<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen DataExchange Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../stylesheets/brand2.css" rel="stylesheet" /><link href="../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a href="/" class="navbar-brand">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a href="http://www.clinicalgenome.org" target="_blank"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>

        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">ACMG Criterion Value Set</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../">Interpretation Model</a></li>
<li><a href="./">Terminologies</a></li>

              </ol>
                <blockquote><p>The ACMG-AMP IVS variant pathogenicity rule or criterion codes including extended codes used in domain specific derivatives of the original guideline. (e.g. BA1, PS2, PM3, PVS1, ...)</p>
                <STRONG>
                  <div>SEPIO:0000395</div>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
             
   
    <!-- list the high level attributes here -->
    <h3>Summary</h3>
    <dl class="dl-horizontal">
      <dt>IRI</dt>
      <dd>SEPIO:0000395</dd>
      <dt>Name</dt>
      <dd>ACMG Criterion Value Set</dd>
        <dt>Description</dt>
        <dd>The ACMG-AMP IVS variant pathogenicity rule or criterion codes including extended codes used in domain specific derivatives of the original guideline. (e.g. BA1, PS2, PM3, PVS1, ...)</dd>
    </dl>
    
    <hr />
    <h3>Included Codes</h3>
    <p>This value set includes codes from the following identifier systems:</p>
    <ul class="list-unstyled none">
        <li>Include any concept from <code>http://purl.obolibrary.org/obo/sepio-clingen#</code> ( ClinGen extensions to SEPIO )</li>
    </ul>
    <hr />
    <!-- build the expanded table if any concepts exist -->
    <h3>Expansion</h3>
    <p>This value set is <em>extensible</em> and therefore 
    can be extended to include concepts from the listed identifier systems by adding on the fly concepts defined by a sending system if no viable option from any of the identifier systems is acceptable.
    </p> 

    <p>This value set contains 28 concepts. </p>
    <table class="table table-hover table-striped table-bordered">
        <thead>
        <th>IRI</th>
        <th>Label</th>
        <th>Description</th>
        </thead>
        <tr>
            <td>SEPIO-CG:99022</td>
            <td>PVS1</td>
            <td>null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99023</td>
            <td>PS1</td>
            <td>Same amino acid change as a previously established pathogenic variant regardless of nucleotide change</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99024</td>
            <td>PS2</td>
            <td>De novo (both maternity and paternity confirmed) in a patient with the disease and no family history</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99025</td>
            <td>PS3</td>
            <td>Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99026</td>
            <td>PS4</td>
            <td>The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99027</td>
            <td>PM1</td>
            <td>Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99028</td>
            <td>PM2</td>
            <td>Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99029</td>
            <td>PM3</td>
            <td>For recessive disorders, detected in trans with a pathogenic variant</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99030</td>
            <td>PM4</td>
            <td>Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99031</td>
            <td>PM5</td>
            <td>Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99032</td>
            <td>PM6</td>
            <td>Assumed de novo, but without confirmation of paternity and maternity</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99033</td>
            <td>PP1</td>
            <td>Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99034</td>
            <td>PP2</td>
            <td>Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99035</td>
            <td>PP3</td>
            <td>Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99036</td>
            <td>PP4</td>
            <td>Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99037</td>
            <td>PP5</td>
            <td>Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99038</td>
            <td>BA1</td>
            <td>Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99039</td>
            <td>BS1</td>
            <td>Allele frequency is greater than expected for disorder</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99040</td>
            <td>BS2</td>
            <td>Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99041</td>
            <td>BS3</td>
            <td>Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99042</td>
            <td>BS4</td>
            <td>Lack of segregation in affected members of a family</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99043</td>
            <td>BP1</td>
            <td>Missense variant in a gene for which primarily truncating variants are known to cause disease</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99044</td>
            <td>BP2</td>
            <td>Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99045</td>
            <td>BP3</td>
            <td>In-frame deletions/insertions in a repetitive region without a known function</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99046</td>
            <td>BP4</td>
            <td>Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99047</td>
            <td>BP5</td>
            <td>Variant found in a case with an alternate molecular basis for disease</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99048</td>
            <td>BP6</td>
            <td>Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation</td>
        </tr>
        <tr>
            <td>SEPIO-CG:99049</td>
            <td>BP7</td>
            <td>A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved</td>
        </tr>
    </table>
    
          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../">Model Overview</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../getting_started.html">Getting Started</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../deeper_dive.html">A Deeper Dive</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../faq.html">FAQ</a></h5>
              </li>
              <li class='list-group-item'><h5><a href="../entities/">Entities</a></h5></li>
              <li class="list-group-item"><h5><a href="./">Terminologies</a></h5><ul class='list-unstyled'><li class="active"><a href="ACMGCriterionValueSet.html"></a></li><li><a href="ACMGEvidenceStrengthValueSet.html"></a></li><li><a href="AlleleComplexityValueSet.html"></a></li><li><a href="AlleleConservationValueSet.html"></a></li><li><a href="AlleleFunctionalAssayValueSet.html"></a></li><li><a href="AlleleNameTypeValueSet.html"></a></li><li><a href="AlleleOriginValueSet.html"></a></li><li><a href="AllelicPhaseValueSet.html"></a></li><li><a href="AllelicStateValueSet.html"></a></li><li><a href="AscertainmentMethodValueSet.html"></a></li><li><a href="BenignMissenseVariationRateOutcomeValueSet.html"></a></li><li><a href="CanonicalAlleleTypeValueSet.html"></a></li><li><a href="ChromosomeValueSet.html"></a></li><li><a href="ClinicalSignificanceValueSet.html"></a></li><li><a href="ConditionMechanismStrengthValueSet.html"></a></li><li><a href="ConditionMechanismValueSet.html"></a></li><li><a href="ConditionPenetranceValueSet.html"></a></li><li><a href="ConditionStatusValueSet.html"></a></li><li><a href="ContextualAlleleTypeValueSet.html"></a></li><li><a href="ContributoryRoleValueSet.html"></a></li><li><a href="CriterionAssessmentOutcomeValueSet.html"></a></li><li><a href="DiseaseValueSet.html"></a></li><li><a href="GeneValueSet.html"></a></li><li><a href="InSilicoPredictionMethodValueSet.html"></a></li><li><a href="InconsistentSegregationObservedValueSet.html"></a></li><li><a href="InconsistentSegregationStatusValueSet.html"></a></li><li><a href="InheritancePatternValueSet.html"></a></li><li><a href="LocusSpecificityValueSet.html"></a></li><li><a href="MolecularConsequenceValueSet.html"></a></li><li><a href="NullAlleleOutcomeValueSet.html"></a></li><li><a href="ParentalConfirmationValueSet.html"></a></li><li><a href="PhenotypeValueSet.html"></a></li><li><a href="PlaceholderEmptyValueSet.html"></a></li><li><a href="PopulationValueSet.html"></a></li><li><a href="ReferenceGenomeBuildValueSet.html"></a></li><li><a href="ReferenceSequenceValueSet.html"></a></li><li><a href="RegionAllelesOutcomeValueSet.html"></a></li><li><a href="RegionAnnotationValueSet.html"></a></li><li><a href="ValueSetExtensibilityValueSet.html"></a></li><li><a href="VariantInterpretationCriterionScoringRulesValueSet.html"></a></li><li><a href="VariantInterpretationGuidelineValueSet.html"></a></li></ul></li>
              <li class="list-group-item">
                <h5><a href="../tools.html">Tools</a></h5>
              </li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2018 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../javascripts/jquery.js"></script><script src="../javascripts/bootstrap.js"></script><script src="../javascripts/jquery.jsonview.js"></script><script src="../javascripts/all.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
