<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      Terminologies
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../stylesheets/brand2.css" rel="stylesheet" /><link href="../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a href="/" class="navbar-brand">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a href="http://www.clinicalgenome.org" target="_blank"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>

        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">Terminologies</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../">Interpretation Model</a></li>
<li><a href="./">Terminologies</a></li>

              </ol>
                <blockquote><p>The description of the controlled sets of terms also known as Value Sets and their sources - Identifer Systems. These value sets and their controlled concept list are required to support the bindings to various attributes for entities used throughout the model.</p>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            <ul class="nav nav-tabs" id="myTabs" role="tablist">
  <li role="presentation" class="active">
    <a href="#value_sets" role="tab" id="value_sets-tab" data-toggle="tab" aria-controls="value_sets" aria-expanded="false">Value Sets</a>
  </li>
  <li role="presentation" class="">
    <a href="#identifier_systems" id="identifier_systems-tab" role="tab" data-toggle="tab" aria-controls="identifier_systems" aria-expanded="true">Identifier Systems</a>
  </li> 
</ul>
<div class="tab-content">
  <div class="tab-pane fade active in" role="tabpanel" id="value_sets" aria-labelledby="value_sets-tab">
    <div class="col-sm-12">
      <h3>Value Sets</h3>
      <p>This table contains a list of all the value sets defined as part of the Variant Pathogenicity Interpretation specification. Some of these value sets include concepts defined elsewhere, some contain their own inline identifier system definitions, and some do both. </p>
      <table class="table table-hover table-striped table-bordered">
        <thead>
          <th>Name</th>
          <th>IRI</th>
          <th>Description</th>
          <th>Source</th>
        </thead>
        <tr>
          <td><a href="ValueSetExtensibilityValueSet.html" target="_blank">Value Set Extensibility Value Set</a></td>
          <td>SEPIO:0000363</td>
          <td>Whether or not a ValueSet can be extended by users</td>
          <td>LOINC<br />SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="MolecularConsequenceValueSet.html" target="_blank">Molecular Consequence Value Set</a></td>
          <td>SEPIO:0000380</td>
          <td>A classification of a genomic variant based on its effect on a transcript or its location with respect to the transcript.</td>
          <td>SO</td>
        </tr>
        <tr>
          <td><a href="ClinicalSignificanceValueSet.html" target="_blank">Clinical Significance Value Set</a></td>
          <td>SEPIO:0000371</td>
          <td>The role that a variant plays in the causing a genetic condition
(see LOINC answer list https://r.details.loinc.org/AnswerList/LL4034-6.html)</td>
          <td>LOINC</td>
        </tr>
        <tr>
          <td><a href="RegionAnnotationValueSet.html" target="_blank">Region Annotation Value Set</a></td>
          <td>SEPIO:0000343</td>
          <td>A label applied to a sequence region</td>
          <td>SO<br />SEPIO</td>
        </tr>
        <tr>
          <td><a href="AllelicPhaseValueSet.html" target="_blank">Allelic Phase Value Set</a></td>
          <td>SEPIO:0000344</td>
          <td>Value denoting whether sequence features lie on the same or different molecules</td>
          <td>GENO</td>
        </tr>
        <tr>
          <td><a href="AllelicStateValueSet.html" target="_blank">Allelic State Value Set</a></td>
          <td>SEPIO:0000338</td>
          <td>Describes the existence of an allele across multiple copies of a chromosome
(Informed by LOINC answer list https://r.details.loinc.org/AnswerList/LL381-5.html but with additional codes for somatic mosaicism and asserted absence.)</td>
          <td>GENO</td>
        </tr>
        <tr>
          <td><a href="InheritancePatternValueSet.html" target="_blank">Inheritance Pattern Value Set</a></td>
          <td>SEPIO:0000416</td>
          <td>Classifies the transmission pattern of a condition in a pedigree.
(see LOINC Answer List https://r.details.loinc.org/AnswerList/LL3731-8.html)</td>
          <td>HP<br />GENO</td>
        </tr>
        <tr>
          <td><a href="ConditionMechanismValueSet.html" target="_blank">Condition Mechanism Value Set</a></td>
          <td>SEPIO-CG:65108</td>
          <td>The particular type of allele that can lead to a condition</td>
          <td>SO</td>
        </tr>
        <tr>
          <td><a href="InSilicoPredictionMethodValueSet.html" target="_blank">In Silico Prediction Method Value Set</a></td>
          <td>SEPIO:0000345</td>
          <td>The type of InSilico Predictor</td>
          <td>SEPIO</td>
        </tr>
        <tr>
          <td><a href="CriterionAssessmentOutcomeValueSet.html" target="_blank">Criterion Assessment Outcome Value Set</a></td>
          <td>SEPIO:0000347</td>
          <td>Denotes whether the collected evidence is sufficient to meet a specified criteria or not</td>
          <td>SEPIO</td>
        </tr>
        <tr>
          <td><a href="AlleleFunctionalAssayValueSet.html" target="_blank">Allele Functional Assay Value Set</a></td>
          <td>SEPIO:0000382</td>
          <td>The functional measurement carried out in a given assay</td>
          <td>OBI</td>
        </tr>
        <tr>
          <td><a href="PopulationValueSet.html" target="_blank">Population Value Set</a></td>
          <td>SEPIO-CG:65112</td>
          <td>Label for a group of individuals with a shared genetic background</td>
          <td>NCIT</td>
        </tr>
        <tr>
          <td><a href="BenignMissenseVariationRateOutcomeValueSet.html" target="_blank">Benign Missense Variation Rate Outcome Value Set</a></td>
          <td>SEPIO:0000349</td>
          <td>The rate of missense variation in a transcript that do not lead to a genetic condition</td>
          <td>SEPIO</td>
        </tr>
        <tr>
          <td><a href="ConditionPenetranceValueSet.html" target="_blank">Condition Penetrance Value Set</a></td>
          <td>SEPIO-CG:65114</td>
          <td>The chance of having a particular genetic condition given the presence of a particular variant or damaged gene</td>
          <td>HP</td>
        </tr>
        <tr>
          <td><a href="ConditionMechanismStrengthValueSet.html" target="_blank">Condition Mechanism Strength Value Set</a></td>
          <td>SEPIO:0000354</td>
          <td>The strength of association between a genetic mechanism and a condition</td>
          <td>SEPIO</td>
        </tr>
        <tr>
          <td><a href="ACMGEvidenceStrengthValueSet.html" target="_blank">ACMG Evidence Strength Value Set</a></td>
          <td>SEPIO:0000353</td>
          <td>Degree of support that a set of evidence lends to a subsequent statement</td>
          <td>SEPIO</td>
        </tr>
        <tr>
          <td><a href="DiseaseValueSet.html" target="_blank">Disease Value Set</a></td>
          <td>SEPIO:0000372</td>
          <td>An alteration of health status resulting from a physiopathological mechanism, and having a homogeneous clinical presentation and evolution and homogeneous therapeutic possibilities. (From MONDO)</td>
          <td>OMIM<br />Orphanet<br />DOID<br />MESH<br />MONDO</td>
        </tr>
        <tr>
          <td><a href="PhenotypeValueSet.html" target="_blank">Phenotype Value Set</a></td>
          <td>SEPIO:0000375</td>
          <td>The assemblage of traits or outward appearance of an individual. It is the product of interactions between genes and between genes and the environment. (From NCIT)</td>
          <td>HP</td>
        </tr>
        <tr>
          <td><a href="ReferenceSequenceValueSet.html" target="_blank">Reference Sequence Value Set</a></td>
          <td>SEPIO-CG:65119</td>
          <td>TODO</td>
          <td>REFSEQ<br />ENSEMBL</td>
        </tr>
        <tr>
          <td><a href="ContributoryRoleValueSet.html" target="_blank">Contributory Role Value Set</a></td>
          <td>SEPIO-CG:65120</td>
          <td>The role played by an agent in the creation of an artifact</td>
          <td>SEPIO</td>
        </tr>
        <tr>
          <td><a href="AlleleOriginValueSet.html" target="_blank">Allele Origin Value Set</a></td>
          <td>SEPIO:0000392</td>
          <td>The source of an allele in an individual</td>
          <td>GENO</td>
        </tr>
        <tr>
          <td><a href="AscertainmentMethodValueSet.html" target="_blank">Ascertainment Method Value Set</a></td>
          <td>SEPIO:0000409</td>
          <td>The method by which a set of people was chosen to define a group</td>
          <td>SEPIO</td>
        </tr>
        <tr>
          <td><a href="LocusSpecificityValueSet.html" target="_blank">Locus Specificity Value Set</a></td>
          <td>SEPIO:0000348</td>
          <td>The degree to which a condition implicates a specific genomic locus, such as a gene.</td>
          <td>SEPIO<br />SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="AlleleConservationValueSet.html" target="_blank">Allele Conservation Value Set</a></td>
          <td>SEPIO-CG:65124</td>
          <td>The degree to which an allele is preserved over evolutionary history</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="ParentalConfirmationValueSet.html" target="_blank">Parental Confirmation Value Set</a></td>
          <td>SEPIO-CG:65125</td>
          <td>Whether or not parentage has been confirmed with a genetic test</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="ConditionStatusValueSet.html" target="_blank">Condition Status Value Set</a></td>
          <td>SEPIO-CG:65126</td>
          <td>Whether or not an individual is affected by a specific genetic condition</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="InconsistentSegregationObservedValueSet.html" target="_blank">Inconsistent Segregation Observed Value Set</a></td>
          <td>SEPIO-CG:65127</td>
          <td>Whether or not inconsistent segretations have been observed in a pedigree</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="NullAlleleOutcomeValueSet.html" target="_blank">Null Allele Outcome Value Set</a></td>
          <td>SEPIO-CG:65128</td>
          <td>Denotes whether an allele leads to a non-functional transcript</td>
          <td>SO</td>
        </tr>
        <tr>
          <td><a href="RegionAllelesOutcomeValueSet.html" target="_blank">Region Alleles Outcome Value Set</a></td>
          <td>SEPIO-CG:65129</td>
          <td>Whether or not an allele overlaps a sequence region</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="GeneValueSet.html" target="_blank">Gene Value Set</a></td>
          <td>SEPIO:0000396</td>
          <td>TODO</td>
          <td>HGNC</td>
        </tr>
        <tr>
          <td><a href="ACMGCriterionValueSet.html" target="_blank">ACMG Criterion Value Set</a></td>
          <td>SEPIO:0000395</td>
          <td>TODO</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="VariantInterpretationGuidelineValueSet.html" target="_blank">Variant Interpretation Guideline Value Set</a></td>
          <td>SEPIO-CG:65132</td>
          <td>TODO</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="VariantInterpretationCriterionScoringRulesValueSet.html" target="_blank">Variant Interpretation Criterion Scoring Rules Value Set</a></td>
          <td>SEPIO-CG:65133</td>
          <td>This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance.</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="InconsistentSegregationStatusValueSet.html" target="_blank">Inconsistent Segregation Status Value Set</a></td>
          <td>SEPIO-CG:65140</td>
          <td>TODO</td>
          <td>SEPIO</td>
        </tr>
        <tr>
          <td><a href="PlaceholderEmptyValueSet.html" target="_blank">Placeholder Empty Value Set</a></td>
          <td>SEPIO-CG:65900</td>
          <td>TODO</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="CanonicalAlleleTypeValueSet.html" target="_blank">Canonical Allele Type Value Set</a></td>
          <td>SEPIO-CG:65901</td>
          <td>TODO</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="AlleleComplexityValueSet.html" target="_blank">Allele Complexity Value Set</a></td>
          <td>SEPIO-CG:65902</td>
          <td>TODO</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="AlleleNameTypeValueSet.html" target="_blank">Allele Name Type Value Set</a></td>
          <td>SEPIO-CG:65903</td>
          <td>TODO</td>
          <td>SEPIO-CG</td>
        </tr>
        <tr>
          <td><a href="ContextualAlleleTypeValueSet.html" target="_blank">Contextual Allele Type Value Set</a></td>
          <td>SEPIO-CG:65904</td>
          <td>TODO</td>
          <td>SEPIO-CG</td>
        </tr>
      </table>
    </div>
  </div>
  <div class="tab-pane fade in" role="tabpanel" id="identifier_systems" aria-labelledby="identifier_systems-tab">
    <div class="col-sm-12">
      <h3>Identifier Systems</h3>
      <p>This table contains a list of all the Identifier Systems referenced by the Value Sets and by the model entity and attributes themselves.</p>
      <table class="table table-hover table-striped table-bordered">
        <thead>
          <th>Name</th>
          <th><a href="https://en.wikipedia.org/wiki/CURIE" target="_blank">CURIE</a></th>
          <th><a href="https://en.wikipedia.org/wiki/Internationalized_Resource_Identifier" target="_blank">IRI</a></th>
          <th>Description</th>
        </thead>
        <tr>
          <td>ClinGen Allele Registry</td>
          <td>CAR</td>
          <td><a href="http://reg.genome.network/allele/" target="_blank">http://reg.genome.network/allele/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>LOINC</td>
          <td>LOINC</td>
          <td><a href="http://loinc.org/" target="_blank">http://loinc.org/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>HGNC</td>
          <td>HGNC</td>
          <td><a href="http://www.genenames.org/" target="_blank">http://www.genenames.org/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>RefSeq</td>
          <td>REFSEQ</td>
          <td><a href="http://www.ncbi.nlm.nih.gov/nuccore/" target="_blank">http://www.ncbi.nlm.nih.gov/nuccore/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>ClinVar</td>
          <td>CLINVAR</td>
          <td><a href="http://www.ncbi.nlm.nih.gov/clinvar/" target="_blank">http://www.ncbi.nlm.nih.gov/clinvar/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Ensembl</td>
          <td>ENSEMBL</td>
          <td><a href="http://www.ensembl.org/" target="_blank">http://www.ensembl.org/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Sequence Ontology</td>
          <td>SO</td>
          <td><a href="http://purl.obolibrary.org/obo/so#" target="_blank">http://purl.obolibrary.org/obo/so#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>dbSNP</td>
          <td>DBSNP</td>
          <td><a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" target="_blank">http://www.ncbi.nlm.nih.gov/projects/SNP/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>HGVS</td>
          <td>HGVS</td>
          <td><a href="http://www.hgvs.org/mutnomen/" target="_blank">http://www.hgvs.org/mutnomen/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>LRG</td>
          <td>LRG</td>
          <td><a href="http://www.lrg-sequence.org/" target="_blank">http://www.lrg-sequence.org/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>OMIM</td>
          <td>OMIM</td>
          <td><a href="http://www.omim.org/" target="_blank">http://www.omim.org/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Pubmed</td>
          <td>PMID</td>
          <td><a href="http://www.ncbi.nlm.nih.gov/pubmed/" target="_blank">http://www.ncbi.nlm.nih.gov/pubmed/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Orphanet</td>
          <td>Orphanet</td>
          <td><a href="http://www.orpha.net/" target="_blank">http://www.orpha.net/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>ExAC/gnomad Populations</td>
          <td>GNOMAD</td>
          <td><a href="http://gnomad.broadinstitute.org/" target="_blank">http://gnomad.broadinstitute.org/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Human Phenotype Ontology</td>
          <td>HP</td>
          <td><a href="http://purl.obolibrary.org/obo/hp#" target="_blank">http://purl.obolibrary.org/obo/hp#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Relations Ontology</td>
          <td>RO</td>
          <td><a href="http://purl.obolibrary.org/obo/ro#" target="_blank">http://purl.obolibrary.org/obo/ro#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Int'l Genome Sample Resource populations</td>
          <td>IGSR</td>
          <td><a href="http://www.internationalgenome.org/" target="_blank">http://www.internationalgenome.org/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Disease Ontology</td>
          <td>DOID</td>
          <td><a href="http://purl.obolibrary.org/obo/doid#" target="_blank">http://purl.obolibrary.org/obo/doid#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Medical Subject Headers</td>
          <td>MESH</td>
          <td><a href="https://meshb.nlm.nih.gov/" target="_blank">https://meshb.nlm.nih.gov/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Exome Variant Server/ESP populations</td>
          <td>EVS</td>
          <td><a href="http://evs.gs.washington.edu/EVS/" target="_blank">http://evs.gs.washington.edu/EVS/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>GENO</td>
          <td>GENO</td>
          <td><a href="http://purl.obolibrary.org/obo/geno#" target="_blank">http://purl.obolibrary.org/obo/geno#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Scientific Evidence Provenance Information Ontology</td>
          <td>SEPIO</td>
          <td><a href="http://purl.obolibrary.org/obo/sepio#" target="_blank">http://purl.obolibrary.org/obo/sepio#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Monarch Initiative Disease Ontology</td>
          <td>MONDO</td>
          <td><a href="http://purl.obolibrary.org/obo/mondo#" target="_blank">http://purl.obolibrary.org/obo/mondo#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>ClinGen extensions to SEPIO</td>
          <td>SEPIO-CG</td>
          <td><a href="http://purl.obolibrary.org/obo/sepio-clingen#" target="_blank">http://purl.obolibrary.org/obo/sepio-clingen#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>OBI</td>
          <td>OBI</td>
          <td><a href="http://purl.obolibrary.org/obo/obi#" target="_blank">http://purl.obolibrary.org/obo/obi#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>Information Artifact Ontology</td>
          <td>IAO</td>
          <td><a href="http://purl.obolibrary.org/obo/iao#" target="_blank">http://purl.obolibrary.org/obo/iao#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>NCI Thesarus</td>
          <td>NCIT</td>
          <td><a href="http://purl.obolibrary.org/obo/ncit#" target="_blank">http://purl.obolibrary.org/obo/ncit#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>BFO</td>
          <td>BFO</td>
          <td><a href="http://purl.obolibrary.org/obo/bfo#" target="_blank">http://purl.obolibrary.org/obo/bfo#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>PCO</td>
          <td>PCO</td>
          <td><a href="http://purl.obolibrary.org/obo/pco#" target="_blank">http://purl.obolibrary.org/obo/pco#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>STATO</td>
          <td>STATO</td>
          <td><a href="http://purl.obolibrary.org/obo/stato#" target="_blank">http://purl.obolibrary.org/obo/stato#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>ERO</td>
          <td>ERO</td>
          <td><a href="http://purl.obolibrary.org/obo/ero#" target="_blank">http://purl.obolibrary.org/obo/ero#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>FALDO</td>
          <td>FALDO</td>
          <td><a href="http://biohackathon.org/resource/faldo#" target="_blank">http://biohackathon.org/resource/faldo#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>DC</td>
          <td>DC</td>
          <td><a href="http://purl.org/dc/elements/1.1/" target="_blank">http://purl.org/dc/elements/1.1/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>RDF</td>
          <td>RDFS</td>
          <td><a href="http://www.w3.org/2000/01/rdf-schema#" target="_blank">http://www.w3.org/2000/01/rdf-schema#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>W3C Provenance</td>
          <td>PROV</td>
          <td><a href="http://www.w3.org/ns/prov#" target="_blank">http://www.w3.org/ns/prov#</a></td>
          <td></td>
        </tr>
        <tr>
          <td>PAV</td>
          <td>PAV</td>
          <td><a href="http://purl.org/pav/" target="_blank">http://purl.org/pav/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>ClinGen Modeling Temporary Namespace</td>
          <td>CG</td>
          <td><a href="http://dataexchange.clinicalgenome.org/temp/" target="_blank">http://dataexchange.clinicalgenome.org/temp/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>ClinGen Modeling Example Namespace</td>
          <td>CGEX</td>
          <td><a href="http://dataexchange.clinicalgenome.org/examples/" target="_blank">http://dataexchange.clinicalgenome.org/examples/</a></td>
          <td></td>
        </tr>
        <tr>
          <td>FOAF Ontology</td>
          <td>FOAF</td>
          <td><a href="http://xmlns.com/foaf/0.1/" target="_blank">http://xmlns.com/foaf/0.1/</a></td>
          <td></td>
        </tr>
      </table>
    </div>
  </div>
</div>

          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../">Model Overview</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../getting_started.html">Getting Started</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../deeper_dive.html">A Deeper Dive</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../faq.html">FAQ</a></h5>
              </li>
              <li class='list-group-item'><h5><a href="../entities/">Entities</a></h5></li>
              <li class="list-group-item"><h5><a href="./">Terminologies</a></h5><ul class='list-unstyled'><li><a href="ACMGCriterionValueSet.html"></a></li>
<li><a href="ACMGEvidenceStrengthValueSet.html"></a></li>
<li><a href="AlleleComplexityValueSet.html"></a></li>
<li><a href="AlleleConservationValueSet.html"></a></li>
<li><a href="AlleleFunctionalAssayValueSet.html"></a></li>
<li><a href="AlleleNameTypeValueSet.html"></a></li>
<li><a href="AlleleOriginValueSet.html"></a></li>
<li><a href="AllelicPhaseValueSet.html"></a></li>
<li><a href="AllelicStateValueSet.html"></a></li>
<li><a href="AscertainmentMethodValueSet.html"></a></li>
<li><a href="BenignMissenseVariationRateOutcomeValueSet.html"></a></li>
<li><a href="CanonicalAlleleTypeValueSet.html"></a></li>
<li><a href="ClinicalSignificanceValueSet.html"></a></li>
<li><a href="ConditionMechanismStrengthValueSet.html"></a></li>
<li><a href="ConditionMechanismValueSet.html"></a></li>
<li><a href="ConditionPenetranceValueSet.html"></a></li>
<li><a href="ConditionStatusValueSet.html"></a></li>
<li><a href="ContextualAlleleTypeValueSet.html"></a></li>
<li><a href="ContributoryRoleValueSet.html"></a></li>
<li><a href="CriterionAssessmentOutcomeValueSet.html"></a></li>
<li><a href="DiseaseValueSet.html"></a></li>
<li><a href="GeneValueSet.html"></a></li>
<li><a href="InSilicoPredictionMethodValueSet.html"></a></li>
<li><a href="InconsistentSegregationObservedValueSet.html"></a></li>
<li><a href="InconsistentSegregationStatusValueSet.html"></a></li>
<li><a href="InheritancePatternValueSet.html"></a></li>
<li><a href="LocusSpecificityValueSet.html"></a></li>
<li><a href="MolecularConsequenceValueSet.html"></a></li>
<li><a href="NullAlleleOutcomeValueSet.html"></a></li>
<li><a href="ParentalConfirmationValueSet.html"></a></li>
<li><a href="PhenotypeValueSet.html"></a></li>
<li><a href="PlaceholderEmptyValueSet.html"></a></li>
<li><a href="PopulationValueSet.html"></a></li>
<li><a href="ReferenceSequenceValueSet.html"></a></li>
<li><a href="RegionAllelesOutcomeValueSet.html"></a></li>
<li><a href="RegionAnnotationValueSet.html"></a></li>
<li><a href="ValueSetExtensibilityValueSet.html"></a></li>
<li><a href="VariantInterpretationCriterionScoringRulesValueSet.html"></a></li>
<li><a href="VariantInterpretationGuidelineValueSet.html"></a></li>
</ul></li>
              <li class="list-group-item">
                <h5><a href="../tools.html">Tools</a></h5>
              </li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          Â© 2018 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../javascripts/jquery.js"></script><script src="../javascripts/bootstrap.js"></script><script src="../javascripts/jquery.jsonview.js"></script><script src="../javascripts/all.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
