<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a class="navbar-brand" href="/">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a target="_blank" href="http://www.clinicalgenome.org"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>
    


    
        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">ScoringAlgorithm</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../../">Interpretation Model</a></li>
<li><a href="../">Technical Reference and Examples</a></li>
<li><a href="./">Domain Entities</a></li>

              </ol>
                <blockquote><p>A specification for how to collectively evaluate the outcomes of one or more variant pathogenicity criterion assessments as evidence, to make an assertion about the pathogenicity of a variant.</p>
                <STRONG>
                  <div>SEPIO:0000224</div>
                      <i>Descended from DomainEntity</i>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>An AssertionMethod is composed of multiple Criteria. Each Criteria is assessed against a set of evidence Statements; the assessment lends support for or against an interpretation.   From this set of multiple assessments, a final interpretation is created.  A ScoringAlgorithm describes the how a set of assessments is combined or reconciled to produce an overall interpretation.</p>

<p>ScoringAlgorithms are properties of AssertionMethods.  A ScoringAlgorithm describes the algorithm, rather than an implementation of that algorithm.  In other words, the ScoringAlgorithm for the ACMG Variant Pathogenicity Guidelines is the algorithm described in text and figures in the guideline paper, rather than e.g. the Baylor Pathogenicity Calculator.  The use of a particular implementation can be denoted using a Contribution on the interpretation.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>description</td>
                <td>String</td>
                <td>0..1</td>
                <td></td>
                <td>DC:description</td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>description</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td></td>
                    <td></td>
                    <td></td>
                </tr>
                <tr>
                    <td>label</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A freeform text (preferred) representation used by the authoring system to identify a concept.</td>
                    <td></td>
                </tr>
                <tr>
                    <td>synonym</td>
                    <td>Synonym</td>
                    <td>0..*</td>
                    <td></td>
                    <td>Codings thtat identify the concept. If more than one then the codings are typically considered equivalent. (FHIR offers additional attributes to qualify the codings further if needed) </td>
                    <td></td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>description</th>
        <th>label</th>
        <th>synonym</th>
        <th>description</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../details/details.html#ScorAlg332">ScorAlg332</a></p></td>
          <td>
              <p>This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance.</p>
         </td>
          <!--
          <td><div class="example_view" id="ScorAlg332"><pre><code>{
  "id": "ScorAlg332",
  "type": "ScoringAlgorithm",
  "description": "This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance."
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../../">Interpretation Overview</a></h5>
              </li>
              <li class='list-group-item'><h5><a href="../../user/">User Guide</a></h5></li>
              <li class="list-group-item"><h5><a href="../">Technical Reference</a></h5><ul class='list-unstyled'><li><a href="../core/">Core</a></li><li class="active"><a href="./">Domain Entities</a></li><ul class='list-unstyled'><li><a href="alleleInheritance_type">AlleleInheritanceType</a></li><li><a href="allele_functional_impact_type.html">AlleleFunctionalImpactType</a></li><li><a href="allelic_state.html">AllelicState</a></li><li><a href="ascertainment_type.html">AscertainmentType</a></li><li><a href="assertion_method.html">AssertionMethod</a></li><li><a href="benign_missense_variation_rate.html">BenignMissenseVariationRate</a></li><li><a href="condition_mechanism_strength.html">ConditionMechanismStrength</a></li><li><a href="condition_status.html">ConditionStatus</a></li><li><a href="conservation.html">Conservation</a></li><li><a href="contributory_role.html">ContributoryRole</a></li><li><a href="criterion_evaluation.html">CriterionEvaluation</a></li><li><a href="disease.html">Disease</a></li><li><a href="evidence_line_strength.html">EvidenceLineStrength</a></li><li><a href="family.html">Family</a></li><li><a href="gene.html">Gene</a></li><li><a href="inconsistent_segregation_observed">InconsistentSegregationObserved</a></li><li><a href="individual.html">Individual</a></li><li><a href="locus_specificity_type.html">LocusSpecificityType</a></li><li><a href="mechanism_type.html">MechanismType</a></li><li><a href="mode_of_inheritance.html">ModeOfInheritance</a></li><li><a href="molecular_consequence.html">MolecularConsequenceType</a></li><li><a href="null_allele.html">NullAllele</a></li><li><a href="parental_confirmation.html">ParentalConfirmation</a></li><li><a href="penetrance_type.html">PenetranceType</a></li><li><a href="phase.html">Phase</a></li><li><a href="phenotype.html">Phenotype</a></li><li><a href="population_type.html">PopulationType</a></li><li><a href="prediction_type.html">PredictionType</a></li><li><a href="reference_sequence.html">ReferenceSequence</a></li><li><a href="reference_sequence_relatedtype.html">ReferenceSequenceRelatedType</a></li><li><a href="region_alleles_outcome.html">RegionAllelesOutcome</a></li><li><a href="region_annotation_type.html">RegionAnnotationType</a></li><li class="active"><a href="scoring_algorithm.html">ScoringAlgorithm</a></li><li><a href="significance.html">Significance</a></li></ul><li><a href="../domain_evidence/">Domain Evidence</a></li><li><a href="../provenance/">Provenance</a></li><li><a href="../terminology/">Terminology</a></li></ul></li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2017 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../javascripts/jquery.js"></script><script src="../../javascripts/bootstrap.js"></script><script src="../../javascripts/jquery.jsonview.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
