<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      User Guide
    </title>

	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../stylesheets/brand2.css" rel="stylesheet" /><link href="../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/user/">0.1</a></li>
                  <li><a href="/development/user/">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../">Interpretation Model</a></li>
<li><a href="./">User Guide</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
         <div class='nav-sidebar nav-sidebar-title'>
<a href="https://clinicalgenome.org">            	<img width="80%" height="80%" alt="ClinGen Documentation" class="img-responsive" src="../images/clingen-doc-logo.png" />
</a>        	</div>
          <ul class="nav nav-sidebar">
            
            <li class="active">
              <a href="../">Welcome</a>
            </li>
            <li class="active"><a href="./">User Guide</a></li><ul><li><a href="sepio.html">SEPIO</a></li>
<li><a href="supported_queries.html">Supported queries</a></li>
<li><a href="w3c-prov.html">W3C-Prov</a></li>
</ul>
            <li><a href="../tech/">Technical Reference</a></li>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>User Guide</h1>
            <p>The overall structure of ClinGen interpretations, and the use of SEPIO.</p>


            <hr class='col-sm-12' />


            <p>This guide provides background, use cases, examples, future planning and
collaborations surrounding ClinGen's Variant Interpretation Model. This portion
of the documentation is targeted at the health care providers, scientists,
geneticists, counselors, and researchers that work in or are interested in
curating and interpreting variants.</p>

<h2 id="introduction">Introduction</h2>

<p>An interpretation is the outcome of structured reasoning applied to evidence.   Interpretations may be made about many kinds of entities using many kinds of evidence, and different formalized reasoning strategies.  Although the overall structure of an interpretation is the same for many cases, the initial version of the interpretation model is limited to interpretations of the pathogenicity of sequence variants using the <a href="">ACMG Pathogenicity Guidelines</a>.   The discussion below will specifically use this context as an example, but the structure of the model is expected to remain largely unchanged as new interpretation contexts are added.</p>

<p>The process of generating this type of interpretation is composed of three kinds of activities:</p>

<ol>
  <li>Collecting various forms of <strong>evidence</strong> from potentially many sources such as literature or databases</li>
  <li>Application of individual <strong>ACMG Guideline Criteria</strong>, in which subsets of this evidence are combined to produce <strong>assessments</strong> using lines of evidence</li>
  <li>Combination of these assertions to create an overall <strong>interpretation</strong> of the evidence: a statement about the pathogenicity of a given allele.</li>
</ol>

<p>The process is visualized as a tree.  The root of the tree is the interpretation, and it draws upon the conclusions/assessments of individual ACMG criteria, and those in turn depend on the evidence (the leaves of the tree).</p>

<p>An interpretation without the supporting evidence and reasoning is merely an assertion by somebody.  Inclusion of the evidence and reasoning allows the receiver to draw their own conclusions, and to reasonably build upon the work carried out by an interpretations original creators.</p>

<h2 id="sepio">SEPIO</h2>

<p>The interpretation model, therefore must contain not only interpretations, but organized and structured information about the evidence and criteria used to create the interpretation, along with metadata about who performed atomic activities and when.  The Scientific Evidence and Provenance Information Ontology (<a href="sepio.html">SEPIO</a>) provides such a structure.</p>

<h2 id="w3c-prov">W3C-Prov</h2>

<p>The <a href="https://www.w3.org/TR/prov-o/">PROV</a> ontology is another standard for structural
representation of data along with provenance which can provide a more direct record
of the process through which data entities (including assertions) were developed.
Much of our early work in this model was informed by the W3C-Prov working group
recommendations, including the delineation of <em>Entities</em> that are produced by <em>Agents</em>
in the context of <em>Activities</em>.</p>

<p>While we follow SEPIO more closely in terms of our current model, we intend to
maintain a defined mapping between our data representation and one that would
more directly follow PROV-O for the purposes of interoperability.</p>

<h2 id="discussion">Discussion</h2>

<p>The model is loosely coupled; individual entities (assertions or primary data elements) have at least a conceptual existence outside a given interpretation, so that the same piece of evidence that is generated for one interpretation may be retained, along with its provenance, and used in later interpretations.</p>

<p>The tree structure of the model is collapsible.  If a user is interested only in the final interpretation, they need to simply access the root node of the structure.  Suppose however, that a user has a discrepant interpretation.  The initial question in such a case would be "which ACMG criteria were used to generate the interpretation?"  The user can answer this question by moving one level in the graph to see which lines of evidence were used by the interpretation.  Suppose further that most of these assessments concur with the users' but that one is not.  Then the user may inspect the branch of the tree corresponding to that conflicting assessment, allowing inspection of the particular evidence and where that evidence came from.</p>

          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../javascripts/jquery.js"></script><script src="../javascripts/bootstrap.js"></script><script src="../javascripts/jquery.jsonview.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
