<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      TranscriptReferenceCoordinate
    </title>

    <link href="../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../stylesheets/brand2.css" rel="stylesheet" />

    <script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-49947422-3', 'auto');
      ga('send', 'pageview');
    </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">About</a>
              <!-- <a href="/">About</a> -->
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                ::
                <span class="glyphicon glyphicon-leaf"></span>Allele
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                <li class="allele" >
                  <li class="allele"><a href="../../"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                </li>
                <li class="assertion" >
                  <li class="assertion"><a href="../../../assertion/"><span class="glyphicon glyphicon-star"></span>Assertion</a></li>
                </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >0.1 release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                <li><a href="/0.1/allele/conceptual/simple_allele/transcript_reference_coordinate.html">0.1</a></li>
                <li><a href="/development/allele/conceptual/simple_allele/transcript_reference_coordinate.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../../../">ClinGen Data Model</a></li>
<li><a href="../../">Allele Model</a></li>
<li><a href="../">Conceptual</a></li>
<li><a href="./">SimpleAllele</a></li>
<li><a href="transcript_reference_coordinate.html">TranscriptReferenceCoordinate</a></li>

        </ol>
      </div>
    </div>

    <div class="container-fluid">
  <div class="row">
    <div class="col-sm-4 col-md-3 sidebar">
      <ul class="nav nav-sidebar">
        <li class='nav-sidebar-title'>
        <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../../../images/clingen-doc-logo.png" />
          <!-- <img src="/images/clingen-doc-logo.png" width="438" height="108" alt="ClinGen Documentation" class="img-responsive"/> -->
        </li>
        <li><a href="../../discussion/">Discussion</a></li>
        <li><a href="../../implementation/">Implementation</a></li>
        <li><a href="../../resource/">Resource</a></li>
        <li class="active"><a href="../">Conceptual</a></li><ul><li><a href="../canonical_allele/">CanonicalAllele</a></li><li><a href="../gene/">Gene</a></li><li><a href="../provenance/">Provenance</a></li><li><a href="../reference_sequence/">ReferenceSequence</a></li><li class="active"><a href="./">SimpleAllele</a></li><ul><li><a href="allele_name.html">AlleleName</a></li><li><a href="amino_acid_reference_coordinate.html">AminoAcidReferenceCoordinate</a></li><li><a href="genomic_reference_coordinate.html">GenomicReferenceCoordinate</a></li><li><a href="intron_offset.html">IntronOffset</a></li><li><a href="nucleotide_reference_coordinate.html">NucleotideReferenceCoordinate</a></li><li><a href="reference_coordinate.html">ReferenceCoordinate</a></li><li><a href="simple_amino_acid_allele.html">SimpleAminoAcidAllele</a></li><li><a href="simple_genomic_allele.html">SimpleGenomicAllele</a></li><li><a href="simple_nucleotide_allele.html">SimpleNucleotideAllele</a></li><li><a href="simple_transcript_allele.html">SimpleTranscriptAllele</a></li><li class="active"><a href="transcript_reference_coordinate.html">TranscriptReferenceCoordinate</a></li></ul></ul>
      </ul>
    </div>
    <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
      <article>
        <h1>TranscriptReferenceCoordinate</h1>
        <hr class='col-sm-12' />
        <!-- <ul class="list-unstyled"> -->
        <!--   <li class='col-sm-6 col-md-4'><a href="#introduction">Introduction</a></li> -->
        <!--   <li class='col-sm-6 col-md-4'><a href="#selects">Selects</a></li> -->
        <!-- </ul> -->
        <!-- <hr class='col-sm-12' id="sticky-nav-anchor" /> -->
        <!-- <div class='row'><div id="sticky-nav"> -->
        <!--     <div class="navbar navbar-default col-sm-12"> -->
        <!--       <ul class="nav navbar-nav"> -->
        <!--         <li class="dropdown"> -->
        <!--           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false"><strong>Query Builder</strong> :: Introduction <span class="caret"></span></a> -->
        <!--           <ul class="dropdown-menu" role="menu"> -->
        <!--             <li><a href="#">Action</a></li> -->
        <!--             <li><a href="#">Another action</a></li> -->
        <!--           </ul> -->
        <!--         </li> -->
        <!--       </ul> -->
        <!--       <form class="navbar-form navbar-right" role="search"> -->
        <!--         <div class="input-group"> -->
        <!--           <div class="input-group-btn"> -->
        <!--             <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-expanded="false">Page <span class="caret"></span></button> -->
        <!--             <ul class="dropdown-menu" role="menu"> -->
        <!--               <li><a href="#">Model</a></li> -->
        <!--               <li><a href="#">Site</a></li> -->
        <!--             </ul> -->

        <!--           </div><\!-- /btn-group -\-> -->
        <!--           <input type="text" class="form-control" aria-label="" placeholder='Search'> -->
        <!--           <span class="input-group-btn"> -->
        <!--             <button class="btn btn-default" type="button">Go!</button> -->
        <!--           </span> -->
        <!--         </div><\!-- /input-group -\-> -->
        <!--       </form> -->
        <!--   </div> </div> -->
        <!-- </div><\!-- /.navbar-collapse -\-> -->
           <h2 id="scope-and-usage">Scope and Usage</h2>

<p><a href="reference_coordinate.html">ReferenceCoordinate</a> representts a particular continguous location at which different alleles may be defined with <a href="simple_allele.html">SimpleAllele</a>.  TranscriptReferenceCoordinate is a <a href="reference_coordinate.html">ReferenceCoordinate</a> where the underlying reference sequence is a <a href="../reference_sequence/transcript_reference_sequence.html">TranscriptReferenceSequence</a>.</p>

<p>TranscriptReferenceCoordinates may refer to either exonic or intronic locations. Because exonic locations are in the <a href="../reference_sequence/transcript_reference_sequence.html">TranscriptReferenceSequence</a>, the coordinate may be defined as described in <a href="reference_coordinate.html">ReferenceCoordinate</a>.</p>

<p>However, intronic locations are not part of the <a href="../reference_sequence/transcript_reference_sequence.html">TranscriptReferenceSequence</a>, and cannot be simply described using a start and end.  In HGVS, intronic variants are described in a transcript by using two integers.  The first is a location in the transcript, which is supposed to be the first or last base in an exon.  The second number is a coordinate into the intron from that location.  This assumes an alignment of this transcript to a particular <a href="genomic_reference_sequence.html">GenomicReferenceSequence</a>.  There may be multiple such alignments to multiple <a href="genomic_reference_sequence.html">GenomicReferenceSequences</a>, though this is not modeled in HGVS expressions.</p>

<p>TranscriptReferenceCoordinate uses a similar system to refer to intronic positions.  Each TranscriptReferenceCoordinate may have one references to an <a href="intron_offset.html">intronOffset</a> entity.  The intronOffset entity contains two integer attributes, which are the intronic offsets that are added to the start and end coordinates of the TranscriptReferenceCoordinate.   In addition, the IntronicCoordinate contains a reference to the particular <a href="genomic_reference_sequence.html">GenomicReferenceSequence</a> where the transcript was mapped.</p>

<p>If an allele occurs in an intron, the allele does not occur in the transcript, but in a genomic sequence; the relation to this sequence is managed by intronOffset.  Note that the transcript may be mapped to multiple genomic sequences.  Rather than allow a multiplicity of intronOffsets for a single SimpleTranscriptAllele, we allow only one.  Each mapping is represented by the creation of a new SimpleTranscriptAllele with that mapping via a single intronOffset.  These SimpleTranscriptAlleles are then aggregated at the CanonicalSimpleAllele level.</p>

<h2 id="exclusions-and-limitations">Exclusions and Limitations</h2>

<p>The relationship between the transcript and the genomic sequence to which it is mapped is partially modeled, the details of the mapping are not modeled.  In other words, for intronic coordinates, using only the data modeled here, it will be impossible to calculate the reference allele.</p>

<h2 id="attributes">Attributes</h2>

<p>When describing observations a concise and interoperable method is required to reduce ambiguity as well as clarify the semantics of nature language and biological meaning. The use of a formal, stable ontological structure allows highly detailed descriptions as well as a logically ordered interrelationship between concepts.</p>

<table class="table">
  <thead>
    <tr>
      <th>Attribute Name</th>
      <th>Attribute Type</th>
      <th>Cardinality</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><a href="/allele/value_set_list/primary_transcript_region_type.html">primaryTranscriptRegionType</a></td>
      <td>Code</td>
      <td>0..1</td>
    </tr>
    <tr>
      <td><a href="/allele/value_set_list/ancillary_transcript_region_type.html">ancillaryTranscriptRegionType</a></td>
      <td>Code</td>
      <td>0..*</td>
    </tr>
  </tbody>
</table>

<h2 id="related-entities">Related Entities</h2>

<table class="table">
  <thead>
    <tr>
      <th>Relationship Name</th>
      <th>Entity Type</th>
      <th>Cardinality</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>intronOffset</td>
      <td><a href="intron_offset.html">IntronOffset</a></td>
      <td>0..*</td>
    </tr>
  </tbody>
</table>

<p>intronOffset: These entities are used to represent coordinates that occur in intronic sequences.   See above for details.  Fully exonic coordinates will not have any intronOffset relationships.  An intron offset will have at least one, and may have more than one, representing mappings of the transcript to different <a href="../reference_sequence/genomic_reference_sequence.html">GenomicReferenceSequences</a>.</p>


      </article>
    </div>
  </div>
</div>


    <!-- Bootstrap core JavaScript
         ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../javascripts/jquery.js"></script><script src="../../../javascripts/bootstrap.js"></script>

  </body>
</html>
