<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

    <link href="../../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/model/entities/evidence/aggregation_segregation_data.html">0.1</a></li>
                  <li><a href="/development/tech/model/entities/evidence/aggregation_segregation_data.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../../../../">Interpretation Model</a></li>
<li><a href="../../../">Technical Reference</a></li>
<li><a href="../../">Model</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../../../../images/clingen-doc-logo.png" />
            </li>

            <li><a href="../../../../welcome/">Welcome</a></li>
            <li><a href="../../../../user/">User Guide</a></li>
            <li class="active"><a href="../../../">Technical Reference</a></li><ul><li><a href="../../../api/">API</a></li><li><a href="../../../examples/">Examples</a></li><li class="active"><a href="../../">Model</a></li><ul><li><a href="../../activities/">Activity</a></li><li><a href="../../agents/">Agent</a></li><li class="active"><a href="../">Entity</a></li><ul><li><a href="../criterion_assessment.html">CriterionAssessment</a></li><li class="active"><a href="./">Data</a></li><ul><li class="active"><a href="aggregation_segregation_data.html">AggregateSegregationData</a></li><li><a href="benign_missense_variation_rate.html">BenignMissenseVariationRate</a></li><li><a href="case_control.html">CaseControl</a></li><li><a href="condition_etiology.html">ConditionEtiology</a></li><li><a href="condition_inheritance.html">ConditionInheritance</a></li><li><a href="condition_mechanism.html">ConditionMechanism</a></li><li><a href="condition_penetrance.html">ConditionPenetrance</a></li><li><a href="condition_prevelance.html">ConditionPrevelance</a></li><li><a href="conservation.html">Conservation</a></li><li><a href="de_novo_allele.html">DeNovoAllele</a></li><li><a href="family_history.html">FamilyHistory</a></li><li><a href="family_segregation_data.html">FamilySegregationData</a></li><li><a href="functional_data.html">FunctionalData</a></li><li><a href="group_allele_frequency.html">GroupAlleleFrequency</a></li><li><a href="in_silico_prediction.html">InSilicoPrediction</a></li><li><a href="individual_allele.html">IndividualAllele</a></li><li><a href="individual_condition.html">IndividualCondition</a></li><li><a href="mendelian_disease_variant_interpretation.html">MendelianDiseaseVariantPathogenicityInterpretation</a></li><li><a href="molecular_consequence.html">MolecularConsequence</a></li><li><a href="population_allele_frequency.html">PopulationAlleleFrequency</a></li><li><a href="region_annotation.html">RegionAnnotation</a></li><li><a href="region_contains_allele.html">RegionContainsAllele</a></li></ul><li><a href="../evidence_data-source.html">DataSource</a></li><li><a href="../publication_status.html">PublicationStatus</a></li><li><a href="../variant_interpretation.html">VariantInterpretation</a></li></ul><li><a href="../../secondary/">Secondary Types</a></li></ul><li><a href="../../../value_sets/">Value Sets</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>AggregateSegregationData</h1>
            <p>Cosegregation of an allele and a condition aggregated across families</p>

                <p>Descended from Data</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>Co-segregation of alleles and conditions is evaluated per-family, but may then be aggregated across families.  This Data type represents such an aggregation.  Instances may refer to FamilySegregationData that represent the per-family segregation results that are aggregated together, but this is not required.</p>

<p>AggregateSegregationData requires a particular CanonicalAllele and Condition, and it is the co-segregation of these entities that is described.   In addition, a modeOfInheritance may be given, indicating the inheritance pattern to be used for interpreting the segregations.</p>

<p>AggregateSegregationData also provides numerous attributes summarizing data across families, counting the numbers of segregations and sets of individuals in the families, based on their allele and condition status.</p>

<p>If LOD scores are used in the family segregations, these LODS may be combined to a total LOD score as well, though this is not required.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>canonicalAllele</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>Allele being tested for cosegregation</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>Condition</td>
                <td>1..1</td>
                <td>Condition being tested for cosegregation</td>
            </tr>
            <tr>
                <td>totalNumberOfSegregations</td>
                <td>int</td>
                <td>0..1</td>
                <td>Total number of segregations across families</td>
            </tr>
            <tr>
                <td>familySegregationData</td>
                <td>FamilySegregationData</td>
                <td>0..*</td>
                <td>List of FamilySegregrationData that have been aggregated</td>
            </tr>
            <tr>
                <td>modeOfInheritance</td>
                <td>CodeableConcept</td>
                <td>0..1</td>
                <td>Mode Of Inheritance used in evaluation of segregation</td>
            </tr>
            <tr>
                <td>totalOfAllelePosConditionPosIndividuals</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of individuals across families that have the allele and the condition</td>
            </tr>
            <tr>
                <td>totalOfAlleleNegConditionNegIndividuals</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of individuals across families that lack both the allele and the condition</td>
            </tr>
            <tr>
                <td>totalOfAllelePosConditionNegIndividuals</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of individuals across families that have the allele but not the condition</td>
            </tr>
            <tr>
                <td>totalOfAlleleNegConditionPosIndividuals</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of individuals across families that do not have the allele but do have the condition</td>
            </tr>
            <tr>
                <td>totalOfUntestedConditionPosIndividuals</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of individuals across families that have the condition but whose allele is untested</td>
            </tr>
            <tr>
                <td>totalOfUntestedConditionNegIndividuals</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of individuals across families that do not have the condition but whose allele is untested</td>
            </tr>
            <tr>
                <td>totalOfFamiliesWithInconsistentSegregations</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of families containing segregations that are inconsistent with the allele causing the condition</td>
            </tr>
            <tr>
                <td>totalLODScore</td>
                <td>float</td>
                <td>0..1</td>
                <td>LOD Score aggregated across families</td>
            </tr>
            <tr>
                <td>LODScoreType</td>
                <td>CodeableConcept</td>
                <td></td>
                <td>Method of calculating LOD</td>
            </tr>
            </tbody>
        </table>

    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>wasGeneratedBy</td>
                    <td>Activity</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>The activity that produced this entity</td>
                </tr>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>A human-readable description of the entity</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <div id="example_view"></div>
    <script>
        run_on_load = function () {return $("#example_view").JSONView(
          {"AggSegData105":{"cg:id":"AggSegData105","cg:type":"AggregateSegregationData","condition":{"cg:id":"Cond020","cg:type":"Condition","name":"Hypertrophic Cardiomyopathy","code":"217569"},"canonicalAllele":{"cg:id":"CanAll063","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll067","identifier":"http://reg.genome.network/allele/CA010136"},"familySegregationData":{"cg:id":"FamSegData104","cg:type":"FamilySegregationData","wasGeneratedBy":{"cg:id":"CapData075","cg:type":"CaptureData","startedAtTime":{"cg:id":"2016-09-13T00:00:00+00:00"},"endedAtTime":{"cg:id":"2016-09-13T00:00:00+00:00"},"wasAssociatedWith":[{"agent":{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"},"role":"capturer"}],"used":{"DataSource":[{"cg:id":"DataSrc003","cg:type":"DataSource","externalIdentifier":"LMM-VARIANT:NM_000256.3(MYBPC3)c.1351+1G>A"}]}},"explanation":"In LMM internal family, c.1351+1G>A was identified in 1 individual with HCM. Variant also found in individual’s sister - who also had clinical dx of HCM.","condition":{"cg:id":"Cond020","cg:type":"Condition","name":"Hypertrophic Cardiomyopathy","code":"217569"},"canonicalAllele":{"cg:id":"CanAll063","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll067","identifier":"http://reg.genome.network/allele/CA010136"},"family":{"cg:id":"Fam002","cg:type":"Family","description":"Family #002"},"phenotypePositiveAllelePositive":1},"totalNumberOfSegregations":4},"AggSegData109":{"cg:id":"AggSegData109","cg:type":"AggregateSegregationData","condition":{"cg:id":"Cond021","cg:type":"Condition","name":"Usher syndrome","code":"886"},"canonicalAllele":{"cg:id":"CanAll062","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll066","identifier":"http://reg.genome.network/allele/CA143213"},"familySegregationData":{"cg:id":"FamSegData108","cg:type":"FamilySegregationData","explanation":"c.1000C>T identified in het state in 1 individual with Usher. Individual compound het with Gly1840Val USH2A variant - confirmed in trans. Gly1840Val called Pathogenic. Individual has 3 affected sibs all of whom are compound het with 1000C>T & Gly1840Val. One additional sib only carried 1000C>T and is unaffected.","condition":{"cg:id":"Cond021","cg:type":"Condition","name":"Usher syndrome","code":"886"},"canonicalAllele":{"cg:id":"CanAll062","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll066","identifier":"http://reg.genome.network/allele/CA143213"},"family":{"cg:id":"Fam004","cg:type":"Family","description":"Family #004"}},"totalNumberOfSegregations":8},"AggSegData115":{"cg:id":"AggSegData115","cg:type":"AggregateSegregationData","condition":{"cg:id":"Cond002","cg:type":"Condition","name":"Dilated Cardiomyopathy","code":"217604"},"canonicalAllele":{"cg:id":"CanAll065","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll069","identifier":"http://reg.genome.network/allele/CA006429"},"familySegregationData":{"cg:id":"FamSegData114","cg:type":"FamilySegregationData","condition":{"cg:id":"Cond002","cg:type":"Condition","name":"Dilated Cardiomyopathy","code":"217604"},"canonicalAllele":{"cg:id":"CanAll065","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll069","identifier":"http://reg.genome.network/allele/CA006429"},"family":{"cg:id":"Fam007","cg:type":"Family","description":"Family #007"},"phenotypePositiveAllelePositive":2,"phenotypeNegativeAllelePositive":3,"columns":"[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”,”ObligateGenotype”]","affectedValues":"{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}","genotypeValues":"{0: “Unknown”,1:”Reference”, 2: CA65}","pedigree":"[[Fam007, Fam007.1, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.2, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.3, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.4, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.5, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.6, Fam007.1, Fam007.2, 2, 3, 1/2, 1],\n   [Fam007, Fam007.7,0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.8, Fam007.1, Fam007.2, 2, 1, 1/2, 1],\n   [Fam007, Fam007.9,Fam007.3, Fam007.4, 1, 1, 0/0, 0],\n   [Fam007, Fam007.10, 0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.11, Fam007.5, Fam007.6, 2, 2, 1/2, 0],\n   [Fam007, Fam007.12, Fam007.5, Fam007.6, 1,2, 0/0, 0],\n   [Fam007, Fam007.13, Fam007.5, Fam007.6, 2,1, 1/1,0],\n   [Fam007, Fam007.14, Fam007.5, Fam007.6, 1, 1, 1/1, 0],\n   [Fam007, Fam007.15, Fam007.7, Fam007.8, 1, 2, 1/2, 0],\n   [Fam007, Fam007.16, Fam007.8, Fam007.9, 1, 2, 1/2, 0],\n   [Fam007, Fam007.17, Fam007.8, Fam007.9, 1, 2, 0/0,0],\n   [Fam007, Fam007.18, Fam007.8, Fam007.9, 1, 2, 0/0, 0],\n   [Fam007, Fam007.19, Fam007.10, Fam007.11, 1, 2, 0/0,0],\n   [Fam007.20,Fam007.10, Fam007.11, 1,1,1/1,0]]"},"totalNumberOfSegregations":9}},
          { collapsed: true }
        )}
    </script>


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../../javascripts/jquery.js"></script><script src="../../../../javascripts/bootstrap.js"></script><script src="../../../../javascripts/jquery.jsonview.js"></script><script src="../../../../javascripts/run-run_on_load.js"></script>

  </body>
</html>
