<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      GenerateVariantInterpretation
    </title>

    <link href="../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/model/activities/generate_variant_interpretation.html">0.1</a></li>
                  <li><a href="/development/tech/model/activities/generate_variant_interpretation.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../../../">Interpretation Model</a></li>
<li><a href="../../">Technical Reference</a></li>
<li><a href="../">Model</a></li>
<li><a href="./">Activities</a></li>
<li><a href="generate_variant_interpretation.html">GenerateVariantInterpretation</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../../../images/clingen-doc-logo.png" />
            </li>

            <li><a href="../../../welcome/">Welcome</a></li>
            <li><a href="../../../user/">User Guide</a></li>
            <li class="active"><a href="../../">Technical Reference</a></li><ul><li><a href="../../api/">API</a></li><li><a href="../../examples/">Examples</a></li><li class="active"><a href="../">Model</a></li><ul><li class="active"><a href="./">Activity</a></li><ul><li><a href="assess_criterion.html">AssessCriterion</a></li><li><a href="capture_evidence_data.html">CaptureData</a></li><li class="active"><a href="generate_variant_interpretation.html">GenerateVariantInterpretation</a></li><li><a href="publish_variant_interpretation.html">PublishVariantInterpretation</a></li></ul><li><a href="../agents/">Agent</a></li><li><a href="../entities/">Entity</a></li><li><a href="../secondary/">Secondary Types</a></li></ul><li><a href="../../value_sets/">Value Sets</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>GenerateVariantInterpretation</h1>
            <p>*TODO* A short description about generate variant interpretation activity.</p>

                <p>Descended from Activity</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>The GenerateVariantInterpretation activity represents the process of combining CriterionAssessments and producing an interpretation.   In other words, this activity involves weighing the different lines of evidence represented by the possibly conflicting CriterionAssessments and combining them into a single statement. In the context of the ACMG Pathogenicity guidelines, this combination can be calculated algorithmically; the assessor in this case may be a computational agent such as the ClinGen Pathogenicity Calculator.</p>



    <!-- image here if available -->
    <img src="../../../images/generate_variant_interpretation.svg" />

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>assertionMethod</td>
                <td>AssertionMethod</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>criterionAssessments</td>
                <td>CriterionAssessment</td>
                <td>0..*</td>
                <td></td>
            </tr>
            <tr>
                <td>condition</td>
                <td>Condition</td>
                <td>1..1</td>
                <td></td>
            </tr>
            <tr>
                <td>variant</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td></td>
            </tr>
            </tbody>
        </table>

            <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>activityEntityId</td>
                    <td>Entity</td>
                    <td>1..1</td>
                    <td>Activity</td>
                    <td></td>
                </tr>
                <tr>
                    <td>startedAtTime</td>
                    <td>Datetime</td>
                    <td>0..1</td>
                    <td>Activity</td>
                    <td>A human-readable description of the reasoning behind the interpretation</td>
                </tr>
                <tr>
                    <td>endedAtTime</td>
                    <td>Datetime</td>
                    <td>0..1</td>
                    <td>Activity</td>
                    <td></td>
                </tr>
                <tr>
                    <td>wasAssociatedWith</td>
                    <td>Agent</td>
                    <td>0..*</td>
                    <td>Activity</td>
                    <td>The Agent that performed the Activity</td>
                </tr>
            </tbody>
        </table>


    <h2>Examples</h2>
    <div id="example_view"></div>
    <script>
window.addEventListener('load', function () {
    var view = $("#example_view").JSONView(
          {"GenVarInterp032":{"cg:id":"GenVarInterp032","cg:type":"GenerateVariantInterpretation","startedAtTime":"2016-07-12T11:00:59+00:00","endedAtTime":"2016-07-12T11:00:59+00:00","assertionMethod":{"cg:id":"AssertMeth001","cg:type":"AssertionMethod","description":"ACMG ISV guidelines 2015","version":"1.0","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf","scoringAlgorithm":{"cg:id":"ScorAlg01","cg:type":"ScoringAlgorithm","description":"This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance."}},"criterionAssessments":[{"cg:id":"CritAssess031","cg:type":"CriterionAssessment","criterion":{"cg:id":"PM2","cg:type":"Criterion","description":"Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium","shortDescription":"Absent from controls","defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"outcome":"CG:I","wasGeneratedBy":{"cg:id":"AssessCrit031","cg:type":"AssessCriterion","startedAtTime":"2016-07-01T10:53:00+00:00","endedAtTime":"2016-07-01T10:53:00+00:00","hasSupportingEvidence":[{"cg:id":"AllFreq082","cg:type":"AlleleFrequency","wasGeneratedBy":{"cg:id":"CapData081","cg:type":"CaptureData","startedAtTime":"2016-06-01T14:15:00+00:00","endedAtTime":"2016-06-01T14:15:00+00:00","dataSource":{"cg:id":"DataSrc009","cg:type":"DataSource","externalIdentifier":"EXAC v0.2","linkedIRI":"ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.2/"},"wasAssociatedWith":[{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"}]},"ascertainment":"ExAC","population":"CG:combined","allele":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"alleleCount":0,"alleleFrequency":0.0,"medianCoverage":11.0}],"criterion":{"cg:id":"PM2","cg:type":"Criterion","description":"Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium","shortDescription":"Absent from controls","defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"wasAssociatedWith":[{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"},{"cg:id":"Agent005","cg:type":"Agent","description":"CSER","name":"CSER Bake Off Project"}]},"explanation":"while the variant is absent from exac, however\n1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”\n2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC)."},{"cg:id":"CritAssess022","cg:type":"CriterionAssessment","criterion":{"cg:id":"PM3","cg:type":"Criterion","description":"For recessive disorders, detected in trans with a pathogenic variant","shortDescription":"Recessive in trans w/ Path","defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"outcome":"CG:M","wasGeneratedBy":{"cg:id":"AssessCrit022","cg:type":"AssessCriterion","startedAtTime":"2016-06-02T08:00:59+00:00","endedAtTime":"2016-06-02T08:00:59+00:00","hasSupportingEvidence":[{"cg:id":"CondInh055","cg:type":"ConditionInheritance","wasGeneratedBy":{"cg:id":"CapData076","cg:type":"CaptureData","startedAtTime":"2016-05-31T08:45:59+00:00","endedAtTime":"2016-05-31T08:45:59+00:00","dataSource":{"cg:id":"DataSrc004","cg:type":"DataSource","externalIdentifier":"OMIM:244400","linkedIRI":"http://omim.org/entry/244400","version":"alopez : 09/15/2016"},"wasAssociatedWith":[{"cg:id":"Agent008","cg:type":"Agent","description":"MDS","name":"Marina DiStefano"}]},"condition":{"cg:id":"Cond004","cg:type":"Condition","name":"Primary Ciliary Dyskenisia","code":"244"},"modeOfInheritance":"HP:0000007"},{"cg:id":"IndCond056","cg:type":"IndividualCondition","wasGeneratedBy":{"cg:id":"CapData077","cg:type":"CaptureData","startedAtTime":"2016-05-31T09:14:59+00:00","endedAtTime":"2016-05-31T09:14:59+00:00","dataSource":{"cg:id":"DataSrc005","cg:type":"DataSource","externalIdentifier":"LMM-CASE:CSER-SUBJECT-123"},"wasAssociatedWith":[{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"}]},"individual":{"cg:id":"Indiv004","cg:type":"Individual","description":"individual #004"},"condition":[{"cg:id":"Cond004","cg:type":"Condition","name":"Primary Ciliary Dyskenisia","code":"244"},{"cg:id":"Cond014","cg:type":"Condition","name":"outer dynein arm (ODA) defects","code":"0200106"}],"hasCondition":true},{"cg:id":"MenDisVarInterp057","cg:type":"MendelianDiseaseVariantPathogenicityInterpretation","wasGeneratedBy":{"cg:id":"CapData078","cg:type":"CaptureData","startedAtTime":"2016-05-31T11:00:59+00:00","endedAtTime":"2016-05-31T11:00:59+00:00","dataSource":{"cg:id":"DataSrc006","cg:type":"DataSource","externalIdentifier":"CLINVAR:SCV000205842.3","linkedIRI":"https://www.ncbi.nlm.nih.gov/clinvar/SCV000205842.3"},"wasAssociatedWith":[{"cg:id":"Agent008","cg:type":"Agent","description":"MDS","name":"Marina DiStefano"}]},"canonicalAllele":{"cg:id":"CanAll036","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll040","identifier":"http://reg.genome.network/allele/CA273615"},"condition":[{"cg:id":"Cond004","cg:type":"Condition","name":"Primary Ciliary Dyskenisia","code":"244"}],"clinicalSignificance":"LN:LA6668-3"},{"cg:id":"IndAll058","cg:type":"IndividualAllele","wasGeneratedBy":{"cg:id":"CapData077","cg:type":"CaptureData","startedAtTime":"2016-05-31T09:14:59+00:00","endedAtTime":"2016-05-31T09:14:59+00:00","dataSource":{"cg:id":"DataSrc005","cg:type":"DataSource","externalIdentifier":"LMM-CASE:CSER-SUBJECT-123"},"wasAssociatedWith":[{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"}]},"explanation":"Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband.","individual":{"cg:id":"Indiv004","cg:type":"Individual","description":"individual #004"},"primaryAllele":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"secondaryAllele":{"cg:id":"CanAll036","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll040","identifier":"http://reg.genome.network/allele/CA273615"},"primaryZygosity":"CG:het","secondaryZygosity":"CG:het","phase":"CG:trans"}],"criterion":{"cg:id":"PM3","cg:type":"Criterion","description":"For recessive disorders, detected in trans with a pathogenic variant","shortDescription":"Recessive in trans w/ Path","defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"condition":[{"cg:id":"Cond004","cg:type":"Condition","name":"Primary Ciliary Dyskenisia","code":"244"},{"cg:id":"Cond014","cg:type":"Condition","name":"outer dynein arm (ODA) defects","code":"0200106"}],"wasAssociatedWith":[{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"}]}},{"cg:id":"CritAssess025","cg:type":"CriterionAssessment","criterion":{"cg:id":"PM4","cg:type":"Criterion","description":"Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants","shortDescription":"Protein length changes or stop-loss","defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"outcome":"CG:M","wasGeneratedBy":{"cg:id":"AssessCrit025","cg:type":"AssessCriterion","startedAtTime":"2016-06-12T07:30:00+00:00","endedAtTime":"2016-06-12T07:30:00+00:00","hasSupportingEvidence":[{"cg:id":"MolCon066","cg:type":"MolecularConsequence","wasGeneratedBy":{"cg:id":"CapData079","cg:type":"CaptureData","startedAtTime":"2016-06-01T14:03:00+00:00","endedAtTime":"2016-06-01T14:03:00+00:00","dataSource":{"cg:id":"DataSrc007","cg:type":"DataSource","externalIdentifier":"LMM:GENEINSIGHT-LAB v5.3.2","linkedIRI":"https://geneinsight-lmm-test.partners.org"},"wasAssociatedWith":[{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"}]},"contextualAllele":{"cg:id":"CtxAll044","cg:type":"ContextualAllele","relatedCanonicalAllele":{"cg:id":"CanAll040","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll044","identifier":"http://reg.genome.network/allele/CA273136"},"alleleName":"NP_001360.1:p.Trp2490_Leu2496del"},"consequence":"SO:0001825"},{"cg:id":"Reg069","cg:type":"RegionTypesAndAlleles","wasGeneratedBy":{"cg:id":"CapData080","cg:type":"CaptureData","startedAtTime":"2016-06-01T14:10:00+00:00","endedAtTime":"2016-06-01T14:10:00+00:00","dataSource":{"cg:id":"DataSrc008","cg:type":"DataSource","externalIdentifier":"LMM:ALAMUT-PROD v123","linkedIRI":"https://pcpgm.partners.org/lmm"},"wasAssociatedWith":[{"cg:id":"Agent008","cg:type":"Agent","description":"MDS","name":"Marina DiStefano"}]},"explanation":"This is not a repreating region.","referenceSequence":{"cg:id":"RefSeq005","cg:type":"ReferenceSequence","identifier":"NP_001360.1"},"start":2490,"stop":2496,"isNotTypes":["CG:repeat"]}],"criterion":{"cg:id":"PM4","cg:type":"Criterion","description":"Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants","shortDescription":"Protein length changes or stop-loss","defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"wasAssociatedWith":[{"cg:id":"Agent003","cg:type":"Agent","description":"TS","name":"Tasha Strande"}]},"explanation":"leads to an in-frame deletion of 7 amino acids (not repeat region)."}],"condition":{"cg:id":"Cond004","cg:type":"Condition","name":"Primary Ciliary Dyskenisia","code":"244"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"wasAssociatedWith":[{"cg:id":"Agent007","cg:type":"Agent","description":"CPC","name":"ClinGen Pathogenicity Calculator v1"},{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"}]}},
          { collapsed: true }
    )

    var fix_ellipses = function () {
        $('#example_view ul.obj li a:contains("cg:id")').parent().parent().parent().children('.ellipsis').each(function () {
            $(this).html('<em>' + $(this).next('ul').find('a:contains("cg:id")').next('.string')[0].innerText + ', …</em>')
        })

        $('#example_view ul.array').each(function () {
            $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
        })
    }
    fix_ellipses()
    $('#example_view .collapser').click(fix_ellipses)
    $('#example_view .collapser').next('a.prop').click(fix_ellipses)
})
    </script>


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../javascripts/jquery.js"></script><script src="../../../javascripts/bootstrap.js"></script><script src="../../../javascripts/jquery.jsonview.js"></script>

  </body>
</html>
