<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

    <link href="../../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/model/entities/guideline/criterion.html">0.1</a></li>
                  <li><a href="/development/tech/model/entities/guideline/criterion.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          
        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../../../../images/clingen-doc-logo.png" />
            </li>

            <li><a href="../../../../welcome/">Welcome</a></li>
            <li><a href="../../../../user/">User Guide</a></li>
            <li class="active"><a href="../../../">Technical Reference</a></li><ul><li><a href="../../../api/">API</a></li><li><a href="../../../examples/">Examples</a></li><li class="active"><a href="../../">Model</a></li><ul><li><a href="../../activities/">Activity</a></li><li><a href="../../agents/">Agent</a></li><li class="active"><a href="../">Entity</a></li><ul><li><a href="../criterion_assessment.html">CriterionAssessment</a></li><li><a href="../evidence/">Data</a></li><li><a href="../evidence_data-source.html">DataSource</a></li><li><a href="../publication_status.html">PublicationStatus</a></li><li><a href="../variant_interpretation.html">VariantInterpretation</a></li></ul><li><a href="../../secondary/">Secondary Types</a></li></ul><li><a href="../../../value_sets/">Value Sets</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>Criterion</h1>
            <p>A single rule for evaluating data in an interpretation.</p>

                <p>Descended from Entity</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>description</td>
                <td>String</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>shortDescription</td>
                <td>String</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>method</td>
                <td>Method</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>defaultOutcome</td>
                <td>CodeableConcept</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>targetInterpretation</td>
                <td>CodeableConcept</td>
                <td>0..1</td>
                <td></td>
            </tr>
            </tbody>
        </table>

    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>wasGeneratedBy</td>
                    <td>Activity</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>The activity that produced this entity</td>
                </tr>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>A human-readable description of the entity</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <div id="example_view"></div>
    <script>
        run_on_load = function () {return $("#example_view").JSONView(
          {"PVS1":{"cg:id":"PVS1","cg:type":"Criterion","description":"null variant (nonsense, frameshift, canonical Â±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease","shortDescription":"Null Variant in known LOF genes","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:VS","targetInterpretation":"LN:LA6668-3"},"PS1":{"cg:id":"PS1","cg:type":"Criterion","description":"Same amino acid change as a previously established pathogenic variant regardless of nucleotide change","shortDescription":"same as previous pathogenic amino acid change","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:S","targetInterpretation":"LN:LA6668-3"},"PS2":{"cg:id":"PS2","cg:type":"Criterion","description":"De novo (both maternity and paternity confirmed) in a patient with the disease and no family history","shortDescription":"confirmed de novo no family history","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:S","targetInterpretation":"LN:LA6668-3"},"PS3":{"cg:id":"PS3","cg:type":"Criterion","description":"Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product","shortDescription":"well-established damaging effect functional studies","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:S","targetInterpretation":"LN:LA6668-3"},"PS4":{"cg:id":"PS4","cg:type":"Criterion","description":"The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls","shortDescription":"Prevelance increase in affected v control","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:S","targetInterpretation":"LN:LA6668-3"},"PM1":{"cg:id":"PM1","cg:type":"Criterion","description":"Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation","shortDescription":"Located in hot spot w/out benign variation","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"PM2":{"cg:id":"PM2","cg:type":"Criterion","description":"Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium","shortDescription":"Absent from controls","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"PM3":{"cg:id":"PM3","cg:type":"Criterion","description":"For recessive disorders, detected in trans with a pathogenic variant","shortDescription":"Recessive in trans w/ Path","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"PM4":{"cg:id":"PM4","cg:type":"Criterion","description":"Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants","shortDescription":"Protein length changes or stop-loss","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"PM5":{"cg:id":"PM5","cg:type":"Criterion","description":"Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before","shortDescription":"novel missense at same aa residue as known path missense","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"PM6":{"cg:id":"PM6","cg:type":"Criterion","description":"Assumed de novo, but without confirmation of paternity and maternity","shortDescription":"assumed de novo","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"PP1":{"cg:id":"PP1","cg:type":"Criterion","description":"Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease","shortDescription":"segregates with disease in affected family members\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6668-3"},"PP2":{"cg:id":"PP2","cg:type":"Criterion","description":"Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease","shortDescription":"missense with pathogenic missense common\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6668-3"},"PP3":{"cg:id":"PP3","cg:type":"Criterion","description":"Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)","shortDescription":"computational evidence supports deleterious\neffect","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6668-3"},"PP4":{"cg:id":"PP4","cg:type":"Criterion","description":"Patientâs phenotype or family history is highly specific for a disease with a single genetic etiology","shortDescription":"disease specific phenotype and family history\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6668-3"},"PP5":{"cg:id":"PP5","cg:type":"Criterion","description":"Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6668-3"},"BA1":{"cg:id":"BA1","cg:type":"Criterion","description":"Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium","shortDescription":"present in population databases at greater than 5%","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:A","targetInterpretation":"LN:LA6675-8"},"BS1":{"cg:id":"BS1","cg:type":"Criterion","description":"Allele frequency is greater than expected for disorder","shortDescription":"MAF too high for disorder","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:S","targetInterpretation":"LN:LA6675-8"},"BS2":{"cg:id":"BS2","cg:type":"Criterion","description":"Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age","shortDescription":"control frequency higher than penetrance predicts\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:S","targetInterpretation":"LN:LA6675-8"},"BS3":{"cg:id":"BS3","cg:type":"Criterion","description":"Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing","shortDescription":"well-established no damaging effect functional studies","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:S","targetInterpretation":"LN:LA6675-8"},"BS4":{"cg:id":"BS4","cg:type":"Criterion","description":"Lack of segregation in affected members of a family","shortDescription":"does not segregate with disease\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:S","targetInterpretation":"LN:LA6675-8"},"BP1":{"cg:id":"BP1","cg:type":"Criterion","description":"Missense variant in a gene for which primarily truncating variants are known to cause disease","shortDescription":"missense variant when truncating is pathogenic\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6675-8"},"BP2":{"cg:id":"BP2","cg:type":"Criterion","description":"Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern","shortDescription":"in trans with dominant OR in cis with path variant\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6675-8"},"BP3":{"cg:id":"BP3","cg:type":"Criterion","description":"In-frame deletions/insertions in a repetitive region without a known function","shortDescription":"in-frame indel unknown function\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6675-8"},"BP4":{"cg:id":"BP4","cg:type":"Criterion","description":"Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)","shortDescription":"computational evidence for\nno gene impact","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6675-8"},"BP5":{"cg:id":"BP5","cg:type":"Criterion","description":"Variant found in a case with an alternate molecular basis for disease","shortDescription":"alternate cause of disease\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6675-8"},"BP6":{"cg:id":"BP6","cg:type":"Criterion","description":"Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation","shortDescription":"benign from reputable source without shared data\n","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6675-8"},"BP7":{"cg:id":"BP7","cg:type":"Criterion","description":"A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved","shortDescription":"silent variant no splicing \nimpact  and low conservation","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:P","targetInterpretation":"LN:LA6675-8"}},
          { collapsed: true }
        )}
    </script>


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../../javascripts/jquery.js"></script><script src="../../../../javascripts/bootstrap.js"></script><script src="../../../../javascripts/jquery.jsonview.js"></script><script src="../../../../javascripts/run-run_on_load.js"></script>

  </body>
</html>
