<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/model/entities/guideline/criterion.html">0.1</a></li>
                  <li><a href="/development/tech/model/entities/guideline/criterion.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          
        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../../../../images/clingen-doc-logo.png" />
            </li>

            <li><a href="../../../../welcome/">Welcome</a></li>
            <li><a href="../../../../user/">User Guide</a></li>
            <li class="active"><a href="../../../">Technical Reference</a></li><ul><li><a href="../../../api/">API</a></li><li><a href="../../../examples/">Examples</a></li><li class="active"><a href="../../">Model</a></li><ul><li><a href="../../agents/">Agent</a></li><li class="active"><a href="../">Entity</a></li><ul><li><a href="../criterion_assessment.html">CriterionAssessment</a></li><li><a href="../evidence/">Data</a></li><li><a href="../evidence_data-source.html">DataSource</a></li><li><a href="../variant_interpretation.html">VariantInterpretation</a></li></ul><li><a href="../../secondary/">Secondary Types</a></li></ul><li><a href="../../../value_sets/">Value Sets</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>Criterion</h1>
            <p>A single rule for evaluating data in an interpretation.</p>

                <p>Descended from Entity</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>description</td>
                <td>String</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>shortDescription</td>
                <td>String</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>method</td>
                <td>Method</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>defaultOutcome</td>
                <td>CodeableConcept</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>targetInterpretation</td>
                <td>CodeableConcept</td>
                <td>0..1</td>
                <td></td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A human-readable description of the entity</td>
                </tr>
                <tr>
                    <td>wasGeneratedBy</td>
                    <td>Activity</td>
                    <td>0..1</td>
                    <td></td>
                    <td>The activity that produced this entity</td>
                </tr>
                <tr>
                    <td>wasDerivedFrom</td>
                    <td>Data</td>
                    <td>0..1</td>
                    <td></td>
                    <td>Another entityy which was a former derivation of this entity.</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>description</th>
        <th>description</th>
        <th>shortDescription</th>
        <th>method</th>
        <th>defaultOutcome</th>
        <th>targetInterpretation</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../../../examples/details.html#PVS1">PVS1</a></p></td>
          <td>null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease</td>
          <td>
              <p>null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease</p>
         </td>
          <td>
              <p>Null Variant in known LOF genes</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:VS</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PVS1"><pre><code>{
  "cg:id": "PVS1",
  "cg:type": "Criterion",
  "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
  "shortDescription": "Null Variant in known LOF genes",
  "defaultOutcome": "CG:VS",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PS1">PS1</a></p></td>
          <td>Same amino acid change as a previously established pathogenic variant regardless of nucleotide change</td>
          <td>
              <p>Same amino acid change as a previously established pathogenic variant regardless of nucleotide change</p>
         </td>
          <td>
              <p>same as previous pathogenic amino acid change</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:S</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PS1"><pre><code>{
  "cg:id": "PS1",
  "cg:type": "Criterion",
  "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
  "shortDescription": "same as previous pathogenic amino acid change",
  "defaultOutcome": "CG:S",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PS2">PS2</a></p></td>
          <td>De novo (both maternity and paternity confirmed) in a patient with the disease and no family history</td>
          <td>
              <p>De novo (both maternity and paternity confirmed) in a patient with the disease and no family history</p>
         </td>
          <td>
              <p>confirmed de novo no family history</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:S</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PS2"><pre><code>{
  "cg:id": "PS2",
  "cg:type": "Criterion",
  "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
  "shortDescription": "confirmed de novo no family history",
  "defaultOutcome": "CG:S",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PS3">PS3</a></p></td>
          <td>Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product</td>
          <td>
              <p>Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product</p>
         </td>
          <td>
              <p>well-established damaging effect functional studies</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:S</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PS3"><pre><code>{
  "cg:id": "PS3",
  "cg:type": "Criterion",
  "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
  "shortDescription": "well-established damaging effect functional studies",
  "defaultOutcome": "CG:S",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PS4">PS4</a></p></td>
          <td>The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls</td>
          <td>
              <p>The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls</p>
         </td>
          <td>
              <p>Prevelance increase in affected v control</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:S</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PS4"><pre><code>{
  "cg:id": "PS4",
  "cg:type": "Criterion",
  "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
  "shortDescription": "Prevelance increase in affected v control",
  "defaultOutcome": "CG:S",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PM1">PM1</a></p></td>
          <td>Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation</td>
          <td>
              <p>Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation</p>
         </td>
          <td>
              <p>Located in hot spot w/out benign variation</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:M</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PM1"><pre><code>{
  "cg:id": "PM1",
  "cg:type": "Criterion",
  "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
  "shortDescription": "Located in hot spot w/out benign variation",
  "defaultOutcome": "CG:M",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PM2">PM2</a></p></td>
          <td>Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium</td>
          <td>
              <p>Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium</p>
         </td>
          <td>
              <p>Absent from controls</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:M</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PM2"><pre><code>{
  "cg:id": "PM2",
  "cg:type": "Criterion",
  "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
  "shortDescription": "Absent from controls",
  "defaultOutcome": "CG:M",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PM3">PM3</a></p></td>
          <td>For recessive disorders, detected in trans with a pathogenic variant</td>
          <td>
              <p>For recessive disorders, detected in trans with a pathogenic variant</p>
         </td>
          <td>
              <p>Recessive in trans w/ Path</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:M</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PM3"><pre><code>{
  "cg:id": "PM3",
  "cg:type": "Criterion",
  "description": "For recessive disorders, detected in trans with a pathogenic variant",
  "shortDescription": "Recessive in trans w/ Path",
  "defaultOutcome": "CG:M",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PM4">PM4</a></p></td>
          <td>Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants</td>
          <td>
              <p>Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants</p>
         </td>
          <td>
              <p>Protein length changes or stop-loss</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:M</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PM4"><pre><code>{
  "cg:id": "PM4",
  "cg:type": "Criterion",
  "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
  "shortDescription": "Protein length changes or stop-loss",
  "defaultOutcome": "CG:M",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PM5">PM5</a></p></td>
          <td>Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before</td>
          <td>
              <p>Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before</p>
         </td>
          <td>
              <p>novel missense at same aa residue as known path missense</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:M</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PM5"><pre><code>{
  "cg:id": "PM5",
  "cg:type": "Criterion",
  "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
  "shortDescription": "novel missense at same aa residue as known path missense",
  "defaultOutcome": "CG:M",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PM6">PM6</a></p></td>
          <td>Assumed de novo, but without confirmation of paternity and maternity</td>
          <td>
              <p>Assumed de novo, but without confirmation of paternity and maternity</p>
         </td>
          <td>
              <p>assumed de novo</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:M</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PM6"><pre><code>{
  "cg:id": "PM6",
  "cg:type": "Criterion",
  "description": "Assumed de novo, but without confirmation of paternity and maternity",
  "shortDescription": "assumed de novo",
  "defaultOutcome": "CG:M",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PP1">PP1</a></p></td>
          <td>Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease</td>
          <td>
              <p>Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease</p>
         </td>
          <td>
              <p>segregates with disease in affected family members</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PP1"><pre><code>{
  "cg:id": "PP1",
  "cg:type": "Criterion",
  "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
  "shortDescription": "segregates with disease in affected family members",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PP2">PP2</a></p></td>
          <td>Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease</td>
          <td>
              <p>Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease</p>
         </td>
          <td>
              <p>missense with pathogenic missense common</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PP2"><pre><code>{
  "cg:id": "PP2",
  "cg:type": "Criterion",
  "description": "Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease",
  "shortDescription": "missense with pathogenic missense common",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PP3">PP3</a></p></td>
          <td>Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)</td>
          <td>
              <p>Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)</p>
         </td>
          <td>
              <p>computational evidence supports deleterious
effect</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PP3"><pre><code>{
  "cg:id": "PP3",
  "cg:type": "Criterion",
  "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
  "shortDescription": "computational evidence supports deleterious\neffect",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PP4">PP4</a></p></td>
          <td>Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology</td>
          <td>
              <p>Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology</p>
         </td>
          <td>
              <p>disease specific phenotype and family history</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PP4"><pre><code>{
  "cg:id": "PP4",
  "cg:type": "Criterion",
  "description": "Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology",
  "shortDescription": "disease specific phenotype and family history",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#PP5">PP5</a></p></td>
          <td>Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation</td>
          <td>
              <p>Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation</p>
         </td>
          <td>
              <p>pathogenic from reputable source without shared data</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <!--
          <td><div class="example_view" id="PP5"><pre><code>{
  "cg:id": "PP5",
  "cg:type": "Criterion",
  "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
  "shortDescription": "pathogenic from reputable source without shared data",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BA1">BA1</a></p></td>
          <td>Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium</td>
          <td>
              <p>Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium</p>
         </td>
          <td>
              <p>present in population databases at greater than 5%</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:A</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BA1"><pre><code>{
  "cg:id": "BA1",
  "cg:type": "Criterion",
  "description": "Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
  "shortDescription": "present in population databases at greater than 5%",
  "defaultOutcome": "CG:A",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BS1">BS1</a></p></td>
          <td>Allele frequency is greater than expected for disorder</td>
          <td>
              <p>Allele frequency is greater than expected for disorder</p>
         </td>
          <td>
              <p>MAF too high for disorder</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:S</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BS1"><pre><code>{
  "cg:id": "BS1",
  "cg:type": "Criterion",
  "description": "Allele frequency is greater than expected for disorder",
  "shortDescription": "MAF too high for disorder",
  "defaultOutcome": "CG:S",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BS2">BS2</a></p></td>
          <td>Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age</td>
          <td>
              <p>Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age</p>
         </td>
          <td>
              <p>control frequency higher than penetrance predicts</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:S</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BS2"><pre><code>{
  "cg:id": "BS2",
  "cg:type": "Criterion",
  "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
  "shortDescription": "control frequency higher than penetrance predicts",
  "defaultOutcome": "CG:S",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BS3">BS3</a></p></td>
          <td>Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing</td>
          <td>
              <p>Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing</p>
         </td>
          <td>
              <p>well-established no damaging effect functional studies</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:S</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BS3"><pre><code>{
  "cg:id": "BS3",
  "cg:type": "Criterion",
  "description": "Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing",
  "shortDescription": "well-established no damaging effect functional studies",
  "defaultOutcome": "CG:S",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BS4">BS4</a></p></td>
          <td>Lack of segregation in affected members of a family</td>
          <td>
              <p>Lack of segregation in affected members of a family</p>
         </td>
          <td>
              <p>does not segregate with disease</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:S</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BS4"><pre><code>{
  "cg:id": "BS4",
  "cg:type": "Criterion",
  "description": "Lack of segregation in affected members of a family",
  "shortDescription": "does not segregate with disease",
  "defaultOutcome": "CG:S",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BP1">BP1</a></p></td>
          <td>Missense variant in a gene for which primarily truncating variants are known to cause disease</td>
          <td>
              <p>Missense variant in a gene for which primarily truncating variants are known to cause disease</p>
         </td>
          <td>
              <p>missense variant when truncating is pathogenic
</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BP1"><pre><code>{
  "cg:id": "BP1",
  "cg:type": "Criterion",
  "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
  "shortDescription": "missense variant when truncating is pathogenic\n",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BP2">BP2</a></p></td>
          <td>Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern</td>
          <td>
              <p>Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern</p>
         </td>
          <td>
              <p>in trans with dominant OR in cis with path variant</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BP2"><pre><code>{
  "cg:id": "BP2",
  "cg:type": "Criterion",
  "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
  "shortDescription": "in trans with dominant OR in cis with path variant",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BP3">BP3</a></p></td>
          <td>In-frame deletions/insertions in a repetitive region without a known function</td>
          <td>
              <p>In-frame deletions/insertions in a repetitive region without a known function</p>
         </td>
          <td>
              <p>in-frame indel unknown function</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BP3"><pre><code>{
  "cg:id": "BP3",
  "cg:type": "Criterion",
  "description": "In-frame deletions/insertions in a repetitive region without a known function",
  "shortDescription": "in-frame indel unknown function",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BP4">BP4</a></p></td>
          <td>Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)</td>
          <td>
              <p>Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)</p>
         </td>
          <td>
              <p>computational evidence for
no gene impact</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BP4"><pre><code>{
  "cg:id": "BP4",
  "cg:type": "Criterion",
  "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
  "shortDescription": "computational evidence for\nno gene impact",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BP5">BP5</a></p></td>
          <td>Variant found in a case with an alternate molecular basis for disease</td>
          <td>
              <p>Variant found in a case with an alternate molecular basis for disease</p>
         </td>
          <td>
              <p>alternate cause of disease</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BP5"><pre><code>{
  "cg:id": "BP5",
  "cg:type": "Criterion",
  "description": "Variant found in a case with an alternate molecular basis for disease",
  "shortDescription": "alternate cause of disease",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BP6">BP6</a></p></td>
          <td>Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation</td>
          <td>
              <p>Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation</p>
         </td>
          <td>
              <p>benign from reputable source without shared data</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BP6"><pre><code>{
  "cg:id": "BP6",
  "cg:type": "Criterion",
  "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
  "shortDescription": "benign from reputable source without shared data",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../../examples/details.html#BP7">BP7</a></p></td>
          <td>A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved</td>
          <td>
              <p>A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved</p>
         </td>
          <td>
              <p>silent variant no splicing 
impact  and low conservation</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>CG:P</p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <!--
          <td><div class="example_view" id="BP7"><pre><code>{
  "cg:id": "BP7",
  "cg:type": "Criterion",
  "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
  "shortDescription": "silent variant no splicing \nimpact  and low conservation",
  "defaultOutcome": "CG:P",
  "targetInterpretation": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("cg:id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("cg:id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../../javascripts/jquery.js"></script><script src="../../../../javascripts/bootstrap.js"></script><script src="../../../../javascripts/jquery.jsonview.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
