<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a class="navbar-brand" href="/">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../../../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a target="_blank" href="http://www.clinicalgenome.org"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>
    


    
        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">VariantPathogenicityInterpretation</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../../../">Interpretation Model</a></li>
<li><a href="../../">Technical Reference</a></li>
<li><a href="../">Entity Definitions</a></li>
<li><a href="./">Core</a></li>

              </ol>
                <blockquote><p>An assertion about the pathogenicity of a genetic sequence variant, based on evaluation of one or more lines of evidence.</p>
                <STRONG>
                  <div>SEPIO:0000190</div>
                      <i>Descended from Statement</i>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>A VariantInterpretation is a statement about the pathogenicity of a given CanonicalAllele, usually for a given MendelianCondition.  A VariantInterpretation does not explicitly contain the reasoning and evidence supporting the conclusion, but that information is referenced in the provenance of the VariantInterpretation entity.  That is, the VariantInterpretation is, for example, the statement that a variant is pathogenic, the reasoning for that conclusion is found by following the EvidenceLines of the VariantInterpretation.</p>

<p>VariantInterpretations for pathogenic variants will include a MendelianCondition for which the conclusion was reached.  However, if the variant is determined to be benign (such as in the case of a very common polymorphism), then the condition attribute may be unused indicating that the allele is benign for all conditions.  If a Condition is specified for a benign variant, then the conclusion that the allele is benign is limited to that specific condition.</p>

<p>While the EvidenceLines should be sufficient to understand the reasoning behind a given VariantInterpretation, a human-readable statement of this reasoning may be included in the explanation attribute (inherited from Entity).</p>

<p>VariantInterpretations are frequently updated when new information becomes available, or during periodic review.  In such a case, a new VariantInterpretation is created.  A tie to the original VariantInterpretation can be maintained using the wasDerivedFrom attribute, which indicates the previous version of the current VariantInterpretation.</p>

<h2 id="primary-examples">Primary Examples</h2>

<p><a href="../../details/details.html#CG-EX:VarInterp274">VarInterp274</a> and
<a href="../../details/details.html#CG-EX:VarInterp063">VarInterp063</a> 
are fully-realized examples and may serve as a starting point for exploring the model.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>variant</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>The variant whose pathogenicity is being described</td>
                <td>SEPIO:0000275</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>GeneticCondition</td>
                <td>0..*</td>
                <td>The condition for which the variant's pathogenicity is described</td>
                <td>SEPIO:0000276</td>
            </tr>
            <tr>
                <td>statementOutcome</td>
                <td>@id</td>
                <td>1..1</td>
                <td>The coded pathogenicity of the variant for the condition</td>
                <td>SEPIO:0000197</td>
            </tr>
            <tr>
                <td>assertionMethod</td>
                <td>AssertionMethod</td>
                <td>0..1</td>
                <td>The  framework that formally defines the methodology used in creating this interpretation.</td>
                <td>SEPIO:0000041</td>
            </tr>
            </tbody>
        </table>

<!--
    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>label</td>
                    <td>string</td>
                    <td>0..1</td>
                    <td>A name given to the resource.</td>
                    <td>RDFS:label</td>
                </tr>
                <tr>
                    <td>description</td>
                    <td>string</td>
                    <td>0..1</td>
                    <td>Description may include but is not limited to: an abstract, a table of contents, a graphical representation, or a free-text account of the resource.</td>
                    <td>DC:description</td>
                </tr>
                <tr>
                    <td>userLabelDictionary</td>
                    <td>UserLabel</td>
                    <td>0..*</td>
                    <td>An optional label defined by the user. Used for custom entities or to clarify the preferred user label on existing entities with non-preferred labels.</td>
                    <td>SEPIO:0000422</td>
                </tr>
                <tr>
                    <td>outcomeQualifier</td>
                    <td>string</td>
                    <td>0..1</td>
                    <td>Use “NOT” as the value of this property to assert that the statement is negated.</td>
                    <td>SEPIO:0000346</td>
                </tr>
                <tr>
                    <td>evidenceLine</td>
                    <td>EvidenceLine</td>
                    <td>0..*</td>
                    <td>supporting evidence</td>
                    <td>SEPIO:0000006</td>
                </tr>
                <tr>
                    <td>contribution</td>
                    <td>Contribution</td>
                    <td>0..*</td>
                    <td>The contributions (agent, time and role) made to this entity</td>
                    <td>SEPIO:0000159</td>
                </tr>
                <tr>
                    <td>source</td>
                    <td>string</td>
                    <td>0..*</td>
                    <td>A string indicating the source of a Statement</td>
                    <td>DC:source</td>
                </tr>
            </tbody>
        </table>
-->

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>label</th>
        <th>description</th>
        <th>userLabelDictionary</th>
        <th>outcomeQualifier</th>
        <th>evidenceLine</th>
        <th>variant</th>
        <th>condition</th>
        <th>statementOutcome</th>
        <th>assertionMethod</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp008">CGEX:VarInterp008</a></p></td>
          <td>
              
         </td>
          <td>
              <p>NM_002834.3:c.855T>G Pathogenic for Noonan Syndrome</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA220155">CAR:CA220155</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond003">CGEX:GenCond003</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp008"><pre><code>{
  "id": "CGEX:VarInterp008",
  "type": "VariantPathogenicityInterpretation",
  "description": "NM_002834.3:c.855T>G Pathogenic for Noonan Syndrome",
  "variant": {
    "id": "CAR:CA220155",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll006"
  },
  "condition": [
    {
      "id": "CGEX:GenCond003",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:648",
          "type": "Entity",
          "label": "noonan syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp009">CGEX:VarInterp009</a></p></td>
          <td>
              
         </td>
          <td>
              <p>NM_001369.2:c.6249G>A Pathogenic for PCD</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA501221">CAR:CA501221</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond004">CGEX:GenCond004</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp009"><pre><code>{
  "id": "CGEX:VarInterp009",
  "type": "VariantPathogenicityInterpretation",
  "description": "NM_001369.2:c.6249G>A Pathogenic for PCD",
  "variant": {
    "id": "CAR:CA501221",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll009"
  },
  "condition": [
    {
      "id": "CGEX:GenCond004",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:244",
          "type": "Entity",
          "label": "Primary ciliary dyskenisia"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp010">CGEX:VarInterp010</a></p></td>
          <td>
              
         </td>
          <td>
              <p>NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Noonan Syndrome 3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA256480">CAR:CA256480</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond006">CGEX:GenCond006</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp010"><pre><code>{
  "id": "CGEX:VarInterp010",
  "type": "VariantPathogenicityInterpretation",
  "description": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Noonan Syndrome 3",
  "variant": {
    "id": "CAR:CA256480",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll014"
  },
  "condition": [
    {
      "id": "CGEX:GenCond006",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "OMIM:609942",
          "type": "Entity",
          "label": "noonan syndrome 3"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp011">CGEX:VarInterp011</a></p></td>
          <td>
              
         </td>
          <td>
              <p>NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Costello syndrome</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA256480">CAR:CA256480</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond055">CGEX:GenCond055</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp011"><pre><code>{
  "id": "CGEX:VarInterp011",
  "type": "VariantPathogenicityInterpretation",
  "description": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Costello syndrome",
  "variant": {
    "id": "CAR:CA256480",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll014"
  },
  "condition": [
    {
      "id": "CGEX:GenCond055",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:648",
          "type": "Entity",
          "label": "noonan syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp055">CGEX:VarInterp055</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The Tyr337Cys variant in SOS1 has been reported in 1 individual with clinical features of Noonan syndrome (Roberts 2007), but has also been reported to be present in 2 unaffected individuals from one family (Personal Communication). This variant was absent from large population studies. Studies have shown that the Tyr337Cys variant may impact protein function by increasing its activity though these results are inconclusive and these in vitro assays may not accurately represent biological function (Smith 2013). Due to the conflicting data about this variant, the clinical significance of the Tyr337Cys variant is uncertain.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:EvLn124">CGEX:EvLn124</a></p><p><a href="../../details/details.html#CGEX:EvLn125">CGEX:EvLn125</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA090930">CAR:CA090930</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond075">CGEX:GenCond075</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26333-7">LOINC:LA26333-7</a></p> (Uncertain Significance)
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:AssertMeth331">CGEX:AssertMeth331</a></p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp055"><pre><code>{
  "id": "CGEX:VarInterp055",
  "type": "VariantPathogenicityInterpretation",
  "description": "The Tyr337Cys variant in SOS1 has been reported in 1 individual with clinical features of Noonan syndrome (Roberts 2007), but has also been reported to be present in 2 unaffected individuals from one family (Personal Communication). This variant was absent from large population studies. Studies have shown that the Tyr337Cys variant may impact protein function by increasing its activity though these results are inconclusive and these in vitro assays may not accurately represent biological function (Smith 2013). Due to the conflicting data about this variant, the clinical significance of the Tyr337Cys variant is uncertain.",
  "variant": {
    "id": "CAR:CA090930",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll130"
  },
  "condition": [
    {
      "id": "CGEX:GenCond075",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:648",
          "type": "Entity",
          "label": "noonan syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26333-7",
    "type": "Entity",
    "label": "Uncertain Significance"
  },
  "assertionMethod": {
    "id": "CGEX:AssertMeth331",
    "type": "VariantPathogenicityInterpretationGuideline",
    "description": "ACMG ISV guidelines 2015",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "version": "1.0",
    "scoringAlgorithm": {
      "id": "SEPIO-CG:99300"
    }
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn124",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess374",
          "type": "CriterionAssessment",
          "description": "Flag-tagged construct with the variant was transfected into HEK cells. As per the RASopathy working group’s approved assays (see example 3 above), Smith et al. showed that the Y337C mutation increased levels of pERK in starved HEK cells via western blotting.",
          "criterion": {
            "id": "SEPIO-CG:99025",
            "type": "VariantInterpretationCriterion",
            "label": "PS3",
            "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
            "usageNotes": "This particular rule will probably be downgraded often, given that there are not many “Functional studies that have been validated and shown to be reproducible and robust in a clinical diagnostic laboratory setting”",
            "defaultStrength": {
              "id": "SEPIO:0000330",
              "type": "Entity",
              "label": "Pathogenic Strong"
            }
          },
          "variant": {
            "id": "CAR:CA090930",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll130"
          },
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn116",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:Func067",
                  "type": "AlleleFunctionalImpactStatement",
                  "resultDescription": "increased ERK activtation",
                  "assayType": {
                    "id": "CGEX:AFAMID076",
                    "type": "Entity",
                    "label": "Activation Assay"
                  },
                  "contextualAllele": {
                    "id": "CGEX:CtxAll130",
                    "type": "ContextualAllele",
                    "canonicalAllele": {
                      "id": "CAR:CA090930",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll130"
                    },
                    "alleleName": {
                      "id": "CGEX:CtxNm9130",
                      "type": "ContextualAlleleName",
                      "name": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)",
                      "preferred": true
                    }
                  },
                  "gene": {
                    "id": "HGNC:11187",
                    "type": "Entity",
                    "label": "SOS1"
                  },
                  "source": [
                    "PMID:23487764"
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "3/14/2017 10:29 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000330",
        "type": "Entity",
        "label": "Pathogenic Strong"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent007",
            "type": "Agent",
            "label": "Stefan DiMarino"
          },
          "contributionDate": "3/14/2017 10:29 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn125",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess375",
          "type": "CriterionAssessment",
          "description": "A personal communication to our lab informed us of an individual and his son who both harbored the variant but with clinical workup did not appear to have Noonan Syndrome or symptoms indicative of Noonan Syndrome.",
          "criterion": {
            "id": "SEPIO-CG:99042",
            "type": "VariantInterpretationCriterion",
            "label": "BS4",
            "description": "Lack of segregation in affected members of a family",
            "usageNotes": "This rule is for “lack of segregation” meaning phenotype positive / genotype negative (and not pheno neg / geno pos). Also need to be careful of phenocopies. Also need rules for how many cases are needed in use this rule",
            "defaultStrength": {
              "id": "SEPIO:0000328",
              "type": "Entity",
              "label": "Benign Strong"
            }
          },
          "variant": {
            "id": "CAR:CA090930",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll130"
          },
          "condition": [
            {
              "id": "CGEX:GenCond056",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:648",
                  "type": "Entity",
                  "label": "noonan syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "type": "Entity",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn123",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:FamSeg076",
                  "type": "FamilyCosegregationStatement",
                  "condition": {
                    "id": "CGEX:GenCond056",
                    "type": "GeneticCondition",
                    "disease": [
                      {
                        "id": "Orphanet:648",
                        "type": "Entity",
                        "label": "noonan syndrome"
                      }
                    ],
                    "inheritancePattern": {
                      "id": "HP:0000006",
                      "type": "Entity",
                      "label": "Autosomal Dominant"
                    }
                  },
                  "canonicalAllele": {
                    "id": "CAR:CA090930",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll130"
                  },
                  "family": {
                    "id": "CGEX:Fam373",
                    "type": "Entity",
                    "label": "Family #011"
                  },
                  "phenotypePositiveAllelePositive": 0,
                  "phenotypeNegativeAllelePositive": 2,
                  "phenotypePositiveAlleleNegative": 0,
                  "inconsistentSegregationsObserved": {
                    "id": "SEPIO-CG:99011",
                    "type": "Entity",
                    "label": "present"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "3/14/2017 10:45 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000328",
        "type": "Entity",
        "label": "Benign Strong"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent007",
            "type": "Agent",
            "label": "Stefan DiMarino"
          },
          "contributionDate": "3/14/2017 10:45 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent008",
        "type": "Agent",
        "label": "Gene Geneticist"
      },
      "contributionDate": "8/14/2014 10:29 AM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ],
  "source": [
    "http://omim.org/entry/244400"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp060">CGEX:VarInterp060</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA273507">CAR:CA273507</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond049">CGEX:GenCond049</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26332-9">LOINC:LA26332-9</a></p> (Likely Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp060"><pre><code>{
  "id": "CGEX:VarInterp060",
  "type": "VariantPathogenicityInterpretation",
  "description": "The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic.",
  "variant": {
    "id": "CAR:CA273507",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll133"
  },
  "condition": [
    {
      "id": "CGEX:GenCond049",
      "type": "GeneticCondition",
      "description": "Rasopathy",
      "disease": [
        {
          "id": "Orphanet:98733",
          "type": "Entity",
          "label": "Noonan syndrome and noonan-related syndrom"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26332-9",
    "type": "Entity",
    "label": "Likely Pathogenic"
  },
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent010",
        "type": "Agent",
        "label": "ACME Molecular Lab"
      },
      "contributionDate": "6/12/2011 9:00 AM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp063">CGEX:VarInterp063</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The Lys483Gln variant in BRAF has been identified in one proband with clinical features of Cardio-facio-cutaneous syndrome, and was not identified in either of this individual's parents (LMM unpublished data). This variant has not been identified in large population studies. In addition, another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). Computational analyses (biochemical amino acid properties, conservation, AlignGVGD, PolyPhen2, and SIFT) suggest that the Lys483Gln variant may impact the normal function of the protein. In summary, this variant is likely pathogenic, though additional studies are required to fully establish its clinical significance.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:EvLn127">CGEX:EvLn127</a></p><p><a href="../../details/details.html#CGEX:EvLn129">CGEX:EvLn129</a></p><p><a href="../../details/details.html#CGEX:EvLn131">CGEX:EvLn131</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA280055">CAR:CA280055</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond070">CGEX:GenCond070</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26332-9">LOINC:LA26332-9</a></p> (Likely Pathogenic)
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:AssertMeth331">CGEX:AssertMeth331</a></p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp063"><pre><code>{
  "id": "CGEX:VarInterp063",
  "type": "VariantPathogenicityInterpretation",
  "description": "The Lys483Gln variant in BRAF has been identified in one proband with clinical features of Cardio-facio-cutaneous syndrome, and was not identified in either of this individual's parents (LMM unpublished data). This variant has not been identified in large population studies. In addition, another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). Computational analyses (biochemical amino acid properties, conservation, AlignGVGD, PolyPhen2, and SIFT) suggest that the Lys483Gln variant may impact the normal function of the protein. In summary, this variant is likely pathogenic, though additional studies are required to fully establish its clinical significance.",
  "variant": {
    "id": "CAR:CA280055",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll131"
  },
  "condition": [
    {
      "id": "CGEX:GenCond070",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:1340",
          "type": "Entity",
          "label": "cardiofaciocutaneous syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26332-9",
    "type": "Entity",
    "label": "Likely Pathogenic"
  },
  "assertionMethod": {
    "id": "CGEX:AssertMeth331",
    "type": "VariantPathogenicityInterpretationGuideline",
    "description": "ACMG ISV guidelines 2015",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "version": "1.0",
    "scoringAlgorithm": {
      "id": "SEPIO-CG:99300"
    }
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn127",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess376",
          "type": "CriterionAssessment",
          "criterion": {
            "id": "SEPIO-CG:99028",
            "type": "VariantInterpretationCriterion",
            "label": "PM2",
            "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
            "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
            "defaultStrength": {
              "id": "SEPIO:0000216",
              "type": "Entity",
              "label": "Pathogenic Moderate"
            }
          },
          "variant": {
            "id": "CAR:CA280055",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll131"
          },
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn126",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllFreq101",
                  "type": "PopulationAlleleFrequencyStatement",
                  "ascertainment": {
                    "id": "SEPIO:0000332",
                    "type": "Entity",
                    "label": "ExAC ascertainment method"
                  },
                  "population": {
                    "id": "SEPIO-CG:98001",
                    "type": "Entity",
                    "label": "Combined"
                  },
                  "allele": {
                    "id": "CAR:CA280055",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll131"
                  },
                  "alleleCount": 0,
                  "alleleFrequency": 0.0,
                  "medianCoverage": 57.0
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent010",
                "type": "Agent",
                "label": "ACME Molecular Lab"
              },
              "contributionDate": "6/11/2011 11:45 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000216",
        "type": "Entity",
        "label": "Pathogenic Moderate"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent010",
            "type": "Agent",
            "label": "ACME Molecular Lab"
          },
          "contributionDate": "6/11/2011 11:45 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn129",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess377",
          "type": "CriterionAssessment",
          "description": "Parental testing in our lab confirmed this variant was present de novo in 1 proband with clinical features of CFC. (LMM internal Data)",
          "criterion": {
            "id": "SEPIO-CG:99032",
            "type": "VariantInterpretationCriterion",
            "label": "PM6",
            "description": "Assumed de novo, but without confirmation of paternity and maternity",
            "defaultStrength": {
              "id": "SEPIO:0000216",
              "type": "Entity",
              "label": "Pathogenic Moderate"
            }
          },
          "variant": {
            "id": "CAR:CA280055",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll131"
          },
          "condition": [
            {
              "id": "CGEX:GenCond069",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "type": "Entity",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn128",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:IndAllInh106",
                  "type": "IndividualAlleleInheritanceStatement",
                  "canonicalAllele": {
                    "id": "CAR:CA280055",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll131"
                  },
                  "individual": {
                    "id": "CGEX:Indiv362",
                    "type": "Entity",
                    "label": "individual #030"
                  },
                  "parentalConfirmation": {
                    "id": "SEPIO-CG:99008",
                    "type": "Entity",
                    "label": "no parental confirmation"
                  },
                  "statementOutcome": [
                    {
                      "id": "GENO:0000880",
                      "type": "Entity",
                      "label": "de novo"
                    }
                  ]
                },
                {
                  "id": "CGEX:IndCond110",
                  "type": "IndividualConditionStatement",
                  "individual": {
                    "id": "CGEX:Indiv362",
                    "type": "Entity",
                    "label": "individual #030"
                  },
                  "condition": [
                    {
                      "id": "CGEX:GenCond069",
                      "type": "GeneticCondition",
                      "disease": [
                        {
                          "id": "Orphanet:1340",
                          "type": "Entity",
                          "label": "cardiofaciocutaneous syndrome"
                        }
                      ]
                    }
                  ],
                  "statementOutcome": {
                    "id": "SEPIO-CG:99009",
                    "type": "Entity",
                    "label": "affected"
                  }
                },
                {
                  "id": "CGEX:FamCond112",
                  "type": "FamilyConditionStatement",
                  "proband": {
                    "id": "CGEX:Indiv362",
                    "type": "Entity",
                    "label": "individual #030"
                  },
                  "condition": [
                    {
                      "id": "CGEX:GenCond069",
                      "type": "GeneticCondition",
                      "disease": [
                        {
                          "id": "Orphanet:1340",
                          "type": "Entity",
                          "label": "cardiofaciocutaneous syndrome"
                        }
                      ]
                    }
                  ],
                  "statementOutcome": {
                    "id": "SEPIO-CG:99010",
                    "type": "Entity",
                    "label": "unaffected"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent010",
                "type": "Agent",
                "label": "ACME Molecular Lab"
              },
              "contributionDate": "6/11/2011 12:45:00",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000216",
        "type": "Entity",
        "label": "Pathogenic Moderate"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent010",
            "type": "Agent",
            "label": "ACME Molecular Lab"
          },
          "contributionDate": "6/11/2011 12:45:00",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn131",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess378",
          "type": "CriterionAssessment",
          "description": "Another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). The other variant: NM_004333.4(BRAF):c.1449A>C (p.Lys483Asn) is classified as likely pathogenic by LMM for Rasopathy.",
          "criterion": {
            "id": "SEPIO-CG:99031",
            "type": "VariantInterpretationCriterion",
            "label": "PM5",
            "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
            "usageNotes": "Need to be sure that pathogenic variant exists at the residue in question. Sometimes discrepancy resolution may be required. Some thing like Variantexplorer.org may be useful.\nSometimes no assertion is provided in ClinVar ...may require curation by lit. search.",
            "defaultStrength": {
              "id": "SEPIO:0000216",
              "type": "Entity",
              "label": "Pathogenic Moderate"
            }
          },
          "variant": {
            "id": "CAR:CA280055",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll131"
          },
          "condition": [
            {
              "id": "CGEX:GenCond049",
              "type": "GeneticCondition",
              "description": "Rasopathy",
              "disease": [
                {
                  "id": "Orphanet:98733",
                  "type": "Entity",
                  "label": "Noonan syndrome and noonan-related syndrom"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn130",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:VarInterp060",
                  "type": "VariantPathogenicityInterpretation",
                  "description": "The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic.",
                  "variant": {
                    "id": "CAR:CA273507",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll133"
                  },
                  "condition": [
                    {
                      "id": "CGEX:GenCond049",
                      "type": "GeneticCondition",
                      "description": "Rasopathy",
                      "disease": [
                        {
                          "id": "Orphanet:98733",
                          "type": "Entity",
                          "label": "Noonan syndrome and noonan-related syndrom"
                        }
                      ]
                    }
                  ],
                  "statementOutcome": {
                    "id": "LOINC:LA26332-9",
                    "type": "Entity",
                    "label": "Likely Pathogenic"
                  },
                  "contribution": [
                    {
                      "type": "Contribution",
                      "agent": {
                        "id": "CGEX:Agent010",
                        "type": "Agent",
                        "label": "ACME Molecular Lab"
                      },
                      "contributionDate": "6/12/2011 9:00 AM",
                      "contributionRole": {
                        "id": "SEPIO:0000331",
                        "type": "Entity",
                        "label": "interpreter role"
                      }
                    }
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent010",
                "type": "Agent",
                "label": "ACME Molecular Lab"
              },
              "contributionDate": "6/10/2011 1:45 PM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000216",
        "type": "Entity",
        "label": "Pathogenic Moderate"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent010",
            "type": "Agent",
            "label": "ACME Molecular Lab"
          },
          "contributionDate": "6/10/2011 1:45 PM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent009",
        "type": "Agent",
        "label": "George Geneticist"
      },
      "contributionDate": "8/14/2014 10:29 AM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp150">CGEX:VarInterp150</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Variant has been classified as “Pathogenic” by Sharing Clinical Reports Project (SCV000054057.3)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA019369">CAR:CA019369</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond023">CGEX:GenCond023</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp150"><pre><code>{
  "id": "CGEX:VarInterp150",
  "type": "VariantPathogenicityInterpretation",
  "description": "Variant has been classified as “Pathogenic” by Sharing Clinical Reports Project (SCV000054057.3)",
  "variant": {
    "id": "CAR:CA019369",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll078"
  },
  "condition": [
    {
      "id": "CGEX:GenCond023",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "OMIM:612555",
          "type": "Entity",
          "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp151">CGEX:VarInterp151</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Variant has been classified as “Pathogenic” by OMIM (SCV000043718.1)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA128479">CAR:CA128479</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond024">CGEX:GenCond024</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp151"><pre><code>{
  "id": "CGEX:VarInterp151",
  "type": "VariantPathogenicityInterpretation",
  "description": "Variant has been classified as “Pathogenic” by OMIM (SCV000043718.1)",
  "variant": {
    "id": "CAR:CA128479",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll079"
  },
  "condition": [
    {
      "id": "CGEX:GenCond024",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "OMIM:614065",
          "type": "Entity",
          "label": "Myopathy, Distal, type 4"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp152">CGEX:VarInterp152</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Variant has been classified as “Pathogenic” by Expert panel CFTR (SCV000245981.1)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA276116">CAR:CA276116</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond025">CGEX:GenCond025</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp152"><pre><code>{
  "id": "CGEX:VarInterp152",
  "type": "VariantPathogenicityInterpretation",
  "description": "Variant has been classified as “Pathogenic” by Expert panel CFTR (SCV000245981.1)",
  "variant": {
    "id": "CAR:CA276116",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll080"
  },
  "condition": [
    {
      "id": "CGEX:GenCond025",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:586",
          "type": "Entity",
          "label": "Cystic Fibrosis"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "type": "Entity",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp255">CGEX:VarInterp255</a></p></td>
          <td>
              
         </td>
          <td>
              <p>classified by ENIGMA.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA276195">CAR:CA276195</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond073">CGEX:GenCond073</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6675-8">LOINC:LA6675-8</a></p> (Benign)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp255"><pre><code>{
  "id": "CGEX:VarInterp255",
  "type": "VariantPathogenicityInterpretation",
  "description": "classified by ENIGMA.",
  "variant": {
    "id": "CAR:CA276195",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll120"
  },
  "condition": [
    {
      "id": "CGEX:GenCond073",
      "type": "GeneticCondition",
      "description": "hereditary breast and ovarian cancer",
      "disease": [
        {
          "id": "Orphanet:145",
          "type": "Entity",
          "label": "hereditary breast and ovarian cancer syndrome"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6675-8",
    "type": "Entity",
    "label": "Benign"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp256">CGEX:VarInterp256</a></p></td>
          <td>
              
         </td>
          <td>
              <p>classified by Sharing Clinical Reports Project.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA016113">CAR:CA016113</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond043">CGEX:GenCond043</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6675-8">LOINC:LA6675-8</a></p> (Benign)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp256"><pre><code>{
  "id": "CGEX:VarInterp256",
  "type": "VariantPathogenicityInterpretation",
  "description": "classified by Sharing Clinical Reports Project.",
  "variant": {
    "id": "CAR:CA016113",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll121"
  },
  "condition": [
    {
      "id": "CGEX:GenCond043",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "OMIM:612555",
          "type": "Entity",
          "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6675-8",
    "type": "Entity",
    "label": "Benign"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp257">CGEX:VarInterp257</a></p></td>
          <td>
              
         </td>
          <td>
              <p>classified by OMIM</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA113946">CAR:CA113946</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond044">CGEX:GenCond044</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6675-8">LOINC:LA6675-8</a></p> (Benign)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp257"><pre><code>{
  "id": "CGEX:VarInterp257",
  "type": "VariantPathogenicityInterpretation",
  "description": "classified by OMIM",
  "variant": {
    "id": "CAR:CA113946",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll122"
  },
  "condition": [
    {
      "id": "CGEX:GenCond044",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:414",
          "type": "Entity",
          "label": "Gyrate atrophy of choroid and retina"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6675-8",
    "type": "Entity",
    "label": "Benign"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp270">CGEX:VarInterp270</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome is caused by a combination of a pathogenic variant in COL2A1 and another in COMP.   So CA113 may be involved, but is not the sole cause.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA254703">CAR:CA254703</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond041">CGEX:GenCond041</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26333-7">LOINC:LA26333-7</a></p> (Uncertain Significance)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp270"><pre><code>{
  "id": "CGEX:VarInterp270",
  "type": "VariantPathogenicityInterpretation",
  "description": "Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome is caused by a combination of a pathogenic variant in COL2A1 and another in COMP.   So CA113 may be involved, but is not the sole cause.",
  "variant": {
    "id": "CAR:CA254703",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll119"
  },
  "condition": [
    {
      "id": "CGEX:GenCond041",
      "type": "GeneticCondition",
      "description": "Pseudoachondroplastic spondyloepiphyseal dysplasia",
      "disease": [
        {
          "id": "Orphanet:750",
          "type": "Entity",
          "label": "pseudoachondroplasia"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26333-7",
    "type": "Entity",
    "label": "Uncertain Significance"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp274">CGEX:VarInterp274</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The Trp2490_Leu2496del variant in DNAH5 leads to an in-frame deletion of 7 amino acids. This variant has been reported together with a second DNAH5 variant (Met2083Ile) in one individual with PCD and situs inversus (Berg 2011). In addition, this variant has been identified in trans configuration with a disease-causing variant in one affected proband (LMM unpublished data). Data from large population studies is insufficient to determine whether this variant is present in the general population. In summary, this variant is likely pathogenic, though additional studies are required to fully establish its clinical significance.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:EvLn110">CGEX:EvLn110</a></p><p><a href="../../details/details.html#CGEX:EvLn111">CGEX:EvLn111</a></p><p><a href="../../details/details.html#CGEX:EvLn112">CGEX:EvLn112</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA090919">CAR:CA090919</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond063">CGEX:GenCond063</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26332-9">LOINC:LA26332-9</a></p> (Likely Pathogenic)
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:AssertMeth331">CGEX:AssertMeth331</a></p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp274"><pre><code>{
  "id": "CGEX:VarInterp274",
  "type": "VariantPathogenicityInterpretation",
  "description": "The Trp2490_Leu2496del variant in DNAH5 leads to an in-frame deletion of 7 amino acids. This variant has been reported together with a second DNAH5 variant (Met2083Ile) in one individual with PCD and situs inversus (Berg 2011). In addition, this variant has been identified in trans configuration with a disease-causing variant in one affected proband (LMM unpublished data). Data from large population studies is insufficient to determine whether this variant is present in the general population. In summary, this variant is likely pathogenic, though additional studies are required to fully establish its clinical significance.",
  "variant": {
    "id": "CAR:CA090919",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll039"
  },
  "condition": [
    {
      "id": "CGEX:GenCond063",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:244",
          "type": "Entity",
          "label": "Primary ciliary dyskenisia"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "type": "Entity",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26332-9",
    "type": "Entity",
    "label": "Likely Pathogenic"
  },
  "assertionMethod": {
    "id": "CGEX:AssertMeth331",
    "type": "VariantPathogenicityInterpretationGuideline",
    "description": "ACMG ISV guidelines 2015",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "version": "1.0",
    "scoringAlgorithm": {
      "id": "SEPIO-CG:99300"
    }
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn110",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess305",
          "type": "CriterionAssessment",
          "description": "while the variant is absent from exac, however\n1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”\n2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC).",
          "criterion": {
            "id": "SEPIO-CG:99028",
            "type": "VariantInterpretationCriterion",
            "label": "PM2",
            "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
            "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
            "defaultStrength": {
              "id": "SEPIO:0000216",
              "type": "Entity",
              "label": "Pathogenic Moderate"
            }
          },
          "variant": {
            "id": "CAR:CA090919",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll039"
          },
          "statementOutcome": {
            "id": "SEPIO:0000224",
            "type": "Entity",
            "label": "Not Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn040",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllFreq082",
                  "type": "PopulationAlleleFrequencyStatement",
                  "ascertainment": {
                    "id": "SEPIO:0000332",
                    "type": "Entity",
                    "label": "ExAC ascertainment method"
                  },
                  "population": {
                    "id": "SEPIO-CG:98001",
                    "type": "Entity",
                    "label": "Combined"
                  },
                  "allele": {
                    "id": "CAR:CA090919",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll039"
                  },
                  "alleleCount": 0,
                  "alleleFrequency": 0.0,
                  "medianCoverage": 11.0,
                  "contribution": [
                    {
                      "type": "Contribution",
                      "agent": {
                        "id": "CGEX:Agent001",
                        "type": "Agent",
                        "label": "Harry Stevenson"
                      },
                      "contributionDate": "6/1/16 2:15 PM",
                      "contributionRole": {
                        "id": "SEPIO:0000156",
                        "type": "Entity",
                        "label": "curator role"
                      }
                    }
                  ],
                  "source": [
                    "ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.2/"
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "7/1/16 10:53 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            },
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent004",
                "type": "Agent",
                "label": "CSER Bakeoff Project"
              },
              "contributionDate": "7/1/16 10:53 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000218",
        "type": "Entity",
        "label": "None"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent001",
            "type": "Agent",
            "label": "Harry Stevenson"
          },
          "contributionDate": "7/1/16 10:53 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        },
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent004",
            "type": "Agent",
            "label": "CSER Bakeoff Project"
          },
          "contributionDate": "7/1/16 10:53 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn111",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess296",
          "type": "CriterionAssessment",
          "criterion": {
            "id": "SEPIO-CG:99029",
            "type": "VariantInterpretationCriterion",
            "label": "PM3",
            "description": "For recessive disorders, detected in trans with a pathogenic variant",
            "usageNotes": "CSER - Laboratories discussed when to modify the strength of PM3, the variant is seen in trans with a pathogenic variant for recessive disorders. Published literature may not always explicitly state the phase of variants found in affected individuals which raises a challenge for invoking PM3.\nWhen phase has not been established, some felt that PM3 could be invoked as supporting evidence. Also, if the variant is seen in trans with a pathogenic variant in more than one individual it was felt that PM3 can be upgraded to strong. However, sites did not agree on how many additional observations were necessary to call the evidence strong (2 vs. 3) but concluded that such guidance would be useful.\n \nLMM - INCREASES IN STRENGTH IF VARIANT SEEN IN MULTIPLE COMPOUND HETS WITH DIFFERENT LIKELY PATHOGENIC/PATHOGENIC VARIANTS",
            "defaultStrength": {
              "id": "SEPIO:0000216",
              "type": "Entity",
              "label": "Pathogenic Moderate"
            }
          },
          "variant": {
            "id": "CAR:CA090919",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll039"
          },
          "condition": [
            {
              "id": "CGEX:GenCond062",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:244",
                  "type": "Entity",
                  "label": "Primary ciliary dyskenisia"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "type": "Entity",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn029",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:IndCond056",
                  "type": "IndividualConditionStatement",
                  "individual": {
                    "id": "CGEX:Indiv336",
                    "type": "Entity",
                    "label": "individual #004"
                  },
                  "condition": [
                    {
                      "id": "CGEX:GenCond014",
                      "type": "GeneticCondition",
                      "description": "PCD autosomal recessive w/ outer dynein arm (ODA) defects",
                      "disease": [
                        {
                          "id": "Orphanet:244",
                          "type": "Entity",
                          "label": "Primary ciliary dyskenisia"
                        }
                      ],
                      "inheritancePattern": {
                        "id": "HP:0000007",
                        "type": "Entity",
                        "label": "Autosomal Recessive"
                      },
                      "phenotype": [
                        {
                          "id": "HP:0200106",
                          "type": "Entity",
                          "label": "Absent/shortened dynein arms"
                        }
                      ]
                    }
                  ],
                  "statementOutcome": {
                    "id": "SEPIO-CG:99009",
                    "type": "Entity",
                    "label": "affected"
                  },
                  "contribution": [
                    {
                      "type": "Contribution",
                      "agent": {
                        "id": "CGEX:Agent007",
                        "type": "Agent",
                        "label": "Stefan DiMarino"
                      },
                      "contributionDate": "5/31/16 8:46 AM",
                      "contributionRole": {
                        "id": "SEPIO:0000156",
                        "type": "Entity",
                        "label": "curator role"
                      }
                    }
                  ],
                  "source": [
                    "http://partners.org/ppm/lmm/gi/case/123"
                  ]
                },
                {
                  "id": "CGEX:VarInterp057",
                  "type": "VariantPathogenicityInterpretation",
                  "variant": {
                    "id": "CAR:CA273615",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll040"
                  },
                  "condition": [
                    {
                      "id": "CGEX:GenCond062",
                      "type": "GeneticCondition",
                      "disease": [
                        {
                          "id": "Orphanet:244",
                          "type": "Entity",
                          "label": "Primary ciliary dyskenisia"
                        }
                      ],
                      "inheritancePattern": {
                        "id": "HP:0000007",
                        "type": "Entity",
                        "label": "Autosomal Recessive"
                      }
                    }
                  ],
                  "statementOutcome": {
                    "id": "LOINC:LA6668-3",
                    "type": "Entity",
                    "label": "Pathogenic"
                  },
                  "contribution": [
                    {
                      "type": "Contribution",
                      "agent": {
                        "id": "CGEX:Agent007",
                        "type": "Agent",
                        "label": "Stefan DiMarino"
                      },
                      "contributionDate": "5/31/16 11:01 AM",
                      "contributionRole": {
                        "id": "SEPIO:0000331",
                        "type": "Entity",
                        "label": "interpreter role"
                      }
                    }
                  ],
                  "source": [
                    "https://www.ncbi.nlm.nih.gov/clinvar/SCV000205842.3"
                  ]
                },
                {
                  "id": "CGEX:IndGeno058",
                  "type": "IndividualGenotypeStatement",
                  "description": "Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband.",
                  "individual": {
                    "id": "CGEX:Indiv336",
                    "type": "Entity",
                    "label": "individual #004"
                  },
                  "genotype": [
                    {
                      "id": "CGEX:Geno378",
                      "type": "Genotype",
                      "haplotype": [
                        {
                          "id": "CGEX:Haplo376",
                          "type": "Haplotype",
                          "allele": [
                            {
                              "id": "CAR:CA090919",
                              "type": "CanonicalAllele",
                              "relatedContextualAllele": "CGEX:CtxAll039"
                            }
                          ]
                        },
                        {
                          "id": "CGEX:Haplo377",
                          "type": "Haplotype",
                          "allele": [
                            {
                              "id": "CAR:CA273615",
                              "type": "CanonicalAllele",
                              "relatedContextualAllele": "CGEX:CtxAll040"
                            }
                          ]
                        }
                      ]
                    }
                  ],
                  "contribution": [
                    {
                      "type": "Contribution",
                      "agent": {
                        "id": "CGEX:Agent001",
                        "type": "Agent",
                        "label": "Harry Stevenson"
                      },
                      "contributionDate": "5/31/16 9:15 AM",
                      "contributionRole": {
                        "id": "SEPIO:0000156",
                        "type": "Entity",
                        "label": "curator role"
                      }
                    }
                  ],
                  "source": [
                    "http://partners.org/ppm/lmm/gi/case/123"
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "6/2/16 8:01 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000216",
        "type": "Entity",
        "label": "Pathogenic Moderate"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent001",
            "type": "Agent",
            "label": "Harry Stevenson"
          },
          "contributionDate": "6/2/16 8:01 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn112",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess299",
          "type": "CriterionAssessment",
          "description": "leads to an in-frame deletion of 7 amino acids (not repeat region).",
          "criterion": {
            "id": "SEPIO-CG:99030",
            "type": "VariantInterpretationCriterion",
            "label": "PM4",
            "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
            "usageNotes": "Need to ensure that variant is not located within a repeat region\n\nThis rule is also (likely) applicable for frameshift variants that result in extension of the protein as opposed to truncation (similar to stop loss).",
            "defaultStrength": {
              "id": "SEPIO:0000216",
              "type": "Entity",
              "label": "Pathogenic Moderate"
            }
          },
          "variant": {
            "id": "CAR:CA090919",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll039"
          },
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn032",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:MolCon066",
                  "type": "AlleleMolecularConsequenceStatement",
                  "contextualAllele": {
                    "id": "CGEX:CtxAll044",
                    "type": "ContextualAllele",
                    "canonicalAllele": {
                      "id": "CAR:CA273136",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll044"
                    },
                    "alleleName": {
                      "id": "CGEX:CtxNm9044",
                      "type": "ContextualAlleleName",
                      "name": "NP_001360.1:p.Trp2490_Leu2496del",
                      "preferred": true
                    }
                  },
                  "statementOutcome": {
                    "id": "SO:0001825",
                    "type": "Entity",
                    "label": "conservative Inframe Deletion"
                  },
                  "contribution": [
                    {
                      "type": "Contribution",
                      "agent": {
                        "id": "CGEX:Agent001",
                        "type": "Agent",
                        "label": "Harry Stevenson"
                      },
                      "contributionDate": "6/1/16 2:03 PM",
                      "contributionRole": {
                        "id": "SEPIO:0000156",
                        "type": "Entity",
                        "label": "curator role"
                      }
                    }
                  ],
                  "source": [
                    "https://geneinsight-lmm-test.partners.org"
                  ]
                },
                {
                  "id": "CGEX:RegTyp069",
                  "type": "RegionTypeStatement",
                  "description": "This is not a repreating region.",
                  "region": {
                    "id": "CGEX:Loc435",
                    "type": "SequenceLocation",
                    "referenceSequence": {
                      "id": "REFSEQ:NP_001360.1",
                      "type": "Entity",
                      "label": "NP_001360.1"
                    },
                    "start": 2490,
                    "stop": 2496
                  },
                  "statementOutcome": [
                    {
                      "id": "SO:0000657",
                      "type": "Entity",
                      "label": "repeat region "
                    }
                  ],
                  "outcomeQualifier": "NOT",
                  "contribution": [
                    {
                      "type": "Contribution",
                      "agent": {
                        "id": "CGEX:Agent007",
                        "type": "Agent",
                        "label": "Stefan DiMarino"
                      },
                      "contributionDate": "6/1/16 2:10 PM",
                      "contributionRole": {
                        "id": "SEPIO:0000156",
                        "type": "Entity",
                        "label": "curator role"
                      }
                    }
                  ],
                  "source": [
                    "https://pcpgm.partners.org/lmm/alamut"
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent003",
                "type": "Agent",
                "label": "Sasha Rande"
              },
              "contributionDate": "6/12/16 7:30 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000216",
        "type": "Entity",
        "label": "Pathogenic Moderate"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent003",
            "type": "Agent",
            "label": "Sasha Rande"
          },
          "contributionDate": "6/12/16 7:30 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent009",
        "type": "Agent",
        "label": "George Geneticist"
      },
      "contributionDate": "6/26/2015 12:03 PM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp041">CGEX:VarInterp041</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Arg343His in Exon 08 of LOXHD1: This variant is not expected to have clinical significance because it has been identified in 2.0% (27/1384) of African American chromosomes by the NHLBI Exome Sequencing Project (http://evs.gs.washington.edu/EVS).</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:EvLn113">CGEX:EvLn113</a></p><p><a href="../../details/details.html#CGEX:EvLn114">CGEX:EvLn114</a></p><p><a href="../../details/details.html#CGEX:EvLn115">CGEX:EvLn115</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA090950">CAR:CA090950</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond061">CGEX:GenCond061</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6675-8">LOINC:LA6675-8</a></p> (Benign)
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:AssertMeth331">CGEX:AssertMeth331</a></p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp041"><pre><code>{
  "id": "CGEX:VarInterp041",
  "type": "VariantPathogenicityInterpretation",
  "description": "Arg343His in Exon 08 of LOXHD1: This variant is not expected to have clinical significance because it has been identified in 2.0% (27/1384) of African American chromosomes by the NHLBI Exome Sequencing Project (http://evs.gs.washington.edu/EVS).",
  "variant": {
    "id": "CAR:CA090950",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll098"
  },
  "condition": [
    {
      "id": "CGEX:GenCond061",
      "type": "GeneticCondition",
      "description": "Hearing Loss, Autosomal Recessive",
      "disease": [
        {
          "id": "Orphanet:87884",
          "type": "Entity",
          "label": "nonsyndromic deafness"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "type": "Entity",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6675-8",
    "type": "Entity",
    "label": "Benign"
  },
  "assertionMethod": {
    "id": "CGEX:AssertMeth331",
    "type": "VariantPathogenicityInterpretationGuideline",
    "description": "ACMG ISV guidelines 2015",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "version": "1.0",
    "scoringAlgorithm": {
      "id": "SEPIO-CG:99300"
    }
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn113",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess372",
          "type": "CriterionAssessment",
          "description": "This variant is present in 2% (27/1384) of African American Chromosomes in ESP.",
          "criterion": {
            "id": "SEPIO-CG:99039",
            "type": "VariantInterpretationCriterion",
            "label": "BS1",
            "description": "Allele frequency is greater than expected for disorder",
            "defaultStrength": {
              "id": "SEPIO:0000328",
              "type": "Entity",
              "label": "Benign Strong"
            }
          },
          "variant": {
            "id": "CAR:CA090950",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll098"
          },
          "condition": [
            {
              "id": "CGEX:GenCond057",
              "type": "GeneticCondition",
              "description": "Hearing Loss",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "type": "Entity",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn109",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllFreq038",
                  "type": "PopulationAlleleFrequencyStatement",
                  "ascertainment": {
                    "id": "SEPIO:0000332",
                    "type": "Entity",
                    "label": "ExAC ascertainment method"
                  },
                  "allele": {
                    "id": "CAR:CA090950",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll098"
                  },
                  "alleleCount": 27,
                  "alleleNumber": 1384,
                  "alleleFrequency": 0.02,
                  "population": {
                    "id": "GNOMAD:afr",
                    "type": "Entity",
                    "label": "African/African American"
                  }
                },
                {
                  "id": "CGEX:CondPrev039",
                  "type": "ConditionPrevelanceStatement",
                  "condition": [
                    {
                      "id": "CGEX:GenCond057",
                      "type": "GeneticCondition",
                      "description": "Hearing Loss",
                      "disease": [
                        {
                          "id": "Orphanet:87884",
                          "type": "Entity",
                          "label": "nonsyndromic deafness"
                        }
                      ]
                    }
                  ],
                  "minimum": 1.0,
                  "maximum": 1.0,
                  "population": {
                    "id": "SEPIO-CG:98002",
                    "type": "Entity",
                    "label": "Overall"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "03/08/2017 9:46 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000328",
        "type": "Entity",
        "label": "Benign Strong"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent007",
            "type": "Agent",
            "label": "Stefan DiMarino"
          },
          "contributionDate": "03/08/2017 9:46 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn114",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess340",
          "type": "CriterionAssessment",
          "description": "Homozygous truncating variants in LOXHD1 are associated with autosomal recessive hearing loss. This was demonstrated by both identification in human patients and a mouse model. (I used the OMIM entry as a source, but you could also directly use the papers.)",
          "criterion": {
            "id": "SEPIO-CG:99043",
            "type": "VariantInterpretationCriterion",
            "label": "BP1",
            "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
            "usageNotes": "Is there any guidance on the proportion of pathogenic variants that are LOF to apply this rule?  99%? 90%? 75%?",
            "defaultStrength": {
              "id": "SEPIO:0000327",
              "type": "Entity",
              "label": "Benign Supporting"
            }
          },
          "variant": {
            "id": "CAR:CA090950",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll098"
          },
          "condition": [
            {
              "id": "CGEX:GenCond059",
              "type": "GeneticCondition",
              "description": "Hearing Loss, Autosomal Recessive",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "type": "Entity",
                  "label": "nonsyndromic deafness"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "type": "Entity",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn083",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:MolCon201",
                  "type": "AlleleMolecularConsequenceStatement",
                  "contextualAllele": {
                    "id": "CGEX:CtxAll098",
                    "type": "ContextualAllele",
                    "canonicalAllele": {
                      "id": "CAR:CA090950",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll098"
                    },
                    "alleleName": {
                      "id": "CGEX:CtxNm9098",
                      "type": "ContextualAlleleName",
                      "name": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)",
                      "preferred": true
                    }
                  },
                  "statementOutcome": {
                    "id": "SO:0001583",
                    "type": "Entity",
                    "label": "missense variant"
                  }
                },
                {
                  "id": "CGEX:Null415",
                  "type": "NullAlleleStatement",
                  "contextualAllele": {
                    "id": "CGEX:CtxAll098",
                    "type": "ContextualAllele",
                    "canonicalAllele": {
                      "id": "CAR:CA090950",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll098"
                    },
                    "alleleName": {
                      "id": "CGEX:CtxNm9098",
                      "type": "ContextualAlleleName",
                      "name": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)",
                      "preferred": true
                    }
                  },
                  "statementOutcome": {
                    "id": "SEPIO-CG:99015",
                    "type": "Entity",
                    "label": "function not lost"
                  }
                },
                {
                  "id": "CGEX:CondMech202",
                  "type": "ConditionMechanismStatement",
                  "description": "shown in both humans and mouse models",
                  "statementOutcome": {
                    "id": "SO:0002054",
                    "type": "Entity",
                    "label": "loss of function variant"
                  },
                  "mechanismConfidence": {
                    "id": "SEPIO:0000270",
                    "type": "Entity",
                    "label": "primary"
                  },
                  "gene": {
                    "id": "HGNC:26521",
                    "type": "Entity",
                    "label": "LOXHD1"
                  },
                  "condition": {
                    "id": "CGEX:GenCond059",
                    "type": "GeneticCondition",
                    "description": "Hearing Loss, Autosomal Recessive",
                    "disease": [
                      {
                        "id": "Orphanet:87884",
                        "type": "Entity",
                        "label": "nonsyndromic deafness"
                      }
                    ],
                    "inheritancePattern": {
                      "id": "HP:0000007",
                      "type": "Entity",
                      "label": "Autosomal Recessive"
                    }
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "9/6/16 10:00 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000327",
        "type": "Entity",
        "label": "Benign Supporting"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent001",
            "type": "Agent",
            "label": "Harry Stevenson"
          },
          "contributionDate": "7/1/16 9:23 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn115",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess363",
          "type": "CriterionAssessment",
          "description": "This variant was identified in one individual with pathogenic variants in another gene. Internal LMM data: The proband had one pathogenic and one likely pathogenic variant in (in trans) in SLC26A4. This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome. SLC26A4 is the only gene associated with Pendred Syndrome",
          "criterion": {
            "id": "SEPIO-CG:99047",
            "type": "VariantInterpretationCriterion",
            "label": "BP5",
            "description": "Variant found in a case with an alternate molecular basis for disease",
            "defaultStrength": {
              "id": "SEPIO:0000327",
              "type": "Entity",
              "label": "Benign Supporting"
            }
          },
          "variant": {
            "id": "CAR:CA090950",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll098"
          },
          "condition": [
            {
              "id": "CGEX:GenCond058",
              "type": "GeneticCondition",
              "description": "Hearing Loss",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "type": "Entity",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn106",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:VarInterp364",
                  "type": "VariantPathogenicityInterpretation",
                  "variant": {
                    "id": "CAR:CA253304",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll128"
                  },
                  "condition": [
                    {
                      "id": "CGEX:GenCond058",
                      "type": "GeneticCondition",
                      "description": "Hearing Loss",
                      "disease": [
                        {
                          "id": "Orphanet:87884",
                          "type": "Entity",
                          "label": "nonsyndromic deafness"
                        }
                      ]
                    }
                  ],
                  "statementOutcome": {
                    "id": "LOINC:LA26332-9",
                    "type": "Entity",
                    "label": "Likely Pathogenic"
                  }
                },
                {
                  "id": "CGEX:VarInterp365",
                  "type": "VariantPathogenicityInterpretation",
                  "variant": {
                    "id": "CAR:CA273665",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll129"
                  },
                  "condition": [
                    {
                      "id": "CGEX:GenCond058",
                      "type": "GeneticCondition",
                      "description": "Hearing Loss",
                      "disease": [
                        {
                          "id": "Orphanet:87884",
                          "type": "Entity",
                          "label": "nonsyndromic deafness"
                        }
                      ]
                    }
                  ],
                  "statementOutcome": {
                    "id": "LOINC:LA6668-3",
                    "type": "Entity",
                    "label": "Pathogenic"
                  }
                },
                {
                  "id": "CGEX:IndGeno366",
                  "type": "IndividualGenotypeStatement",
                  "individual": {
                    "id": "CGEX:Indiv361",
                    "type": "Entity",
                    "label": "individual #029"
                  },
                  "genotype": [
                    {
                      "id": "CGEX:Geno428",
                      "type": "Genotype",
                      "haplotype": [
                        {
                          "id": "CGEX:Haplo426",
                          "type": "Haplotype",
                          "allele": [
                            {
                              "id": "CAR:CA090950",
                              "type": "CanonicalAllele",
                              "relatedContextualAllele": "CGEX:CtxAll098"
                            }
                          ]
                        },
                        {
                          "id": "CGEX:Haplo427",
                          "type": "Haplotype",
                          "allele": [
                            {
                              "id": "CAR:CA253304",
                              "type": "CanonicalAllele",
                              "relatedContextualAllele": "CGEX:CtxAll128"
                            }
                          ]
                        }
                      ]
                    },
                    {
                      "id": "CGEX:Geno431",
                      "type": "Genotype",
                      "haplotype": [
                        {
                          "id": "CGEX:Haplo429",
                          "type": "Haplotype",
                          "allele": [
                            {
                              "id": "CAR:CA090950",
                              "type": "CanonicalAllele",
                              "relatedContextualAllele": "CGEX:CtxAll098"
                            }
                          ]
                        },
                        {
                          "id": "CGEX:Haplo430",
                          "type": "Haplotype",
                          "allele": [
                            {
                              "id": "CAR:CA273665",
                              "type": "CanonicalAllele",
                              "relatedContextualAllele": "CGEX:CtxAll129"
                            }
                          ]
                        }
                      ]
                    }
                  ]
                },
                {
                  "id": "CGEX:IndCond368",
                  "type": "IndividualConditionStatement",
                  "description": "This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome.",
                  "individual": {
                    "id": "CGEX:Indiv361",
                    "type": "Entity",
                    "label": "individual #029"
                  },
                  "condition": [
                    {
                      "id": "CGEX:GenCond058",
                      "type": "GeneticCondition",
                      "description": "Hearing Loss",
                      "disease": [
                        {
                          "id": "Orphanet:87884",
                          "type": "Entity",
                          "label": "nonsyndromic deafness"
                        }
                      ]
                    }
                  ],
                  "statementOutcome": {
                    "id": "SEPIO-CG:99009",
                    "type": "Entity",
                    "label": "affected"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "03/07/2017 4:30 PM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000327",
        "type": "Entity",
        "label": "Benign Supporting"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent007",
            "type": "Agent",
            "label": "Stefan DiMarino"
          },
          "contributionDate": "03/07/2017 4:30 PM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent006",
        "type": "Agent",
        "label": "CSER classifier #6"
      },
      "contributionDate": "6/26/2015 12:06 PM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp042">CGEX:VarInterp042</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Uncertain Sig(not specified) NM_000363.4(TNNI3):c.484C>T (p.Arg162Trp)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA021738">CAR:CA021738</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26333-7">LOINC:LA26333-7</a></p> (Uncertain Significance)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp042"><pre><code>{
  "id": "CGEX:VarInterp042",
  "type": "VariantPathogenicityInterpretation",
  "description": "Uncertain Sig(not specified) NM_000363.4(TNNI3):c.484C>T (p.Arg162Trp)",
  "variant": {
    "id": "CAR:CA021738",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll053"
  },
  "statementOutcome": {
    "id": "LOINC:LA26333-7",
    "type": "Entity",
    "label": "Uncertain Significance"
  },
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/13/16 8:46 AM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ],
  "source": [
    "https://www.ncbi.nlm.nih.gov/clinvar/SCV000190642.1"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp044">CGEX:VarInterp044</a></p></td>
          <td>
              
         </td>
          <td>
              <p>p.Tyr109Tyr in exon 4 of MYH7: This variant is not expected to have clinical significance because it does not alter an amino acid residue and is not located near a splice junction. It has been identified in 0.2% (31/16512) of South Asian chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs36211408).</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:EvLn120">CGEX:EvLn120</a></p><p><a href="../../details/details.html#CGEX:EvLn118">CGEX:EvLn118</a></p><p><a href="../../details/details.html#CGEX:EvLn119">CGEX:EvLn119</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA013536">CAR:CA013536</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26334-5">LOINC:LA26334-5</a></p> (Likely Benign)
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:AssertMeth331">CGEX:AssertMeth331</a></p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp044"><pre><code>{
  "id": "CGEX:VarInterp044",
  "type": "VariantPathogenicityInterpretation",
  "description": "p.Tyr109Tyr in exon 4 of MYH7: This variant is not expected to have clinical significance because it does not alter an amino acid residue and is not located near a splice junction. It has been identified in 0.2% (31/16512) of South Asian chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs36211408).",
  "variant": {
    "id": "CAR:CA013536",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll124"
  },
  "statementOutcome": {
    "id": "LOINC:LA26334-5",
    "type": "Entity",
    "label": "Likely Benign"
  },
  "assertionMethod": {
    "id": "CGEX:AssertMeth331",
    "type": "VariantPathogenicityInterpretationGuideline",
    "description": "ACMG ISV guidelines 2015",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "version": "1.0",
    "scoringAlgorithm": {
      "id": "SEPIO-CG:99300"
    }
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn120",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess371",
          "type": "CriterionAssessment",
          "description": "This variant is present in 0.2% (31/16512) of South Asian Chromosomes in ExAC.",
          "criterion": {
            "id": "SEPIO-CG:99039",
            "type": "VariantInterpretationCriterion",
            "label": "BS1",
            "description": "Allele frequency is greater than expected for disorder",
            "defaultStrength": {
              "id": "SEPIO:0000328",
              "type": "Entity",
              "label": "Benign Strong"
            }
          },
          "variant": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn108",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllFreq037",
                  "type": "PopulationAlleleFrequencyStatement",
                  "ascertainment": {
                    "id": "SEPIO:0000332",
                    "type": "Entity",
                    "label": "ExAC ascertainment method"
                  },
                  "allele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "alleleCount": 31,
                  "alleleNumber": 16512,
                  "alleleFrequency": 0.02,
                  "population": {
                    "id": "GNOMAD:sas",
                    "type": "Entity",
                    "label": "South Asian"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "03/08/2017 7:43 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000328",
        "type": "Entity",
        "label": "Benign Strong"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent007",
            "type": "Agent",
            "label": "Stefan DiMarino"
          },
          "contributionDate": "03/08/2017 7:43 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn118",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess373",
          "type": "CriterionAssessment",
          "description": "This is a synonymous variant with little conservation and no predicted impact on splicing. AllConservation and splicing algorithm values are listed below.",
          "criterion": {
            "id": "SEPIO-CG:99046",
            "type": "VariantInterpretationCriterion",
            "label": "BP4",
            "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
            "usageNotes": "same issues as PP3.",
            "defaultStrength": {
              "id": "SEPIO:0000327",
              "type": "Entity",
              "label": "Benign Supporting"
            }
          },
          "variant": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn117",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:InSilicoPred068",
                  "type": "InSilicoPredictionStatement",
                  "description": "Provenance would show that this comes from Alamut",
                  "transcript": {
                    "id": "REFSEQ:NM_000257.2",
                    "type": "Entity",
                    "label": "NM_000257.2"
                  },
                  "canonicalAllele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "predictionType": {
                    "id": "SEPIO:0000323",
                    "type": "Entity",
                    "label": "splicing prediction"
                  },
                  "statementOutcome": "no impact"
                },
                {
                  "id": "CGEX:AllConserv071",
                  "type": "AlleleConservationStatement",
                  "allele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "statementOutcome": {
                    "id": "SEPIO-CG:99017",
                    "type": "Entity",
                    "label": "not conserved region"
                  },
                  "algorithm": "phyloP",
                  "evidenceLine": [
                    {
                      "id": "CGEX:EvLn133",
                      "type": "EvidenceLine",
                      "evidenceItem": [
                        {
                          "id": "CGEX:AllConservScore116",
                          "type": "AlleleConservationScoreStatement",
                          "allele": {
                            "id": "CAR:CA013536",
                            "type": "CanonicalAllele",
                            "relatedContextualAllele": "CGEX:CtxAll124"
                          },
                          "algorithm": "phyloP",
                          "score": -0.76
                        }
                      ]
                    }
                  ]
                },
                {
                  "id": "CGEX:AllConserv072",
                  "type": "AlleleConservationStatement",
                  "allele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "statementOutcome": {
                    "id": "SEPIO-CG:99017",
                    "type": "Entity",
                    "label": "not conserved region"
                  },
                  "algorithm": "PhastCons",
                  "evidenceLine": [
                    {
                      "id": "CGEX:EvLn134",
                      "type": "EvidenceLine",
                      "evidenceItem": [
                        {
                          "id": "CGEX:AllConservScore171",
                          "type": "AlleleConservationScoreStatement",
                          "allele": {
                            "id": "CAR:CA013536",
                            "type": "CanonicalAllele",
                            "relatedContextualAllele": "CGEX:CtxAll124"
                          },
                          "algorithm": "PhastCons",
                          "score": 0.63
                        }
                      ]
                    }
                  ]
                },
                {
                  "id": "CGEX:MolCon075",
                  "type": "AlleleMolecularConsequenceStatement",
                  "statementOutcome": {
                    "id": "SO:0001819",
                    "type": "Entity",
                    "label": "synonymous variant"
                  },
                  "contextualAllele": {
                    "id": "CGEX:CtxAll124",
                    "type": "ContextualAllele",
                    "canonicalAllele": {
                      "id": "CAR:CA013536",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll124"
                    },
                    "alleleName": {
                      "id": "CGEX:CtxNm9124",
                      "type": "ContextualAlleleName",
                      "name": "NM_000257.2(MYH7):c.327C>T",
                      "preferred": true
                    }
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "03/08/2017 10:02 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000327",
        "type": "Entity",
        "label": "Benign Supporting"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent007",
            "type": "Agent",
            "label": "Stefan DiMarino"
          },
          "contributionDate": "03/08/2017 10:02 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn119",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess359",
          "type": "CriterionAssessment",
          "criterion": {
            "id": "SEPIO-CG:99049",
            "type": "VariantInterpretationCriterion",
            "label": "BP7",
            "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
            "defaultStrength": {
              "id": "SEPIO:0000327",
              "type": "Entity",
              "label": "Benign Supporting"
            }
          },
          "variant": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn101",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:InSilicoPred262",
                  "type": "InSilicoPredictionStatement",
                  "description": "Provenance would show that this comes from Alamut",
                  "transcript": {
                    "id": "REFSEQ:NM_000257.2",
                    "type": "Entity",
                    "label": "NM_000257.2"
                  },
                  "canonicalAllele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "predictionType": {
                    "id": "SEPIO:0000323",
                    "type": "Entity",
                    "label": "splicing prediction"
                  },
                  "statementOutcome": "no impact"
                },
                {
                  "id": "CGEX:AllConserv263",
                  "type": "AlleleConservationStatement",
                  "allele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "statementOutcome": {
                    "id": "SEPIO-CG:99017",
                    "type": "Entity",
                    "label": "not conserved region"
                  },
                  "algorithm": "phyloP",
                  "evidenceLine": [
                    {
                      "id": "CGEX:EvLn137",
                      "type": "EvidenceLine",
                      "evidenceItem": [
                        {
                          "id": "CGEX:AllConservScore230",
                          "type": "AlleleConservationScoreStatement",
                          "allele": {
                            "id": "CAR:CA013536",
                            "type": "CanonicalAllele",
                            "relatedContextualAllele": "CGEX:CtxAll124"
                          },
                          "algorithm": "phyloP",
                          "score": -0.76
                        }
                      ]
                    }
                  ]
                },
                {
                  "id": "CGEX:AllConserv264",
                  "type": "AlleleConservationStatement",
                  "allele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "statementOutcome": {
                    "id": "SEPIO-CG:99017",
                    "type": "Entity",
                    "label": "not conserved region"
                  },
                  "algorithm": "PhastCons",
                  "evidenceLine": [
                    {
                      "id": "CGEX:EvLn138",
                      "type": "EvidenceLine",
                      "evidenceItem": [
                        {
                          "id": "CGEX:AllConservScore226",
                          "type": "AlleleConservationScoreStatement",
                          "allele": {
                            "id": "CAR:CA013536",
                            "type": "CanonicalAllele",
                            "relatedContextualAllele": "CGEX:CtxAll124"
                          },
                          "algorithm": "PhastCons",
                          "score": 0.63
                        }
                      ]
                    }
                  ]
                },
                {
                  "id": "CGEX:MolCon265",
                  "type": "AlleleMolecularConsequenceStatement",
                  "statementOutcome": {
                    "id": "SO:0001819",
                    "type": "Entity",
                    "label": "synonymous variant"
                  },
                  "contextualAllele": {
                    "id": "CGEX:CtxAll124",
                    "type": "ContextualAllele",
                    "canonicalAllele": {
                      "id": "CAR:CA013536",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll124"
                    },
                    "alleleName": {
                      "id": "CGEX:CtxNm9124",
                      "type": "ContextualAlleleName",
                      "name": "NM_000257.2(MYH7):c.327C>T",
                      "preferred": true
                    }
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "10/10/16 8:05 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000327",
        "type": "Entity",
        "label": "Benign Supporting"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent007",
            "type": "Agent",
            "label": "Stefan DiMarino"
          },
          "contributionDate": "10/10/16 8:05 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent006",
        "type": "Agent",
        "label": "CSER classifier #6"
      },
      "contributionDate": "6/26/2015 12:06 PM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp045">CGEX:VarInterp045</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The c.631-2A>G variant in DSC2 has been reported in 1 Caucasian individual with ARVC (Heuser 2006). This variant has also been identified by our laboratory in 2 individuals (1 with ARVC and 1 with LVNC). This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042). This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/). In summary, although additional studies are required to fully establish its clinical significance, the c.631-2A>G variant is likely pathogenic.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:EvLn121">CGEX:EvLn121</a></p><p><a href="../../details/details.html#CGEX:EvLn122">CGEX:EvLn122</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA022880">CAR:CA022880</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond074">CGEX:GenCond074</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26332-9">LOINC:LA26332-9</a></p> (Likely Pathogenic)
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:AssertMeth331">CGEX:AssertMeth331</a></p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp045"><pre><code>{
  "id": "CGEX:VarInterp045",
  "type": "VariantPathogenicityInterpretation",
  "description": "The c.631-2A>G variant in DSC2 has been reported in 1 Caucasian individual with ARVC (Heuser 2006). This variant has also been identified by our laboratory in 2 individuals (1 with ARVC and 1 with LVNC). This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042). This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/). In summary, although additional studies are required to fully establish its clinical significance, the c.631-2A>G variant is likely pathogenic.",
  "variant": {
    "id": "CAR:CA022880",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll127"
  },
  "condition": [
    {
      "id": "CGEX:GenCond074",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:247",
          "type": "Entity",
          "label": "arrhythmogenic right ventricular cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26332-9",
    "type": "Entity",
    "label": "Likely Pathogenic"
  },
  "assertionMethod": {
    "id": "CGEX:AssertMeth331",
    "type": "VariantPathogenicityInterpretationGuideline",
    "description": "ACMG ISV guidelines 2015",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "version": "1.0",
    "scoringAlgorithm": {
      "id": "SEPIO-CG:99300"
    }
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn121",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess361",
          "type": "CriterionAssessment",
          "description": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID:17186466). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).",
          "criterion": {
            "id": "SEPIO-CG:99022",
            "type": "VariantInterpretationCriterion",
            "label": "PVS1",
            "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
            "usageNotes": "The two main concerns for applying this rule is:\na. Is LOF an established disease mechanism?\nb. Is LOF actually occurring?",
            "defaultStrength": {
              "id": "SEPIO:0000220",
              "type": "Entity",
              "label": "Pathogenic Very Strong"
            }
          },
          "variant": {
            "id": "CAR:CA022880",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll127"
          },
          "condition": [
            {
              "id": "CGEX:GenCond046",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:247",
                  "type": "Entity",
                  "label": "arrhythmogenic right ventricular cardiomyopathy"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn103",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:MolCon272",
                  "type": "AlleleMolecularConsequenceStatement",
                  "description": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID:17186466).",
                  "contextualAllele": {
                    "id": "CGEX:CtxAll127",
                    "type": "ContextualAllele",
                    "canonicalAllele": {
                      "id": "CAR:CA022880",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll127"
                    },
                    "alleleName": {
                      "id": "CGEX:CtxNm9127",
                      "type": "ContextualAlleleName",
                      "name": "NM_024422.3(DSC2): c.631-2A>G (p.?)",
                      "preferred": true
                    }
                  },
                  "statementOutcome": {
                    "id": "SO:0001629",
                    "type": "Entity",
                    "label": "splice-site variant"
                  },
                  "source": [
                    "PMID:17186466"
                  ]
                },
                {
                  "id": "CGEX:CondMech273",
                  "type": "ConditionMechanismStatement",
                  "description": "Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).",
                  "statementOutcome": {
                    "id": "SO:0002054",
                    "type": "Entity",
                    "label": "loss of function variant"
                  },
                  "condition": {
                    "id": "CGEX:GenCond046",
                    "type": "GeneticCondition",
                    "disease": [
                      {
                        "id": "Orphanet:247",
                        "type": "Entity",
                        "label": "arrhythmogenic right ventricular cardiomyopathy"
                      }
                    ]
                  },
                  "mechanismConfidence": {
                    "id": "SEPIO:0000269",
                    "type": "Entity",
                    "label": "established"
                  },
                  "gene": {
                    "id": "HGNC:3036",
                    "type": "Entity",
                    "label": "DSC2"
                  }
                },
                {
                  "id": "CGEX:Null404",
                  "type": "NullAlleleStatement",
                  "contextualAllele": {
                    "id": "CGEX:CtxAll127",
                    "type": "ContextualAllele",
                    "canonicalAllele": {
                      "id": "CAR:CA022880",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll127"
                    },
                    "alleleName": {
                      "id": "CGEX:CtxNm9127",
                      "type": "ContextualAlleleName",
                      "name": "NM_024422.3(DSC2): c.631-2A>G (p.?)",
                      "preferred": true
                    }
                  },
                  "statementOutcome": {
                    "id": "SO:0002055",
                    "type": "Entity",
                    "label": "null mutation"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "01/17/17 9:12 AM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ],
          "source": [
            "PMID:17186466"
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000220",
        "type": "Entity",
        "label": "Pathogenic Very Strong"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent007",
            "type": "Agent",
            "label": "Stefan DiMarino"
          },
          "contributionDate": "01/17/17 9:12 AM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn122",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CritAssess369",
          "type": "CriterionAssessment",
          "description": "This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042).",
          "criterion": {
            "id": "SEPIO-CG:99028",
            "type": "VariantInterpretationCriterion",
            "label": "PM2",
            "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
            "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
            "defaultStrength": {
              "id": "SEPIO:0000216",
              "type": "Entity",
              "label": "Pathogenic Moderate"
            }
          },
          "variant": {
            "id": "CAR:CA022880",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll127"
          },
          "statementOutcome": {
            "id": "SEPIO:0000223",
            "type": "Entity",
            "label": "Met"
          },
          "evidenceLine": [
            {
              "id": "CGEX:EvLn107",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllFreq370",
                  "type": "PopulationAlleleFrequencyStatement",
                  "ascertainment": {
                    "id": "SEPIO:0000332",
                    "type": "Entity",
                    "label": "ExAC ascertainment method"
                  },
                  "population": {
                    "id": "GNOMAD:nfe",
                    "type": "Entity",
                    "label": "Non-Finnish European"
                  },
                  "allele": {
                    "id": "CAR:CA022880",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll127"
                  },
                  "alleleCount": 1,
                  "alleleNumber": 66564,
                  "alleleFrequency": 1.5e-05,
                  "homozygousAlleleIndividualCount": 0
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "03/07/2017 5:22 PM",
              "contributionRole": {
                "id": "SEPIO:0000154",
                "type": "Entity",
                "label": "assessor role"
              }
            }
          ]
        }
      ],
      "evidenceStrength": {
        "id": "SEPIO:0000216",
        "type": "Entity",
        "label": "Pathogenic Moderate"
      },
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CGEX:Agent001",
            "type": "Agent",
            "label": "Harry Stevenson"
          },
          "contributionDate": "03/07/2017 5:22 PM",
          "contributionRole": {
            "id": "SEPIO:0000154",
            "type": "Entity",
            "label": "assessor role"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent006",
        "type": "Agent",
        "label": "CSER classifier #6"
      },
      "contributionDate": "6/26/2015 12:06 PM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp047">CGEX:VarInterp047</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA022141">CAR:CA022141</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond013">CGEX:GenCond013</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp047"><pre><code>{
  "id": "CGEX:VarInterp047",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA022141",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll031"
  },
  "condition": [
    {
      "id": "CGEX:GenCond013",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:324",
          "type": "Entity",
          "label": "Fabry disease"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0001419",
        "type": "Entity",
        "label": "X linked recessive inheritance"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp048">CGEX:VarInterp048</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA022153">CAR:CA022153</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond013">CGEX:GenCond013</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp048"><pre><code>{
  "id": "CGEX:VarInterp048",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA022153",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll032"
  },
  "condition": [
    {
      "id": "CGEX:GenCond013",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:324",
          "type": "Entity",
          "label": "Fabry disease"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0001419",
        "type": "Entity",
        "label": "X linked recessive inheritance"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp049">CGEX:VarInterp049</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA022159">CAR:CA022159</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond013">CGEX:GenCond013</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp049"><pre><code>{
  "id": "CGEX:VarInterp049",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA022159",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll033"
  },
  "condition": [
    {
      "id": "CGEX:GenCond013",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:324",
          "type": "Entity",
          "label": "Fabry disease"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0001419",
        "type": "Entity",
        "label": "X linked recessive inheritance"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp050">CGEX:VarInterp050</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA022189">CAR:CA022189</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond013">CGEX:GenCond013</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp050"><pre><code>{
  "id": "CGEX:VarInterp050",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA022189",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll034"
  },
  "condition": [
    {
      "id": "CGEX:GenCond013",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:324",
          "type": "Entity",
          "label": "Fabry disease"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0001419",
        "type": "Entity",
        "label": "X linked recessive inheritance"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp057">CGEX:VarInterp057</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA273615">CAR:CA273615</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond062">CGEX:GenCond062</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp057"><pre><code>{
  "id": "CGEX:VarInterp057",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA273615",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll040"
  },
  "condition": [
    {
      "id": "CGEX:GenCond062",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:244",
          "type": "Entity",
          "label": "Primary ciliary dyskenisia"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "type": "Entity",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  },
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "5/31/16 11:01 AM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ],
  "source": [
    "https://www.ncbi.nlm.nih.gov/clinvar/SCV000205842.3"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp059">CGEX:VarInterp059</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA501223">CAR:CA501223</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond010">CGEX:GenCond010</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp059"><pre><code>{
  "id": "CGEX:VarInterp059",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA501223",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll042"
  },
  "condition": [
    {
      "id": "CGEX:GenCond010",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:87884",
          "type": "Entity",
          "label": "nonsyndromic deafness"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "type": "Entity",
        "label": "Autosomal Recessive"
      },
      "gene": {
        "id": "HGNC:18183",
        "type": "Entity",
        "label": "GIPC3"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp078">CGEX:VarInterp078</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA021749">CAR:CA021749</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond016">CGEX:GenCond016</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26332-9">LOINC:LA26332-9</a></p> (Likely Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp078"><pre><code>{
  "id": "CGEX:VarInterp078",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA021749",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll051"
  },
  "condition": [
    {
      "id": "CGEX:GenCond016",
      "type": "GeneticCondition",
      "description": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant",
      "disease": [
        {
          "id": "Orphanet:217569",
          "type": "Entity",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26332-9",
    "type": "Entity",
    "label": "Likely Pathogenic"
  },
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent011",
        "type": "Agent",
        "label": "Genetics Diagnostic Laboratory,Children's Hospital of Eastern Ontario"
      },
      "contributionDate": "6/1/2014 9:02 AM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ],
  "source": [
    "https://www.ncbi.nlm.nih.gov/clinvar/SCV000059953.4"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp079">CGEX:VarInterp079</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA021738">CAR:CA021738</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond016">CGEX:GenCond016</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp079"><pre><code>{
  "id": "CGEX:VarInterp079",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA021738",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll053"
  },
  "condition": [
    {
      "id": "CGEX:GenCond016",
      "type": "GeneticCondition",
      "description": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant",
      "disease": [
        {
          "id": "Orphanet:217569",
          "type": "Entity",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  },
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent012",
        "type": "Agent",
        "label": "GeneDx"
      },
      "contributionDate": "6/20/2016 11:12 AM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "type": "Entity",
        "label": "interpreter role"
      }
    }
  ],
  "source": [
    "https://www.ncbi.nlm.nih.gov/clinvar/SCV000203864.3"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp080">CGEX:VarInterp080</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA224454">CAR:CA224454</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond017">CGEX:GenCond017</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp080"><pre><code>{
  "id": "CGEX:VarInterp080",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA224454",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll057"
  },
  "condition": [
    {
      "id": "CGEX:GenCond017",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:664",
          "type": "Entity",
          "label": "Ornithine carbamoyltransferase deficiency"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp081">CGEX:VarInterp081</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA003449">CAR:CA003449</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond050">CGEX:GenCond050</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp081"><pre><code>{
  "id": "CGEX:VarInterp081",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA003449",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll061"
  },
  "condition": [
    {
      "id": "CGEX:GenCond050",
      "type": "GeneticCondition",
      "description": "Breast Ovarian Cancer, Autosomal Dominant",
      "disease": [
        {
          "id": "MESH:D061325",
          "type": "Entity",
          "label": "hereditary breast ovarian cancer"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp210">CGEX:VarInterp210</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA252233">CAR:CA252233</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond038">CGEX:GenCond038</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp210"><pre><code>{
  "id": "CGEX:VarInterp210",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA252233",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll101"
  },
  "condition": [
    {
      "id": "CGEX:GenCond038",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:791",
          "type": "Entity",
          "label": "Retinitis Pigmentosa"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp213">CGEX:VarInterp213</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA182839">CAR:CA182839</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond039">CGEX:GenCond039</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp213"><pre><code>{
  "id": "CGEX:VarInterp213",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA182839",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll103"
  },
  "condition": [
    {
      "id": "CGEX:GenCond039",
      "type": "GeneticCondition",
      "phenotype": [
        {
          "id": "HP:0012873",
          "type": "Entity",
          "label": "Absent vas deferens"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp216">CGEX:VarInterp216</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA501236">CAR:CA501236</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond007">CGEX:GenCond007</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26333-7">LOINC:LA26333-7</a></p> (Uncertain Significance)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp216"><pre><code>{
  "id": "CGEX:VarInterp216",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA501236",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll105"
  },
  "condition": [
    {
      "id": "CGEX:GenCond007",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:167848",
          "type": "Entity",
          "label": "cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "type": "Entity",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26333-7",
    "type": "Entity",
    "label": "Uncertain Significance"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp243">CGEX:VarInterp243</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA012381">CAR:CA012381</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond067">CGEX:GenCond067</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26332-9">LOINC:LA26332-9</a></p> (Likely Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp243"><pre><code>{
  "id": "CGEX:VarInterp243",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA012381",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll112"
  },
  "condition": [
    {
      "id": "CGEX:GenCond067",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217569",
          "type": "Entity",
          "label": "hypertrophic cardiomyopathy"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26332-9",
    "type": "Entity",
    "label": "Likely Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp247">CGEX:VarInterp247</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA172240">CAR:CA172240</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond060">CGEX:GenCond060</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26332-9">LOINC:LA26332-9</a></p> (Likely Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp247"><pre><code>{
  "id": "CGEX:VarInterp247",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA172240",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll115"
  },
  "condition": [
    {
      "id": "CGEX:GenCond060",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:87884",
          "type": "Entity",
          "label": "nonsyndromic deafness"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26332-9",
    "type": "Entity",
    "label": "Likely Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp248">CGEX:VarInterp248</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA274460">CAR:CA274460</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond060">CGEX:GenCond060</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26332-9">LOINC:LA26332-9</a></p> (Likely Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp248"><pre><code>{
  "id": "CGEX:VarInterp248",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA274460",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll116"
  },
  "condition": [
    {
      "id": "CGEX:GenCond060",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:87884",
          "type": "Entity",
          "label": "nonsyndromic deafness"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26332-9",
    "type": "Entity",
    "label": "Likely Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp251">CGEX:VarInterp251</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA137502">CAR:CA137502</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond071">CGEX:GenCond071</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp251"><pre><code>{
  "id": "CGEX:VarInterp251",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA137502",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll114"
  },
  "condition": [
    {
      "id": "CGEX:GenCond071",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:886",
          "type": "Entity",
          "label": "Usher syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "type": "Entity",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp254">CGEX:VarInterp254</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA254703">CAR:CA254703</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond042">CGEX:GenCond042</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp254"><pre><code>{
  "id": "CGEX:VarInterp254",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA254703",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll119"
  },
  "condition": [
    {
      "id": "CGEX:GenCond042",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:750",
          "type": "Entity",
          "label": "pseudoachondroplasia"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp364">CGEX:VarInterp364</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA253304">CAR:CA253304</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond058">CGEX:GenCond058</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA26332-9">LOINC:LA26332-9</a></p> (Likely Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp364"><pre><code>{
  "id": "CGEX:VarInterp364",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA253304",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll128"
  },
  "condition": [
    {
      "id": "CGEX:GenCond058",
      "type": "GeneticCondition",
      "description": "Hearing Loss",
      "disease": [
        {
          "id": "Orphanet:87884",
          "type": "Entity",
          "label": "nonsyndromic deafness"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA26332-9",
    "type": "Entity",
    "label": "Likely Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:VarInterp365">CGEX:VarInterp365</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA273665">CAR:CA273665</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:GenCond058">CGEX:GenCond058</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#LOINC:LA6668-3">LOINC:LA6668-3</a></p> (Pathogenic)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:VarInterp365"><pre><code>{
  "id": "CGEX:VarInterp365",
  "type": "VariantPathogenicityInterpretation",
  "variant": {
    "id": "CAR:CA273665",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll129"
  },
  "condition": [
    {
      "id": "CGEX:GenCond058",
      "type": "GeneticCondition",
      "description": "Hearing Loss",
      "disease": [
        {
          "id": "Orphanet:87884",
          "type": "Entity",
          "label": "nonsyndromic deafness"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "LOINC:LA6668-3",
    "type": "Entity",
    "label": "Pathogenic"
  }
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../../../">Interpretation Overview</a></h5>
              </li>
              <li class='list-group-item'><h5><a href="../../../user/">User Guide</a></h5></li>
              <li class="list-group-item"><h5><a href="../../">Technical Reference</a></h5><ul class='list-unstyled'><li class="active"><a href="../">Entity Definitions</a></li><ul class='list-unstyled'><li class="active"><a href="./">Core</a></li><ul class='list-unstyled'><li><a href="criterion_assessment.html">CriterionAssessment</a></li><li><a href="domain_entity.html">DomainEntity</a></li><li><a href="evidence_line.html">EvidenceLine</a></li><li><a href="statement.html">Statement</a></li><li class="active"><a href="variant_interpretation.html">VariantPathogenicityInterpretation</a></li></ul><li><a href="../domain_entities/">Domain Entities</a></li><li><a href="../evidence_items/">Evidence Item</a></li><li><a href="../provenance/">Provenance</a></li><li><a href="../terminology/">Terminology</a></li></ul><li><a href="../../json_schema.html">JSON Schema</a></li></ul></li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2017 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../javascripts/jquery.js"></script><script src="../../../javascripts/bootstrap.js"></script><script src="../../../javascripts/jquery.jsonview.js"></script><script src="../../../javascripts/all.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
