<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a class="navbar-brand" href="/">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a target="_blank" href="http://www.clinicalgenome.org"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>
    


    
        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">CriterionAssessment</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../../">Interpretation Model</a></li>
<li><a href="../">Technical Reference and Examples</a></li>
<li><a href="./">Core</a></li>

              </ol>
                <blockquote><p>Preferred: An assertion of whether sufficient evidence exists to satisfy some pre-defined 'variant interpretation criterion', which defines requirements for a particular type of argument relevant to the pathogenicity of a genetic variant.

An assertion of whether sufficient evidence exists about a particular variant to satisfy some pre-defined 'variant interpretation criterion'. 

------

(then see clarifying comments to the right)
</p>
                <STRONG>
                  <div>SEPIO:0000191</div>
                      <i>Descended from Statement</i>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>A CriterionAssessment represents the result of applying a named Criterion to a set of Data.</p>

<p>A VariantInterpretation is based on assessing different lines of evidence, such as population frequency data, or case-control studies.  A CriterionAssessment is the result of gathering together the data necessary to understand a single line of evidence, and deciding how that set of evidence hangs together to create a single unit of understanding. One or more CriterionAssessments, each of which describes a judgement about a particular line of evidence, are then subsequently combined to produce the final interpretation.</p>

<p>In addition to the Data, the CriterionAssessment references a particular Criterion.  The Criterion is a pre-defined rule that describes how to use a particular type of data to reach a conclusion.</p>

<p>In the context of ACMG Variant Pathogenicity interpretations, a Criterion is a particular rule, such as PM1, and the CriterionAssessment is the statement, for example, that this set of evidence provides moderate support that the variant is pathogenic, or that the evidence is insufficient to apply the Criterion.</p>

<p>A CriterionAssessment that lacks the Criterion attribute represents a conclusion based on a set of evidence, but which is evaluated in a way outside of a pre-defined set of criteria.  While a Criterion is preferable, and will be expressly required in certain cases, making this attribute optional allows the model to express interpretation structures that are constructed outside of the ACMG Guideline framework.</p>

<p>Given the Criterion and the Data, the logic of a particular CriterionAssessment is meant to be transparent.  However, to be complete, a human-readable description of the reasoning behind a CriterionAssessment may be included in the explanation attribute (inherited from Entity).</p>

<p>CriterionAssessments are generated by an AssessCriterion Activity, and are subsequently used as inputs to a GenerateVariantInterpretation Activity.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>criterion</td>
                <td>Criterion</td>
                <td>0..1</td>
                <td>The rule describing how the data is being used</td>
                <td>SEPIO:0000041</td>
            </tr>
            <tr>
                <td>variant</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>Variant about which the assemssment is made</td>
                <td>SEPIO:0000275</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>GeneticCondition</td>
                <td>0..*</td>
                <td>Condition for which the assessment is made</td>
                <td>SEPIO:0000276</td>
            </tr>
            <tr>
                <td>outcome</td>
                <td>CriterionEvaluation</td>
                <td>1..1</td>
                <td>Result of assessing the data and criterion</td>
                <td>SEPIO:0000197</td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>qualifier</td>
                    <td>string</td>
                    <td>0..1</td>
                    <td></td>
                    <td>Use “NOT” as the value of this property to assert that the statement is negated.</td>
                    <td>??</td>
                </tr>
                <tr>
                    <td>evidenceLine</td>
                    <td>EvidenceLine</td>
                    <td>0..*</td>
                    <td></td>
                    <td>supporting evidence</td>
                    <td>SEPIO:0000006</td>
                </tr>
                <tr>
                    <td>description</td>
                    <td>string</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A human-readable description of the entity</td>
                    <td>DC:description</td>
                </tr>
                <tr>
                    <td>contribution</td>
                    <td>Contribution</td>
                    <td>0..*</td>
                    <td></td>
                    <td>The contributions (agent, time and role) made to this entity</td>
                    <td>SEPIO:0000159</td>
                </tr>
                <tr>
                    <td>source</td>
                    <td>string</td>
                    <td>0..*</td>
                    <td></td>
                    <td></td>
                    <td>DC:source</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>qualifier</th>
        <th>evidenceLine</th>
        <th>description</th>
        <th>criterion</th>
        <th>variant</th>
        <th>condition</th>
        <th>outcome</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../details/details.html#CritAssess156">CritAssess156</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn132">EvLn132</a></p>
         </td>
          <td>
              <p>Most pathogenic variants in TTN are truncating. However, a small number of missense variants in TTN are associated with ARVC (Taylor 2011 PMID:21810661 ) Because TTN is such a large gene, the statistical expectation of benign missense variants is very high.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit315">Crit315</a></p> (PP2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll093">CanAll093</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess156"><pre><code>{
  "id": "CritAssess156",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit315",
    "type": "Criterion",
    "label": "PP2",
    "description": "Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    },
    "comments": "Seems no one has established firm rules for when this rule is applicable so not sure how great any examples will be\nAgree with the lack of firm rules. Seems like one criteria may be observed missense variants over 5% allele frequency in ExAC. If this seems reasonable then I could generate a couple of examples."
  },
  "variant": {
    "id": "CanAll093",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll099",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll093",
      "alleleName": "NM_133378.4(TTN): c.2926T>C"
    }
  },
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn132",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "BMVR161",
          "type": "BenignMissenseVariationRate",
          "gene": {
            "id": "G229",
            "type": "Gene",
            "geneSymbol": "TTN"
          },
          "value": {
            "id": "BMVROID128",
            "type": "BenignMissenseVariationRateOutcome",
            "label": "High"
          },
          "description": "In LMM data alone, we have over 150 benign missense variants in TTN."
        },
        {
          "id": "MolCon169",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll099",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll093",
            "alleleName": "NM_133378.4(TTN): c.2926T>C"
          },
          "consequence": {
            "id": "MolConID011",
            "type": "MolecularConsequence",
            "label": "missense variant"
          }
        },
        {
          "id": "CondMech165",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MolConID011",
            "type": "MolecularConsequence",
            "label": "missense variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID133",
            "type": "ConditionMechanismStrength",
            "label": "common"
          },
          "gene": {
            "id": "G229",
            "type": "Gene",
            "geneSymbol": "TTN"
          },
          "condition": {
            "id": "GenCond054",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID164",
                "type": "Disease",
                "label": "arrhythmogenic right ventricular cardiomyopathy"
              }
            ]
          },
          "description": "Taylor 2011 PMID:21810661"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent010",
        "type": "Agent",
        "label": "ACME Molecular Lab"
      },
      "contributionDate": "7/2/2012 8:45 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Most pathogenic variants in TTN are truncating. However, a small number of missense variants in TTN are associated with ARVC (Taylor 2011 PMID:21810661 ) Because TTN is such a large gene, the statistical expectation of benign missense variants is very high."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess275">CritAssess275</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn001">EvLn001</a></p>
         </td>
          <td>
              <p>Variant is not LOF and LOF is not a known mechanism of disease associated with this gene for any condition.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit303">Crit303</a></p> (PVS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll001">CanAll001</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess275"><pre><code>{
  "id": "CritAssess275",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit303",
    "type": "Criterion",
    "label": "PVS1",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "defaultStrength": {
      "id": "EvLnStrgID146",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Very Strong"
    },
    "comments": "The two main concerns for applying this rule is:\na. Is LOF an established disease mechanism?\nb. Is LOF actually occurring?"
  },
  "variant": {
    "id": "CanAll001",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll001",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll001",
      "alleleName": "NM_031844.2(HNRNPU):c.2304_2305del (p.Gly769Glufs*83)"
    }
  },
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn001",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon001",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll001",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll001",
            "alleleName": "NM_031844.2(HNRNPU):c.2304_2305del (p.Gly769Glufs*83)"
          },
          "consequence": {
            "id": "MolConID014",
            "type": "MolecularConsequence",
            "label": "frameshift variant"
          },
          "description": "Protein ends at p.826 and this frameshift would create stop at p.852 (769+83) thus NMD would not occur. Instead an elongated protein"
        },
        {
          "id": "Null396",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll001",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll001",
            "alleleName": "NM_031844.2(HNRNPU):c.2304_2305del (p.Gly769Glufs*83)"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          },
          "qualifier": "NOT"
        },
        {
          "id": "CondMech002",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MechID071",
            "type": "Mechanism",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID134",
            "type": "ConditionMechanismStrength",
            "label": "established"
          },
          "gene": {
            "id": "G227",
            "type": "Gene",
            "geneSymbol": "HNRNPU"
          },
          "description": "HNRNPU is not associated with any conditions, and therefore LOF is not an established mechanism of any condition for this gene."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/1/16 10:10 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Variant is not LOF and LOF is not a known mechanism of disease associated with this gene for any condition."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess276">CritAssess276</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn002">EvLn002</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit303">Crit303</a></p> (PVS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll002">CanAll002</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond001">GenCond001</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess276"><pre><code>{
  "id": "CritAssess276",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit303",
    "type": "Criterion",
    "label": "PVS1",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "defaultStrength": {
      "id": "EvLnStrgID146",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Very Strong"
    },
    "comments": "The two main concerns for applying this rule is:\na. Is LOF an established disease mechanism?\nb. Is LOF actually occurring?"
  },
  "variant": {
    "id": "CanAll002",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll002",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll002",
      "alleleName": "NM_000059.3(BRCA2):c.7762_7764delinsTT"
    }
  },
  "condition": [
    {
      "id": "GenCond001",
      "type": "GeneticCondition",
      "name": "Breast Ovarian Cancer, Autosomal Dominant",
      "disease": [
        {
          "id": "DisID148",
          "type": "Disease",
          "label": "hereditary breast ovarian cancer"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn002",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon003",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll002",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll002",
            "alleleName": "NM_000059.3(BRCA2):c.7762_7764delinsTT"
          },
          "consequence": {
            "id": "MolConID014",
            "type": "MolecularConsequence",
            "label": "frameshift variant"
          },
          "description": "LOF is predicted to occur as variant is >55bp from penultimate exon"
        },
        {
          "id": "Null397",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll002",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll002",
            "alleleName": "NM_000059.3(BRCA2):c.7762_7764delinsTT"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          }
        },
        {
          "id": "CondMech004",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MechID071",
            "type": "Mechanism",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID134",
            "type": "ConditionMechanismStrength",
            "label": "established"
          },
          "gene": {
            "id": "G228",
            "type": "Gene",
            "geneSymbol": "BRCA2"
          },
          "condition": {
            "id": "GenCond001",
            "type": "GeneticCondition",
            "name": "Breast Ovarian Cancer, Autosomal Dominant",
            "disease": [
              {
                "id": "DisID148",
                "type": "Disease",
                "label": "hereditary breast ovarian cancer"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "description": "This is so well-known that it does not require a citation."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 12:15 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess277">CritAssess277</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn003">EvLn003</a></p><p><a href="../details/details.html#EvLn004">EvLn004</a></p>
         </td>
          <td>
              <p>This variant only occurs as LOF in an uncommon transcript (NM_133437.4).  For most transcripts, the variant is intronic.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit303">Crit303</a></p> (PVS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll003">CanAll003</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond047">GenCond047</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess277"><pre><code>{
  "id": "CritAssess277",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit303",
    "type": "Criterion",
    "label": "PVS1",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "defaultStrength": {
      "id": "EvLnStrgID146",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Very Strong"
    },
    "comments": "The two main concerns for applying this rule is:\na. Is LOF an established disease mechanism?\nb. Is LOF actually occurring?"
  },
  "variant": {
    "id": "CanAll003",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll003",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll003",
      "alleleName": "NM_133437.4(TTN):c.10670dupG"
    }
  },
  "condition": [
    {
      "id": "GenCond047",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID162",
          "type": "Disease",
          "label": "dilated cardiomyopathy"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn003",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon005",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll003",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll003",
            "alleleName": "NM_133437.4(TTN):c.10670dupG"
          },
          "consequence": {
            "id": "MolConID014",
            "type": "MolecularConsequence",
            "label": "frameshift variant"
          },
          "description": "LOF is predicted but this transcript is rare"
        },
        {
          "id": "Null398",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll003",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll003",
            "alleleName": "NM_133437.4(TTN):c.10670dupG"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          }
        },
        {
          "id": "CondMech007",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MechID071",
            "type": "Mechanism",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID134",
            "type": "ConditionMechanismStrength",
            "label": "established"
          },
          "gene": {
            "id": "G229",
            "type": "Gene",
            "geneSymbol": "TTN"
          },
          "condition": {
            "id": "GenCond047",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID162",
                "type": "Disease",
                "label": "dilated cardiomyopathy"
              }
            ]
          }
        }
      ]
    },
    {
      "id": "EvLn004",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon006",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll004",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll003",
            "alleleName": "LRG_391t2:c.10303+2691dupG"
          },
          "consequence": {
            "id": "MolConID002",
            "type": "MolecularConsequence",
            "label": "interior intron"
          }
        },
        {
          "id": "Null399",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll004",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll003",
            "alleleName": "LRG_391t2:c.10303+2691dupG"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          },
          "qualifier": "NOT"
        },
        {
          "id": "CondMech007",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MechID071",
            "type": "Mechanism",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID134",
            "type": "ConditionMechanismStrength",
            "label": "established"
          },
          "gene": {
            "id": "G229",
            "type": "Gene",
            "geneSymbol": "TTN"
          },
          "condition": {
            "id": "GenCond047",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID162",
                "type": "Disease",
                "label": "dilated cardiomyopathy"
              }
            ]
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 2:22 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This variant only occurs as LOF in an uncommon transcript (NM_133437.4).  For most transcripts, the variant is intronic."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess278">CritAssess278</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn005">EvLn005</a></p>
         </td>
          <td>
              <p>The assessed variant produces the same amino acid change (p.Phe285Leu) as known pathogenic variant c.853T>C</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit304">Crit304</a></p> (PS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll004">CanAll004</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond003">GenCond003</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess278"><pre><code>{
  "id": "CritAssess278",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit304",
    "type": "Criterion",
    "label": "PS1",
    "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "Beware of changes that impact splicing rather than at the amino acid/protein level\nFrom CSER paper: One site erroneously used this to apply prior publication of the same exact variant while this rule, as described in more depth on the ACMG/AMP guideline, only applies when the established pathogenic variant has a different nucleotide change than the variant being interpreted."
  },
  "variant": {
    "id": "CanAll004",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll005",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll004",
      "alleleName": "NM_002834.3(PTPN11):c.855T>G (p.Phe285Leu)",
      "produces": [
        {
          "id": "CtxAll007",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll006",
          "alleleName": "NP_002825.3:p.Phe285Leu",
          "producedBy": "CtxAll006"
        }
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond003",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID175",
          "type": "Disease",
          "label": "noonan syndrome"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn005",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp008",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll005",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll006",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll005",
              "alleleName": "NM_002834.3(PTPN11):c.853T>C (p.Phe285Leu)",
              "produces": [
                {
                  "id": "CtxAll007",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll006",
                  "alleleName": "NP_002825.3:p.Phe285Leu",
                  "producedBy": "CtxAll006"
                }
              ]
            }
          },
          "condition": [
            {
              "id": "GenCond003",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID175",
                  "type": "Disease",
                  "label": "noonan syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          },
          "description": "NM_002834.3:c.855T>G Pathogenic for Noonan Syndrome"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 3:22 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "The assessed variant produces the same amino acid change (p.Phe285Leu) as known pathogenic variant c.853T>C"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess279">CritAssess279</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn006">EvLn006</a></p>
         </td>
          <td>
              <p>The assessed variant produces the same amino acid change (p.Met2083Ile) as known pathogenic variant c.6249G>A.  But that change is pathogenic because it has been shown to disrupt splicing and create a premature stop codon. Prediction tools do not suggest c.6249G>C disrupts splicing.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit304">Crit304</a></p> (PS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll007">CanAll007</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond004">GenCond004</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess279"><pre><code>{
  "id": "CritAssess279",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit304",
    "type": "Criterion",
    "label": "PS1",
    "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "Beware of changes that impact splicing rather than at the amino acid/protein level\nFrom CSER paper: One site erroneously used this to apply prior publication of the same exact variant while this rule, as described in more depth on the ACMG/AMP guideline, only applies when the established pathogenic variant has a different nucleotide change than the variant being interpreted."
  },
  "variant": {
    "id": "CanAll007",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll008",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll007",
      "alleleName": "NM_001369.2(DNAH5):c.6249G>C (p.Met2083Ile)",
      "produces": [
        {
          "id": "CtxAll010",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll009",
          "alleleName": "NP_001360.1:p.Met2083Ile",
          "producedBy": "CtxAll009"
        }
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond004",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID163",
          "type": "Disease",
          "label": "Primary ciliary dyskenisia"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn006",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp009",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll008",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll009",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll008",
              "alleleName": "NM_001369.2(DNAH5):c.6249G>A (p.Met2083Ile)",
              "produces": [
                {
                  "id": "CtxAll010",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll009",
                  "alleleName": "NP_001360.1:p.Met2083Ile",
                  "producedBy": "CtxAll009"
                }
              ]
            }
          },
          "condition": [
            {
              "id": "GenCond004",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID163",
                  "type": "Disease",
                  "label": "Primary ciliary dyskenisia"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          },
          "description": "NM_001369.2:c.6249G>A Pathogenic for PCD"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 4:22 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "The assessed variant produces the same amino acid change (p.Met2083Ile) as known pathogenic variant c.6249G>A.  But that change is pathogenic because it has been shown to disrupt splicing and create a premature stop codon. Prediction tools do not suggest c.6249G>C disrupts splicing."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess280">CritAssess280</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn007">EvLn007</a></p><p><a href="../details/details.html#EvLn008">EvLn008</a></p>
         </td>
          <td>
              <p>While the two amino acid changes are not identical, they are the same amino acid change in analagous residues of HRAS and KRAS.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit304">Crit304</a></p> (PS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll010">CanAll010</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond005">GenCond005</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess280"><pre><code>{
  "id": "CritAssess280",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit304",
    "type": "Criterion",
    "label": "PS1",
    "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "Beware of changes that impact splicing rather than at the amino acid/protein level\nFrom CSER paper: One site erroneously used this to apply prior publication of the same exact variant while this rule, as described in more depth on the ACMG/AMP guideline, only applies when the established pathogenic variant has a different nucleotide change than the variant being interpreted."
  },
  "variant": {
    "id": "CanAll010",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll012",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll010",
      "alleleName": "NM_176795.4(HRAS):c.173C>T (p.Thr58Ile)",
      "produces": [
        {
          "id": "CtxAll015",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll012",
          "alleleName": "NP_005334.1:p.Thr58Ile",
          "producedBy": "CtxAll012"
        }
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond005",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID167",
          "type": "Disease",
          "label": "costello syndrome"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn007",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp010",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll011",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll014",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll011",
              "alleleName": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile)",
              "produces": [
                {
                  "id": "CtxAll016",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll013",
                  "alleleName": "NP_004976.2:p.Thr58Ile",
                  "producedBy": "CtxAll014"
                }
              ]
            }
          },
          "condition": [
            {
              "id": "GenCond006",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID152",
                  "type": "Disease",
                  "label": "noonan syndrome 3"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          },
          "description": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Noonan Syndrome 3"
        }
      ]
    },
    {
      "id": "EvLn008",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp011",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll011",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll014",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll011",
              "alleleName": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile)",
              "produces": [
                {
                  "id": "CtxAll016",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll013",
                  "alleleName": "NP_004976.2:p.Thr58Ile",
                  "producedBy": "CtxAll014"
                }
              ]
            }
          },
          "condition": [
            {
              "id": "GenCond055",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID175",
                  "type": "Disease",
                  "label": "noonan syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          },
          "description": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Costello syndrome"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 5:22 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent002",
        "type": "Agent",
        "label": "Noonan/Rasopathy WG (ClinGen)"
      },
      "contributionDate": "6/2/16 5:22 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "While the two amino acid changes are not identical, they are the same amino acid change in analagous residues of HRAS and KRAS."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess281">CritAssess281</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn009">EvLn009</a></p>
         </td>
          <td>
              <p>IC1: Individual has condition, FH1: No family history, DNA1: variant is denovo (parentage confirmed)</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit305">Crit305</a></p> (PS2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll014">CanAll014</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond052">GenCond052</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess281"><pre><code>{
  "id": "CritAssess281",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit305",
    "type": "Criterion",
    "label": "PS2",
    "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "MYH7 is adding specificity to this rule with\na. No family history = parents have had ECHO and ECG\nb. Only paternity confirmation required"
  },
  "variant": {
    "id": "CanAll014",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll017",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll014",
      "alleleName": "NM_000257.3(MYH7):c.5740G>A (p.Glu1914Lys)"
    }
  },
  "condition": [
    {
      "id": "GenCond052",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID162",
          "type": "Disease",
          "label": "dilated cardiomyopathy"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn009",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond012",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv333",
            "type": "Individual",
            "description": "individual #001"
          },
          "condition": [
            {
              "id": "GenCond052",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID162",
                  "type": "Disease",
                  "label": "dilated cardiomyopathy"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "FamCond013",
          "type": "FamilyCondition",
          "proband": {
            "id": "Indiv333",
            "type": "Individual",
            "description": "individual #001"
          },
          "condition": [
            {
              "id": "GenCond052",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID162",
                  "type": "Disease",
                  "label": "dilated cardiomyopathy"
                }
              ]
            }
          ],
          "familyHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        },
        {
          "id": "IndAllInh014",
          "type": "IndividualAlleleInheritance",
          "canonicalAllele": {
            "id": "CanAll014",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll017",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll014",
              "alleleName": "NM_000257.3(MYH7):c.5740G>A (p.Glu1914Lys)"
            }
          },
          "individual": {
            "id": "Indiv333",
            "type": "Individual",
            "description": "individual #001"
          },
          "parentalConfirmation": {
            "id": "ParenConfID216",
            "type": "ParentalConfirmation",
            "label": "maternal and paternal confirmation"
          },
          "alleleInheritance": [
            {
              "id": "AlleleInhID197",
              "type": "AlleleInheritance",
              "label": "de novo"
            }
          ],
          "description": "parentage confirmed via haplotyping."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/3/16 8:00 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "IC1: Individual has condition, FH1: No family history, DNA1: variant is denovo (parentage confirmed)"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess282">CritAssess282</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn010">EvLn010</a></p>
         </td>
          <td>
              <p>De novo variant occurred on haplotype inherited from the father. Maternity no explicitly confirmed - but if variant occurred on haplotype inherited from the father, then confirmation of maternity not required.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit305">Crit305</a></p> (PS2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll015">CanAll015</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond008">GenCond008</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess282"><pre><code>{
  "id": "CritAssess282",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit305",
    "type": "Criterion",
    "label": "PS2",
    "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "MYH7 is adding specificity to this rule with\na. No family history = parents have had ECHO and ECG\nb. Only paternity confirmation required"
  },
  "variant": {
    "id": "CanAll015",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll018",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll015",
      "alleleName": "NM_001927.3(DES):c.1216C>T (p.Arg406Trp)"
    }
  },
  "condition": [
    {
      "id": "GenCond008",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID151",
          "type": "Disease",
          "label": "myofibrillar myopathy 1"
        }
      ],
      "name": "desmin related myopathy"
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn010",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond015",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv334",
            "type": "Individual",
            "description": "individual #002"
          },
          "condition": [
            {
              "id": "GenCond008",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID151",
                  "type": "Disease",
                  "label": "myofibrillar myopathy 1"
                }
              ],
              "name": "desmin related myopathy"
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "FamCond016",
          "type": "FamilyCondition",
          "proband": {
            "id": "Indiv334",
            "type": "Individual",
            "description": "individual #002"
          },
          "condition": [
            {
              "id": "GenCond008",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID151",
                  "type": "Disease",
                  "label": "myofibrillar myopathy 1"
                }
              ],
              "name": "desmin related myopathy"
            }
          ],
          "familyHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        },
        {
          "id": "IndAllInh017",
          "type": "IndividualAlleleInheritance",
          "canonicalAllele": {
            "id": "CanAll015",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll018",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll015",
              "alleleName": "NM_001927.3(DES):c.1216C>T (p.Arg406Trp)"
            }
          },
          "individual": {
            "id": "Indiv334",
            "type": "Individual",
            "description": "individual #002"
          },
          "parentalConfirmation": {
            "id": "ParenConfID215",
            "type": "ParentalConfirmation",
            "label": "paternal confirmation"
          },
          "alleleInheritance": [
            {
              "id": "AlleleInhID197",
              "type": "AlleleInheritance",
              "label": "de novo"
            }
          ],
          "description": "Non-paternity was excluded by assessment with microsatellites"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/3/16 9:17 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "De novo variant occurred on haplotype inherited from the father. Maternity no explicitly confirmed - but if variant occurred on haplotype inherited from the father, then confirmation of maternity not required."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess283">CritAssess283</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit305">Crit305</a></p> (PS2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll016">CanAll016</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond009">GenCond009</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess283"><pre><code>{
  "id": "CritAssess283",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit305",
    "type": "Criterion",
    "label": "PS2",
    "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "MYH7 is adding specificity to this rule with\na. No family history = parents have had ECHO and ECG\nb. Only paternity confirmation required"
  },
  "variant": {
    "id": "CanAll016",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll019",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll016",
      "alleleName": "NM_000833.4 (GRIN2A):c.4375A>G"
    }
  },
  "condition": [
    {
      "id": "GenCond009",
      "type": "GeneticCondition",
      "name": "Epilepsy with Mental Retardation"
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "phenotypePositiveAlleleNegative": 0,
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/3/16 10:25 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess284">CritAssess284</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn012">EvLn012</a></p><p><a href="../details/details.html#EvLn013">EvLn013</a></p><p><a href="../details/details.html#EvLn014">EvLn014</a></p><p><a href="../details/details.html#EvLn015">EvLn015</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit306">Crit306</a></p> (PS3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll017">CanAll017</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess284"><pre><code>{
  "id": "CritAssess284",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit306",
    "type": "Criterion",
    "label": "PS3",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "This particular rule will probably be downgraded often, given that there are not many “Functional studies that have been validated and shown to be reproducible and robust in a clinical diagnostic laboratory setting”"
  },
  "variant": {
    "id": "CanAll017",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll020",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll017",
      "alleleName": "NM_001001431.2(TNNT2):c.391C>T",
      "produces": [
        {
          "id": "CtxAll021",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll018",
          "alleleName": "NP_001001431.1:p.Arg131Trp",
          "producedBy": "CtxAll020"
        }
      ]
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn012",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func021",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "the mutation lowered the Ca2+  affinity of the reconstituted thin filaments;",
          "assayType": {
            "id": "AFAMID078",
            "type": "AlleleFunctionalAssayMethod",
            "label": "Binding assay"
          },
          "contextualAllele": {
            "id": "CtxAll021",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll018",
            "alleleName": "NP_001001431.1:p.Arg131Trp",
            "producedBy": "CtxAll020"
          },
          "gene": {
            "id": "G237",
            "type": "Gene",
            "geneSymbol": "TNNT2"
          }
        }
      ]
    },
    {
      "id": "EvLn013",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func022",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "reduced Ca2 sensitivity of activation",
          "assayType": {
            "id": "AFAMID077",
            "type": "AlleleFunctionalAssayMethod",
            "label": "ATPase assay"
          },
          "contextualAllele": {
            "id": "CtxAll021",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll018",
            "alleleName": "NP_001001431.1:p.Arg131Trp",
            "producedBy": "CtxAll020"
          },
          "gene": {
            "id": "G237",
            "type": "Gene",
            "geneSymbol": "TNNT2"
          }
        }
      ]
    },
    {
      "id": "EvLn014",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func023",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "reduced Ca2 sensitivity of activation",
          "assayType": {
            "id": "AFAMID082",
            "type": "AlleleFunctionalAssayMethod",
            "label": "motility assay"
          },
          "contextualAllele": {
            "id": "CtxAll021",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll018",
            "alleleName": "NP_001001431.1:p.Arg131Trp",
            "producedBy": "CtxAll020"
          },
          "gene": {
            "id": "G237",
            "type": "Gene",
            "geneSymbol": "TNNT2"
          }
        }
      ]
    },
    {
      "id": "EvLn015",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func024",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "desensitized Ca2+ binding to the thin filament.",
          "assayType": {
            "id": "AFAMID078",
            "type": "AlleleFunctionalAssayMethod",
            "label": "Binding assay"
          },
          "contextualAllele": {
            "id": "CtxAll021",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll018",
            "alleleName": "NP_001001431.1:p.Arg131Trp",
            "producedBy": "CtxAll020"
          },
          "gene": {
            "id": "G237",
            "type": "Gene",
            "geneSymbol": "TNNT2"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/3/16 2:50 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "6/3/16 2:50 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess285">CritAssess285</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn016">EvLn016</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit306">Crit306</a></p> (PS3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll019">CanAll019</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess285"><pre><code>{
  "id": "CritAssess285",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit306",
    "type": "Criterion",
    "label": "PS3",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "This particular rule will probably be downgraded often, given that there are not many “Functional studies that have been validated and shown to be reproducible and robust in a clinical diagnostic laboratory setting”"
  },
  "variant": {
    "id": "CanAll019",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll022",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll019",
      "alleleName": "NM_003119.2 (SPG7):c.1529C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn016",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func025",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "Functionally Damaging",
          "assayType": {
            "id": "AFAMID084",
            "type": "AlleleFunctionalAssayMethod",
            "label": "Yeast complementation assay"
          },
          "contextualAllele": {
            "id": "CtxAll022",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll019",
            "alleleName": "NM_003119.2 (SPG7):c.1529C>T"
          },
          "gene": {
            "id": "G238",
            "type": "Gene",
            "geneSymbol": "SPG7"
          },
          "description": "Expression of human SPG7 (with this mutation) in yeast with damaged m-AAA protease did not restore respiratory growth or MrpL32 processing, if the function of AFG3L2 was impaired."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/3/16 6:09 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "6/3/16 6:09 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess286">CritAssess286</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn017">EvLn017</a></p><p><a href="../details/details.html#EvLn018">EvLn018</a></p>
         </td>
          <td>
              <p>The RASopathy group considers MEK and ERK activation assays as well validated.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit306">Crit306</a></p> (PS3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll020">CanAll020</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess286"><pre><code>{
  "id": "CritAssess286",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit306",
    "type": "Criterion",
    "label": "PS3",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "This particular rule will probably be downgraded often, given that there are not many “Functional studies that have been validated and shown to be reproducible and robust in a clinical diagnostic laboratory setting”"
  },
  "variant": {
    "id": "CanAll020",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll023",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll020",
      "alleleName": "NM_004333.4(BRAF):c.1787G>T",
      "produces": [
        {
          "id": "CtxAll024",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll021",
          "alleleName": "NP_004324.2:p.Gly596Val",
          "producedBy": "CtxAll023"
        }
      ]
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn017",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func026",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "abnormal/deceased MEK activtation.",
          "assayType": {
            "id": "AFAMID076",
            "type": "AlleleFunctionalAssayMethod",
            "label": "Activation Assay"
          },
          "contextualAllele": {
            "id": "CtxAll024",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll021",
            "alleleName": "NP_004324.2:p.Gly596Val",
            "producedBy": "CtxAll023"
          },
          "gene": {
            "id": "G240",
            "type": "Gene",
            "geneSymbol": "MEK"
          }
        }
      ]
    },
    {
      "id": "EvLn018",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func027",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "abnormal/deceased ERK activtation.",
          "assayType": {
            "id": "AFAMID076",
            "type": "AlleleFunctionalAssayMethod",
            "label": "Activation Assay"
          },
          "contextualAllele": {
            "id": "CtxAll024",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll021",
            "alleleName": "NP_004324.2:p.Gly596Val",
            "producedBy": "CtxAll023"
          },
          "gene": {
            "id": "G241",
            "type": "Gene",
            "geneSymbol": "ERK"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/4/16 7:12 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent002",
        "type": "Agent",
        "label": "Noonan/Rasopathy WG (ClinGen)"
      },
      "contributionDate": "6/4/16 7:12 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "The RASopathy group considers MEK and ERK activation assays as well validated."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess287">CritAssess287</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn019">EvLn019</a></p>
         </td>
          <td>
              <p>statistically significant difference between affects and unaffecteds</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit307">Crit307</a></p> (PS4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll022">CanAll022</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond048">GenCond048</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess287"><pre><code>{
  "id": "CritAssess287",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit307",
    "type": "Criterion",
    "label": "PS4",
    "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CanAll022",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll025",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll022",
      "alleleName": "NM_000441.1(SLC26A4):c.1003T>C"
    }
  },
  "condition": [
    {
      "id": "GenCond048",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID183",
          "type": "Disease",
          "label": "nonsyndromic deafness"
        }
      ],
      "name": "Hearing Loss"
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn019",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CaseCtrl030",
          "type": "CaseControl",
          "canonicalAllele": {
            "id": "CanAll022",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll025",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll022",
              "alleleName": "NM_000441.1(SLC26A4):c.1003T>C"
            }
          },
          "condition": {
            "id": "GenCond048",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID183",
                "type": "Disease",
                "label": "nonsyndromic deafness"
              }
            ],
            "name": "Hearing Loss"
          },
          "oddsRatio": 9.1361,
          "confidenceLevel": 99.0,
          "confidenceIntervalLower": 6.1425,
          "confidenceIntervalUpper": 13.5889,
          "caseGroupFrequency": {
            "id": "AllFreq029",
            "type": "AlleleFrequency",
            "ascertainment": {
              "id": "AscrtID206",
              "type": "Ascertainment",
              "userLabel": {
                "id": "UsrLabl002",
                "type": "UserLabel",
                "description": "Case group is composed of cases from six cohorts from the GENIUS T2D consortium.  Case status is defined per-cohort."
              }
            },
            "allele": {
              "id": "CanAll022",
              "type": "CanonicalAllele",
              "preferredCtxAllele": {
                "id": "CtxAll025",
                "type": "ContextualAllele",
                "relatedCanonicalAllele": "CanAll022",
                "alleleName": "NM_000441.1(SLC26A4):c.1003T>C"
              }
            },
            "alleleCount": 32,
            "alleleNumber": 4076
          },
          "controlGroupFrequency": {
            "id": "AllFreq028",
            "type": "AlleleFrequency",
            "ascertainment": {
              "id": "AscrtID203",
              "type": "Ascertainment",
              "label": "ExAc ascertainment method"
            },
            "allele": {
              "id": "CanAll022",
              "type": "CanonicalAllele",
              "preferredCtxAllele": {
                "id": "CtxAll025",
                "type": "ContextualAllele",
                "relatedCanonicalAllele": "CanAll022",
                "alleleName": "NM_000441.1(SLC26A4):c.1003T>C"
              }
            },
            "alleleCount": 105,
            "alleleNumber": 121336
          },
          "description": "Treating ExAC data as controls for"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/5/16 10:24 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "statistically significant difference between affects and unaffecteds"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess288">CritAssess288</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn020">EvLn020</a></p>
         </td>
          <td>
              <p>Across publications, this variant is thought to account for 2%-10% of patients with Malignant hyperthermia. But variant is also only found in 9/66740 Europeans in ExAC (.01%). Based on these differences, groups applied PS4 to this variant classification (without a formal OR analysis).</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit307">Crit307</a></p> (PS4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll023">CanAll023</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond011">GenCond011</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess288"><pre><code>{
  "id": "CritAssess288",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit307",
    "type": "Criterion",
    "label": "PS4",
    "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CanAll023",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll026",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll023",
      "alleleName": "NM_000540.2 (RYR1):c.1840C>T"
    }
  },
  "condition": [
    {
      "id": "GenCond011",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID172",
          "type": "Disease",
          "label": "Malignant hyperthermia"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn020",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CaseCtrl033",
          "type": "CaseControl",
          "canonicalAllele": {
            "id": "CanAll023",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll026",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll023",
              "alleleName": "NM_000540.2 (RYR1):c.1840C>T"
            }
          },
          "condition": {
            "id": "GenCond011",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID172",
                "type": "Disease",
                "label": "Malignant hyperthermia"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "caseGroupFrequency": {
            "id": "AllFreq032",
            "type": "AlleleFrequency",
            "allele": {
              "id": "CanAll023",
              "type": "CanonicalAllele",
              "preferredCtxAllele": {
                "id": "CtxAll026",
                "type": "ContextualAllele",
                "relatedCanonicalAllele": "CanAll023",
                "alleleName": "NM_000540.2 (RYR1):c.1840C>T"
              }
            },
            "description": "This allele is thought to account for between 2% and 10% of patients with Malignant hyperthermia."
          },
          "controlGroupFrequency": {
            "id": "AllFreq031",
            "type": "AlleleFrequency",
            "ascertainment": {
              "id": "AscrtID203",
              "type": "Ascertainment",
              "label": "ExAc ascertainment method"
            },
            "allele": {
              "id": "CanAll023",
              "type": "CanonicalAllele",
              "preferredCtxAllele": {
                "id": "CtxAll026",
                "type": "ContextualAllele",
                "relatedCanonicalAllele": "CanAll023",
                "alleleName": "NM_000540.2 (RYR1):c.1840C>T"
              }
            },
            "alleleCount": 9,
            "alleleNumber": 66740
          },
          "description": "Did not calculate formal odds ratio"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/5/16 2:39 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Across publications, this variant is thought to account for 2%-10% of patients with Malignant hyperthermia. But variant is also only found in 9/66740 Europeans in ExAC (.01%). Based on these differences, groups applied PS4 to this variant classification (without a formal OR analysis)."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess289">CritAssess289</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn021">EvLn021</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit307">Crit307</a></p> (PS4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll024">CanAll024</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond012">GenCond012</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess289"><pre><code>{
  "id": "CritAssess289",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit307",
    "type": "Criterion",
    "label": "PS4",
    "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CanAll024",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll027",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll024",
      "alleleName": "NM_006208.2(ENPP1):c.517A>C"
    }
  },
  "condition": [
    {
      "id": "GenCond012",
      "type": "GeneticCondition",
      "phenotype": [
        {
          "id": "PhenID189",
          "type": "Phenotype",
          "label": "Abnormal Glucose Homeostasis"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn021",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CaseCtrl036",
          "type": "CaseControl",
          "canonicalAllele": {
            "id": "CanAll024",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll027",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll024",
              "alleleName": "NM_006208.2(ENPP1):c.517A>C"
            }
          },
          "condition": {
            "id": "GenCond012",
            "type": "GeneticCondition",
            "phenotype": [
              {
                "id": "PhenID189",
                "type": "Phenotype",
                "label": "Abnormal Glucose Homeostasis"
              }
            ]
          },
          "oddsRatio": 1.06,
          "confidenceLevel": 95.0,
          "caseGroupFrequency": {
            "id": "AllFreq034",
            "type": "AlleleFrequency",
            "ascertainment": {
              "id": "AscrtID207",
              "type": "Ascertainment",
              "userLabel": {
                "id": "UsrLabl002",
                "type": "UserLabel",
                "description": "Case group is composed of cases from six cohorts from the GENIUS T2D consortium.  Case status is defined per-cohort."
              }
            },
            "allele": {
              "id": "CanAll024",
              "type": "CanonicalAllele",
              "preferredCtxAllele": {
                "id": "CtxAll027",
                "type": "ContextualAllele",
                "relatedCanonicalAllele": "CanAll024",
                "alleleName": "NM_006208.2(ENPP1):c.517A>C"
              }
            },
            "alleleCount": 1198,
            "alleleNumber": 7344,
            "individualCount": 3672,
            "homozygousAlleleIndividualCount": 113,
            "heterozygousAlleleIndividualCount": 972,
            "population": {
              "id": "PopID090",
              "type": "Population",
              "label": "Non-Finnish European"
            }
          },
          "controlGroupFrequency": {
            "id": "AllFreq035",
            "type": "AlleleFrequency",
            "ascertainment": {
              "id": "AscrtID208",
              "type": "Ascertainment",
              "userLabel": {
                "id": "UsrLabl003",
                "type": "UserLabel",
                "description": "Controls group is composed of controls from six cohorts from the GENIUS T2D consortium.  Controls status is defined per-cohort."
              }
            },
            "allele": {
              "id": "CanAll024",
              "type": "CanonicalAllele",
              "preferredCtxAllele": {
                "id": "CtxAll027",
                "type": "ContextualAllele",
                "relatedCanonicalAllele": "CanAll024",
                "alleleName": "NM_006208.2(ENPP1):c.517A>C"
              }
            },
            "alleleCount": 936,
            "alleleNumber": 5870,
            "individualCount": 2935,
            "homozygousAlleleIndividualCount": 77,
            "heterozygousAlleleIndividualCount": 782,
            "population": {
              "id": "PopID090",
              "type": "Population",
              "label": "Non-Finnish European"
            }
          },
          "confidenceIntervalLower": 0.96,
          "confidenceIntervalUpper": 1.18,
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "9/13/16",
              "contributionRole": {
                "id": "ContribRoleID194",
                "type": "ContributoryRole",
                "label": "curator role"
              }
            }
          ],
          "source": [
            "http://www.ncbi.nlm.nih.gov/pubmed/23633196"
          ],
          "description": "Odds ratio is calculated by pooling samples from the six GENIUS T2D cohorts before calculation."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/5/16 4:12 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess290">CritAssess290</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn022">EvLn022</a></p>
         </td>
          <td>
              <p>Variant is located within the kinase domain (where multiple other pathogenic variants are located)</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit308">Crit308</a></p> (PM1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll025">CanAll025</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess290"><pre><code>{
  "id": "CritAssess290",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit308",
    "type": "Criterion",
    "label": "PM1",
    "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Suggested that this rule is NOT appropriate for LOF variants.\nAlso, note that when the region of interest is a “well-established functional domain” then it’s not necessary to show through evidence that it is intolerant of benign missense variants.  However, when a mutational hotspot is being defined, then showing it does not have benign missense variants is important."
  },
  "variant": {
    "id": "CanAll025",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll028",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll025",
      "alleleName": "NM_005228.3(EGFR):c.2369C>T",
      "produces": [
        {
          "id": "CtxAll126",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll120",
          "alleleName": "NP_005219.2:p.Thr790Met",
          "producedBy": "CtxAll028"
        }
      ]
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn022",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "RegTyp040",
          "type": "RegionType",
          "region": {
            "id": "Loc379",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq288",
              "type": "ReferenceSequence",
              "accession": "NP_005219.2",
              "producedBy": "RefSeq287"
            },
            "start": 704,
            "stop": 1016
          },
          "annotation": [
            {
              "id": "RegAnnID033",
              "type": "RegionAnnotation",
              "label": "functional region"
            }
          ],
          "description": "This region is the catalytic (kinase) domain."
        },
        {
          "id": "RegAll380",
          "type": "RegionAlleles",
          "region": {
            "id": "Loc379",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq288",
              "type": "ReferenceSequence",
              "accession": "NP_005219.2",
              "producedBy": "RefSeq287"
            },
            "start": 704,
            "stop": 1016
          },
          "allele": [
            {
              "id": "CtxAll126",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll120",
              "alleleName": "NP_005219.2:p.Thr790Met",
              "producedBy": "CtxAll028"
            }
          ],
          "outcome": {
            "id": "RegAllOutID225",
            "type": "RegionAllelesOutcome",
            "label": "overlaps region"
          },
          "description": "This region is the catalytic (kinase) domain."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/6/16 8:12 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Variant is located within the kinase domain (where multiple other pathogenic variants are located)"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess291">CritAssess291</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn023">EvLn023</a></p>
         </td>
          <td>
              <p>Variant is located in exon 15, which the RASopathy group has decided is an established functional domain for BRAF</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit308">Crit308</a></p> (PM1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll026">CanAll026</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess291"><pre><code>{
  "id": "CritAssess291",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit308",
    "type": "Criterion",
    "label": "PM1",
    "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Suggested that this rule is NOT appropriate for LOF variants.\nAlso, note that when the region of interest is a “well-established functional domain” then it’s not necessary to show through evidence that it is intolerant of benign missense variants.  However, when a mutational hotspot is being defined, then showing it does not have benign missense variants is important."
  },
  "variant": {
    "id": "CanAll026",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll029",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll026",
      "alleleName": "NM_004333.4(BRAF):c.1789C>G"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn023",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "RegTyp043",
          "type": "RegionType",
          "region": {
            "id": "Loc381",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq289",
              "type": "ReferenceSequence",
              "accession": "NM_004333.4"
            },
            "start": 1803,
            "stop": 1921
          },
          "annotation": [
            {
              "id": "RegAnnID033",
              "type": "RegionAnnotation",
              "label": "functional region"
            }
          ],
          "description": "This region is exon 15 in the transcript, and codes the serine-threonine-protein kinase catalytic domain."
        },
        {
          "id": "RegAll382",
          "type": "RegionAlleles",
          "region": {
            "id": "Loc381",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq289",
              "type": "ReferenceSequence",
              "accession": "NM_004333.4"
            },
            "start": 1803,
            "stop": 1921
          },
          "allele": [
            {
              "id": "CtxAll029",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll026",
              "alleleName": "NM_004333.4(BRAF):c.1789C>G"
            }
          ],
          "outcome": {
            "id": "RegAllOutID225",
            "type": "RegionAllelesOutcome",
            "label": "overlaps region"
          },
          "description": "This region is exon 15 in the transcript, and codes the serine-threonine-protein kinase catalytic domain."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/6/16 10:53 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent002",
        "type": "Agent",
        "label": "Noonan/Rasopathy WG (ClinGen)"
      },
      "contributionDate": "6/6/16 10:53 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Variant is located in exon 15, which the RASopathy group has decided is an established functional domain for BRAF"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess292">CritAssess292</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn024">EvLn024</a></p>
         </td>
          <td>
              <p>The variant is near 4 other pathogenic variants</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit308">Crit308</a></p> (PM1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll027">CanAll027</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess292"><pre><code>{
  "id": "CritAssess292",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit308",
    "type": "Criterion",
    "label": "PM1",
    "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Suggested that this rule is NOT appropriate for LOF variants.\nAlso, note that when the region of interest is a “well-established functional domain” then it’s not necessary to show through evidence that it is intolerant of benign missense variants.  However, when a mutational hotspot is being defined, then showing it does not have benign missense variants is important."
  },
  "variant": {
    "id": "CanAll027",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll030",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll027",
      "alleleName": "NM_000169.2(GLA):c.902G>C"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn024",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "RegTyp046",
          "type": "RegionType",
          "region": {
            "id": "Loc383",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq290",
              "type": "ReferenceSequence",
              "accession": "NM_000169.2"
            },
            "start": 996,
            "stop": 1012
          },
          "annotation": [
            {
              "id": "RegAnnID034",
              "type": "RegionAnnotation",
              "label": "mutational hotspot"
            }
          ],
          "description": "Region(NM_0001692[996..1012]) is a mutational hotspot."
        },
        {
          "id": "RegAll384",
          "type": "RegionAlleles",
          "region": {
            "id": "Loc383",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq290",
              "type": "ReferenceSequence",
              "accession": "NM_000169.2"
            },
            "start": 996,
            "stop": 1012
          },
          "allele": [
            {
              "id": "CtxAll031",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll028",
              "alleleName": "NM_000169.2(GLA): c.886A>G"
            },
            {
              "id": "CtxAll032",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll029",
              "alleleName": "NM_000169.2(GLA):c.890C>T"
            },
            {
              "id": "CtxAll033",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll030",
              "alleleName": "NM_000169.2(GLA):c.899T>C"
            },
            {
              "id": "CtxAll034",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll031",
              "alleleName": "NM_000169.2(GLA):c.902G>A"
            }
          ],
          "outcome": {
            "id": "RegAllOutID225",
            "type": "RegionAllelesOutcome",
            "label": "overlaps region"
          },
          "description": "The region contains the following pathogenic variants: c.886A>G, c.890C>T, c.899T>C, & c.902G>A."
        },
        {
          "id": "VarInterp047",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll028",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll031",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll028",
              "alleleName": "NM_000169.2(GLA): c.886A>G"
            }
          },
          "condition": [
            {
              "id": "GenCond013",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID169",
                  "type": "Disease",
                  "label": "Fabry disease"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID054",
                "type": "ModeOfInheritance",
                "label": "X linked recessive inheritance"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        },
        {
          "id": "VarInterp048",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll029",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll032",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll029",
              "alleleName": "NM_000169.2(GLA):c.890C>T"
            }
          },
          "condition": [
            {
              "id": "GenCond013",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID169",
                  "type": "Disease",
                  "label": "Fabry disease"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID054",
                "type": "ModeOfInheritance",
                "label": "X linked recessive inheritance"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        },
        {
          "id": "VarInterp049",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll030",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll033",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll030",
              "alleleName": "NM_000169.2(GLA):c.899T>C"
            }
          },
          "condition": [
            {
              "id": "GenCond013",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID169",
                  "type": "Disease",
                  "label": "Fabry disease"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID054",
                "type": "ModeOfInheritance",
                "label": "X linked recessive inheritance"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        },
        {
          "id": "VarInterp050",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll031",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll034",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll031",
              "alleleName": "NM_000169.2(GLA):c.902G>A"
            }
          },
          "condition": [
            {
              "id": "GenCond013",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID169",
                  "type": "Disease",
                  "label": "Fabry disease"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID054",
                "type": "ModeOfInheritance",
                "label": "X linked recessive inheritance"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        },
        {
          "id": "BMVR102",
          "type": "BenignMissenseVariationRate",
          "region": {
            "id": "Loc383",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq290",
              "type": "ReferenceSequence",
              "accession": "NM_000169.2"
            },
            "start": 996,
            "stop": 1012
          },
          "value": {
            "id": "BMVROID129",
            "type": "BenignMissenseVariationRateOutcome",
            "label": "Low"
          },
          "description": "ClinVar shows no benign missense variants in (or near) this region.  The closest is NM_000169.2:c.8T>C"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/3/16 7:24 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "The variant is near 4 other pathogenic variants"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess293">CritAssess293</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn025">EvLn025</a></p><p><a href="../details/details.html#EvLn026">EvLn026</a></p>
         </td>
          <td>
              <p>PM2 is applicable as even though variant is not 100% absent, it is practically absent (a 95% CI for the population goes from 0.00% to 0.01%)</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit309">Crit309</a></p> (PM2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll032">CanAll032</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess293"><pre><code>{
  "id": "CritAssess293",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit309",
    "type": "Criterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing"
  },
  "variant": {
    "id": "CanAll032",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll035",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll032",
      "alleleName": "NM_000257.2(MYH7):c.5135G>A"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn025",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq051",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID090",
            "type": "Population",
            "label": "Non-Finnish European"
          },
          "allele": {
            "id": "CanAll032",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll035",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll032",
              "alleleName": "NM_000257.2(MYH7):c.5135G>A"
            }
          },
          "alleleCount": 66706,
          "alleleNumber": 1,
          "alleleFrequency": 1.499e-05,
          "homozygousAlleleIndividualCount": 0
        }
      ]
    },
    {
      "id": "EvLn026",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq052",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll032",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll035",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll032",
              "alleleName": "NM_000257.2(MYH7):c.5135G>A"
            }
          },
          "alleleCount": 121366,
          "alleleNumber": 1,
          "alleleFrequency": 8.24e-06,
          "homozygousAlleleIndividualCount": 0
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/4/16 9:08 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "PM2 is applicable as even though variant is not 100% absent, it is practically absent (a 95% CI for the population goes from 0.00% to 0.01%)"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess294">CritAssess294</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn027">EvLn027</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit309">Crit309</a></p> (PM2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll033">CanAll033</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess294"><pre><code>{
  "id": "CritAssess294",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit309",
    "type": "Criterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing"
  },
  "variant": {
    "id": "CanAll033",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll037",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll033",
      "alleleName": "NM_001128425.1(MUTYH):c.64G>A (p.Val22Met)"
    }
  },
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn027",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq053",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID089",
            "type": "Population",
            "label": "Finnish"
          },
          "allele": {
            "id": "CanAll033",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll037",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll033",
              "alleleName": "NM_001128425.1(MUTYH):c.64G>A (p.Val22Met)"
            }
          },
          "alleleCount": 6614,
          "alleleNumber": 514,
          "alleleFrequency": 0.07771,
          "homozygousAlleleIndividualCount": 21,
          "medianCoverage": 100.0
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/5/16 3:13 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess295">CritAssess295</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn028">EvLn028</a></p>
         </td>
          <td>
              <p>while the variant is absent from exac, mean coverage (11X) is insufficient to be sure that variant was fully assayed.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit309">Crit309</a></p> (PM2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll034">CanAll034</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess295"><pre><code>{
  "id": "CritAssess295",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit309",
    "type": "Criterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing"
  },
  "variant": {
    "id": "CanAll034",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll038",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll034",
      "alleleName": "NM_001399.4(EDA): c.676C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn028",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq054",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll034",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll038",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll034",
              "alleleName": "NM_001399.4(EDA): c.676C>T"
            }
          },
          "alleleCount": 0,
          "alleleNumber": 0,
          "alleleFrequency": 0.0,
          "medianCoverage": 11.0
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/6/16 11:28 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "while the variant is absent from exac, mean coverage (11X) is insufficient to be sure that variant was fully assayed."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess296">CritAssess296</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn029">EvLn029</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit310">Crit310</a></p> (PM3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll035">CanAll035</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond062">GenCond062</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess296"><pre><code>{
  "id": "CritAssess296",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit310",
    "type": "Criterion",
    "label": "PM3",
    "description": "For recessive disorders, detected in trans with a pathogenic variant",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "CSER - Laboratories discussed when to modify the strength of PM3, the variant is seen in trans with a pathogenic variant for recessive disorders. Published literature may not always explicitly state the phase of variants found in affected individuals which raises a challenge for invoking PM3.\nWhen phase has not been established, some felt that PM3 could be invoked as supporting evidence. Also, if the variant is seen in trans with a pathogenic variant in more than one individual it was felt that PM3 can be upgraded to strong. However, sites did not agree on how many additional observations were necessary to call the evidence strong (2 vs. 3) but concluded that such guidance would be useful.\n \nLMM - INCREASES IN STRENGTH IF VARIANT SEEN IN MULTIPLE COMPOUND HETS WITH DIFFERENT LIKELY PATHOGENIC/PATHOGENIC VARIANTS"
  },
  "variant": {
    "id": "CanAll035",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll039",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll035",
      "alleleName": "NM_001369.2(DNAH5):c.7468_7488del",
      "produces": [
        {
          "id": "CtxAll044",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll040",
          "alleleName": "NP_001360.1:p.Trp2490_Leu2496del",
          "producedBy": "CtxAll039"
        }
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond062",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID163",
          "type": "Disease",
          "label": "Primary ciliary dyskenisia"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID052",
        "type": "ModeOfInheritance",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn029",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond056",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv336",
            "type": "Individual",
            "description": "individual #004"
          },
          "condition": [
            {
              "id": "GenCond014",
              "type": "GeneticCondition",
              "name": "PCD autosomal recessive w/ outer dynein arm (ODA) defects",
              "disease": [
                {
                  "id": "DisID163",
                  "type": "Disease",
                  "label": "Primary ciliary dyskenisia"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              },
              "phenotype": [
                {
                  "id": "PhenID191",
                  "type": "Phenotype",
                  "label": "Absent/shortened dynein arms"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          },
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "5/31/16 8:46 AM",
              "contributionRole": {
                "id": "ContribRoleID194",
                "type": "ContributoryRole",
                "label": "curator role"
              }
            }
          ],
          "source": [
            "http://partners.org/ppm/lmm/gi/case/123"
          ]
        },
        {
          "id": "VarInterp057",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll036",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll040",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll036",
              "alleleName": "NM_001369.2(DNAH5):c.9449delG"
            }
          },
          "condition": [
            {
              "id": "GenCond062",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID163",
                  "type": "Disease",
                  "label": "Primary ciliary dyskenisia"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          },
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "5/31/16 11:01 AM",
              "contributionRole": {
                "id": "ContribRoleID196",
                "type": "ContributoryRole",
                "label": "interpreter role"
              }
            }
          ],
          "source": [
            "https://www.ncbi.nlm.nih.gov/clinvar/SCV000205842.3"
          ]
        },
        {
          "id": "IndGeno058",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv336",
            "type": "Individual",
            "description": "individual #004"
          },
          "genotype": [
            {
              "id": "Geno378",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo376",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll035",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll039",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll035",
                        "alleleName": "NM_001369.2(DNAH5):c.7468_7488del",
                        "produces": [
                          {
                            "id": "CtxAll044",
                            "type": "ContextualAllele",
                            "relatedCanonicalAllele": "CanAll040",
                            "alleleName": "NP_001360.1:p.Trp2490_Leu2496del",
                            "producedBy": "CtxAll039"
                          }
                        ]
                      }
                    }
                  ]
                },
                {
                  "id": "Haplo377",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll036",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll040",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll036",
                        "alleleName": "NM_001369.2(DNAH5):c.9449delG"
                      }
                    }
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "5/31/16 9:15 AM",
              "contributionRole": {
                "id": "ContribRoleID194",
                "type": "ContributoryRole",
                "label": "curator role"
              }
            }
          ],
          "source": [
            "http://partners.org/ppm/lmm/gi/case/123"
          ],
          "description": "Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 8:01 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess297">CritAssess297</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn030">EvLn030</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit310">Crit310</a></p> (PM3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll037">CanAll037</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond010">GenCond010</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess297"><pre><code>{
  "id": "CritAssess297",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit310",
    "type": "Criterion",
    "label": "PM3",
    "description": "For recessive disorders, detected in trans with a pathogenic variant",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "CSER - Laboratories discussed when to modify the strength of PM3, the variant is seen in trans with a pathogenic variant for recessive disorders. Published literature may not always explicitly state the phase of variants found in affected individuals which raises a challenge for invoking PM3.\nWhen phase has not been established, some felt that PM3 could be invoked as supporting evidence. Also, if the variant is seen in trans with a pathogenic variant in more than one individual it was felt that PM3 can be upgraded to strong. However, sites did not agree on how many additional observations were necessary to call the evidence strong (2 vs. 3) but concluded that such guidance would be useful.\n \nLMM - INCREASES IN STRENGTH IF VARIANT SEEN IN MULTIPLE COMPOUND HETS WITH DIFFERENT LIKELY PATHOGENIC/PATHOGENIC VARIANTS"
  },
  "variant": {
    "id": "CanAll037",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll041",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll037",
      "alleleName": "NM_133261.2(GIPC3):c.298G>A"
    }
  },
  "condition": [
    {
      "id": "GenCond010",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID183",
          "type": "Disease",
          "label": "nonsyndromic deafness"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID052",
        "type": "ModeOfInheritance",
        "label": "Autosomal Recessive"
      },
      "gene": {
        "id": "G250",
        "type": "Gene",
        "geneSymbol": "GIPC3"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn030",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp059",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll038",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll042",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll038",
              "alleleName": "NM_133261.2(GIPC3):c.411+1G>A"
            }
          },
          "condition": [
            {
              "id": "GenCond010",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID183",
                  "type": "Disease",
                  "label": "nonsyndromic deafness"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              },
              "gene": {
                "id": "G250",
                "type": "Gene",
                "geneSymbol": "GIPC3"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        },
        {
          "id": "IndCond061",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv337",
            "type": "Individual",
            "description": "individual #005"
          },
          "condition": [
            {
              "id": "GenCond010",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID183",
                  "type": "Disease",
                  "label": "nonsyndromic deafness"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              },
              "gene": {
                "id": "G250",
                "type": "Gene",
                "geneSymbol": "GIPC3"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "IndGeno062",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv337",
            "type": "Individual",
            "description": "individual #005"
          },
          "genotype": [
            {
              "id": "Geno381",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo379",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll037",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll041",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll037",
                        "alleleName": "NM_133261.2(GIPC3):c.298G>A"
                      }
                    }
                  ]
                },
                {
                  "id": "Haplo380",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll038",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll042",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll038",
                        "alleleName": "NM_133261.2(GIPC3):c.411+1G>A"
                      }
                    }
                  ]
                }
              ]
            }
          ],
          "description": "Based on the NGS data, we are able to determine that c.298G>A and c.411+1G>A are in trans as no NGS reads carry both variants."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/10/16 9:02 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess298">CritAssess298</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn031">EvLn031</a></p>
         </td>
          <td>
              <p>Application of PM3 is incorrect as the variant in trans is know known to be pathogenic (it is the same variant).</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit310">Crit310</a></p> (PM3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll039">CanAll039</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond015">GenCond015</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess298"><pre><code>{
  "id": "CritAssess298",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit310",
    "type": "Criterion",
    "label": "PM3",
    "description": "For recessive disorders, detected in trans with a pathogenic variant",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "CSER - Laboratories discussed when to modify the strength of PM3, the variant is seen in trans with a pathogenic variant for recessive disorders. Published literature may not always explicitly state the phase of variants found in affected individuals which raises a challenge for invoking PM3.\nWhen phase has not been established, some felt that PM3 could be invoked as supporting evidence. Also, if the variant is seen in trans with a pathogenic variant in more than one individual it was felt that PM3 can be upgraded to strong. However, sites did not agree on how many additional observations were necessary to call the evidence strong (2 vs. 3) but concluded that such guidance would be useful.\n \nLMM - INCREASES IN STRENGTH IF VARIANT SEEN IN MULTIPLE COMPOUND HETS WITH DIFFERENT LIKELY PATHOGENIC/PATHOGENIC VARIANTS"
  },
  "variant": {
    "id": "CanAll039",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll043",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll039",
      "alleleName": "NM_000391.3(TTP1):c.1678_1679delCT"
    }
  },
  "condition": [
    {
      "id": "GenCond015",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID159",
          "type": "Disease",
          "label": "neuronal ceroid lipofuscinosis"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID052",
        "type": "ModeOfInheritance",
        "label": "Autosomal Recessive"
      },
      "gene": {
        "id": "G251",
        "type": "Gene",
        "geneSymbol": "TTP1"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn031",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond064",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv338",
            "type": "Individual",
            "description": "individual #006"
          },
          "condition": [
            {
              "id": "GenCond015",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID159",
                  "type": "Disease",
                  "label": "neuronal ceroid lipofuscinosis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              },
              "gene": {
                "id": "G251",
                "type": "Gene",
                "geneSymbol": "TTP1"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "IndGeno065",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv338",
            "type": "Individual",
            "description": "individual #006"
          },
          "genotype": [
            {
              "id": "Geno383",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo382",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll039",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll043",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll039",
                        "alleleName": "NM_000391.3(TTP1):c.1678_1679delCT"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID043",
                "type": "AllelicState",
                "label": "homozygous"
              }
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/10/16 9:15 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Application of PM3 is incorrect as the variant in trans is know known to be pathogenic (it is the same variant)."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess299">CritAssess299</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn032">EvLn032</a></p>
         </td>
          <td>
              <p>leads to an in-frame deletion of 7 amino acids (not repeat region).</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit311">Crit311</a></p> (PM4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll035">CanAll035</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess299"><pre><code>{
  "id": "CritAssess299",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit311",
    "type": "Criterion",
    "label": "PM4",
    "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Need to ensure that variant is not located within a repeat region\n\nThis rule is also (likely) applicable for frameshift variants that result in extension of the protein as opposed to truncation (similar to stop loss)."
  },
  "variant": {
    "id": "CanAll035",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll039",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll035",
      "alleleName": "NM_001369.2(DNAH5):c.7468_7488del",
      "produces": [
        {
          "id": "CtxAll044",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll040",
          "alleleName": "NP_001360.1:p.Trp2490_Leu2496del",
          "producedBy": "CtxAll039"
        }
      ]
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn032",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon066",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll044",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll040",
            "alleleName": "NP_001360.1:p.Trp2490_Leu2496del",
            "producedBy": "CtxAll039"
          },
          "consequence": {
            "id": "MolConID019",
            "type": "MolecularConsequence",
            "label": "conservative Inframe Deletion"
          },
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "6/1/16 2:03 PM",
              "contributionRole": {
                "id": "ContribRoleID194",
                "type": "ContributoryRole",
                "label": "curator role"
              }
            }
          ],
          "source": [
            "https://geneinsight-lmm-test.partners.org"
          ]
        },
        {
          "id": "RegTyp069",
          "type": "RegionType",
          "region": {
            "id": "Loc385",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq291",
              "type": "ReferenceSequence",
              "accession": "NP_001360.1"
            },
            "start": 2490,
            "stop": 2496
          },
          "annotation": [
            {
              "id": "RegAnnID035",
              "type": "RegionAnnotation",
              "label": "repeat region "
            }
          ],
          "qualifier": "NOT",
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "6/1/16 2:10 PM",
              "contributionRole": {
                "id": "ContribRoleID194",
                "type": "ContributoryRole",
                "label": "curator role"
              }
            }
          ],
          "source": [
            "https://pcpgm.partners.org/lmm/alamut"
          ],
          "description": "This is not a repreating region."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/12/16 7:30 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "leads to an in-frame deletion of 7 amino acids (not repeat region)."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess300">CritAssess300</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn033">EvLn033</a></p>
         </td>
          <td>
              <p>leads to an in-frame deletion of 5 amino acids (not repeat region).</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit311">Crit311</a></p> (PM4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll041">CanAll041</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess300"><pre><code>{
  "id": "CritAssess300",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit311",
    "type": "Criterion",
    "label": "PM4",
    "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Need to ensure that variant is not located within a repeat region\n\nThis rule is also (likely) applicable for frameshift variants that result in extension of the protein as opposed to truncation (similar to stop loss)."
  },
  "variant": {
    "id": "CanAll041",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll045",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll041",
      "alleleName": "NM_015560.2(OPA1):c.113_130del18",
      "produces": [
        {
          "id": "CtxAll046",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll042",
          "alleleName": "NP_056375.2:p.Arg38_Ser43del",
          "producedBy": "CtxAll045"
        }
      ]
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn033",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon070",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll046",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll042",
            "alleleName": "NP_056375.2:p.Arg38_Ser43del",
            "producedBy": "CtxAll045"
          },
          "consequence": {
            "id": "MolConID019",
            "type": "MolecularConsequence",
            "label": "conservative Inframe Deletion"
          }
        },
        {
          "id": "RegTyp073",
          "type": "RegionType",
          "region": {
            "id": "Loc386",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq292",
              "type": "ReferenceSequence",
              "accession": "NP_056375.2"
            },
            "start": 35,
            "stop": 43
          },
          "annotation": [
            {
              "id": "RegAnnID035",
              "type": "RegionAnnotation",
              "label": "repeat region "
            }
          ],
          "qualifier": "NOT",
          "description": "NP_056375.2[35..43] is not repeating."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/12/16 8:31 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "leads to an in-frame deletion of 5 amino acids (not repeat region)."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess301">CritAssess301</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn034">EvLn034</a></p>
         </td>
          <td>
              <p>This 15bp duplication variant results in an expansion of a 20 residue polyalanine tract to a 25 residue polyalanine tract.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit311">Crit311</a></p> (PM4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll043">CanAll043</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess301"><pre><code>{
  "id": "CritAssess301",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit311",
    "type": "Criterion",
    "label": "PM4",
    "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Need to ensure that variant is not located within a repeat region\n\nThis rule is also (likely) applicable for frameshift variants that result in extension of the protein as opposed to truncation (similar to stop loss)."
  },
  "variant": {
    "id": "CanAll043",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll047",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll043",
      "alleleName": "NM_003924.3(PHOX2B):c.753_767dup",
      "produces": [
        {
          "id": "CtxAll048",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll044",
          "alleleName": "NP_003915.2:p.Ala260_Gly261insAlaAlaAlaAlaAla",
          "producedBy": "CtxAll047"
        }
      ]
    }
  },
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn034",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon074",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll048",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll044",
            "alleleName": "NP_003915.2:p.Ala260_Gly261insAlaAlaAlaAlaAla",
            "producedBy": "CtxAll047"
          },
          "consequence": {
            "id": "MolConID017",
            "type": "MolecularConsequence",
            "label": "conservative Inframe Insertion"
          }
        },
        {
          "id": "RegTyp077",
          "type": "RegionType",
          "region": {
            "id": "Loc387",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq293",
              "type": "ReferenceSequence",
              "accession": "NP_003915.2"
            },
            "start": 241,
            "stop": 260
          },
          "annotation": [
            {
              "id": "RegAnnID035",
              "type": "RegionAnnotation",
              "label": "repeat region "
            }
          ],
          "description": "NP_003915.2[241..260] is repeating."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/12/16 9:32 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This 15bp duplication variant results in an expansion of a 20 residue polyalanine tract to a 25 residue polyalanine tract."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess302">CritAssess302</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn035">EvLn035</a></p><p><a href="../details/details.html#EvLn036">EvLn036</a></p><p><a href="../details/details.html#EvLn037">EvLn037</a></p>
         </td>
          <td>
              <p>From ClinVar: Doolan et al. (2005) (PMID:15698845) reported mutations at the same residue (R162P) in patients with HCM, supporting the functional importance of this residue in the protein. This rare R162P variant observed multiple times in is Likely Pathogenic in ClinVar with 1 star assertion.  Other variants in the same residue R162W has conflicting evidence of pathogenicity(Path by GeneDx : SCV000209174.2 and VUS by LMM:SCV000203864.2). Considered these two only as they have provided assertion criteria.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit312">Crit312</a></p> (PM5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll045">CanAll045</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond016">GenCond016</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess302"><pre><code>{
  "id": "CritAssess302",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit312",
    "type": "Criterion",
    "label": "PM5",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Need to be sure that pathogenic variant exists at the residue in question. Sometimes discrepancy resolution may be required. Some thing like Variantexplorer.org may be useful.\nSometimes no assertion is provided in ClinVar ...may require curation by lit. search."
  },
  "variant": {
    "id": "CanAll045",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll049",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll045",
      "alleleName": "NM_000363.4(TNNI3):c.485G>A",
      "produces": [
        {
          "id": "CtxAll050",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll046",
          "alleleName": "NP_000354.4:p.Arg162Gln",
          "producedBy": "CtxAll049"
        }
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond016",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID160",
          "type": "Disease",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      },
      "name": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant"
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn035",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp078",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll047",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll051",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll047",
              "alleleName": "NM_000363.4(TNNI3):c.485G>C",
              "produces": [
                {
                  "id": "CtxAll052",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll048",
                  "alleleName": "NP_000354.4:p.Arg162Pro",
                  "producedBy": "CtxAll050"
                }
              ]
            }
          },
          "condition": [
            {
              "id": "GenCond016",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID160",
                  "type": "Disease",
                  "label": "hypertrophic cardiomyopathy"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              },
              "name": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant"
            }
          ],
          "outcome": {
            "id": "SigID028",
            "type": "Significance",
            "label": "Likely Pathogenic"
          },
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent011",
                "type": "Agent",
                "label": "Genetics Diagnostic Laboratory,Children's Hospital of Eastern Ontario"
              },
              "contributionDate": "6/1/2014 9:02 AM",
              "contributionRole": {
                "id": "ContribRoleID196",
                "type": "ContributoryRole",
                "label": "interpreter role"
              }
            }
          ],
          "source": [
            "https://www.ncbi.nlm.nih.gov/clinvar/SCV000059953.4"
          ]
        }
      ]
    },
    {
      "id": "EvLn036",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp079",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll049",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll053",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll049",
              "alleleName": "NM_000363.4(TNNI3):c.484C>T",
              "produces": [
                {
                  "id": "CtxAll054",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll050",
                  "alleleName": "NP_000354.4:p.Arg162Trp",
                  "producedBy": "CtxAll053"
                }
              ]
            }
          },
          "condition": [
            {
              "id": "GenCond016",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID160",
                  "type": "Disease",
                  "label": "hypertrophic cardiomyopathy"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              },
              "name": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant"
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          },
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent012",
                "type": "Agent",
                "label": "GeneDx"
              },
              "contributionDate": "6/20/2016 11:12 AM",
              "contributionRole": {
                "id": "ContribRoleID196",
                "type": "ContributoryRole",
                "label": "interpreter role"
              }
            }
          ],
          "source": [
            "https://www.ncbi.nlm.nih.gov/clinvar/SCV000203864.3"
          ]
        }
      ]
    },
    {
      "id": "EvLn037",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp042",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll049",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll053",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll049",
              "alleleName": "NM_000363.4(TNNI3):c.484C>T",
              "produces": [
                {
                  "id": "CtxAll054",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll050",
                  "alleleName": "NP_000354.4:p.Arg162Trp",
                  "producedBy": "CtxAll053"
                }
              ]
            }
          },
          "outcome": {
            "id": "SigID029",
            "type": "Significance",
            "label": "Uncertain Signficance"
          },
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "7/13/16 8:46 AM",
              "contributionRole": {
                "id": "ContribRoleID196",
                "type": "ContributoryRole",
                "label": "interpreter role"
              }
            }
          ],
          "source": [
            "https://www.ncbi.nlm.nih.gov/clinvar/SCV000190642.1"
          ],
          "description": "Uncertain Sig(not specified) NM_000363.4(TNNI3):c.484C>T (p.Arg162Trp)"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "6/12/16 10:48 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "From ClinVar: Doolan et al. (2005) (PMID:15698845) reported mutations at the same residue (R162P) in patients with HCM, supporting the functional importance of this residue in the protein. This rare R162P variant observed multiple times in is Likely Pathogenic in ClinVar with 1 star assertion.  Other variants in the same residue R162W has conflicting evidence of pathogenicity(Path by GeneDx : SCV000209174.2 and VUS by LMM:SCV000203864.2). Considered these two only as they have provided assertion criteria."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess303">CritAssess303</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn038">EvLn038</a></p>
         </td>
          <td>
              <p>Other pathogenic variants at the same residue per ClinVar : NM_000531.5(OTC):c.119G>A (p.Arg40His) However no assertion criteria is provided for these variants. So pathogenicity needs to be determined by literature search. 

So PM5 is applicable when assessing p.Leu1764Il</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit312">Crit312</a></p> (PM5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll051">CanAll051</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond017">GenCond017</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess303"><pre><code>{
  "id": "CritAssess303",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit312",
    "type": "Criterion",
    "label": "PM5",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Need to be sure that pathogenic variant exists at the residue in question. Sometimes discrepancy resolution may be required. Some thing like Variantexplorer.org may be useful.\nSometimes no assertion is provided in ClinVar ...may require curation by lit. search."
  },
  "variant": {
    "id": "CanAll051",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll055",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll051",
      "alleleName": "NM_000531.5(OTC):c.118C>T",
      "produces": [
        {
          "id": "CtxAll056",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll052",
          "alleleName": "NP_000522.3:p.Arg40Cys",
          "producedBy": "CtxAll055"
        }
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond017",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID176",
          "type": "Disease",
          "label": "Ornithine carbamoyltransferase deficiency"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn038",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp080",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll053",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll057",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll053",
              "alleleName": "NM_000531.5(OTC):c.119G>A",
              "produces": [
                {
                  "id": "CtxAll058",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll054",
                  "alleleName": "NP_000522.3:p.Arg40His",
                  "producedBy": "CtxAll057"
                },
                {
                  "id": "CtxAll058",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll054",
                  "alleleName": "NP_000522.3:p.Arg40His",
                  "producedBy": "CtxAll057"
                }
              ]
            }
          },
          "condition": [
            {
              "id": "GenCond017",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID176",
                  "type": "Disease",
                  "label": "Ornithine carbamoyltransferase deficiency"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "6/12/16 11:03 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Other pathogenic variants at the same residue per ClinVar : NM_000531.5(OTC):c.119G>A (p.Arg40His) However no assertion criteria is provided for these variants. So pathogenicity needs to be determined by literature search. \n\nSo PM5 is applicable when assessing p.Leu1764Il"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess304">CritAssess304</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn039">EvLn039</a></p>
         </td>
          <td>
              <p>Another variant at this position, but different AA change, has been observed and called pathogenic for breast ovarian cancer by ENIGMA expert panel:</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit312">Crit312</a></p> (PM5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll055">CanAll055</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond050">GenCond050</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess304"><pre><code>{
  "id": "CritAssess304",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit312",
    "type": "Criterion",
    "label": "PM5",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Need to be sure that pathogenic variant exists at the residue in question. Sometimes discrepancy resolution may be required. Some thing like Variantexplorer.org may be useful.\nSometimes no assertion is provided in ClinVar ...may require curation by lit. search."
  },
  "variant": {
    "id": "CanAll055",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll059",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll055",
      "alleleName": "NM_007294.3(BRCA1):c.5290C>A",
      "produces": [
        {
          "id": "CtxAll060",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll056",
          "alleleName": "NP_009225.1:p.Leu1764Ile",
          "producedBy": "CtxAll059"
        }
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond050",
      "type": "GeneticCondition",
      "name": "Breast Ovarian Cancer, Autosomal Dominant",
      "disease": [
        {
          "id": "DisID148",
          "type": "Disease",
          "label": "hereditary breast ovarian cancer"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn039",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp081",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll057",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll061",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll057",
              "alleleName": "NM_007294.3(BRCA1):c.5291T>C",
              "produces": [
                {
                  "id": "CtxAll062",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll058",
                  "alleleName": "NP_009225.1:p.Leu1764Pro",
                  "producedBy": "CtxAll061"
                }
              ]
            }
          },
          "condition": [
            {
              "id": "GenCond050",
              "type": "GeneticCondition",
              "name": "Breast Ovarian Cancer, Autosomal Dominant",
              "disease": [
                {
                  "id": "DisID148",
                  "type": "Disease",
                  "label": "hereditary breast ovarian cancer"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/1/16 9:23 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Another variant at this position, but different AA change, has been observed and called pathogenic for breast ovarian cancer by ENIGMA expert panel:"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess305">CritAssess305</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn040">EvLn040</a></p>
         </td>
          <td>
              <p>while the variant is absent from exac, however
1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”
2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC).</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit309">Crit309</a></p> (PM2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll035">CanAll035</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess305"><pre><code>{
  "id": "CritAssess305",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit309",
    "type": "Criterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing"
  },
  "variant": {
    "id": "CanAll035",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll039",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll035",
      "alleleName": "NM_001369.2(DNAH5):c.7468_7488del",
      "produces": [
        {
          "id": "CtxAll044",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll040",
          "alleleName": "NP_001360.1:p.Trp2490_Leu2496del",
          "producedBy": "CtxAll039"
        }
      ]
    }
  },
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn040",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq082",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll035",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll039",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll035",
              "alleleName": "NM_001369.2(DNAH5):c.7468_7488del",
              "produces": [
                {
                  "id": "CtxAll044",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll040",
                  "alleleName": "NP_001360.1:p.Trp2490_Leu2496del",
                  "producedBy": "CtxAll039"
                }
              ]
            }
          },
          "alleleCount": 0,
          "alleleFrequency": 0.0,
          "medianCoverage": 11.0,
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "6/1/16 2:15 PM",
              "contributionRole": {
                "id": "ContribRoleID194",
                "type": "ContributoryRole",
                "label": "curator role"
              }
            }
          ],
          "source": [
            "ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.2/"
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/1/16 10:53 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "7/1/16 10:53 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "while the variant is absent from exac, however\n1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”\n2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC)."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess306">CritAssess306</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn041">EvLn041</a></p><p><a href="../details/details.html#EvLn042">EvLn042</a></p>
         </td>
          <td>
              <p>Two sporadic cases of BHD contain this allele</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit313">Crit313</a></p> (PM6)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll059">CanAll059</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond018">GenCond018</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess306"><pre><code>{
  "id": "CritAssess306",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit313",
    "type": "Criterion",
    "label": "PM6",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CanAll059",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll063",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll059",
      "alleleName": "NM_144997.5(FLCN):c.1285dupC",
      "legacyNames": [
        "c.1733insC"
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond018",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID155",
          "type": "Disease",
          "label": "Birt-Hogg-Dube Syndrome"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn041",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond083",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv339",
            "type": "Individual",
            "description": "individual #007"
          },
          "condition": [
            {
              "id": "GenCond018",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID155",
                  "type": "Disease",
                  "label": "Birt-Hogg-Dube Syndrome"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "FamCond085",
          "type": "FamilyCondition",
          "proband": {
            "id": "Indiv339",
            "type": "Individual",
            "description": "individual #007"
          },
          "condition": [
            {
              "id": "GenCond018",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID155",
                  "type": "Disease",
                  "label": "Birt-Hogg-Dube Syndrome"
                }
              ]
            }
          ],
          "familyHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        },
        {
          "id": "IndAllInh087",
          "type": "IndividualAlleleInheritance",
          "canonicalAllele": {
            "id": "CanAll059",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll063",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll059",
              "alleleName": "NM_144997.5(FLCN):c.1285dupC",
              "legacyNames": [
                "c.1733insC"
              ]
            }
          },
          "individual": {
            "id": "Indiv339",
            "type": "Individual",
            "description": "individual #007"
          },
          "parentalConfirmation": {
            "id": "ParenConfID217",
            "type": "ParentalConfirmation",
            "label": "no parental confirmation"
          },
          "alleleInheritance": [
            {
              "id": "AlleleInhID197",
              "type": "AlleleInheritance",
              "label": "de novo"
            }
          ],
          "description": "parentage confirmation not mentioned in the source article"
        }
      ]
    },
    {
      "id": "EvLn042",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond084",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv340",
            "type": "Individual",
            "description": "individual #008"
          },
          "condition": [
            {
              "id": "GenCond018",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID155",
                  "type": "Disease",
                  "label": "Birt-Hogg-Dube Syndrome"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "FamCond086",
          "type": "FamilyCondition",
          "proband": {
            "id": "Indiv340",
            "type": "Individual",
            "description": "individual #008"
          },
          "condition": [
            {
              "id": "GenCond018",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID155",
                  "type": "Disease",
                  "label": "Birt-Hogg-Dube Syndrome"
                }
              ]
            }
          ],
          "familyHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        },
        {
          "id": "IndAllInh088",
          "type": "IndividualAlleleInheritance",
          "canonicalAllele": {
            "id": "CanAll059",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll063",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll059",
              "alleleName": "NM_144997.5(FLCN):c.1285dupC",
              "legacyNames": [
                "c.1733insC"
              ]
            }
          },
          "individual": {
            "id": "Indiv340",
            "type": "Individual",
            "description": "individual #008"
          },
          "parentalConfirmation": {
            "id": "ParenConfID217",
            "type": "ParentalConfirmation",
            "label": "no parental confirmation"
          },
          "alleleInheritance": [
            {
              "id": "AlleleInhID197",
              "type": "AlleleInheritance",
              "label": "de novo"
            }
          ],
          "description": "parentage confirmation not mentioned in the source article"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "7/20/16 7:22 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "7/20/16 7:22 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Two sporadic cases of BHD contain this allele"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess307">CritAssess307</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn043">EvLn043</a></p>
         </td>
          <td>
              <p>affected child with CFC was found to carry c.158T>C. Both parents are unaffected and from Sanger seq neither parent was found to carry c.158T>C variant - so authors conclude variant occurred de novo. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit313">Crit313</a></p> (PM6)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll060">CanAll060</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond019">GenCond019</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess307"><pre><code>{
  "id": "CritAssess307",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit313",
    "type": "Criterion",
    "label": "PM6",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CanAll060",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll064",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll060",
      "alleleName": "NM_002755.3(MAP2K1):c.158T>C"
    }
  },
  "condition": [
    {
      "id": "GenCond019",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID156",
          "type": "Disease",
          "label": "cardiofaciocutaneous syndrome"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn043",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond089",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv341",
            "type": "Individual",
            "description": "individual #009"
          },
          "condition": [
            {
              "id": "GenCond019",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "FamCond090",
          "type": "FamilyCondition",
          "proband": {
            "id": "Indiv341",
            "type": "Individual",
            "description": "individual #009"
          },
          "condition": [
            {
              "id": "GenCond019",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "familyHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        },
        {
          "id": "IndAllInh091",
          "type": "IndividualAlleleInheritance",
          "canonicalAllele": {
            "id": "CanAll060",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll064",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll060",
              "alleleName": "NM_002755.3(MAP2K1):c.158T>C"
            }
          },
          "individual": {
            "id": "Indiv341",
            "type": "Individual",
            "description": "individual #009"
          },
          "parentalConfirmation": {
            "id": "ParenConfID217",
            "type": "ParentalConfirmation",
            "label": "no parental confirmation"
          },
          "alleleInheritance": [
            {
              "id": "AlleleInhID197",
              "type": "AlleleInheritance",
              "label": "de novo"
            }
          ],
          "description": "parentage confirmation not mentioned in the source article"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/20/16 7:49 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "affected child with CFC was found to carry c.158T>C. Both parents are unaffected and from Sanger seq neither parent was found to carry c.158T>C variant - so authors conclude variant occurred de novo. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess308">CritAssess308</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn044">EvLn044</a></p><p><a href="../details/details.html#EvLn045">EvLn045</a></p><p><a href="../details/details.html#EvLn046">EvLn046</a></p>
         </td>
          <td>
              <p>3 unrelated probands with CFC were found to carry c.1741A>G variant. There is no family history of CFC in any family and all three sets of parents were confirmed to not have the variant. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable. However, since there are 3 de novo observations, many groups (including Noonan and MYH7) would allow PM6 to be upgraded to Strong (PM6_S)</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit313">Crit313</a></p> (PM6)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll061">CanAll061</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond068">GenCond068</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess308"><pre><code>{
  "id": "CritAssess308",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit313",
    "type": "Criterion",
    "label": "PM6",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CanAll061",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll065",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll061",
      "alleleName": "NM_004333.4(BRAF):c.1741A>G"
    }
  },
  "condition": [
    {
      "id": "GenCond068",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID156",
          "type": "Disease",
          "label": "cardiofaciocutaneous syndrome"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn044",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond092",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv342",
            "type": "Individual",
            "description": "individual #010"
          },
          "condition": [
            {
              "id": "GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "FamCond093",
          "type": "FamilyCondition",
          "proband": {
            "id": "Indiv342",
            "type": "Individual",
            "description": "individual #010"
          },
          "condition": [
            {
              "id": "GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "familyHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        },
        {
          "id": "IndAllInh094",
          "type": "IndividualAlleleInheritance",
          "canonicalAllele": {
            "id": "CanAll061",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll065",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll061",
              "alleleName": "NM_004333.4(BRAF):c.1741A>G"
            }
          },
          "individual": {
            "id": "Indiv342",
            "type": "Individual",
            "description": "individual #010"
          },
          "parentalConfirmation": {
            "id": "ParenConfID217",
            "type": "ParentalConfirmation",
            "label": "no parental confirmation"
          },
          "alleleInheritance": [
            {
              "id": "AlleleInhID197",
              "type": "AlleleInheritance",
              "label": "de novo"
            }
          ]
        }
      ]
    },
    {
      "id": "EvLn045",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond095",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv343",
            "type": "Individual",
            "description": "individual #011"
          },
          "condition": [
            {
              "id": "GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "FamCond096",
          "type": "FamilyCondition",
          "proband": {
            "id": "Indiv343",
            "type": "Individual",
            "description": "individual #011"
          },
          "condition": [
            {
              "id": "GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "familyHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        },
        {
          "id": "IndAllInh097",
          "type": "IndividualAlleleInheritance",
          "canonicalAllele": {
            "id": "CanAll061",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll065",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll061",
              "alleleName": "NM_004333.4(BRAF):c.1741A>G"
            }
          },
          "individual": {
            "id": "Indiv343",
            "type": "Individual",
            "description": "individual #011"
          },
          "parentalConfirmation": {
            "id": "ParenConfID217",
            "type": "ParentalConfirmation",
            "label": "no parental confirmation"
          },
          "alleleInheritance": [
            {
              "id": "AlleleInhID197",
              "type": "AlleleInheritance",
              "label": "de novo"
            }
          ]
        }
      ]
    },
    {
      "id": "EvLn046",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond098",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv344",
            "type": "Individual",
            "description": "individual #012"
          },
          "condition": [
            {
              "id": "GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "FamCond099",
          "type": "FamilyCondition",
          "proband": {
            "id": "Indiv344",
            "type": "Individual",
            "description": "individual #012"
          },
          "condition": [
            {
              "id": "GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "familyHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        },
        {
          "id": "IndAllInh100",
          "type": "IndividualAlleleInheritance",
          "canonicalAllele": {
            "id": "CanAll061",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll065",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll061",
              "alleleName": "NM_004333.4(BRAF):c.1741A>G"
            }
          },
          "individual": {
            "id": "Indiv344",
            "type": "Individual",
            "description": "individual #012"
          },
          "parentalConfirmation": {
            "id": "ParenConfID217",
            "type": "ParentalConfirmation",
            "label": "no parental confirmation"
          },
          "alleleInheritance": [
            {
              "id": "AlleleInhID197",
              "type": "AlleleInheritance",
              "label": "de novo"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/20/16 8:02 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "3 unrelated probands with CFC were found to carry c.1741A>G variant. There is no family history of CFC in any family and all three sets of parents were confirmed to not have the variant. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable. However, since there are 3 de novo observations, many groups (including Noonan and MYH7) would allow PM6 to be upgraded to Strong (PM6_S)"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess309">CritAssess309</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn047">EvLn047</a></p>
         </td>
          <td>
              <p>In total, 4 segs from 2 families</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit314">Crit314</a></p> (PP1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll063">CanAll063</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond020">GenCond020</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess309"><pre><code>{
  "id": "CritAssess309",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit314",
    "type": "Criterion",
    "label": "PP1",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CanAll063",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll067",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll063",
      "alleleName": "NM_000256.3(MYBPC3)c.1351+1G>A"
    }
  },
  "condition": [
    {
      "id": "GenCond020",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID160",
          "type": "Disease",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn047",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AggSeg105",
          "type": "AggregateSegregation",
          "condition": {
            "id": "GenCond020",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID160",
                "type": "Disease",
                "label": "hypertrophic cardiomyopathy"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CanAll063",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll067",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll063",
              "alleleName": "NM_000256.3(MYBPC3)c.1351+1G>A"
            }
          },
          "familySegregation": [
            {
              "id": "FamSeg103",
              "type": "FamilySegregation",
              "condition": {
                "id": "GenCond020",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "DisID160",
                    "type": "Disease",
                    "label": "hypertrophic cardiomyopathy"
                  }
                ],
                "modeOfInheritance": {
                  "id": "ModeOfInhID051",
                  "type": "ModeOfInheritance",
                  "label": "Autosomal Dominant"
                }
              },
              "canonicalAllele": {
                "id": "CanAll063",
                "type": "CanonicalAllele",
                "preferredCtxAllele": {
                  "id": "CtxAll067",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll063",
                  "alleleName": "NM_000256.3(MYBPC3)c.1351+1G>A"
                }
              },
              "family": {
                "id": "Fam363",
                "type": "Family",
                "label": "12A",
                "description": "Family #001"
              },
              "phenotypePositiveAllelePositive": 3,
              "phenotypeNegativeAllelePositive": 2,
              "contribution": [
                {
                  "type": "Contribution",
                  "agent": {
                    "id": "Agent007",
                    "type": "Agent",
                    "label": "Stefan DiMarino"
                  },
                  "contributionDate": "9/13/16",
                  "contributionRole": {
                    "id": "ContribRoleID194",
                    "type": "ContributoryRole",
                    "label": "curator role"
                  }
                }
              ],
              "source": [
                "http://www.ncbi.nlm.nih.gov/pubmed/17081393"
              ],
              "description": "identified in 1 individual with HCM. Segregation analysis determined 5 other members of the family also carried this variant. However, only 3 of those family members had HCM (aged 48-75). The 2 other family members were young and asymptomatic. So from this paper, only count 3 segregations. The two apparently refuting segregations can be ignored due to the age-dependent penetrance of HCM."
            },
            {
              "id": "FamSeg104",
              "type": "FamilySegregation",
              "condition": {
                "id": "GenCond020",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "DisID160",
                    "type": "Disease",
                    "label": "hypertrophic cardiomyopathy"
                  }
                ],
                "modeOfInheritance": {
                  "id": "ModeOfInhID051",
                  "type": "ModeOfInheritance",
                  "label": "Autosomal Dominant"
                }
              },
              "canonicalAllele": {
                "id": "CanAll063",
                "type": "CanonicalAllele",
                "preferredCtxAllele": {
                  "id": "CtxAll067",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll063",
                  "alleleName": "NM_000256.3(MYBPC3)c.1351+1G>A"
                }
              },
              "family": {
                "id": "Fam364",
                "type": "Family",
                "description": "Family #002"
              },
              "phenotypePositiveAllelePositive": 1,
              "contribution": [
                {
                  "type": "Contribution",
                  "agent": {
                    "id": "Agent001",
                    "type": "Agent",
                    "label": "Harry Stevenson"
                  },
                  "contributionDate": "9/13/16",
                  "contributionRole": {
                    "id": "ContribRoleID194",
                    "type": "ContributoryRole",
                    "label": "curator role"
                  }
                }
              ],
              "source": [
                "http://partners.org/ppm/lmm/gi/variant/12345"
              ],
              "description": "In LMM internal family, c.1351+1G>A was identified in 1 individual with HCM. Variant also found in individual’s sister - who also had clinical dx of HCM."
            }
          ],
          "totalNumberOfSegregations": 4
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 6:45 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "In total, 4 segs from 2 families"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess310">CritAssess310</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn048">EvLn048</a></p>
         </td>
          <td>
              <p>In total, 8 segs from 2 families (can combine hom segs and compound het segs). Most groups would allow this to be shifted up the Moderate (PP1_M)</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit314">Crit314</a></p> (PP1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll062">CanAll062</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond021">GenCond021</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess310"><pre><code>{
  "id": "CritAssess310",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit314",
    "type": "Criterion",
    "label": "PP1",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CanAll062",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll066",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll062",
      "alleleName": "NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)"
    }
  },
  "condition": [
    {
      "id": "GenCond021",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID184",
          "type": "Disease",
          "label": "Usher syndrome"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID052",
        "type": "ModeOfInheritance",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn048",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AggSeg109",
          "type": "AggregateSegregation",
          "condition": {
            "id": "GenCond021",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID184",
                "type": "Disease",
                "label": "Usher syndrome"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID052",
              "type": "ModeOfInheritance",
              "label": "Autosomal Recessive"
            }
          },
          "canonicalAllele": {
            "id": "CanAll062",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll066",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll062",
              "alleleName": "NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)"
            }
          },
          "familySegregation": [
            {
              "id": "FamSeg107",
              "type": "FamilySegregation",
              "condition": {
                "id": "GenCond021",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "DisID184",
                    "type": "Disease",
                    "label": "Usher syndrome"
                  }
                ],
                "modeOfInheritance": {
                  "id": "ModeOfInhID052",
                  "type": "ModeOfInheritance",
                  "label": "Autosomal Recessive"
                }
              },
              "canonicalAllele": {
                "id": "CanAll062",
                "type": "CanonicalAllele",
                "preferredCtxAllele": {
                  "id": "CtxAll066",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll062",
                  "alleleName": "NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)"
                }
              },
              "family": {
                "id": "Fam365",
                "type": "Family",
                "description": "Family #003"
              },
              "phenotypePositiveAllelePositive": 5,
              "description": "proband and 5 affected family members all homozygous for NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)."
            },
            {
              "id": "FamSeg108",
              "type": "FamilySegregation",
              "condition": {
                "id": "GenCond021",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "DisID184",
                    "type": "Disease",
                    "label": "Usher syndrome"
                  }
                ],
                "modeOfInheritance": {
                  "id": "ModeOfInhID052",
                  "type": "ModeOfInheritance",
                  "label": "Autosomal Recessive"
                }
              },
              "canonicalAllele": {
                "id": "CanAll062",
                "type": "CanonicalAllele",
                "preferredCtxAllele": {
                  "id": "CtxAll066",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll062",
                  "alleleName": "NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)"
                }
              },
              "family": {
                "id": "Fam366",
                "type": "Family",
                "description": "Family #004"
              },
              "description": "c.1000C>T identified in het state in 1 individual with Usher. Individual compound het with Gly1840Val USH2A variant - confirmed in trans. Gly1840Val called Pathogenic. Individual has 3 affected sibs all of whom are compound het with 1000C>T & Gly1840Val. One additional sib only carried 1000C>T and is unaffected."
            }
          ],
          "totalNumberOfSegregations": 8
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 7:02 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "In total, 8 segs from 2 families (can combine hom segs and compound het segs). Most groups would allow this to be shifted up the Moderate (PP1_M)"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess311">CritAssess311</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn049">EvLn049</a></p>
         </td>
          <td>
              <p>PP1 is for “Co-segregation with disease in multiple affected family members” PP1 is not applicable for this variant because it’s only a single segregation.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit314">Crit314</a></p> (PP1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll064">CanAll064</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond051">GenCond051</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess311"><pre><code>{
  "id": "CritAssess311",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit314",
    "type": "Criterion",
    "label": "PP1",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CanAll064",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll068",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll064",
      "alleleName": "NM_000059.3(BRCA2):c.1832C>G"
    }
  },
  "condition": [
    {
      "id": "GenCond051",
      "type": "GeneticCondition",
      "name": "Breast Ovarian Cancer, Autosomal Dominant",
      "disease": [
        {
          "id": "DisID148",
          "type": "Disease",
          "label": "hereditary breast ovarian cancer"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn049",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "FamSeg111",
          "type": "FamilySegregation",
          "condition": {
            "id": "GenCond051",
            "type": "GeneticCondition",
            "name": "Breast Ovarian Cancer, Autosomal Dominant",
            "disease": [
              {
                "id": "DisID148",
                "type": "Disease",
                "label": "hereditary breast ovarian cancer"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CanAll064",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll068",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll064",
              "alleleName": "NM_000059.3(BRCA2):c.1832C>G"
            }
          },
          "family": {
            "id": "Fam367",
            "type": "Family",
            "description": "Family #005"
          },
          "phenotypePositiveAllelePositive": 2
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 7:39 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "PP1 is for “Co-segregation with disease in multiple affected family members” PP1 is not applicable for this variant because it’s only a single segregation."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess312">CritAssess312</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn050">EvLn050</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit314">Crit314</a></p> (PP1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll065">CanAll065</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond002">GenCond002</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess312"><pre><code>{
  "id": "CritAssess312",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit314",
    "type": "Criterion",
    "label": "PP1",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CanAll065",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll069",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll065",
      "alleleName": "NM_001001430.2(TNNT2):c.629_631delAGA"
    }
  },
  "condition": [
    {
      "id": "GenCond002",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID162",
          "type": "Disease",
          "label": "dilated cardiomyopathy"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn050",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AggSeg115",
          "type": "AggregateSegregation",
          "canonicalAllele": {
            "id": "CanAll065",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll069",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll065",
              "alleleName": "NM_001001430.2(TNNT2):c.629_631delAGA"
            }
          },
          "condition": {
            "id": "GenCond002",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID162",
                "type": "Disease",
                "label": "dilated cardiomyopathy"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "familySegregation": [
            {
              "id": "FamSeg113",
              "type": "FamilySegregation",
              "condition": {
                "id": "GenCond002",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "DisID162",
                    "type": "Disease",
                    "label": "dilated cardiomyopathy"
                  }
                ],
                "modeOfInheritance": {
                  "id": "ModeOfInhID051",
                  "type": "ModeOfInheritance",
                  "label": "Autosomal Dominant"
                }
              },
              "canonicalAllele": {
                "id": "CanAll065",
                "type": "CanonicalAllele",
                "preferredCtxAllele": {
                  "id": "CtxAll069",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll065",
                  "alleleName": "NM_001001430.2(TNNT2):c.629_631delAGA"
                }
              },
              "family": {
                "id": "Fam368",
                "type": "Family",
                "description": "Family #006"
              },
              "phenotypePositiveAllelePositive": 1,
              "phenotypeNegativeAllelePositive": 1,
              "columns": [
                "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”]"
              ],
              "affectedValues": [
                "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
              ],
              "genotypeValues": [
                "{0: “Unknown”,1:”Reference”, 2: CA65}"
              ],
              "pedigree": [
                "[[Fam006,Fam006.1,0,0,1,2,0/0],\n   [Fam006,Fam006.2,0,0,2,1,0/0],\n   [Fam006,Fam006.3,Fam006.1,Fam006.2,2,3,0/0],\n   [Fam006,Fam006.4,Fam006.1,Fam006.2,1,2,1/2],\n   [Fam006,Fam006.5,Fam006.1,Fam006.2,2,1,1/1],\n   [Fam006,Fam006.6,Fam006.1,Fam006.2,1,2,1/2]]"
              ]
            },
            {
              "id": "FamSeg114",
              "type": "FamilySegregation",
              "condition": {
                "id": "GenCond002",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "DisID162",
                    "type": "Disease",
                    "label": "dilated cardiomyopathy"
                  }
                ],
                "modeOfInheritance": {
                  "id": "ModeOfInhID051",
                  "type": "ModeOfInheritance",
                  "label": "Autosomal Dominant"
                }
              },
              "canonicalAllele": {
                "id": "CanAll065",
                "type": "CanonicalAllele",
                "preferredCtxAllele": {
                  "id": "CtxAll069",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll065",
                  "alleleName": "NM_001001430.2(TNNT2):c.629_631delAGA"
                }
              },
              "family": {
                "id": "Fam369",
                "type": "Family",
                "description": "Family #007"
              },
              "phenotypePositiveAllelePositive": 2,
              "phenotypeNegativeAllelePositive": 3,
              "columns": [
                "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”,”ObligateGenotype”]"
              ],
              "affectedValues": [
                "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
              ],
              "genotypeValues": [
                "{0: “Unknown”,1:”Reference”, 2: CA65}"
              ],
              "pedigree": [
                "[[Fam007, Fam007.1, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.2, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.3, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.4, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.5, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.6, Fam007.1, Fam007.2, 2, 3, 1/2, 1],\n   [Fam007, Fam007.7,0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.8, Fam007.1, Fam007.2, 2, 1, 1/2, 1],\n   [Fam007, Fam007.9,Fam007.3, Fam007.4, 1, 1, 0/0, 0],\n   [Fam007, Fam007.10, 0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.11, Fam007.5, Fam007.6, 2, 2, 1/2, 0],\n   [Fam007, Fam007.12, Fam007.5, Fam007.6, 1,2, 0/0, 0],\n   [Fam007, Fam007.13, Fam007.5, Fam007.6, 2,1, 1/1,0],\n   [Fam007, Fam007.14, Fam007.5, Fam007.6, 1, 1, 1/1, 0],\n   [Fam007, Fam007.15, Fam007.7, Fam007.8, 1, 2, 1/2, 0],\n   [Fam007, Fam007.16, Fam007.8, Fam007.9, 1, 2, 1/2, 0],\n   [Fam007, Fam007.17, Fam007.8, Fam007.9, 1, 2, 0/0,0],\n   [Fam007, Fam007.18, Fam007.8, Fam007.9, 1, 2, 0/0, 0],\n   [Fam007, Fam007.19, Fam007.10, Fam007.11, 1, 2, 0/0,0],\n   [Fam007.20,Fam007.10, Fam007.11, 1,1,1/1,0]]"
              ]
            }
          ],
          "totalNumberOfSegregations": 9
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent002",
        "type": "Agent",
        "label": "Noonan/Rasopathy WG (ClinGen)"
      },
      "contributionDate": "7/21/16 8:11 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess313">CritAssess313</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn051">EvLn051</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit315">Crit315</a></p> (PP2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll066">CanAll066</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess313"><pre><code>{
  "id": "CritAssess313",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit315",
    "type": "Criterion",
    "label": "PP2",
    "description": "Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    },
    "comments": "Seems no one has established firm rules for when this rule is applicable so not sure how great any examples will be\nAgree with the lack of firm rules. Seems like one criteria may be observed missense variants over 5% allele frequency in ExAC. If this seems reasonable then I could generate a couple of examples."
  },
  "variant": {
    "id": "CanAll066",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll070",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll066",
      "alleleName": "NM_000257.3(MYH7):c.5401G>A"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn051",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon117",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll070",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll066",
            "alleleName": "NM_000257.3(MYH7):c.5401G>A"
          },
          "consequence": {
            "id": "MolConID011",
            "type": "MolecularConsequence",
            "label": "missense variant"
          }
        },
        {
          "id": "CondMech118",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MolConID011",
            "type": "MolecularConsequence",
            "label": "missense variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID133",
            "type": "ConditionMechanismStrength",
            "label": "common"
          },
          "gene": {
            "id": "G234",
            "type": "Gene",
            "geneSymbol": "MYH7"
          },
          "description": "Of 178 pathogenic  alleles in ClinVar, 149 are missense."
        },
        {
          "id": "BMVR119",
          "type": "BenignMissenseVariationRate",
          "gene": {
            "id": "G234",
            "type": "Gene",
            "geneSymbol": "MYH7"
          },
          "value": {
            "id": "BMVROID129",
            "type": "BenignMissenseVariationRateOutcome",
            "label": "Low"
          },
          "description": "Based on the gene size, ExAC expects 740 missense variants to be observed, but actually observes only 420, giving a z score of 6.5, which is in the top 0.02% of genes."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 9:41 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess314">CritAssess314</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn052">EvLn052</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit316">Crit316</a></p> (PP3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll067">CanAll067</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess314"><pre><code>{
  "id": "CritAssess314",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit316",
    "type": "Criterion",
    "label": "PP3",
    "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    },
    "comments": "Don’t know if there is a definition of “multiple” . If >1 then the algorithms chosen are at the discretion of the curator/lab.  There is a difference between lines of evidence and number of implementations."
  },
  "variant": {
    "id": "CanAll067",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll071",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll067",
      "alleleName": "NM_005228.3(EGFR):c.2369C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn052",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "InSilicoPred120",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq287",
            "type": "ReferenceSequence",
            "accession": "NM_005228.3",
            "produces": [
              {
                "id": "RefSeq288",
                "type": "ReferenceSequence",
                "accession": "NP_005219.2",
                "producedBy": "RefSeq287"
              }
            ]
          },
          "canonicalAllele": {
            "id": "CanAll067",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll071",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll067",
              "alleleName": "NM_005228.3(EGFR):c.2369C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "SIFT",
          "prediction": "Damaging"
        },
        {
          "id": "InSilicoPred121",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq287",
            "type": "ReferenceSequence",
            "accession": "NM_005228.3",
            "produces": [
              {
                "id": "RefSeq288",
                "type": "ReferenceSequence",
                "accession": "NP_005219.2",
                "producedBy": "RefSeq287"
              }
            ]
          },
          "canonicalAllele": {
            "id": "CanAll067",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll071",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll067",
              "alleleName": "NM_005228.3(EGFR):c.2369C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Polyphen",
          "prediction": "Probably damaging"
        },
        {
          "id": "InSilicoPred122",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq287",
            "type": "ReferenceSequence",
            "accession": "NM_005228.3",
            "produces": [
              {
                "id": "RefSeq288",
                "type": "ReferenceSequence",
                "accession": "NP_005219.2",
                "producedBy": "RefSeq287"
              }
            ]
          },
          "canonicalAllele": {
            "id": "CanAll067",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll071",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll067",
              "alleleName": "NM_005228.3(EGFR):c.2369C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "FATHMM",
          "prediction": "Tolerated"
        },
        {
          "id": "InSilicoPred123",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq287",
            "type": "ReferenceSequence",
            "accession": "NM_005228.3",
            "produces": [
              {
                "id": "RefSeq288",
                "type": "ReferenceSequence",
                "accession": "NP_005219.2",
                "producedBy": "RefSeq287"
              }
            ]
          },
          "canonicalAllele": {
            "id": "CanAll067",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll071",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll067",
              "alleleName": "NM_005228.3(EGFR):c.2369C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "LRT",
          "prediction": "Damaging"
        },
        {
          "id": "InSilicoPred124",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq287",
            "type": "ReferenceSequence",
            "accession": "NM_005228.3",
            "produces": [
              {
                "id": "RefSeq288",
                "type": "ReferenceSequence",
                "accession": "NP_005219.2",
                "producedBy": "RefSeq287"
              }
            ]
          },
          "canonicalAllele": {
            "id": "CanAll067",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll071",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll067",
              "alleleName": "NM_005228.3(EGFR):c.2369C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "MetaSVM",
          "prediction": "Tolerated"
        },
        {
          "id": "InSilicoPred125",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq287",
            "type": "ReferenceSequence",
            "accession": "NM_005228.3",
            "produces": [
              {
                "id": "RefSeq288",
                "type": "ReferenceSequence",
                "accession": "NP_005219.2",
                "producedBy": "RefSeq287"
              }
            ]
          },
          "canonicalAllele": {
            "id": "CanAll067",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll071",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll067",
              "alleleName": "NM_005228.3(EGFR):c.2369C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Mutation assessor",
          "prediction": "Tolerated"
        },
        {
          "id": "InSilicoPred126",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq287",
            "type": "ReferenceSequence",
            "accession": "NM_005228.3",
            "produces": [
              {
                "id": "RefSeq288",
                "type": "ReferenceSequence",
                "accession": "NP_005219.2",
                "producedBy": "RefSeq287"
              }
            ]
          },
          "canonicalAllele": {
            "id": "CanAll067",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll071",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll067",
              "alleleName": "NM_005228.3(EGFR):c.2369C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Mutation taster",
          "prediction": "Disease-causing"
        },
        {
          "id": "InSilicoPred127",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq287",
            "type": "ReferenceSequence",
            "accession": "NM_005228.3",
            "produces": [
              {
                "id": "RefSeq288",
                "type": "ReferenceSequence",
                "accession": "NP_005219.2",
                "producedBy": "RefSeq287"
              }
            ]
          },
          "canonicalAllele": {
            "id": "CanAll067",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll071",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll067",
              "alleleName": "NM_005228.3(EGFR):c.2369C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Provean",
          "prediction": "Damaging"
        },
        {
          "id": "InSilicoPred128",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq287",
            "type": "ReferenceSequence",
            "accession": "NM_005228.3",
            "produces": [
              {
                "id": "RefSeq288",
                "type": "ReferenceSequence",
                "accession": "NP_005219.2",
                "producedBy": "RefSeq287"
              }
            ]
          },
          "canonicalAllele": {
            "id": "CanAll067",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll071",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll067",
              "alleleName": "NM_005228.3(EGFR):c.2369C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Condel",
          "prediction": "deleterious"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "7/21/16 10:29 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "7/21/16 10:29 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess315">CritAssess315</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn053">EvLn053</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit316">Crit316</a></p> (PP3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll068">CanAll068</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess315"><pre><code>{
  "id": "CritAssess315",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit316",
    "type": "Criterion",
    "label": "PP3",
    "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    },
    "comments": "Don’t know if there is a definition of “multiple” . If >1 then the algorithms chosen are at the discretion of the curator/lab.  There is a difference between lines of evidence and number of implementations."
  },
  "variant": {
    "id": "CanAll068",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll072",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll068",
      "alleleName": "NM_003119.3(SPG7):c.1529C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn053",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "InSilicoPred129",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq298",
            "type": "ReferenceSequence",
            "accession": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CanAll068",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll072",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll068",
              "alleleName": "NM_003119.3(SPG7):c.1529C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "SIFT",
          "prediction": "deleterious"
        },
        {
          "id": "InSilicoPred130",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq298",
            "type": "ReferenceSequence",
            "accession": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CanAll068",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll072",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll068",
              "alleleName": "NM_003119.3(SPG7):c.1529C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Polyphen",
          "prediction": "Probably damaging"
        },
        {
          "id": "InSilicoPred131",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq298",
            "type": "ReferenceSequence",
            "accession": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CanAll068",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll072",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll068",
              "alleleName": "NM_003119.3(SPG7):c.1529C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "FATHMM",
          "prediction": "Damaging"
        },
        {
          "id": "InSilicoPred132",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq298",
            "type": "ReferenceSequence",
            "accession": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CanAll068",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll072",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll068",
              "alleleName": "NM_003119.3(SPG7):c.1529C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "LRT",
          "prediction": "Damaging"
        },
        {
          "id": "InSilicoPred133",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq298",
            "type": "ReferenceSequence",
            "accession": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CanAll068",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll072",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll068",
              "alleleName": "NM_003119.3(SPG7):c.1529C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "MetaSVM",
          "prediction": "Tolerated"
        },
        {
          "id": "InSilicoPred134",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq298",
            "type": "ReferenceSequence",
            "accession": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CanAll068",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll072",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll068",
              "alleleName": "NM_003119.3(SPG7):c.1529C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Mutation assessor",
          "prediction": "Medium functional impact"
        },
        {
          "id": "InSilicoPred135",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq298",
            "type": "ReferenceSequence",
            "accession": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CanAll068",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll072",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll068",
              "alleleName": "NM_003119.3(SPG7):c.1529C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Mutation taster",
          "prediction": "Disease-causing"
        },
        {
          "id": "InSilicoPred136",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq298",
            "type": "ReferenceSequence",
            "accession": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CanAll068",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll072",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll068",
              "alleleName": "NM_003119.3(SPG7):c.1529C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Provean",
          "prediction": "Damaging"
        },
        {
          "id": "InSilicoPred137",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq298",
            "type": "ReferenceSequence",
            "accession": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CanAll068",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll072",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll068",
              "alleleName": "NM_003119.3(SPG7):c.1529C>T"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Condel",
          "prediction": "deleterious"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "7/21/16 12:15 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess316">CritAssess316</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn054">EvLn054</a></p>
         </td>
          <td>
              <p>even though the computational tools for AA changes are NOT predicting an impact, splicing tools are predicting an impact - so PP3 is applicable</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit316">Crit316</a></p> (PP3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll069">CanAll069</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess316"><pre><code>{
  "id": "CritAssess316",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit316",
    "type": "Criterion",
    "label": "PP3",
    "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    },
    "comments": "Don’t know if there is a definition of “multiple” . If >1 then the algorithms chosen are at the discretion of the curator/lab.  There is a difference between lines of evidence and number of implementations."
  },
  "variant": {
    "id": "CanAll069",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll075",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll069",
      "alleleName": "NM_000363.4(TNNI3):c.592C>G"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn054",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "InSilicoPred138",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "SIFT",
          "prediction": "Tolerated"
        },
        {
          "id": "InSilicoPred139",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Polyphen2",
          "prediction": "benign"
        },
        {
          "id": "InSilicoPred140",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "AlignGVGD",
          "prediction": "C0 (benign)"
        },
        {
          "id": "InSilicoPred141",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "MAPP",
          "prediction": "good (benign)"
        },
        {
          "id": "InSilicoPred142",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "MutationTaster",
          "prediction": "disease-causing"
        },
        {
          "id": "InSilicoPredScore143",
          "type": "InSilicoPredictionScore",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID073",
            "type": "Prediction",
            "label": "splicing prediction"
          },
          "algorithm": "SSF",
          "prediction": 71.53
        },
        {
          "id": "InSilicoPredScore144",
          "type": "InSilicoPredictionScore",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID073",
            "type": "Prediction",
            "label": "splicing prediction"
          },
          "algorithm": "MaxEnt",
          "prediction": 7.39
        },
        {
          "id": "InSilicoPredScore145",
          "type": "InSilicoPredictionScore",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID073",
            "type": "Prediction",
            "label": "splicing prediction"
          },
          "algorithm": "NNSPLICE",
          "prediction": 0.87
        },
        {
          "id": "InSilicoPredScore146",
          "type": "InSilicoPredictionScore",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID073",
            "type": "Prediction",
            "label": "splicing prediction"
          },
          "algorithm": "GeneSplicer",
          "prediction": 7.75
        },
        {
          "id": "InSilicoPredScore147",
          "type": "InSilicoPredictionScore",
          "transcript": {
            "id": "RefSeq299",
            "type": "ReferenceSequence",
            "accession": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CanAll069",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll075",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll069",
              "alleleName": "NM_000363.4(TNNI3):c.592C>G"
            }
          },
          "predictionType": {
            "id": "PredID073",
            "type": "Prediction",
            "label": "splicing prediction"
          },
          "algorithm": "HSF",
          "prediction": 81.75
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 2:46 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "even though the computational tools for AA changes are NOT predicting an impact, splicing tools are predicting an impact - so PP3 is applicable"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess318">CritAssess318</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn055">EvLn055</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit317">Crit317</a></p> (PP4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll070">CanAll070</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond022">GenCond022</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess318"><pre><code>{
  "id": "CritAssess318",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit317",
    "type": "Criterion",
    "label": "PP4",
    "description": "Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CanAll070",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll076",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll070",
      "alleleName": "NM_001114753.1(ENG):c.818C>T"
    }
  },
  "condition": [
    {
      "id": "GenCond022",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID180",
          "type": "Disease",
          "label": "hereditary hemorrhagic telangiectasia"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn055",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "LocHet148",
          "type": "LocusHeterogeneity",
          "condition": {
            "id": "GenCond022",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID180",
                "type": "Disease",
                "label": "hereditary hemorrhagic telangiectasia"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "gene": {
            "id": "G261",
            "type": "Gene",
            "geneSymbol": "ENG"
          },
          "specificity": {
            "id": "LocSpecID211",
            "type": "LocusSpecificity",
            "label": "complete"
          },
          "description": "HHT type 1 is only known to be caused by variants in ENG -"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/28/16 2:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "8/28/16 2:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess319">CritAssess319</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn056">EvLn056</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit317">Crit317</a></p> (PP4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll071">CanAll071</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond064">GenCond064</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess319"><pre><code>{
  "id": "CritAssess319",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit317",
    "type": "Criterion",
    "label": "PP4",
    "description": "Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CanAll071",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll077",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll071",
      "alleleName": "NM_000257.2(MYH7):c.1822T>G"
    }
  },
  "condition": [
    {
      "id": "GenCond064",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID160",
          "type": "Disease",
          "label": "hypertrophic cardiomyopathy"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn056",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "LocHet149",
          "type": "LocusHeterogeneity",
          "condition": {
            "id": "GenCond064",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID160",
                "type": "Disease",
                "label": "hypertrophic cardiomyopathy"
              }
            ]
          },
          "gene": {
            "id": "G234",
            "type": "Gene",
            "geneSymbol": "MYH7"
          },
          "specificity": {
            "id": "LocSpecID209",
            "type": "LocusSpecificity",
            "label": "low"
          },
          "description": "1.Only ~50% of HCM is genetic (can be caused by environment) \n2. More than 10 genes can cause HCM (specifically 9 sarcomere genes)"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 3:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess320">CritAssess320</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn057">EvLn057</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit318">Crit318</a></p> (PP5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll072">CanAll072</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond023">GenCond023</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess320"><pre><code>{
  "id": "CritAssess320",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit318",
    "type": "Criterion",
    "label": "PP5",
    "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CanAll072",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll078",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll072",
      "alleleName": "NM_000059.3(BRCA2):c.3G>T"
    }
  },
  "condition": [
    {
      "id": "GenCond023",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID153",
          "type": "Disease",
          "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn057",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp150",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll072",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll078",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll072",
              "alleleName": "NM_000059.3(BRCA2):c.3G>T"
            }
          },
          "condition": [
            {
              "id": "GenCond023",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID153",
                  "type": "Disease",
                  "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          },
          "description": "Variant has been classified as “Pathogenic” by Sharing Clinical Reports Project (SCV000054057.3)"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 4:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess321">CritAssess321</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn058">EvLn058</a></p>
         </td>
          <td>
              <p>OMIM is not considered an expert or reputable source for variant interpretations.
Further, OMIM’s evidence for this pathogenic call is available (the publication link - PMID:21620354 - and free text summary) so people should use the evidence from OMIM in their assessment NOT OMIM’s interpretation.  This is marked as Insufficient Evidence rather than refuted because the OMIM entry does not rule out a later application of PP5 based on eg. an expert panel.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit318">Crit318</a></p> (PP5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll073">CanAll073</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond024">GenCond024</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess321"><pre><code>{
  "id": "CritAssess321",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit318",
    "type": "Criterion",
    "label": "PP5",
    "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CanAll073",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll079",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll073",
      "alleleName": "NM_001458.4(FLNC):c.577G>A"
    }
  },
  "condition": [
    {
      "id": "GenCond024",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID154",
          "type": "Disease",
          "label": "Myopathy, Distal, type 4"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn058",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp151",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll073",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll079",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll073",
              "alleleName": "NM_001458.4(FLNC):c.577G>A"
            }
          },
          "condition": [
            {
              "id": "GenCond024",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID154",
                  "type": "Disease",
                  "label": "Myopathy, Distal, type 4"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          },
          "description": "Variant has been classified as “Pathogenic” by OMIM (SCV000043718.1)"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 5:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "OMIM is not considered an expert or reputable source for variant interpretations.\nFurther, OMIM’s evidence for this pathogenic call is available (the publication link - PMID:21620354 - and free text summary) so people should use the evidence from OMIM in their assessment NOT OMIM’s interpretation.  This is marked as Insufficient Evidence rather than refuted because the OMIM entry does not rule out a later application of PP5 based on eg. an expert panel."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess322">CritAssess322</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn059">EvLn059</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit318">Crit318</a></p> (PP5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll074">CanAll074</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond025">GenCond025</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess322"><pre><code>{
  "id": "CritAssess322",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit318",
    "type": "Criterion",
    "label": "PP5",
    "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "EvLnStrgID144",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CanAll074",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll080",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll074",
      "alleleName": "NM_000492.3(CFTR):c.2053dupC"
    }
  },
  "condition": [
    {
      "id": "GenCond025",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID174",
          "type": "Disease",
          "label": "Cystic Fibrosis"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID052",
        "type": "ModeOfInheritance",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn059",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp152",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll074",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll080",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll074",
              "alleleName": "NM_000492.3(CFTR):c.2053dupC"
            }
          },
          "condition": [
            {
              "id": "GenCond025",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID174",
                  "type": "Disease",
                  "label": "Cystic Fibrosis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          },
          "description": "Variant has been classified as “Pathogenic” by Expert panel CFTR (SCV000245981.1)"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 6:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess323">CritAssess323</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn060">EvLn060</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit319">Crit319</a></p> (BA1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll075">CanAll075</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess323"><pre><code>{
  "id": "CritAssess323",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit319",
    "type": "Criterion",
    "label": "BA1",
    "description": "Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "defaultStrength": {
      "id": "EvLnStrgID138",
      "type": "EvidenceLineStrength",
      "label": "Benign Stand Alone"
    },
    "comments": "Several groups have lowered the BA1 threshold (e.g. PTEN-1%). Also the sequence variant interpretation group is working on a modified version of this guideline including the minimum # of alleles that need to be examined and the population stratification effects, if any."
  },
  "variant": {
    "id": "CanAll075",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll081",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll075",
      "alleleName": "NM_000057.2 (BLM):c.2603C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn060",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq153",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll075",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll081",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll075",
              "alleleName": "NM_000057.2 (BLM):c.2603C>T"
            }
          },
          "alleleCount": 6732,
          "alleleNumber": 120402,
          "alleleFrequency": 0.05591,
          "homozygousAlleleIndividualCount": 223
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 7:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess324">CritAssess324</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn061">EvLn061</a></p><p><a href="../details/details.html#EvLn062">EvLn062</a></p>
         </td>
          <td>
              <p>Globally, this variant has a MAF of 0.0454 (4.5%) which would put it under the 5% rule and thus BA1 would not be applicable. However, variant is enriched in East Asian population - 29% MAF - so way over the 5% cut-off. So now BA1 is applicable</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit319">Crit319</a></p> (BA1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll076">CanAll076</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess324"><pre><code>{
  "id": "CritAssess324",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit319",
    "type": "Criterion",
    "label": "BA1",
    "description": "Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "defaultStrength": {
      "id": "EvLnStrgID138",
      "type": "EvidenceLineStrength",
      "label": "Benign Stand Alone"
    },
    "comments": "Several groups have lowered the BA1 threshold (e.g. PTEN-1%). Also the sequence variant interpretation group is working on a modified version of this guideline including the minimum # of alleles that need to be examined and the population stratification effects, if any."
  },
  "variant": {
    "id": "CanAll076",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll082",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll076",
      "alleleName": "NM_004004.5(GJB2):c.79G>A"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn061",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq154",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll076",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll082",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll076",
              "alleleName": "NM_004004.5(GJB2):c.79G>A"
            }
          },
          "alleleCount": 5507,
          "alleleNumber": 121350,
          "alleleFrequency": 0.04538,
          "homozygousAlleleIndividualCount": 768
        }
      ]
    },
    {
      "id": "EvLn062",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq155",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID088",
            "type": "Population",
            "label": "East Asian"
          },
          "allele": {
            "id": "CanAll076",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll082",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll076",
              "alleleName": "NM_004004.5(GJB2):c.79G>A"
            }
          },
          "alleleCount": 2518,
          "alleleNumber": 8632,
          "alleleFrequency": 0.2917,
          "homozygousAlleleIndividualCount": 405
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 8:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Globally, this variant has a MAF of 0.0454 (4.5%) which would put it under the 5% rule and thus BA1 would not be applicable. However, variant is enriched in East Asian population - 29% MAF - so way over the 5% cut-off. So now BA1 is applicable"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess325">CritAssess325</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn063">EvLn063</a></p>
         </td>
          <td>
              <p>Although this variant has been seen in the general population, its frequency is low enough to be consistent with a recessive carrier frequency.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit309">Crit309</a></p> (PM2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll077">CanAll077</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond045">GenCond045</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess325"><pre><code>{
  "id": "CritAssess325",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit309",
    "type": "Criterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing"
  },
  "variant": {
    "id": "CanAll077",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll083",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll077",
      "alleleName": "NM_023036.4(DNAI2):c.1304G>A"
    }
  },
  "condition": [
    {
      "id": "GenCond045",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID163",
          "type": "Disease",
          "label": "Primary ciliary dyskenisia"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID052",
        "type": "ModeOfInheritance",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn063",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq157",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll077",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll083",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll077",
              "alleleName": "NM_023036.4(DNAI2):c.1304G>A"
            }
          },
          "alleleCount": 14,
          "alleleNumber": 121100,
          "alleleFrequency": 0.0001158,
          "homozygousAlleleIndividualCount": 0
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 9:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Although this variant has been seen in the general population, its frequency is low enough to be consistent with a recessive carrier frequency."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess326">CritAssess326</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn064">EvLn064</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit320">Crit320</a></p> (BS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll078">CanAll078</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond026">GenCond026</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess326"><pre><code>{
  "id": "CritAssess326",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit320",
    "type": "Criterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CanAll078",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll084",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll078",
      "alleleName": "NM_000138.4(FBN1):c.8176C>T"
    }
  },
  "condition": [
    {
      "id": "GenCond026",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID173",
          "type": "Disease",
          "label": "Marfan Syndrome"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn064",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq158",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll078",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll084",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll078",
              "alleleName": "NM_000138.4(FBN1):c.8176C>T"
            }
          },
          "alleleCount": 89,
          "alleleNumber": 121410,
          "alleleFrequency": 0.0007,
          "homozygousAlleleIndividualCount": 1
        },
        {
          "id": "CondPrev159",
          "type": "ConditionPrevelance",
          "condition": [
            {
              "id": "GenCond026",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID173",
                  "type": "Disease",
                  "label": "Marfan Syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "minimum": 0.0017,
          "maximum": 0.0007,
          "population": {
            "id": "PopID094",
            "type": "Population",
            "label": "Overall"
          }
        },
        {
          "id": "CondPenet160",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond026",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID173",
                  "type": "Disease",
                  "label": "Marfan Syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID131",
            "type": "Penetrance",
            "label": "High"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "8/29/16 10:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess327">CritAssess327</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn065">EvLn065</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit320">Crit320</a></p> (BS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll079">CanAll079</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond027">GenCond027</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess327"><pre><code>{
  "id": "CritAssess327",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit320",
    "type": "Criterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CanAll079",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll085",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll079",
      "alleleName": "NM_004541.3(NDUFA1):c.94G>C"
    }
  },
  "condition": [
    {
      "id": "GenCond027",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID165",
          "type": "Disease",
          "label": "Mitochondrial complex 1 deficiency"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID045",
        "type": "ModeOfInheritance",
        "label": "X-linked dominant inheritance"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn065",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq162",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll079",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll085",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll079",
              "alleleName": "NM_004541.3(NDUFA1):c.94G>C"
            }
          },
          "alleleCount": 545,
          "alleleNumber": 87696,
          "alleleFrequency": 0.00625,
          "homozygousAlleleIndividualCount": 1,
          "hemizygousAlleleIndividualCount": 206
        },
        {
          "id": "CondPrev163",
          "type": "ConditionPrevelance",
          "condition": [
            {
              "id": "GenCond027",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID165",
                  "type": "Disease",
                  "label": "Mitochondrial complex 1 deficiency"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID045",
                "type": "ModeOfInheritance",
                "label": "X-linked dominant inheritance"
              }
            }
          ],
          "prevelance": 0.00012,
          "population": {
            "id": "PopID094",
            "type": "Population",
            "label": "Overall"
          }
        },
        {
          "id": "CondPenet164",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond027",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID165",
                  "type": "Disease",
                  "label": "Mitochondrial complex 1 deficiency"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID045",
                "type": "ModeOfInheritance",
                "label": "X-linked dominant inheritance"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID131",
            "type": "Penetrance",
            "label": "High"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "8/29/16 11:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "8/29/16 11:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess328">CritAssess328</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn066">EvLn066</a></p>
         </td>
          <td>
              <p>Perrault syndrome may be underreported due to misdiagnosis.  If the prevalence estimates rise, this assessment may need to be revisited</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit320">Crit320</a></p> (BS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll080">CanAll080</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond028">GenCond028</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess328"><pre><code>{
  "id": "CritAssess328",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit320",
    "type": "Criterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CanAll080",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll086",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll080",
      "alleleName": "NM_000414.3(HSD17B4):c.56C>G"
    }
  },
  "condition": [
    {
      "id": "GenCond028",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID166",
          "type": "Disease",
          "label": "Perrault Syndrome"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID052",
        "type": "ModeOfInheritance",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn066",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq166",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll080",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll086",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll080",
              "alleleName": "NM_000414.3(HSD17B4):c.56C>G"
            }
          },
          "alleleCount": 3,
          "alleleNumber": 120870,
          "alleleFrequency": 2.4e-05
        },
        {
          "id": "CondPrev167",
          "type": "ConditionPrevelance",
          "condition": [
            {
              "id": "GenCond028",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID166",
                  "type": "Disease",
                  "label": "Perrault Syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "maximum": 1.0e-06,
          "population": {
            "id": "PopID094",
            "type": "Population",
            "label": "Overall"
          }
        },
        {
          "id": "CondPenet168",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond028",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID166",
                  "type": "Disease",
                  "label": "Perrault Syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID131",
            "type": "Penetrance",
            "label": "High"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "8/30/16 6:00 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Perrault syndrome may be underreported due to misdiagnosis.  If the prevalence estimates rise, this assessment may need to be revisited"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess329">CritAssess329</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn067">EvLn067</a></p><p><a href="../details/details.html#EvLn068">EvLn068</a></p>
         </td>
          <td>
              <p>According to Baylor internal data on this variant, it has been observed multiple times in asymptomatic parents internally and in controls and observed once as a homozygotes internally in unaffected.  Although the evidence for full penetrance is weak, the fact that there are multiple asymptomatic occurrences helps the case.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit321">Crit321</a></p> (BS2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll081">CanAll081</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond029">GenCond029</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess329"><pre><code>{
  "id": "CritAssess329",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit321",
    "type": "Criterion",
    "label": "BS2",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    },
    "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
    "comments": "This rule should not be used for diseases with reduced penetrance or late/variable age of  onset (such as certain cardiomyopathies)."
  },
  "variant": {
    "id": "CanAll081",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll087",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll081",
      "alleleName": "NM_001197104.1(KMT2A):c.6572G>A"
    }
  },
  "condition": [
    {
      "id": "GenCond029",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID168",
          "type": "Disease",
          "label": "Wiedemann-Steiner syndrome"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn067",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CondPenet170",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond029",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID168",
                  "type": "Disease",
                  "label": "Wiedemann-Steiner syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID130",
            "type": "Penetrance",
            "label": "Complete"
          },
          "description": "The evidence for full penetrance is not great, due to the rarity of the disease, but the fact that it's diagnosed at a young age and majority of cases are de novo is suggestive"
        },
        {
          "id": "IndGeno172",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv345",
            "type": "Individual",
            "description": "individual #013"
          },
          "genotype": [
            {
              "id": "Geno385",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo384",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll081",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll087",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll081",
                        "alleleName": "NM_001197104.1(KMT2A):c.6572G>A"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID041",
                "type": "AllelicState",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "IndCond174",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv345",
            "type": "Individual",
            "description": "individual #013"
          },
          "condition": [
            {
              "id": "GenCond029",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID168",
                  "type": "Disease",
                  "label": "Wiedemann-Steiner syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "EvLn068",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CondPenet170",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond029",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID168",
                  "type": "Disease",
                  "label": "Wiedemann-Steiner syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID130",
            "type": "Penetrance",
            "label": "Complete"
          },
          "description": "The evidence for full penetrance is not great, due to the rarity of the disease, but the fact that it's diagnosed at a young age and majority of cases are de novo is suggestive"
        },
        {
          "id": "IndGeno173",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv346",
            "type": "Individual",
            "description": "individual #014"
          },
          "genotype": [
            {
              "id": "Geno387",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo386",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll081",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll087",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll081",
                        "alleleName": "NM_001197104.1(KMT2A):c.6572G>A"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID043",
                "type": "AllelicState",
                "label": "homozygous"
              }
            }
          ]
        },
        {
          "id": "IndCond175",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv346",
            "type": "Individual",
            "description": "individual #014"
          },
          "condition": [
            {
              "id": "GenCond029",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID168",
                  "type": "Disease",
                  "label": "Wiedemann-Steiner syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/5/16 6:00 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "According to Baylor internal data on this variant, it has been observed multiple times in asymptomatic parents internally and in controls and observed once as a homozygotes internally in unaffected.  Although the evidence for full penetrance is weak, the fact that there are multiple asymptomatic occurrences helps the case."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess330">CritAssess330</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn069">EvLn069</a></p><p><a href="../details/details.html#EvLn070">EvLn070</a></p><p><a href="../details/details.html#EvLn071">EvLn071</a></p><p><a href="../details/details.html#EvLn072">EvLn072</a></p>
         </td>
          <td>
              <p>This variant was observed in unaffected healthy control chromosomes. PMIDs: 15122587, 18199528, 22703879, 24728327</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit321">Crit321</a></p> (BS2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll082">CanAll082</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond030">GenCond030</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess330"><pre><code>{
  "id": "CritAssess330",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit321",
    "type": "Criterion",
    "label": "BS2",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    },
    "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
    "comments": "This rule should not be used for diseases with reduced penetrance or late/variable age of  onset (such as certain cardiomyopathies)."
  },
  "variant": {
    "id": "CanAll082",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll088",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll082",
      "alleleName": "NM_000038.5(APC ):c.3386T>C"
    }
  },
  "condition": [
    {
      "id": "GenCond030",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID178",
          "type": "Disease",
          "label": "Familial Adenomatous Polyposis"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn069",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CondPenet176",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID178",
                  "type": "Disease",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID130",
            "type": "Penetrance",
            "label": "Complete"
          },
          "description": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early."
        },
        {
          "id": "IndGeno178",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv347",
            "type": "Individual",
            "description": "individual #015"
          },
          "genotype": [
            {
              "id": "Geno389",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo388",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll082",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll088",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll082",
                        "alleleName": "NM_000038.5(APC ):c.3386T>C"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID041",
                "type": "AllelicState",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "IndCond182",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv347",
            "type": "Individual",
            "description": "individual #015"
          },
          "condition": [
            {
              "id": "GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID178",
                  "type": "Disease",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "EvLn070",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CondPenet176",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID178",
                  "type": "Disease",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID130",
            "type": "Penetrance",
            "label": "Complete"
          },
          "description": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early."
        },
        {
          "id": "IndGeno179",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv348",
            "type": "Individual",
            "description": "individual #016"
          },
          "genotype": [
            {
              "id": "Geno391",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo390",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll082",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll088",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll082",
                        "alleleName": "NM_000038.5(APC ):c.3386T>C"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID041",
                "type": "AllelicState",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "IndCond183",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv348",
            "type": "Individual",
            "description": "individual #016"
          },
          "condition": [
            {
              "id": "GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID178",
                  "type": "Disease",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "EvLn071",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CondPenet176",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID178",
                  "type": "Disease",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID130",
            "type": "Penetrance",
            "label": "Complete"
          },
          "description": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early."
        },
        {
          "id": "IndGeno180",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv349",
            "type": "Individual",
            "description": "individual #017"
          },
          "genotype": [
            {
              "id": "Geno393",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo392",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll082",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll088",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll082",
                        "alleleName": "NM_000038.5(APC ):c.3386T>C"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID041",
                "type": "AllelicState",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "IndCond184",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv349",
            "type": "Individual",
            "description": "individual #017"
          },
          "condition": [
            {
              "id": "GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID178",
                  "type": "Disease",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "EvLn072",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CondPenet176",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID178",
                  "type": "Disease",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID130",
            "type": "Penetrance",
            "label": "Complete"
          },
          "description": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early."
        },
        {
          "id": "IndGeno181",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv350",
            "type": "Individual",
            "description": "individual #018"
          },
          "genotype": [
            {
              "id": "Geno395",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo394",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll082",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll088",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll082",
                        "alleleName": "NM_000038.5(APC ):c.3386T>C"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID041",
                "type": "AllelicState",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "IndCond185",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv350",
            "type": "Individual",
            "description": "individual #018"
          },
          "condition": [
            {
              "id": "GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID178",
                  "type": "Disease",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/5/16 6:30 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "9/5/16 6:30 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This variant was observed in unaffected healthy control chromosomes. PMIDs: 15122587, 18199528, 22703879, 24728327"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess331">CritAssess331</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn073">EvLn073</a></p><p><a href="../details/details.html#EvLn074">EvLn074</a></p>
         </td>
          <td>
              <p>Although this variant was found in 5 individuals with Marfan Syndrome, it was identified in 2 unaffected relatives from two separate families. PMID: 12402346</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit321">Crit321</a></p> (BS2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll083">CanAll083</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond072">GenCond072</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess331"><pre><code>{
  "id": "CritAssess331",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit321",
    "type": "Criterion",
    "label": "BS2",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    },
    "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
    "comments": "This rule should not be used for diseases with reduced penetrance or late/variable age of  onset (such as certain cardiomyopathies)."
  },
  "variant": {
    "id": "CanAll083",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll089",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll083",
      "alleleName": "NM_000138.4(FBN1 ):c.2956G>A"
    }
  },
  "condition": [
    {
      "id": "GenCond072",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID173",
          "type": "Disease",
          "label": "Marfan Syndrome"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn073",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CondPenet186",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond072",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID173",
                  "type": "Disease",
                  "label": "Marfan Syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID130",
            "type": "Penetrance",
            "label": "Complete"
          },
          "description": "Marfan syndrome involves a range of skeletal features that should be identifiable at birth, if not during adolescence. The ocular and cardiovascular aspects may manifest later. 1/4 of mutations are de novo."
        },
        {
          "id": "IndGeno188",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv351",
            "type": "Individual",
            "description": "individual #019"
          },
          "genotype": [
            {
              "id": "Geno397",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo396",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll083",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll089",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll083",
                        "alleleName": "NM_000138.4(FBN1 ):c.2956G>A"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID041",
                "type": "AllelicState",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "IndCond190",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv351",
            "type": "Individual",
            "description": "individual #019"
          },
          "condition": [
            {
              "id": "GenCond072",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID173",
                  "type": "Disease",
                  "label": "Marfan Syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "EvLn074",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CondPenet186",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond072",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID173",
                  "type": "Disease",
                  "label": "Marfan Syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID130",
            "type": "Penetrance",
            "label": "Complete"
          },
          "description": "Marfan syndrome involves a range of skeletal features that should be identifiable at birth, if not during adolescence. The ocular and cardiovascular aspects may manifest later. 1/4 of mutations are de novo."
        },
        {
          "id": "IndGeno189",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv352",
            "type": "Individual",
            "description": "individual #020"
          },
          "genotype": [
            {
              "id": "Geno399",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo398",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll083",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll089",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll083",
                        "alleleName": "NM_000138.4(FBN1 ):c.2956G>A"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID041",
                "type": "AllelicState",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "IndCond191",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv352",
            "type": "Individual",
            "description": "individual #020"
          },
          "condition": [
            {
              "id": "GenCond072",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID173",
                  "type": "Disease",
                  "label": "Marfan Syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/5/16 7:01 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    },
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "9/5/16 7:01 AM",
      "contributionRole": {
        "id": "ContribRoleID196",
        "type": "ContributoryRole",
        "label": "interpreter role"
      }
    }
  ],
  "description": "Although this variant was found in 5 individuals with Marfan Syndrome, it was identified in 2 unaffected relatives from two separate families. PMID: 12402346"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess333">CritAssess333</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn075">EvLn075</a></p><p><a href="../details/details.html#EvLn076">EvLn076</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit322">Crit322</a></p> (BS3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll085">CanAll085</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess333"><pre><code>{
  "id": "CritAssess333",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit322",
    "type": "Criterion",
    "label": "BS3",
    "description": "Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    },
    "comments": "\nThis will all depend on what is “established”, which could end up being defined by the different clinical domain working groups.\n"
  },
  "variant": {
    "id": "CanAll085",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll091",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll085",
      "alleleName": "NM_000441.1(SLC26A4): c.349C>T (p.Leu117Phe)",
      "produces": [
        {
          "id": "CtxAll092",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll086",
          "alleleName": "NP_000432..1:p.Leu117Phe",
          "producedBy": "CtxAll091"
        }
      ]
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn075",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func192",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "Unchanged from reference",
          "assayType": {
            "id": "AFAMID081",
            "type": "AlleleFunctionalAssayMethod",
            "label": "iodide efflux"
          },
          "contextualAllele": {
            "id": "CtxAll092",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll086",
            "alleleName": "NP_000432..1:p.Leu117Phe",
            "producedBy": "CtxAll091"
          },
          "gene": {
            "id": "G242",
            "type": "Gene",
            "geneSymbol": "SLC26A4"
          }
        }
      ]
    },
    {
      "id": "EvLn076",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func193",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "localized to apical membrane",
          "assayType": {
            "id": "AFAMID079",
            "type": "AlleleFunctionalAssayMethod",
            "label": "cellular localization"
          },
          "contextualAllele": {
            "id": "CtxAll092",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll086",
            "alleleName": "NP_000432..1:p.Leu117Phe",
            "producedBy": "CtxAll091"
          },
          "gene": {
            "id": "G242",
            "type": "Gene",
            "geneSymbol": "SLC26A4"
          },
          "description": "Both wild type and NP_000432..1:p.Leu117Phe localize to the same location"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/5/16 9:01 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess334">CritAssess334</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn077">EvLn077</a></p><p><a href="../details/details.html#EvLn078">EvLn078</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit322">Crit322</a></p> (BS3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll087">CanAll087</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess334"><pre><code>{
  "id": "CritAssess334",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit322",
    "type": "Criterion",
    "label": "BS3",
    "description": "Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    },
    "comments": "\nThis will all depend on what is “established”, which could end up being defined by the different clinical domain working groups.\n"
  },
  "variant": {
    "id": "CanAll087",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll093",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll087",
      "alleleName": "NM_152594.2(SPRED1): c.587C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn077",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func194",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "Unchanged from reference",
          "assayType": {
            "id": "AFAMID080",
            "type": "AlleleFunctionalAssayMethod",
            "label": "Elk1 pohsphorylation via luciferase assay"
          },
          "contextualAllele": {
            "id": "CtxAll093",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll087",
            "alleleName": "NM_152594.2(SPRED1): c.587C>T"
          },
          "gene": {
            "id": "G271",
            "type": "Gene",
            "geneSymbol": "SPRED1"
          }
        }
      ]
    },
    {
      "id": "EvLn078",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func195",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "localized to apical membrane",
          "assayType": {
            "id": "AFAMID080",
            "type": "AlleleFunctionalAssayMethod",
            "label": "Elk1 pohsphorylation via luciferase assay"
          },
          "contextualAllele": {
            "id": "CtxAll093",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll087",
            "alleleName": "NM_152594.2(SPRED1): c.587C>T"
          },
          "gene": {
            "id": "G271",
            "type": "Gene",
            "geneSymbol": "SPRED1"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/5/16 9:31 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess335">CritAssess335</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn079">EvLn079</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit323">Crit323</a></p> (BS4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll088">CanAll088</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond035">GenCond035</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess335"><pre><code>{
  "id": "CritAssess335",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit323",
    "type": "Criterion",
    "label": "BS4",
    "description": "Lack of segregation in affected members of a family",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    },
    "comments": "This rule is for “lack of segregation” meaning phenotype positive / genotype negative (and not pheno neg / geno pos). Also need to be careful of phenocopies. Also need rules for how many cases are needed in use this rule"
  },
  "variant": {
    "id": "CanAll088",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll094",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll088",
      "alleleName": "NM_000527.4(LDLR):c.313+2T>C"
    }
  },
  "condition": [
    {
      "id": "GenCond035",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID150",
          "type": "Disease",
          "label": "Hypercholesterolemia, Familial"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn079",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "FamSeg196",
          "type": "FamilySegregation",
          "condition": {
            "id": "GenCond035",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID150",
                "type": "Disease",
                "label": "Hypercholesterolemia, Familial"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CanAll088",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll094",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll088",
              "alleleName": "NM_000527.4(LDLR):c.313+2T>C"
            }
          },
          "family": {
            "id": "Fam370",
            "type": "Family",
            "description": "Family #008"
          },
          "phenotypePositiveAllelePositive": 0,
          "phenotypePositiveAlleleNegative": 2,
          "inconsistentSegregationsObserved": {
            "id": "InconSegObsID220",
            "type": "InconsistentSegregationObserved",
            "label": "present"
          },
          "description": "observed in one case diagnosed with Familial Hypercholesterolemia, characterized by elevated LDLC levels. Two siblings of the proband also have elevated LDLC levels but neither carry the 313+2T>C variant."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/5/16 9:51 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess336">CritAssess336</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn080">EvLn080</a></p>
         </td>
          <td>
              <p>The inconsistent segregation data is reasonably explainable as a phenocopy, and therefore does not provide evidence for the benignity of this allele.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit323">Crit323</a></p> (BS4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll089">CanAll089</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond065">GenCond065</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess336"><pre><code>{
  "id": "CritAssess336",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit323",
    "type": "Criterion",
    "label": "BS4",
    "description": "Lack of segregation in affected members of a family",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    },
    "comments": "This rule is for “lack of segregation” meaning phenotype positive / genotype negative (and not pheno neg / geno pos). Also need to be careful of phenocopies. Also need rules for how many cases are needed in use this rule"
  },
  "variant": {
    "id": "CanAll089",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll095",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll089",
      "alleleName": "NM_000256.3(MYBPC3):c.1624+4A>T"
    }
  },
  "condition": [
    {
      "id": "GenCond065",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID160",
          "type": "Disease",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn080",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "FamSeg197",
          "type": "FamilySegregation",
          "condition": {
            "id": "GenCond065",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID160",
                "type": "Disease",
                "label": "hypertrophic cardiomyopathy"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CanAll089",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll095",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll089",
              "alleleName": "NM_000256.3(MYBPC3):c.1624+4A>T"
            }
          },
          "family": {
            "id": "Fam371",
            "type": "Family",
            "description": "Family #009"
          },
          "phenotypePositiveAllelePositive": 3,
          "phenotypePositiveAlleleNegative": 1,
          "inconsistentSegregationsObserved": {
            "id": "InconSegObsID220",
            "type": "InconsistentSegregationObserved",
            "label": "present"
          },
          "description": "The individual with HCM but without the allele is likely a phenocopy explained\nby environmental factors (obese, alcohol)"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/5/16 10:51 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "The inconsistent segregation data is reasonably explainable as a phenocopy, and therefore does not provide evidence for the benignity of this allele."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess337">CritAssess337</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn104">EvLn104</a></p>
         </td>
          <td>
              <p>In a family tested by the LMM, the c.1484C>T variant was identified in a proband with clinical diagnosis of HCM. This proband has two relatives that also have clinical diagnosis of HCM however never relative carries the c.1484C>T variant.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit323">Crit323</a></p> (BS4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll090">CanAll090</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond066">GenCond066</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess337"><pre><code>{
  "id": "CritAssess337",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit323",
    "type": "Criterion",
    "label": "BS4",
    "description": "Lack of segregation in affected members of a family",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    },
    "comments": "This rule is for “lack of segregation” meaning phenotype positive / genotype negative (and not pheno neg / geno pos). Also need to be careful of phenocopies. Also need rules for how many cases are needed in use this rule"
  },
  "variant": {
    "id": "CanAll090",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll096",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll090",
      "alleleName": "NM_001103.3(ACTN2):c.1484C>T (p.Thr495Met)"
    }
  },
  "condition": [
    {
      "id": "GenCond066",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID160",
          "type": "Disease",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn104",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "FamSeg271",
          "type": "FamilySegregation",
          "condition": {
            "id": "GenCond066",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID160",
                "type": "Disease",
                "label": "hypertrophic cardiomyopathy"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CanAll090",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll096",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll090",
              "alleleName": "NM_001103.3(ACTN2):c.1484C>T (p.Thr495Met)"
            }
          },
          "family": {
            "id": "Fam372",
            "type": "Family",
            "description": "Family #010"
          },
          "phenotypePositiveAllelePositive": 0,
          "phenotypePositiveAlleleNegative": 2,
          "inconsistentSegregationsObserved": {
            "id": "InconSegObsID220",
            "type": "InconsistentSegregationObserved",
            "label": "present"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/5/16 11:00 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "In a family tested by the LMM, the c.1484C>T variant was identified in a proband with clinical diagnosis of HCM. This proband has two relatives that also have clinical diagnosis of HCM however never relative carries the c.1484C>T variant."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess338">CritAssess338</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn081">EvLn081</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit324">Crit324</a></p> (BP1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll091">CanAll091</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond036">GenCond036</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess338"><pre><code>{
  "id": "CritAssess338",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit324",
    "type": "Criterion",
    "label": "BP1",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    },
    "comments": "Is there any guidance on the proportion of pathogenic variants that are LOF to apply this rule?  99%? 90%? 75%?"
  },
  "variant": {
    "id": "CanAll091",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll097",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll091",
      "alleleName": "NM_000059.3(BRCA2) :c.4779A>C"
    }
  },
  "condition": [
    {
      "id": "GenCond036",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID157",
          "type": "Disease",
          "label": "hereditary breast and ovarian cancer syndrome"
        }
      ],
      "name": "hereditary breast and ovarian cancer"
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn081",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon198",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll097",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll091",
            "alleleName": "NM_000059.3(BRCA2) :c.4779A>C"
          },
          "consequence": {
            "id": "MolConID011",
            "type": "MolecularConsequence",
            "label": "missense variant"
          }
        },
        {
          "id": "Null404",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll070",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll066",
            "alleleName": "NM_000257.3(MYH7):c.5401G>A"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          },
          "qualifier": "NOT"
        },
        {
          "id": "CondMech199",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MechID071",
            "type": "Mechanism",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID135",
            "type": "ConditionMechanismStrength",
            "label": "primary"
          },
          "gene": {
            "id": "G228",
            "type": "Gene",
            "geneSymbol": "BRCA2"
          },
          "condition": {
            "id": "GenCond036",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID157",
                "type": "Disease",
                "label": "hereditary breast and ovarian cancer syndrome"
              }
            ],
            "name": "hereditary breast and ovarian cancer"
          },
          "description": "ClinVar shows 421 variants in BRCA2 that are pathogenic for HBOC. 402 of these (~95%) are frameshift, nonsense, or splice-site variants."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "9/6/16 8:00 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess339">CritAssess339</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn082">EvLn082</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit324">Crit324</a></p> (BP1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll091">CanAll091</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond037">GenCond037</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess339"><pre><code>{
  "id": "CritAssess339",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit324",
    "type": "Criterion",
    "label": "BP1",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    },
    "comments": "Is there any guidance on the proportion of pathogenic variants that are LOF to apply this rule?  99%? 90%? 75%?"
  },
  "variant": {
    "id": "CanAll091",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll097",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll091",
      "alleleName": "NM_000059.3(BRCA2) :c.4779A>C"
    }
  },
  "condition": [
    {
      "id": "GenCond037",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID182",
          "type": "Disease",
          "label": "Fanconi Anemia"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn082",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon198",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll097",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll091",
            "alleleName": "NM_000059.3(BRCA2) :c.4779A>C"
          },
          "consequence": {
            "id": "MolConID011",
            "type": "MolecularConsequence",
            "label": "missense variant"
          }
        },
        {
          "id": "Null405",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll099",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll093",
            "alleleName": "NM_133378.4(TTN): c.2926T>C"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          },
          "qualifier": "NOT"
        },
        {
          "id": "CondMech200",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MechID071",
            "type": "Mechanism",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID135",
            "type": "ConditionMechanismStrength",
            "label": "primary"
          },
          "gene": {
            "id": "G228",
            "type": "Gene",
            "geneSymbol": "BRCA2"
          },
          "condition": {
            "id": "GenCond037",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID182",
                "type": "Disease",
                "label": "Fanconi Anemia"
              }
            ]
          },
          "description": "ClinVar shows 61 variants in BRCA2 that are pathogenic for Fanconi Anemia. 55 of these (~90%) are frameshift, nonsense, or splice-site variants."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "9/6/16 9:00 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess340">CritAssess340</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn083">EvLn083</a></p>
         </td>
          <td>
              <p>Homozygous truncating variants in LOXHD1 are associated with autosomal recessive hearing loss. This was demonstrated by both identification in human patients and a mouse model. (I used the OMIM entry as a source, but you could also directly use the papers.)</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit324">Crit324</a></p> (BP1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll092">CanAll092</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond059">GenCond059</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess340"><pre><code>{
  "id": "CritAssess340",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit324",
    "type": "Criterion",
    "label": "BP1",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    },
    "comments": "Is there any guidance on the proportion of pathogenic variants that are LOF to apply this rule?  99%? 90%? 75%?"
  },
  "variant": {
    "id": "CanAll092",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll098",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll092",
      "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
    }
  },
  "condition": [
    {
      "id": "GenCond059",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID183",
          "type": "Disease",
          "label": "nonsyndromic deafness"
        }
      ],
      "name": "Hearing Loss, Autosomal Recessive",
      "modeOfInheritance": {
        "id": "ModeOfInhID052",
        "type": "ModeOfInheritance",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn083",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon201",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll098",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll092",
            "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
          },
          "consequence": {
            "id": "MolConID011",
            "type": "MolecularConsequence",
            "label": "missense variant"
          }
        },
        {
          "id": "Null406",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll097",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll091",
            "alleleName": "NM_000059.3(BRCA2) :c.4779A>C"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          }
        },
        {
          "id": "CondMech202",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MechID071",
            "type": "Mechanism",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID135",
            "type": "ConditionMechanismStrength",
            "label": "primary"
          },
          "gene": {
            "id": "G274",
            "type": "Gene",
            "geneSymbol": "LOXHD1"
          },
          "condition": {
            "id": "GenCond059",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID183",
                "type": "Disease",
                "label": "nonsyndromic deafness"
              }
            ],
            "name": "Hearing Loss, Autosomal Recessive",
            "modeOfInheritance": {
              "id": "ModeOfInhID052",
              "type": "ModeOfInheritance",
              "label": "Autosomal Recessive"
            }
          },
          "description": "shown in both humans and mouse models"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 10:00 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Homozygous truncating variants in LOXHD1 are associated with autosomal recessive hearing loss. This was demonstrated by both identification in human patients and a mouse model. (I used the OMIM entry as a source, but you could also directly use the papers.)"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess341">CritAssess341</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn084">EvLn084</a></p>
         </td>
          <td>
              <p>While truncations are usually the cause of TTN related DCM, missense variants in certain regions can also cause disease.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit324">Crit324</a></p> (BP1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll093">CanAll093</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond053">GenCond053</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess341"><pre><code>{
  "id": "CritAssess341",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit324",
    "type": "Criterion",
    "label": "BP1",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    },
    "comments": "Is there any guidance on the proportion of pathogenic variants that are LOF to apply this rule?  99%? 90%? 75%?"
  },
  "variant": {
    "id": "CanAll093",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll099",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll093",
      "alleleName": "NM_133378.4(TTN): c.2926T>C"
    }
  },
  "condition": [
    {
      "id": "GenCond053",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID162",
          "type": "Disease",
          "label": "dilated cardiomyopathy"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn084",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon203",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll099",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll093",
            "alleleName": "NM_133378.4(TTN): c.2926T>C"
          },
          "consequence": {
            "id": "MolConID011",
            "type": "MolecularConsequence",
            "label": "missense variant"
          }
        },
        {
          "id": "Null407",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll098",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll092",
            "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          },
          "qualifier": "NOT"
        },
        {
          "id": "CondMech204",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MechID071",
            "type": "Mechanism",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID135",
            "type": "ConditionMechanismStrength",
            "label": "primary"
          },
          "gene": {
            "id": "G229",
            "type": "Gene",
            "geneSymbol": "TTN"
          },
          "condition": {
            "id": "GenCond053",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID162",
                "type": "Disease",
                "label": "dilated cardiomyopathy"
              }
            ]
          }
        },
        {
          "id": "CondMech205",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MolConID011",
            "type": "MolecularConsequence",
            "label": "missense variant"
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID134",
            "type": "ConditionMechanismStrength",
            "label": "established"
          },
          "gene": {
            "id": "G229",
            "type": "Gene",
            "geneSymbol": "TTN"
          },
          "condition": {
            "id": "GenCond053",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID162",
                "type": "Disease",
                "label": "dilated cardiomyopathy"
              }
            ]
          },
          "description": "Truncating variants in TTN are associated with DCM. However, if missense variants are located in the A-band, impact the A-band, or are in a highly expressed region, they can also be associated with DCM."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 11:00 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "While truncations are usually the cause of TTN related DCM, missense variants in certain regions can also cause disease."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess342">CritAssess342</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn085">EvLn085</a></p><p><a href="../details/details.html#EvLn086">EvLn086</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit325">Crit325</a></p> (BP2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll094">CanAll094</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond038">GenCond038</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess342"><pre><code>{
  "id": "CritAssess342",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit325",
    "type": "Criterion",
    "label": "BP2",
    "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll094",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll100",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll094",
      "alleleName": "NM_206933.2 (USH2A): c.6721C>T (p.Pro2241Ser)"
    }
  },
  "condition": [
    {
      "id": "GenCond038",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID181",
          "type": "Disease",
          "label": "Retinitis Pigmentosa"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn085",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndGeno206",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv353",
            "type": "Individual",
            "description": "individual #021"
          },
          "genotype": [
            {
              "id": "Geno401",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo400",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll094",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll100",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll094",
                        "alleleName": "NM_206933.2 (USH2A): c.6721C>T (p.Pro2241Ser)"
                      }
                    },
                    {
                      "id": "CanAll095",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll101",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll095",
                        "alleleName": "NM_206933.2(USH2A):c.2276G>T"
                      }
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "IndCond208",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv353",
            "type": "Individual",
            "description": "individual #021"
          },
          "condition": [
            {
              "id": "GenCond038",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID181",
                  "type": "Disease",
                  "label": "Retinitis Pigmentosa"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "VarInterp210",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll095",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll101",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll095",
              "alleleName": "NM_206933.2(USH2A):c.2276G>T"
            }
          },
          "condition": [
            {
              "id": "GenCond038",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID181",
                  "type": "Disease",
                  "label": "Retinitis Pigmentosa"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        }
      ]
    },
    {
      "id": "EvLn086",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndGeno207",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv354",
            "type": "Individual",
            "description": "individual #022"
          },
          "genotype": [
            {
              "id": "Geno403",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo402",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll094",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll100",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll094",
                        "alleleName": "NM_206933.2 (USH2A): c.6721C>T (p.Pro2241Ser)"
                      }
                    },
                    {
                      "id": "CanAll095",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll101",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll095",
                        "alleleName": "NM_206933.2(USH2A):c.2276G>T"
                      }
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "IndCond209",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv354",
            "type": "Individual",
            "description": "individual #022"
          },
          "condition": [
            {
              "id": "GenCond038",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID181",
                  "type": "Disease",
                  "label": "Retinitis Pigmentosa"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "VarInterp210",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll095",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll101",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll095",
              "alleleName": "NM_206933.2(USH2A):c.2276G>T"
            }
          },
          "condition": [
            {
              "id": "GenCond038",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID181",
                  "type": "Disease",
                  "label": "Retinitis Pigmentosa"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 12:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess343">CritAssess343</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn087">EvLn087</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit325">Crit325</a></p> (BP2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll096">CanAll096</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond039">GenCond039</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess343"><pre><code>{
  "id": "CritAssess343",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit325",
    "type": "Criterion",
    "label": "BP2",
    "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll096",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll102",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll096",
      "alleleName": "NM_000492.3(CFTR): c.2002C>T"
    }
  },
  "condition": [
    {
      "id": "GenCond039",
      "type": "GeneticCondition",
      "phenotype": [
        {
          "id": "PhenID190",
          "type": "Phenotype",
          "label": "Absent vas deferens"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn087",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndGeno211",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv355",
            "type": "Individual",
            "description": "individual #023"
          },
          "genotype": [
            {
              "id": "Geno405",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo404",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll096",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll102",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll096",
                        "alleleName": "NM_000492.3(CFTR): c.2002C>T"
                      }
                    },
                    {
                      "id": "CanAll097",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll103",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll097",
                        "alleleName": "NP_000483.3:p.Gly576Ala"
                      }
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "IndCond212",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv355",
            "type": "Individual",
            "description": "individual #023"
          },
          "condition": [
            {
              "id": "GenCond039",
              "type": "GeneticCondition",
              "phenotype": [
                {
                  "id": "PhenID190",
                  "type": "Phenotype",
                  "label": "Absent vas deferens"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "VarInterp213",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll097",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll103",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll097",
              "alleleName": "NP_000483.3:p.Gly576Ala"
            }
          },
          "condition": [
            {
              "id": "GenCond039",
              "type": "GeneticCondition",
              "phenotype": [
                {
                  "id": "PhenID190",
                  "type": "Phenotype",
                  "label": "Absent vas deferens"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 1:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess344">CritAssess344</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn088">EvLn088</a></p>
         </td>
          <td>
              <p>although this variant was found in trans with another variant in the same gene, that variant is VUS and not an established pathogenic variant and cardiomyopathies can be variable in penetrance. </p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit325">Crit325</a></p> (BP2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll098">CanAll098</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond007">GenCond007</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess344"><pre><code>{
  "id": "CritAssess344",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit325",
    "type": "Criterion",
    "label": "BP2",
    "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll098",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll104",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll098",
      "alleleName": "NM_000257.2(MYH7): c.2559G>A"
    }
  },
  "condition": [
    {
      "id": "GenCond007",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID158",
          "type": "Disease",
          "label": "cardiomyopathy"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn088",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndGeno214",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv356",
            "type": "Individual",
            "description": "individual #024"
          },
          "genotype": [
            {
              "id": "Geno408",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo406",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll098",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll104",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll098",
                        "alleleName": "NM_000257.2(MYH7): c.2559G>A"
                      }
                    }
                  ]
                },
                {
                  "id": "Haplo407",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll099",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll105",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll099",
                        "alleleName": "NM_000257.2(MYH7): c.2537A>G"
                      }
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "IndCond215",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv356",
            "type": "Individual",
            "description": "individual #024"
          },
          "condition": [
            {
              "id": "GenCond007",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID158",
                  "type": "Disease",
                  "label": "cardiomyopathy"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "VarInterp216",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll099",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll105",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll099",
              "alleleName": "NM_000257.2(MYH7): c.2537A>G"
            }
          },
          "condition": [
            {
              "id": "GenCond007",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID158",
                  "type": "Disease",
                  "label": "cardiomyopathy"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "outcome": {
            "id": "SigID029",
            "type": "Significance",
            "label": "Uncertain Signficance"
          }
        },
        {
          "id": "CondPenet218",
          "type": "ConditionPenetrance",
          "condition": [
            {
              "id": "GenCond007",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID158",
                  "type": "Disease",
                  "label": "cardiomyopathy"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID051",
                "type": "ModeOfInheritance",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "penetrance": {
            "id": "PenetID132",
            "type": "Penetrance",
            "label": "Incomplete"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 2:00 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "although this variant was found in trans with another variant in the same gene, that variant is VUS and not an established pathogenic variant and cardiomyopathies can be variable in penetrance. "
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess345">CritAssess345</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn089">EvLn089</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit326">Crit326</a></p> (BP3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll100">CanAll100</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess345"><pre><code>{
  "id": "CritAssess345",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit326",
    "type": "Criterion",
    "label": "BP3",
    "description": "In-frame deletions/insertions in a repetitive region without a known function",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll100",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll106",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll100",
      "alleleName": "NM_001271208.1(NEB):c.169_183delCTGGCACAGCCAGCA"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn089",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "RegTyp220",
          "type": "RegionType",
          "region": {
            "id": "Loc390",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq294",
              "type": "ReferenceSequence",
              "accession": "NM_001271208.1"
            },
            "start": 154,
            "stop": 183
          },
          "annotation": [
            {
              "id": "RegAnnID033",
              "type": "RegionAnnotation",
              "label": "functional region"
            }
          ],
          "qualifier": "NOT",
          "description": "region does not have a known function."
        },
        {
          "id": "RegTyp221",
          "type": "RegionType",
          "region": {
            "id": "Loc390",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq294",
              "type": "ReferenceSequence",
              "accession": "NM_001271208.1"
            },
            "start": 154,
            "stop": 183
          },
          "annotation": [
            {
              "id": "RegAnnID035",
              "type": "RegionAnnotation",
              "label": "repeat region "
            }
          ],
          "description": "region is repeating."
        },
        {
          "id": "RegAll391",
          "type": "RegionAlleles",
          "region": {
            "id": "Loc390",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq294",
              "type": "ReferenceSequence",
              "accession": "NM_001271208.1"
            },
            "start": 154,
            "stop": 183
          },
          "allele": [
            {
              "id": "CtxAll106",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll100",
              "alleleName": "NM_001271208.1(NEB):c.169_183delCTGGCACAGCCAGCA"
            }
          ],
          "outcome": {
            "id": "RegAllOutID225",
            "type": "RegionAllelesOutcome",
            "label": "overlaps region"
          },
          "description": "region is repeating but does not have a known function."
        },
        {
          "id": "MolCon223",
          "type": "AlleleMolecularConsequence",
          "consequence": {
            "id": "MolConID019",
            "type": "MolecularConsequence",
            "label": "conservative Inframe Deletion"
          },
          "contextualAllele": {
            "id": "CtxAll106",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll100",
            "alleleName": "NM_001271208.1(NEB):c.169_183delCTGGCACAGCCAGCA"
          }
        },
        {
          "id": "Null408",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll099",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll093",
            "alleleName": "NM_133378.4(TTN): c.2926T>C"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          },
          "qualifier": "NOT"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/15/16 8:12 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess346">CritAssess346</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn090">EvLn090</a></p>
         </td>
          <td>
              <p>BP3 does not apply for this variant because it is not an in-frame deletion (it's a deletion of 10 bp) and, although the I Band is a repetitive region that contains tandem arrays of immunoglobulin domains, these folds may be an important component for TTN's elasticity.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit326">Crit326</a></p> (BP3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll101">CanAll101</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess346"><pre><code>{
  "id": "CritAssess346",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit326",
    "type": "Criterion",
    "label": "BP3",
    "description": "In-frame deletions/insertions in a repetitive region without a known function",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll101",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll107",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll101",
      "alleleName": "NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC"
    }
  },
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn090",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "RegTyp225",
          "type": "RegionType",
          "region": {
            "id": "Loc392",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq295",
              "type": "ReferenceSequence",
              "accession": "NM_133378.4"
            },
            "start": 6509,
            "stop": 39262
          },
          "annotation": [
            {
              "id": "RegAnnID035",
              "type": "RegionAnnotation",
              "label": "repeat region "
            }
          ],
          "description": "The I-Band is a large region of repetitive Igg domains. "
        },
        {
          "id": "RegAll393",
          "type": "RegionAlleles",
          "region": {
            "id": "Loc392",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq295",
              "type": "ReferenceSequence",
              "accession": "NM_133378.4"
            },
            "start": 6509,
            "stop": 39262
          },
          "allele": [
            {
              "id": "CtxAll107",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll101",
              "alleleName": "NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC"
            }
          ],
          "outcome": {
            "id": "RegAllOutID225",
            "type": "RegionAllelesOutcome",
            "label": "overlaps region"
          },
          "description": "Allele NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC is contained by this region."
        },
        {
          "id": "MolCon227",
          "type": "AlleleMolecularConsequence",
          "consequence": {
            "id": "MolConID014",
            "type": "MolecularConsequence",
            "label": "frameshift variant"
          },
          "contextualAllele": {
            "id": "CtxAll107",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll101",
            "alleleName": "NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/15/16 9:23 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "BP3 does not apply for this variant because it is not an in-frame deletion (it's a deletion of 10 bp) and, although the I Band is a repetitive region that contains tandem arrays of immunoglobulin domains, these folds may be an important component for TTN's elasticity."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess347">CritAssess347</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn091">EvLn091</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit326">Crit326</a></p> (BP3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll102">CanAll102</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess347"><pre><code>{
  "id": "CritAssess347",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit326",
    "type": "Criterion",
    "label": "BP3",
    "description": "In-frame deletions/insertions in a repetitive region without a known function",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll102",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll108",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll102",
      "alleleName": "NM_147196.2(TMIE):c.391_393dupAAG"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn091",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "RegTyp229",
          "type": "RegionType",
          "region": {
            "id": "Loc394",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq296",
              "type": "ReferenceSequence",
              "accession": "NM_147196.2"
            },
            "start": 522,
            "stop": 548
          },
          "annotation": [
            {
              "id": "RegAnnID035",
              "type": "RegionAnnotation",
              "label": "repeat region "
            }
          ]
        },
        {
          "id": "RegAll395",
          "type": "RegionAlleles",
          "region": {
            "id": "Loc394",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "RefSeq296",
              "type": "ReferenceSequence",
              "accession": "NM_147196.2"
            },
            "start": 522,
            "stop": 548
          },
          "allele": [
            {
              "id": "CtxAll108",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll102",
              "alleleName": "NM_147196.2(TMIE):c.391_393dupAAG"
            }
          ],
          "outcome": {
            "id": "RegAllOutID225",
            "type": "RegionAllelesOutcome",
            "label": "overlaps region"
          }
        },
        {
          "id": "MolCon231",
          "type": "AlleleMolecularConsequence",
          "consequence": {
            "id": "MolConID017",
            "type": "MolecularConsequence",
            "label": "conservative Inframe Insertion"
          },
          "contextualAllele": {
            "id": "CtxAll108",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll102",
            "alleleName": "NM_147196.2(TMIE):c.391_393dupAAG"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/15/16 10:40 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess348">CritAssess348</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn092">EvLn092</a></p>
         </td>
          <td>
              <p>Seven of the nine predictions found this variant to be benign, which we consider sufficient to support.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit327">Crit327</a></p> (BP4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll103">CanAll103</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess348"><pre><code>{
  "id": "CritAssess348",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit327",
    "type": "Criterion",
    "label": "BP4",
    "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    },
    "comments": "same issues as PP3."
  },
  "variant": {
    "id": "CanAll103",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll109",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll103",
      "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn092",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "InSilicoPred232",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq300",
            "type": "ReferenceSequence",
            "accession": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CanAll103",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll109",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll103",
              "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "SIFT",
          "prediction": "Benign"
        },
        {
          "id": "InSilicoPred233",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq300",
            "type": "ReferenceSequence",
            "accession": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CanAll103",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll109",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll103",
              "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Polyphen",
          "prediction": "Possibly damaging"
        },
        {
          "id": "InSilicoPred234",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq300",
            "type": "ReferenceSequence",
            "accession": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CanAll103",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll109",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll103",
              "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "FATHMM",
          "prediction": "Benign"
        },
        {
          "id": "InSilicoPred235",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq300",
            "type": "ReferenceSequence",
            "accession": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CanAll103",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll109",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll103",
              "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "LRT",
          "prediction": "Benign"
        },
        {
          "id": "InSilicoPred236",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq300",
            "type": "ReferenceSequence",
            "accession": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CanAll103",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll109",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll103",
              "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "MetaSVM",
          "prediction": "Benign"
        },
        {
          "id": "InSilicoPred237",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq300",
            "type": "ReferenceSequence",
            "accession": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CanAll103",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll109",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll103",
              "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Mutation assessor",
          "prediction": "Benign"
        },
        {
          "id": "InSilicoPred238",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq300",
            "type": "ReferenceSequence",
            "accession": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CanAll103",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll109",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll103",
              "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Mutation taster",
          "prediction": "Benign"
        },
        {
          "id": "InSilicoPred239",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq300",
            "type": "ReferenceSequence",
            "accession": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CanAll103",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll109",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll103",
              "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Provean",
          "prediction": "Benign"
        },
        {
          "id": "InSilicoPred240",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq300",
            "type": "ReferenceSequence",
            "accession": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CanAll103",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll109",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll103",
              "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
            }
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "algorithm": "Condel",
          "prediction": "deleterious"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "10/4/16 8:02 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Seven of the nine predictions found this variant to be benign, which we consider sufficient to support."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess350">CritAssess350</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn105">EvLn105</a></p>
         </td>
          <td>
              <p>While AI111 is considered probably damaging by at least one in silico predictor, the transcript for AI111 ( NM_133379.3 ) is not clinically relevant.  In clinically relevant transcripts, thisthis canonical allele is intronic.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit327">Crit327</a></p> (BP4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll105">CanAll105</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess350"><pre><code>{
  "id": "CritAssess350",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit327",
    "type": "Criterion",
    "label": "BP4",
    "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    },
    "comments": "same issues as PP3."
  },
  "variant": {
    "id": "CanAll105",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll111",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll105",
      "alleleName": "NM_133379.3(TTN) : c.15283T>C"
    }
  },
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn105",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "InSilicoPred362",
          "type": "InSilicoPrediction",
          "canonicalAllele": {
            "id": "CtxAll111",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll105",
            "alleleName": "NM_133379.3(TTN) : c.15283T>C"
          },
          "predictionType": {
            "id": "PredID072",
            "type": "Prediction",
            "label": "missense effect"
          },
          "prediction": "ProbablyDamaging",
          "algorithm": "Polyphen-2 HumDiv"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/4/16 10:37 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "While AI111 is considered probably damaging by at least one in silico predictor, the transcript for AI111 ( NM_133379.3 ) is not clinically relevant.  In clinically relevant transcripts, thisthis canonical allele is intronic."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess351">CritAssess351</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn093">EvLn093</a></p>
         </td>
          <td>
              <p>The evidence here only explicitly describes a single individual with a likely pathogenic variant in TNNT2, but there are numerous other examples that have been observed in internal LMM data.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit328">Crit328</a></p> (BP5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll106">CanAll106</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond067">GenCond067</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess351"><pre><code>{
  "id": "CritAssess351",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit328",
    "type": "Criterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll106",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll112",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll106",
      "alleleName": "NM_000256.3(MYBPC3): c.2497G>A"
    }
  },
  "condition": [
    {
      "id": "GenCond067",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID160",
          "type": "Disease",
          "label": "hypertrophic cardiomyopathy"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn093",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndGeno241",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv357",
            "type": "Individual",
            "description": "individual #025"
          },
          "genotype": [
            {
              "id": "Geno410",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo409",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll106",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll112",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll106",
                        "alleleName": "NM_000256.3(MYBPC3): c.2497G>A"
                      }
                    }
                  ]
                }
              ]
            },
            {
              "id": "Geno412",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo411",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll107",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll113",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll107",
                        "alleleName": "NM_000364.3(TNNT2):c.878G>A"
                      }
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "IndCond242",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv357",
            "type": "Individual",
            "description": "individual #025"
          },
          "condition": [
            {
              "id": "GenCond067",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID160",
                  "type": "Disease",
                  "label": "hypertrophic cardiomyopathy"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "VarInterp243",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll106",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll112",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll106",
              "alleleName": "NM_000256.3(MYBPC3): c.2497G>A"
            }
          },
          "condition": [
            {
              "id": "GenCond067",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID160",
                  "type": "Disease",
                  "label": "hypertrophic cardiomyopathy"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID028",
            "type": "Significance",
            "label": "Likely Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/4/16 11:42 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "The evidence here only explicitly describes a single individual with a likely pathogenic variant in TNNT2, but there are numerous other examples that have been observed in internal LMM data."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess352">CritAssess352</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn094">EvLn094</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit328">Crit328</a></p> (BP5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll108">CanAll108</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond060">GenCond060</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess352"><pre><code>{
  "id": "CritAssess352",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit328",
    "type": "Criterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll108",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll114",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll108",
      "alleleName": "NM_022124.5(CDH23):c.5660C>T"
    }
  },
  "condition": [
    {
      "id": "GenCond060",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID183",
          "type": "Disease",
          "label": "nonsyndromic deafness"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn094",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndGeno244",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv358",
            "type": "Individual",
            "description": "individual #026"
          },
          "genotype": [
            {
              "id": "Geno414",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo413",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll108",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll114",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll108",
                        "alleleName": "NM_022124.5(CDH23):c.5660C>T"
                      }
                    }
                  ]
                }
              ]
            },
            {
              "id": "Geno417",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo415",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll109",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll115",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll109",
                        "alleleName": "NM_004004.5(GJB2):c.71G>A"
                      }
                    }
                  ]
                },
                {
                  "id": "Haplo416",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll110",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll116",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll110",
                        "alleleName": "NM_004004.5(GJB2):c.230G>A"
                      }
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "IndCond246",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv358",
            "type": "Individual",
            "description": "individual #026"
          },
          "condition": [
            {
              "id": "GenCond060",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID183",
                  "type": "Disease",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "VarInterp247",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll109",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll115",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll109",
              "alleleName": "NM_004004.5(GJB2):c.71G>A"
            }
          },
          "condition": [
            {
              "id": "GenCond060",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID183",
                  "type": "Disease",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID028",
            "type": "Significance",
            "label": "Likely Pathogenic"
          }
        },
        {
          "id": "VarInterp248",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll110",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll116",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll110",
              "alleleName": "NM_004004.5(GJB2):c.230G>A"
            }
          },
          "condition": [
            {
              "id": "GenCond060",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID183",
                  "type": "Disease",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID028",
            "type": "Significance",
            "label": "Likely Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/6/16 8:30 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess353">CritAssess353</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn095">EvLn095</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit328">Crit328</a></p> (BP5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll108">CanAll108</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond071">GenCond071</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess353"><pre><code>{
  "id": "CritAssess353",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit328",
    "type": "Criterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll108",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll114",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll108",
      "alleleName": "NM_022124.5(CDH23):c.5660C>T"
    }
  },
  "condition": [
    {
      "id": "GenCond071",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID184",
          "type": "Disease",
          "label": "Usher syndrome"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID052",
        "type": "ModeOfInheritance",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn095",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndGeno249",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv359",
            "type": "Individual",
            "description": "individual #027"
          },
          "genotype": [
            {
              "id": "Geno419",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo418",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll108",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll114",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll108",
                        "alleleName": "NM_022124.5(CDH23):c.5660C>T"
                      }
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "AllStateID043",
                "type": "AllelicState",
                "label": "homozygous"
              }
            },
            {
              "id": "Geno421",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo420",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll111",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll117",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll111",
                        "alleleName": "NM_015404.3(WHRN):c.643delG"
                      }
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "IndCond250",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv359",
            "type": "Individual",
            "description": "individual #027"
          },
          "condition": [
            {
              "id": "GenCond071",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID184",
                  "type": "Disease",
                  "label": "Usher syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "VarInterp251",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll108",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll114",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll108",
              "alleleName": "NM_022124.5(CDH23):c.5660C>T"
            }
          },
          "condition": [
            {
              "id": "GenCond071",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID184",
                  "type": "Disease",
                  "label": "Usher syndrome"
                }
              ],
              "modeOfInheritance": {
                "id": "ModeOfInhID052",
                "type": "ModeOfInheritance",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/6/16 8:54 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess354">CritAssess354</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn096">EvLn096</a></p>
         </td>
          <td>
              <p>Variants in COL2A1 are associated with dominant spondyloepiphyseal dysplasia congenita (a primary bone dysplasia). CA112 was identified in a  proband with CA113, which is pathogenic for another primary bone dysplasia, dominant Pseudoachondroplasia. However, this does not provide supporting evidence for CA112 being benign because the combination of a COL2A1 and COMP variant are known to produce a more severe phenotype, Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome, which is the phenotype of the proband.  Because the combination of alleles may produce a qualitatively different phenotype, the bone dysplasia caused by the COMP variant does not provide evidence that the COL2A1 variant is benign for any bone dysplasias in general.
</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit328">Crit328</a></p> (BP5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll112">CanAll112</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond040">GenCond040</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess354"><pre><code>{
  "id": "CritAssess354",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit328",
    "type": "Criterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll112",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll118",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll112",
      "alleleName": "NM_001844.4(COL2A1):c.4316C>T"
    }
  },
  "condition": [
    {
      "id": "GenCond040",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID170",
          "type": "Disease",
          "label": "Primary bone dysplasia"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn096",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndCond252",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv360",
            "type": "Individual",
            "description": "individual #028"
          },
          "condition": [
            {
              "id": "GenCond041",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID179",
                  "type": "Disease",
                  "label": "pseudoachondroplasia"
                }
              ],
              "name": "Pseudoachondroplastic spondyloepiphyseal dysplasia"
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "IndGeno253",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv360",
            "type": "Individual",
            "description": "individual #028"
          },
          "genotype": [
            {
              "id": "Geno423",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo422",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll112",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll118",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll112",
                        "alleleName": "NM_001844.4(COL2A1):c.4316C>T"
                      }
                    }
                  ]
                }
              ]
            },
            {
              "id": "Geno425",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo424",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll113",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll119",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll113",
                        "alleleName": "NM_000095.2(COMP):c.1042T>C"
                      }
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "VarInterp254",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll113",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll119",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll113",
              "alleleName": "NM_000095.2(COMP):c.1042T>C"
            }
          },
          "condition": [
            {
              "id": "GenCond042",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID179",
                  "type": "Disease",
                  "label": "pseudoachondroplasia"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        },
        {
          "id": "VarInterp270",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll113",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll119",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll113",
              "alleleName": "NM_000095.2(COMP):c.1042T>C"
            }
          },
          "condition": [
            {
              "id": "GenCond041",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID179",
                  "type": "Disease",
                  "label": "pseudoachondroplasia"
                }
              ],
              "name": "Pseudoachondroplastic spondyloepiphyseal dysplasia"
            }
          ],
          "outcome": {
            "id": "SigID029",
            "type": "Significance",
            "label": "Uncertain Signficance"
          },
          "description": "Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome is caused by a combination of a pathogenic variant in COL2A1 and another in COMP.   So CA113 may be involved, but is not the sole cause."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/6/16 9:09 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Variants in COL2A1 are associated with dominant spondyloepiphyseal dysplasia congenita (a primary bone dysplasia). CA112 was identified in a  proband with CA113, which is pathogenic for another primary bone dysplasia, dominant Pseudoachondroplasia. However, this does not provide supporting evidence for CA112 being benign because the combination of a COL2A1 and COMP variant are known to produce a more severe phenotype, Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome, which is the phenotype of the proband.  Because the combination of alleles may produce a qualitatively different phenotype, the bone dysplasia caused by the COMP variant does not provide evidence that the COL2A1 variant is benign for any bone dysplasias in general.\n"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess355">CritAssess355</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn097">EvLn097</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit329">Crit329</a></p> (BP6)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll114">CanAll114</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond073">GenCond073</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess355"><pre><code>{
  "id": "CritAssess355",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit329",
    "type": "Criterion",
    "label": "BP6",
    "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll114",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll120",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll114",
      "alleleName": "NM_007294.3(BRCA1):c.*6207C>T"
    }
  },
  "condition": [
    {
      "id": "GenCond073",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID157",
          "type": "Disease",
          "label": "hereditary breast and ovarian cancer syndrome"
        }
      ],
      "name": "hereditary breast and ovarian cancer"
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn097",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp255",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll114",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll120",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll114",
              "alleleName": "NM_007294.3(BRCA1):c.*6207C>T"
            }
          },
          "condition": [
            {
              "id": "GenCond073",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID157",
                  "type": "Disease",
                  "label": "hereditary breast and ovarian cancer syndrome"
                }
              ],
              "name": "hereditary breast and ovarian cancer"
            }
          ],
          "outcome": {
            "id": "SigID032",
            "type": "Significance",
            "label": "Benign"
          },
          "description": "classified by ENIGMA."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/6/16 9:45 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess356">CritAssess356</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn098">EvLn098</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit329">Crit329</a></p> (BP6)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll115">CanAll115</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond043">GenCond043</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess356"><pre><code>{
  "id": "CritAssess356",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit329",
    "type": "Criterion",
    "label": "BP6",
    "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll115",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll121",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll115",
      "alleleName": "NM_000059.3(BRCA2):c.-26G>A"
    }
  },
  "condition": [
    {
      "id": "GenCond043",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID153",
          "type": "Disease",
          "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn098",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp256",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll115",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll121",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll115",
              "alleleName": "NM_000059.3(BRCA2):c.-26G>A"
            }
          },
          "condition": [
            {
              "id": "GenCond043",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID153",
                  "type": "Disease",
                  "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID032",
            "type": "Significance",
            "label": "Benign"
          },
          "description": "classified by Sharing Clinical Reports Project."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:02 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess357">CritAssess357</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn099">EvLn099</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit329">Crit329</a></p> (BP6)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll116">CanAll116</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond044">GenCond044</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess357"><pre><code>{
  "id": "CritAssess357",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit329",
    "type": "Criterion",
    "label": "BP6",
    "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll116",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll122",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll116",
      "alleleName": "NM_000274.3(OAT):c.1311G>C"
    }
  },
  "condition": [
    {
      "id": "GenCond044",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID171",
          "type": "Disease",
          "label": "Gyrate atrophy of choroid and retina"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn099",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp257",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll116",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll122",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll116",
              "alleleName": "NM_000274.3(OAT):c.1311G>C"
            }
          },
          "condition": [
            {
              "id": "GenCond044",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID171",
                  "type": "Disease",
                  "label": "Gyrate atrophy of choroid and retina"
                }
              ]
            }
          ],
          "outcome": {
            "id": "SigID032",
            "type": "Significance",
            "label": "Benign"
          },
          "description": "classified by OMIM"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:03 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess358">CritAssess358</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn100">EvLn100</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit330">Crit330</a></p> (BP7)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll117">CanAll117</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess358"><pre><code>{
  "id": "CritAssess358",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit330",
    "type": "Criterion",
    "label": "BP7",
    "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll117",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll123",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll117",
      "alleleName": "NM_138691.2 (TMC1): c.684C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn100",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "InSilicoPred258",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq301",
            "type": "ReferenceSequence",
            "accession": "NM_138691.2"
          },
          "canonicalAllele": {
            "id": "CanAll117",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll123",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll117",
              "alleleName": "NM_138691.2 (TMC1): c.684C>T"
            }
          },
          "predictionType": {
            "id": "PredID073",
            "type": "Prediction",
            "label": "splicing prediction"
          },
          "prediction": "no impact"
        },
        {
          "id": "AllConserv259",
          "type": "AlleleConservation",
          "allele": {
            "id": "CanAll117",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll123",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll117",
              "alleleName": "NM_138691.2 (TMC1): c.684C>T"
            }
          },
          "conservationType": {
            "id": "ConsrvID212",
            "type": "Conservation",
            "label": "conserved region"
          },
          "qualifier": "NOT",
          "algorithm": "phyloP",
          "evidenceLine": [
            {
              "id": "EvLn135",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "AllConservScore245",
                  "type": "AlleleConservationScore",
                  "allele": {
                    "id": "CanAll117",
                    "type": "CanonicalAllele",
                    "preferredCtxAllele": {
                      "id": "CtxAll123",
                      "type": "ContextualAllele",
                      "relatedCanonicalAllele": "CanAll117",
                      "alleleName": "NM_138691.2 (TMC1): c.684C>T"
                    }
                  },
                  "conservationType": {
                    "id": "ConsrvID212",
                    "type": "Conservation",
                    "label": "conserved region"
                  },
                  "algorithm": "phyloP",
                  "score": -2.297
                }
              ]
            }
          ]
        },
        {
          "id": "AllConserv260",
          "type": "AlleleConservation",
          "allele": {
            "id": "CanAll117",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll123",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll117",
              "alleleName": "NM_138691.2 (TMC1): c.684C>T"
            }
          },
          "conservationType": {
            "id": "ConsrvID212",
            "type": "Conservation",
            "label": "conserved region"
          },
          "qualifier": "NOT",
          "algorithm": "PhastCons",
          "evidenceLine": [
            {
              "id": "EvLn136",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "AllConservScore228",
                  "type": "AlleleConservationScore",
                  "allele": {
                    "id": "CanAll117",
                    "type": "CanonicalAllele",
                    "preferredCtxAllele": {
                      "id": "CtxAll123",
                      "type": "ContextualAllele",
                      "relatedCanonicalAllele": "CanAll117",
                      "alleleName": "NM_138691.2 (TMC1): c.684C>T"
                    }
                  },
                  "conservationType": {
                    "id": "ConsrvID212",
                    "type": "Conservation",
                    "label": "conserved region"
                  },
                  "algorithm": "PhastCons",
                  "score": 0.039
                }
              ]
            }
          ]
        },
        {
          "id": "MolCon261",
          "type": "AlleleMolecularConsequence",
          "consequence": {
            "id": "MolConID016",
            "type": "MolecularConsequence",
            "label": "synonymous variant"
          },
          "contextualAllele": {
            "id": "CtxAll123",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll117",
            "alleleName": "NM_138691.2 (TMC1): c.684C>T"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:04 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess359">CritAssess359</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn101">EvLn101</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit330">Crit330</a></p> (BP7)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll118">CanAll118</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess359"><pre><code>{
  "id": "CritAssess359",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit330",
    "type": "Criterion",
    "label": "BP7",
    "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll118",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll124",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll118",
      "alleleName": "NM_000257.2(MYH7):c.327C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn101",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "InSilicoPred262",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq297",
            "type": "ReferenceSequence",
            "accession": "NM_000257.2"
          },
          "canonicalAllele": {
            "id": "CanAll118",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll124",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll118",
              "alleleName": "NM_000257.2(MYH7):c.327C>T"
            }
          },
          "predictionType": {
            "id": "PredID073",
            "type": "Prediction",
            "label": "splicing prediction"
          },
          "prediction": "no impact",
          "description": "Provenance would show that this comes from Alamut"
        },
        {
          "id": "AllConserv263",
          "type": "AlleleConservation",
          "allele": {
            "id": "CanAll118",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll124",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll118",
              "alleleName": "NM_000257.2(MYH7):c.327C>T"
            }
          },
          "conservationType": {
            "id": "ConsrvID212",
            "type": "Conservation",
            "label": "conserved region"
          },
          "qualifier": "NOT",
          "algorithm": "phyloP",
          "evidenceLine": [
            {
              "id": "EvLn137",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "AllConservScore230",
                  "type": "AlleleConservationScore",
                  "allele": {
                    "id": "CanAll118",
                    "type": "CanonicalAllele",
                    "preferredCtxAllele": {
                      "id": "CtxAll124",
                      "type": "ContextualAllele",
                      "relatedCanonicalAllele": "CanAll118",
                      "alleleName": "NM_000257.2(MYH7):c.327C>T"
                    }
                  },
                  "conservationType": {
                    "id": "ConsrvID212",
                    "type": "Conservation",
                    "label": "conserved region"
                  },
                  "algorithm": "phyloP",
                  "score": -0.76
                }
              ]
            }
          ]
        },
        {
          "id": "AllConserv264",
          "type": "AlleleConservation",
          "allele": {
            "id": "CanAll118",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll124",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll118",
              "alleleName": "NM_000257.2(MYH7):c.327C>T"
            }
          },
          "conservationType": {
            "id": "ConsrvID212",
            "type": "Conservation",
            "label": "conserved region"
          },
          "qualifier": "NOT",
          "algorithm": "PhastCons",
          "evidenceLine": [
            {
              "id": "EvLn138",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "AllConservScore226",
                  "type": "AlleleConservationScore",
                  "allele": {
                    "id": "CanAll118",
                    "type": "CanonicalAllele",
                    "preferredCtxAllele": {
                      "id": "CtxAll124",
                      "type": "ContextualAllele",
                      "relatedCanonicalAllele": "CanAll118",
                      "alleleName": "NM_000257.2(MYH7):c.327C>T"
                    }
                  },
                  "conservationType": {
                    "id": "ConsrvID212",
                    "type": "Conservation",
                    "label": "conserved region"
                  },
                  "algorithm": "PhastCons",
                  "score": 0.63
                }
              ]
            }
          ]
        },
        {
          "id": "MolCon265",
          "type": "AlleleMolecularConsequence",
          "consequence": {
            "id": "MolConID016",
            "type": "MolecularConsequence",
            "label": "synonymous variant"
          },
          "contextualAllele": {
            "id": "CtxAll124",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll118",
            "alleleName": "NM_000257.2(MYH7):c.327C>T"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:05 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess360">CritAssess360</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn102">EvLn102</a></p>
         </td>
          <td>
              <p>This variant is a silent variant that does not result in an amino acid change. It is not very conserved and splicing tools suggest no change. However, BP7 does not apply because the variant is located in the first base of exon 4 of RAF1, which is part of the 3’ splice region of the exon. For that reason, we can’t fully trust computational tools. Functional assays would have to confirm that this variant truly does not alter splicing.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit330">Crit330</a></p> (BP7)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll119">CanAll119</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID075">CritEvalID075</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess360"><pre><code>{
  "id": "CritAssess360",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit330",
    "type": "Criterion",
    "label": "BP7",
    "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll119",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll125",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll119",
      "alleleName": "NM_002880.3(RAF1):c.321T>C"
    }
  },
  "outcome": {
    "id": "CritEvalID075",
    "type": "CriterionEvaluation",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn102",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "InSilicoPred266",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq302",
            "type": "ReferenceSequence",
            "accession": "NM_002880.3"
          },
          "canonicalAllele": {
            "id": "CanAll119",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll125",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll119",
              "alleleName": "NM_002880.3(RAF1):c.321T>C"
            }
          },
          "predictionType": {
            "id": "PredID073",
            "type": "Prediction",
            "label": "splicing prediction"
          },
          "prediction": "no impact"
        },
        {
          "id": "AllConserv267",
          "type": "AlleleConservation",
          "allele": {
            "id": "CanAll119",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll125",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll119",
              "alleleName": "NM_002880.3(RAF1):c.321T>C"
            }
          },
          "conservationType": {
            "id": "ConsrvID212",
            "type": "Conservation",
            "label": "conserved region"
          },
          "qualifier": "NOT",
          "algorithm": "phyloP",
          "evidenceLine": [
            {
              "id": "EvLn139",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "AllConservScore222",
                  "type": "AlleleConservationScore",
                  "allele": {
                    "id": "CanAll119",
                    "type": "CanonicalAllele",
                    "preferredCtxAllele": {
                      "id": "CtxAll125",
                      "type": "ContextualAllele",
                      "relatedCanonicalAllele": "CanAll119",
                      "alleleName": "NM_002880.3(RAF1):c.321T>C"
                    }
                  },
                  "conservationType": {
                    "id": "ConsrvID212",
                    "type": "Conservation",
                    "label": "conserved region"
                  },
                  "algorithm": "phyloP",
                  "score": 0.45
                }
              ]
            }
          ]
        },
        {
          "id": "AllConserv268",
          "type": "AlleleConservation",
          "allele": {
            "id": "CanAll119",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll125",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll119",
              "alleleName": "NM_002880.3(RAF1):c.321T>C"
            }
          },
          "conservationType": {
            "id": "ConsrvID212",
            "type": "Conservation",
            "label": "conserved region"
          },
          "qualifier": "NOT",
          "algorithm": "PhastCons",
          "evidenceLine": [
            {
              "id": "EvLn140",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "AllConservScore224",
                  "type": "AlleleConservationScore",
                  "allele": {
                    "id": "CanAll119",
                    "type": "CanonicalAllele",
                    "preferredCtxAllele": {
                      "id": "CtxAll125",
                      "type": "ContextualAllele",
                      "relatedCanonicalAllele": "CanAll119",
                      "alleleName": "NM_002880.3(RAF1):c.321T>C"
                    }
                  },
                  "conservationType": {
                    "id": "ConsrvID212",
                    "type": "Conservation",
                    "label": "conserved region"
                  },
                  "algorithm": "PhastCons",
                  "score": 1.0
                }
              ]
            }
          ]
        },
        {
          "id": "MolCon269",
          "type": "AlleleMolecularConsequence",
          "consequence": {
            "id": "MolConID016",
            "type": "MolecularConsequence",
            "label": "synonymous variant"
          },
          "contextualAllele": {
            "id": "CtxAll125",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll119",
            "alleleName": "NM_002880.3(RAF1):c.321T>C"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:06 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This variant is a silent variant that does not result in an amino acid change. It is not very conserved and splicing tools suggest no change. However, BP7 does not apply because the variant is located in the first base of exon 4 of RAF1, which is part of the 3’ splice region of the exon. For that reason, we can’t fully trust computational tools. Functional assays would have to confirm that this variant truly does not alter splicing."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess361">CritAssess361</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn103">EvLn103</a></p>
         </td>
          <td>
              <p>This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit303">Crit303</a></p> (PVS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll121">CanAll121</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond046">GenCond046</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess361"><pre><code>{
  "id": "CritAssess361",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit303",
    "type": "Criterion",
    "label": "PVS1",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "defaultStrength": {
      "id": "EvLnStrgID146",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Very Strong"
    },
    "comments": "The two main concerns for applying this rule is:\na. Is LOF an established disease mechanism?\nb. Is LOF actually occurring?"
  },
  "variant": {
    "id": "CanAll121",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll127",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll121",
      "alleleName": "NM_024422.3(DSC2): c.631-2A>G (p.?)"
    }
  },
  "condition": [
    {
      "id": "GenCond046",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID164",
          "type": "Disease",
          "label": "arrhythmogenic right ventricular cardiomyopathy"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn103",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "MolCon272",
          "type": "AlleleMolecularConsequence",
          "contextualAllele": {
            "id": "CtxAll127",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll121",
            "alleleName": "NM_024422.3(DSC2): c.631-2A>G (p.?)"
          },
          "consequence": {
            "id": "MolConID015",
            "type": "MolecularConsequence",
            "label": "splice-site variant"
          },
          "description": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466)."
        },
        {
          "id": "CondMech273",
          "type": "ConditionMechanism",
          "mechanism": {
            "id": "MechID071",
            "type": "Mechanism",
            "label": "loss of function variant"
          },
          "condition": {
            "id": "GenCond046",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID164",
                "type": "Disease",
                "label": "arrhythmogenic right ventricular cardiomyopathy"
              }
            ]
          },
          "mechanismConfidence": {
            "id": "CondMechStrgID134",
            "type": "ConditionMechanismStrength",
            "label": "established"
          },
          "gene": {
            "id": "G286",
            "type": "Gene",
            "geneSymbol": "DSC2"
          },
          "description": "Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/)."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "01/17/17 9:12 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/)."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess363">CritAssess363</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn106">EvLn106</a></p>
         </td>
          <td>
              <p>This variant was identified in one individual with pathogenic variants in another gene. Internal LMM data: The proband had one pathogenic and one likely pathogenic variant in (in trans) in SLC26A4. This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome. SLC26A4 is the only gene associated with Pendred Syndrome</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit328">Crit328</a></p> (BP5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll092">CanAll092</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond058">GenCond058</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess363"><pre><code>{
  "id": "CritAssess363",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit328",
    "type": "Criterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CanAll092",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll098",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll092",
      "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
    }
  },
  "condition": [
    {
      "id": "GenCond058",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID183",
          "type": "Disease",
          "label": "nonsyndromic deafness"
        }
      ],
      "name": "Hearing Loss"
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn106",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp364",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll122",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll128",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll122",
              "alleleName": "NM_000441.1(SLC26A4): c.626G>T"
            }
          },
          "condition": [
            {
              "id": "GenCond058",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID183",
                  "type": "Disease",
                  "label": "nonsyndromic deafness"
                }
              ],
              "name": "Hearing Loss"
            }
          ],
          "outcome": {
            "id": "SigID028",
            "type": "Significance",
            "label": "Likely Pathogenic"
          }
        },
        {
          "id": "VarInterp365",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll123",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll129",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll123",
              "alleleName": "NM_000441.1(SLC26A4): c.1707+6T>C"
            }
          },
          "condition": [
            {
              "id": "GenCond058",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID183",
                  "type": "Disease",
                  "label": "nonsyndromic deafness"
                }
              ],
              "name": "Hearing Loss"
            }
          ],
          "outcome": {
            "id": "SigID031",
            "type": "Significance",
            "label": "Pathogenic"
          }
        },
        {
          "id": "IndGeno366",
          "type": "IndividualGenotype",
          "individual": {
            "id": "Indiv361",
            "type": "Individual",
            "description": "individual #029"
          },
          "genotype": [
            {
              "id": "Geno428",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo426",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll092",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll098",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll092",
                        "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
                      }
                    }
                  ]
                },
                {
                  "id": "Haplo427",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll122",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll128",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll122",
                        "alleleName": "NM_000441.1(SLC26A4): c.626G>T"
                      }
                    }
                  ]
                }
              ]
            },
            {
              "id": "Geno431",
              "type": "Genotype",
              "haplotype": [
                {
                  "id": "Haplo429",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll092",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll098",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll092",
                        "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
                      }
                    }
                  ]
                },
                {
                  "id": "Haplo430",
                  "type": "Haplotype",
                  "allele": [
                    {
                      "id": "CanAll123",
                      "type": "CanonicalAllele",
                      "preferredCtxAllele": {
                        "id": "CtxAll129",
                        "type": "ContextualAllele",
                        "relatedCanonicalAllele": "CanAll123",
                        "alleleName": "NM_000441.1(SLC26A4): c.1707+6T>C"
                      }
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "IndCond368",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv361",
            "type": "Individual",
            "description": "individual #029"
          },
          "condition": [
            {
              "id": "GenCond058",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID183",
                  "type": "Disease",
                  "label": "nonsyndromic deafness"
                }
              ],
              "name": "Hearing Loss"
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          },
          "description": "This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "03/07/2017 4:30 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This variant was identified in one individual with pathogenic variants in another gene. Internal LMM data: The proband had one pathogenic and one likely pathogenic variant in (in trans) in SLC26A4. This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome. SLC26A4 is the only gene associated with Pendred Syndrome"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess369">CritAssess369</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn107">EvLn107</a></p>
         </td>
          <td>
              <p>This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042).</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit309">Crit309</a></p> (PM2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll121">CanAll121</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess369"><pre><code>{
  "id": "CritAssess369",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit309",
    "type": "Criterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing"
  },
  "variant": {
    "id": "CanAll121",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll127",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll121",
      "alleleName": "NM_024422.3(DSC2): c.631-2A>G (p.?)"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn107",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq370",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID090",
            "type": "Population",
            "label": "Non-Finnish European"
          },
          "allele": {
            "id": "CanAll121",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll127",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll121",
              "alleleName": "NM_024422.3(DSC2): c.631-2A>G (p.?)"
            }
          },
          "alleleCount": 1,
          "alleleNumber": 66564,
          "alleleFrequency": 1.5e-05,
          "homozygousAlleleIndividualCount": 0
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "03/07/2017 5:22 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042)."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess371">CritAssess371</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn108">EvLn108</a></p>
         </td>
          <td>
              <p>This variant is present in 0.2% (31/16512) of South Asian Chromosomes in ExAC.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit320">Crit320</a></p> (BS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll118">CanAll118</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess371"><pre><code>{
  "id": "CritAssess371",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit320",
    "type": "Criterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CanAll118",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll124",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll118",
      "alleleName": "NM_000257.2(MYH7):c.327C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn108",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq037",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "allele": {
            "id": "CanAll118",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll124",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll118",
              "alleleName": "NM_000257.2(MYH7):c.327C>T"
            }
          },
          "alleleCount": 31,
          "alleleNumber": 16512,
          "alleleFrequency": 0.02,
          "population": {
            "id": "PopID092",
            "type": "Population",
            "label": "South Asian"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "03/08/2017 7:43 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This variant is present in 0.2% (31/16512) of South Asian Chromosomes in ExAC."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess372">CritAssess372</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn109">EvLn109</a></p>
         </td>
          <td>
              <p>This variant is present in 2% (27/1384) of African American Chromosomes in ESP.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit320">Crit320</a></p> (BS1)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll092">CanAll092</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond057">GenCond057</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess372"><pre><code>{
  "id": "CritAssess372",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit320",
    "type": "Criterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CanAll092",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll098",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll092",
      "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
    }
  },
  "condition": [
    {
      "id": "GenCond057",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID183",
          "type": "Disease",
          "label": "nonsyndromic deafness"
        }
      ],
      "name": "Hearing Loss"
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn109",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq038",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "allele": {
            "id": "CanAll092",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll098",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll092",
              "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
            }
          },
          "alleleCount": 27,
          "alleleNumber": 1384,
          "alleleFrequency": 0.02,
          "population": {
            "id": "PopID085",
            "type": "Population",
            "label": "African/African American"
          }
        },
        {
          "id": "CondPrev039",
          "type": "ConditionPrevelance",
          "condition": [
            {
              "id": "GenCond057",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID183",
                  "type": "Disease",
                  "label": "nonsyndromic deafness"
                }
              ],
              "name": "Hearing Loss"
            }
          ],
          "minimum": 1.0,
          "maximum": 1.0,
          "population": {
            "id": "PopID094",
            "type": "Population",
            "label": "Overall"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "03/08/2017 9:46 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This variant is present in 2% (27/1384) of African American Chromosomes in ESP."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess373">CritAssess373</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn117">EvLn117</a></p>
         </td>
          <td>
              <p>This is a synonymous variant with little conservation and no predicted impact on splicing. AllConservation and splicing algorithm values are listed below.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit327">Crit327</a></p> (BP4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll118">CanAll118</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess373"><pre><code>{
  "id": "CritAssess373",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit327",
    "type": "Criterion",
    "label": "BP4",
    "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "defaultStrength": {
      "id": "EvLnStrgID140",
      "type": "EvidenceLineStrength",
      "label": "Benign Supporting"
    },
    "comments": "same issues as PP3."
  },
  "variant": {
    "id": "CanAll118",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll124",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll118",
      "alleleName": "NM_000257.2(MYH7):c.327C>T"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn117",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "InSilicoPred068",
          "type": "InSilicoPrediction",
          "transcript": {
            "id": "RefSeq297",
            "type": "ReferenceSequence",
            "accession": "NM_000257.2"
          },
          "canonicalAllele": {
            "id": "CanAll118",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll124",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll118",
              "alleleName": "NM_000257.2(MYH7):c.327C>T"
            }
          },
          "predictionType": {
            "id": "PredID073",
            "type": "Prediction",
            "label": "splicing prediction"
          },
          "prediction": "no impact",
          "description": "Provenance would show that this comes from Alamut"
        },
        {
          "id": "AllConserv071",
          "type": "AlleleConservation",
          "allele": {
            "id": "CanAll118",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll124",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll118",
              "alleleName": "NM_000257.2(MYH7):c.327C>T"
            }
          },
          "conservationType": {
            "id": "ConsrvID212",
            "type": "Conservation",
            "label": "conserved region"
          },
          "qualifier": "NOT",
          "algorithm": "phyloP",
          "evidenceLine": [
            {
              "id": "EvLn133",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "AllConservScore116",
                  "type": "AlleleConservationScore",
                  "allele": {
                    "id": "CanAll118",
                    "type": "CanonicalAllele",
                    "preferredCtxAllele": {
                      "id": "CtxAll124",
                      "type": "ContextualAllele",
                      "relatedCanonicalAllele": "CanAll118",
                      "alleleName": "NM_000257.2(MYH7):c.327C>T"
                    }
                  },
                  "conservationType": {
                    "id": "ConsrvID212",
                    "type": "Conservation",
                    "label": "conserved region"
                  },
                  "algorithm": "phyloP",
                  "score": -0.76
                }
              ]
            }
          ]
        },
        {
          "id": "AllConserv072",
          "type": "AlleleConservation",
          "allele": {
            "id": "CanAll118",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll124",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll118",
              "alleleName": "NM_000257.2(MYH7):c.327C>T"
            }
          },
          "conservationType": {
            "id": "ConsrvID212",
            "type": "Conservation",
            "label": "conserved region"
          },
          "qualifier": "NOT",
          "algorithm": "PhastCons",
          "evidenceLine": [
            {
              "id": "EvLn134",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "AllConservScore171",
                  "type": "AlleleConservationScore",
                  "allele": {
                    "id": "CanAll118",
                    "type": "CanonicalAllele",
                    "preferredCtxAllele": {
                      "id": "CtxAll124",
                      "type": "ContextualAllele",
                      "relatedCanonicalAllele": "CanAll118",
                      "alleleName": "NM_000257.2(MYH7):c.327C>T"
                    }
                  },
                  "conservationType": {
                    "id": "ConsrvID212",
                    "type": "Conservation",
                    "label": "conserved region"
                  },
                  "algorithm": "PhastCons",
                  "score": 0.63
                }
              ]
            }
          ]
        },
        {
          "id": "MolCon075",
          "type": "AlleleMolecularConsequence",
          "consequence": {
            "id": "MolConID016",
            "type": "MolecularConsequence",
            "label": "synonymous variant"
          },
          "contextualAllele": {
            "id": "CtxAll124",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll118",
            "alleleName": "NM_000257.2(MYH7):c.327C>T"
          }
        },
        {
          "id": "Null415",
          "type": "NullAllele",
          "contextualAllele": {
            "id": "CtxAll127",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll121",
            "alleleName": "NM_024422.3(DSC2): c.631-2A>G (p.?)"
          },
          "annotation": {
            "id": "NullAllOutID223",
            "type": "NullAlleleOutcome",
            "label": "null mutation"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "03/08/2017 10:02 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "This is a synonymous variant with little conservation and no predicted impact on splicing. AllConservation and splicing algorithm values are listed below."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess374">CritAssess374</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn116">EvLn116</a></p>
         </td>
          <td>
              <p>Flag-tagged construct with the variant was transfected into HEK cells. As per the RASopathy working group’s approved assays (see example 3 above), Smith et al. showed that the Y337C mutation increased levels of pERK in starved HEK cells via western blotting.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit306">Crit306</a></p> (PS3)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll124">CanAll124</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess374"><pre><code>{
  "id": "CritAssess374",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit306",
    "type": "Criterion",
    "label": "PS3",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "defaultStrength": {
      "id": "EvLnStrgID145",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Strong"
    },
    "comments": "This particular rule will probably be downgraded often, given that there are not many “Functional studies that have been validated and shown to be reproducible and robust in a clinical diagnostic laboratory setting”"
  },
  "variant": {
    "id": "CanAll124",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll130",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll124",
      "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn116",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "Func067",
          "type": "AlleleFunctionalImpact",
          "resultDescription": "increased ERK activtation",
          "assayType": {
            "id": "AFAMID076",
            "type": "AlleleFunctionalAssayMethod",
            "label": "Activation Assay"
          },
          "contextualAllele": {
            "id": "CtxAll130",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": "CanAll124",
            "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
          },
          "gene": {
            "id": "G278",
            "type": "Gene",
            "geneSymbol": "SOS1"
          },
          "description": "PMID: 23487764"
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "3/14/2017 10:29 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Flag-tagged construct with the variant was transfected into HEK cells. As per the RASopathy working group’s approved assays (see example 3 above), Smith et al. showed that the Y337C mutation increased levels of pERK in starved HEK cells via western blotting."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess375">CritAssess375</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn123">EvLn123</a></p>
         </td>
          <td>
              <p>A personal communication to our lab informed us of an individual and his son who both harbored the variant but with clinical workup did not appear to have Noonan Syndrome or symptoms indicative of Noonan Syndrome.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit323">Crit323</a></p> (BS4)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll124">CanAll124</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond056">GenCond056</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess375"><pre><code>{
  "id": "CritAssess375",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit323",
    "type": "Criterion",
    "label": "BS4",
    "description": "Lack of segregation in affected members of a family",
    "defaultStrength": {
      "id": "EvLnStrgID141",
      "type": "EvidenceLineStrength",
      "label": "Benign Strong"
    },
    "comments": "This rule is for “lack of segregation” meaning phenotype positive / genotype negative (and not pheno neg / geno pos). Also need to be careful of phenocopies. Also need rules for how many cases are needed in use this rule"
  },
  "variant": {
    "id": "CanAll124",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll130",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll124",
      "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
    }
  },
  "condition": [
    {
      "id": "GenCond056",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID175",
          "type": "Disease",
          "label": "noonan syndrome"
        }
      ],
      "modeOfInheritance": {
        "id": "ModeOfInhID051",
        "type": "ModeOfInheritance",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn123",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "FamSeg076",
          "type": "FamilySegregation",
          "condition": {
            "id": "GenCond056",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "DisID175",
                "type": "Disease",
                "label": "noonan syndrome"
              }
            ],
            "modeOfInheritance": {
              "id": "ModeOfInhID051",
              "type": "ModeOfInheritance",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CanAll124",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll130",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll124",
              "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
            }
          },
          "family": {
            "id": "Fam373",
            "type": "Family",
            "description": "Family #011"
          },
          "phenotypePositiveAllelePositive": 0,
          "phenotypeNegativeAllelePositive": 2,
          "phenotypePositiveAlleleNegative": 0,
          "inconsistentSegregationsObserved": {
            "id": "InconSegObsID220",
            "type": "InconsistentSegregationObserved",
            "label": "present"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "3/14/2017 10:45 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "A personal communication to our lab informed us of an individual and his son who both harbored the variant but with clinical workup did not appear to have Noonan Syndrome or symptoms indicative of Noonan Syndrome."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess376">CritAssess376</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn126">EvLn126</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#Crit309">Crit309</a></p> (PM2)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll125">CanAll125</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess376"><pre><code>{
  "id": "CritAssess376",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit309",
    "type": "Criterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing"
  },
  "variant": {
    "id": "CanAll125",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll131",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll125",
      "alleleName": "NM_004333.4(BRAF):c.1447A>C (p.Lys483Gln)",
      "produces": [
        {
          "id": "CtxAll132",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll126",
          "alleleName": "NP_004324.2:p.Lys483Gln",
          "producedBy": "CtxAll131"
        }
      ]
    }
  },
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn126",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "AllFreq101",
          "type": "AlleleFrequency",
          "ascertainment": {
            "id": "AscrtID203",
            "type": "Ascertainment",
            "label": "ExAc ascertainment method"
          },
          "population": {
            "id": "PopID093",
            "type": "Population",
            "label": "Combined"
          },
          "allele": {
            "id": "CanAll125",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll131",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll125",
              "alleleName": "NM_004333.4(BRAF):c.1447A>C (p.Lys483Gln)",
              "produces": [
                {
                  "id": "CtxAll132",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll126",
                  "alleleName": "NP_004324.2:p.Lys483Gln",
                  "producedBy": "CtxAll131"
                }
              ]
            }
          },
          "alleleCount": 0,
          "alleleFrequency": 0.0,
          "medianCoverage": 57.0
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent010",
        "type": "Agent",
        "label": "ACME Molecular Lab"
      },
      "contributionDate": "6/11/2011 11:45 AM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess377">CritAssess377</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn128">EvLn128</a></p>
         </td>
          <td>
              <p>Parental testing in our lab confirmed this variant was present de novo in 1 proband with clinical features of CFC. (LMM internal Data)</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit313">Crit313</a></p> (PM6)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll125">CanAll125</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond069">GenCond069</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess377"><pre><code>{
  "id": "CritAssess377",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit313",
    "type": "Criterion",
    "label": "PM6",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CanAll125",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll131",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll125",
      "alleleName": "NM_004333.4(BRAF):c.1447A>C (p.Lys483Gln)",
      "produces": [
        {
          "id": "CtxAll132",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll126",
          "alleleName": "NP_004324.2:p.Lys483Gln",
          "producedBy": "CtxAll131"
        }
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond069",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID156",
          "type": "Disease",
          "label": "cardiofaciocutaneous syndrome"
        }
      ]
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn128",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "IndAllInh106",
          "type": "IndividualAlleleInheritance",
          "canonicalAllele": {
            "id": "CanAll125",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll131",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll125",
              "alleleName": "NM_004333.4(BRAF):c.1447A>C (p.Lys483Gln)",
              "produces": [
                {
                  "id": "CtxAll132",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll126",
                  "alleleName": "NP_004324.2:p.Lys483Gln",
                  "producedBy": "CtxAll131"
                }
              ]
            }
          },
          "individual": {
            "id": "Indiv362",
            "type": "Individual",
            "description": "individual #030"
          },
          "parentalConfirmation": {
            "id": "ParenConfID217",
            "type": "ParentalConfirmation",
            "label": "no parental confirmation"
          },
          "alleleInheritance": [
            {
              "id": "AlleleInhID197",
              "type": "AlleleInheritance",
              "label": "de novo"
            }
          ]
        },
        {
          "id": "IndCond110",
          "type": "IndividualCondition",
          "individual": {
            "id": "Indiv362",
            "type": "Individual",
            "description": "individual #030"
          },
          "condition": [
            {
              "id": "GenCond069",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "individualHasCondition": {
            "id": "CondStatID218",
            "type": "ConditionStatus",
            "label": "affected"
          }
        },
        {
          "id": "FamCond112",
          "type": "FamilyCondition",
          "proband": {
            "id": "Indiv362",
            "type": "Individual",
            "description": "individual #030"
          },
          "condition": [
            {
              "id": "GenCond069",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID156",
                  "type": "Disease",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "familyHasCondition": {
            "id": "CondStatID219",
            "type": "ConditionStatus",
            "label": "unaffected"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent010",
        "type": "Agent",
        "label": "ACME Molecular Lab"
      },
      "contributionDate": "6/11/2011 12:45:00",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Parental testing in our lab confirmed this variant was present de novo in 1 proband with clinical features of CFC. (LMM internal Data)"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CritAssess378">CritAssess378</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#EvLn130">EvLn130</a></p>
         </td>
          <td>
              <p>Another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). The other variant: NM_004333.4(BRAF):c.1449A>C (p.Lys483Asn) is classified as likely pathogenic by LMM for Rasopathy.</p>
         </td>
          <td>
              <p><a href="../details/details.html#Crit312">Crit312</a></p> (PM5)
         </td>
          <td>
              <p><a href="../details/details.html#CanAll125">CanAll125</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond049">GenCond049</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#CritEvalID074">CritEvalID074</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CritAssess378"><pre><code>{
  "id": "CritAssess378",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "Crit312",
    "type": "Criterion",
    "label": "PM5",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "defaultStrength": {
      "id": "EvLnStrgID143",
      "type": "EvidenceLineStrength",
      "label": "Pathogenic Moderate"
    },
    "comments": "Need to be sure that pathogenic variant exists at the residue in question. Sometimes discrepancy resolution may be required. Some thing like Variantexplorer.org may be useful.\nSometimes no assertion is provided in ClinVar ...may require curation by lit. search."
  },
  "variant": {
    "id": "CanAll125",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll131",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll125",
      "alleleName": "NM_004333.4(BRAF):c.1447A>C (p.Lys483Gln)",
      "produces": [
        {
          "id": "CtxAll132",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll126",
          "alleleName": "NP_004324.2:p.Lys483Gln",
          "producedBy": "CtxAll131"
        }
      ]
    }
  },
  "condition": [
    {
      "id": "GenCond049",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "DisID185",
          "type": "Disease",
          "label": "Noonan syndrome and noonan-related syndrom"
        }
      ],
      "name": "Rasopathy"
    }
  ],
  "outcome": {
    "id": "CritEvalID074",
    "type": "CriterionEvaluation",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "EvLn130",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "VarInterp060",
          "type": "VariantInterpretation",
          "variant": {
            "id": "CanAll127",
            "type": "CanonicalAllele",
            "preferredCtxAllele": {
              "id": "CtxAll133",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": "CanAll127",
              "alleleName": "NM_004333.4(BRAF):c.1449A>C",
              "produces": [
                {
                  "id": "CtxAll134",
                  "type": "ContextualAllele",
                  "relatedCanonicalAllele": "CanAll128",
                  "alleleName": "NP_004324.2:p.Lys483Asn",
                  "producedBy": "CtxAll133"
                }
              ]
            }
          },
          "condition": [
            {
              "id": "GenCond049",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "DisID185",
                  "type": "Disease",
                  "label": "Noonan syndrome and noonan-related syndrom"
                }
              ],
              "name": "Rasopathy"
            }
          ],
          "outcome": {
            "id": "SigID028",
            "type": "Significance",
            "label": "Likely Pathogenic"
          },
          "contribution": [
            {
              "type": "Contribution",
              "agent": {
                "id": "Agent010",
                "type": "Agent",
                "label": "ACME Molecular Lab"
              },
              "contributionDate": "6/12/2011 9:00 AM",
              "contributionRole": {
                "id": "ContribRoleID196",
                "type": "ContributoryRole",
                "label": "interpreter role"
              }
            }
          ],
          "description": "The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic."
        }
      ]
    }
  ],
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent010",
        "type": "Agent",
        "label": "ACME Molecular Lab"
      },
      "contributionDate": "6/10/2011 1:45 PM",
      "contributionRole": {
        "id": "ContribRoleID193",
        "type": "ContributoryRole",
        "label": "assessor role"
      }
    }
  ],
  "description": "Another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). The other variant: NM_004333.4(BRAF):c.1449A>C (p.Lys483Asn) is classified as likely pathogenic by LMM for Rasopathy."
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../../">Interpretation Overview</a></h5>
              </li>
              <li class='list-group-item'><h5><a href="../../user/">User Guide</a></h5></li>
              <li class="list-group-item"><h5><a href="../">Technical Reference</a></h5><ul class='list-unstyled'><li class="active"><a href="./">Core</a></li><ul class='list-unstyled'><li class="active"><a href="criterion_assessment.html">CriterionAssessment</a></li><li><a href="domain_entity.html">DomainEntity</a></li><li><a href="evidence_line.html">EvidenceLine</a></li><li><a href="genetic_condition.html">GeneticCondition</a></li><li><a href="statement.html">Statement</a></li><li><a href="variant_interpretation.html">VariantInterpretation</a></li></ul><li><a href="../domain_entities/">Domain Entities</a></li><li><a href="../evidence_items/">Evidence Item</a></li><li><a href="../provenance/">Provenance</a></li><li><a href="../terminology/">Terminology</a></li></ul></li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2017 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../javascripts/jquery.js"></script><script src="../../javascripts/bootstrap.js"></script><script src="../../javascripts/jquery.jsonview.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
