<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../stylesheets/brand2.css" rel="stylesheet" /><link href="../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/evidence_line.html">0.1</a></li>
                  <li><a href="/development/tech/evidence_line.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../">Interpretation Model</a></li>
<li><a href="./">Technical Reference and Examples</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../images/clingen-doc-logo.png" />
            </li>

            <li class="active">
              <a href="../">Welcome</a>
            </li>
            <li><a href="../user/">User Guide</a></li>
            <li class="active"><a href="./">Technical Reference</a></li><ul><li><a href="criterion_assessment.html">CriterionAssessment</a></li><li><a href="evidence/">Information</a></li><li class="active"><a href="evidence_line.html">EvidenceLine</a></li><li><a href="variant_interpretation.html">VariantInterpretation</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>EvidenceLine</h1>
            <p>A reification of the underlying supporting data which may be one or more data points or assertions. Typically grouped in a specific manner for use in an owning assertion activity (either Variant Interpretation or CriterionAssessment. All evidence may optionally be given a strength by an agent.</p>


            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>explanation</td>
                <td>String</td>
                <td>1..1</td>
                <td>A human-readable description of the entity</td>
            </tr>
            <tr>
                <td>strength</td>
                <td>CodeableConcept</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>hasInformation</td>
                <td>Information</td>
                <td>0..*</td>
                <td></td>
            </tr>
            <tr>
                <td>contribution</td>
                <td>Contribution</td>
                <td>0..*</td>
                <td>The contributions (agent, time and role) made to this entity</td>
            </tr>
            </tbody>
        </table>



    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>description</th>
        <th>explanation</th>
        <th>strength</th>
        <th>hasInformation</th>
        <th>contribution</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="details/details.html#EvLn001">EvLn001</a></p></td>
          <td>EvLn001</td>
          <td>
              <p>EvLn001</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon001">MolCon001</a></p><p><a href="details/details.html#CondMech002">CondMech002</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn001"><pre><code>{
  "cg:id": "EvLn001",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn001",
  "hasInformation": [
    {
      "cg:id": "MolCon001",
      "cg:type": "MolecularConsequence",
      "explanation": "Protein ends at p.826 and this frameshift would create stop at p.852 (769+83) thus NMD would not occur. Instead an elongated protein",
      "contextualAllele": {
        "cg:id": "CtxAll001",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll001",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll001",
          "identifier": "http://reg.genome.network/allele/CA090943"
        },
        "alleleName": "NM_031844.2(HNRNPU):c.2304_2305del (p.Gly769Glufs*83)"
      },
      "consequence": {
        "cg:id": "CC001",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001589",
            "cg:type": "Coding",
            "code": "0001589",
            "system": "http://sequenceontology.org/",
            "display": "frameshift variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech002",
      "cg:type": "ConditionMechanism",
      "explanation": "HNRNPU is not associated with any conditions, and therefore LOF is not an established mechanism of any condition for this gene.",
      "mechanism": {
        "cg:id": "CC002",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0002054",
            "cg:type": "Coding",
            "code": "0002054",
            "system": "http://sequenceontology.org/",
            "display": "loss of function variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC003",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:established",
            "cg:type": "Coding",
            "code": "established",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Established"
          }
        ]
      },
      "gene": {
        "cg:id": "G001",
        "cg:type": "Gene",
        "symbol": "HNRNPU"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn002">EvLn002</a></p></td>
          <td>EvLn002</td>
          <td>
              <p>EvLn002</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon003">MolCon003</a></p><p><a href="details/details.html#CondMech004">CondMech004</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn002"><pre><code>{
  "cg:id": "EvLn002",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn002",
  "hasInformation": [
    {
      "cg:id": "MolCon003",
      "cg:type": "MolecularConsequence",
      "explanation": "LOF is predicted to occur as variant is >55bp from penultimate exon",
      "contextualAllele": {
        "cg:id": "CtxAll002",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll002",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll002",
          "identifier": "http://reg.genome.network/allele/CA090898"
        },
        "alleleName": "NM_000059.3(BRCA2):c.7762_7764delinsTT"
      },
      "consequence": {
        "cg:id": "CC004",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001589",
            "cg:type": "Coding",
            "code": "0001589",
            "system": "http://sequenceontology.org/",
            "display": "frameshift variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech004",
      "cg:type": "ConditionMechanism",
      "explanation": "This is so well-known that it does not require a citation.",
      "mechanism": {
        "cg:id": "CC005",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0002054",
            "cg:type": "Coding",
            "code": "0002054",
            "system": "http://sequenceontology.org/",
            "display": "loss of function variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC006",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:established",
            "cg:type": "Coding",
            "code": "established",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Established"
          }
        ]
      },
      "gene": {
        "cg:id": "G002",
        "cg:type": "Gene",
        "symbol": "BRCA2"
      },
      "condition": {
        "cg:id": "MendCond001",
        "cg:type": "MendelianCondition",
        "explanation": "Breast Ovarian Cancer, Autosomal Dominant",
        "name": "Breast Ovarian Cancer, Autosomal Dominant",
        "disease": [
          {
            "cg:id": "MESH:D061325",
            "cg:type": "Coding",
            "code": "D061325",
            "system": "https://meshb.nlm.nih.gov/",
            "display": "hereditary breast ovarian cancer"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn003">EvLn003</a></p></td>
          <td>EvLn003</td>
          <td>
              <p>EvLn003</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon005">MolCon005</a></p><p><a href="details/details.html#CondMech007">CondMech007</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn003"><pre><code>{
  "cg:id": "EvLn003",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn003",
  "hasInformation": [
    {
      "cg:id": "MolCon005",
      "cg:type": "MolecularConsequence",
      "explanation": "LOF is predicted but this transcript is rare",
      "contextualAllele": {
        "cg:id": "CtxAll003",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll003",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll003",
          "identifier": "http://reg.genome.network/allele/CA238217"
        },
        "alleleName": "NM_133437.4(TTN):c.10670dupG "
      },
      "consequence": {
        "cg:id": "CC007",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001589",
            "cg:type": "Coding",
            "code": "0001589",
            "system": "http://sequenceontology.org/",
            "display": "frameshift variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech007",
      "cg:type": "ConditionMechanism",
      "mechanism": {
        "cg:id": "CC009",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0002054",
            "cg:type": "Coding",
            "code": "0002054",
            "system": "http://sequenceontology.org/",
            "display": "loss of function variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC010",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:established",
            "cg:type": "Coding",
            "code": "established",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Established"
          }
        ]
      },
      "gene": {
        "cg:id": "G003",
        "cg:type": "Gene",
        "symbol": "TTN"
      },
      "condition": {
        "cg:id": "MendCond047",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:217604",
        "disease": [
          {
            "cg:id": "Orphanet:217604"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn004">EvLn004</a></p></td>
          <td>EvLn004</td>
          <td>
              <p>EvLn004</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon006">MolCon006</a></p><p><a href="details/details.html#CondMech007">CondMech007</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn004"><pre><code>{
  "cg:id": "EvLn004",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn004",
  "hasInformation": [
    {
      "cg:id": "MolCon006",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll004",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll003",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll003",
          "identifier": "http://reg.genome.network/allele/CA238217"
        },
        "alleleName": "LRG_391t2:c.10303+2691dupG"
      },
      "consequence": {
        "cg:id": "CC008",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0000191",
            "cg:type": "Coding",
            "code": "0000191",
            "system": "http://sequenceontology.org/",
            "display": "interior intron"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech007",
      "cg:type": "ConditionMechanism",
      "mechanism": {
        "cg:id": "CC009",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0002054",
            "cg:type": "Coding",
            "code": "0002054",
            "system": "http://sequenceontology.org/",
            "display": "loss of function variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC010",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:established",
            "cg:type": "Coding",
            "code": "established",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Established"
          }
        ]
      },
      "gene": {
        "cg:id": "G003",
        "cg:type": "Gene",
        "symbol": "TTN"
      },
      "condition": {
        "cg:id": "MendCond047",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:217604",
        "disease": [
          {
            "cg:id": "Orphanet:217604"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn005">EvLn005</a></p></td>
          <td>EvLn005</td>
          <td>
              <p>EvLn005</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp008">VarInterp008</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn005"><pre><code>{
  "cg:id": "EvLn005",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn005",
  "hasInformation": [
    {
      "cg:id": "VarInterp008",
      "cg:type": "VariantInterpretation",
      "explanation": "NM_002834.3:c.855T>G Pathogenic for Noonan Syndrome",
      "variant": {
        "cg:id": "CanAll005",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll006",
        "identifier": "http://reg.genome.network/allele/CA220155"
      },
      "condition": [
        {
          "cg:id": "MendCond003",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:648, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:648"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC011",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn006">EvLn006</a></p></td>
          <td>EvLn006</td>
          <td>
              <p>EvLn006</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp009">VarInterp009</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn006"><pre><code>{
  "cg:id": "EvLn006",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn006",
  "hasInformation": [
    {
      "cg:id": "VarInterp009",
      "cg:type": "VariantInterpretation",
      "explanation": "NM_001369.2:c.6249G>A Pathogenic for PCD",
      "variant": {
        "cg:id": "CanAll008",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll009",
        "identifier": "http://reg.genome.network/allele/CA501221"
      },
      "condition": [
        {
          "cg:id": "MendCond004",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:244",
          "disease": [
            {
              "cg:id": "Orphanet:244"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC012",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn007">EvLn007</a></p></td>
          <td>EvLn007</td>
          <td>
              <p>EvLn007</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp010">VarInterp010</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn007"><pre><code>{
  "cg:id": "EvLn007",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn007",
  "hasInformation": [
    {
      "cg:id": "VarInterp010",
      "cg:type": "VariantInterpretation",
      "explanation": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Noonan Syndrome 3",
      "variant": {
        "cg:id": "CanAll011",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll014",
        "identifier": "http://reg.genome.network/allele/CA256480"
      },
      "condition": [
        {
          "cg:id": "MendCond006",
          "cg:type": "MendelianCondition",
          "explanation": "OMIM:609942",
          "disease": [
            {
              "cg:id": "OMIM:609942",
              "cg:type": "Coding",
              "code": "609942",
              "system": "http://www.omim.org/",
              "display": "noonan syndrome 3"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC013",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn008">EvLn008</a></p></td>
          <td>EvLn008</td>
          <td>
              <p>EvLn008</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp011">VarInterp011</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn008"><pre><code>{
  "cg:id": "EvLn008",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn008",
  "hasInformation": [
    {
      "cg:id": "VarInterp011",
      "cg:type": "VariantInterpretation",
      "explanation": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Costello syndrome",
      "variant": {
        "cg:id": "CanAll011",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll014",
        "identifier": "http://reg.genome.network/allele/CA256480"
      },
      "condition": [
        {
          "cg:id": "MendCond055",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:648, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:648"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC014",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn009">EvLn009</a></p></td>
          <td>EvLn009</td>
          <td>
              <p>EvLn009</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond012">IndCond012</a></p><p><a href="details/details.html#FamHist013">FamHist013</a></p><p><a href="details/details.html#DeNovo014">DeNovo014</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn009"><pre><code>{
  "cg:id": "EvLn009",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn009",
  "hasInformation": [
    {
      "cg:id": "IndCond012",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv001",
        "cg:type": "Individual",
        "description": "individual #001"
      },
      "condition": [
        {
          "cg:id": "MendCond052",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:217604",
          "disease": [
            {
              "cg:id": "Orphanet:217604"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist013",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Indiv001",
        "cg:type": "Individual",
        "description": "individual #001"
      },
      "condition": [
        {
          "cg:id": "MendCond052",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:217604",
          "disease": [
            {
              "cg:id": "Orphanet:217604"
            }
          ]
        }
      ],
      "familyHasCondition": true
    },
    {
      "cg:id": "DeNovo014",
      "cg:type": "DeNovoAllele",
      "explanation": "parentage confirmed via haplotyping.",
      "canonicalAllele": {
        "cg:id": "CanAll014",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll017",
        "identifier": "http://reg.genome.network/allele/CA016441"
      },
      "individual": {
        "cg:id": "Indiv001",
        "cg:type": "Individual",
        "description": "individual #001"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn010">EvLn010</a></p></td>
          <td>EvLn010</td>
          <td>
              <p>EvLn010</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond015">IndCond015</a></p><p><a href="details/details.html#FamHist016">FamHist016</a></p><p><a href="details/details.html#DeNovo017">DeNovo017</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn010"><pre><code>{
  "cg:id": "EvLn010",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn010",
  "hasInformation": [
    {
      "cg:id": "IndCond015",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv002",
        "cg:type": "Individual",
        "description": "individual #002"
      },
      "condition": [
        {
          "cg:id": "MendCond008",
          "cg:type": "MendelianCondition",
          "explanation": "desmin related myopathy",
          "disease": [
            {
              "cg:id": "OMIM:601419",
              "cg:type": "Coding",
              "code": "601419",
              "system": "http://www.omim.org/",
              "display": "myofibrillar myopathy 1"
            }
          ],
          "name": "desmin related myopathy"
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist016",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Indiv002",
        "cg:type": "Individual",
        "description": "individual #002"
      },
      "condition": [
        {
          "cg:id": "MendCond008",
          "cg:type": "MendelianCondition",
          "explanation": "desmin related myopathy",
          "disease": [
            {
              "cg:id": "OMIM:601419",
              "cg:type": "Coding",
              "code": "601419",
              "system": "http://www.omim.org/",
              "display": "myofibrillar myopathy 1"
            }
          ],
          "name": "desmin related myopathy"
        }
      ],
      "familyHasCondition": true
    },
    {
      "cg:id": "DeNovo017",
      "cg:type": "DeNovoAllele",
      "explanation": "Non-paternity was excluded by assessment with microsatellites",
      "canonicalAllele": {
        "cg:id": "CanAll015",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll018",
        "identifier": "http://reg.genome.network/allele/CA257646"
      },
      "individual": {
        "cg:id": "Indiv002",
        "cg:type": "Individual",
        "description": "individual #002"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn011">EvLn011</a></p></td>
          <td>EvLn011</td>
          <td>
              <p>EvLn011</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond018">IndCond018</a></p><p><a href="details/details.html#FamHist019">FamHist019</a></p><p><a href="details/details.html#DeNovo020">DeNovo020</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn011"><pre><code>{
  "cg:id": "EvLn011",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn011",
  "hasInformation": [
    {
      "cg:id": "IndCond018",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv003",
        "cg:type": "Individual",
        "description": "individual #003"
      },
      "condition": [
        {
          "cg:id": "MendCond009",
          "cg:type": "MendelianCondition",
          "explanation": "Epilepsy with Mental Retardation",
          "name": "Epilepsy with Mental Retardation"
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist019",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Indiv003",
        "cg:type": "Individual",
        "description": "individual #003"
      },
      "condition": [
        {
          "cg:id": "MendCond009",
          "cg:type": "MendelianCondition",
          "explanation": "Epilepsy with Mental Retardation",
          "name": "Epilepsy with Mental Retardation"
        }
      ],
      "familyHasCondition": true
    },
    {
      "cg:id": "DeNovo020",
      "cg:type": "DeNovoAllele",
      "explanation": "Variant from trio WES, which confirmed parentage",
      "canonicalAllele": {
        "cg:id": "CanAll016",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll019",
        "identifier": "http://reg.genome.network/allele/CA354186"
      },
      "individual": {
        "cg:id": "Indiv003",
        "cg:type": "Individual",
        "description": "individual #003"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn012">EvLn012</a></p></td>
          <td>EvLn012</td>
          <td>
              <p>EvLn012</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func021">Func021</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn012"><pre><code>{
  "cg:id": "EvLn012",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn012",
  "hasInformation": [
    {
      "cg:id": "Func021",
      "cg:type": "FunctionalData",
      "result": "the mutation lowered the Ca2+  affinity of the reconstituted thin filaments;",
      "dataType": {
        "cg:id": "CC015",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:binding",
            "cg:type": "Coding",
            "code": "binding",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "Binding assay?"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll021",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll018",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll021",
          "identifier": "http://reg.genome.network/allele/CA501233"
        },
        "alleleName": "NP_001001431.1:p.Arg131Trp"
      },
      "gene": {
        "cg:id": "G011",
        "cg:type": "Gene",
        "symbol": "TNNT2"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn013">EvLn013</a></p></td>
          <td>EvLn013</td>
          <td>
              <p>EvLn013</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func022">Func022</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn013"><pre><code>{
  "cg:id": "EvLn013",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn013",
  "hasInformation": [
    {
      "cg:id": "Func022",
      "cg:type": "FunctionalData",
      "result": "reduced Ca2 sensitivity of activation",
      "dataType": {
        "cg:id": "CC016",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:atpase",
            "cg:type": "Coding",
            "code": "atpase",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "ATPase assay?"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll021",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll018",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll021",
          "identifier": "http://reg.genome.network/allele/CA501233"
        },
        "alleleName": "NP_001001431.1:p.Arg131Trp"
      },
      "gene": {
        "cg:id": "G011",
        "cg:type": "Gene",
        "symbol": "TNNT2"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn014">EvLn014</a></p></td>
          <td>EvLn014</td>
          <td>
              <p>EvLn014</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func023">Func023</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn014"><pre><code>{
  "cg:id": "EvLn014",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn014",
  "hasInformation": [
    {
      "cg:id": "Func023",
      "cg:type": "FunctionalData",
      "result": "reduced Ca2 sensitivity of activation",
      "dataType": {
        "cg:id": "CC017",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:motility",
            "cg:type": "Coding",
            "code": "motility",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "motility assay?"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll021",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll018",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll021",
          "identifier": "http://reg.genome.network/allele/CA501233"
        },
        "alleleName": "NP_001001431.1:p.Arg131Trp"
      },
      "gene": {
        "cg:id": "G011",
        "cg:type": "Gene",
        "symbol": "TNNT2"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn015">EvLn015</a></p></td>
          <td>EvLn015</td>
          <td>
              <p>EvLn015</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func024">Func024</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn015"><pre><code>{
  "cg:id": "EvLn015",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn015",
  "hasInformation": [
    {
      "cg:id": "Func024",
      "cg:type": "FunctionalData",
      "result": "desensitized Ca2+ binding to the thin filament.",
      "dataType": {
        "cg:id": "CC018",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:binding",
            "cg:type": "Coding",
            "code": "binding",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "Binding assay?"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll021",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll018",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll021",
          "identifier": "http://reg.genome.network/allele/CA501233"
        },
        "alleleName": "NP_001001431.1:p.Arg131Trp"
      },
      "gene": {
        "cg:id": "G011",
        "cg:type": "Gene",
        "symbol": "TNNT2"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn016">EvLn016</a></p></td>
          <td>EvLn016</td>
          <td>
              <p>EvLn016</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func025">Func025</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn016"><pre><code>{
  "cg:id": "EvLn016",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn016",
  "hasInformation": [
    {
      "cg:id": "Func025",
      "cg:type": "FunctionalData",
      "explanation": "Expression of human SPG7 (with this mutation) in yeast with damaged m-AAA protease did not restore respiratory growth or MrpL32 processing, if the function of AFG3L2 was impaired.",
      "result": "Functionally Damaging",
      "dataType": {
        "cg:id": "CC019",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:yeastcomp",
            "cg:type": "Coding",
            "code": "yeastcomp",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "Yeast complementation assay"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll022",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll019",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll022",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "NM_003119.2 (SPG7):c.1529C>T"
      },
      "gene": {
        "cg:id": "G012",
        "cg:type": "Gene",
        "symbol": "SPG7"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn017">EvLn017</a></p></td>
          <td>EvLn017</td>
          <td>
              <p>EvLn017</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func026">Func026</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn017"><pre><code>{
  "cg:id": "EvLn017",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn017",
  "hasInformation": [
    {
      "cg:id": "Func026",
      "cg:type": "FunctionalData",
      "result": "abnormal/deceased MEK activtation.",
      "dataType": {
        "cg:id": "CC020",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:activation",
            "cg:type": "Coding",
            "code": "activation",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "Activation Assay"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll024",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll021",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll024",
          "identifier": "http://reg.genome.network/allele/CA093846"
        },
        "alleleName": "NP_004324.2:p.Gly596Val"
      },
      "gene": {
        "cg:id": "G014",
        "cg:type": "Gene",
        "symbol": "MEK"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn018">EvLn018</a></p></td>
          <td>EvLn018</td>
          <td>
              <p>EvLn018</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func027">Func027</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn018"><pre><code>{
  "cg:id": "EvLn018",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn018",
  "hasInformation": [
    {
      "cg:id": "Func027",
      "cg:type": "FunctionalData",
      "result": "abnormal/deceased ERK activtation.",
      "dataType": {
        "cg:id": "CC021",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:activation",
            "cg:type": "Coding",
            "code": "activation",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "Activation Assay"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll024",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll021",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll024",
          "identifier": "http://reg.genome.network/allele/CA093846"
        },
        "alleleName": "NP_004324.2:p.Gly596Val"
      },
      "gene": {
        "cg:id": "G015",
        "cg:type": "Gene",
        "symbol": "ERK"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn019">EvLn019</a></p></td>
          <td>EvLn019</td>
          <td>
              <p>EvLn019</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CaseCtrl030">CaseCtrl030</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn019"><pre><code>{
  "cg:id": "EvLn019",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn019",
  "hasInformation": [
    {
      "cg:id": "CaseCtrl030",
      "cg:type": "CaseControl",
      "explanation": "Treating ExAC data as controls for",
      "canonicalAllele": {
        "cg:id": "CanAll022",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll025",
        "identifier": "http://reg.genome.network/allele/CA253316"
      },
      "condition": {
        "cg:id": "MendCond048",
        "cg:type": "MendelianCondition",
        "explanation": "Hearing Loss",
        "disease": [
          {
            "cg:id": "Orphanet:87884"
          }
        ],
        "name": "Hearing Loss"
      },
      "oddsRatio": 9.1361,
      "confidenceLevel": 99.0,
      "confidenceIntervalLower": 6.1425,
      "confidenceIntervalUpper": 13.5889,
      "caseGroupFrequency": {
        "cg:id": "AllFreq029",
        "cg:type": "AlleleFrequency",
        "ascertainment": "Across all published data (and LMM data) with hearing loss.",
        "allele": {
          "cg:id": "CanAll022",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll025",
          "identifier": "http://reg.genome.network/allele/CA253316"
        },
        "alleleCount": 32,
        "alleleNumber": 4076
      },
      "controlGroupFrequency": {
        "cg:id": "AllFreq028",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "allele": {
          "cg:id": "CanAll022",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll025",
          "identifier": "http://reg.genome.network/allele/CA253316"
        },
        "alleleCount": 105,
        "alleleNumber": 121336
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn020">EvLn020</a></p></td>
          <td>EvLn020</td>
          <td>
              <p>EvLn020</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CaseCtrl033">CaseCtrl033</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn020"><pre><code>{
  "cg:id": "EvLn020",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn020",
  "hasInformation": [
    {
      "cg:id": "CaseCtrl033",
      "cg:type": "CaseControl",
      "explanation": "Did not calculate formal odds ratio",
      "canonicalAllele": {
        "cg:id": "CanAll023",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll026",
        "identifier": "http://reg.genome.network/allele/CA024311"
      },
      "condition": {
        "cg:id": "MendCond011",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:423, HP:0000006",
        "disease": [
          {
            "cg:id": "Orphanet:423"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      },
      "caseGroupFrequency": {
        "cg:id": "AllFreq032",
        "cg:type": "AlleleFrequency",
        "explanation": "This allele is thought to account for between 2% and 10% of patients with Malignant hyperthermia.",
        "allele": {
          "cg:id": "CanAll023",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll026",
          "identifier": "http://reg.genome.network/allele/CA024311"
        }
      },
      "controlGroupFrequency": {
        "cg:id": "AllFreq031",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "allele": {
          "cg:id": "CanAll023",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll026",
          "identifier": "http://reg.genome.network/allele/CA024311"
        },
        "alleleCount": 9,
        "alleleNumber": 66740
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn021">EvLn021</a></p></td>
          <td>EvLn021</td>
          <td>
              <p>EvLn021</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CaseCtrl036">CaseCtrl036</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn021"><pre><code>{
  "cg:id": "EvLn021",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn021",
  "hasInformation": [
    {
      "cg:id": "CaseCtrl036",
      "cg:type": "CaseControl",
      "explanation": "Odds ratio is calculated by pooling samples from the six GENIUS T2D cohorts before calculation.",
      "canonicalAllele": {
        "cg:id": "CanAll024",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll027",
        "identifier": "http://reg.genome.network/allele/CA123287"
      },
      "condition": {
        "cg:id": "MendCond012",
        "cg:type": "MendelianCondition",
        "explanation": "HP:0011014",
        "phenotype": [
          {
            "cg:id": "HP:0011014",
            "cg:type": "Coding",
            "code": "0011014",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Abnormal Glucose Homeostasis"
          }
        ]
      },
      "oddsRatio": 1.06,
      "confidenceLevel": 95.0,
      "caseGroupFrequency": {
        "cg:id": "AllFreq034",
        "cg:type": "AlleleFrequency",
        "ascertainment": "Case group is composed of cases from six cohorts from the GENIUS T2D consortium.  Case status is defined per-cohort.",
        "allele": {
          "cg:id": "CanAll024",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll027",
          "identifier": "http://reg.genome.network/allele/CA123287"
        },
        "alleleCount": 1198,
        "alleleNumber": 7344,
        "individualCount": 3672,
        "homozygousAlleleIndividualCount": 113,
        "heterozygousAlleleIndividualCount": 972,
        "population": {
          "cg:id": "CC022",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "EXAC:NFE"
            }
          ]
        }
      },
      "controlGroupFrequency": {
        "cg:id": "AllFreq035",
        "cg:type": "AlleleFrequency",
        "ascertainment": "Controls group is composed of controls from six cohorts from the GENIUS T2D consortium.  Controls status is defined per-cohort.",
        "allele": {
          "cg:id": "CanAll024",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll027",
          "identifier": "http://reg.genome.network/allele/CA123287"
        },
        "alleleCount": 936,
        "alleleNumber": 5870,
        "individualCount": 2935,
        "homozygousAlleleIndividualCount": 77,
        "heterozygousAlleleIndividualCount": 782,
        "population": {
          "cg:id": "CC023",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "EXAC:NFE"
            }
          ]
        }
      },
      "confidenceIntervalLower": 0.96,
      "confidenceIntervalUpper": 1.18,
      "contribution": [
        {
          "cg:id": "Contrib480",
          "cg:type": "Contribution",
          "agent": "Harry Stevenson",
          "onDate": "9/13/16",
          "role": {
            "cg:id": "CC382",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:curator",
                "cg:type": "Coding",
                "code": "curator",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "curator"
              }
            ]
          }
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn022">EvLn022</a></p></td>
          <td>EvLn022</td>
          <td>
              <p>EvLn022</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#RegAnn040">RegAnn040</a></p><p><a href="details/details.html#RegAll380">RegAll380</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn022"><pre><code>{
  "cg:id": "EvLn022",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn022",
  "hasInformation": [
    {
      "cg:id": "RegAnn040",
      "cg:type": "RegionAnnotation",
      "explanation": "This region is the catalytic (kinase) domain.",
      "region": {
        "cg:id": "Loc379",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq002",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NP_005219.2"
          }
        },
        "start": 704,
        "stop": 1016
      },
      "isTypes": [
        {
          "cg:id": "CC027",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:functional",
              "cg:type": "Coding",
              "code": "functional",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Functional"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "RegAll380",
      "cg:type": "RegionAlleles",
      "explanation": "This region is the catalytic (kinase) domain.",
      "region": {
        "cg:id": "Loc379",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq002",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NP_005219.2"
          }
        },
        "start": 704,
        "stop": 1016
      },
      "includedAlleles": [
        {
          "cg:id": "CtxAll126",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll120",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll126",
            "identifier": "http://reg.genome.network/allele/CA126726"
          },
          "alleleName": "NP_005219.2:p.Thr790Met"
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn023">EvLn023</a></p></td>
          <td>EvLn023</td>
          <td>
              <p>EvLn023</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#RegAnn043">RegAnn043</a></p><p><a href="details/details.html#RegAll382">RegAll382</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn023"><pre><code>{
  "cg:id": "EvLn023",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn023",
  "hasInformation": [
    {
      "cg:id": "RegAnn043",
      "cg:type": "RegionAnnotation",
      "explanation": "This region is exon 15 in the transcript, and codes the serine-threonine-protein kinase catalytic domain.",
      "region": {
        "cg:id": "Loc381",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq003",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NM_004333.4"
          }
        },
        "start": 1803,
        "stop": 1921
      },
      "isTypes": [
        {
          "cg:id": "CC030",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:functional",
              "cg:type": "Coding",
              "code": "functional",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Functional"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "RegAll382",
      "cg:type": "RegionAlleles",
      "explanation": "This region is exon 15 in the transcript, and codes the serine-threonine-protein kinase catalytic domain.",
      "region": {
        "cg:id": "Loc381",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq003",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NM_004333.4"
          }
        },
        "start": 1803,
        "stop": 1921
      },
      "includedAlleles": [
        {
          "cg:id": "CtxAll029",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll026",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll029",
            "identifier": "http://reg.genome.network/allele/CA123651"
          },
          "alleleName": "NM_004333.4(BRAF):c.1789C>G"
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn024">EvLn024</a></p></td>
          <td>EvLn024</td>
          <td>
              <p>EvLn024</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#RegAnn046">RegAnn046</a></p><p><a href="details/details.html#RegAll384">RegAll384</a></p><p><a href="details/details.html#VarInterp047">VarInterp047</a></p><p><a href="details/details.html#VarInterp048">VarInterp048</a></p><p><a href="details/details.html#VarInterp049">VarInterp049</a></p><p><a href="details/details.html#VarInterp050">VarInterp050</a></p><p><a href="details/details.html#BMVR102">BMVR102</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn024"><pre><code>{
  "cg:id": "EvLn024",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn024",
  "hasInformation": [
    {
      "cg:id": "RegAnn046",
      "cg:type": "RegionAnnotation",
      "explanation": "Region(NM_0001692[996..1012]) is a mutational hotspot.",
      "region": {
        "cg:id": "Loc383",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq004",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NM_000169.2"
          }
        },
        "start": 996,
        "stop": 1012
      },
      "isTypes": [
        {
          "cg:id": "CC033",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:hotspot",
              "cg:type": "Coding",
              "code": "hotspot",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Mutational Hotspot"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "RegAll384",
      "cg:type": "RegionAlleles",
      "explanation": "The region contains the following pathogenic variants: c.886A>G, c.890C>T, c.899T>C, & c.902G>A.",
      "region": {
        "cg:id": "Loc383",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq004",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NM_000169.2"
          }
        },
        "start": 996,
        "stop": 1012
      },
      "includedAlleles": [
        {
          "cg:id": "CtxAll031",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll028",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll031",
            "identifier": "http://reg.genome.network/allele/CA022141"
          },
          "alleleName": "NM_000169.2(GLA): c.886A>G"
        },
        {
          "cg:id": "CtxAll032",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll029",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll032",
            "identifier": "http://reg.genome.network/allele/CA022153"
          },
          "alleleName": "NM_000169.2(GLA):c.890C>T"
        },
        {
          "cg:id": "CtxAll033",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll030",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll033",
            "identifier": "http://reg.genome.network/allele/CA022159"
          },
          "alleleName": "NM_000169.2(GLA):c.899T>C"
        },
        {
          "cg:id": "CtxAll034",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll031",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll034",
            "identifier": "http://reg.genome.network/allele/CA022189"
          },
          "alleleName": "NM_000169.2(GLA):c.902G>A"
        }
      ]
    },
    {
      "cg:id": "VarInterp047",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll028",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll031",
        "identifier": "http://reg.genome.network/allele/CA022141"
      },
      "condition": [
        {
          "cg:id": "MendCond013",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:324, HP:0001419",
          "disease": [
            {
              "cg:id": "Orphanet:324"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0001419",
            "cg:type": "Coding",
            "code": "0001419",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "X-linked recessive inheritance"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC034",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    },
    {
      "cg:id": "VarInterp048",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll029",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll032",
        "identifier": "http://reg.genome.network/allele/CA022153"
      },
      "condition": [
        {
          "cg:id": "MendCond013",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:324, HP:0001419",
          "disease": [
            {
              "cg:id": "Orphanet:324"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0001419",
            "cg:type": "Coding",
            "code": "0001419",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "X-linked recessive inheritance"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC035",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    },
    {
      "cg:id": "VarInterp049",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll030",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll033",
        "identifier": "http://reg.genome.network/allele/CA022159"
      },
      "condition": [
        {
          "cg:id": "MendCond013",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:324, HP:0001419",
          "disease": [
            {
              "cg:id": "Orphanet:324"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0001419",
            "cg:type": "Coding",
            "code": "0001419",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "X-linked recessive inheritance"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC036",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    },
    {
      "cg:id": "VarInterp050",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll031",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll034",
        "identifier": "http://reg.genome.network/allele/CA022189"
      },
      "condition": [
        {
          "cg:id": "MendCond013",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:324, HP:0001419",
          "disease": [
            {
              "cg:id": "Orphanet:324"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0001419",
            "cg:type": "Coding",
            "code": "0001419",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "X-linked recessive inheritance"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC037",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    },
    {
      "cg:id": "BMVR102",
      "cg:type": "BenignMissenseVariationRate",
      "explanation": "ClinVar shows no benign missense variants in (or near) this region.  The closest is NM_000169.2:c.8T>C",
      "region": {
        "cg:id": "Loc383",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq004",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NM_000169.2"
          }
        },
        "start": 996,
        "stop": 1012
      },
      "value": {
        "cg:id": "CC069",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-ben-missense-var-rate:low",
            "cg:type": "Coding",
            "code": "low",
            "system": "http://clinicalgenome.org/datamodel/benign-missense-variation-rate/",
            "display": "Low"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn025">EvLn025</a></p></td>
          <td>EvLn025</td>
          <td>
              <p>EvLn025</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq051">AllFreq051</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn025"><pre><code>{
  "cg:id": "EvLn025",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn025",
  "hasInformation": [
    {
      "cg:id": "AllFreq051",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC038",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "EXAC:NFE"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll032",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll035",
        "identifier": "http://reg.genome.network/allele/CA011797"
      },
      "alleleCount": 66706,
      "alleleNumber": 1,
      "alleleFrequency": 1.499e-05,
      "homozygousAlleleIndividualCount": 0
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn026">EvLn026</a></p></td>
          <td>EvLn026</td>
          <td>
              <p>EvLn026</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq052">AllFreq052</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn026"><pre><code>{
  "cg:id": "EvLn026",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn026",
  "hasInformation": [
    {
      "cg:id": "AllFreq052",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC039",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll032",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll035",
        "identifier": "http://reg.genome.network/allele/CA011797"
      },
      "alleleCount": 121366,
      "alleleNumber": 1,
      "alleleFrequency": 8.24e-06,
      "homozygousAlleleIndividualCount": 0
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn027">EvLn027</a></p></td>
          <td>EvLn027</td>
          <td>
              <p>EvLn027</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq053">AllFreq053</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn027"><pre><code>{
  "cg:id": "EvLn027",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn027",
  "hasInformation": [
    {
      "cg:id": "AllFreq053",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC040",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "EXAC:FIN"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll033",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll037",
        "identifier": "http://reg.genome.network/allele/CA011797"
      },
      "alleleCount": 6614,
      "alleleNumber": 514,
      "alleleFrequency": 0.07771,
      "homozygousAlleleIndividualCount": 21,
      "medianCoverage": 100.0
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn028">EvLn028</a></p></td>
          <td>EvLn028</td>
          <td>
              <p>EvLn028</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq054">AllFreq054</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn028"><pre><code>{
  "cg:id": "EvLn028",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn028",
  "hasInformation": [
    {
      "cg:id": "AllFreq054",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC041",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll034",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll038",
        "identifier": "http://reg.genome.network/allele/CA273141"
      },
      "alleleCount": 0,
      "alleleNumber": 0,
      "alleleFrequency": 0.0,
      "medianCoverage": 11.0
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn029">EvLn029</a></p></td>
          <td>EvLn029</td>
          <td>
              <p>EvLn029</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond056">IndCond056</a></p><p><a href="details/details.html#VarInterp057">VarInterp057</a></p><p><a href="details/details.html#IndAll058">IndAll058</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn029"><pre><code>{
  "cg:id": "EvLn029",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn029",
  "hasInformation": [
    {
      "cg:id": "IndCond056",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv004",
        "cg:type": "Individual",
        "description": "individual #004"
      },
      "condition": [
        {
          "cg:id": "MendCond014",
          "cg:type": "MendelianCondition",
          "explanation": "PCD autosomal recessive w/ outer dynein arm (ODA) defects",
          "name": "PCD autosomal recessive w/ outer dynein arm (ODA) defects",
          "disease": [
            {
              "cg:id": "Orphanet:244"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          },
          "phenotype": [
            {
              "cg:id": "HP:0200106",
              "cg:type": "Coding",
              "code": "0200106",
              "system": "http://human-phenotype-ontology.github.io/",
              "display": "Absent/shortened dynein arms"
            }
          ]
        }
      ],
      "hasCondition": true,
      "contribution": [
        {
          "cg:id": "Contrib483",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "5/31/16 8:46 AM",
          "role": {
            "cg:id": "CC385",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:curator",
                "cg:type": "Coding",
                "code": "curator",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "curator"
              }
            ]
          }
        }
      ]
    },
    {
      "cg:id": "VarInterp057",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll036",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll040",
        "identifier": "http://reg.genome.network/allele/CA273615"
      },
      "condition": [
        {
          "cg:id": "MendCond062",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:244, HP:0000007",
          "disease": [
            {
              "cg:id": "Orphanet:244"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC043",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      },
      "contribution": [
        {
          "cg:id": "Contrib485",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "5/31/16 11:01 AM",
          "role": {
            "cg:id": "CC387",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:interpreter",
                "cg:type": "Coding",
                "code": "interpreter",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "interpreter"
              }
            ]
          }
        }
      ]
    },
    {
      "cg:id": "IndAll058",
      "cg:type": "IndividualAllele",
      "explanation": "Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband.",
      "individual": {
        "cg:id": "Indiv004",
        "cg:type": "Individual",
        "description": "individual #004"
      },
      "primaryAllele": {
        "cg:id": "CanAll035",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll039",
        "identifier": "http://reg.genome.network/allele/CA090919"
      },
      "secondaryAllele": {
        "cg:id": "CanAll036",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll040",
        "identifier": "http://reg.genome.network/allele/CA273615"
      },
      "primaryZygosity": {
        "cg:id": "CC044",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      },
      "secondaryZygosity": {
        "cg:id": "CC045",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      },
      "phase": {
        "cg:id": "CC046",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-phase:trans",
            "cg:type": "Coding",
            "code": "trans",
            "system": "http://clinicalgenome.org/datamodel/phase/",
            "display": "in trans"
          }
        ]
      },
      "contribution": [
        {
          "cg:id": "Contrib484",
          "cg:type": "Contribution",
          "agent": "Harry Stevenson",
          "onDate": "5/31/16 9:15 AM",
          "role": {
            "cg:id": "CC386",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:curator",
                "cg:type": "Coding",
                "code": "curator",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "curator"
              }
            ]
          }
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn030">EvLn030</a></p></td>
          <td>EvLn030</td>
          <td>
              <p>EvLn030</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp059">VarInterp059</a></p><p><a href="details/details.html#IndCond061">IndCond061</a></p><p><a href="details/details.html#IndAll062">IndAll062</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn030"><pre><code>{
  "cg:id": "EvLn030",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn030",
  "hasInformation": [
    {
      "cg:id": "VarInterp059",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll038",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll042",
        "identifier": "http://reg.genome.network/allele/CA501223"
      },
      "condition": [
        {
          "cg:id": "MendCond010",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:87884, HP:0000007, GIPC3",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          },
          "gene": {
            "cg:id": "G024",
            "cg:type": "Gene",
            "symbol": "GIPC3"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC047",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond061",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv005",
        "cg:type": "Individual",
        "description": "individual #005"
      },
      "condition": [
        {
          "cg:id": "MendCond010",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:87884, HP:0000007, GIPC3",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          },
          "gene": {
            "cg:id": "G024",
            "cg:type": "Gene",
            "symbol": "GIPC3"
          }
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "IndAll062",
      "cg:type": "IndividualAllele",
      "explanation": "Based on the NGS data, we are able to determine that c.298G>A and c.411+1G>A are in trans as no NGS reads carry both variants.",
      "individual": {
        "cg:id": "Indiv005",
        "cg:type": "Individual",
        "description": "individual #005"
      },
      "primaryAllele": {
        "cg:id": "CanAll037",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll041",
        "identifier": "http://reg.genome.network/allele/CA176145"
      },
      "secondaryAllele": {
        "cg:id": "CanAll038",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll042",
        "identifier": "http://reg.genome.network/allele/CA501223"
      },
      "primaryZygosity": {
        "cg:id": "CC049",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      },
      "secondaryZygosity": {
        "cg:id": "CC050",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      },
      "phase": {
        "cg:id": "CC051",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-phase:trans",
            "cg:type": "Coding",
            "code": "trans",
            "system": "http://clinicalgenome.org/datamodel/phase/",
            "display": "in trans"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn031">EvLn031</a></p></td>
          <td>EvLn031</td>
          <td>
              <p>EvLn031</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond064">IndCond064</a></p><p><a href="details/details.html#IndAll065">IndAll065</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn031"><pre><code>{
  "cg:id": "EvLn031",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn031",
  "hasInformation": [
    {
      "cg:id": "IndCond064",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv006",
        "cg:type": "Individual",
        "description": "individual #006"
      },
      "condition": [
        {
          "cg:id": "MendCond015",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:216, HP:0000007, TTP1",
          "disease": [
            {
              "cg:id": "Orphanet:216"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          },
          "gene": {
            "cg:id": "G025",
            "cg:type": "Gene",
            "symbol": "TTP1"
          }
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "IndAll065",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv006",
        "cg:type": "Individual",
        "description": "individual #006"
      },
      "primaryAllele": {
        "cg:id": "CanAll039",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll043",
        "identifier": "http://reg.genome.network/allele/CA090904"
      },
      "primaryZygosity": {
        "cg:id": "CC053",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:hom",
            "cg:type": "Coding",
            "code": "hom",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "homozygous"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn032">EvLn032</a></p></td>
          <td>EvLn032</td>
          <td>
              <p>EvLn032</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon066">MolCon066</a></p><p><a href="details/details.html#RegAnn069">RegAnn069</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn032"><pre><code>{
  "cg:id": "EvLn032",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn032",
  "hasInformation": [
    {
      "cg:id": "MolCon066",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll044",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll040",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll044",
          "identifier": "http://reg.genome.network/allele/CA273136"
        },
        "alleleName": "NP_001360.1:p.Trp2490_Leu2496del"
      },
      "consequence": {
        "cg:id": "CC054",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001825",
            "cg:type": "Coding",
            "code": "0001825",
            "system": "http://sequenceontology.org/",
            "display": "Conservative Inframe Deletion"
          }
        ]
      },
      "contribution": [
        {
          "cg:id": "Contrib486",
          "cg:type": "Contribution",
          "agent": "Harry Stevenson",
          "onDate": "6/1/16 2:03 PM",
          "role": {
            "cg:id": "CC388",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:curator",
                "cg:type": "Coding",
                "code": "curator",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "curator"
              }
            ]
          }
        }
      ]
    },
    {
      "cg:id": "RegAnn069",
      "cg:type": "RegionAnnotation",
      "explanation": "This is not a repreating region.",
      "region": {
        "cg:id": "Loc385",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq005",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NP_001360.1"
          }
        },
        "start": 2490,
        "stop": 2496
      },
      "isTypes": [
        {
          "cg:id": "CC057",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:repeat",
              "cg:type": "Coding",
              "code": "repeat",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Repeat"
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib487",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "6/1/16 2:10 PM",
          "role": {
            "cg:id": "CC389",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:curator",
                "cg:type": "Coding",
                "code": "curator",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "curator"
              }
            ]
          }
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn033">EvLn033</a></p></td>
          <td>EvLn033</td>
          <td>
              <p>EvLn033</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon070">MolCon070</a></p><p><a href="details/details.html#RegAnn073">RegAnn073</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn033"><pre><code>{
  "cg:id": "EvLn033",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn033",
  "hasInformation": [
    {
      "cg:id": "MolCon070",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll046",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll042",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll046",
          "identifier": "http://reg.genome.network/allele/CA322149"
        },
        "alleleName": "NP_056375.2:p.Arg38_Ser43del"
      },
      "consequence": {
        "cg:id": "CC058",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001825",
            "cg:type": "Coding",
            "code": "0001825",
            "system": "http://sequenceontology.org/",
            "display": "Conservative Inframe Deletion"
          }
        ]
      }
    },
    {
      "cg:id": "RegAnn073",
      "cg:type": "RegionAnnotation",
      "explanation": "NP_056375.2[35..43] is not repeating.",
      "region": {
        "cg:id": "Loc386",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq006",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NP_056375.2"
          }
        },
        "start": 35,
        "stop": 43
      },
      "isNotTypes": [
        {
          "cg:id": "CC059",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:repeat",
              "cg:type": "Coding",
              "code": "repeat",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Repeat"
            }
          ]
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn034">EvLn034</a></p></td>
          <td>EvLn034</td>
          <td>
              <p>EvLn034</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon074">MolCon074</a></p><p><a href="details/details.html#RegAnn077">RegAnn077</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn034"><pre><code>{
  "cg:id": "EvLn034",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn034",
  "hasInformation": [
    {
      "cg:id": "MolCon074",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll048",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll044",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll048",
          "identifier": "http://reg.genome.network/allele/CA501225"
        },
        "alleleName": "NP_003915.2:p.Ala260_Gly261insAlaAlaAlaAlaAla"
      },
      "consequence": {
        "cg:id": "CC060",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001823",
            "cg:type": "Coding",
            "code": "0001823",
            "system": "http://sequenceontology.org/",
            "display": "Conservative Inframe Insertion"
          }
        ]
      }
    },
    {
      "cg:id": "RegAnn077",
      "cg:type": "RegionAnnotation",
      "explanation": "NP_003915.2[241..260] is repeating.",
      "region": {
        "cg:id": "Loc387",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq007",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NP_003915.2"
          }
        },
        "start": 241,
        "stop": 260
      },
      "isTypes": [
        {
          "cg:id": "CC062",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:repeat",
              "cg:type": "Coding",
              "code": "repeat",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Repeat"
            }
          ]
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn035">EvLn035</a></p></td>
          <td>EvLn035</td>
          <td>
              <p>EvLn035</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp078">VarInterp078</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn035"><pre><code>{
  "cg:id": "EvLn035",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn035",
  "hasInformation": [
    {
      "cg:id": "VarInterp078",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll047",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll051",
        "identifier": "http://reg.genome.network/allele/CA021749"
      },
      "condition": [
        {
          "cg:id": "MendCond016",
          "cg:type": "MendelianCondition",
          "explanation": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant",
          "disease": [
            {
              "cg:id": "Orphanet:217569"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          },
          "name": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant"
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC063",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA26332-9",
            "cg:type": "Coding",
            "code": "LA26332-9",
            "system": "http://loinc.org/",
            "display": "Likely Pathogenic"
          }
        ]
      },
      "contribution": [
        {
          "cg:id": "Contrib520",
          "cg:type": "Contribution",
          "agent": "Genetics Diagnostic Laboratory,Children's Hospital of Eastern Ontario",
          "onDate": "6/1/2014 9:02 AM",
          "role": {
            "cg:id": "CC422",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:interpreter",
                "cg:type": "Coding",
                "code": "interpreter",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "interpreter"
              }
            ]
          }
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn036">EvLn036</a></p></td>
          <td>EvLn036</td>
          <td>
              <p>EvLn036</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp079">VarInterp079</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn036"><pre><code>{
  "cg:id": "EvLn036",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn036",
  "hasInformation": [
    {
      "cg:id": "VarInterp079",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll049",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll053",
        "identifier": "http://reg.genome.network/allele/CA021738"
      },
      "condition": [
        {
          "cg:id": "MendCond016",
          "cg:type": "MendelianCondition",
          "explanation": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant",
          "disease": [
            {
              "cg:id": "Orphanet:217569"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          },
          "name": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant"
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC064",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      },
      "contribution": [
        {
          "cg:id": "Contrib521",
          "cg:type": "Contribution",
          "agent": "GeneDx",
          "onDate": "6/20/2016 11:12 AM",
          "role": {
            "cg:id": "CC423",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:interpreter",
                "cg:type": "Coding",
                "code": "interpreter",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "interpreter"
              }
            ]
          }
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn037">EvLn037</a></p></td>
          <td>EvLn037</td>
          <td>
              <p>EvLn037</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp042">VarInterp042</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn037"><pre><code>{
  "cg:id": "EvLn037",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn037",
  "hasInformation": [
    {
      "cg:id": "VarInterp042",
      "cg:type": "VariantInterpretation",
      "explanation": "Uncertain Sig(not specified) NM_000363.4(TNNI3):c.484C>T (p.Arg162Trp)",
      "variant": {
        "cg:id": "CanAll049",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll053",
        "identifier": "http://reg.genome.network/allele/CA021738"
      },
      "clinicalSignificance": {
        "cg:id": "CC029",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA26333-7",
            "cg:type": "Coding",
            "code": "LA26333-7",
            "system": "http://loinc.org/",
            "display": "Uncertain Signficance"
          }
        ]
      },
      "contribution": [
        {
          "cg:id": "Contrib434",
          "cg:type": "Contribution",
          "agent": "Harry Stevenson",
          "onDate": "7/13/16 8:46 AM",
          "role": {
            "cg:id": "CC336",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn038">EvLn038</a></p></td>
          <td>EvLn038</td>
          <td>
              <p>EvLn038</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp080">VarInterp080</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn038"><pre><code>{
  "cg:id": "EvLn038",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn038",
  "hasInformation": [
    {
      "cg:id": "VarInterp080",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll053",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll057",
        "identifier": "http://reg.genome.network/allele/CA224454"
      },
      "condition": [
        {
          "cg:id": "MendCond017",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:664",
          "disease": [
            {
              "cg:id": "Orphanet:664"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC065",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn039">EvLn039</a></p></td>
          <td>EvLn039</td>
          <td>
              <p>EvLn039</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp081">VarInterp081</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn039"><pre><code>{
  "cg:id": "EvLn039",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn039",
  "hasInformation": [
    {
      "cg:id": "VarInterp081",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll057",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll061",
        "identifier": "http://reg.genome.network/allele/CA003449"
      },
      "condition": [
        {
          "cg:id": "MendCond050",
          "cg:type": "MendelianCondition",
          "explanation": "Breast Ovarian Cancer, Autosomal Dominant",
          "name": "Breast Ovarian Cancer, Autosomal Dominant",
          "disease": [
            {
              "cg:id": "MESH:D061325",
              "cg:type": "Coding",
              "code": "D061325",
              "system": "https://meshb.nlm.nih.gov/",
              "display": "hereditary breast ovarian cancer"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC066",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn040">EvLn040</a></p></td>
          <td>EvLn040</td>
          <td>
              <p>EvLn040</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq082">AllFreq082</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn040"><pre><code>{
  "cg:id": "EvLn040",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn040",
  "hasInformation": [
    {
      "cg:id": "AllFreq082",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC067",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll035",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll039",
        "identifier": "http://reg.genome.network/allele/CA090919"
      },
      "alleleCount": 0,
      "alleleFrequency": 0.0,
      "medianCoverage": 11.0,
      "contribution": [
        {
          "cg:id": "Contrib488",
          "cg:type": "Contribution",
          "agent": "Harry Stevenson",
          "onDate": "6/1/16 2:15 PM",
          "role": {
            "cg:id": "CC390",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:curator",
                "cg:type": "Coding",
                "code": "curator",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "curator"
              }
            ]
          }
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn041">EvLn041</a></p></td>
          <td>EvLn041</td>
          <td>
              <p>EvLn041</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond083">IndCond083</a></p><p><a href="details/details.html#FamHist085">FamHist085</a></p><p><a href="details/details.html#DeNovo087">DeNovo087</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn041"><pre><code>{
  "cg:id": "EvLn041",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn041",
  "hasInformation": [
    {
      "cg:id": "IndCond083",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Ind007"
      },
      "condition": [
        {
          "cg:id": "MendCond018",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:122",
          "disease": [
            {
              "cg:id": "Orphanet:122"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist085",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Ind007"
      },
      "condition": [
        {
          "cg:id": "MendCond018",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:122",
          "disease": [
            {
              "cg:id": "Orphanet:122"
            }
          ]
        }
      ],
      "familyHasCondition": true
    },
    {
      "cg:id": "DeNovo087",
      "cg:type": "DeNovoAllele",
      "explanation": "parentage confirmation not mentioned in the source article",
      "canonicalAllele": {
        "cg:id": "CanAll059",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll063",
        "identifier": "http://reg.genome.network/allele/CA090951"
      },
      "individual": {
        "cg:id": "Ind007"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn042">EvLn042</a></p></td>
          <td>EvLn042</td>
          <td>
              <p>EvLn042</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond084">IndCond084</a></p><p><a href="details/details.html#FamHist086">FamHist086</a></p><p><a href="details/details.html#DeNovo088">DeNovo088</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn042"><pre><code>{
  "cg:id": "EvLn042",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn042",
  "hasInformation": [
    {
      "cg:id": "IndCond084",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Ind008"
      },
      "condition": [
        {
          "cg:id": "MendCond018",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:122",
          "disease": [
            {
              "cg:id": "Orphanet:122"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist086",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Ind008"
      },
      "condition": [
        {
          "cg:id": "MendCond018",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:122",
          "disease": [
            {
              "cg:id": "Orphanet:122"
            }
          ]
        }
      ],
      "familyHasCondition": true
    },
    {
      "cg:id": "DeNovo088",
      "cg:type": "DeNovoAllele",
      "explanation": "parentage confirmation not mentioned in the source article",
      "canonicalAllele": {
        "cg:id": "CanAll059",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll063",
        "identifier": "http://reg.genome.network/allele/CA090951"
      },
      "individual": {
        "cg:id": "Ind008"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn043">EvLn043</a></p></td>
          <td>EvLn043</td>
          <td>
              <p>EvLn043</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond089">IndCond089</a></p><p><a href="details/details.html#FamHist090">FamHist090</a></p><p><a href="details/details.html#DeNovo091">DeNovo091</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn043"><pre><code>{
  "cg:id": "EvLn043",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn043",
  "hasInformation": [
    {
      "cg:id": "IndCond089",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Ind009"
      },
      "condition": [
        {
          "cg:id": "MendCond019",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist090",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Ind009"
      },
      "condition": [
        {
          "cg:id": "MendCond019",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "familyHasCondition": true
    },
    {
      "cg:id": "DeNovo091",
      "cg:type": "DeNovoAllele",
      "explanation": "parentage confirmation not mentioned in the source article",
      "canonicalAllele": {
        "cg:id": "CanAll060",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll064",
        "identifier": "http://reg.genome.network/allele/CA279966"
      },
      "individual": {
        "cg:id": "Ind009"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn044">EvLn044</a></p></td>
          <td>EvLn044</td>
          <td>
              <p>EvLn044</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond092">IndCond092</a></p><p><a href="details/details.html#FamHist093">FamHist093</a></p><p><a href="details/details.html#DeNovo094">DeNovo094</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn044"><pre><code>{
  "cg:id": "EvLn044",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn044",
  "hasInformation": [
    {
      "cg:id": "IndCond092",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Ind010"
      },
      "condition": [
        {
          "cg:id": "MendCond068",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist093",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Ind010"
      },
      "condition": [
        {
          "cg:id": "MendCond068",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "familyHasCondition": true
    },
    {
      "cg:id": "DeNovo094",
      "cg:type": "DeNovoAllele",
      "canonicalAllele": {
        "cg:id": "CanAll061",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll065",
        "identifier": "http://reg.genome.network/allele/CA279976"
      },
      "individual": {
        "cg:id": "Ind010"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn045">EvLn045</a></p></td>
          <td>EvLn045</td>
          <td>
              <p>EvLn045</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond095">IndCond095</a></p><p><a href="details/details.html#FamHist096">FamHist096</a></p><p><a href="details/details.html#DeNovo097">DeNovo097</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn045"><pre><code>{
  "cg:id": "EvLn045",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn045",
  "hasInformation": [
    {
      "cg:id": "IndCond095",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Ind011"
      },
      "condition": [
        {
          "cg:id": "MendCond068",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist096",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Ind011"
      },
      "condition": [
        {
          "cg:id": "MendCond068",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "familyHasCondition": true
    },
    {
      "cg:id": "DeNovo097",
      "cg:type": "DeNovoAllele",
      "canonicalAllele": {
        "cg:id": "CanAll061",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll065",
        "identifier": "http://reg.genome.network/allele/CA279976"
      },
      "individual": {
        "cg:id": "Ind011"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn046">EvLn046</a></p></td>
          <td>EvLn046</td>
          <td>
              <p>EvLn046</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond098">IndCond098</a></p><p><a href="details/details.html#FamHist099">FamHist099</a></p><p><a href="details/details.html#DeNovo100">DeNovo100</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn046"><pre><code>{
  "cg:id": "EvLn046",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn046",
  "hasInformation": [
    {
      "cg:id": "IndCond098",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Ind012"
      },
      "condition": [
        {
          "cg:id": "MendCond068",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist099",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Ind012"
      },
      "condition": [
        {
          "cg:id": "MendCond068",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "familyHasCondition": true
    },
    {
      "cg:id": "DeNovo100",
      "cg:type": "DeNovoAllele",
      "canonicalAllele": {
        "cg:id": "CanAll061",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll065",
        "identifier": "http://reg.genome.network/allele/CA279976"
      },
      "individual": {
        "cg:id": "Ind012"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn047">EvLn047</a></p></td>
          <td>EvLn047</td>
          <td>
              <p>EvLn047</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AggSegData105">AggSegData105</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn047"><pre><code>{
  "cg:id": "EvLn047",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn047",
  "hasInformation": [
    {
      "cg:id": "AggSegData105",
      "cg:type": "AggregateSegregationData",
      "condition": {
        "cg:id": "MendCond020",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:217569, HP:0000006",
        "disease": [
          {
            "cg:id": "Orphanet:217569"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "cg:id": "CanAll063",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll067",
        "identifier": "http://reg.genome.network/allele/CA010136"
      },
      "familySegregationData": [
        {
          "cg:id": "FamSegData103",
          "cg:type": "FamilySegregationData",
          "explanation": "identified in 1 individual with HCM. Segregation analysis determined 5 other members of the family also carried this variant. However, only 3 of those family members had HCM (aged 48-75). The 2 other family members were young and asymptomatic. So from this paper, only count 3 segregations. The two apparently refuting segregations can be ignored due to the age-dependent penetrance of HCM.",
          "condition": {
            "cg:id": "MendCond020",
            "cg:type": "MendelianCondition",
            "explanation": "Orphanet:217569, HP:0000006",
            "disease": [
              {
                "cg:id": "Orphanet:217569"
              }
            ],
            "modeOfInheritance": {
              "cg:id": "HP:0000006",
              "cg:type": "Coding",
              "code": "0000006",
              "system": "http://human-phenotype-ontology.github.io/",
              "display": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "cg:id": "CanAll063",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll067",
            "identifier": "http://reg.genome.network/allele/CA010136"
          },
          "family": {
            "cg:id": "Fam001",
            "cg:type": "Family",
            "label": "12A",
            "description": "Family #001"
          },
          "phenotypePositiveAllelePositive": 3,
          "phenotypeNegativeAllelePositive": 2,
          "contribution": [
            {
              "cg:id": "Contrib481",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": "9/13/16",
              "role": {
                "cg:id": "CC383",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-contributory-role:curator",
                    "cg:type": "Coding",
                    "code": "curator",
                    "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                    "display": "curator"
                  }
                ]
              }
            }
          ]
        },
        {
          "cg:id": "FamSegData104",
          "cg:type": "FamilySegregationData",
          "explanation": "In LMM internal family, c.1351+1G>A was identified in 1 individual with HCM. Variant also found in individuals sister - who also had clinical dx of HCM.",
          "condition": {
            "cg:id": "MendCond020",
            "cg:type": "MendelianCondition",
            "explanation": "Orphanet:217569, HP:0000006",
            "disease": [
              {
                "cg:id": "Orphanet:217569"
              }
            ],
            "modeOfInheritance": {
              "cg:id": "HP:0000006",
              "cg:type": "Coding",
              "code": "0000006",
              "system": "http://human-phenotype-ontology.github.io/",
              "display": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "cg:id": "CanAll063",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll067",
            "identifier": "http://reg.genome.network/allele/CA010136"
          },
          "family": {
            "cg:id": "Fam002",
            "cg:type": "Family",
            "description": "Family #002"
          },
          "phenotypePositiveAllelePositive": 1,
          "contribution": [
            {
              "cg:id": "Contrib482",
              "cg:type": "Contribution",
              "agent": "Harry Stevenson",
              "onDate": "9/13/16",
              "role": {
                "cg:id": "CC384",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-contributory-role:curator",
                    "cg:type": "Coding",
                    "code": "curator",
                    "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                    "display": "curator"
                  }
                ]
              }
            }
          ]
        }
      ],
      "totalNumberOfSegregations": 4
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn048">EvLn048</a></p></td>
          <td>EvLn048</td>
          <td>
              <p>EvLn048</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AggSegData109">AggSegData109</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn048"><pre><code>{
  "cg:id": "EvLn048",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn048",
  "hasInformation": [
    {
      "cg:id": "AggSegData109",
      "cg:type": "AggregateSegregationData",
      "condition": {
        "cg:id": "MendCond021",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:886, HP:0000007",
        "disease": [
          {
            "cg:id": "Orphanet:886"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000007",
          "cg:type": "Coding",
          "code": "0000007",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Recessive"
        }
      },
      "canonicalAllele": {
        "cg:id": "CanAll062",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll066",
        "identifier": "http://reg.genome.network/allele/CA143213"
      },
      "familySegregationData": [
        {
          "cg:id": "FamSegData107",
          "cg:type": "FamilySegregationData",
          "explanation": "proband and 5 affected family members all homozygous for NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly).",
          "condition": {
            "cg:id": "MendCond021",
            "cg:type": "MendelianCondition",
            "explanation": "Orphanet:886, HP:0000007",
            "disease": [
              {
                "cg:id": "Orphanet:886"
              }
            ],
            "modeOfInheritance": {
              "cg:id": "HP:0000007",
              "cg:type": "Coding",
              "code": "0000007",
              "system": "http://human-phenotype-ontology.github.io/",
              "display": "Autosomal Recessive"
            }
          },
          "canonicalAllele": {
            "cg:id": "CanAll062",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll066",
            "identifier": "http://reg.genome.network/allele/CA143213"
          },
          "family": {
            "cg:id": "Fam003",
            "cg:type": "Family",
            "description": "Family #003"
          },
          "phenotypePositiveAllelePositive": 5
        },
        {
          "cg:id": "FamSegData108",
          "cg:type": "FamilySegregationData",
          "explanation": "c.1000C>T identified in het state in 1 individual with Usher. Individual compound het with Gly1840Val USH2A variant - confirmed in trans. Gly1840Val called Pathogenic. Individual has 3 affected sibs all of whom are compound het with 1000C>T & Gly1840Val. One additional sib only carried 1000C>T and is unaffected.",
          "condition": {
            "cg:id": "MendCond021",
            "cg:type": "MendelianCondition",
            "explanation": "Orphanet:886, HP:0000007",
            "disease": [
              {
                "cg:id": "Orphanet:886"
              }
            ],
            "modeOfInheritance": {
              "cg:id": "HP:0000007",
              "cg:type": "Coding",
              "code": "0000007",
              "system": "http://human-phenotype-ontology.github.io/",
              "display": "Autosomal Recessive"
            }
          },
          "canonicalAllele": {
            "cg:id": "CanAll062",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll066",
            "identifier": "http://reg.genome.network/allele/CA143213"
          },
          "family": {
            "cg:id": "Fam004",
            "cg:type": "Family",
            "description": "Family #004"
          }
        }
      ],
      "totalNumberOfSegregations": 8
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn049">EvLn049</a></p></td>
          <td>EvLn049</td>
          <td>
              <p>EvLn049</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#FamSegData111">FamSegData111</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn049"><pre><code>{
  "cg:id": "EvLn049",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn049",
  "hasInformation": [
    {
      "cg:id": "FamSegData111",
      "cg:type": "FamilySegregationData",
      "condition": {
        "cg:id": "MendCond051",
        "cg:type": "MendelianCondition",
        "explanation": "Breast Ovarian Cancer, Autosomal Dominant",
        "name": "Breast Ovarian Cancer, Autosomal Dominant",
        "disease": [
          {
            "cg:id": "MESH:D061325",
            "cg:type": "Coding",
            "code": "D061325",
            "system": "https://meshb.nlm.nih.gov/",
            "display": "hereditary breast ovarian cancer"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "cg:id": "CanAll064",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll068",
        "identifier": "http://reg.genome.network/allele/CA501226"
      },
      "family": {
        "cg:id": "Fam005",
        "cg:type": "Family",
        "description": "Family #005"
      },
      "phenotypePositiveAllelePositive": 2
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn050">EvLn050</a></p></td>
          <td>EvLn050</td>
          <td>
              <p>EvLn050</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AggSegData115">AggSegData115</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn050"><pre><code>{
  "cg:id": "EvLn050",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn050",
  "hasInformation": [
    {
      "cg:id": "AggSegData115",
      "cg:type": "AggregateSegregationData",
      "canonicalAllele": {
        "cg:id": "CanAll065",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll069",
        "identifier": "http://reg.genome.network/allele/CA006429"
      },
      "condition": {
        "cg:id": "MendCond002",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:217604, HP:0000006",
        "disease": [
          {
            "cg:id": "Orphanet:217604"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      },
      "familySegregationData": [
        {
          "cg:id": "FamSegData113",
          "cg:type": "FamilySegregationData",
          "condition": {
            "cg:id": "MendCond002",
            "cg:type": "MendelianCondition",
            "explanation": "Orphanet:217604, HP:0000006",
            "disease": [
              {
                "cg:id": "Orphanet:217604"
              }
            ],
            "modeOfInheritance": {
              "cg:id": "HP:0000006",
              "cg:type": "Coding",
              "code": "0000006",
              "system": "http://human-phenotype-ontology.github.io/",
              "display": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "cg:id": "CanAll065",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll069",
            "identifier": "http://reg.genome.network/allele/CA006429"
          },
          "family": {
            "cg:id": "Fam006",
            "cg:type": "Family",
            "description": "Family #006"
          },
          "phenotypePositiveAllelePositive": 1,
          "phenotypeNegativeAllelePositive": 1,
          "columns": [
            "[Family,Individual,Father,Mother,Sex,Affected,Genotype]"
          ],
          "affectedValues": [
            "{1: Unaffected, 2: Affected, 3: Any Cardiovascular Abnormality}"
          ],
          "genotypeValues": [
            "{0: Unknown,1:Reference, 2: CA65}"
          ],
          "pedigree": [
            "[[Fam006,Fam006.1,0,0,1,2,0/0],\n   [Fam006,Fam006.2,0,0,2,1,0/0],\n   [Fam006,Fam006.3,Fam006.1,Fam006.2,2,3,0/0],\n   [Fam006,Fam006.4,Fam006.1,Fam006.2,1,2,1/2],\n   [Fam006,Fam006.5,Fam006.1,Fam006.2,2,1,1/1],\n   [Fam006,Fam006.6,Fam006.1,Fam006.2,1,2,1/2]]"
          ]
        },
        {
          "cg:id": "FamSegData114",
          "cg:type": "FamilySegregationData",
          "condition": {
            "cg:id": "MendCond002",
            "cg:type": "MendelianCondition",
            "explanation": "Orphanet:217604, HP:0000006",
            "disease": [
              {
                "cg:id": "Orphanet:217604"
              }
            ],
            "modeOfInheritance": {
              "cg:id": "HP:0000006",
              "cg:type": "Coding",
              "code": "0000006",
              "system": "http://human-phenotype-ontology.github.io/",
              "display": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "cg:id": "CanAll065",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll069",
            "identifier": "http://reg.genome.network/allele/CA006429"
          },
          "family": {
            "cg:id": "Fam007",
            "cg:type": "Family",
            "description": "Family #007"
          },
          "phenotypePositiveAllelePositive": 2,
          "phenotypeNegativeAllelePositive": 3,
          "columns": [
            "[Family,Individual,Father,Mother,Sex,Affected,Genotype,ObligateGenotype]"
          ],
          "affectedValues": [
            "{1: Unaffected, 2: Affected, 3: Any Cardiovascular Abnormality}"
          ],
          "genotypeValues": [
            "{0: Unknown,1:Reference, 2: CA65}"
          ],
          "pedigree": [
            "[[Fam007, Fam007.1, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.2, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.3, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.4, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.5, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.6, Fam007.1, Fam007.2, 2, 3, 1/2, 1],\n   [Fam007, Fam007.7,0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.8, Fam007.1, Fam007.2, 2, 1, 1/2, 1],\n   [Fam007, Fam007.9,Fam007.3, Fam007.4, 1, 1, 0/0, 0],\n   [Fam007, Fam007.10, 0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.11, Fam007.5, Fam007.6, 2, 2, 1/2, 0],\n   [Fam007, Fam007.12, Fam007.5, Fam007.6, 1,2, 0/0, 0],\n   [Fam007, Fam007.13, Fam007.5, Fam007.6, 2,1, 1/1,0],\n   [Fam007, Fam007.14, Fam007.5, Fam007.6, 1, 1, 1/1, 0],\n   [Fam007, Fam007.15, Fam007.7, Fam007.8, 1, 2, 1/2, 0],\n   [Fam007, Fam007.16, Fam007.8, Fam007.9, 1, 2, 1/2, 0],\n   [Fam007, Fam007.17, Fam007.8, Fam007.9, 1, 2, 0/0,0],\n   [Fam007, Fam007.18, Fam007.8, Fam007.9, 1, 2, 0/0, 0],\n   [Fam007, Fam007.19, Fam007.10, Fam007.11, 1, 2, 0/0,0],\n   [Fam007.20,Fam007.10, Fam007.11, 1,1,1/1,0]]"
          ]
        }
      ],
      "totalNumberOfSegregations": 9
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn051">EvLn051</a></p></td>
          <td>EvLn051</td>
          <td>
              <p>EvLn051</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon117">MolCon117</a></p><p><a href="details/details.html#CondMech118">CondMech118</a></p><p><a href="details/details.html#BMVR119">BMVR119</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn051"><pre><code>{
  "cg:id": "EvLn051",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn051",
  "hasInformation": [
    {
      "cg:id": "MolCon117",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll070",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll066",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll070",
          "identifier": "http://reg.genome.network/allele/CA016087"
        },
        "alleleName": "NM_000257.3(MYH7):c.5401G>A"
      },
      "consequence": {
        "cg:id": "CC070",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001583",
            "cg:type": "Coding",
            "code": "0001583",
            "system": "http://sequenceontology.org/",
            "display": "missense variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech118",
      "cg:type": "ConditionMechanism",
      "explanation": "Of 178 pathogenic  alleles in ClinVar, 149 are missense.",
      "mechanism": {
        "cg:id": "CC071",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001583",
            "cg:type": "Coding",
            "code": "0001583",
            "system": "http://sequenceontology.org/",
            "display": "missense variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC072",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:common",
            "cg:type": "Coding",
            "code": "common",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Common"
          }
        ]
      },
      "gene": {
        "cg:id": "G008",
        "cg:type": "Gene",
        "symbol": "MYH7"
      }
    },
    {
      "cg:id": "BMVR119",
      "cg:type": "BenignMissenseVariationRate",
      "explanation": "Based on the gene size, ExAC expects 740 missense variants to be observed, but actually observes only 420, giving a z score of 6.5, which is in the top 0.02% of genes.",
      "gene": {
        "cg:id": "G008",
        "cg:type": "Gene",
        "symbol": "MYH7"
      },
      "value": {
        "cg:id": "CC073",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-ben-missense-var-rate:low",
            "cg:type": "Coding",
            "code": "low",
            "system": "http://clinicalgenome.org/datamodel/benign-missense-variation-rate/",
            "display": "Low"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn052">EvLn052</a></p></td>
          <td>EvLn052</td>
          <td>
              <p>EvLn052</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#InSilicoPred120">InSilicoPred120</a></p><p><a href="details/details.html#InSilicoPred121">InSilicoPred121</a></p><p><a href="details/details.html#InSilicoPred122">InSilicoPred122</a></p><p><a href="details/details.html#InSilicoPred123">InSilicoPred123</a></p><p><a href="details/details.html#InSilicoPred124">InSilicoPred124</a></p><p><a href="details/details.html#InSilicoPred125">InSilicoPred125</a></p><p><a href="details/details.html#InSilicoPred126">InSilicoPred126</a></p><p><a href="details/details.html#InSilicoPred127">InSilicoPred127</a></p><p><a href="details/details.html#InSilicoPred128">InSilicoPred128</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn052"><pre><code>{
  "cg:id": "EvLn052",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn052",
  "hasInformation": [
    {
      "cg:id": "InSilicoPred120",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll071",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll067",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll071",
          "identifier": "http://reg.genome.network/allele/CA090928"
        },
        "alleleName": "NM_005228.3(EGFR):c.2369C>T"
      },
      "type": {
        "cg:id": "CC074",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "SIFT",
      "value": "Damaging"
    },
    {
      "cg:id": "InSilicoPred121",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll071",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll067",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll071",
          "identifier": "http://reg.genome.network/allele/CA090928"
        },
        "alleleName": "NM_005228.3(EGFR):c.2369C>T"
      },
      "type": {
        "cg:id": "CC075",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Polyphen",
      "value": "Probably damaging"
    },
    {
      "cg:id": "InSilicoPred122",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll071",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll067",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll071",
          "identifier": "http://reg.genome.network/allele/CA090928"
        },
        "alleleName": "NM_005228.3(EGFR):c.2369C>T"
      },
      "type": {
        "cg:id": "CC076",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "FATHMM",
      "value": "Tolerated"
    },
    {
      "cg:id": "InSilicoPred123",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll071",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll067",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll071",
          "identifier": "http://reg.genome.network/allele/CA090928"
        },
        "alleleName": "NM_005228.3(EGFR):c.2369C>T"
      },
      "type": {
        "cg:id": "CC077",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "LRT",
      "value": "Damaging"
    },
    {
      "cg:id": "InSilicoPred124",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll071",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll067",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll071",
          "identifier": "http://reg.genome.network/allele/CA090928"
        },
        "alleleName": "NM_005228.3(EGFR):c.2369C>T"
      },
      "type": {
        "cg:id": "CC078",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "MetaSVM",
      "value": "Tolerated"
    },
    {
      "cg:id": "InSilicoPred125",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll071",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll067",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll071",
          "identifier": "http://reg.genome.network/allele/CA090928"
        },
        "alleleName": "NM_005228.3(EGFR):c.2369C>T"
      },
      "type": {
        "cg:id": "CC079",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Mutation assessor",
      "value": "Tolerated"
    },
    {
      "cg:id": "InSilicoPred126",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll071",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll067",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll071",
          "identifier": "http://reg.genome.network/allele/CA090928"
        },
        "alleleName": "NM_005228.3(EGFR):c.2369C>T"
      },
      "type": {
        "cg:id": "CC080",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Mutation taster",
      "value": "Disease-causing"
    },
    {
      "cg:id": "InSilicoPred127",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll071",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll067",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll071",
          "identifier": "http://reg.genome.network/allele/CA090928"
        },
        "alleleName": "NM_005228.3(EGFR):c.2369C>T"
      },
      "type": {
        "cg:id": "CC081",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Provean",
      "value": "Damaging"
    },
    {
      "cg:id": "InSilicoPred128",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll071",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll067",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll071",
          "identifier": "http://reg.genome.network/allele/CA090928"
        },
        "alleleName": "NM_005228.3(EGFR):c.2369C>T"
      },
      "type": {
        "cg:id": "CC082",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Condel",
      "value": "deleterious"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn053">EvLn053</a></p></td>
          <td>EvLn053</td>
          <td>
              <p>EvLn053</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#InSilicoPred129">InSilicoPred129</a></p><p><a href="details/details.html#InSilicoPred130">InSilicoPred130</a></p><p><a href="details/details.html#InSilicoPred131">InSilicoPred131</a></p><p><a href="details/details.html#InSilicoPred132">InSilicoPred132</a></p><p><a href="details/details.html#InSilicoPred133">InSilicoPred133</a></p><p><a href="details/details.html#InSilicoPred134">InSilicoPred134</a></p><p><a href="details/details.html#InSilicoPred135">InSilicoPred135</a></p><p><a href="details/details.html#InSilicoPred136">InSilicoPred136</a></p><p><a href="details/details.html#InSilicoPred137">InSilicoPred137</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn053"><pre><code>{
  "cg:id": "EvLn053",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn053",
  "hasInformation": [
    {
      "cg:id": "InSilicoPred129",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll074",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll068",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll072",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "ENST00000566221.5:c.127C>T"
      },
      "type": {
        "cg:id": "CC083",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "SIFT",
      "value": "deleterious"
    },
    {
      "cg:id": "InSilicoPred130",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll074",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll068",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll072",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "ENST00000566221.5:c.127C>T"
      },
      "type": {
        "cg:id": "CC084",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Polyphen",
      "value": "Probably damaging"
    },
    {
      "cg:id": "InSilicoPred131",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll074",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll068",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll072",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "ENST00000566221.5:c.127C>T"
      },
      "type": {
        "cg:id": "CC085",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "FATHMM",
      "value": "Damaging"
    },
    {
      "cg:id": "InSilicoPred132",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll074",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll068",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll072",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "ENST00000566221.5:c.127C>T"
      },
      "type": {
        "cg:id": "CC086",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "LRT",
      "value": "Damaging"
    },
    {
      "cg:id": "InSilicoPred133",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll074",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll068",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll072",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "ENST00000566221.5:c.127C>T"
      },
      "type": {
        "cg:id": "CC087",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "MetaSVM",
      "value": "Tolerated"
    },
    {
      "cg:id": "InSilicoPred134",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll074",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll068",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll072",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "ENST00000566221.5:c.127C>T"
      },
      "type": {
        "cg:id": "CC088",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Mutation assessor",
      "value": "Medium functional impact"
    },
    {
      "cg:id": "InSilicoPred135",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll074",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll068",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll072",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "ENST00000566221.5:c.127C>T"
      },
      "type": {
        "cg:id": "CC089",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Mutation taster",
      "value": "Disease-causing"
    },
    {
      "cg:id": "InSilicoPred136",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll074",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll068",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll072",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "ENST00000566221.5:c.127C>T"
      },
      "type": {
        "cg:id": "CC090",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Provean",
      "value": "Damaging"
    },
    {
      "cg:id": "InSilicoPred137",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll074",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll068",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll072",
          "identifier": "http://reg.genome.network/allele/CA090884"
        },
        "alleleName": "ENST00000566221.5:c.127C>T"
      },
      "type": {
        "cg:id": "CC091",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Condel",
      "value": "deleterious"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn054">EvLn054</a></p></td>
          <td>EvLn054</td>
          <td>
              <p>EvLn054</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#InSilicoPred138">InSilicoPred138</a></p><p><a href="details/details.html#InSilicoPred139">InSilicoPred139</a></p><p><a href="details/details.html#InSilicoPred140">InSilicoPred140</a></p><p><a href="details/details.html#InSilicoPred141">InSilicoPred141</a></p><p><a href="details/details.html#InSilicoPred142">InSilicoPred142</a></p><p><a href="details/details.html#InSilicoPred143">InSilicoPred143</a></p><p><a href="details/details.html#InSilicoPred144">InSilicoPred144</a></p><p><a href="details/details.html#InSilicoPred145">InSilicoPred145</a></p><p><a href="details/details.html#InSilicoPred146">InSilicoPred146</a></p><p><a href="details/details.html#InSilicoPred147">InSilicoPred147</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn054"><pre><code>{
  "cg:id": "EvLn054",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn054",
  "hasInformation": [
    {
      "cg:id": "InSilicoPred138",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC092",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "SIFT",
      "value": "Tolerated"
    },
    {
      "cg:id": "InSilicoPred139",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC093",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Polyphen2",
      "value": "benign"
    },
    {
      "cg:id": "InSilicoPred140",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC094",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "AlignGVGD",
      "value": "C0 (benign)"
    },
    {
      "cg:id": "InSilicoPred141",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC095",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "MAPP",
      "value": "good (benign)"
    },
    {
      "cg:id": "InSilicoPred142",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC096",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "MutationTaster",
      "value": "disease-causing"
    },
    {
      "cg:id": "InSilicoPred143",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC097",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:sp",
            "cg:type": "Coding",
            "code": "sp",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "splicing prediction"
          }
        ]
      },
      "tool": "SSF",
      "value": 71.53
    },
    {
      "cg:id": "InSilicoPred144",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC098",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:sp",
            "cg:type": "Coding",
            "code": "sp",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "splicing prediction"
          }
        ]
      },
      "tool": "MaxEnt",
      "value": 7.39
    },
    {
      "cg:id": "InSilicoPred145",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC099",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:sp",
            "cg:type": "Coding",
            "code": "sp",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "splicing prediction"
          }
        ]
      },
      "tool": "NNSPLICE",
      "value": 0.87
    },
    {
      "cg:id": "InSilicoPred146",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC100",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:sp",
            "cg:type": "Coding",
            "code": "sp",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "splicing prediction"
          }
        ]
      },
      "tool": "GeneSplicer",
      "value": 7.75
    },
    {
      "cg:id": "InSilicoPred147",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll075",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll069",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll075",
          "identifier": "http://reg.genome.network/allele/CA022019"
        },
        "alleleName": "NM_000363.4(TNNI3):c.592C>G"
      },
      "type": {
        "cg:id": "CC101",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:sp",
            "cg:type": "Coding",
            "code": "sp",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "splicing prediction"
          }
        ]
      },
      "tool": "HSF",
      "value": 81.75
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn055">EvLn055</a></p></td>
          <td>EvLn055</td>
          <td>
              <p>EvLn055</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondEtiol148">CondEtiol148</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn055"><pre><code>{
  "cg:id": "EvLn055",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn055",
  "hasInformation": [
    {
      "cg:id": "CondEtiol148",
      "cg:type": "ConditionEtiology",
      "explanation": "HHT type 1 is only known to be caused by variants in ENG -",
      "condition": {
        "cg:id": "MendCond022",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:774, HP:0000006",
        "disease": [
          {
            "cg:id": "Orphanet:774"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      },
      "gene": {
        "cg:id": "G035",
        "cg:type": "Gene",
        "symbol": "ENG"
      },
      "specificity": "complete"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn056">EvLn056</a></p></td>
          <td>EvLn056</td>
          <td>
              <p>EvLn056</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondEtiol149">CondEtiol149</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn056"><pre><code>{
  "cg:id": "EvLn056",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn056",
  "hasInformation": [
    {
      "cg:id": "CondEtiol149",
      "cg:type": "ConditionEtiology",
      "explanation": "1.Only ~50% of HCM is genetic (can be caused by environment) \n2. More than 10 genes can cause HCM (specifically 9 sarcomere genes)",
      "condition": {
        "cg:id": "MendCond064",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:217569",
        "disease": [
          {
            "cg:id": "Orphanet:217569"
          }
        ]
      },
      "gene": {
        "cg:id": "G008",
        "cg:type": "Gene",
        "symbol": "MYH7"
      },
      "specificity": "low"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn057">EvLn057</a></p></td>
          <td>EvLn057</td>
          <td>
              <p>EvLn057</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp150">VarInterp150</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn057"><pre><code>{
  "cg:id": "EvLn057",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn057",
  "hasInformation": [
    {
      "cg:id": "VarInterp150",
      "cg:type": "VariantInterpretation",
      "explanation": "Variant has been classified as Pathogenic by Sharing Clinical Reports Project (SCV000054057.3)",
      "variant": {
        "cg:id": "CanAll072",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll078",
        "identifier": "http://reg.genome.network/allele/CA019369"
      },
      "condition": [
        {
          "cg:id": "MendCond023",
          "cg:type": "MendelianCondition",
          "explanation": "OMIM:612555, HP:0000006",
          "disease": [
            {
              "cg:id": "OMIM:612555",
              "cg:type": "Coding",
              "code": "612555",
              "system": "http://www.omim.org/",
              "display": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC102",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn058">EvLn058</a></p></td>
          <td>EvLn058</td>
          <td>
              <p>EvLn058</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp151">VarInterp151</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn058"><pre><code>{
  "cg:id": "EvLn058",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn058",
  "hasInformation": [
    {
      "cg:id": "VarInterp151",
      "cg:type": "VariantInterpretation",
      "explanation": "Variant has been classified as Pathogenic by OMIM (SCV000043718.1)",
      "variant": {
        "cg:id": "CanAll073",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll079",
        "identifier": "http://reg.genome.network/allele/CA128479"
      },
      "condition": [
        {
          "cg:id": "MendCond024",
          "cg:type": "MendelianCondition",
          "explanation": "OMIM:614065, HP:0000006",
          "disease": [
            {
              "cg:id": "OMIM:614065",
              "cg:type": "Coding",
              "code": "614065",
              "system": "http://www.omim.org/",
              "display": "Myopathy, Distal, type 4"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC103",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn059">EvLn059</a></p></td>
          <td>EvLn059</td>
          <td>
              <p>EvLn059</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp152">VarInterp152</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn059"><pre><code>{
  "cg:id": "EvLn059",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn059",
  "hasInformation": [
    {
      "cg:id": "VarInterp152",
      "cg:type": "VariantInterpretation",
      "explanation": "Variant has been classified as Pathogenic by Expert panel CFTR (SCV000245981.1)",
      "variant": {
        "cg:id": "CanAll074",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll080",
        "identifier": "http://reg.genome.network/allele/CA276116"
      },
      "condition": [
        {
          "cg:id": "MendCond025",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:586, HP:0000007",
          "disease": [
            {
              "cg:id": "Orphanet:586"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC104",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn060">EvLn060</a></p></td>
          <td>EvLn060</td>
          <td>
              <p>EvLn060</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq153">AllFreq153</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn060"><pre><code>{
  "cg:id": "EvLn060",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn060",
  "hasInformation": [
    {
      "cg:id": "AllFreq153",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC105",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll075",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll081",
        "identifier": "http://reg.genome.network/allele/CA090891"
      },
      "alleleCount": 6732,
      "alleleNumber": 120402,
      "alleleFrequency": 0.05591,
      "homozygousAlleleIndividualCount": 223
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn061">EvLn061</a></p></td>
          <td>EvLn061</td>
          <td>
              <p>EvLn061</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq154">AllFreq154</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn061"><pre><code>{
  "cg:id": "EvLn061",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn061",
  "hasInformation": [
    {
      "cg:id": "AllFreq154",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC106",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll076",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll082",
        "identifier": "http://reg.genome.network/allele/CA134997"
      },
      "alleleCount": 5507,
      "alleleNumber": 121350,
      "alleleFrequency": 0.04538,
      "homozygousAlleleIndividualCount": 768
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn062">EvLn062</a></p></td>
          <td>EvLn062</td>
          <td>
              <p>EvLn062</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq155">AllFreq155</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn062"><pre><code>{
  "cg:id": "EvLn062",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn062",
  "hasInformation": [
    {
      "cg:id": "AllFreq155",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC107",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "EXAC:EAS"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll076",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll082",
        "identifier": "http://reg.genome.network/allele/CA134997"
      },
      "alleleCount": 2518,
      "alleleNumber": 8632,
      "alleleFrequency": 0.2917,
      "homozygousAlleleIndividualCount": 405
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn063">EvLn063</a></p></td>
          <td>EvLn063</td>
          <td>
              <p>EvLn063</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq157">AllFreq157</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn063"><pre><code>{
  "cg:id": "EvLn063",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn063",
  "hasInformation": [
    {
      "cg:id": "AllFreq157",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC109",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll077",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll083",
        "identifier": "http://reg.genome.network/allele/CA501229"
      },
      "alleleCount": 14,
      "alleleNumber": 121100,
      "alleleFrequency": 0.0001158,
      "homozygousAlleleIndividualCount": 0
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn064">EvLn064</a></p></td>
          <td>EvLn064</td>
          <td>
              <p>EvLn064</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq158">AllFreq158</a></p><p><a href="details/details.html#CondPrev159">CondPrev159</a></p><p><a href="details/details.html#CondPenet160">CondPenet160</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn064"><pre><code>{
  "cg:id": "EvLn064",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn064",
  "hasInformation": [
    {
      "cg:id": "AllFreq158",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC110",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll078",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll084",
        "identifier": "http://reg.genome.network/allele/CA017597"
      },
      "alleleCount": 89,
      "alleleNumber": 121410,
      "alleleFrequency": 0.0007,
      "homozygousAlleleIndividualCount": 1
    },
    {
      "cg:id": "CondPrev159",
      "cg:type": "ConditionPrevelance",
      "condition": [
        {
          "cg:id": "MendCond026",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:558, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:558"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "minimum": 0.0017,
      "maximum": 0.0007,
      "population": {
        "cg:id": "CC111",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:overall",
            "cg:type": "Coding",
            "code": "overall",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Overall"
          }
        ]
      }
    },
    {
      "cg:id": "CondPenet160",
      "cg:type": "ConditionPenetrance",
      "condition": [
        {
          "cg:id": "MendCond026",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:558, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:558"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC112",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:high",
            "cg:type": "Coding",
            "code": "high",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "High"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn065">EvLn065</a></p></td>
          <td>EvLn065</td>
          <td>
              <p>EvLn065</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq162">AllFreq162</a></p><p><a href="details/details.html#CondPrev163">CondPrev163</a></p><p><a href="details/details.html#CondPenet164">CondPenet164</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn065"><pre><code>{
  "cg:id": "EvLn065",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn065",
  "hasInformation": [
    {
      "cg:id": "AllFreq162",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC114",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll079",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll085",
        "identifier": "http://reg.genome.network/allele/CA090926"
      },
      "alleleCount": 545,
      "alleleNumber": 87696,
      "alleleFrequency": 0.00625,
      "homozygousAlleleIndividualCount": 1,
      "hemizygousAlleleIndividualCount": 206
    },
    {
      "cg:id": "CondPrev163",
      "cg:type": "ConditionPrevelance",
      "condition": [
        {
          "cg:id": "MendCond027",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:2609, HP:0001423",
          "disease": [
            {
              "cg:id": "Orphanet:2609"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0001423",
            "cg:type": "Coding",
            "code": "0001423",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "X-linked dominant inheritance"
          }
        }
      ],
      "value": 0.00012,
      "population": {
        "cg:id": "CC115",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:overall",
            "cg:type": "Coding",
            "code": "overall",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Overall"
          }
        ]
      }
    },
    {
      "cg:id": "CondPenet164",
      "cg:type": "ConditionPenetrance",
      "condition": [
        {
          "cg:id": "MendCond027",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:2609, HP:0001423",
          "disease": [
            {
              "cg:id": "Orphanet:2609"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0001423",
            "cg:type": "Coding",
            "code": "0001423",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "X-linked dominant inheritance"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC116",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:high",
            "cg:type": "Coding",
            "code": "high",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "High"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn066">EvLn066</a></p></td>
          <td>EvLn066</td>
          <td>
              <p>EvLn066</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq166">AllFreq166</a></p><p><a href="details/details.html#CondPrev167">CondPrev167</a></p><p><a href="details/details.html#CondPenet168">CondPenet168</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn066"><pre><code>{
  "cg:id": "EvLn066",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn066",
  "hasInformation": [
    {
      "cg:id": "AllFreq166",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC119",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll080",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll086",
        "identifier": "http://reg.genome.network/allele/CA501230"
      },
      "alleleCount": 3,
      "alleleNumber": 120870,
      "alleleFrequency": 2.4e-05
    },
    {
      "cg:id": "CondPrev167",
      "cg:type": "ConditionPrevelance",
      "condition": [
        {
          "cg:id": "MendCond028",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:2855, HP:0000007",
          "disease": [
            {
              "cg:id": "Orphanet:2855"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          }
        }
      ],
      "maximum": 1.0e-06,
      "population": {
        "cg:id": "CC120",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:overall",
            "cg:type": "Coding",
            "code": "overall",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Overall"
          }
        ]
      }
    },
    {
      "cg:id": "CondPenet168",
      "cg:type": "ConditionPenetrance",
      "condition": [
        {
          "cg:id": "MendCond028",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:2855, HP:0000007",
          "disease": [
            {
              "cg:id": "Orphanet:2855"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC121",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:high",
            "cg:type": "Coding",
            "code": "high",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "High"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn067">EvLn067</a></p></td>
          <td>EvLn067</td>
          <td>
              <p>EvLn067</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondPenet170">CondPenet170</a></p><p><a href="details/details.html#IndAll172">IndAll172</a></p><p><a href="details/details.html#IndCond174">IndCond174</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn067"><pre><code>{
  "cg:id": "EvLn067",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn067",
  "hasInformation": [
    {
      "cg:id": "CondPenet170",
      "cg:type": "ConditionPenetrance",
      "explanation": "The evidence for full penetrance is not great, due to the rarity of the disease, but the fact that it's diagnosed at a young age and majority of cases are de novo is suggestive",
      "condition": [
        {
          "cg:id": "MendCond029",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:319182, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:319182"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC123",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:complete",
            "cg:type": "Coding",
            "code": "complete",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "Complete"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll172",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv013",
        "cg:type": "Individual",
        "description": "individual #013"
      },
      "primaryAllele": {
        "cg:id": "CanAll081",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll087",
        "identifier": "http://reg.genome.network/allele/CA090918"
      },
      "primaryZygosity": {
        "cg:id": "CC124",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond174",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv013",
        "cg:type": "Individual",
        "description": "individual #013"
      },
      "condition": [
        {
          "cg:id": "MendCond029",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:319182, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:319182"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "hasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn068">EvLn068</a></p></td>
          <td>EvLn068</td>
          <td>
              <p>EvLn068</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondPenet170">CondPenet170</a></p><p><a href="details/details.html#IndAll173">IndAll173</a></p><p><a href="details/details.html#IndCond175">IndCond175</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn068"><pre><code>{
  "cg:id": "EvLn068",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn068",
  "hasInformation": [
    {
      "cg:id": "CondPenet170",
      "cg:type": "ConditionPenetrance",
      "explanation": "The evidence for full penetrance is not great, due to the rarity of the disease, but the fact that it's diagnosed at a young age and majority of cases are de novo is suggestive",
      "condition": [
        {
          "cg:id": "MendCond029",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:319182, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:319182"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC123",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:complete",
            "cg:type": "Coding",
            "code": "complete",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "Complete"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll173",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv014",
        "cg:type": "Individual",
        "description": "individual #014"
      },
      "primaryAllele": {
        "cg:id": "CanAll081",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll087",
        "identifier": "http://reg.genome.network/allele/CA090918"
      },
      "primaryZygosity": {
        "cg:id": "CC125",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:hom",
            "cg:type": "Coding",
            "code": "hom",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "homozygous"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond175",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv014",
        "cg:type": "Individual",
        "description": "individual #014"
      },
      "condition": [
        {
          "cg:id": "MendCond029",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:319182, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:319182"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "hasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn069">EvLn069</a></p></td>
          <td>EvLn069</td>
          <td>
              <p>EvLn069</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondPenet176">CondPenet176</a></p><p><a href="details/details.html#IndAll178">IndAll178</a></p><p><a href="details/details.html#IndCond182">IndCond182</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn069"><pre><code>{
  "cg:id": "EvLn069",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn069",
  "hasInformation": [
    {
      "cg:id": "CondPenet176",
      "cg:type": "ConditionPenetrance",
      "explanation": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early.",
      "condition": [
        {
          "cg:id": "MendCond030",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:733, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:733"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC126",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:complete",
            "cg:type": "Coding",
            "code": "complete",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "Complete"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll178",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv015",
        "cg:type": "Individual",
        "description": "individual #015"
      },
      "primaryAllele": {
        "cg:id": "CanAll082",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll088",
        "identifier": "http://reg.genome.network/allele/CA008351"
      },
      "primaryZygosity": {
        "cg:id": "CC127",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond182",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv015",
        "cg:type": "Individual",
        "description": "individual #015"
      },
      "condition": [
        {
          "cg:id": "MendCond030",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:733, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:733"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "hasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn070">EvLn070</a></p></td>
          <td>EvLn070</td>
          <td>
              <p>EvLn070</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondPenet176">CondPenet176</a></p><p><a href="details/details.html#IndAll179">IndAll179</a></p><p><a href="details/details.html#IndCond183">IndCond183</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn070"><pre><code>{
  "cg:id": "EvLn070",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn070",
  "hasInformation": [
    {
      "cg:id": "CondPenet176",
      "cg:type": "ConditionPenetrance",
      "explanation": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early.",
      "condition": [
        {
          "cg:id": "MendCond030",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:733, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:733"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC126",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:complete",
            "cg:type": "Coding",
            "code": "complete",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "Complete"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll179",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv016",
        "cg:type": "Individual",
        "description": "individual #016"
      },
      "primaryAllele": {
        "cg:id": "CanAll082",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll088",
        "identifier": "http://reg.genome.network/allele/CA008351"
      },
      "primaryZygosity": {
        "cg:id": "CC128",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond183",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv016",
        "cg:type": "Individual",
        "description": "individual #016"
      },
      "condition": [
        {
          "cg:id": "MendCond030",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:733, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:733"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "hasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn071">EvLn071</a></p></td>
          <td>EvLn071</td>
          <td>
              <p>EvLn071</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondPenet176">CondPenet176</a></p><p><a href="details/details.html#IndAll180">IndAll180</a></p><p><a href="details/details.html#IndCond184">IndCond184</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn071"><pre><code>{
  "cg:id": "EvLn071",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn071",
  "hasInformation": [
    {
      "cg:id": "CondPenet176",
      "cg:type": "ConditionPenetrance",
      "explanation": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early.",
      "condition": [
        {
          "cg:id": "MendCond030",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:733, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:733"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC126",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:complete",
            "cg:type": "Coding",
            "code": "complete",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "Complete"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll180",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv017",
        "cg:type": "Individual",
        "description": "individual #017"
      },
      "primaryAllele": {
        "cg:id": "CanAll082",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll088",
        "identifier": "http://reg.genome.network/allele/CA008351"
      },
      "primaryZygosity": {
        "cg:id": "CC129",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond184",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv017",
        "cg:type": "Individual",
        "description": "individual #017"
      },
      "condition": [
        {
          "cg:id": "MendCond030",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:733, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:733"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "hasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn072">EvLn072</a></p></td>
          <td>EvLn072</td>
          <td>
              <p>EvLn072</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondPenet176">CondPenet176</a></p><p><a href="details/details.html#IndAll181">IndAll181</a></p><p><a href="details/details.html#IndCond185">IndCond185</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn072"><pre><code>{
  "cg:id": "EvLn072",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn072",
  "hasInformation": [
    {
      "cg:id": "CondPenet176",
      "cg:type": "ConditionPenetrance",
      "explanation": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early.",
      "condition": [
        {
          "cg:id": "MendCond030",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:733, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:733"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC126",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:complete",
            "cg:type": "Coding",
            "code": "complete",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "Complete"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll181",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv018",
        "cg:type": "Individual",
        "description": "individual #018"
      },
      "primaryAllele": {
        "cg:id": "CanAll082",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll088",
        "identifier": "http://reg.genome.network/allele/CA008351"
      },
      "primaryZygosity": {
        "cg:id": "CC130",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond185",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv018",
        "cg:type": "Individual",
        "description": "individual #018"
      },
      "condition": [
        {
          "cg:id": "MendCond030",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:733, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:733"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "hasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn073">EvLn073</a></p></td>
          <td>EvLn073</td>
          <td>
              <p>EvLn073</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondPenet186">CondPenet186</a></p><p><a href="details/details.html#IndAll188">IndAll188</a></p><p><a href="details/details.html#IndCond190">IndCond190</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn073"><pre><code>{
  "cg:id": "EvLn073",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn073",
  "hasInformation": [
    {
      "cg:id": "CondPenet186",
      "cg:type": "ConditionPenetrance",
      "explanation": "Marfan syndrome involves a range of skeletal features that should be identifiable at birth, if not during adolescence. The ocular and cardiovascular aspects may manifest later. 1/4 of mutations are de novo.",
      "condition": [
        {
          "cg:id": "MendCond072",
          "disease": [
            {
              "cg:id": "Orphanet:558"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC131",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:complete",
            "cg:type": "Coding",
            "code": "complete",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "Complete"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll188",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv019",
        "cg:type": "Individual",
        "description": "individual #019"
      },
      "primaryAllele": {
        "cg:id": "CanAll083",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll089",
        "identifier": "http://reg.genome.network/allele/CA013604"
      },
      "primaryZygosity": {
        "cg:id": "CC132",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond190",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv019",
        "cg:type": "Individual",
        "description": "individual #019"
      },
      "condition": [
        {
          "cg:id": "MendCond072",
          "disease": [
            {
              "cg:id": "Orphanet:558"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "hasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn074">EvLn074</a></p></td>
          <td>EvLn074</td>
          <td>
              <p>EvLn074</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CondPenet186">CondPenet186</a></p><p><a href="details/details.html#IndAll189">IndAll189</a></p><p><a href="details/details.html#IndCond191">IndCond191</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn074"><pre><code>{
  "cg:id": "EvLn074",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn074",
  "hasInformation": [
    {
      "cg:id": "CondPenet186",
      "cg:type": "ConditionPenetrance",
      "explanation": "Marfan syndrome involves a range of skeletal features that should be identifiable at birth, if not during adolescence. The ocular and cardiovascular aspects may manifest later. 1/4 of mutations are de novo.",
      "condition": [
        {
          "cg:id": "MendCond072",
          "disease": [
            {
              "cg:id": "Orphanet:558"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC131",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:complete",
            "cg:type": "Coding",
            "code": "complete",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "Complete"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll189",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv020",
        "cg:type": "Individual",
        "description": "individual #020"
      },
      "primaryAllele": {
        "cg:id": "CanAll083",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll089",
        "identifier": "http://reg.genome.network/allele/CA013604"
      },
      "primaryZygosity": {
        "cg:id": "CC133",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond191",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv020",
        "cg:type": "Individual",
        "description": "individual #020"
      },
      "condition": [
        {
          "cg:id": "MendCond072",
          "disease": [
            {
              "cg:id": "Orphanet:558"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "hasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn075">EvLn075</a></p></td>
          <td>EvLn075</td>
          <td>
              <p>EvLn075</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func192">Func192</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn075"><pre><code>{
  "cg:id": "EvLn075",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn075",
  "hasInformation": [
    {
      "cg:id": "Func192",
      "cg:type": "FunctionalData",
      "result": "Unchanged from reference",
      "dataType": {
        "cg:id": "CC134",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:iodeff",
            "cg:type": "Coding",
            "code": "iodeff",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "iodide efflux"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll092",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll086",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll092",
          "identifier": "http://reg.genome.network/allele/CA109417"
        },
        "alleleName": "NP_000432..1:p.Leu117Phe"
      },
      "gene": {
        "cg:id": "G016",
        "cg:type": "Gene",
        "symbol": "SLC26A4"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn076">EvLn076</a></p></td>
          <td>EvLn076</td>
          <td>
              <p>EvLn076</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func193">Func193</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn076"><pre><code>{
  "cg:id": "EvLn076",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn076",
  "hasInformation": [
    {
      "cg:id": "Func193",
      "cg:type": "FunctionalData",
      "explanation": "Both wild type and NP_000432..1:p.Leu117Phe localize to the same location",
      "result": "localized to apical membrane",
      "dataType": {
        "cg:id": "CC135",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:celloc",
            "cg:type": "Coding",
            "code": "celloc",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "cellular localization"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll092",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll086",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll092",
          "identifier": "http://reg.genome.network/allele/CA109417"
        },
        "alleleName": "NP_000432..1:p.Leu117Phe"
      },
      "gene": {
        "cg:id": "G016",
        "cg:type": "Gene",
        "symbol": "SLC26A4"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn077">EvLn077</a></p></td>
          <td>EvLn077</td>
          <td>
              <p>EvLn077</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func194">Func194</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn077"><pre><code>{
  "cg:id": "EvLn077",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn077",
  "hasInformation": [
    {
      "cg:id": "Func194",
      "cg:type": "FunctionalData",
      "result": "Unchanged from reference",
      "dataType": {
        "cg:id": "CC136",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:neuritegrowth",
            "cg:type": "Coding",
            "code": "neuritegrowth",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "neurite outgrowth"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll093",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll087",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll093",
          "identifier": "http://reg.genome.network/allele/CA501231"
        },
        "alleleName": "NM_152594.2(SPRED1): c.587C>T"
      },
      "gene": {
        "cg:id": "G045",
        "cg:type": "Gene",
        "symbol": "SPRED1"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn078">EvLn078</a></p></td>
          <td>EvLn078</td>
          <td>
              <p>EvLn078</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func195">Func195</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn078"><pre><code>{
  "cg:id": "EvLn078",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn078",
  "hasInformation": [
    {
      "cg:id": "Func195",
      "cg:type": "FunctionalData",
      "result": "localized to apical membrane",
      "dataType": {
        "cg:id": "CC137",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:elk1phosvialucif",
            "cg:type": "Coding",
            "code": "elk1phosvialucif",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "Elk1 pohsphorylation via luciferase assay"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll093",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll087",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll093",
          "identifier": "http://reg.genome.network/allele/CA501231"
        },
        "alleleName": "NM_152594.2(SPRED1): c.587C>T"
      },
      "gene": {
        "cg:id": "G045",
        "cg:type": "Gene",
        "symbol": "SPRED1"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn079">EvLn079</a></p></td>
          <td>EvLn079</td>
          <td>
              <p>EvLn079</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#FamSegData196">FamSegData196</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn079"><pre><code>{
  "cg:id": "EvLn079",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn079",
  "hasInformation": [
    {
      "cg:id": "FamSegData196",
      "cg:type": "FamilySegregationData",
      "explanation": "observed in one case diagnosed with Familial Hypercholesterolemia, characterized by elevated LDLC levels. Two siblings of the proband also have elevated LDLC levels but neither carry the 313+2T>C variant.",
      "condition": {
        "cg:id": "MendCond035",
        "cg:type": "MendelianCondition",
        "explanation": "OMIM:143890, HP:0000006",
        "disease": [
          {
            "cg:id": "OMIM:143890",
            "cg:type": "Coding",
            "code": "143890",
            "system": "http://www.omim.org/",
            "display": "Hypercholesterolemia, Familial"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "cg:id": "CanAll088",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll094",
        "identifier": "http://reg.genome.network/allele/CA023690"
      },
      "family": {
        "cg:id": "Fam008",
        "cg:type": "Family",
        "description": "Family #008"
      },
      "phenotypePositiveAllelePositive": 0,
      "phenotypePositiveAlleleNegative": 2,
      "anyInconsistentSegregations": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn080">EvLn080</a></p></td>
          <td>EvLn080</td>
          <td>
              <p>EvLn080</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#FamSegData197">FamSegData197</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn080"><pre><code>{
  "cg:id": "EvLn080",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn080",
  "hasInformation": [
    {
      "cg:id": "FamSegData197",
      "cg:type": "FamilySegregationData",
      "explanation": "The individual with HCM but without the allele is likely a phenocopy explained\nby environmental factors (obese, alcohol)",
      "condition": {
        "cg:id": "MendCond065",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:217569, HP:0000006",
        "disease": [
          {
            "cg:id": "Orphanet:217569"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "cg:id": "CanAll089",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll095",
        "identifier": "http://reg.genome.network/allele/CA010777"
      },
      "family": {
        "cg:id": "Fam009",
        "cg:type": "Family",
        "description": "Family #009"
      },
      "phenotypePositiveAllelePositive": 3,
      "phenotypePositiveAlleleNegative": 1,
      "anyInconsistentSegregations": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn081">EvLn081</a></p></td>
          <td>EvLn081</td>
          <td>
              <p>EvLn081</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon198">MolCon198</a></p><p><a href="details/details.html#CondMech199">CondMech199</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn081"><pre><code>{
  "cg:id": "EvLn081",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn081",
  "hasInformation": [
    {
      "cg:id": "MolCon198",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll097",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll091",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll097",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2) :c.4779A>C"
      },
      "consequence": {
        "cg:id": "CC138",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001583",
            "cg:type": "Coding",
            "code": "0001583",
            "system": "http://sequenceontology.org/",
            "display": "missense variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech199",
      "cg:type": "ConditionMechanism",
      "explanation": "ClinVar shows 421 variants in BRCA2 that are pathogenic for HBOC. 402 of these (~95%) are frameshift, nonsense, or splice-site variants.",
      "mechanism": {
        "cg:id": "CC139",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0002054",
            "cg:type": "Coding",
            "code": "0002054",
            "system": "http://sequenceontology.org/",
            "display": "loss of function variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC140",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:primary",
            "cg:type": "Coding",
            "code": "primary",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Primary"
          }
        ]
      },
      "gene": {
        "cg:id": "G002",
        "cg:type": "Gene",
        "symbol": "BRCA2"
      },
      "condition": {
        "cg:id": "MendCond036",
        "cg:type": "MendelianCondition",
        "explanation": "hereditary breast and ovarian cancer",
        "disease": [
          {
            "cg:id": "Orphanet:145"
          }
        ],
        "name": "hereditary breast and ovarian cancer"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn082">EvLn082</a></p></td>
          <td>EvLn082</td>
          <td>
              <p>EvLn082</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon198">MolCon198</a></p><p><a href="details/details.html#CondMech200">CondMech200</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn082"><pre><code>{
  "cg:id": "EvLn082",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn082",
  "hasInformation": [
    {
      "cg:id": "MolCon198",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll097",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll091",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll097",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2) :c.4779A>C"
      },
      "consequence": {
        "cg:id": "CC138",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001583",
            "cg:type": "Coding",
            "code": "0001583",
            "system": "http://sequenceontology.org/",
            "display": "missense variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech200",
      "cg:type": "ConditionMechanism",
      "explanation": "ClinVar shows 61 variants in BRCA2 that are pathogenic for Fanconi Anemia. 55 of these (~90%) are frameshift, nonsense, or splice-site variants.",
      "mechanism": {
        "cg:id": "CC141",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0002054",
            "cg:type": "Coding",
            "code": "0002054",
            "system": "http://sequenceontology.org/",
            "display": "loss of function variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC142",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:primary",
            "cg:type": "Coding",
            "code": "primary",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Primary"
          }
        ]
      },
      "gene": {
        "cg:id": "G002",
        "cg:type": "Gene",
        "symbol": "BRCA2"
      },
      "condition": {
        "cg:id": "MendCond037",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:84",
        "disease": [
          {
            "cg:id": "Orphanet:84"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn083">EvLn083</a></p></td>
          <td>EvLn083</td>
          <td>
              <p>EvLn083</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon201">MolCon201</a></p><p><a href="details/details.html#CondMech202">CondMech202</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn083"><pre><code>{
  "cg:id": "EvLn083",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn083",
  "hasInformation": [
    {
      "cg:id": "MolCon201",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll098",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll092",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll098",
          "identifier": "http://reg.genome.network/allele/CA090950"
        },
        "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
      },
      "consequence": {
        "cg:id": "CC143",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001583",
            "cg:type": "Coding",
            "code": "0001583",
            "system": "http://sequenceontology.org/",
            "display": "missense variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech202",
      "cg:type": "ConditionMechanism",
      "explanation": "shown in both humans and mouse models",
      "mechanism": {
        "cg:id": "CC144",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0002054",
            "cg:type": "Coding",
            "code": "0002054",
            "system": "http://sequenceontology.org/",
            "display": "loss of function variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC145",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:primary",
            "cg:type": "Coding",
            "code": "primary",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Primary"
          }
        ]
      },
      "gene": {
        "cg:id": "G048",
        "cg:type": "Gene",
        "symbol": "LOXHD1"
      },
      "condition": {
        "cg:id": "MendCond059",
        "cg:type": "MendelianCondition",
        "explanation": "Hearing Loss, Autosomal Recessive",
        "disease": [
          {
            "cg:id": "Orphanet:87884"
          }
        ],
        "name": "Hearing Loss, Autosomal Recessive",
        "modeOfInheritance": {
          "cg:id": "HP:0000007",
          "cg:type": "Coding",
          "code": "0000007",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Recessive"
        }
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn084">EvLn084</a></p></td>
          <td>EvLn084</td>
          <td>
              <p>EvLn084</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon203">MolCon203</a></p><p><a href="details/details.html#CondMech204">CondMech204</a></p><p><a href="details/details.html#CondMech205">CondMech205</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn084"><pre><code>{
  "cg:id": "EvLn084",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn084",
  "hasInformation": [
    {
      "cg:id": "MolCon203",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll099",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll093",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll099",
          "identifier": "http://reg.genome.network/allele/CA256496"
        },
        "alleleName": "NM_133378.4(TTN): c.2926T>C"
      },
      "consequence": {
        "cg:id": "CC146",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001583",
            "cg:type": "Coding",
            "code": "0001583",
            "system": "http://sequenceontology.org/",
            "display": "missense variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech204",
      "cg:type": "ConditionMechanism",
      "mechanism": {
        "cg:id": "CC147",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0002054",
            "cg:type": "Coding",
            "code": "0002054",
            "system": "http://sequenceontology.org/",
            "display": "loss of function variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC148",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:primary",
            "cg:type": "Coding",
            "code": "primary",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Primary"
          }
        ]
      },
      "gene": {
        "cg:id": "G003",
        "cg:type": "Gene",
        "symbol": "TTN"
      },
      "condition": {
        "cg:id": "MendCond053",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:217604",
        "disease": [
          {
            "cg:id": "Orphanet:217604"
          }
        ]
      }
    },
    {
      "cg:id": "CondMech205",
      "cg:type": "ConditionMechanism",
      "explanation": "Truncating variants in TTN are associated with DCM. However, if missense variants are located in the A-band, impact the A-band, or are in a highly expressed region, they can also be associated with DCM.",
      "mechanism": {
        "cg:id": "CC149",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001583",
            "cg:type": "Coding",
            "code": "0001583",
            "system": "http://sequenceontology.org/",
            "display": "missense variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC150",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:established",
            "cg:type": "Coding",
            "code": "established",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Established"
          }
        ]
      },
      "gene": {
        "cg:id": "G003",
        "cg:type": "Gene",
        "symbol": "TTN"
      },
      "condition": {
        "cg:id": "MendCond053",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:217604",
        "disease": [
          {
            "cg:id": "Orphanet:217604"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn085">EvLn085</a></p></td>
          <td>EvLn085</td>
          <td>
              <p>EvLn085</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndAll206">IndAll206</a></p><p><a href="details/details.html#IndCond208">IndCond208</a></p><p><a href="details/details.html#VarInterp210">VarInterp210</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn085"><pre><code>{
  "cg:id": "EvLn085",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn085",
  "hasInformation": [
    {
      "cg:id": "IndAll206",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv021",
        "cg:type": "Individual",
        "description": "individual #021"
      },
      "primaryAllele": {
        "cg:id": "CanAll094",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll100",
        "identifier": "http://reg.genome.network/allele/CA143572"
      },
      "secondaryAllele": {
        "cg:id": "CanAll095",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll101",
        "identifier": "http://reg.genome.network/allele/CA252233"
      },
      "phase": {
        "cg:id": "CC151",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-phase:cis",
            "cg:type": "Coding",
            "code": "cis",
            "system": "http://clinicalgenome.org/datamodel/phase/",
            "display": "in cis"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond208",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv021",
        "cg:type": "Individual",
        "description": "individual #021"
      },
      "condition": [
        {
          "cg:id": "MendCond038",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:791",
          "disease": [
            {
              "cg:id": "Orphanet:791"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "VarInterp210",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll095",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll101",
        "identifier": "http://reg.genome.network/allele/CA252233"
      },
      "condition": [
        {
          "cg:id": "MendCond038",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:791",
          "disease": [
            {
              "cg:id": "Orphanet:791"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC153",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn086">EvLn086</a></p></td>
          <td>EvLn086</td>
          <td>
              <p>EvLn086</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndAll207">IndAll207</a></p><p><a href="details/details.html#IndCond209">IndCond209</a></p><p><a href="details/details.html#VarInterp210">VarInterp210</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn086"><pre><code>{
  "cg:id": "EvLn086",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn086",
  "hasInformation": [
    {
      "cg:id": "IndAll207",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv022",
        "cg:type": "Individual",
        "description": "individual #022"
      },
      "primaryAllele": {
        "cg:id": "CanAll094",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll100",
        "identifier": "http://reg.genome.network/allele/CA143572"
      },
      "secondaryAllele": {
        "cg:id": "CanAll095",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll101",
        "identifier": "http://reg.genome.network/allele/CA252233"
      },
      "phase": {
        "cg:id": "CC152",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-phase:cis",
            "cg:type": "Coding",
            "code": "cis",
            "system": "http://clinicalgenome.org/datamodel/phase/",
            "display": "in cis"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond209",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv022",
        "cg:type": "Individual",
        "description": "individual #022"
      },
      "condition": [
        {
          "cg:id": "MendCond038",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:791",
          "disease": [
            {
              "cg:id": "Orphanet:791"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "VarInterp210",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll095",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll101",
        "identifier": "http://reg.genome.network/allele/CA252233"
      },
      "condition": [
        {
          "cg:id": "MendCond038",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:791",
          "disease": [
            {
              "cg:id": "Orphanet:791"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC153",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn087">EvLn087</a></p></td>
          <td>EvLn087</td>
          <td>
              <p>EvLn087</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndAll211">IndAll211</a></p><p><a href="details/details.html#IndCond212">IndCond212</a></p><p><a href="details/details.html#VarInterp213">VarInterp213</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn087"><pre><code>{
  "cg:id": "EvLn087",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn087",
  "hasInformation": [
    {
      "cg:id": "IndAll211",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv023",
        "cg:type": "Individual",
        "description": "individual #023"
      },
      "primaryAllele": {
        "cg:id": "CanAll096",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll102",
        "identifier": "http://reg.genome.network/allele/CA182840"
      },
      "secondaryAllele": {
        "cg:id": "CanAll097",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll103",
        "identifier": "http://reg.genome.network/allele/CA182839"
      },
      "phase": {
        "cg:id": "CC154",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-phase:cis",
            "cg:type": "Coding",
            "code": "cis",
            "system": "http://clinicalgenome.org/datamodel/phase/",
            "display": "in cis"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond212",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv023",
        "cg:type": "Individual",
        "description": "individual #023"
      },
      "condition": [
        {
          "cg:id": "MendCond039",
          "cg:type": "MendelianCondition",
          "explanation": "HP:0012873",
          "phenotype": [
            {
              "cg:id": "HP:0012873",
              "cg:type": "Coding",
              "code": "0012873",
              "system": "http://human-phenotype-ontology.github.io/",
              "display": "Absent vas deferens"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "VarInterp213",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll097",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll103",
        "identifier": "http://reg.genome.network/allele/CA182839"
      },
      "condition": [
        {
          "cg:id": "MendCond039",
          "cg:type": "MendelianCondition",
          "explanation": "HP:0012873",
          "phenotype": [
            {
              "cg:id": "HP:0012873",
              "cg:type": "Coding",
              "code": "0012873",
              "system": "http://human-phenotype-ontology.github.io/",
              "display": "Absent vas deferens"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC155",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn088">EvLn088</a></p></td>
          <td>EvLn088</td>
          <td>
              <p>EvLn088</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndAll214">IndAll214</a></p><p><a href="details/details.html#IndCond215">IndCond215</a></p><p><a href="details/details.html#VarInterp216">VarInterp216</a></p><p><a href="details/details.html#CondPenet218">CondPenet218</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn088"><pre><code>{
  "cg:id": "EvLn088",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn088",
  "hasInformation": [
    {
      "cg:id": "IndAll214",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv024",
        "cg:type": "Individual",
        "description": "individual #024"
      },
      "primaryAllele": {
        "cg:id": "CanAll098",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll104",
        "identifier": "http://reg.genome.network/allele/CA501235"
      },
      "secondaryAllele": {
        "cg:id": "CanAll099",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll105",
        "identifier": "http://reg.genome.network/allele/CA501236"
      },
      "phase": {
        "cg:id": "CC156",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-phase:trans",
            "cg:type": "Coding",
            "code": "trans",
            "system": "http://clinicalgenome.org/datamodel/phase/",
            "display": "in trans"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond215",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv024",
        "cg:type": "Individual",
        "description": "individual #024"
      },
      "condition": [
        {
          "cg:id": "MendCond007",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:167848, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:167848"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "VarInterp216",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll099",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll105",
        "identifier": "http://reg.genome.network/allele/CA501236"
      },
      "condition": [
        {
          "cg:id": "MendCond007",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:167848, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:167848"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC157",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA26333-7",
            "cg:type": "Coding",
            "code": "LA26333-7",
            "system": "http://loinc.org/",
            "display": "Uncertain Signficance"
          }
        ]
      }
    },
    {
      "cg:id": "CondPenet218",
      "cg:type": "ConditionPenetrance",
      "condition": [
        {
          "cg:id": "MendCond007",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:167848, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:167848"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "penetrance": {
        "cg:id": "CC158",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-penetrance-type:incomplete",
            "cg:type": "Coding",
            "code": "incomplete",
            "system": "http://clinicalgenome.org/datamodel/penetrance-type/",
            "display": "Incomplete"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn089">EvLn089</a></p></td>
          <td>EvLn089</td>
          <td>
              <p>EvLn089</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#RegAnn220">RegAnn220</a></p><p><a href="details/details.html#RegAll391">RegAll391</a></p><p><a href="details/details.html#MolCon223">MolCon223</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn089"><pre><code>{
  "cg:id": "EvLn089",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn089",
  "hasInformation": [
    {
      "cg:id": "RegAnn220",
      "cg:type": "RegionAnnotation",
      "explanation": "region is repeating but does not have a known function.",
      "region": {
        "cg:id": "Loc390",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq008",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NM_001271208.1"
          }
        },
        "start": 154,
        "stop": 183
      },
      "isNotTypes": [
        {
          "cg:id": "CC159",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:functional",
              "cg:type": "Coding",
              "code": "functional",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Functional"
            }
          ]
        }
      ],
      "isTypes": [
        {
          "cg:id": "CC160",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:repeat",
              "cg:type": "Coding",
              "code": "repeat",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Repeat"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "RegAll391",
      "cg:type": "RegionAlleles",
      "explanation": "region is repeating but does not have a known function.",
      "region": {
        "cg:id": "Loc390",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq008",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NM_001271208.1"
          }
        },
        "start": 154,
        "stop": 183
      },
      "includedAlleles": [
        {
          "cg:id": "CtxAll106",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll100",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll106",
            "identifier": "http://reg.genome.network/allele/CA148212"
          },
          "alleleName": "NM_001271208.1(NEB):c.169_183delCTGGCACAGCCAGCA"
        }
      ]
    },
    {
      "cg:id": "MolCon223",
      "cg:type": "MolecularConsequence",
      "consequence": {
        "cg:id": "CC161",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001825",
            "cg:type": "Coding",
            "code": "0001825",
            "system": "http://sequenceontology.org/",
            "display": "Conservative Inframe Deletion"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll106",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll100",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll106",
          "identifier": "http://reg.genome.network/allele/CA148212"
        },
        "alleleName": "NM_001271208.1(NEB):c.169_183delCTGGCACAGCCAGCA"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn090">EvLn090</a></p></td>
          <td>EvLn090</td>
          <td>
              <p>EvLn090</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#RegAnn225">RegAnn225</a></p><p><a href="details/details.html#RegAll393">RegAll393</a></p><p><a href="details/details.html#MolCon227">MolCon227</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn090"><pre><code>{
  "cg:id": "EvLn090",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn090",
  "hasInformation": [
    {
      "cg:id": "RegAnn225",
      "cg:type": "RegionAnnotation",
      "explanation": "The I-Band is a large region of repetitive Igg domains. ",
      "region": {
        "cg:id": "Loc392",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RegSeq009"
        },
        "start": 6509,
        "stop": 39262
      },
      "isTypes": [
        {
          "cg:id": "CC162",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:repeat",
              "cg:type": "Coding",
              "code": "repeat",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Repeat"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "RegAll393",
      "cg:type": "RegionAlleles",
      "explanation": "Allele NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC is contained by this region.",
      "region": {
        "cg:id": "Loc392",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RegSeq009"
        },
        "start": 6509,
        "stop": 39262
      },
      "includedAlleles": [
        {
          "cg:id": "CtxAll107",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll101",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll107",
            "identifier": "http://reg.genome.network/allele/CA273551"
          },
          "alleleName": "NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC"
        }
      ]
    },
    {
      "cg:id": "MolCon227",
      "cg:type": "MolecularConsequence",
      "consequence": {
        "cg:id": "CC163",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001589",
            "cg:type": "Coding",
            "code": "0001589",
            "system": "http://sequenceontology.org/",
            "display": "frameshift variant"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll107",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll101",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll107",
          "identifier": "http://reg.genome.network/allele/CA273551"
        },
        "alleleName": "NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn091">EvLn091</a></p></td>
          <td>EvLn091</td>
          <td>
              <p>EvLn091</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#RegAnn229">RegAnn229</a></p><p><a href="details/details.html#RegAll395">RegAll395</a></p><p><a href="details/details.html#MolCon231">MolCon231</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn091"><pre><code>{
  "cg:id": "EvLn091",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn091",
  "hasInformation": [
    {
      "cg:id": "RegAnn229",
      "cg:type": "RegionAnnotation",
      "region": {
        "cg:id": "Loc394",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq010",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NM_147196.2"
          }
        },
        "start": 522,
        "stop": 548
      },
      "isTypes": [
        {
          "cg:id": "CC164",
          "cg:type": "CodeableConcept",
          "coding": [
            {
              "cg:id": "CG-reg-anno-type:repeat",
              "cg:type": "Coding",
              "code": "repeat",
              "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
              "display": "Repeat"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "RegAll395",
      "cg:type": "RegionAlleles",
      "region": {
        "cg:id": "Loc394",
        "cg:type": "Locus",
        "referenceSequence": {
          "cg:id": "RefSeq010",
          "cg:type": "ReferenceSequence",
          "accession": {
            "cg:id": "NM_147196.2"
          }
        },
        "start": 522,
        "stop": 548
      },
      "includedAlleles": [
        {
          "cg:id": "CtxAll108",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll102",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll108",
            "identifier": "http://reg.genome.network/allele/CA184168"
          },
          "alleleName": "NM_147196.2(TMIE):c.391_393dupAAG"
        }
      ]
    },
    {
      "cg:id": "MolCon231",
      "cg:type": "MolecularConsequence",
      "consequence": {
        "cg:id": "CC165",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001823",
            "cg:type": "Coding",
            "code": "0001823",
            "system": "http://sequenceontology.org/",
            "display": "Conservative Inframe Insertion"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll108",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll102",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll108",
          "identifier": "http://reg.genome.network/allele/CA184168"
        },
        "alleleName": "NM_147196.2(TMIE):c.391_393dupAAG"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn092">EvLn092</a></p></td>
          <td>EvLn092</td>
          <td>
              <p>EvLn092</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#InSilicoPred232">InSilicoPred232</a></p><p><a href="details/details.html#InSilicoPred233">InSilicoPred233</a></p><p><a href="details/details.html#InSilicoPred234">InSilicoPred234</a></p><p><a href="details/details.html#InSilicoPred235">InSilicoPred235</a></p><p><a href="details/details.html#InSilicoPred236">InSilicoPred236</a></p><p><a href="details/details.html#InSilicoPred237">InSilicoPred237</a></p><p><a href="details/details.html#InSilicoPred238">InSilicoPred238</a></p><p><a href="details/details.html#InSilicoPred239">InSilicoPred239</a></p><p><a href="details/details.html#InSilicoPred240">InSilicoPred240</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn092"><pre><code>{
  "cg:id": "EvLn092",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn092",
  "hasInformation": [
    {
      "cg:id": "InSilicoPred232",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll109",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll103",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll109",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
      },
      "type": {
        "cg:id": "CC166",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "SIFT",
      "value": "Benign"
    },
    {
      "cg:id": "InSilicoPred233",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll109",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll103",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll109",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
      },
      "type": {
        "cg:id": "CC167",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Polyphen",
      "value": "Possibly damaging"
    },
    {
      "cg:id": "InSilicoPred234",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll109",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll103",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll109",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
      },
      "type": {
        "cg:id": "CC168",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "FATHMM",
      "value": "Benign"
    },
    {
      "cg:id": "InSilicoPred235",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll109",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll103",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll109",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
      },
      "type": {
        "cg:id": "CC169",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "LRT",
      "value": "Benign"
    },
    {
      "cg:id": "InSilicoPred236",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll109",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll103",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll109",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
      },
      "type": {
        "cg:id": "CC170",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "MetaSVM",
      "value": "Benign"
    },
    {
      "cg:id": "InSilicoPred237",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll109",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll103",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll109",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
      },
      "type": {
        "cg:id": "CC171",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Mutation assessor",
      "value": "Benign"
    },
    {
      "cg:id": "InSilicoPred238",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll109",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll103",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll109",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
      },
      "type": {
        "cg:id": "CC172",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Mutation taster",
      "value": "Benign"
    },
    {
      "cg:id": "InSilicoPred239",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll109",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll103",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll109",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
      },
      "type": {
        "cg:id": "CC173",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Provean",
      "value": "Benign"
    },
    {
      "cg:id": "InSilicoPred240",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll109",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll103",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll109",
          "identifier": "http://reg.genome.network/allele/CA020822"
        },
        "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
      },
      "type": {
        "cg:id": "CC174",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "tool": "Condel",
      "value": "deleterious"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn093">EvLn093</a></p></td>
          <td>EvLn093</td>
          <td>
              <p>EvLn093</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndAll241">IndAll241</a></p><p><a href="details/details.html#IndCond242">IndCond242</a></p><p><a href="details/details.html#VarInterp243">VarInterp243</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn093"><pre><code>{
  "cg:id": "EvLn093",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn093",
  "hasInformation": [
    {
      "cg:id": "IndAll241",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv025",
        "cg:type": "Individual",
        "description": "individual #025"
      },
      "primaryAllele": {
        "cg:id": "CanAll106",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll112",
        "identifier": "http://reg.genome.network/allele/CA012381"
      },
      "secondaryAllele": {
        "cg:id": "CanAll107",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll113",
        "identifier": "http://reg.genome.network/allele/CA005304"
      }
    },
    {
      "cg:id": "IndCond242",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv025",
        "cg:type": "Individual",
        "description": "individual #025"
      },
      "condition": [
        {
          "cg:id": "MendCond067",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:217569",
          "disease": [
            {
              "cg:id": "Orphanet:217569"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "VarInterp243",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll106",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll112",
        "identifier": "http://reg.genome.network/allele/CA012381"
      },
      "condition": [
        {
          "cg:id": "MendCond067",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:217569",
          "disease": [
            {
              "cg:id": "Orphanet:217569"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC175",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA26332-9",
            "cg:type": "Coding",
            "code": "LA26332-9",
            "system": "http://loinc.org/",
            "display": "Likely Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn094">EvLn094</a></p></td>
          <td>EvLn094</td>
          <td>
              <p>EvLn094</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndAll244">IndAll244</a></p><p><a href="details/details.html#IndAll245">IndAll245</a></p><p><a href="details/details.html#IndCond246">IndCond246</a></p><p><a href="details/details.html#VarInterp247">VarInterp247</a></p><p><a href="details/details.html#VarInterp248">VarInterp248</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn094"><pre><code>{
  "cg:id": "EvLn094",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn094",
  "hasInformation": [
    {
      "cg:id": "IndAll244",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv026",
        "cg:type": "Individual",
        "description": "individual #026"
      },
      "primaryAllele": {
        "cg:id": "CanAll108",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll114",
        "identifier": "http://reg.genome.network/allele/CA137502"
      }
    },
    {
      "cg:id": "IndAll245",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv026",
        "cg:type": "Individual",
        "description": "individual #026"
      },
      "primaryAllele": {
        "cg:id": "CanAll109",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll115",
        "identifier": "http://reg.genome.network/allele/CA172240"
      },
      "secondaryAllele": {
        "cg:id": "CanAll110",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll116",
        "identifier": "http://reg.genome.network/allele/CA274460"
      },
      "primaryZygosity": {
        "cg:id": "CC176",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      },
      "secondaryZygosity": {
        "cg:id": "CC177",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:het",
            "cg:type": "Coding",
            "code": "het",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "heterozygous"
          }
        ]
      },
      "phase": {
        "cg:id": "CC178",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-phase:trans",
            "cg:type": "Coding",
            "code": "trans",
            "system": "http://clinicalgenome.org/datamodel/phase/",
            "display": "in trans"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond246",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv026",
        "cg:type": "Individual",
        "description": "individual #026"
      },
      "condition": [
        {
          "cg:id": "MendCond060",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:87884",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "VarInterp247",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll109",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll115",
        "identifier": "http://reg.genome.network/allele/CA172240"
      },
      "condition": [
        {
          "cg:id": "MendCond060",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:87884",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC179",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA26332-9",
            "cg:type": "Coding",
            "code": "LA26332-9",
            "system": "http://loinc.org/",
            "display": "Likely Pathogenic"
          }
        ]
      }
    },
    {
      "cg:id": "VarInterp248",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll110",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll116",
        "identifier": "http://reg.genome.network/allele/CA274460"
      },
      "condition": [
        {
          "cg:id": "MendCond060",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:87884",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC180",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA26332-9",
            "cg:type": "Coding",
            "code": "LA26332-9",
            "system": "http://loinc.org/",
            "display": "Likely Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn095">EvLn095</a></p></td>
          <td>EvLn095</td>
          <td>
              <p>EvLn095</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndAll249">IndAll249</a></p><p><a href="details/details.html#IndCond250">IndCond250</a></p><p><a href="details/details.html#VarInterp251">VarInterp251</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn095"><pre><code>{
  "cg:id": "EvLn095",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn095",
  "hasInformation": [
    {
      "cg:id": "IndAll249",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv027",
        "cg:type": "Individual",
        "description": "individual #027"
      },
      "primaryAllele": {
        "cg:id": "CanAll108",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll114",
        "identifier": "http://reg.genome.network/allele/CA137502"
      },
      "secondaryAllele": {
        "cg:id": "CanAll111",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll117",
        "identifier": "http://reg.genome.network/allele/CA277980"
      },
      "secondaryZygosity": {
        "cg:id": "CC181",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-allelic-state:hom",
            "cg:type": "Coding",
            "code": "hom",
            "system": "http://clinicalgenome.org/datamodel/allelic-state/",
            "display": "homozygous"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond250",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv027",
        "cg:type": "Individual",
        "description": "individual #027"
      },
      "condition": [
        {
          "cg:id": "MendCond071",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:886, HP:0000007",
          "disease": [
            {
              "cg:id": "Orphanet:886"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          }
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "VarInterp251",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll108",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll114",
        "identifier": "http://reg.genome.network/allele/CA137502"
      },
      "condition": [
        {
          "cg:id": "MendCond071",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:886, HP:0000007",
          "disease": [
            {
              "cg:id": "Orphanet:886"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          }
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC182",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn096">EvLn096</a></p></td>
          <td>EvLn096</td>
          <td>
              <p>EvLn096</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#IndCond252">IndCond252</a></p><p><a href="details/details.html#IndAll253">IndAll253</a></p><p><a href="details/details.html#VarInterp254">VarInterp254</a></p><p><a href="details/details.html#VarInterp270">VarInterp270</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn096"><pre><code>{
  "cg:id": "EvLn096",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn096",
  "hasInformation": [
    {
      "cg:id": "IndCond252",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Indiv028",
        "cg:type": "Individual",
        "description": "individual #028"
      },
      "condition": [
        {
          "cg:id": "MendCond041",
          "cg:type": "MendelianCondition",
          "explanation": "Pseudoachondroplastic spondyloepiphyseal dysplasia",
          "disease": [
            {
              "cg:id": "Orphanet:750"
            }
          ],
          "name": "Pseudoachondroplastic spondyloepiphyseal dysplasia"
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "IndAll253",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv028",
        "cg:type": "Individual",
        "description": "individual #028"
      },
      "primaryAllele": {
        "cg:id": "CanAll112",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll118",
        "identifier": "http://reg.genome.network/allele/CA204551"
      },
      "secondaryAllele": {
        "cg:id": "CanAll113",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll119",
        "identifier": "http://reg.genome.network/allele/CA254703"
      }
    },
    {
      "cg:id": "VarInterp254",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll113",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll119",
        "identifier": "http://reg.genome.network/allele/CA254703"
      },
      "condition": [
        {
          "cg:id": "MendCond042",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:750",
          "disease": [
            {
              "cg:id": "Orphanet:750"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC183",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    },
    {
      "cg:id": "VarInterp270",
      "cg:type": "VariantInterpretation",
      "explanation": "Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome is caused by a combination of a pathogenic variant in COL2A1 and another in COMP.   So CA113 may be involved, but is not the sole cause.",
      "variant": {
        "cg:id": "CanAll113",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll119",
        "identifier": "http://reg.genome.network/allele/CA254703"
      },
      "condition": [
        {
          "cg:id": "MendCond041",
          "cg:type": "MendelianCondition",
          "explanation": "Pseudoachondroplastic spondyloepiphyseal dysplasia",
          "disease": [
            {
              "cg:id": "Orphanet:750"
            }
          ],
          "name": "Pseudoachondroplastic spondyloepiphyseal dysplasia"
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC193",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA26333-7",
            "cg:type": "Coding",
            "code": "LA26333-7",
            "system": "http://loinc.org/",
            "display": "Uncertain Signficance"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn097">EvLn097</a></p></td>
          <td>EvLn097</td>
          <td>
              <p>EvLn097</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp255">VarInterp255</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn097"><pre><code>{
  "cg:id": "EvLn097",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn097",
  "hasInformation": [
    {
      "cg:id": "VarInterp255",
      "cg:type": "VariantInterpretation",
      "explanation": "classified by ENIGMA.",
      "variant": {
        "cg:id": "CanAll114",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll120",
        "identifier": "http://reg.genome.network/allele/CA276195"
      },
      "condition": [
        {
          "cg:id": "MendCond073",
          "disease": [
            {
              "cg:id": "Orphanet:145"
            }
          ],
          "name": "hereditary breast and ovarian cancer"
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC184",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6675-8",
            "cg:type": "Coding",
            "code": "LA6675-8",
            "system": "http://loinc.org/",
            "display": "Benign"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn098">EvLn098</a></p></td>
          <td>EvLn098</td>
          <td>
              <p>EvLn098</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp256">VarInterp256</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn098"><pre><code>{
  "cg:id": "EvLn098",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn098",
  "hasInformation": [
    {
      "cg:id": "VarInterp256",
      "cg:type": "VariantInterpretation",
      "explanation": "classified by Sharing Clinical Reports Project.",
      "variant": {
        "cg:id": "CanAll115",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll121",
        "identifier": "http://reg.genome.network/allele/CA016113"
      },
      "condition": [
        {
          "cg:id": "MendCond043",
          "cg:type": "MendelianCondition",
          "explanation": "OMIM:612555",
          "disease": [
            {
              "cg:id": "OMIM:612555",
              "cg:type": "Coding",
              "code": "612555",
              "system": "http://www.omim.org/",
              "display": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC185",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6675-8",
            "cg:type": "Coding",
            "code": "LA6675-8",
            "system": "http://loinc.org/",
            "display": "Benign"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn099">EvLn099</a></p></td>
          <td>EvLn099</td>
          <td>
              <p>EvLn099</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp257">VarInterp257</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn099"><pre><code>{
  "cg:id": "EvLn099",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn099",
  "hasInformation": [
    {
      "cg:id": "VarInterp257",
      "cg:type": "VariantInterpretation",
      "explanation": "classified by OMIM",
      "variant": {
        "cg:id": "CanAll116",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll122",
        "identifier": "http://reg.genome.network/allele/CA113946"
      },
      "condition": [
        {
          "cg:id": "MendCond044",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:414",
          "disease": [
            {
              "cg:id": "Orphanet:414"
            }
          ]
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC186",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6675-8",
            "cg:type": "Coding",
            "code": "LA6675-8",
            "system": "http://loinc.org/",
            "display": "Benign"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn100">EvLn100</a></p></td>
          <td>EvLn100</td>
          <td>
              <p>EvLn100</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#InSilicoPred258">InSilicoPred258</a></p><p><a href="details/details.html#Conserv259">Conserv259</a></p><p><a href="details/details.html#Conserv260">Conserv260</a></p><p><a href="details/details.html#MolCon261">MolCon261</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn100"><pre><code>{
  "cg:id": "EvLn100",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn100",
  "hasInformation": [
    {
      "cg:id": "InSilicoPred258",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll123",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll117",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll123",
          "identifier": "http://reg.genome.network/allele/CA501232"
        },
        "alleleName": "NM_138691.2 (TMC1): c.684C>T"
      },
      "type": {
        "cg:id": "CC187",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:sp",
            "cg:type": "Coding",
            "code": "sp",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "splicing prediction"
          }
        ]
      },
      "value": "no impact"
    },
    {
      "cg:id": "Conserv259",
      "cg:type": "Conservation",
      "allele": {
        "cg:id": "CanAll117",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll123",
        "identifier": "http://reg.genome.network/allele/CA501232"
      },
      "conserved": true,
      "method": "phyloP",
      "score": -2.297
    },
    {
      "cg:id": "Conserv260",
      "cg:type": "Conservation",
      "allele": {
        "cg:id": "CanAll117",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll123",
        "identifier": "http://reg.genome.network/allele/CA501232"
      },
      "conserved": true,
      "method": "PhastCons",
      "score": 0.039
    },
    {
      "cg:id": "MolCon261",
      "cg:type": "MolecularConsequence",
      "consequence": {
        "cg:id": "CC188",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001819",
            "cg:type": "Coding",
            "code": "0001819",
            "system": "http://sequenceontology.org/",
            "display": "synonymous variant"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll123",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll117",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll123",
          "identifier": "http://reg.genome.network/allele/CA501232"
        },
        "alleleName": "NM_138691.2 (TMC1): c.684C>T"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn101">EvLn101</a></p></td>
          <td>EvLn101</td>
          <td>
              <p>EvLn101</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#InSilicoPred262">InSilicoPred262</a></p><p><a href="details/details.html#Conserv263">Conserv263</a></p><p><a href="details/details.html#Conserv264">Conserv264</a></p><p><a href="details/details.html#MolCon265">MolCon265</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn101"><pre><code>{
  "cg:id": "EvLn101",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn101",
  "hasInformation": [
    {
      "cg:id": "InSilicoPred262",
      "cg:type": "InSilicoPrediction",
      "explanation": "Provenance would show that this comes from Alamut",
      "contextualAllele": {
        "cg:id": "CtxAll124",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll118",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll124",
          "identifier": "http://reg.genome.network/allele/CA013536"
        },
        "alleleName": "NM_000257.2(MYH7):c.327C>T"
      },
      "type": {
        "cg:id": "CC189",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:sp",
            "cg:type": "Coding",
            "code": "sp",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "splicing prediction"
          }
        ]
      },
      "value": "no impact"
    },
    {
      "cg:id": "Conserv263",
      "cg:type": "Conservation",
      "allele": {
        "cg:id": "CanAll118",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll124",
        "identifier": "http://reg.genome.network/allele/CA013536"
      },
      "conserved": true,
      "method": "phyloP",
      "score": -0.76
    },
    {
      "cg:id": "Conserv264",
      "cg:type": "Conservation",
      "allele": {
        "cg:id": "CanAll118",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll124",
        "identifier": "http://reg.genome.network/allele/CA013536"
      },
      "conserved": true,
      "method": "PhastCons",
      "score": 0.63
    },
    {
      "cg:id": "MolCon265",
      "cg:type": "MolecularConsequence",
      "consequence": {
        "cg:id": "CC190",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001819",
            "cg:type": "Coding",
            "code": "0001819",
            "system": "http://sequenceontology.org/",
            "display": "synonymous variant"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll124",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll118",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll124",
          "identifier": "http://reg.genome.network/allele/CA013536"
        },
        "alleleName": "NM_000257.2(MYH7):c.327C>T"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn102">EvLn102</a></p></td>
          <td>EvLn102</td>
          <td>
              <p>EvLn102</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#InSilicoPred266">InSilicoPred266</a></p><p><a href="details/details.html#Conserv267">Conserv267</a></p><p><a href="details/details.html#Conserv268">Conserv268</a></p><p><a href="details/details.html#MolCon269">MolCon269</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn102"><pre><code>{
  "cg:id": "EvLn102",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn102",
  "hasInformation": [
    {
      "cg:id": "InSilicoPred266",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll125",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll119",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll125",
          "identifier": "http://reg.genome.network/allele/CA177682"
        },
        "alleleName": "NM_002880.3(RAF1):c.321T>C"
      },
      "type": {
        "cg:id": "CC191",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:sp",
            "cg:type": "Coding",
            "code": "sp",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "splicing prediction"
          }
        ]
      },
      "value": "no impact"
    },
    {
      "cg:id": "Conserv267",
      "cg:type": "Conservation",
      "allele": {
        "cg:id": "CanAll119",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll125",
        "identifier": "http://reg.genome.network/allele/CA177682"
      },
      "conserved": true,
      "method": "phyloP",
      "score": 0.45
    },
    {
      "cg:id": "Conserv268",
      "cg:type": "Conservation",
      "allele": {
        "cg:id": "CanAll119",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll125",
        "identifier": "http://reg.genome.network/allele/CA177682"
      },
      "conserved": true,
      "method": "PhastCons",
      "score": 1.0
    },
    {
      "cg:id": "MolCon269",
      "cg:type": "MolecularConsequence",
      "consequence": {
        "cg:id": "CC192",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001819",
            "cg:type": "Coding",
            "code": "0001819",
            "system": "http://sequenceontology.org/",
            "display": "synonymous variant"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll125",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll119",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll125",
          "identifier": "http://reg.genome.network/allele/CA177682"
        },
        "alleleName": "NM_002880.3(RAF1):c.321T>C"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn103">EvLn103</a></p></td>
          <td>EvLn103</td>
          <td>
              <p>EvLn103</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#MolCon272">MolCon272</a></p><p><a href="details/details.html#CondMech273">CondMech273</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn103"><pre><code>{
  "cg:id": "EvLn103",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn103",
  "hasInformation": [
    {
      "cg:id": "MolCon272",
      "cg:type": "MolecularConsequence",
      "explanation": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466).",
      "contextualAllele": {
        "cg:id": "CtxAll127",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll121",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll127",
          "identifier": "http://reg.genome.network/allele/CA022880"
        },
        "alleleName": "NM_024422.3(DSC2): c.631-2A>G (p.?)"
      },
      "consequence": {
        "cg:id": "CC194",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001629",
            "cg:type": "Coding",
            "code": "0001629",
            "system": "http://sequenceontology.org/",
            "display": "splice-site variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech273",
      "cg:type": "ConditionMechanism",
      "explanation": "Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).",
      "mechanism": {
        "cg:id": "CC195",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0002054",
            "cg:type": "Coding",
            "code": "0002054",
            "system": "http://sequenceontology.org/",
            "display": "loss of function variant"
          }
        ]
      },
      "condition": {
        "cg:id": "MendCond046",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:247",
        "disease": [
          {
            "cg:id": "Orphanet:247"
          }
        ]
      },
      "strength": {
        "cg:id": "CC196",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:established",
            "cg:type": "Coding",
            "code": "established",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Established"
          }
        ]
      },
      "gene": {
        "cg:id": "G060",
        "cg:type": "Gene",
        "symbol": "DSC2"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn104">EvLn104</a></p></td>
          <td>EvLn104</td>
          <td>
              <p>EvLn104</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#FamSegData271">FamSegData271</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn104"><pre><code>{
  "cg:id": "EvLn104",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn104",
  "hasInformation": [
    {
      "cg:id": "FamSegData271",
      "cg:type": "FamilySegregationData",
      "condition": {
        "cg:id": "MendCond066",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:217569, HP:0000006",
        "disease": [
          {
            "cg:id": "Orphanet:217569"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "cg:id": "CanAll090",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll096",
        "identifier": "http://reg.genome.network/allele/CA133141"
      },
      "family": {
        "cg:id": "Fam010",
        "cg:type": "Family",
        "description": "Family #010"
      },
      "phenotypePositiveAllelePositive": 0,
      "phenotypePositiveAlleleNegative": 2,
      "anyInconsistentSegregations": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn105">EvLn105</a></p></td>
          <td>EvLn105</td>
          <td>
              <p>EvLn105</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#InSilicoPred362">InSilicoPred362</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn105"><pre><code>{
  "cg:id": "EvLn105",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn105",
  "hasInformation": [
    {
      "cg:id": "InSilicoPred362",
      "cg:type": "InSilicoPrediction",
      "contextualAllele": {
        "cg:id": "CtxAll111",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll105",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll111",
          "identifier": "http://reg.genome.network/allele/CA501237"
        },
        "alleleName": "NM_133379.3(TTN) : c.15283T>C"
      },
      "type": {
        "cg:id": "CC282",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:me",
            "cg:type": "Coding",
            "code": "me",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "missense effect"
          }
        ]
      },
      "value": "ProbablyDamaging",
      "tool": "Polyphen-2 HumDiv"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn106">EvLn106</a></p></td>
          <td>EvLn106</td>
          <td>
              <p>EvLn106</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp364">VarInterp364</a></p><p><a href="details/details.html#VarInterp365">VarInterp365</a></p><p><a href="details/details.html#IndAll366">IndAll366</a></p><p><a href="details/details.html#IndAll367">IndAll367</a></p><p><a href="details/details.html#IndCond368">IndCond368</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn106"><pre><code>{
  "cg:id": "EvLn106",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn106",
  "hasInformation": [
    {
      "cg:id": "VarInterp364",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll122",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll128",
        "identifier": "http://reg.genome.network/allele/CA253304"
      },
      "condition": [
        {
          "cg:id": "MendCond058",
          "cg:type": "MendelianCondition",
          "explanation": "Hearing Loss",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ],
          "name": "Hearing Loss"
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC284",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA26332-9",
            "cg:type": "Coding",
            "code": "LA26332-9",
            "system": "http://loinc.org/",
            "display": "Likely Pathogenic"
          }
        ]
      }
    },
    {
      "cg:id": "VarInterp365",
      "cg:type": "VariantInterpretation",
      "variant": {
        "cg:id": "CanAll123",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll129",
        "identifier": "http://reg.genome.network/allele/CA273665"
      },
      "condition": [
        {
          "cg:id": "MendCond058",
          "cg:type": "MendelianCondition",
          "explanation": "Hearing Loss",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ],
          "name": "Hearing Loss"
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC285",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA6668-3",
            "cg:type": "Coding",
            "code": "LA6668-3",
            "system": "http://loinc.org/",
            "display": "Pathogenic"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll366",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv029",
        "cg:type": "Individual",
        "description": "individual #029"
      },
      "primaryAllele": {
        "cg:id": "CanAll092",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll098",
        "identifier": "http://reg.genome.network/allele/CA090950"
      },
      "secondaryAllele": {
        "cg:id": "CanAll122",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll128",
        "identifier": "http://reg.genome.network/allele/CA253304"
      },
      "phase": {
        "cg:id": "CC286",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-phase:trans",
            "cg:type": "Coding",
            "code": "trans",
            "system": "http://clinicalgenome.org/datamodel/phase/",
            "display": "in trans"
          }
        ]
      }
    },
    {
      "cg:id": "IndAll367",
      "cg:type": "IndividualAllele",
      "individual": {
        "cg:id": "Indiv029",
        "cg:type": "Individual",
        "description": "individual #029"
      },
      "primaryAllele": {
        "cg:id": "CanAll092",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll098",
        "identifier": "http://reg.genome.network/allele/CA090950"
      },
      "secondaryAllele": {
        "cg:id": "CanAll123",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll129",
        "identifier": "http://reg.genome.network/allele/CA273665"
      },
      "phase": {
        "cg:id": "CC287",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-phase:trans",
            "cg:type": "Coding",
            "code": "trans",
            "system": "http://clinicalgenome.org/datamodel/phase/",
            "display": "in trans"
          }
        ]
      }
    },
    {
      "cg:id": "IndCond368",
      "cg:type": "IndividualCondition",
      "explanation": "This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome.",
      "individual": {
        "cg:id": "Indiv029",
        "cg:type": "Individual",
        "description": "individual #029"
      },
      "condition": [
        {
          "cg:id": "MendCond058",
          "cg:type": "MendelianCondition",
          "explanation": "Hearing Loss",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ],
          "name": "Hearing Loss"
        }
      ],
      "hasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn107">EvLn107</a></p></td>
          <td>EvLn107</td>
          <td>
              <p>EvLn107</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq370">AllFreq370</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn107"><pre><code>{
  "cg:id": "EvLn107",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn107",
  "hasInformation": [
    {
      "cg:id": "AllFreq370",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC289",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "EXAC:NFE"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll121",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll127",
        "identifier": "http://reg.genome.network/allele/CA022880"
      },
      "alleleCount": 1,
      "alleleNumber": 66564,
      "alleleFrequency": 1.5e-05,
      "homozygousAlleleIndividualCount": 0
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn108">EvLn108</a></p></td>
          <td>EvLn108</td>
          <td>
              <p>EvLn108</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq037">AllFreq037</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn108"><pre><code>{
  "cg:id": "EvLn108",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn108",
  "hasInformation": [
    {
      "cg:id": "AllFreq037",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "allele": {
        "cg:id": "CanAll118",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll124",
        "identifier": "http://reg.genome.network/allele/CA013536"
      },
      "alleleCount": 31,
      "alleleNumber": 16512,
      "alleleFrequency": 0.02,
      "population": {
        "cg:id": "CC024",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "EXAC:SAS"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn109">EvLn109</a></p></td>
          <td>EvLn109</td>
          <td>
              <p>EvLn109</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq038">AllFreq038</a></p><p><a href="details/details.html#CondPrev039">CondPrev039</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn109"><pre><code>{
  "cg:id": "EvLn109",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn109",
  "hasInformation": [
    {
      "cg:id": "AllFreq038",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "allele": {
        "cg:id": "CanAll092",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll098",
        "identifier": "http://reg.genome.network/allele/CA090950"
      },
      "alleleCount": 27,
      "alleleNumber": 1384,
      "alleleFrequency": 0.02,
      "population": {
        "cg:id": "CC025",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "EXAC:AFR"
          }
        ]
      }
    },
    {
      "cg:id": "CondPrev039",
      "cg:type": "ConditionPrevelance",
      "condition": [
        {
          "cg:id": "MendCond057",
          "cg:type": "MendelianCondition",
          "explanation": "Hearing Loss",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ],
          "name": "Hearing Loss"
        }
      ],
      "minimum": 1.0,
      "maximum": 1.0,
      "population": {
        "cg:id": "CC026",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:overall",
            "cg:type": "Coding",
            "code": "overall",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Overall"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn110">EvLn110</a></p></td>
          <td>EvLn110</td>
          <td>
              <p>EvLn110</p>
         </td>
          <td>
              <p><a href="details/details.html#CC441">CC441</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess305">CritAssess305</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib522">Contrib522</a></p><p><a href="details/details.html#Contrib523">Contrib523</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn110"><pre><code>{
  "cg:id": "EvLn110",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn110",
  "hasInformation": [
    {
      "cg:id": "CritAssess305",
      "cg:type": "CriterionAssessment",
      "explanation": "while the variant is absent from exac, however\n1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say absent\n2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC).",
      "criterion": {
        "cg:id": "PM2",
        "cg:type": "Criterion",
        "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
        "shortDescription": "Absent from controls",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:pm"
        }
      },
      "variant": {
        "cg:id": "CanAll035",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll039",
        "identifier": "http://reg.genome.network/allele/CA090919"
      },
      "outcome": {
        "cg:id": "CC228",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:not-met",
            "cg:type": "Coding",
            "code": "not-met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Not Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn040",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn040",
          "hasInformation": [
            {
              "cg:id": "AllFreq082",
              "cg:type": "AlleleFrequency",
              "ascertainment": "ExAC",
              "population": {
                "cg:id": "CC067",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-population-type:combined",
                    "cg:type": "Coding",
                    "code": "combined",
                    "system": "http://clinicalgenome.org/datamodel/population-type/",
                    "display": "Combined"
                  }
                ]
              },
              "allele": {
                "cg:id": "CanAll035",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll039",
                "identifier": "http://reg.genome.network/allele/CA090919"
              },
              "alleleCount": 0,
              "alleleFrequency": 0.0,
              "medianCoverage": 11.0,
              "contribution": [
                {
                  "cg:id": "Contrib488",
                  "cg:type": "Contribution",
                  "agent": "Harry Stevenson",
                  "onDate": "6/1/16 2:15 PM",
                  "role": {
                    "cg:id": "CC390",
                    "cg:type": "CodeableConcept",
                    "coding": [
                      {
                        "cg:id": "CG-contributory-role:curator",
                        "cg:type": "Coding",
                        "code": "curator",
                        "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                        "display": "curator"
                      }
                    ]
                  }
                }
              ]
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib431",
          "cg:type": "Contribution",
          "agent": "Harry Stevenson",
          "onDate": "7/1/16 10:53 AM",
          "role": {
            "cg:id": "CC333",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        },
        {
          "cg:id": "Contrib432",
          "cg:type": "Contribution",
          "agent": "CSER Bakeoff Project",
          "onDate": "7/1/16 10:53 AM",
          "role": {
            "cg:id": "CC334",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC441",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:na",
        "cg:type": "Coding",
        "code": "na",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "None"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib522",
      "cg:type": "Contribution",
      "agent": "Harry Stevenson",
      "onDate": "7/1/16 10:53 AM",
      "role": {
        "cg:id": "CC424",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    },
    {
      "cg:id": "Contrib523",
      "cg:type": "Contribution",
      "agent": "CSER Bakeoff Project",
      "onDate": "7/1/16 10:53 AM",
      "role": {
        "cg:id": "CC425",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn111">EvLn111</a></p></td>
          <td>EvLn111</td>
          <td>
              <p>EvLn111</p>
         </td>
          <td>
              <p><a href="details/details.html#CC442">CC442</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess296">CritAssess296</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib524">Contrib524</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn111"><pre><code>{
  "cg:id": "EvLn111",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn111",
  "hasInformation": [
    {
      "cg:id": "CritAssess296",
      "cg:type": "CriterionAssessment",
      "criterion": {
        "cg:id": "PM3",
        "cg:type": "Criterion",
        "description": "For recessive disorders, detected in trans with a pathogenic variant",
        "shortDescription": "Recessive in trans with pathogenic variant",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:pm"
        }
      },
      "variant": {
        "cg:id": "CanAll035",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll039",
        "identifier": "http://reg.genome.network/allele/CA090919"
      },
      "condition": [
        {
          "cg:id": "MendCond062",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:244, HP:0000007",
          "disease": [
            {
              "cg:id": "Orphanet:244"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          }
        }
      ],
      "outcome": {
        "cg:id": "CC219",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn029",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn029",
          "hasInformation": [
            {
              "cg:id": "IndCond056",
              "cg:type": "IndividualCondition",
              "individual": {
                "cg:id": "Indiv004",
                "cg:type": "Individual",
                "description": "individual #004"
              },
              "condition": [
                {
                  "cg:id": "MendCond014",
                  "cg:type": "MendelianCondition",
                  "explanation": "PCD autosomal recessive w/ outer dynein arm (ODA) defects",
                  "name": "PCD autosomal recessive w/ outer dynein arm (ODA) defects",
                  "disease": [
                    {
                      "cg:id": "Orphanet:244"
                    }
                  ],
                  "modeOfInheritance": {
                    "cg:id": "HP:0000007",
                    "cg:type": "Coding",
                    "code": "0000007",
                    "system": "http://human-phenotype-ontology.github.io/",
                    "display": "Autosomal Recessive"
                  },
                  "phenotype": [
                    {
                      "cg:id": "HP:0200106",
                      "cg:type": "Coding",
                      "code": "0200106",
                      "system": "http://human-phenotype-ontology.github.io/",
                      "display": "Absent/shortened dynein arms"
                    }
                  ]
                }
              ],
              "hasCondition": true,
              "contribution": [
                {
                  "cg:id": "Contrib483",
                  "cg:type": "Contribution",
                  "agent": "Stefan DiMarino",
                  "onDate": "5/31/16 8:46 AM",
                  "role": {
                    "cg:id": "CC385",
                    "cg:type": "CodeableConcept",
                    "coding": [
                      {
                        "cg:id": "CG-contributory-role:curator",
                        "cg:type": "Coding",
                        "code": "curator",
                        "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                        "display": "curator"
                      }
                    ]
                  }
                }
              ]
            },
            {
              "cg:id": "VarInterp057",
              "cg:type": "VariantInterpretation",
              "variant": {
                "cg:id": "CanAll036",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll040",
                "identifier": "http://reg.genome.network/allele/CA273615"
              },
              "condition": [
                {
                  "cg:id": "MendCond062",
                  "cg:type": "MendelianCondition",
                  "explanation": "Orphanet:244, HP:0000007",
                  "disease": [
                    {
                      "cg:id": "Orphanet:244"
                    }
                  ],
                  "modeOfInheritance": {
                    "cg:id": "HP:0000007",
                    "cg:type": "Coding",
                    "code": "0000007",
                    "system": "http://human-phenotype-ontology.github.io/",
                    "display": "Autosomal Recessive"
                  }
                }
              ],
              "clinicalSignificance": {
                "cg:id": "CC043",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "LN:LA6668-3",
                    "cg:type": "Coding",
                    "code": "LA6668-3",
                    "system": "http://loinc.org/",
                    "display": "Pathogenic"
                  }
                ]
              },
              "contribution": [
                {
                  "cg:id": "Contrib485",
                  "cg:type": "Contribution",
                  "agent": "Stefan DiMarino",
                  "onDate": "5/31/16 11:01 AM",
                  "role": {
                    "cg:id": "CC387",
                    "cg:type": "CodeableConcept",
                    "coding": [
                      {
                        "cg:id": "CG-contributory-role:interpreter",
                        "cg:type": "Coding",
                        "code": "interpreter",
                        "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                        "display": "interpreter"
                      }
                    ]
                  }
                }
              ]
            },
            {
              "cg:id": "IndAll058",
              "cg:type": "IndividualAllele",
              "explanation": "Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband.",
              "individual": {
                "cg:id": "Indiv004",
                "cg:type": "Individual",
                "description": "individual #004"
              },
              "primaryAllele": {
                "cg:id": "CanAll035",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll039",
                "identifier": "http://reg.genome.network/allele/CA090919"
              },
              "secondaryAllele": {
                "cg:id": "CanAll036",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll040",
                "identifier": "http://reg.genome.network/allele/CA273615"
              },
              "primaryZygosity": {
                "cg:id": "CC044",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-allelic-state:het",
                    "cg:type": "Coding",
                    "code": "het",
                    "system": "http://clinicalgenome.org/datamodel/allelic-state/",
                    "display": "heterozygous"
                  }
                ]
              },
              "secondaryZygosity": {
                "cg:id": "CC045",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-allelic-state:het",
                    "cg:type": "Coding",
                    "code": "het",
                    "system": "http://clinicalgenome.org/datamodel/allelic-state/",
                    "display": "heterozygous"
                  }
                ]
              },
              "phase": {
                "cg:id": "CC046",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-phase:trans",
                    "cg:type": "Coding",
                    "code": "trans",
                    "system": "http://clinicalgenome.org/datamodel/phase/",
                    "display": "in trans"
                  }
                ]
              },
              "contribution": [
                {
                  "cg:id": "Contrib484",
                  "cg:type": "Contribution",
                  "agent": "Harry Stevenson",
                  "onDate": "5/31/16 9:15 AM",
                  "role": {
                    "cg:id": "CC386",
                    "cg:type": "CodeableConcept",
                    "coding": [
                      {
                        "cg:id": "CG-contributory-role:curator",
                        "cg:type": "Coding",
                        "code": "curator",
                        "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                        "display": "curator"
                      }
                    ]
                  }
                }
              ]
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib422",
          "cg:type": "Contribution",
          "agent": "Harry Stevenson",
          "onDate": "6/2/16 8:01 AM",
          "role": {
            "cg:id": "CC324",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC442",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:pm",
        "cg:type": "Coding",
        "code": "pm",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Pathogenic Moderate"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib524",
      "cg:type": "Contribution",
      "agent": "Harry Stevenson",
      "onDate": "6/2/16 8:01 AM",
      "role": {
        "cg:id": "CC426",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn112">EvLn112</a></p></td>
          <td>EvLn112</td>
          <td>
              <p>EvLn112</p>
         </td>
          <td>
              <p><a href="details/details.html#CC443">CC443</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess299">CritAssess299</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib525">Contrib525</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn112"><pre><code>{
  "cg:id": "EvLn112",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn112",
  "hasInformation": [
    {
      "cg:id": "CritAssess299",
      "cg:type": "CriterionAssessment",
      "explanation": "leads to an in-frame deletion of 7 amino acids (not repeat region).",
      "criterion": {
        "cg:id": "PM4",
        "cg:type": "Criterion",
        "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
        "shortDescription": "Protein length changes or stop-loss",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:pm"
        }
      },
      "variant": {
        "cg:id": "CanAll035",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll039",
        "identifier": "http://reg.genome.network/allele/CA090919"
      },
      "outcome": {
        "cg:id": "CC222",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn032",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn032",
          "hasInformation": [
            {
              "cg:id": "MolCon066",
              "cg:type": "MolecularConsequence",
              "contextualAllele": {
                "cg:id": "CtxAll044",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll040",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll044",
                  "identifier": "http://reg.genome.network/allele/CA273136"
                },
                "alleleName": "NP_001360.1:p.Trp2490_Leu2496del"
              },
              "consequence": {
                "cg:id": "CC054",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "SO:0001825",
                    "cg:type": "Coding",
                    "code": "0001825",
                    "system": "http://sequenceontology.org/",
                    "display": "Conservative Inframe Deletion"
                  }
                ]
              },
              "contribution": [
                {
                  "cg:id": "Contrib486",
                  "cg:type": "Contribution",
                  "agent": "Harry Stevenson",
                  "onDate": "6/1/16 2:03 PM",
                  "role": {
                    "cg:id": "CC388",
                    "cg:type": "CodeableConcept",
                    "coding": [
                      {
                        "cg:id": "CG-contributory-role:curator",
                        "cg:type": "Coding",
                        "code": "curator",
                        "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                        "display": "curator"
                      }
                    ]
                  }
                }
              ]
            },
            {
              "cg:id": "RegAnn069",
              "cg:type": "RegionAnnotation",
              "explanation": "This is not a repreating region.",
              "region": {
                "cg:id": "Loc385",
                "cg:type": "Locus",
                "referenceSequence": {
                  "cg:id": "RefSeq005",
                  "cg:type": "ReferenceSequence",
                  "accession": {
                    "cg:id": "NP_001360.1"
                  }
                },
                "start": 2490,
                "stop": 2496
              },
              "isTypes": [
                {
                  "cg:id": "CC057",
                  "cg:type": "CodeableConcept",
                  "coding": [
                    {
                      "cg:id": "CG-reg-anno-type:repeat",
                      "cg:type": "Coding",
                      "code": "repeat",
                      "system": "http://clinicalgenome.org/datamodel/region-annotation-type/",
                      "display": "Repeat"
                    }
                  ]
                }
              ],
              "contribution": [
                {
                  "cg:id": "Contrib487",
                  "cg:type": "Contribution",
                  "agent": "Stefan DiMarino",
                  "onDate": "6/1/16 2:10 PM",
                  "role": {
                    "cg:id": "CC389",
                    "cg:type": "CodeableConcept",
                    "coding": [
                      {
                        "cg:id": "CG-contributory-role:curator",
                        "cg:type": "Coding",
                        "code": "curator",
                        "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                        "display": "curator"
                      }
                    ]
                  }
                }
              ]
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib425",
          "cg:type": "Contribution",
          "agent": "Sasha Rande",
          "onDate": "6/12/16 7:30 AM",
          "role": {
            "cg:id": "CC327",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC443",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:pm",
        "cg:type": "Coding",
        "code": "pm",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Pathogenic Moderate"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib525",
      "cg:type": "Contribution",
      "agent": "Sasha Rande",
      "onDate": "6/12/16 7:30 AM",
      "role": {
        "cg:id": "CC427",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn113">EvLn113</a></p></td>
          <td>EvLn113</td>
          <td>
              <p>EvLn113</p>
         </td>
          <td>
              <p><a href="details/details.html#CC444">CC444</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess372">CritAssess372</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib526">Contrib526</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn113"><pre><code>{
  "cg:id": "EvLn113",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn113",
  "hasInformation": [
    {
      "cg:id": "CritAssess372",
      "cg:type": "CriterionAssessment",
      "explanation": "This variant is present in 2% (27/1384) of African American Chromosomes in ESP.",
      "criterion": {
        "cg:id": "BS1",
        "cg:type": "Criterion",
        "description": "Allele frequency is greater than expected for disorder",
        "shortDescription": "MAF too high for disorder",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:bs"
        }
      },
      "variant": {
        "cg:id": "CanAll092",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll098",
        "identifier": "http://reg.genome.network/allele/CA090950"
      },
      "condition": [
        {
          "cg:id": "MendCond057",
          "cg:type": "MendelianCondition",
          "explanation": "Hearing Loss",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ],
          "name": "Hearing Loss"
        }
      ],
      "outcome": {
        "cg:id": "CC291",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn109",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn109",
          "hasInformation": [
            {
              "cg:id": "AllFreq038",
              "cg:type": "AlleleFrequency",
              "ascertainment": "ExAC",
              "allele": {
                "cg:id": "CanAll092",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll098",
                "identifier": "http://reg.genome.network/allele/CA090950"
              },
              "alleleCount": 27,
              "alleleNumber": 1384,
              "alleleFrequency": 0.02,
              "population": {
                "cg:id": "CC025",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "EXAC:AFR"
                  }
                ]
              }
            },
            {
              "cg:id": "CondPrev039",
              "cg:type": "ConditionPrevelance",
              "condition": [
                {
                  "cg:id": "MendCond057",
                  "cg:type": "MendelianCondition",
                  "explanation": "Hearing Loss",
                  "disease": [
                    {
                      "cg:id": "Orphanet:87884"
                    }
                  ],
                  "name": "Hearing Loss"
                }
              ],
              "minimum": 1.0,
              "maximum": 1.0,
              "population": {
                "cg:id": "CC026",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-population-type:overall",
                    "cg:type": "Coding",
                    "code": "overall",
                    "system": "http://clinicalgenome.org/datamodel/population-type/",
                    "display": "Overall"
                  }
                ]
              }
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib509",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "03/08/2017 9:46 AM",
          "role": {
            "cg:id": "CC411",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC444",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:bs",
        "cg:type": "Coding",
        "code": "bs",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Benign Strong"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib526",
      "cg:type": "Contribution",
      "agent": "Stefan DiMarino",
      "onDate": "03/08/2017 9:46 AM",
      "role": {
        "cg:id": "CC428",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn114">EvLn114</a></p></td>
          <td>EvLn114</td>
          <td>
              <p>EvLn114</p>
         </td>
          <td>
              <p><a href="details/details.html#CC445">CC445</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess340">CritAssess340</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib527">Contrib527</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn114"><pre><code>{
  "cg:id": "EvLn114",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn114",
  "hasInformation": [
    {
      "cg:id": "CritAssess340",
      "cg:type": "CriterionAssessment",
      "explanation": "Homozygous truncating variants in LOXHD1 are associated with autosomal recessive hearing loss. This was demonstrated by both identification in human patients and a mouse model. (I used the OMIM entry as a source, but you could also directly use the papers.)",
      "criterion": {
        "cg:id": "BP1",
        "cg:type": "Criterion",
        "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
        "shortDescription": "Missense variant when truncating is expected mechanism",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:bp"
        }
      },
      "variant": {
        "cg:id": "CanAll092",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll098",
        "identifier": "http://reg.genome.network/allele/CA090950"
      },
      "condition": [
        {
          "cg:id": "MendCond059",
          "cg:type": "MendelianCondition",
          "explanation": "Hearing Loss, Autosomal Recessive",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ],
          "name": "Hearing Loss, Autosomal Recessive",
          "modeOfInheritance": {
            "cg:id": "HP:0000007",
            "cg:type": "Coding",
            "code": "0000007",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Recessive"
          }
        }
      ],
      "outcome": {
        "cg:id": "CC261",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn083",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn083",
          "hasInformation": [
            {
              "cg:id": "MolCon201",
              "cg:type": "MolecularConsequence",
              "contextualAllele": {
                "cg:id": "CtxAll098",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll092",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll098",
                  "identifier": "http://reg.genome.network/allele/CA090950"
                },
                "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
              },
              "consequence": {
                "cg:id": "CC143",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "SO:0001583",
                    "cg:type": "Coding",
                    "code": "0001583",
                    "system": "http://sequenceontology.org/",
                    "display": "missense variant"
                  }
                ]
              },
              "lof": true
            },
            {
              "cg:id": "CondMech202",
              "cg:type": "ConditionMechanism",
              "explanation": "shown in both humans and mouse models",
              "mechanism": {
                "cg:id": "CC144",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "SO:0002054",
                    "cg:type": "Coding",
                    "code": "0002054",
                    "system": "http://sequenceontology.org/",
                    "display": "loss of function variant"
                  }
                ]
              },
              "strength": {
                "cg:id": "CC145",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-cond-mech-strength:primary",
                    "cg:type": "Coding",
                    "code": "primary",
                    "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
                    "display": "Primary"
                  }
                ]
              },
              "gene": {
                "cg:id": "G048",
                "cg:type": "Gene",
                "symbol": "LOXHD1"
              },
              "condition": {
                "cg:id": "MendCond059",
                "cg:type": "MendelianCondition",
                "explanation": "Hearing Loss, Autosomal Recessive",
                "disease": [
                  {
                    "cg:id": "Orphanet:87884"
                  }
                ],
                "name": "Hearing Loss, Autosomal Recessive",
                "modeOfInheritance": {
                  "cg:id": "HP:0000007",
                  "cg:type": "Coding",
                  "code": "0000007",
                  "system": "http://human-phenotype-ontology.github.io/",
                  "display": "Autosomal Recessive"
                }
              }
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib475",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "9/6/16 10:00 AM",
          "role": {
            "cg:id": "CC377",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC445",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:bp",
        "cg:type": "Coding",
        "code": "bp",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Benign Supporting"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib527",
      "cg:type": "Contribution",
      "agent": "Harry Stevenson",
      "onDate": "7/1/16 9:23 AM",
      "role": {
        "cg:id": "CC429",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn115">EvLn115</a></p></td>
          <td>EvLn115</td>
          <td>
              <p>EvLn115</p>
         </td>
          <td>
              <p><a href="details/details.html#CC446">CC446</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess363">CritAssess363</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib528">Contrib528</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn115"><pre><code>{
  "cg:id": "EvLn115",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn115",
  "hasInformation": [
    {
      "cg:id": "CritAssess363",
      "cg:type": "CriterionAssessment",
      "explanation": "This variant was identified in one individual with pathogenic variants in another gene. Internal LMM data: The proband had one pathogenic and one likely pathogenic variant in (in trans) in SLC26A4. This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome. SLC26A4 is the only gene associated with Pendred Syndrome",
      "criterion": {
        "cg:id": "BP5",
        "cg:type": "Criterion",
        "description": "Variant found in a case with an alternate molecular basis for disease",
        "shortDescription": "Variant found in case with alternate explanation",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:bp"
        }
      },
      "variant": {
        "cg:id": "CanAll092",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll098",
        "identifier": "http://reg.genome.network/allele/CA090950"
      },
      "condition": [
        {
          "cg:id": "MendCond058",
          "cg:type": "MendelianCondition",
          "explanation": "Hearing Loss",
          "disease": [
            {
              "cg:id": "Orphanet:87884"
            }
          ],
          "name": "Hearing Loss"
        }
      ],
      "outcome": {
        "cg:id": "CC283",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn106",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn106",
          "hasInformation": [
            {
              "cg:id": "VarInterp364",
              "cg:type": "VariantInterpretation",
              "variant": {
                "cg:id": "CanAll122",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll128",
                "identifier": "http://reg.genome.network/allele/CA253304"
              },
              "condition": [
                {
                  "cg:id": "MendCond058",
                  "cg:type": "MendelianCondition",
                  "explanation": "Hearing Loss",
                  "disease": [
                    {
                      "cg:id": "Orphanet:87884"
                    }
                  ],
                  "name": "Hearing Loss"
                }
              ],
              "clinicalSignificance": {
                "cg:id": "CC284",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "LN:LA26332-9",
                    "cg:type": "Coding",
                    "code": "LA26332-9",
                    "system": "http://loinc.org/",
                    "display": "Likely Pathogenic"
                  }
                ]
              }
            },
            {
              "cg:id": "VarInterp365",
              "cg:type": "VariantInterpretation",
              "variant": {
                "cg:id": "CanAll123",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll129",
                "identifier": "http://reg.genome.network/allele/CA273665"
              },
              "condition": [
                {
                  "cg:id": "MendCond058",
                  "cg:type": "MendelianCondition",
                  "explanation": "Hearing Loss",
                  "disease": [
                    {
                      "cg:id": "Orphanet:87884"
                    }
                  ],
                  "name": "Hearing Loss"
                }
              ],
              "clinicalSignificance": {
                "cg:id": "CC285",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "LN:LA6668-3",
                    "cg:type": "Coding",
                    "code": "LA6668-3",
                    "system": "http://loinc.org/",
                    "display": "Pathogenic"
                  }
                ]
              }
            },
            {
              "cg:id": "IndAll366",
              "cg:type": "IndividualAllele",
              "individual": {
                "cg:id": "Indiv029",
                "cg:type": "Individual",
                "description": "individual #029"
              },
              "primaryAllele": {
                "cg:id": "CanAll092",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll098",
                "identifier": "http://reg.genome.network/allele/CA090950"
              },
              "secondaryAllele": {
                "cg:id": "CanAll122",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll128",
                "identifier": "http://reg.genome.network/allele/CA253304"
              },
              "phase": {
                "cg:id": "CC286",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-phase:trans",
                    "cg:type": "Coding",
                    "code": "trans",
                    "system": "http://clinicalgenome.org/datamodel/phase/",
                    "display": "in trans"
                  }
                ]
              }
            },
            {
              "cg:id": "IndAll367",
              "cg:type": "IndividualAllele",
              "individual": {
                "cg:id": "Indiv029",
                "cg:type": "Individual",
                "description": "individual #029"
              },
              "primaryAllele": {
                "cg:id": "CanAll092",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll098",
                "identifier": "http://reg.genome.network/allele/CA090950"
              },
              "secondaryAllele": {
                "cg:id": "CanAll123",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll129",
                "identifier": "http://reg.genome.network/allele/CA273665"
              },
              "phase": {
                "cg:id": "CC287",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-phase:trans",
                    "cg:type": "Coding",
                    "code": "trans",
                    "system": "http://clinicalgenome.org/datamodel/phase/",
                    "display": "in trans"
                  }
                ]
              }
            },
            {
              "cg:id": "IndCond368",
              "cg:type": "IndividualCondition",
              "explanation": "This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome.",
              "individual": {
                "cg:id": "Indiv029",
                "cg:type": "Individual",
                "description": "individual #029"
              },
              "condition": [
                {
                  "cg:id": "MendCond058",
                  "cg:type": "MendelianCondition",
                  "explanation": "Hearing Loss",
                  "disease": [
                    {
                      "cg:id": "Orphanet:87884"
                    }
                  ],
                  "name": "Hearing Loss"
                }
              ],
              "hasCondition": true
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib506",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "03/07/2017 4:30 PM",
          "role": {
            "cg:id": "CC408",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC446",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:bp",
        "cg:type": "Coding",
        "code": "bp",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Benign Supporting"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib528",
      "cg:type": "Contribution",
      "agent": "Stefan DiMarino",
      "onDate": "03/07/2017 4:30 PM",
      "role": {
        "cg:id": "CC430",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn116">EvLn116</a></p></td>
          <td>EvLn116</td>
          <td>
              <p>EvLn116</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#Func067">Func067</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn116"><pre><code>{
  "cg:id": "EvLn116",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn116",
  "hasInformation": [
    {
      "cg:id": "Func067",
      "cg:type": "FunctionalData",
      "explanation": "PMID: 23487764",
      "result": "increased ERK activtation",
      "dataType": {
        "cg:id": "CC055",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-func-data-type:activation",
            "cg:type": "Coding",
            "code": "activation",
            "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
            "display": "Activation Assay"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll130",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll124",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll130",
          "identifier": "http://reg.genome.network/allele/CA090930"
        },
        "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
      },
      "gene": {
        "cg:id": "G052",
        "cg:type": "Gene",
        "symbol": "SOS1"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn117">EvLn117</a></p></td>
          <td>EvLn117</td>
          <td>
              <p>EvLn117</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#InSilicoPred068">InSilicoPred068</a></p><p><a href="details/details.html#Conserv071">Conserv071</a></p><p><a href="details/details.html#Conserv072">Conserv072</a></p><p><a href="details/details.html#MolCon075">MolCon075</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn117"><pre><code>{
  "cg:id": "EvLn117",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn117",
  "hasInformation": [
    {
      "cg:id": "InSilicoPred068",
      "cg:type": "InSilicoPrediction",
      "explanation": "Provenance would show that this comes from Alamut",
      "contextualAllele": {
        "cg:id": "CtxAll124",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll118",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll124",
          "identifier": "http://reg.genome.network/allele/CA013536"
        },
        "alleleName": "NM_000257.2(MYH7):c.327C>T"
      },
      "type": {
        "cg:id": "CC056",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-prediction-type:sp",
            "cg:type": "Coding",
            "code": "sp",
            "system": "http://clinicalgenome.org/datamodel/prediction-type/",
            "display": "splicing prediction"
          }
        ]
      },
      "value": "no impact"
    },
    {
      "cg:id": "Conserv071",
      "cg:type": "Conservation",
      "allele": {
        "cg:id": "CanAll118",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll124",
        "identifier": "http://reg.genome.network/allele/CA013536"
      },
      "conserved": true,
      "method": "phyloP",
      "score": -0.76
    },
    {
      "cg:id": "Conserv072",
      "cg:type": "Conservation",
      "allele": {
        "cg:id": "CanAll118",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll124",
        "identifier": "http://reg.genome.network/allele/CA013536"
      },
      "conserved": true,
      "method": "PhastCons",
      "score": 0.63
    },
    {
      "cg:id": "MolCon075",
      "cg:type": "MolecularConsequence",
      "consequence": {
        "cg:id": "CC061",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001819",
            "cg:type": "Coding",
            "code": "0001819",
            "system": "http://sequenceontology.org/",
            "display": "synonymous variant"
          }
        ]
      },
      "contextualAllele": {
        "cg:id": "CtxAll124",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll118",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll124",
          "identifier": "http://reg.genome.network/allele/CA013536"
        },
        "alleleName": "NM_000257.2(MYH7):c.327C>T"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn118">EvLn118</a></p></td>
          <td>EvLn118</td>
          <td>
              <p>EvLn118</p>
         </td>
          <td>
              <p><a href="details/details.html#CC447">CC447</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess373">CritAssess373</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib529">Contrib529</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn118"><pre><code>{
  "cg:id": "EvLn118",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn118",
  "hasInformation": [
    {
      "cg:id": "CritAssess373",
      "cg:type": "CriterionAssessment",
      "explanation": "This is a synonymous variant with little conservation and no predicted impact on splicing. Conservation and splicing algorithm values are listed below.",
      "criterion": {
        "cg:id": "BP4",
        "cg:type": "Criterion",
        "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
        "shortDescription": "Computational evidence supports benign impact",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:bp"
        }
      },
      "variant": {
        "cg:id": "CanAll118",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll124",
        "identifier": "http://reg.genome.network/allele/CA013536"
      },
      "outcome": {
        "cg:id": "CC292",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn117",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn117",
          "hasInformation": [
            {
              "cg:id": "InSilicoPred068",
              "cg:type": "InSilicoPrediction",
              "explanation": "Provenance would show that this comes from Alamut",
              "contextualAllele": {
                "cg:id": "CtxAll124",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll118",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll124",
                  "identifier": "http://reg.genome.network/allele/CA013536"
                },
                "alleleName": "NM_000257.2(MYH7):c.327C>T"
              },
              "type": {
                "cg:id": "CC056",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-prediction-type:sp",
                    "cg:type": "Coding",
                    "code": "sp",
                    "system": "http://clinicalgenome.org/datamodel/prediction-type/",
                    "display": "splicing prediction"
                  }
                ]
              },
              "value": "no impact"
            },
            {
              "cg:id": "Conserv071",
              "cg:type": "Conservation",
              "allele": {
                "cg:id": "CanAll118",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll124",
                "identifier": "http://reg.genome.network/allele/CA013536"
              },
              "conserved": true,
              "method": "phyloP",
              "score": -0.76
            },
            {
              "cg:id": "Conserv072",
              "cg:type": "Conservation",
              "allele": {
                "cg:id": "CanAll118",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll124",
                "identifier": "http://reg.genome.network/allele/CA013536"
              },
              "conserved": true,
              "method": "PhastCons",
              "score": 0.63
            },
            {
              "cg:id": "MolCon075",
              "cg:type": "MolecularConsequence",
              "consequence": {
                "cg:id": "CC061",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "SO:0001819",
                    "cg:type": "Coding",
                    "code": "0001819",
                    "system": "http://sequenceontology.org/",
                    "display": "synonymous variant"
                  }
                ]
              },
              "contextualAllele": {
                "cg:id": "CtxAll124",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll118",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll124",
                  "identifier": "http://reg.genome.network/allele/CA013536"
                },
                "alleleName": "NM_000257.2(MYH7):c.327C>T"
              }
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib510",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "03/08/2017 10:02 AM",
          "role": {
            "cg:id": "CC412",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC447",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:bp",
        "cg:type": "Coding",
        "code": "bp",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Benign Supporting"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib529",
      "cg:type": "Contribution",
      "agent": "Stefan DiMarino",
      "onDate": "03/08/2017 10:02 AM",
      "role": {
        "cg:id": "CC431",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn119">EvLn119</a></p></td>
          <td>EvLn119</td>
          <td>
              <p>EvLn119</p>
         </td>
          <td>
              <p><a href="details/details.html#CC448">CC448</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess359">CritAssess359</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib530">Contrib530</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn119"><pre><code>{
  "cg:id": "EvLn119",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn119",
  "hasInformation": [
    {
      "cg:id": "CritAssess359",
      "cg:type": "CriterionAssessment",
      "criterion": {
        "cg:id": "BP7",
        "cg:type": "Criterion",
        "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
        "shortDescription": "Synonymous variant without predicted splice effect in region with low conservation",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:bp"
        }
      },
      "variant": {
        "cg:id": "CanAll118",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll124",
        "identifier": "http://reg.genome.network/allele/CA013536"
      },
      "outcome": {
        "cg:id": "CC279",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn101",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn101",
          "hasInformation": [
            {
              "cg:id": "InSilicoPred262",
              "cg:type": "InSilicoPrediction",
              "explanation": "Provenance would show that this comes from Alamut",
              "contextualAllele": {
                "cg:id": "CtxAll124",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll118",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll124",
                  "identifier": "http://reg.genome.network/allele/CA013536"
                },
                "alleleName": "NM_000257.2(MYH7):c.327C>T"
              },
              "type": {
                "cg:id": "CC189",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-prediction-type:sp",
                    "cg:type": "Coding",
                    "code": "sp",
                    "system": "http://clinicalgenome.org/datamodel/prediction-type/",
                    "display": "splicing prediction"
                  }
                ]
              },
              "value": "no impact"
            },
            {
              "cg:id": "Conserv263",
              "cg:type": "Conservation",
              "allele": {
                "cg:id": "CanAll118",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll124",
                "identifier": "http://reg.genome.network/allele/CA013536"
              },
              "conserved": true,
              "method": "phyloP",
              "score": -0.76
            },
            {
              "cg:id": "Conserv264",
              "cg:type": "Conservation",
              "allele": {
                "cg:id": "CanAll118",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll124",
                "identifier": "http://reg.genome.network/allele/CA013536"
              },
              "conserved": true,
              "method": "PhastCons",
              "score": 0.63
            },
            {
              "cg:id": "MolCon265",
              "cg:type": "MolecularConsequence",
              "consequence": {
                "cg:id": "CC190",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "SO:0001819",
                    "cg:type": "Coding",
                    "code": "0001819",
                    "system": "http://sequenceontology.org/",
                    "display": "synonymous variant"
                  }
                ]
              },
              "contextualAllele": {
                "cg:id": "CtxAll124",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll118",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll124",
                  "identifier": "http://reg.genome.network/allele/CA013536"
                },
                "alleleName": "NM_000257.2(MYH7):c.327C>T"
              }
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib503",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "10/10/16 8:05 AM",
          "role": {
            "cg:id": "CC405",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC448",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:bp",
        "cg:type": "Coding",
        "code": "bp",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Benign Supporting"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib530",
      "cg:type": "Contribution",
      "agent": "Stefan DiMarino",
      "onDate": "10/10/16 8:05 AM",
      "role": {
        "cg:id": "CC432",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn120">EvLn120</a></p></td>
          <td>EvLn120</td>
          <td>
              <p>EvLn120</p>
         </td>
          <td>
              <p><a href="details/details.html#CC449">CC449</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess371">CritAssess371</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib531">Contrib531</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn120"><pre><code>{
  "cg:id": "EvLn120",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn120",
  "hasInformation": [
    {
      "cg:id": "CritAssess371",
      "cg:type": "CriterionAssessment",
      "explanation": "This variant is present in 0.2% (31/16512) of South Asian Chromosomes in ExAC.",
      "criterion": {
        "cg:id": "BS1",
        "cg:type": "Criterion",
        "description": "Allele frequency is greater than expected for disorder",
        "shortDescription": "MAF too high for disorder",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:bs"
        }
      },
      "variant": {
        "cg:id": "CanAll118",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll124",
        "identifier": "http://reg.genome.network/allele/CA013536"
      },
      "outcome": {
        "cg:id": "CC290",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn108",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn108",
          "hasInformation": [
            {
              "cg:id": "AllFreq037",
              "cg:type": "AlleleFrequency",
              "ascertainment": "ExAC",
              "allele": {
                "cg:id": "CanAll118",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll124",
                "identifier": "http://reg.genome.network/allele/CA013536"
              },
              "alleleCount": 31,
              "alleleNumber": 16512,
              "alleleFrequency": 0.02,
              "population": {
                "cg:id": "CC024",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "EXAC:SAS"
                  }
                ]
              }
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib508",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "03/08/2017 7:43 AM",
          "role": {
            "cg:id": "CC410",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC449",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:bs",
        "cg:type": "Coding",
        "code": "bs",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Benign Strong"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib531",
      "cg:type": "Contribution",
      "agent": "Stefan DiMarino",
      "onDate": "03/08/2017 7:43 AM",
      "role": {
        "cg:id": "CC433",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn121">EvLn121</a></p></td>
          <td>EvLn121</td>
          <td>
              <p>EvLn121</p>
         </td>
          <td>
              <p><a href="details/details.html#CC450">CC450</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess361">CritAssess361</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib532">Contrib532</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn121"><pre><code>{
  "cg:id": "EvLn121",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn121",
  "hasInformation": [
    {
      "cg:id": "CritAssess361",
      "cg:type": "CriterionAssessment",
      "explanation": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).",
      "criterion": {
        "cg:id": "PVS1",
        "cg:type": "Criterion",
        "description": "null variant (nonsense, frameshift, canonical 1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
        "shortDescription": "Null variant in gene with known LOF mechanism",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:pvs"
        }
      },
      "variant": {
        "cg:id": "CanAll121",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll127",
        "identifier": "http://reg.genome.network/allele/CA022880"
      },
      "condition": [
        {
          "cg:id": "MendCond046",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:247",
          "disease": [
            {
              "cg:id": "Orphanet:247"
            }
          ]
        }
      ],
      "outcome": {
        "cg:id": "CC281",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn103",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn103",
          "hasInformation": [
            {
              "cg:id": "MolCon272",
              "cg:type": "MolecularConsequence",
              "explanation": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466).",
              "contextualAllele": {
                "cg:id": "CtxAll127",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll121",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll127",
                  "identifier": "http://reg.genome.network/allele/CA022880"
                },
                "alleleName": "NM_024422.3(DSC2): c.631-2A>G (p.?)"
              },
              "consequence": {
                "cg:id": "CC194",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "SO:0001629",
                    "cg:type": "Coding",
                    "code": "0001629",
                    "system": "http://sequenceontology.org/",
                    "display": "splice-site variant"
                  }
                ]
              },
              "lof": true
            },
            {
              "cg:id": "CondMech273",
              "cg:type": "ConditionMechanism",
              "explanation": "Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).",
              "mechanism": {
                "cg:id": "CC195",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "SO:0002054",
                    "cg:type": "Coding",
                    "code": "0002054",
                    "system": "http://sequenceontology.org/",
                    "display": "loss of function variant"
                  }
                ]
              },
              "condition": {
                "cg:id": "MendCond046",
                "cg:type": "MendelianCondition",
                "explanation": "Orphanet:247",
                "disease": [
                  {
                    "cg:id": "Orphanet:247"
                  }
                ]
              },
              "strength": {
                "cg:id": "CC196",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-cond-mech-strength:established",
                    "cg:type": "Coding",
                    "code": "established",
                    "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
                    "display": "Established"
                  }
                ]
              },
              "gene": {
                "cg:id": "G060",
                "cg:type": "Gene",
                "symbol": "DSC2"
              }
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib505",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "01/17/17 9:12 AM",
          "role": {
            "cg:id": "CC407",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC450",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:pvs",
        "cg:type": "Coding",
        "code": "pvs",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Pathogenic Very Strong"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib532",
      "cg:type": "Contribution",
      "agent": "Stefan DiMarino",
      "onDate": "01/17/17 9:12 AM",
      "role": {
        "cg:id": "CC434",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn122">EvLn122</a></p></td>
          <td>EvLn122</td>
          <td>
              <p>EvLn122</p>
         </td>
          <td>
              <p><a href="details/details.html#CC451">CC451</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess369">CritAssess369</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib533">Contrib533</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn122"><pre><code>{
  "cg:id": "EvLn122",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn122",
  "hasInformation": [
    {
      "cg:id": "CritAssess369",
      "cg:type": "CriterionAssessment",
      "explanation": "This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042).",
      "criterion": {
        "cg:id": "PM2",
        "cg:type": "Criterion",
        "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
        "shortDescription": "Absent from controls",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:pm"
        }
      },
      "variant": {
        "cg:id": "CanAll121",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll127",
        "identifier": "http://reg.genome.network/allele/CA022880"
      },
      "outcome": {
        "cg:id": "CC288",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn107",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn107",
          "hasInformation": [
            {
              "cg:id": "AllFreq370",
              "cg:type": "AlleleFrequency",
              "ascertainment": "ExAC",
              "population": {
                "cg:id": "CC289",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "EXAC:NFE"
                  }
                ]
              },
              "allele": {
                "cg:id": "CanAll121",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll127",
                "identifier": "http://reg.genome.network/allele/CA022880"
              },
              "alleleCount": 1,
              "alleleNumber": 66564,
              "alleleFrequency": 1.5e-05,
              "homozygousAlleleIndividualCount": 0
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib507",
          "cg:type": "Contribution",
          "agent": "Harry Stevenson",
          "onDate": "03/07/2017 5:22 PM",
          "role": {
            "cg:id": "CC409",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC451",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:pm",
        "cg:type": "Coding",
        "code": "pm",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Pathogenic Moderate"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib533",
      "cg:type": "Contribution",
      "agent": "Harry Stevenson",
      "onDate": "03/07/2017 5:22 PM",
      "role": {
        "cg:id": "CC435",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn123">EvLn123</a></p></td>
          <td>EvLn123</td>
          <td>
              <p>EvLn123</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#FamSegData076">FamSegData076</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn123"><pre><code>{
  "cg:id": "EvLn123",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn123",
  "hasInformation": [
    {
      "cg:id": "FamSegData076",
      "cg:type": "FamilySegregationData",
      "condition": {
        "cg:id": "MendCond056",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:648, HP:0000006",
        "disease": [
          {
            "cg:id": "Orphanet:648"
          }
        ],
        "modeOfInheritance": {
          "cg:id": "HP:0000006",
          "cg:type": "Coding",
          "code": "0000006",
          "system": "http://human-phenotype-ontology.github.io/",
          "display": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "cg:id": "CanAll124",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll130",
        "identifier": "http://reg.genome.network/allele/CA090930"
      },
      "family": {
        "cg:id": "Fam011",
        "cg:type": "Family",
        "description": "Family #011"
      },
      "phenotypePositiveAllelePositive": 0,
      "phenotypeNegativeAllelePositive": 2,
      "phenotypePositiveAlleleNegative": 0,
      "anyInconsistentSegregations": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn124">EvLn124</a></p></td>
          <td>EvLn124</td>
          <td>
              <p>EvLn124</p>
         </td>
          <td>
              <p><a href="details/details.html#CC452">CC452</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess374">CritAssess374</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib534">Contrib534</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn124"><pre><code>{
  "cg:id": "EvLn124",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn124",
  "hasInformation": [
    {
      "cg:id": "CritAssess374",
      "cg:type": "CriterionAssessment",
      "explanation": "Flag-tagged construct with the variant was transfected into HEK cells. As per the RASopathy working groups approved assays (see example 3 above), Smith et al. showed that the Y337C mutation increased levels of pERK in starved HEK cells via western blotting.",
      "criterion": {
        "cg:id": "PS3",
        "cg:type": "Criterion",
        "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
        "shortDescription": "Well-established functional studies showing damaging effect",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:ps"
        }
      },
      "variant": {
        "cg:id": "CanAll124",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll130",
        "identifier": "http://reg.genome.network/allele/CA090930"
      },
      "outcome": {
        "cg:id": "CC293",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn116",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn116",
          "hasInformation": [
            {
              "cg:id": "Func067",
              "cg:type": "FunctionalData",
              "explanation": "PMID: 23487764",
              "result": "increased ERK activtation",
              "dataType": {
                "cg:id": "CC055",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-func-data-type:activation",
                    "cg:type": "Coding",
                    "code": "activation",
                    "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
                    "display": "Activation Assay"
                  }
                ]
              },
              "contextualAllele": {
                "cg:id": "CtxAll130",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll124",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll130",
                  "identifier": "http://reg.genome.network/allele/CA090930"
                },
                "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
              },
              "gene": {
                "cg:id": "G052",
                "cg:type": "Gene",
                "symbol": "SOS1"
              }
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib513",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "3/14/2017 10:29 AM",
          "role": {
            "cg:id": "CC415",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC452",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:ps",
        "cg:type": "Coding",
        "code": "ps",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Pathogenic Strong"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib534",
      "cg:type": "Contribution",
      "agent": "Stefan DiMarino",
      "onDate": "3/14/2017 10:29 AM",
      "role": {
        "cg:id": "CC436",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn125">EvLn125</a></p></td>
          <td>EvLn125</td>
          <td>
              <p>EvLn125</p>
         </td>
          <td>
              <p><a href="details/details.html#CC453">CC453</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess375">CritAssess375</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib535">Contrib535</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn125"><pre><code>{
  "cg:id": "EvLn125",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn125",
  "hasInformation": [
    {
      "cg:id": "CritAssess375",
      "cg:type": "CriterionAssessment",
      "explanation": "A personal communication to our lab informed us of an individual and his son who both harbored the variant but with clinical workup did not appear to have Noonan Syndrome or symptoms indicative of Noonan Syndrome.",
      "criterion": {
        "cg:id": "BS4",
        "cg:type": "Criterion",
        "description": "Lack of segregation in affected members of a family",
        "shortDescription": "Does not segregate with disease",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:bs"
        }
      },
      "variant": {
        "cg:id": "CanAll124",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll130",
        "identifier": "http://reg.genome.network/allele/CA090930"
      },
      "condition": [
        {
          "cg:id": "MendCond056",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:648, HP:0000006",
          "disease": [
            {
              "cg:id": "Orphanet:648"
            }
          ],
          "modeOfInheritance": {
            "cg:id": "HP:0000006",
            "cg:type": "Coding",
            "code": "0000006",
            "system": "http://human-phenotype-ontology.github.io/",
            "display": "Autosomal Dominant"
          }
        }
      ],
      "outcome": {
        "cg:id": "CC294",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn123",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn123",
          "hasInformation": [
            {
              "cg:id": "FamSegData076",
              "cg:type": "FamilySegregationData",
              "condition": {
                "cg:id": "MendCond056",
                "cg:type": "MendelianCondition",
                "explanation": "Orphanet:648, HP:0000006",
                "disease": [
                  {
                    "cg:id": "Orphanet:648"
                  }
                ],
                "modeOfInheritance": {
                  "cg:id": "HP:0000006",
                  "cg:type": "Coding",
                  "code": "0000006",
                  "system": "http://human-phenotype-ontology.github.io/",
                  "display": "Autosomal Dominant"
                }
              },
              "canonicalAllele": {
                "cg:id": "CanAll124",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll130",
                "identifier": "http://reg.genome.network/allele/CA090930"
              },
              "family": {
                "cg:id": "Fam011",
                "cg:type": "Family",
                "description": "Family #011"
              },
              "phenotypePositiveAllelePositive": 0,
              "phenotypeNegativeAllelePositive": 2,
              "phenotypePositiveAlleleNegative": 0,
              "anyInconsistentSegregations": true
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib514",
          "cg:type": "Contribution",
          "agent": "Stefan DiMarino",
          "onDate": "3/14/2017 10:45 AM",
          "role": {
            "cg:id": "CC416",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC453",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:bs",
        "cg:type": "Coding",
        "code": "bs",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Benign Strong"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib535",
      "cg:type": "Contribution",
      "agent": "Stefan DiMarino",
      "onDate": "3/14/2017 10:45 AM",
      "role": {
        "cg:id": "CC437",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn126">EvLn126</a></p></td>
          <td>EvLn126</td>
          <td>
              <p>EvLn126</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#AllFreq101">AllFreq101</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn126"><pre><code>{
  "cg:id": "EvLn126",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn126",
  "hasInformation": [
    {
      "cg:id": "AllFreq101",
      "cg:type": "AlleleFrequency",
      "ascertainment": "ExAC",
      "population": {
        "cg:id": "CC068",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-population-type:combined",
            "cg:type": "Coding",
            "code": "combined",
            "system": "http://clinicalgenome.org/datamodel/population-type/",
            "display": "Combined"
          }
        ]
      },
      "allele": {
        "cg:id": "CanAll125",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll131",
        "identifier": "http://reg.genome.network/allele/CA280055"
      },
      "alleleCount": 0,
      "alleleFrequency": 0.0,
      "medianCoverage": 57.0
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn127">EvLn127</a></p></td>
          <td>EvLn127</td>
          <td>
              <p>EvLn127</p>
         </td>
          <td>
              <p><a href="details/details.html#CC454">CC454</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess376">CritAssess376</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib536">Contrib536</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn127"><pre><code>{
  "cg:id": "EvLn127",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn127",
  "hasInformation": [
    {
      "cg:id": "CritAssess376",
      "cg:type": "CriterionAssessment",
      "criterion": {
        "cg:id": "PM2",
        "cg:type": "Criterion",
        "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
        "shortDescription": "Absent from controls",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:pm"
        }
      },
      "variant": {
        "cg:id": "CanAll125",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll131",
        "identifier": "http://reg.genome.network/allele/CA280055"
      },
      "outcome": {
        "cg:id": "CC295",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn126",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn126",
          "hasInformation": [
            {
              "cg:id": "AllFreq101",
              "cg:type": "AlleleFrequency",
              "ascertainment": "ExAC",
              "population": {
                "cg:id": "CC068",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "CG-population-type:combined",
                    "cg:type": "Coding",
                    "code": "combined",
                    "system": "http://clinicalgenome.org/datamodel/population-type/",
                    "display": "Combined"
                  }
                ]
              },
              "allele": {
                "cg:id": "CanAll125",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll131",
                "identifier": "http://reg.genome.network/allele/CA280055"
              },
              "alleleCount": 0,
              "alleleFrequency": 0.0,
              "medianCoverage": 57.0
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib515",
          "cg:type": "Contribution",
          "agent": "ACME Molecular Lab",
          "onDate": "6/11/2011 11:45 AM",
          "role": {
            "cg:id": "CC417",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC454",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:pm",
        "cg:type": "Coding",
        "code": "pm",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Pathogenic Moderate"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib536",
      "cg:type": "Contribution",
      "agent": "ACME Molecular Lab",
      "onDate": "6/11/2011 11:45 AM",
      "role": {
        "cg:id": "CC438",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn128">EvLn128</a></p></td>
          <td>EvLn128</td>
          <td>
              <p>EvLn128</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#DeNovo106">DeNovo106</a></p><p><a href="details/details.html#IndCond110">IndCond110</a></p><p><a href="details/details.html#FamHist112">FamHist112</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn128"><pre><code>{
  "cg:id": "EvLn128",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn128",
  "hasInformation": [
    {
      "cg:id": "DeNovo106",
      "cg:type": "DeNovoAllele",
      "canonicalAllele": {
        "cg:id": "CanAll125",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll131",
        "identifier": "http://reg.genome.network/allele/CA280055"
      },
      "individual": {
        "cg:id": "Ind03"
      },
      "maternityConfirmed": true,
      "paternityConfirmed": true
    },
    {
      "cg:id": "IndCond110",
      "cg:type": "IndividualCondition",
      "individual": {
        "cg:id": "Ind030"
      },
      "condition": [
        {
          "cg:id": "MendCond069",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "hasCondition": true
    },
    {
      "cg:id": "FamHist112",
      "cg:type": "FamilyHistory",
      "proband": {
        "cg:id": "Ind030"
      },
      "condition": [
        {
          "cg:id": "MendCond069",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "familyHasCondition": true
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn129">EvLn129</a></p></td>
          <td>EvLn129</td>
          <td>
              <p>EvLn129</p>
         </td>
          <td>
              <p><a href="details/details.html#CC455">CC455</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess377">CritAssess377</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib537">Contrib537</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn129"><pre><code>{
  "cg:id": "EvLn129",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn129",
  "hasInformation": [
    {
      "cg:id": "CritAssess377",
      "cg:type": "CriterionAssessment",
      "explanation": "Parental testing in our lab confirmed this variant was present de novo in 1 proband with clinical features of CFC. (LMM internal Data)",
      "criterion": {
        "cg:id": "PM6",
        "cg:type": "Criterion",
        "description": "Assumed de novo, but without confirmation of paternity and maternity",
        "shortDescription": "Assumed de novo (parental status not confirmed)",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:pm"
        }
      },
      "variant": {
        "cg:id": "CanAll125",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll131",
        "identifier": "http://reg.genome.network/allele/CA280055"
      },
      "condition": [
        {
          "cg:id": "MendCond069",
          "cg:type": "MendelianCondition",
          "explanation": "Orphanet:1340",
          "disease": [
            {
              "cg:id": "Orphanet:1340"
            }
          ]
        }
      ],
      "outcome": {
        "cg:id": "CC296",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "contribution": [
        {
          "cg:id": "Contrib516",
          "cg:type": "Contribution",
          "agent": "ACME Molecular Lab",
          "onDate": "6/11/2011 12:45:00",
          "role": {
            "cg:id": "CC418",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC455",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:pm",
        "cg:type": "Coding",
        "code": "pm",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Pathogenic Moderate"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib537",
      "cg:type": "Contribution",
      "agent": "ACME Molecular Lab",
      "onDate": "6/11/2011 12:45:00",
      "role": {
        "cg:id": "CC439",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn130">EvLn130</a></p></td>
          <td>EvLn130</td>
          <td>
              <p>EvLn130</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#VarInterp060">VarInterp060</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn130"><pre><code>{
  "cg:id": "EvLn130",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn130",
  "hasInformation": [
    {
      "cg:id": "VarInterp060",
      "cg:type": "VariantInterpretation",
      "explanation": "The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic.",
      "variant": {
        "cg:id": "CanAll127",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll133",
        "identifier": "http://reg.genome.network/allele/CA273507"
      },
      "condition": [
        {
          "cg:id": "MendCond049",
          "cg:type": "MendelianCondition",
          "explanation": "Rasopathy",
          "disease": [
            {
              "cg:id": "Orphanet:98733"
            }
          ],
          "name": "Rasopathy"
        }
      ],
      "clinicalSignificance": {
        "cg:id": "CC048",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "LN:LA26332-9",
            "cg:type": "Coding",
            "code": "LA26332-9",
            "system": "http://loinc.org/",
            "display": "Likely Pathogenic"
          }
        ]
      },
      "contribution": [
        {
          "cg:id": "Contrib518",
          "cg:type": "Contribution",
          "agent": "ACME Molecular Lab",
          "onDate": "6/12/2011 9:00 AM",
          "role": {
            "cg:id": "CC420",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:interpreter",
                "cg:type": "Coding",
                "code": "interpreter",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "interpreter"
              }
            ]
          }
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn131">EvLn131</a></p></td>
          <td>EvLn131</td>
          <td>
              <p>EvLn131</p>
         </td>
          <td>
              <p><a href="details/details.html#CC456">CC456</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess378">CritAssess378</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Contrib538">Contrib538</a></p>
         </td>
          <!--
          <td><div class="example_view" id="EvLn131"><pre><code>{
  "cg:id": "EvLn131",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn131",
  "hasInformation": [
    {
      "cg:id": "CritAssess378",
      "cg:type": "CriterionAssessment",
      "explanation": "Another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). The other variant: NM_004333.4(BRAF):c.1449A>C (p.Lys483Asn) is classified as likely pathogenic by LMM for Rasopathy.",
      "criterion": {
        "cg:id": "PM5",
        "cg:type": "Criterion",
        "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
        "shortDescription": "Novel missense at same amino acid as known pathogenic missense",
        "defaultStrength": {
          "cg:id": "CG-evidence-strength:pm"
        }
      },
      "variant": {
        "cg:id": "CanAll125",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll131",
        "identifier": "http://reg.genome.network/allele/CA280055"
      },
      "condition": [
        {
          "cg:id": "MendCond049",
          "cg:type": "MendelianCondition",
          "explanation": "Rasopathy",
          "disease": [
            {
              "cg:id": "Orphanet:98733"
            }
          ],
          "name": "Rasopathy"
        }
      ],
      "outcome": {
        "cg:id": "CC297",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-criterion-outcome:met",
            "cg:type": "Coding",
            "code": "met",
            "system": "http://clinicalgenome.org/datamodel/criterion-assertion-outcome/",
            "display": "Met"
          }
        ]
      },
      "hasEvidence": [
        {
          "cg:id": "EvLn130",
          "cg:type": "EvidenceLine",
          "explanation": "EvLn130",
          "hasInformation": [
            {
              "cg:id": "VarInterp060",
              "cg:type": "VariantInterpretation",
              "explanation": "The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic.",
              "variant": {
                "cg:id": "CanAll127",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll133",
                "identifier": "http://reg.genome.network/allele/CA273507"
              },
              "condition": [
                {
                  "cg:id": "MendCond049",
                  "cg:type": "MendelianCondition",
                  "explanation": "Rasopathy",
                  "disease": [
                    {
                      "cg:id": "Orphanet:98733"
                    }
                  ],
                  "name": "Rasopathy"
                }
              ],
              "clinicalSignificance": {
                "cg:id": "CC048",
                "cg:type": "CodeableConcept",
                "coding": [
                  {
                    "cg:id": "LN:LA26332-9",
                    "cg:type": "Coding",
                    "code": "LA26332-9",
                    "system": "http://loinc.org/",
                    "display": "Likely Pathogenic"
                  }
                ]
              },
              "contribution": [
                {
                  "cg:id": "Contrib518",
                  "cg:type": "Contribution",
                  "agent": "ACME Molecular Lab",
                  "onDate": "6/12/2011 9:00 AM",
                  "role": {
                    "cg:id": "CC420",
                    "cg:type": "CodeableConcept",
                    "coding": [
                      {
                        "cg:id": "CG-contributory-role:interpreter",
                        "cg:type": "Coding",
                        "code": "interpreter",
                        "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                        "display": "interpreter"
                      }
                    ]
                  }
                }
              ]
            }
          ]
        }
      ],
      "contribution": [
        {
          "cg:id": "Contrib517",
          "cg:type": "Contribution",
          "agent": "ACME Molecular Lab",
          "onDate": "6/10/2011 1:45 PM",
          "role": {
            "cg:id": "CC419",
            "cg:type": "CodeableConcept",
            "coding": [
              {
                "cg:id": "CG-contributory-role:assessor",
                "cg:type": "Coding",
                "code": "assessor",
                "system": "http://clinicalgenome.org/datamodel/contributory-role/",
                "display": "assessor"
              }
            ]
          }
        }
      ]
    }
  ],
  "strength": {
    "cg:id": "CC456",
    "cg:type": "CodeableConcept",
    "coding": [
      {
        "cg:id": "CG-evidence-strength:pm",
        "cg:type": "Coding",
        "code": "pm",
        "system": "http://clinicalgenome.org/datamodel/criterion-evidence-strength/",
        "display": "Pathogenic Moderate"
      }
    ]
  },
  "contribution": [
    {
      "cg:id": "Contrib538",
      "cg:type": "Contribution",
      "agent": "ACME Molecular Lab",
      "onDate": "6/10/2011 1:45 PM",
      "role": {
        "cg:id": "CC440",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-contributory-role:assessor",
            "cg:type": "Coding",
            "code": "assessor",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "assessor"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn132">EvLn132</a></p></td>
          <td>EvLn132</td>
          <td>
              <p>EvLn132</p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#BMVR161">BMVR161</a></p><p><a href="details/details.html#MolCon169">MolCon169</a></p><p><a href="details/details.html#CondMech165">CondMech165</a></p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn132"><pre><code>{
  "cg:id": "EvLn132",
  "cg:type": "EvidenceLine",
  "explanation": "EvLn132",
  "hasInformation": [
    {
      "cg:id": "BMVR161",
      "cg:type": "BenignMissenseVariationRate",
      "explanation": "In LMM data alone, we have over 150 benign missense variants in TTN.",
      "gene": {
        "cg:id": "G003",
        "cg:type": "Gene",
        "symbol": "TTN"
      },
      "value": {
        "cg:id": "CC113",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-ben-missense-var-rate:high",
            "cg:type": "Coding",
            "code": "high",
            "system": "http://clinicalgenome.org/datamodel/benign-missense-variation-rate/",
            "display": "High"
          }
        ]
      }
    },
    {
      "cg:id": "MolCon169",
      "cg:type": "MolecularConsequence",
      "contextualAllele": {
        "cg:id": "CtxAll099",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll093",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll099",
          "identifier": "http://reg.genome.network/allele/CA256496"
        },
        "alleleName": "NM_133378.4(TTN): c.2926T>C"
      },
      "consequence": {
        "cg:id": "CC122",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001583",
            "cg:type": "Coding",
            "code": "0001583",
            "system": "http://sequenceontology.org/",
            "display": "missense variant"
          }
        ]
      },
      "lof": true
    },
    {
      "cg:id": "CondMech165",
      "cg:type": "ConditionMechanism",
      "explanation": "Taylor 2011 PMID:21810661",
      "mechanism": {
        "cg:id": "CC117",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "SO:0001583",
            "cg:type": "Coding",
            "code": "0001583",
            "system": "http://sequenceontology.org/",
            "display": "missense variant"
          }
        ]
      },
      "strength": {
        "cg:id": "CC118",
        "cg:type": "CodeableConcept",
        "coding": [
          {
            "cg:id": "CG-cond-mech-strength:common",
            "cg:type": "Coding",
            "code": "common",
            "system": "http://clinicalgenome.org/datamodel/condition-mechanism-strength/",
            "display": "Common"
          }
        ]
      },
      "gene": {
        "cg:id": "G003",
        "cg:type": "Gene",
        "symbol": "TTN"
      },
      "condition": {
        "cg:id": "MendCond054",
        "cg:type": "MendelianCondition",
        "explanation": "Orphanet:247",
        "disease": [
          {
            "cg:id": "Orphanet:247"
          }
        ]
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("cg:id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("cg:id")').next('.string')[0].innerText + ', </em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em> [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../javascripts/jquery.js"></script><script src="../javascripts/bootstrap.js"></script><script src="../javascripts/jquery.jsonview.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
