<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../stylesheets/brand2.css" rel="stylesheet" /><link href="../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/evidence_line.html">0.1</a></li>
                  <li><a href="/development/tech/evidence_line.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../">Interpretation Model</a></li>
<li><a href="./">Technical Reference and Examples</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../images/clingen-doc-logo.png" />
            </li>

            <li class="active">
              <a href="../">Welcome</a>
            </li>
            <li><a href="../user/">User Guide</a></li>
            <li class="active"><a href="./">Technical Reference</a></li><ul><li><a href="criterion_assessment.html">CriterionAssessment</a></li><li><a href="evidence/">Data</a></li><li class="active"><a href="evidence_line.html">EvidenceLine</a></li><li><a href="variant_interpretation.html">VariantInterpretation</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>EvidenceLine</h1>
            <p>A reification of the underlying supporting data which may be one or more data points or assertions. Typically grouped in a specific manner for use in an owning assertion activity (either Variant Interpretation or CriterionAssessment. All evidence may optionally be given a strength by an agent.</p>

                <p>Descended from Entity</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>description</td>
                <td>String</td>
                <td>1..1</td>
                <td></td>
            </tr>
            <tr>
                <td>strength</td>
                <td>CodeableConcept</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>supportingData</td>
                <td>Data</td>
                <td>0..*</td>
                <td></td>
            </tr>
            <tr>
                <td>strengthBy</td>
                <td>Agent</td>
                <td>0..1</td>
                <td></td>
            </tr>
            <tr>
                <td>strengthOn</td>
                <td>Datetime</td>
                <td>0..1</td>
                <td></td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A human-readable description of the entity</td>
                </tr>
                <tr>
                    <td>wasGeneratedBy</td>
                    <td>Activity</td>
                    <td>0..1</td>
                    <td></td>
                    <td>The activity that produced this entity</td>
                </tr>
                <tr>
                    <td>wasDerivedFrom</td>
                    <td>Data</td>
                    <td>0..1</td>
                    <td></td>
                    <td>Another entityy which was a former derivation of this entity.</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>description</th>
        <th>description</th>
        <th>strength</th>
        <th>supportingData</th>
        <th>strengthBy</th>
        <th>strengthOn</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="details/details.html#EvLn110">EvLn110</a></p></td>
          <td></td>
          <td>
              
         </td>
          <td>
              <p>CG:INSUFFICIENT</p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess305">CritAssess305</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn110"><pre><code>{
  "cg:id": "EvLn110",
  "cg:type": "EvidenceLine",
  "strength": "CG:INSUFFICIENT",
  "supportingData": [
    {
      "cg:id": "CritAssess305",
      "cg:type": "CriterionAssessment",
      "explanation": "while the variant is absent from exac, however\n1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”\n2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC).",
      "wasGeneratedBy": {
        "cg:id": "Act031",
        "cg:type": "Activity",
        "activityTypeId": {
          "cg:id": "E10"
        },
        "timeStamp": "2016-07-01T10:53:00+00:00",
        "wasAssociatedWith": [
          {
            "cg:id": "Agent001",
            "cg:type": "Agent",
            "description": "SH",
            "name": "Steven Harrison"
          },
          {
            "cg:id": "Agent005",
            "cg:type": "Agent",
            "description": "CSER",
            "name": "CSER Bake Off Project"
          }
        ]
      },
      "criterion": {
        "cg:id": "PM2",
        "cg:type": "Criterion",
        "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
        "shortDescription": "Absent from controls",
        "defaultOutcome": "CG:M",
        "targetInterpretation": "LN:LA6668-3"
      },
      "variant": {
        "cg:id": "CanAll035",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll039",
        "identifier": "http://reg.genome.network/allele/CA090919"
      },
      "condition": [
        {
          "cg:id": "-"
        }
      ],
      "outcome": "CG:INSUFFICIENT",
      "hasSupportingEvidence": [
        {
          "cg:id": "EvLn040",
          "supportingData": [
            {
              "cg:id": "AllFreq082",
              "cg:type": "AlleleFrequency",
              "wasGeneratedBy": {
                "cg:id": "Act081",
                "cg:type": "Activity",
                "activityTypeId": {
                  "cg:id": "E10"
                },
                "timeStamp": "2016-06-01T14:15:00+00:00",
                "wasAssociatedWith": [
                  {
                    "cg:id": "Agent001",
                    "cg:type": "Agent",
                    "description": "SH",
                    "name": "Steven Harrison"
                  }
                ]
              },
              "ascertainment": "ExAC",
              "population": "CG:combined",
              "allele": {
                "cg:id": "CanAll035",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll039",
                "identifier": "http://reg.genome.network/allele/CA090919"
              },
              "alleleCount": 0,
              "alleleFrequency": 0.0,
              "medianCoverage": 11.0
            }
          ]
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn111">EvLn111</a></p></td>
          <td></td>
          <td>
              
         </td>
          <td>
              <p>CG:MODERATE</p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess296">CritAssess296</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn111"><pre><code>{
  "cg:id": "EvLn111",
  "cg:type": "EvidenceLine",
  "strength": "CG:MODERATE",
  "supportingData": [
    {
      "cg:id": "CritAssess296",
      "cg:type": "CriterionAssessment",
      "wasGeneratedBy": {
        "cg:id": "Act022",
        "cg:type": "Activity",
        "activityTypeId": {
          "cg:id": "E10"
        },
        "timeStamp": "2016-06-02T08:00:59+00:00",
        "wasAssociatedWith": [
          {
            "cg:id": "Agent001",
            "cg:type": "Agent",
            "description": "SH",
            "name": "Steven Harrison"
          }
        ]
      },
      "criterion": {
        "cg:id": "PM3",
        "cg:type": "Criterion",
        "description": "For recessive disorders, detected in trans with a pathogenic variant",
        "shortDescription": "Recessive in trans with pathogenic variant",
        "defaultOutcome": "CG:M",
        "targetInterpretation": "LN:LA6668-3"
      },
      "variant": {
        "cg:id": "CanAll035",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll039",
        "identifier": "http://reg.genome.network/allele/CA090919"
      },
      "condition": [
        {
          "cg:id": "MendCond004",
          "cg:type": "MendelianCondition",
          "explanation": "Primary ciliary dyskenisia, Autosomal Recessive"
        }
      ],
      "outcome": "CG:MET",
      "hasSupportingEvidence": [
        {
          "cg:id": "EvLn029",
          "supportingData": [
            {
              "cg:id": "IndCond056",
              "cg:type": "IndividualCondition",
              "wasGeneratedBy": {
                "cg:id": "Act077",
                "cg:type": "Activity",
                "activityTypeId": {
                  "cg:id": "E10"
                },
                "timeStamp": "2016-05-31T09:14:59+00:00",
                "wasAssociatedWith": [
                  {
                    "cg:id": "Agent001",
                    "cg:type": "Agent",
                    "description": "SH",
                    "name": "Steven Harrison"
                  }
                ]
              },
              "individual": {
                "cg:id": "Indiv004",
                "cg:type": "Individual",
                "description": "individual #004"
              },
              "condition": [
                {
                  "cg:id": "MendCond014",
                  "cg:type": "MendelianCondition",
                  "explanation": "PCD autosomal recessive w/ outer dynein arm (ODA) defects"
                }
              ],
              "hasCondition": true
            },
            {
              "cg:id": "VarInterp057",
              "cg:type": "VariantInterpretation",
              "wasGeneratedBy": {
                "cg:id": "Act078",
                "cg:type": "Activity",
                "activityTypeId": {
                  "cg:id": "E10"
                },
                "timeStamp": "2016-05-31T11:00:59+00:00",
                "wasAssociatedWith": [
                  {
                    "cg:id": "Agent008",
                    "cg:type": "Agent",
                    "description": "MDS",
                    "name": "Marina DiStefano"
                  }
                ]
              },
              "variant": {
                "cg:id": "CanAll036",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll040",
                "identifier": "http://reg.genome.network/allele/CA273615"
              },
              "condition": [
                {
                  "cg:id": "MendCond004",
                  "cg:type": "MendelianCondition",
                  "explanation": "Primary ciliary dyskenisia, Autosomal Recessive"
                }
              ],
              "clinicalSignificance": "LN:LA6668-3"
            },
            {
              "cg:id": "IndAll058",
              "cg:type": "IndividualAllele",
              "explanation": "Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband.",
              "wasGeneratedBy": {
                "cg:id": "Act077",
                "cg:type": "Activity",
                "activityTypeId": {
                  "cg:id": "E10"
                },
                "timeStamp": "2016-05-31T09:14:59+00:00",
                "wasAssociatedWith": [
                  {
                    "cg:id": "Agent001",
                    "cg:type": "Agent",
                    "description": "SH",
                    "name": "Steven Harrison"
                  }
                ]
              },
              "individual": {
                "cg:id": "Indiv004",
                "cg:type": "Individual",
                "description": "individual #004"
              },
              "primaryAllele": {
                "cg:id": "CanAll035",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll039",
                "identifier": "http://reg.genome.network/allele/CA090919"
              },
              "secondaryAllele": {
                "cg:id": "CanAll036",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll040",
                "identifier": "http://reg.genome.network/allele/CA273615"
              },
              "primaryZygosity": "CG:het",
              "secondaryZygosity": "CG:het",
              "phase": "CG:trans"
            }
          ]
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn112">EvLn112</a></p></td>
          <td></td>
          <td>
              
         </td>
          <td>
              <p>CG:MODERATE</p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess299">CritAssess299</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn112"><pre><code>{
  "cg:id": "EvLn112",
  "cg:type": "EvidenceLine",
  "strength": "CG:MODERATE",
  "supportingData": [
    {
      "cg:id": "CritAssess299",
      "cg:type": "CriterionAssessment",
      "explanation": "leads to an in-frame deletion of 7 amino acids (not repeat region).",
      "wasGeneratedBy": {
        "cg:id": "Act025",
        "cg:type": "Activity",
        "activityTypeId": {
          "cg:id": "E10"
        },
        "timeStamp": "2016-06-12T07:30:00+00:00",
        "wasAssociatedWith": [
          {
            "cg:id": "Agent003",
            "cg:type": "Agent",
            "description": "TS",
            "name": "Tasha Strande"
          }
        ]
      },
      "criterion": {
        "cg:id": "PM4",
        "cg:type": "Criterion",
        "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
        "shortDescription": "Protein length changes or stop-loss",
        "defaultOutcome": "CG:M",
        "targetInterpretation": "LN:LA6668-3"
      },
      "variant": {
        "cg:id": "CanAll035",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll039",
        "identifier": "http://reg.genome.network/allele/CA090919"
      },
      "condition": [
        {
          "cg:id": "-"
        }
      ],
      "outcome": "CG:MET",
      "hasSupportingEvidence": [
        {
          "cg:id": "EvLn032",
          "supportingData": [
            {
              "cg:id": "MolCon066",
              "cg:type": "MolecularConsequence",
              "wasGeneratedBy": {
                "cg:id": "Act079",
                "cg:type": "Activity",
                "activityTypeId": {
                  "cg:id": "E10"
                },
                "timeStamp": "2016-06-01T14:03:00+00:00",
                "wasAssociatedWith": [
                  {
                    "cg:id": "Agent001",
                    "cg:type": "Agent",
                    "description": "SH",
                    "name": "Steven Harrison"
                  }
                ]
              },
              "contextualAllele": {
                "cg:id": "CtxAll044",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll040",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll044",
                  "identifier": "http://reg.genome.network/allele/CA273136"
                },
                "alleleName": "NP_001360.1:p.Trp2490_Leu2496del"
              },
              "consequence": "SO:0001825"
            },
            {
              "cg:id": "Reg069",
              "cg:type": "RegionTypesAndAlleles",
              "explanation": "This is not a repreating region.",
              "wasGeneratedBy": {
                "cg:id": "Act080",
                "cg:type": "Activity",
                "activityTypeId": {
                  "cg:id": "E10"
                },
                "timeStamp": "2016-06-01T14:10:00+00:00",
                "wasAssociatedWith": [
                  {
                    "cg:id": "Agent008",
                    "cg:type": "Agent",
                    "description": "MDS",
                    "name": "Marina DiStefano"
                  }
                ]
              },
              "referenceSequence": {
                "cg:id": "RefSeq005",
                "cg:type": "ReferenceSequence",
                "identifier": "NP_001360.1"
              },
              "start": 2490,
              "stop": 2496,
              "isNotTypes": [
                "CG:repeat"
              ]
            }
          ]
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn113">EvLn113</a></p></td>
          <td></td>
          <td>
              
         </td>
          <td>
              <p>CG:STRONG</p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess372">CritAssess372</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn113"><pre><code>{
  "cg:id": "EvLn113",
  "cg:type": "EvidenceLine",
  "strength": "CG:STRONG",
  "supportingData": [
    {
      "cg:id": "CritAssess372",
      "cg:type": "CriterionAssessment",
      "explanation": "This variant is present in 2% (27/1384) of African American Chromosomes in ESP.",
      "wasGeneratedBy": {
        "cg:id": "Act102",
        "cg:type": "Activity",
        "activityTypeId": {
          "cg:id": "E10"
        },
        "timeStamp": "2017-03-08T09:46:00+00:00",
        "wasAssociatedWith": [
          {
            "cg:id": "Agent008",
            "cg:type": "Agent",
            "description": "MDS",
            "name": "Marina DiStefano"
          }
        ]
      },
      "criterion": {
        "cg:id": "BS1",
        "cg:type": "Criterion",
        "description": "Allele frequency is greater than expected for disorder",
        "shortDescription": "MAF too high for disorder",
        "defaultOutcome": "CG:S",
        "targetInterpretation": "LN:LA6675-8"
      },
      "variant": {
        "cg:id": "CanAll092",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll098",
        "identifier": "http://reg.genome.network/allele/CA090950"
      },
      "outcome": "CG:MET",
      "hasSupportingEvidence": [
        {
          "cg:id": "EvLn109",
          "supportingData": [
            {
              "cg:id": "AllFreq038",
              "cg:type": "AlleleFrequency",
              "ascertainment": "ExAC",
              "allele": {
                "cg:id": "CanAll092",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll098",
                "identifier": "http://reg.genome.network/allele/CA090950"
              },
              "alleleCount": 27,
              "alleleNumber": 1384,
              "alleleFrequency": 0.02,
              "population": "EX:afr"
            },
            {
              "cg:id": "CondPrev039",
              "cg:type": "ConditionPrevelance",
              "condition": [
                {
                  "cg:id": "MendCond010",
                  "cg:type": "MendelianCondition",
                  "explanation": "nonsyndromic deafness, Autosomal Recessive, GIPC3"
                }
              ],
              "minimum": 1.0,
              "maximum": 1.0,
              "population": "CG:overall"
            }
          ]
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn114">EvLn114</a></p></td>
          <td></td>
          <td>
              
         </td>
          <td>
              <p>CG:SUPPORTING</p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess340">CritAssess340</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn114"><pre><code>{
  "cg:id": "EvLn114",
  "cg:type": "EvidenceLine",
  "strength": "CG:SUPPORTING",
  "supportingData": [
    {
      "cg:id": "CritAssess340",
      "cg:type": "CriterionAssessment",
      "explanation": "Homozygous truncating variants in LOXHD1 are associated with autosomal recessive hearing loss. This was demonstrated by both identification in human patients and a mouse model. (I used the OMIM entry as a source, but you could also directly use the papers.)",
      "wasGeneratedBy": {
        "cg:id": "Act068",
        "cg:type": "Activity",
        "activityTypeId": {
          "cg:id": "E10"
        },
        "timeStamp": "2016-09-06T09:59:59+00:00",
        "wasAssociatedWith": [
          {
            "cg:id": "Agent008",
            "cg:type": "Agent",
            "description": "MDS",
            "name": "Marina DiStefano"
          }
        ]
      },
      "criterion": {
        "cg:id": "BP1",
        "cg:type": "Criterion",
        "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
        "shortDescription": "Missense variant when truncating is expected mechanism",
        "defaultOutcome": "CG:P",
        "targetInterpretation": "LN:LA6675-8"
      },
      "variant": {
        "cg:id": "CanAll092",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll098",
        "identifier": "http://reg.genome.network/allele/CA090950"
      },
      "condition": [
        {
          "cg:id": "MendCond010",
          "cg:type": "MendelianCondition",
          "explanation": "nonsyndromic deafness, Autosomal Recessive, GIPC3"
        }
      ],
      "outcome": "CG:MET",
      "hasSupportingEvidence": [
        {
          "cg:id": "EvLn083",
          "supportingData": [
            {
              "cg:id": "MolCon201",
              "cg:type": "MolecularConsequence",
              "contextualAllele": {
                "cg:id": "CtxAll098",
                "cg:type": "ContextualAllele",
                "relatedCanonicalAllele": {
                  "cg:id": "CanAll092",
                  "cg:type": "CanonicalAllele",
                  "preferredCtxAllele": "CtxAll098",
                  "identifier": "http://reg.genome.network/allele/CA090950"
                },
                "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
              },
              "consequence": "SO:0001583",
              "lof": true
            },
            {
              "cg:id": "CondMech202",
              "cg:type": "ConditionMechanism",
              "explanation": "shown in both humans and mouse models",
              "mechanism": "SO:0002054",
              "strength": "CG:primary",
              "gene": {
                "cg:id": "G048",
                "cg:type": "Gene",
                "symbol": "LOXHD1"
              },
              "condition": {
                "cg:id": "MendCond010",
                "cg:type": "MendelianCondition",
                "explanation": "nonsyndromic deafness, Autosomal Recessive, GIPC3"
              }
            }
          ]
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn115">EvLn115</a></p></td>
          <td></td>
          <td>
              
         </td>
          <td>
              <p>CG:SUPPORTING</p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess363">CritAssess363</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn115"><pre><code>{
  "cg:id": "EvLn115",
  "cg:type": "EvidenceLine",
  "strength": "CG:SUPPORTING",
  "supportingData": [
    {
      "cg:id": "CritAssess363",
      "cg:type": "CriterionAssessment",
      "explanation": "This variant was identified in one individual with pathogenic variants in another gene. Internal LMM data: The proband had one pathogenic and one likely pathogenic variant in (in trans) in SLC26A4. This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome. SLC26A4 is the only gene associated with Pendred Syndrome",
      "wasGeneratedBy": {
        "cg:id": "Act099",
        "cg:type": "Activity",
        "activityTypeId": {
          "cg:id": "E10"
        },
        "timeStamp": "2017-03-07T16:30:00+00:00",
        "wasAssociatedWith": [
          {
            "cg:id": "Agent008",
            "cg:type": "Agent",
            "description": "MDS",
            "name": "Marina DiStefano"
          }
        ]
      },
      "criterion": {
        "cg:id": "BP5",
        "cg:type": "Criterion",
        "description": "Variant found in a case with an alternate molecular basis for disease",
        "shortDescription": "Variant found in case with alternate explanation",
        "defaultOutcome": "CG:P",
        "targetInterpretation": "LN:LA6675-8"
      },
      "variant": {
        "cg:id": "CanAll092",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll098",
        "identifier": "http://reg.genome.network/allele/CA090950"
      },
      "condition": [
        {
          "cg:id": "MendCond010",
          "cg:type": "MendelianCondition",
          "explanation": "nonsyndromic deafness, Autosomal Recessive, GIPC3"
        }
      ],
      "outcome": "CG:MET",
      "hasSupportingEvidence": [
        {
          "cg:id": "EvLn106",
          "supportingData": [
            {
              "cg:id": "VarInterp364",
              "cg:type": "VariantInterpretation",
              "variant": {
                "cg:id": "CanAll122",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll128"
              },
              "condition": [
                {
                  "cg:id": "MendCond010",
                  "cg:type": "MendelianCondition",
                  "explanation": "nonsyndromic deafness, Autosomal Recessive, GIPC3"
                }
              ],
              "clinicalSignificance": "LN:LA26332-9"
            },
            {
              "cg:id": "VarInterp365",
              "cg:type": "VariantInterpretation",
              "variant": {
                "cg:id": "CanAll123",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll129"
              },
              "condition": [
                {
                  "cg:id": "MendCond010",
                  "cg:type": "MendelianCondition",
                  "explanation": "nonsyndromic deafness, Autosomal Recessive, GIPC3"
                }
              ],
              "clinicalSignificance": "LN:LA6668-3"
            },
            {
              "cg:id": "IndAll366",
              "cg:type": "IndividualAllele",
              "individual": {
                "cg:id": "Indiv029",
                "cg:type": "Individual",
                "description": "individual #029"
              },
              "primaryAllele": {
                "cg:id": "CanAll092",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll098",
                "identifier": "http://reg.genome.network/allele/CA090950"
              },
              "secondaryAllele": {
                "cg:id": "CanAll122",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll128"
              },
              "phase": "CG:trans"
            },
            {
              "cg:id": "IndAll367",
              "cg:type": "IndividualAllele",
              "individual": {
                "cg:id": "Indiv029",
                "cg:type": "Individual",
                "description": "individual #029"
              },
              "primaryAllele": {
                "cg:id": "CanAll092",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll098",
                "identifier": "http://reg.genome.network/allele/CA090950"
              },
              "secondaryAllele": {
                "cg:id": "CanAll123",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll129"
              },
              "phase": "CG:trans"
            },
            {
              "cg:id": "IndCond368",
              "cg:type": "IndividualCondition",
              "explanation": "This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome.",
              "individual": {
                "cg:id": "Indiv029",
                "cg:type": "Individual",
                "description": "individual #029"
              },
              "condition": [
                {
                  "cg:id": "MendCond010",
                  "cg:type": "MendelianCondition",
                  "explanation": "nonsyndromic deafness, Autosomal Recessive, GIPC3"
                }
              ],
              "hasCondition": true
            }
          ]
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn116">EvLn116</a></p></td>
          <td></td>
          <td>
              
         </td>
          <td>
              <p>CG:STRONG</p>
         </td>
          <td>
              <p><a href="details/details.html#Func067">Func067</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn116"><pre><code>{
  "cg:id": "EvLn116",
  "cg:type": "EvidenceLine",
  "strength": "CG:STRONG",
  "supportingData": [
    {
      "cg:id": "Func067",
      "cg:type": "FunctionalData",
      "explanation": "PMID: 23487764",
      "result": "increased ERK activtation",
      "dataType": "CG:activation",
      "contextualAllele": {
        "cg:id": "CtxAll130",
        "cg:type": "ContextualAllele",
        "relatedCanonicalAllele": {
          "cg:id": "CanAll124",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll130"
        },
        "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
      },
      "gene": {
        "cg:id": "G052",
        "cg:type": "Gene",
        "symbol": "SOS1"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn127">EvLn127</a></p></td>
          <td></td>
          <td>
              
         </td>
          <td>
              <p>CG:MODERATE</p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess376">CritAssess376</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn127"><pre><code>{
  "cg:id": "EvLn127",
  "cg:type": "EvidenceLine",
  "strength": "CG:MODERATE",
  "supportingData": [
    {
      "cg:id": "CritAssess376",
      "cg:type": "CriterionAssessment",
      "wasGeneratedBy": {
        "cg:id": "Act108",
        "cg:type": "Activity",
        "activityTypeId": {
          "cg:id": "E10"
        },
        "timeStamp": "2017-03-14T11:45:00+00:00",
        "wasAssociatedWith": [
          {
            "cg:id": "Agent008",
            "cg:type": "Agent",
            "description": "MDS",
            "name": "Marina DiStefano"
          }
        ]
      },
      "criterion": {
        "cg:id": "PM2",
        "cg:type": "Criterion",
        "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
        "shortDescription": "Absent from controls",
        "defaultOutcome": "CG:M",
        "targetInterpretation": "LN:LA6668-3"
      },
      "variant": {
        "cg:id": "CanAll125",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll131"
      },
      "outcome": "CG:MET",
      "hasSupportingEvidence": [
        {
          "cg:id": "EvLn126",
          "supportingData": [
            {
              "cg:id": "AllFreq101",
              "cg:type": "AlleleFrequency",
              "ascertainment": "ExAC",
              "population": "CG:combined",
              "allele": {
                "cg:id": "CanAll125",
                "cg:type": "CanonicalAllele",
                "preferredCtxAllele": "CtxAll131"
              },
              "alleleCount": 0,
              "alleleFrequency": 0.0,
              "medianCoverage": 57.0
            }
          ]
        }
      ]
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#EvLn129">EvLn129</a></p></td>
          <td></td>
          <td>
              
         </td>
          <td>
              <p>CG:MODERATE</p>
         </td>
          <td>
              <p><a href="details/details.html#CritAssess377">CritAssess377</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="EvLn129"><pre><code>{
  "cg:id": "EvLn129",
  "cg:type": "EvidenceLine",
  "strength": "CG:MODERATE",
  "supportingData": [
    {
      "cg:id": "CritAssess377",
      "cg:type": "CriterionAssessment",
      "explanation": "Parental testing in our lab confirmed this variant was present de novo in 1 proband with clinical features of CFC. (LMM internal Data)",
      "wasGeneratedBy": {
        "cg:id": "Act109",
        "cg:type": "Activity",
        "activityTypeId": {
          "cg:id": "E10"
        },
        "timeStamp": "2017-03-14T12:45:00+00:00",
        "wasAssociatedWith": [
          {
            "cg:id": "Agent008",
            "cg:type": "Agent",
            "description": "MDS",
            "name": "Marina DiStefano"
          }
        ]
      },
      "criterion": {
        "cg:id": "PM6",
        "cg:type": "Criterion",
        "description": "Assumed de novo, but without confirmation of paternity and maternity",
        "shortDescription": "Assumed de novo (parental status not confirmed)",
        "defaultOutcome": "CG:M",
        "targetInterpretation": "LN:LA6668-3"
      },
      "variant": {
        "cg:id": "CanAll125",
        "cg:type": "CanonicalAllele",
        "preferredCtxAllele": "CtxAll131"
      },
      "outcome": "CG:MET"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("cg:id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("cg:id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../javascripts/jquery.js"></script><script src="../javascripts/bootstrap.js"></script><script src="../javascripts/jquery.jsonview.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
