<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../stylesheets/brand2.css" rel="stylesheet" /><link href="../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/variant_interpretation.html">0.1</a></li>
                  <li><a href="/development/tech/variant_interpretation.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../">Interpretation Model</a></li>
<li><a href="./">Technical Reference and Examples</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../images/clingen-doc-logo.png" />
            </li>

            <li class="active">
              <a href="../">Welcome</a>
            </li>
            <li><a href="../user/">User Guide</a></li>
            <li class="active"><a href="./">Technical Reference</a></li><ul><li><a href="criterion_assessment.html">CriterionAssessment</a></li><li><a href="evidence/">Data</a></li><li><a href="evidence_line.html">EvidenceLine</a></li><li class="active"><a href="variant_interpretation.html">VariantInterpretation</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>VariantInterpretation</h1>
            <p>Evidence-based conclusion about the pathogenicity of a sequence variant</p>

                <p>Descended from Data</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>A VariantInterpretation is a statement about the pathogenicity of a given CanonicalAllele, usually for a given MendelianCondition.  A VariantInterpretation does not explicitly contain the reasoning and evidence supporting the conclusion, but that information is referenced in the provenance of the VariantInterpretation entity.  That is, the VariantInterpretation is, for example, the statement that a variant is pathogenic, the reasoning for that conclusion is found by following the EvidenceLines of the VariantInterpretation.</p>

<p>VariantInterpretations for pathogenic variants will include a MendelianCondition for which the conclusion was reached.  However, if the variant is determined to be benign (such as in the case of a very common polymorphism), then the condition attribute may be unused indicating that the allele is benign for all conditions.  If a Condition is specified for a benign variant, then the conclusion that the allele is benign is limited to that specific condition.</p>

<p>While the EvidenceLines should be sufficient to understand the reasoning behind a given VariantInterpretation, a human-readable statement of this reasoning may be included in the explanation attribute (inherited from Entity).</p>

<p>VariantInterpretations are frequently updated when new information becomes available, or during periodic review.  In such a case, a new VariantInterpretation is created.  A tie to the original VariantInterpretation can be maintained using the wasDerivedFrom attribute, which indicates the previous version of the current VariantInterpretation.</p>

<h2 id="primary-examples">Primary Examples</h2>

<p><a href="details/details.html#VarInterp274">VarInterp274</a> and
<a href="details/details.html#VarInterp063">VarInterp063</a> are fully-realized examples
and may serve as a starting point for exploring the model.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>variant</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>The variant whose pathogenicity is being described</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>MendelianCondition</td>
                <td>0..*</td>
                <td>The condition for which the variant's pathogenicity is described</td>
            </tr>
            <tr>
                <td>clinicalSignificance</td>
                <td>CodeableConcept</td>
                <td>1..1</td>
                <td>The coded pathogenicity of the variant for the condition</td>
            </tr>
            <tr>
                <td>assertionMethod</td>
                <td>AssertionMethod</td>
                <td>0..1</td>
                <td>The  framework that formally defines the methodology used in creating this interpretation.</td>
            </tr>
            <tr>
                <td>hasEvidence</td>
                <td>EvidenceLine</td>
                <td>0..*</td>
                <td>** TODO **</td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A human-readable description of the entity</td>
                </tr>
                <tr>
                    <td>contribution</td>
                    <td>Contribution</td>
                    <td>0..*</td>
                    <td></td>
                    <td>The contributions (agent, time and role) made to this entity</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>description</th>
        <th>variant</th>
        <th>condition</th>
        <th>clinicalSignificance</th>
        <th>assertionMethod</th>
        <th>hasEvidence</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="details/details.html#VarInterp008">VarInterp008</a></p></td>
          <td>NM_002834.3:c.855T>G Pathogenic for Noonan Syndrome</td>
          <td>
              <p><a href="details/details.html#CanAll005">CanAll005</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond003">MendCond003</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp008"><pre><code>{
  "cg:id": "VarInterp008",
  "cg:type": "VariantInterpretation",
  "explanation": "NM_002834.3:c.855T>G Pathogenic for Noonan Syndrome",
  "variant": {
    "cg:id": "CanAll005",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll006",
    "identifier": "http://reg.genome.network/allele/CA220155"
  },
  "condition": [
    {
      "cg:id": "MendCond003",
      "cg:type": "MendelianCondition",
      "explanation": "noonan syndrome, Autosomal Dominant",
      "disease": [
        "Orphanet:648"
      ],
      "modeOfInheritance": "HP:0000006"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp009">VarInterp009</a></p></td>
          <td>NM_001369.2:c.6249G>A Pathogenic for PCD</td>
          <td>
              <p><a href="details/details.html#CanAll008">CanAll008</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond004">MendCond004</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp009"><pre><code>{
  "cg:id": "VarInterp009",
  "cg:type": "VariantInterpretation",
  "explanation": "NM_001369.2:c.6249G>A Pathogenic for PCD",
  "variant": {
    "cg:id": "CanAll008",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll009",
    "identifier": "http://reg.genome.network/allele/CA501221"
  },
  "condition": [
    {
      "cg:id": "MendCond004",
      "cg:type": "MendelianCondition",
      "explanation": "Primary ciliary dyskenisia",
      "disease": [
        "Orphanet:244"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp010">VarInterp010</a></p></td>
          <td>NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Noonan Syndrome 3</td>
          <td>
              <p><a href="details/details.html#CanAll011">CanAll011</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond006">MendCond006</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp010"><pre><code>{
  "cg:id": "VarInterp010",
  "cg:type": "VariantInterpretation",
  "explanation": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Noonan Syndrome 3",
  "variant": {
    "cg:id": "CanAll011",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll014",
    "identifier": "http://reg.genome.network/allele/CA256480"
  },
  "condition": [
    {
      "cg:id": "MendCond006",
      "cg:type": "MendelianCondition",
      "explanation": "noonan syndrome 3",
      "disease": [
        "OMIM:609942"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp011">VarInterp011</a></p></td>
          <td>NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Costello syndrome</td>
          <td>
              <p><a href="details/details.html#CanAll011">CanAll011</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond055">MendCond055</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp011"><pre><code>{
  "cg:id": "VarInterp011",
  "cg:type": "VariantInterpretation",
  "explanation": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Costello syndrome",
  "variant": {
    "cg:id": "CanAll011",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll014",
    "identifier": "http://reg.genome.network/allele/CA256480"
  },
  "condition": [
    {
      "cg:id": "MendCond055",
      "cg:type": "MendelianCondition",
      "explanation": "noonan syndrome, Autosomal Dominant",
      "disease": [
        "Orphanet:648"
      ],
      "modeOfInheritance": "HP:0000006"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp041">VarInterp041</a></p></td>
          <td>Arg343His in Exon 08 of LOXHD1: This variant is not expected to have clinical significance because it has been identified in 2.0% (27/1384) of African American chromosomes by the NHLBI Exome Sequencing Project (http://evs.gs.washington.edu/EVS).</td>
          <td>
              <p><a href="details/details.html#CanAll092">CanAll092</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond061">MendCond061</a></p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <td>
              <p><a href="details/details.html#AssertMeth001">AssertMeth001</a></p>
         </td>
          <td>
              <p><a href="details/details.html#EvLn113">EvLn113</a></p><p><a href="details/details.html#EvLn114">EvLn114</a></p><p><a href="details/details.html#EvLn115">EvLn115</a></p>
         </td>
          <!--
          <td><div class="example_view" id="VarInterp041"><pre><code>{
  "cg:id": "VarInterp041",
  "cg:type": "VariantInterpretation",
  "explanation": "Arg343His in Exon 08 of LOXHD1: This variant is not expected to have clinical significance because it has been identified in 2.0% (27/1384) of African American chromosomes by the NHLBI Exome Sequencing Project (http://evs.gs.washington.edu/EVS).",
  "variant": {
    "cg:id": "CanAll092",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll098",
    "identifier": "http://reg.genome.network/allele/CA090950"
  },
  "condition": [
    {
      "cg:id": "MendCond061",
      "cg:type": "MendelianCondition",
      "explanation": "Hearing Loss, Autosomal Recessive",
      "disease": [
        "Orphanet:87884"
      ],
      "name": "Hearing Loss, Autosomal Recessive",
      "modeOfInheritance": "HP:0000007"
    }
  ],
  "clinicalSignificance": "LN:LA6675-8",
  "assertionMethod": {
    "cg:id": "AssertMeth001",
    "cg:type": "AssertionMethod",
    "description": "ACMG ISV guidelines 2015",
    "version": "1.0",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "scoringAlgorithm": {
      "cg:id": "ScorAlg01",
      "cg:type": "ScoringAlgorithm",
      "description": "This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance."
    }
  },
  "hasEvidence": [
    {
      "cg:id": "EvLn113",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:bs",
      "supportingData": [
        {
          "cg:id": "CritAssess372",
          "cg:type": "CriterionAssessment",
          "explanation": "This variant is present in 2% (27/1384) of African American Chromosomes in ESP.",
          "criterion": {
            "cg:id": "BS1",
            "cg:type": "Criterion",
            "description": "Allele frequency is greater than expected for disorder",
            "shortDescription": "MAF too high for disorder",
            "defaultStrength": "CG-evidence-strength:bs",
            "targetInterpretation": "LN:LA6675-8"
          },
          "variant": {
            "cg:id": "CanAll092",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll098",
            "identifier": "http://reg.genome.network/allele/CA090950"
          },
          "condition": [
            {
              "cg:id": "MendCond057",
              "cg:type": "MendelianCondition",
              "explanation": "Hearing Loss",
              "disease": [
                "Orphanet:87884"
              ],
              "name": "Hearing Loss"
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn109",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "AllFreq038",
                  "cg:type": "AlleleFrequency",
                  "ascertainment": "ExAC",
                  "allele": {
                    "cg:id": "CanAll092",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll098",
                    "identifier": "http://reg.genome.network/allele/CA090950"
                  },
                  "alleleCount": 27,
                  "alleleNumber": 1384,
                  "alleleFrequency": 0.02,
                  "population": "EXAC:AFR"
                },
                {
                  "cg:id": "CondPrev039",
                  "cg:type": "ConditionPrevelance",
                  "condition": [
                    {
                      "cg:id": "MendCond057",
                      "cg:type": "MendelianCondition",
                      "explanation": "Hearing Loss",
                      "disease": [
                        "Orphanet:87884"
                      ],
                      "name": "Hearing Loss"
                    }
                  ],
                  "minimum": 1.0,
                  "maximum": 1.0,
                  "population": "CG-population-type:overall"
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib509",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": {
                "cg:id": "2017-03-08T09:46:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn114",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:bp",
      "supportingData": [
        {
          "cg:id": "CritAssess340",
          "cg:type": "CriterionAssessment",
          "explanation": "Homozygous truncating variants in LOXHD1 are associated with autosomal recessive hearing loss. This was demonstrated by both identification in human patients and a mouse model. (I used the OMIM entry as a source, but you could also directly use the papers.)",
          "criterion": {
            "cg:id": "BP1",
            "cg:type": "Criterion",
            "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
            "shortDescription": "Missense variant when truncating is expected mechanism",
            "defaultStrength": "CG-evidence-strength:bp",
            "targetInterpretation": "LN:LA6675-8"
          },
          "variant": {
            "cg:id": "CanAll092",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll098",
            "identifier": "http://reg.genome.network/allele/CA090950"
          },
          "condition": [
            {
              "cg:id": "MendCond059",
              "cg:type": "MendelianCondition",
              "explanation": "Hearing Loss, Autosomal Recessive",
              "disease": [
                "Orphanet:87884"
              ],
              "name": "Hearing Loss, Autosomal Recessive",
              "modeOfInheritance": "HP:0000007"
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn083",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "MolCon201",
                  "cg:type": "MolecularConsequence",
                  "contextualAllele": {
                    "cg:id": "CtxAll098",
                    "cg:type": "ContextualAllele",
                    "relatedCanonicalAllele": {
                      "cg:id": "CanAll092",
                      "cg:type": "CanonicalAllele",
                      "preferredCtxAllele": "CtxAll098",
                      "identifier": "http://reg.genome.network/allele/CA090950"
                    },
                    "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
                  },
                  "consequence": "SO:0001583",
                  "lof": true
                },
                {
                  "cg:id": "CondMech202",
                  "cg:type": "ConditionMechanism",
                  "explanation": "shown in both humans and mouse models",
                  "mechanism": "SO:0002054",
                  "strength": "CG-cond-mech-strength:primary",
                  "gene": {
                    "cg:id": "G048",
                    "cg:type": "Gene",
                    "symbol": "LOXHD1"
                  },
                  "condition": {
                    "cg:id": "MendCond059",
                    "cg:type": "MendelianCondition",
                    "explanation": "Hearing Loss, Autosomal Recessive",
                    "disease": [
                      "Orphanet:87884"
                    ],
                    "name": "Hearing Loss, Autosomal Recessive",
                    "modeOfInheritance": "HP:0000007"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib475",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": {
                "cg:id": "2016-09-06T09:59:59+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn115",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:bp",
      "supportingData": [
        {
          "cg:id": "CritAssess363",
          "cg:type": "CriterionAssessment",
          "explanation": "This variant was identified in one individual with pathogenic variants in another gene. Internal LMM data: The proband had one pathogenic and one likely pathogenic variant in (in trans) in SLC26A4. This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome. SLC26A4 is the only gene associated with Pendred Syndrome",
          "criterion": {
            "cg:id": "BP5",
            "cg:type": "Criterion",
            "description": "Variant found in a case with an alternate molecular basis for disease",
            "shortDescription": "Variant found in case with alternate explanation",
            "defaultStrength": "CG-evidence-strength:bp",
            "targetInterpretation": "LN:LA6675-8"
          },
          "variant": {
            "cg:id": "CanAll092",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll098",
            "identifier": "http://reg.genome.network/allele/CA090950"
          },
          "condition": [
            {
              "cg:id": "MendCond058",
              "cg:type": "MendelianCondition",
              "explanation": "Hearing Loss",
              "disease": [
                "Orphanet:87884"
              ],
              "name": "Hearing Loss"
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn106",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "VarInterp364",
                  "cg:type": "VariantInterpretation",
                  "variant": {
                    "cg:id": "CanAll122",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll128"
                  },
                  "condition": [
                    {
                      "cg:id": "MendCond058",
                      "cg:type": "MendelianCondition",
                      "explanation": "Hearing Loss",
                      "disease": [
                        "Orphanet:87884"
                      ],
                      "name": "Hearing Loss"
                    }
                  ],
                  "clinicalSignificance": "LN:LA26332-9"
                },
                {
                  "cg:id": "VarInterp365",
                  "cg:type": "VariantInterpretation",
                  "variant": {
                    "cg:id": "CanAll123",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll129"
                  },
                  "condition": [
                    {
                      "cg:id": "MendCond058",
                      "cg:type": "MendelianCondition",
                      "explanation": "Hearing Loss",
                      "disease": [
                        "Orphanet:87884"
                      ],
                      "name": "Hearing Loss"
                    }
                  ],
                  "clinicalSignificance": "LN:LA6668-3"
                },
                {
                  "cg:id": "IndAll366",
                  "cg:type": "IndividualAllele",
                  "individual": {
                    "cg:id": "Indiv029",
                    "cg:type": "Individual",
                    "description": "individual #029"
                  },
                  "primaryAllele": {
                    "cg:id": "CanAll092",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll098",
                    "identifier": "http://reg.genome.network/allele/CA090950"
                  },
                  "secondaryAllele": {
                    "cg:id": "CanAll122",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll128"
                  },
                  "phase": "CG-phase:trans"
                },
                {
                  "cg:id": "IndAll367",
                  "cg:type": "IndividualAllele",
                  "individual": {
                    "cg:id": "Indiv029",
                    "cg:type": "Individual",
                    "description": "individual #029"
                  },
                  "primaryAllele": {
                    "cg:id": "CanAll092",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll098",
                    "identifier": "http://reg.genome.network/allele/CA090950"
                  },
                  "secondaryAllele": {
                    "cg:id": "CanAll123",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll129"
                  },
                  "phase": "CG-phase:trans"
                },
                {
                  "cg:id": "IndCond368",
                  "cg:type": "IndividualCondition",
                  "explanation": "This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome.",
                  "individual": {
                    "cg:id": "Indiv029",
                    "cg:type": "Individual",
                    "description": "individual #029"
                  },
                  "condition": [
                    {
                      "cg:id": "MendCond058",
                      "cg:type": "MendelianCondition",
                      "explanation": "Hearing Loss",
                      "disease": [
                        "Orphanet:87884"
                      ],
                      "name": "Hearing Loss"
                    }
                  ],
                  "hasCondition": true
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib506",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": {
                "cg:id": "2017-03-07T16:30:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "cg:id": "Contrib448",
      "cg:type": "Contribution",
      "agent": "CSER classifier #6",
      "onDate": {
        "cg:id": "2015-06-26T12:05:59+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp042">VarInterp042</a></p></td>
          <td>Uncertain Sig(not specified) NM_000363.4(TNNI3):c.484C>T (p.Arg162Trp)</td>
          <td>
              <p><a href="details/details.html#CanAll049">CanAll049</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p>LN:LA26333-7</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp042"><pre><code>{
  "cg:id": "VarInterp042",
  "cg:type": "VariantInterpretation",
  "explanation": "Uncertain Sig(not specified) NM_000363.4(TNNI3):c.484C>T (p.Arg162Trp)",
  "variant": {
    "cg:id": "CanAll049",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll053",
    "identifier": "http://reg.genome.network/allele/CA021738"
  },
  "clinicalSignificance": "LN:LA26333-7",
  "contribution": [
    {
      "cg:id": "Contrib434",
      "cg:type": "Contribution",
      "agent": "Harry Stevenson",
      "onDate": {
        "cg:id": "2016-07-13T08:45:59+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp044">VarInterp044</a></p></td>
          <td>p.Tyr109Tyr in exon 4 of MYH7: This variant is not expected to have clinical significance because it does not alter an amino acid residue and is not located near a splice junction. It has been identified in 0.2% (31/16512) of South Asian chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs36211408).</td>
          <td>
              <p><a href="details/details.html#CanAll118">CanAll118</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p>LN:LA26334-5</p>
         </td>
          <td>
              <p><a href="details/details.html#AssertMeth001">AssertMeth001</a></p>
         </td>
          <td>
              <p><a href="details/details.html#EvLn120">EvLn120</a></p><p><a href="details/details.html#EvLn118">EvLn118</a></p><p><a href="details/details.html#EvLn119">EvLn119</a></p>
         </td>
          <!--
          <td><div class="example_view" id="VarInterp044"><pre><code>{
  "cg:id": "VarInterp044",
  "cg:type": "VariantInterpretation",
  "explanation": "p.Tyr109Tyr in exon 4 of MYH7: This variant is not expected to have clinical significance because it does not alter an amino acid residue and is not located near a splice junction. It has been identified in 0.2% (31/16512) of South Asian chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs36211408).",
  "variant": {
    "cg:id": "CanAll118",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll124",
    "identifier": "http://reg.genome.network/allele/CA013536"
  },
  "clinicalSignificance": "LN:LA26334-5",
  "assertionMethod": {
    "cg:id": "AssertMeth001",
    "cg:type": "AssertionMethod",
    "description": "ACMG ISV guidelines 2015",
    "version": "1.0",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "scoringAlgorithm": {
      "cg:id": "ScorAlg01",
      "cg:type": "ScoringAlgorithm",
      "description": "This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance."
    }
  },
  "hasEvidence": [
    {
      "cg:id": "EvLn120",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:bs",
      "supportingData": [
        {
          "cg:id": "CritAssess371",
          "cg:type": "CriterionAssessment",
          "explanation": "This variant is present in 0.2% (31/16512) of South Asian Chromosomes in ExAC.",
          "criterion": {
            "cg:id": "BS1",
            "cg:type": "Criterion",
            "description": "Allele frequency is greater than expected for disorder",
            "shortDescription": "MAF too high for disorder",
            "defaultStrength": "CG-evidence-strength:bs",
            "targetInterpretation": "LN:LA6675-8"
          },
          "variant": {
            "cg:id": "CanAll118",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll124",
            "identifier": "http://reg.genome.network/allele/CA013536"
          },
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn108",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "AllFreq037",
                  "cg:type": "AlleleFrequency",
                  "ascertainment": "ExAC",
                  "allele": {
                    "cg:id": "CanAll118",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll124",
                    "identifier": "http://reg.genome.network/allele/CA013536"
                  },
                  "alleleCount": 31,
                  "alleleNumber": 16512,
                  "alleleFrequency": 0.02,
                  "population": "EXAC:SAS"
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib508",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": {
                "cg:id": "2017-03-08T07:43:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn118",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:bp",
      "supportingData": [
        {
          "cg:id": "CritAssess373",
          "cg:type": "CriterionAssessment",
          "explanation": "This is a synonymous variant with little conservation and no predicted impact on splicing. Conservation and splicing algorithm values are listed below.",
          "criterion": {
            "cg:id": "BP4",
            "cg:type": "Criterion",
            "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
            "shortDescription": "Computational evidence supports benign impact",
            "defaultStrength": "CG-evidence-strength:bp",
            "targetInterpretation": "LN:LA6675-8"
          },
          "variant": {
            "cg:id": "CanAll118",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll124",
            "identifier": "http://reg.genome.network/allele/CA013536"
          },
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn117",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "InSilicoPred068",
                  "cg:type": "InSilicoPrediction",
                  "explanation": "Provenance would show that this comes from Alamut",
                  "contextualAllele": {
                    "cg:id": "CtxAll124",
                    "cg:type": "ContextualAllele",
                    "relatedCanonicalAllele": {
                      "cg:id": "CanAll118",
                      "cg:type": "CanonicalAllele",
                      "preferredCtxAllele": "CtxAll124",
                      "identifier": "http://reg.genome.network/allele/CA013536"
                    },
                    "alleleName": "NM_000257.2(MYH7):c.327C>T"
                  },
                  "type": "CG-prediction-type:sp",
                  "value": "no impact"
                },
                {
                  "cg:id": "Conserv071",
                  "cg:type": "Conservation",
                  "allele": {
                    "cg:id": "CanAll118",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll124",
                    "identifier": "http://reg.genome.network/allele/CA013536"
                  },
                  "conserved": true,
                  "method": "phyloP",
                  "score": -0.76
                },
                {
                  "cg:id": "Conserv072",
                  "cg:type": "Conservation",
                  "allele": {
                    "cg:id": "CanAll118",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll124",
                    "identifier": "http://reg.genome.network/allele/CA013536"
                  },
                  "conserved": true,
                  "method": "PhastCons",
                  "score": 0.63
                },
                {
                  "cg:id": "MolCon075",
                  "cg:type": "MolecularConsequence",
                  "consequence": "SO:0001819",
                  "contextualAllele": {
                    "cg:id": "CtxAll124",
                    "cg:type": "ContextualAllele",
                    "relatedCanonicalAllele": {
                      "cg:id": "CanAll118",
                      "cg:type": "CanonicalAllele",
                      "preferredCtxAllele": "CtxAll124",
                      "identifier": "http://reg.genome.network/allele/CA013536"
                    },
                    "alleleName": "NM_000257.2(MYH7):c.327C>T"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib510",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": {
                "cg:id": "2017-03-08T10:02:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn119",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:bp",
      "supportingData": [
        {
          "cg:id": "CritAssess359",
          "cg:type": "CriterionAssessment",
          "criterion": {
            "cg:id": "BP7",
            "cg:type": "Criterion",
            "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
            "shortDescription": "Synonymous variant without predicted splice effect in region with low conservation",
            "defaultStrength": "CG-evidence-strength:bp",
            "targetInterpretation": "LN:LA6675-8"
          },
          "variant": {
            "cg:id": "CanAll118",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll124",
            "identifier": "http://reg.genome.network/allele/CA013536"
          },
          "condition": [
            {
              "cg:id": "-"
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn101",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "InSilicoPred262",
                  "cg:type": "InSilicoPrediction",
                  "explanation": "Provenance would show that this comes from Alamut",
                  "contextualAllele": {
                    "cg:id": "CtxAll124",
                    "cg:type": "ContextualAllele",
                    "relatedCanonicalAllele": {
                      "cg:id": "CanAll118",
                      "cg:type": "CanonicalAllele",
                      "preferredCtxAllele": "CtxAll124",
                      "identifier": "http://reg.genome.network/allele/CA013536"
                    },
                    "alleleName": "NM_000257.2(MYH7):c.327C>T"
                  },
                  "type": "CG-prediction-type:sp",
                  "value": "no impact"
                },
                {
                  "cg:id": "Conserv263",
                  "cg:type": "Conservation",
                  "allele": {
                    "cg:id": "CanAll118",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll124",
                    "identifier": "http://reg.genome.network/allele/CA013536"
                  },
                  "conserved": true,
                  "method": "phyloP",
                  "score": -0.76
                },
                {
                  "cg:id": "Conserv264",
                  "cg:type": "Conservation",
                  "allele": {
                    "cg:id": "CanAll118",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll124",
                    "identifier": "http://reg.genome.network/allele/CA013536"
                  },
                  "conserved": true,
                  "method": "PhastCons",
                  "score": 0.63
                },
                {
                  "cg:id": "MolCon265",
                  "cg:type": "MolecularConsequence",
                  "consequence": "SO:0001819",
                  "contextualAllele": {
                    "cg:id": "CtxAll124",
                    "cg:type": "ContextualAllele",
                    "relatedCanonicalAllele": {
                      "cg:id": "CanAll118",
                      "cg:type": "CanonicalAllele",
                      "preferredCtxAllele": "CtxAll124",
                      "identifier": "http://reg.genome.network/allele/CA013536"
                    },
                    "alleleName": "NM_000257.2(MYH7):c.327C>T"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib503",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": {
                "cg:id": "2016-10-10T08:04:59+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "cg:id": "Contrib467",
      "cg:type": "Contribution",
      "agent": "CSER classifier #6",
      "onDate": {
        "cg:id": "2015-06-26T12:05:59+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp045">VarInterp045</a></p></td>
          <td>The c.631-2A>G variant in DSC2 has been reported in 1 Caucasian individual with ARVC (Heuser 2006). This variant has also been identified by our laboratory in 2 individuals (1 with ARVC and 1 with LVNC). This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042). This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/). In summary, although additional studies are required to fully establish its clinical significance, the c.631-2A>G variant is likely pathogenic.</td>
          <td>
              <p><a href="details/details.html#CanAll121">CanAll121</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond074">MendCond074</a></p>
         </td>
          <td>
              <p>LN:LA26332-9</p>
         </td>
          <td>
              <p><a href="details/details.html#AssertMeth001">AssertMeth001</a></p>
         </td>
          <td>
              <p><a href="details/details.html#EvLn121">EvLn121</a></p><p><a href="details/details.html#EvLn122">EvLn122</a></p>
         </td>
          <!--
          <td><div class="example_view" id="VarInterp045"><pre><code>{
  "cg:id": "VarInterp045",
  "cg:type": "VariantInterpretation",
  "explanation": "The c.631-2A>G variant in DSC2 has been reported in 1 Caucasian individual with ARVC (Heuser 2006). This variant has also been identified by our laboratory in 2 individuals (1 with ARVC and 1 with LVNC). This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042). This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/). In summary, although additional studies are required to fully establish its clinical significance, the c.631-2A>G variant is likely pathogenic.",
  "variant": {
    "cg:id": "CanAll121",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll127"
  },
  "condition": [
    {
      "cg:id": "MendCond074",
      "disease": [
        "Orphanet:247"
      ],
      "modeOfInheritance": "HP:0000006"
    }
  ],
  "clinicalSignificance": "LN:LA26332-9",
  "assertionMethod": {
    "cg:id": "AssertMeth001",
    "cg:type": "AssertionMethod",
    "description": "ACMG ISV guidelines 2015",
    "version": "1.0",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "scoringAlgorithm": {
      "cg:id": "ScorAlg01",
      "cg:type": "ScoringAlgorithm",
      "description": "This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance."
    }
  },
  "hasEvidence": [
    {
      "cg:id": "EvLn121",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:pvs",
      "supportingData": [
        {
          "cg:id": "CritAssess361",
          "cg:type": "CriterionAssessment",
          "explanation": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).",
          "criterion": {
            "cg:id": "PVS1",
            "cg:type": "Criterion",
            "description": "null variant (nonsense, frameshift, canonical Â±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
            "shortDescription": "Null variant in gene with known LOF mechanism",
            "defaultStrength": "CG-evidence-strength:pvs",
            "targetInterpretation": "LN:LA6668-3"
          },
          "variant": {
            "cg:id": "CanAll121",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll127"
          },
          "condition": [
            {
              "cg:id": "MendCond046",
              "cg:type": "MendelianCondition",
              "explanation": "arrhythmogenic right ventricular cardiomyopathy",
              "disease": [
                "Orphanet:247"
              ]
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn103",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "MolCon272",
                  "cg:type": "MolecularConsequence",
                  "explanation": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466).",
                  "contextualAllele": {
                    "cg:id": "CtxAll127",
                    "cg:type": "ContextualAllele",
                    "relatedCanonicalAllele": {
                      "cg:id": "CanAll121",
                      "cg:type": "CanonicalAllele",
                      "preferredCtxAllele": "CtxAll127"
                    },
                    "alleleName": "NM_024422.3(DSC2): c.631-2A>G (p.?)"
                  },
                  "consequence": "SO:0001629",
                  "lof": true
                },
                {
                  "cg:id": "CondMech273",
                  "cg:type": "ConditionMechanism",
                  "explanation": "Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).",
                  "mechanism": "SO:0002054",
                  "condition": {
                    "cg:id": "MendCond046",
                    "cg:type": "MendelianCondition",
                    "explanation": "arrhythmogenic right ventricular cardiomyopathy",
                    "disease": [
                      "Orphanet:247"
                    ]
                  },
                  "strength": "CG-cond-mech-strength:established",
                  "gene": {
                    "cg:id": "G060",
                    "cg:type": "Gene",
                    "symbol": "DSC2"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib505",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": {
                "cg:id": "2017-01-17T09:11:59+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn122",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:pm",
      "supportingData": [
        {
          "cg:id": "CritAssess369",
          "cg:type": "CriterionAssessment",
          "explanation": "This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042).",
          "criterion": {
            "cg:id": "PM2",
            "cg:type": "Criterion",
            "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
            "shortDescription": "Absent from controls",
            "defaultStrength": "CG-evidence-strength:pm",
            "targetInterpretation": "LN:LA6668-3"
          },
          "variant": {
            "cg:id": "CanAll121",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll127"
          },
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn107",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "AllFreq370",
                  "cg:type": "AlleleFrequency",
                  "ascertainment": "ExAC",
                  "population": "EXAC:NFE",
                  "allele": {
                    "cg:id": "CanAll121",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll127"
                  },
                  "alleleCount": 1,
                  "alleleNumber": 66564,
                  "alleleFrequency": 1.5e-05,
                  "homozygousAlleleIndividualCount": 0
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib507",
              "cg:type": "Contribution",
              "agent": "Harry Stevenson",
              "onDate": {
                "cg:id": "2017-03-07T17:22:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "cg:id": "Contrib493",
      "cg:type": "Contribution",
      "agent": "CSER classifier #6",
      "onDate": {
        "cg:id": "2015-06-26T12:05:59+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp047">VarInterp047</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll028">CanAll028</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond013">MendCond013</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp047"><pre><code>{
  "cg:id": "VarInterp047",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll028",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll031",
    "identifier": "http://reg.genome.network/allele/CA022141"
  },
  "condition": [
    {
      "cg:id": "MendCond013",
      "cg:type": "MendelianCondition",
      "explanation": "Fabry disease, X-linked recessive inheritance",
      "disease": [
        "Orphanet:324"
      ],
      "modeOfInheritance": "HP:0001419"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp048">VarInterp048</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll029">CanAll029</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond013">MendCond013</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp048"><pre><code>{
  "cg:id": "VarInterp048",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll029",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll032",
    "identifier": "http://reg.genome.network/allele/CA022153"
  },
  "condition": [
    {
      "cg:id": "MendCond013",
      "cg:type": "MendelianCondition",
      "explanation": "Fabry disease, X-linked recessive inheritance",
      "disease": [
        "Orphanet:324"
      ],
      "modeOfInheritance": "HP:0001419"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp049">VarInterp049</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll030">CanAll030</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond013">MendCond013</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp049"><pre><code>{
  "cg:id": "VarInterp049",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll030",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll033",
    "identifier": "http://reg.genome.network/allele/CA022159"
  },
  "condition": [
    {
      "cg:id": "MendCond013",
      "cg:type": "MendelianCondition",
      "explanation": "Fabry disease, X-linked recessive inheritance",
      "disease": [
        "Orphanet:324"
      ],
      "modeOfInheritance": "HP:0001419"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp050">VarInterp050</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll031">CanAll031</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond013">MendCond013</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp050"><pre><code>{
  "cg:id": "VarInterp050",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll031",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll034",
    "identifier": "http://reg.genome.network/allele/CA022189"
  },
  "condition": [
    {
      "cg:id": "MendCond013",
      "cg:type": "MendelianCondition",
      "explanation": "Fabry disease, X-linked recessive inheritance",
      "disease": [
        "Orphanet:324"
      ],
      "modeOfInheritance": "HP:0001419"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp055">VarInterp055</a></p></td>
          <td>The Tyr337Cys variant in SOS1 has been reported in 1 individual with clinical features of Noonan syndrome (Roberts 2007), but has also been reported to be present in 2 unaffected individuals from one family (Personal Communication). This variant was absent from large population studies. Studies have shown that the Tyr337Cys variant may impact protein function by increasing its activity though these results are inconclusive and these in vitro assays may not accurately represent biological function (Smith 2013). Due to the conflicting data about this variant, the clinical significance of the Tyr337Cys variant is uncertain.</td>
          <td>
              <p><a href="details/details.html#CanAll124">CanAll124</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond075">MendCond075</a></p>
         </td>
          <td>
              <p>LN:LA26333-7</p>
         </td>
          <td>
              <p><a href="details/details.html#AssertMeth001">AssertMeth001</a></p>
         </td>
          <td>
              <p><a href="details/details.html#EvLn124">EvLn124</a></p><p><a href="details/details.html#EvLn125">EvLn125</a></p>
         </td>
          <!--
          <td><div class="example_view" id="VarInterp055"><pre><code>{
  "cg:id": "VarInterp055",
  "cg:type": "VariantInterpretation",
  "explanation": "The Tyr337Cys variant in SOS1 has been reported in 1 individual with clinical features of Noonan syndrome (Roberts 2007), but has also been reported to be present in 2 unaffected individuals from one family (Personal Communication). This variant was absent from large population studies. Studies have shown that the Tyr337Cys variant may impact protein function by increasing its activity though these results are inconclusive and these in vitro assays may not accurately represent biological function (Smith 2013). Due to the conflicting data about this variant, the clinical significance of the Tyr337Cys variant is uncertain.",
  "variant": {
    "cg:id": "CanAll124",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll130"
  },
  "condition": [
    {
      "cg:id": "MendCond075",
      "disease": [
        "Orphanet:648"
      ],
      "modeOfInheritance": "HP:0000006"
    }
  ],
  "clinicalSignificance": "LN:LA26333-7",
  "assertionMethod": {
    "cg:id": "AssertMeth001",
    "cg:type": "AssertionMethod",
    "description": "ACMG ISV guidelines 2015",
    "version": "1.0",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "scoringAlgorithm": {
      "cg:id": "ScorAlg01",
      "cg:type": "ScoringAlgorithm",
      "description": "This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance."
    }
  },
  "hasEvidence": [
    {
      "cg:id": "EvLn124",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:PP",
      "supportingData": [
        {
          "cg:id": "CritAssess374",
          "cg:type": "CriterionAssessment",
          "explanation": "Flag-tagged construct with the variant was transfected into HEK cells. As per the RASopathy working groupâs approved assays (see example 3 above), Smith et al. showed that the Y337C mutation increased levels of pERK in starved HEK cells via western blotting.",
          "criterion": {
            "cg:id": "PS3",
            "cg:type": "Criterion",
            "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
            "shortDescription": "Well-established functional studies showing damaging effect",
            "defaultStrength": "CG-evidence-strength:ps",
            "targetInterpretation": "LN:LA6668-3"
          },
          "variant": {
            "cg:id": "CanAll124",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll130"
          },
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn116",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "Func067",
                  "cg:type": "FunctionalData",
                  "explanation": "PMID: 23487764",
                  "result": "increased ERK activtation",
                  "dataType": "CG-func-data-type:activation",
                  "contextualAllele": {
                    "cg:id": "CtxAll130",
                    "cg:type": "ContextualAllele",
                    "relatedCanonicalAllele": {
                      "cg:id": "CanAll124",
                      "cg:type": "CanonicalAllele",
                      "preferredCtxAllele": "CtxAll130"
                    },
                    "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
                  },
                  "gene": {
                    "cg:id": "G052",
                    "cg:type": "Gene",
                    "symbol": "SOS1"
                  }
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib513",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": {
                "cg:id": "2017-03-14T10:28:59+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn125",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:bs",
      "supportingData": [
        {
          "cg:id": "CritAssess375",
          "cg:type": "CriterionAssessment",
          "explanation": "A personal communication to our lab informed us of an individual and his son who both harbored the variant but with clinical workup did not appear to have Noonan Syndrome or symptoms indicative of Noonan Syndrome.",
          "criterion": {
            "cg:id": "BS4",
            "cg:type": "Criterion",
            "description": "Lack of segregation in affected members of a family",
            "shortDescription": "Does not segregate with disease",
            "defaultStrength": "CG-evidence-strength:bs",
            "targetInterpretation": "LN:LA6675-8"
          },
          "variant": {
            "cg:id": "CanAll124",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll130"
          },
          "condition": [
            {
              "cg:id": "MendCond056",
              "cg:type": "MendelianCondition",
              "explanation": "noonan syndrome, Autosomal Dominant",
              "disease": [
                "Orphanet:648"
              ],
              "modeOfInheritance": "HP:0000006"
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn123",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "FamSegData076",
                  "cg:type": "FamilySegregationData",
                  "condition": {
                    "cg:id": "MendCond056",
                    "cg:type": "MendelianCondition",
                    "explanation": "noonan syndrome, Autosomal Dominant",
                    "disease": [
                      "Orphanet:648"
                    ],
                    "modeOfInheritance": "HP:0000006"
                  },
                  "canonicalAllele": {
                    "cg:id": "CanAll124",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll130"
                  },
                  "family": {
                    "cg:id": "Fam011",
                    "cg:type": "Family",
                    "description": "Family #011"
                  },
                  "phenotypePositiveAllelePositive": 0,
                  "phenotypeNegativeAllelePositive": 2,
                  "phenotypePositiveAlleleNegative": 0,
                  "anyInconsistentSegregations": true
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib514",
              "cg:type": "Contribution",
              "agent": "Stefan DiMarino",
              "onDate": {
                "cg:id": "2017-03-14T10:44:59+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "cg:id": "Contrib511",
      "cg:type": "Contribution",
      "agent": "Gene Geneticist",
      "onDate": {
        "cg:id": "2014-08-14T10:28:59+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp057">VarInterp057</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll036">CanAll036</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond062">MendCond062</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp057"><pre><code>{
  "cg:id": "VarInterp057",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll036",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll040",
    "identifier": "http://reg.genome.network/allele/CA273615"
  },
  "condition": [
    {
      "cg:id": "MendCond062",
      "cg:type": "MendelianCondition",
      "explanation": "Primary ciliary dyskenisia, Autosomal Recessive",
      "disease": [
        "Orphanet:244"
      ],
      "modeOfInheritance": "HP:0000007"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3",
  "contribution": [
    {
      "cg:id": "Contrib485",
      "cg:type": "Contribution",
      "agent": "Stefan DiMarino",
      "onDate": {
        "cg:id": "2016-05-31T11:00:59+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp059">VarInterp059</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll038">CanAll038</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond010">MendCond010</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp059"><pre><code>{
  "cg:id": "VarInterp059",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll038",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll042",
    "identifier": "http://reg.genome.network/allele/CA501223"
  },
  "condition": [
    {
      "cg:id": "MendCond010",
      "cg:type": "MendelianCondition",
      "explanation": "nonsyndromic deafness, Autosomal Recessive, GIPC3",
      "disease": [
        "Orphanet:87884"
      ],
      "modeOfInheritance": "HP:0000007",
      "gene": {
        "cg:id": "G024",
        "cg:type": "Gene",
        "symbol": "GIPC3"
      }
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp060">VarInterp060</a></p></td>
          <td>The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic.</td>
          <td>
              <p><a href="details/details.html#CanAll127">CanAll127</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond049">MendCond049</a></p>
         </td>
          <td>
              <p>LN:LA26332-9</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp060"><pre><code>{
  "cg:id": "VarInterp060",
  "cg:type": "VariantInterpretation",
  "explanation": "The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic.",
  "variant": {
    "cg:id": "CanAll127",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll133"
  },
  "condition": [
    {
      "cg:id": "MendCond049",
      "cg:type": "MendelianCondition",
      "explanation": "Rasopathy",
      "disease": [
        "Orphanet:98733"
      ],
      "name": "Rasopathy"
    }
  ],
  "clinicalSignificance": "LN:LA26332-9",
  "contribution": [
    {
      "cg:id": "Contrib518",
      "cg:type": "Contribution",
      "agent": "ACME Molecular Lab",
      "onDate": {
        "cg:id": "2011-06-12T09:00:00+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp063">VarInterp063</a></p></td>
          <td>The Lys483Gln variant in BRAF has been identified in one proband with clinical features of Cardio-facio-cutaneous syndrome, and was not identified in either of this individual's parents (LMM unpublished data). This variant has not been identified in large population studies. In addition, another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). Computational analyses (biochemical amino acid properties, conservation, AlignGVGD, PolyPhen2, and SIFT) suggest that the Lys483Gln variant may impact the normal function of the protein. In summary, this variant is likely pathogenic, though additional studies are required to fully establish its clinical significance.</td>
          <td>
              <p><a href="details/details.html#CanAll125">CanAll125</a></p>
         </td>
          <td>
              <p><a href="details/details.html#Cond070">Cond070</a></p>
         </td>
          <td>
              <p>LN:LA26332-9</p>
         </td>
          <td>
              <p><a href="details/details.html#AssertMeth001">AssertMeth001</a></p>
         </td>
          <td>
              <p><a href="details/details.html#EvLn127">EvLn127</a></p><p><a href="details/details.html#EvLn129">EvLn129</a></p><p><a href="details/details.html#EvLn131">EvLn131</a></p>
         </td>
          <!--
          <td><div class="example_view" id="VarInterp063"><pre><code>{
  "cg:id": "VarInterp063",
  "cg:type": "VariantInterpretation",
  "explanation": "The Lys483Gln variant in BRAF has been identified in one proband with clinical features of Cardio-facio-cutaneous syndrome, and was not identified in either of this individual's parents (LMM unpublished data). This variant has not been identified in large population studies. In addition, another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). Computational analyses (biochemical amino acid properties, conservation, AlignGVGD, PolyPhen2, and SIFT) suggest that the Lys483Gln variant may impact the normal function of the protein. In summary, this variant is likely pathogenic, though additional studies are required to fully establish its clinical significance.",
  "variant": {
    "cg:id": "CanAll125",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll131"
  },
  "condition": [
    {
      "cg:id": "Cond070"
    }
  ],
  "clinicalSignificance": "LN:LA26332-9",
  "assertionMethod": {
    "cg:id": "AssertMeth001",
    "cg:type": "AssertionMethod",
    "description": "ACMG ISV guidelines 2015",
    "version": "1.0",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "scoringAlgorithm": {
      "cg:id": "ScorAlg01",
      "cg:type": "ScoringAlgorithm",
      "description": "This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance."
    }
  },
  "hasEvidence": [
    {
      "cg:id": "EvLn127",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:pm",
      "supportingData": [
        {
          "cg:id": "CritAssess376",
          "cg:type": "CriterionAssessment",
          "criterion": {
            "cg:id": "PM2",
            "cg:type": "Criterion",
            "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
            "shortDescription": "Absent from controls",
            "defaultStrength": "CG-evidence-strength:pm",
            "targetInterpretation": "LN:LA6668-3"
          },
          "variant": {
            "cg:id": "CanAll125",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll131"
          },
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn126",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "AllFreq101",
                  "cg:type": "AlleleFrequency",
                  "ascertainment": "ExAC",
                  "population": "CG-population-type:combined",
                  "allele": {
                    "cg:id": "CanAll125",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll131"
                  },
                  "alleleCount": 0,
                  "alleleFrequency": 0.0,
                  "medianCoverage": 57.0
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib515",
              "cg:type": "Contribution",
              "agent": "ACME Molecular Lab",
              "onDate": {
                "cg:id": "2011-06-11T11:45:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn129",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:ps",
      "supportingData": [
        {
          "cg:id": "CritAssess377",
          "cg:type": "CriterionAssessment",
          "explanation": "Parental testing in our lab confirmed this variant was present de novo in 1 proband with clinical features of CFC. (LMM internal Data)",
          "criterion": {
            "cg:id": "PM6",
            "cg:type": "Criterion",
            "description": "Assumed de novo, but without confirmation of paternity and maternity",
            "shortDescription": "Assumed de novo (parental status not confirmed)",
            "defaultStrength": "CG-evidence-strength:pm",
            "targetInterpretation": "LN:LA6668-3"
          },
          "variant": {
            "cg:id": "CanAll125",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll131"
          },
          "condition": [
            {
              "cg:id": "MendCond069",
              "cg:type": "MendelianCondition",
              "explanation": "cardiofaciocutaneous syndrome",
              "disease": [
                "Orphanet:1340"
              ]
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "contribution": [
            {
              "cg:id": "Contrib516",
              "cg:type": "Contribution",
              "agent": "ACME Molecular Lab",
              "onDate": {
                "cg:id": "2011-06-11T12:45:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn131",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:pm",
      "supportingData": [
        {
          "cg:id": "CritAssess378",
          "cg:type": "CriterionAssessment",
          "explanation": "Another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). The other variant: NM_004333.4(BRAF):c.1449A>C (p.Lys483Asn) is classified as likely pathogenic by LMM for Rasopathy.",
          "criterion": {
            "cg:id": "PM5",
            "cg:type": "Criterion",
            "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
            "shortDescription": "Novel missense at same amino acid as known pathogenic missense",
            "defaultStrength": "CG-evidence-strength:pm",
            "targetInterpretation": "LN:LA6668-3"
          },
          "variant": {
            "cg:id": "CanAll125",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll131"
          },
          "condition": [
            {
              "cg:id": "MendCond049",
              "cg:type": "MendelianCondition",
              "explanation": "Rasopathy",
              "disease": [
                "Orphanet:98733"
              ],
              "name": "Rasopathy"
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn130",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "VarInterp060",
                  "cg:type": "VariantInterpretation",
                  "explanation": "The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic.",
                  "variant": {
                    "cg:id": "CanAll127",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll133"
                  },
                  "condition": [
                    {
                      "cg:id": "MendCond049",
                      "cg:type": "MendelianCondition",
                      "explanation": "Rasopathy",
                      "disease": [
                        "Orphanet:98733"
                      ],
                      "name": "Rasopathy"
                    }
                  ],
                  "clinicalSignificance": "LN:LA26332-9",
                  "contribution": [
                    {
                      "cg:id": "Contrib518",
                      "cg:type": "Contribution",
                      "agent": "ACME Molecular Lab",
                      "onDate": {
                        "cg:id": "2011-06-12T09:00:00+00:00"
                      },
                      "role": "CG-contributory-role:interpreter"
                    }
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib517",
              "cg:type": "Contribution",
              "agent": "ACME Molecular Lab",
              "onDate": {
                "cg:id": "2011-06-10T13:44:59+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "cg:id": "Contrib512",
      "cg:type": "Contribution",
      "agent": "George Geneticist",
      "onDate": {
        "cg:id": "2014-08-14T10:28:59+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp078">VarInterp078</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll047">CanAll047</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond016">MendCond016</a></p>
         </td>
          <td>
              <p>LN:LA26332-9</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp078"><pre><code>{
  "cg:id": "VarInterp078",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll047",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll051",
    "identifier": "http://reg.genome.network/allele/CA021749"
  },
  "condition": [
    {
      "cg:id": "MendCond016",
      "cg:type": "MendelianCondition",
      "explanation": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant",
      "disease": [
        "Orphanet:217569"
      ],
      "modeOfInheritance": "HP:0000006",
      "name": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant"
    }
  ],
  "clinicalSignificance": "LN:LA26332-9",
  "contribution": [
    {
      "cg:id": "Contrib520",
      "cg:type": "Contribution",
      "agent": "Genetics Diagnostic Laboratory,Children's Hospital of Eastern Ontario",
      "onDate": {
        "cg:id": "2014-06-01T09:01:59+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp079">VarInterp079</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll049">CanAll049</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond016">MendCond016</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp079"><pre><code>{
  "cg:id": "VarInterp079",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll049",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll053",
    "identifier": "http://reg.genome.network/allele/CA021738"
  },
  "condition": [
    {
      "cg:id": "MendCond016",
      "cg:type": "MendelianCondition",
      "explanation": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant",
      "disease": [
        "Orphanet:217569"
      ],
      "modeOfInheritance": "HP:0000006",
      "name": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3",
  "contribution": [
    {
      "cg:id": "Contrib521",
      "cg:type": "Contribution",
      "agent": "GeneDx",
      "onDate": {
        "cg:id": "2016-06-20T11:12:00+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp080">VarInterp080</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll053">CanAll053</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond017">MendCond017</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp080"><pre><code>{
  "cg:id": "VarInterp080",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll053",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll057",
    "identifier": "http://reg.genome.network/allele/CA224454"
  },
  "condition": [
    {
      "cg:id": "MendCond017",
      "cg:type": "MendelianCondition",
      "explanation": "Ornithine carbamoyltransferase deficiency",
      "disease": [
        "Orphanet:664"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp081">VarInterp081</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll057">CanAll057</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond050">MendCond050</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp081"><pre><code>{
  "cg:id": "VarInterp081",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll057",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll061",
    "identifier": "http://reg.genome.network/allele/CA003449"
  },
  "condition": [
    {
      "cg:id": "MendCond050",
      "cg:type": "MendelianCondition",
      "explanation": "Breast Ovarian Cancer, Autosomal Dominant",
      "name": "Breast Ovarian Cancer, Autosomal Dominant",
      "disease": [
        "MESH:D061325"
      ],
      "modeOfInheritance": "HP:0000006"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp150">VarInterp150</a></p></td>
          <td>Variant has been classified as âPathogenicâ by Sharing Clinical Reports Project (SCV000054057.3)</td>
          <td>
              <p><a href="details/details.html#CanAll072">CanAll072</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond023">MendCond023</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp150"><pre><code>{
  "cg:id": "VarInterp150",
  "cg:type": "VariantInterpretation",
  "explanation": "Variant has been classified as âPathogenicâ by Sharing Clinical Reports Project (SCV000054057.3)",
  "variant": {
    "cg:id": "CanAll072",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll078",
    "identifier": "http://reg.genome.network/allele/CA019369"
  },
  "condition": [
    {
      "cg:id": "MendCond023",
      "cg:type": "MendelianCondition",
      "explanation": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2, Autosomal Dominant",
      "disease": [
        "OMIM:612555"
      ],
      "modeOfInheritance": "HP:0000006"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp151">VarInterp151</a></p></td>
          <td>Variant has been classified as âPathogenicâ by OMIM (SCV000043718.1)</td>
          <td>
              <p><a href="details/details.html#CanAll073">CanAll073</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond024">MendCond024</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp151"><pre><code>{
  "cg:id": "VarInterp151",
  "cg:type": "VariantInterpretation",
  "explanation": "Variant has been classified as âPathogenicâ by OMIM (SCV000043718.1)",
  "variant": {
    "cg:id": "CanAll073",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll079",
    "identifier": "http://reg.genome.network/allele/CA128479"
  },
  "condition": [
    {
      "cg:id": "MendCond024",
      "cg:type": "MendelianCondition",
      "explanation": "Myopathy, Distal, type 4, Autosomal Dominant",
      "disease": [
        "OMIM:614065"
      ],
      "modeOfInheritance": "HP:0000006"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp152">VarInterp152</a></p></td>
          <td>Variant has been classified as âPathogenicâ by Expert panel CFTR (SCV000245981.1)</td>
          <td>
              <p><a href="details/details.html#CanAll074">CanAll074</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond025">MendCond025</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp152"><pre><code>{
  "cg:id": "VarInterp152",
  "cg:type": "VariantInterpretation",
  "explanation": "Variant has been classified as âPathogenicâ by Expert panel CFTR (SCV000245981.1)",
  "variant": {
    "cg:id": "CanAll074",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll080",
    "identifier": "http://reg.genome.network/allele/CA276116"
  },
  "condition": [
    {
      "cg:id": "MendCond025",
      "cg:type": "MendelianCondition",
      "explanation": "Cystic Fibrosis, Autosomal Recessive",
      "disease": [
        "Orphanet:586"
      ],
      "modeOfInheritance": "HP:0000007"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp210">VarInterp210</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll095">CanAll095</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond038">MendCond038</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp210"><pre><code>{
  "cg:id": "VarInterp210",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll095",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll101",
    "identifier": "http://reg.genome.network/allele/CA252233"
  },
  "condition": [
    {
      "cg:id": "MendCond038",
      "cg:type": "MendelianCondition",
      "explanation": "Retinitis Pigmentosa",
      "disease": [
        "Orphanet:791"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp213">VarInterp213</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll097">CanAll097</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond039">MendCond039</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp213"><pre><code>{
  "cg:id": "VarInterp213",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll097",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll103",
    "identifier": "http://reg.genome.network/allele/CA182839"
  },
  "condition": [
    {
      "cg:id": "MendCond039",
      "cg:type": "MendelianCondition",
      "explanation": "Absent vas deferens",
      "phenotype": [
        "HP:0012873"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp216">VarInterp216</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll099">CanAll099</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond007">MendCond007</a></p>
         </td>
          <td>
              <p>LN:LA26333-7</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp216"><pre><code>{
  "cg:id": "VarInterp216",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll099",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll105",
    "identifier": "http://reg.genome.network/allele/CA501236"
  },
  "condition": [
    {
      "cg:id": "MendCond007",
      "cg:type": "MendelianCondition",
      "explanation": "cardiomyopathy, Autosomal Dominant",
      "disease": [
        "Orphanet:167848"
      ],
      "modeOfInheritance": "HP:0000006"
    }
  ],
  "clinicalSignificance": "LN:LA26333-7"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp243">VarInterp243</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll106">CanAll106</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond067">MendCond067</a></p>
         </td>
          <td>
              <p>LN:LA26332-9</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp243"><pre><code>{
  "cg:id": "VarInterp243",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll106",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll112",
    "identifier": "http://reg.genome.network/allele/CA012381"
  },
  "condition": [
    {
      "cg:id": "MendCond067",
      "cg:type": "MendelianCondition",
      "explanation": "hypertrophic cardiomyopathy",
      "disease": [
        "Orphanet:217569"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA26332-9"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp247">VarInterp247</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll109">CanAll109</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond060">MendCond060</a></p>
         </td>
          <td>
              <p>LN:LA26332-9</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp247"><pre><code>{
  "cg:id": "VarInterp247",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll109",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll115",
    "identifier": "http://reg.genome.network/allele/CA172240"
  },
  "condition": [
    {
      "cg:id": "MendCond060",
      "cg:type": "MendelianCondition",
      "explanation": "nonsyndromic deafness",
      "disease": [
        "Orphanet:87884"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA26332-9"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp248">VarInterp248</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll110">CanAll110</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond060">MendCond060</a></p>
         </td>
          <td>
              <p>LN:LA26332-9</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp248"><pre><code>{
  "cg:id": "VarInterp248",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll110",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll116",
    "identifier": "http://reg.genome.network/allele/CA274460"
  },
  "condition": [
    {
      "cg:id": "MendCond060",
      "cg:type": "MendelianCondition",
      "explanation": "nonsyndromic deafness",
      "disease": [
        "Orphanet:87884"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA26332-9"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp251">VarInterp251</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll108">CanAll108</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond071">MendCond071</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp251"><pre><code>{
  "cg:id": "VarInterp251",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll108",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll114",
    "identifier": "http://reg.genome.network/allele/CA137502"
  },
  "condition": [
    {
      "cg:id": "MendCond071",
      "cg:type": "MendelianCondition",
      "explanation": "Usher syndrome, Autosomal Recessive",
      "disease": [
        "Orphanet:886"
      ],
      "modeOfInheritance": "HP:0000007"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp254">VarInterp254</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll113">CanAll113</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond042">MendCond042</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp254"><pre><code>{
  "cg:id": "VarInterp254",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll113",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll119",
    "identifier": "http://reg.genome.network/allele/CA254703"
  },
  "condition": [
    {
      "cg:id": "MendCond042",
      "cg:type": "MendelianCondition",
      "explanation": "pseudoachondroplasia",
      "disease": [
        "Orphanet:750"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp255">VarInterp255</a></p></td>
          <td>classified by ENIGMA.</td>
          <td>
              <p><a href="details/details.html#CanAll114">CanAll114</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond073">MendCond073</a></p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp255"><pre><code>{
  "cg:id": "VarInterp255",
  "cg:type": "VariantInterpretation",
  "explanation": "classified by ENIGMA.",
  "variant": {
    "cg:id": "CanAll114",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll120"
  },
  "condition": [
    {
      "cg:id": "MendCond073",
      "disease": [
        "Orphanet:145"
      ],
      "name": "hereditary breast and ovarian cancer"
    }
  ],
  "clinicalSignificance": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp256">VarInterp256</a></p></td>
          <td>classified by Sharing Clinical Reports Project.</td>
          <td>
              <p><a href="details/details.html#CanAll115">CanAll115</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond043">MendCond043</a></p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp256"><pre><code>{
  "cg:id": "VarInterp256",
  "cg:type": "VariantInterpretation",
  "explanation": "classified by Sharing Clinical Reports Project.",
  "variant": {
    "cg:id": "CanAll115",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll121",
    "identifier": "http://reg.genome.network/allele/CA016113"
  },
  "condition": [
    {
      "cg:id": "MendCond043",
      "cg:type": "MendelianCondition",
      "explanation": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2",
      "disease": [
        "OMIM:612555"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp257">VarInterp257</a></p></td>
          <td>classified by OMIM</td>
          <td>
              <p><a href="details/details.html#CanAll116">CanAll116</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond044">MendCond044</a></p>
         </td>
          <td>
              <p>LN:LA6675-8</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp257"><pre><code>{
  "cg:id": "VarInterp257",
  "cg:type": "VariantInterpretation",
  "explanation": "classified by OMIM",
  "variant": {
    "cg:id": "CanAll116",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll122",
    "identifier": "http://reg.genome.network/allele/CA113946"
  },
  "condition": [
    {
      "cg:id": "MendCond044",
      "cg:type": "MendelianCondition",
      "explanation": "Gyrate atrophy of choroid and retina",
      "disease": [
        "Orphanet:414"
      ]
    }
  ],
  "clinicalSignificance": "LN:LA6675-8"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp270">VarInterp270</a></p></td>
          <td>Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome is caused by a combination of a pathogenic variant in COL2A1 and another in COMP.   So CA113 may be involved, but is not the sole cause.</td>
          <td>
              <p><a href="details/details.html#CanAll113">CanAll113</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond041">MendCond041</a></p>
         </td>
          <td>
              <p>LN:LA26333-7</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp270"><pre><code>{
  "cg:id": "VarInterp270",
  "cg:type": "VariantInterpretation",
  "explanation": "Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome is caused by a combination of a pathogenic variant in COL2A1 and another in COMP.   So CA113 may be involved, but is not the sole cause.",
  "variant": {
    "cg:id": "CanAll113",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll119",
    "identifier": "http://reg.genome.network/allele/CA254703"
  },
  "condition": [
    {
      "cg:id": "MendCond041",
      "cg:type": "MendelianCondition",
      "explanation": "Pseudoachondroplastic spondyloepiphyseal dysplasia",
      "disease": [
        "Orphanet:750"
      ],
      "name": "Pseudoachondroplastic spondyloepiphyseal dysplasia"
    }
  ],
  "clinicalSignificance": "LN:LA26333-7"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp274">VarInterp274</a></p></td>
          <td>The Trp2490_Leu2496del variant in DNAH5 leads to an in-frame deletion of 7 amino acids. This variant has been reported together with a second DNAH5 variant (Met2083Ile) in one individual with PCD and situs inversus (Berg 2011). In addition, this variant has been identified in trans configuration with a disease-causing variant in one affected proband (LMM unpublished data). Data from large population studies is insufficient to determine whether this variant is present in the general population. In summary, this variant is likely pathogenic, though additional studies are required to fully establish its clinical significance.</td>
          <td>
              <p><a href="details/details.html#CanAll035">CanAll035</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond063">MendCond063</a></p>
         </td>
          <td>
              <p>LN:LA26332-9</p>
         </td>
          <td>
              <p><a href="details/details.html#AssertMeth001">AssertMeth001</a></p>
         </td>
          <td>
              <p><a href="details/details.html#EvLn110">EvLn110</a></p><p><a href="details/details.html#EvLn111">EvLn111</a></p><p><a href="details/details.html#EvLn112">EvLn112</a></p>
         </td>
          <!--
          <td><div class="example_view" id="VarInterp274"><pre><code>{
  "cg:id": "VarInterp274",
  "cg:type": "VariantInterpretation",
  "explanation": "The Trp2490_Leu2496del variant in DNAH5 leads to an in-frame deletion of 7 amino acids. This variant has been reported together with a second DNAH5 variant (Met2083Ile) in one individual with PCD and situs inversus (Berg 2011). In addition, this variant has been identified in trans configuration with a disease-causing variant in one affected proband (LMM unpublished data). Data from large population studies is insufficient to determine whether this variant is present in the general population. In summary, this variant is likely pathogenic, though additional studies are required to fully establish its clinical significance.",
  "variant": {
    "cg:id": "CanAll035",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll039",
    "identifier": "http://reg.genome.network/allele/CA090919"
  },
  "condition": [
    {
      "cg:id": "MendCond063",
      "cg:type": "MendelianCondition",
      "explanation": "Primary ciliary dyskenisia, Autosomal Recessive",
      "disease": [
        "Orphanet:244"
      ],
      "modeOfInheritance": "HP:0000007"
    }
  ],
  "clinicalSignificance": "LN:LA26332-9",
  "assertionMethod": {
    "cg:id": "AssertMeth001",
    "cg:type": "AssertionMethod",
    "description": "ACMG ISV guidelines 2015",
    "version": "1.0",
    "url": "https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf",
    "scoringAlgorithm": {
      "cg:id": "ScorAlg01",
      "cg:type": "ScoringAlgorithm",
      "description": "This is the description of how the weighting of the satisfied criterion are typically evaluated to arrive at a specific clinical significance."
    }
  },
  "hasEvidence": [
    {
      "cg:id": "EvLn110",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:na",
      "supportingData": [
        {
          "cg:id": "CritAssess305",
          "cg:type": "CriterionAssessment",
          "explanation": "while the variant is absent from exac, however\n1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say âabsentâ\n2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC).",
          "criterion": {
            "cg:id": "PM2",
            "cg:type": "Criterion",
            "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
            "shortDescription": "Absent from controls",
            "defaultStrength": "CG-evidence-strength:pm",
            "targetInterpretation": "LN:LA6668-3"
          },
          "variant": {
            "cg:id": "CanAll035",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll039",
            "identifier": "http://reg.genome.network/allele/CA090919"
          },
          "condition": [
            {
              "cg:id": "-"
            }
          ],
          "outcome": "CG-criterion-outcome:not-met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn040",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "AllFreq082",
                  "cg:type": "AlleleFrequency",
                  "ascertainment": "ExAC",
                  "population": "CG-population-type:combined",
                  "allele": {
                    "cg:id": "CanAll035",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll039",
                    "identifier": "http://reg.genome.network/allele/CA090919"
                  },
                  "alleleCount": 0,
                  "alleleFrequency": 0.0,
                  "medianCoverage": 11.0,
                  "contribution": [
                    {
                      "cg:id": "Contrib488",
                      "cg:type": "Contribution",
                      "agent": "Harry Stevenson",
                      "onDate": {
                        "cg:id": "2016-06-01T14:15:00+00:00"
                      },
                      "role": "CG-contributory-role:curator"
                    }
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib431",
              "cg:type": "Contribution",
              "agent": "Harry Stevenson",
              "onDate": {
                "cg:id": "2016-07-01T10:53:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            },
            {
              "cg:id": "Contrib432",
              "cg:type": "Contribution",
              "agent": "CSER Bakeoff Project",
              "onDate": {
                "cg:id": "2016-07-01T10:53:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn111",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:pm",
      "supportingData": [
        {
          "cg:id": "CritAssess296",
          "cg:type": "CriterionAssessment",
          "criterion": {
            "cg:id": "PM3",
            "cg:type": "Criterion",
            "description": "For recessive disorders, detected in trans with a pathogenic variant",
            "shortDescription": "Recessive in trans with pathogenic variant",
            "defaultStrength": "CG-evidence-strength:pm",
            "targetInterpretation": "LN:LA6668-3"
          },
          "variant": {
            "cg:id": "CanAll035",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll039",
            "identifier": "http://reg.genome.network/allele/CA090919"
          },
          "condition": [
            {
              "cg:id": "MendCond062",
              "cg:type": "MendelianCondition",
              "explanation": "Primary ciliary dyskenisia, Autosomal Recessive",
              "disease": [
                "Orphanet:244"
              ],
              "modeOfInheritance": "HP:0000007"
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn029",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "IndCond056",
                  "cg:type": "IndividualCondition",
                  "individual": {
                    "cg:id": "Indiv004",
                    "cg:type": "Individual",
                    "description": "individual #004"
                  },
                  "condition": [
                    {
                      "cg:id": "MendCond014",
                      "cg:type": "MendelianCondition",
                      "explanation": "PCD autosomal recessive w/ outer dynein arm (ODA) defects",
                      "name": "PCD autosomal recessive w/ outer dynein arm (ODA) defects",
                      "disease": [
                        "Orphanet:244"
                      ],
                      "modeOfInheritance": "HP:0000007",
                      "phenotype": [
                        "HP:0200106"
                      ]
                    }
                  ],
                  "hasCondition": true,
                  "contribution": [
                    {
                      "cg:id": "Contrib483",
                      "cg:type": "Contribution",
                      "agent": "Stefan DiMarino",
                      "onDate": {
                        "cg:id": "2016-05-31T08:45:59+00:00"
                      },
                      "role": "CG-contributory-role:curator"
                    }
                  ]
                },
                {
                  "cg:id": "VarInterp057",
                  "cg:type": "VariantInterpretation",
                  "variant": {
                    "cg:id": "CanAll036",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll040",
                    "identifier": "http://reg.genome.network/allele/CA273615"
                  },
                  "condition": [
                    {
                      "cg:id": "MendCond062",
                      "cg:type": "MendelianCondition",
                      "explanation": "Primary ciliary dyskenisia, Autosomal Recessive",
                      "disease": [
                        "Orphanet:244"
                      ],
                      "modeOfInheritance": "HP:0000007"
                    }
                  ],
                  "clinicalSignificance": "LN:LA6668-3",
                  "contribution": [
                    {
                      "cg:id": "Contrib485",
                      "cg:type": "Contribution",
                      "agent": "Stefan DiMarino",
                      "onDate": {
                        "cg:id": "2016-05-31T11:00:59+00:00"
                      },
                      "role": "CG-contributory-role:interpreter"
                    }
                  ]
                },
                {
                  "cg:id": "IndAll058",
                  "cg:type": "IndividualAllele",
                  "explanation": "Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband.",
                  "individual": {
                    "cg:id": "Indiv004",
                    "cg:type": "Individual",
                    "description": "individual #004"
                  },
                  "primaryAllele": {
                    "cg:id": "CanAll035",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll039",
                    "identifier": "http://reg.genome.network/allele/CA090919"
                  },
                  "secondaryAllele": {
                    "cg:id": "CanAll036",
                    "cg:type": "CanonicalAllele",
                    "preferredCtxAllele": "CtxAll040",
                    "identifier": "http://reg.genome.network/allele/CA273615"
                  },
                  "primaryZygosity": "CG-allelic-state:het",
                  "secondaryZygosity": "CG-allelic-state:het",
                  "phase": "CG-phase:trans",
                  "contribution": [
                    {
                      "cg:id": "Contrib484",
                      "cg:type": "Contribution",
                      "agent": "Harry Stevenson",
                      "onDate": {
                        "cg:id": "2016-05-31T09:14:59+00:00"
                      },
                      "role": "CG-contributory-role:curator"
                    }
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib422",
              "cg:type": "Contribution",
              "agent": "Harry Stevenson",
              "onDate": {
                "cg:id": "2016-06-02T08:00:59+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    },
    {
      "cg:id": "EvLn112",
      "cg:type": "EvidenceLine",
      "strength": "CG-evidence-strength:pm",
      "supportingData": [
        {
          "cg:id": "CritAssess299",
          "cg:type": "CriterionAssessment",
          "explanation": "leads to an in-frame deletion of 7 amino acids (not repeat region).",
          "criterion": {
            "cg:id": "PM4",
            "cg:type": "Criterion",
            "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
            "shortDescription": "Protein length changes or stop-loss",
            "defaultStrength": "CG-evidence-strength:pm",
            "targetInterpretation": "LN:LA6668-3"
          },
          "variant": {
            "cg:id": "CanAll035",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll039",
            "identifier": "http://reg.genome.network/allele/CA090919"
          },
          "condition": [
            {
              "cg:id": "-"
            }
          ],
          "outcome": "CG-criterion-outcome:met",
          "hasSupportingEvidence": [
            {
              "cg:id": "EvLn032",
              "cg:type": "EvidenceLine",
              "supportingData": [
                {
                  "cg:id": "MolCon066",
                  "cg:type": "MolecularConsequence",
                  "contextualAllele": {
                    "cg:id": "CtxAll044",
                    "cg:type": "ContextualAllele",
                    "relatedCanonicalAllele": {
                      "cg:id": "CanAll040",
                      "cg:type": "CanonicalAllele",
                      "preferredCtxAllele": "CtxAll044",
                      "identifier": "http://reg.genome.network/allele/CA273136"
                    },
                    "alleleName": "NP_001360.1:p.Trp2490_Leu2496del"
                  },
                  "consequence": "SO:0001825",
                  "contribution": [
                    {
                      "cg:id": "Contrib486",
                      "cg:type": "Contribution",
                      "agent": "Harry Stevenson",
                      "onDate": {
                        "cg:id": "2016-06-01T14:03:00+00:00"
                      },
                      "role": "CG-contributory-role:curator"
                    }
                  ]
                },
                {
                  "cg:id": "RegAnn069",
                  "cg:type": "RegionAnnotation",
                  "explanation": "This is not a repreating region.",
                  "region": {
                    "cg:id": "Loc385",
                    "cg:type": "Locus",
                    "referenceSequence": {
                      "cg:id": "RefSeq005",
                      "cg:type": "ReferenceSequence",
                      "accession": "NP_001360.1"
                    },
                    "start": 2490,
                    "stop": 2496
                  },
                  "isTypes": [
                    "CG-reg-anno-type:repeat"
                  ],
                  "contribution": [
                    {
                      "cg:id": "Contrib487",
                      "cg:type": "Contribution",
                      "agent": "Stefan DiMarino",
                      "onDate": {
                        "cg:id": "2016-06-01T14:10:00+00:00"
                      },
                      "role": "CG-contributory-role:curator"
                    }
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "cg:id": "Contrib425",
              "cg:type": "Contribution",
              "agent": "Sasha Rande",
              "onDate": {
                "cg:id": "2016-06-12T07:30:00+00:00"
              },
              "role": "CG-contributory-role:assessor"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "cg:id": "Contrib433",
      "cg:type": "Contribution",
      "agent": "Agent009",
      "onDate": {
        "cg:id": "2015-06-26T12:02:59+00:00"
      },
      "role": "CG-contributory-role:interpreter"
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp364">VarInterp364</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll122">CanAll122</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond058">MendCond058</a></p>
         </td>
          <td>
              <p>LN:LA26332-9</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp364"><pre><code>{
  "cg:id": "VarInterp364",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll122",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll128"
  },
  "condition": [
    {
      "cg:id": "MendCond058",
      "cg:type": "MendelianCondition",
      "explanation": "Hearing Loss",
      "disease": [
        "Orphanet:87884"
      ],
      "name": "Hearing Loss"
    }
  ],
  "clinicalSignificance": "LN:LA26332-9"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#VarInterp365">VarInterp365</a></p></td>
          <td></td>
          <td>
              <p><a href="details/details.html#CanAll123">CanAll123</a></p>
         </td>
          <td>
              <p><a href="details/details.html#MendCond058">MendCond058</a></p>
         </td>
          <td>
              <p>LN:LA6668-3</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="VarInterp365"><pre><code>{
  "cg:id": "VarInterp365",
  "cg:type": "VariantInterpretation",
  "variant": {
    "cg:id": "CanAll123",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll129"
  },
  "condition": [
    {
      "cg:id": "MendCond058",
      "cg:type": "MendelianCondition",
      "explanation": "Hearing Loss",
      "disease": [
        "Orphanet:87884"
      ],
      "name": "Hearing Loss"
    }
  ],
  "clinicalSignificance": "LN:LA6668-3"
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("cg:id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("cg:id")').next('.string')[0].innerText + ', â¦</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>â¦ [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../javascripts/jquery.js"></script><script src="../javascripts/bootstrap.js"></script><script src="../javascripts/jquery.jsonview.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
