<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      Genetic Condition
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a class="navbar-brand" href="/">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a target="_blank" href="http://www.clinicalgenome.org"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>
    


    
        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">Genetic Condition</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../../">Interpretation Model</a></li>
<li><a href="../">User Guide</a></li>
<li><a href="./">Discussion</a></li>
<li><a href="condition.html">Genetic Condition</a></li>

              </ol>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            <h2 id="introduction">Introduction</h2>

<p>Clinical geneticists use a variety of methods and systems to define the conditions for which a variant is being interpreted.  In some systems, such as OMIM, the gene is an integral part of the definition of the condition (as described in a phenotype MIM record). If a different gene is involved, then the condition is considered to be different (but may be part of the same OMIM phenotypic series).   Other systems include higher-level descriptions where a disease is defined phenotypically, and may result from variants affecting many different genes.  Similar situations hold for mode of inheritance, which may be part of the disease definition in some systems, and not in others.  Furthermore, ontologies may contain terms in which all of these situations occur.  Finally, a condition may not have a previous entry in any nosology, and may only be defined by a given patient with a rare collection of phenotypes.</p>

<h2 id="disease-phenotype-and-geneticcondition">Disease, Phenotype, and GeneticCondition</h2>

<p>Rather than enforce one particular view of genetic disease, which would necessarily exclude many popular terminology sets, the ClinGen model provides flexibility for describing conditions.  Specifically, the model includes classes for both Disease and Phenotype, which are DomainEntities corresponding to terms from externally managed vocabularies such as MONDO for diseases or HPO for phenotypes.</p>

<p>To allow for the wide variety of condition definitions described in the introduction, we introduce the GeneticCondition class as the structure used in VariantInterpretations.  That is, a GeneticCondition is the entity for which a variant can be described as being benign or pathogenic.  GeneticConditions include a large number of optional items that can be used to define conditions in a flexible way.  For instance, if a single disease from a terminology is used, the GeneticCondition may only have a single disease attribute set.  In the case of a non-predefined disease, a collection of phenotypes may be used.  The two may even be combined to indicate a subset of a disease in which other phenotypes additionally occur.</p>

<h2 id="conditions-for-benign-variants">Conditions for Benign Variants</h2>

<p>What should be used as the condition for benign variants?  Some users see benign variants as fully benign - a benign variant is one that does not cause any genetic condition. However, it is not feasible or meaningful to create individual statements for "Variant 1 is benign for Condition A", "Variant 1 is benign for Condition B", etc., for all known genetic conditions. Furthermore, this is not always the most descriptive interpretation when an assertion may be made regarding a specific condition.  We distinguish two cases:</p>

<ol>
  <li>
    <p>The evidence examined indicates that the variant is benign for any sufficiently rare, penetrant condition.  This is often the case if the variant has a high population frequency.  In this case, the ACMG Guidelines would allow a stand-alone benign interpretation of the variant for any condition to which the guidelines are applicable, and a GeneticCondition does not need to be included on the VariantInterpretation.</p>
  </li>
  <li>
    <p>The evidence examined is specific to a particular condition. For instance, if the frequency of the condition is much lower than the frequency of the allele, then there is strong reason to believe that the variant is benign, but only for the condition with that frequency.  Another example would be a statement that a specific variant in a gene may cause a particular phenotype but be benign for another phenotype associated with some variants in a gene (this would typically be supported by clinical data).  In these cases, the evidence only supports a benign call for a particular condition, and the condition must be included as an element in the VariantInterpretation.</p>
  </li>
</ol>

          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../../">Interpretation Overview</a></h5>
              </li>
              <li class="list-group-item"><h5><a href="../">User Guide</a></h5><ul class='list-unstyled'><li class="active"><a href="./">Discussion</a></li><ul class='list-unstyled'><li><a href="alleles.html">Alleles, Haplotypes, and Genotypes</a></li><li class="active"><a href="condition.html">Genetic Condition</a></li><li><a href="domain_entity.html">DomainEntities and ValueSets</a></li><li><a href="json.html">JSON-LD</a></li><li><a href="qualifiers.html">Statement Qualifiers</a></li></ul><li><a href="../examples.html">Examples</a></li><li><a href="../projects.html">Projects</a></li><li><a href="../release_notes.html">Release Notes</a></li><li><a href="../sepio.html">SEPIO</a></li></ul></li>
              <li class='list-group-item'><h5><a href="../../tech/">Technical Reference</a></h5></li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          Â© 2017 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../javascripts/jquery.js"></script><script src="../../javascripts/bootstrap.js"></script><script src="../../javascripts/jquery.jsonview.js"></script><script src="../../javascripts/all.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
