<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

    <link href="../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/model/entities/criterion_assessment.html">0.1</a></li>
                  <li><a href="/development/tech/model/entities/criterion_assessment.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../../../">Interpretation Model</a></li>
<li><a href="../../">Technical Reference</a></li>
<li><a href="../">Model</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../../../images/clingen-doc-logo.png" />
            </li>

            <li><a href="../../../welcome/">Welcome</a></li>
            <li><a href="../../../user/">User Guide</a></li>
            <li class="active"><a href="../../">Technical Reference</a></li><ul><li><a href="../../api/">API</a></li><li><a href="../../examples/">Examples</a></li><li class="active"><a href="../">Model</a></li><ul><li><a href="../activities/">Activity</a></li><li><a href="../agents/">Agent</a></li><li class="active"><a href="./">Entity</a></li><ul><li class="active"><a href="criterion_assessment.html">CriterionAssessment</a></li><li><a href="evidence/">Data</a></li><li><a href="evidence_data-source.html">DataSource</a></li><li><a href="publication_status.html">PublicationStatus</a></li><li><a href="variant_interpretation.html">VariantInterpretation</a></li></ul><li><a href="../secondary/">Secondary Types</a></li></ul><li><a href="../../value_sets/">Value Sets</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>CriterionAssessment</h1>
            <p>Intermediate interpretation of a line of evidence</p>

                <p>Descended from Entity</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>A CriterionAssessment represents the result of applying a named Criterion to a set of Data.</p>

<p>A VariantInterpretation is based on assessing different lines of evidence, such as population frequency data, or case-control studies.  A CriterionAssessment is the result of gathering together the data necessary to understand a single line of evidence, and deciding how that set of evidence hangs together to create a single unit of understanding. One or more CriterionAssessments, each of which describes a judgement about a particular line of evidence, are then subsequently combined to produce the final interpretation.</p>

<p>In addition to the Data, the CriterionAssessment references a particular Criterion.  The Criterion is a pre-defined rule that describes how to use a particular type of data to reach a conclusion.</p>

<p>In the context of ACMG Variant Pathogenicity interpretations, a Criterion is a particular rule, such as PM1, and the CriterionAssessment is the statement, for example, that this set of evidence provides moderate support that the variant is pathogenic, or that the evidence is insufficient to apply the Criterion.</p>

<p>A CriterionAssessment that lacks the Criterion attribute represents a conclusion based on a set of evidence, but which is evaluated in a way outside of a pre-defined set of criteria.  While a Criterion is preferable, and will be expressly required in certain cases, making this attribute optional allows the model to express interpretation structures that are constructed outside of the ACMG Guideline framework.</p>

<p>Given the Criterion and the Data, the logic of a particular CriterionAssessment is meant to be transparent.  However, to be complete, a human-readable description of the reasoning behind a CriterionAssessment may be included in the explanation attribute (inherited from Entity).</p>

<p>CriterionAssessments are generated by an AssessCriterion Activity, and are subsequently used as inputs to a GenerateVariantInterpretation Activity.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>criterion</td>
                <td>Criterion</td>
                <td>0..1</td>
                <td>The rule describing how the data is being used</td>
            </tr>
            <tr>
                <td>variant</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>Variant about which the assemssment is made</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>Condition</td>
                <td>0..*</td>
                <td>Condition for which the assessment is made</td>
            </tr>
            <tr>
                <td>outcome</td>
                <td>CodeableConcept</td>
                <td>1..1</td>
                <td>Result of assessing the data and criterion</td>
            </tr>
            </tbody>
        </table>

    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>wasGeneratedBy</td>
                    <td>Activity</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>The activity that produced this entity</td>
                </tr>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>A human-readable description of the entity</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>description</th>
        <th>json</th>
        <th>diagram</th>
      </tr></thead>
      <tbody>
        <tr>
          <td>CritAssess001</td>
          <td>Variant is not LOF and LOF is not a known mechanism of disease associated with this gene for any condition.</td>
          <td><div class="example_view" id="CritAssess001"><pre><code>{
  "cg:id": "CritAssess001",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PVS1",
    "cg:type": "Criterion",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "shortDescription": "Null Variant in known LOF genes",
    "defaultOutcome": "CG:VS",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll001",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll001",
    "identifier": "http://reg.genome.network/allele/CA090943"
  },
  "outcome": "CG:R",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit001",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-01T10:10:29+00:00",
    "endedAtTime": "2016-06-01T10:10:29+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon001",
        "cg:type": "MolecularConsequence",
        "explanation": "Protein ends at p.826 and this frameshift would create stop at p.852 (769+83) thus NMD would not occur. Instead an elongated protein",
        "contextualAllele": {
          "cg:id": "CtxAll001",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll001",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll001",
            "identifier": "http://reg.genome.network/allele/CA090943"
          },
          "alleleName": "NM_031844.2(HNRNPU):c.2304_2305del (p.Gly769Glufs*83)"
        },
        "consequence": "SO:0001589",
        "lof": true
      },
      {
        "cg:id": "CondMech002",
        "cg:type": "ConditionMechanism",
        "explanation": "HNRNPU is not associated with any conditions, and therefore LOF is not an established mechanism of any condition for this gene.",
        "mechanism": "SO:0002054",
        "strength": "CG:established",
        "gene": {
          "cg:id": "G001",
          "cg:type": "Gene",
          "symbol": "HNRNPU"
        }
      }
    ],
    "criterion": {
      "cg:id": "PVS1",
      "cg:type": "Criterion",
      "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
      "shortDescription": "Null Variant in known LOF genes",
      "defaultOutcome": "CG:VS",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll001",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll001",
      "identifier": "http://reg.genome.network/allele/CA090943"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "Variant is not LOF and LOF is not a known mechanism of disease associated with this gene for any condition."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess001">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess002</td>
          <td></td>
          <td><div class="example_view" id="CritAssess002"><pre><code>{
  "cg:id": "CritAssess002",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PVS1",
    "cg:type": "Criterion",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "shortDescription": "Null Variant in known LOF genes",
    "defaultOutcome": "CG:VS",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll002",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll002"
  },
  "outcome": "CG:VS",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit002",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-02T12:14:59+00:00",
    "endedAtTime": "2016-06-02T12:14:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon003",
        "cg:type": "MolecularConsequence",
        "explanation": "LOF is predicted to occur as variant is >55bp from penultimate exon",
        "contextualAllele": {
          "cg:id": "CtxAll002",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll002",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll002"
          },
          "alleleName": "NM_000059.3(BRCA2):c.7762_7764delinsTT"
        },
        "consequence": "SO:0001589",
        "lof": true
      },
      {
        "cg:id": "CondMech004",
        "cg:type": "ConditionMechanism",
        "explanation": "This is so well-known that it does not require a citation.",
        "mechanism": "SO:0002054",
        "strength": "CG:established",
        "gene": {
          "cg:id": "G002",
          "cg:type": "Gene",
          "symbol": "BRCA2"
        },
        "condition": {
          "cg:id": "Cond001",
          "cg:type": "Condition",
          "name": "Breast Ovarian Cancer",
          "code": "D061325"
        }
      }
    ],
    "criterion": {
      "cg:id": "PVS1",
      "cg:type": "Criterion",
      "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
      "shortDescription": "Null Variant in known LOF genes",
      "defaultOutcome": "CG:VS",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll002",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll002"
    },
    "condition": [
      {
        "cg:id": "Cond001",
        "cg:type": "Condition",
        "name": "Breast Ovarian Cancer",
        "code": "D061325"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess002">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess003</td>
          <td>This variant only occurs as LOF in an uncommon transcript (NM_133437.4).  For most transcripts, the variant is intronic.</td>
          <td><div class="example_view" id="CritAssess003"><pre><code>{
  "cg:id": "CritAssess003",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PVS1",
    "cg:type": "Criterion",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "shortDescription": "Null Variant in known LOF genes",
    "defaultOutcome": "CG:VS",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll003",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll003",
    "identifier": "http://reg.genome.network/allele/CA238217"
  },
  "outcome": "CG:R",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit003",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-02T14:22:00+00:00",
    "endedAtTime": "2016-06-02T14:22:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon005",
        "cg:type": "MolecularConsequence",
        "explanation": "LOF is predicted but this transcript is rare",
        "contextualAllele": {
          "cg:id": "CtxAll003",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll003",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll003",
            "identifier": "http://reg.genome.network/allele/CA238217"
          },
          "alleleName": "NM_133437.4(TTN):c.10670dupG "
        },
        "consequence": "SO:0001589",
        "lof": true
      },
      {
        "cg:id": "MolCon006",
        "cg:type": "MolecularConsequence",
        "contextualAllele": {
          "cg:id": "CtxAll004",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll003",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll003",
            "identifier": "http://reg.genome.network/allele/CA238217"
          },
          "alleleName": "LRG_391t2:c.10303+2691dupG"
        },
        "consequence": "SO:0000191",
        "lof": true
      },
      {
        "cg:id": "CondMech007",
        "cg:type": "ConditionMechanism",
        "mechanism": "SO:0002054",
        "strength": "CG:established",
        "gene": {
          "cg:id": "G003",
          "cg:type": "Gene",
          "symbol": "TTN"
        },
        "condition": {
          "cg:id": "Cond002",
          "cg:type": "Condition",
          "name": "Dilated Cardiomyopathy",
          "code": "217604"
        }
      }
    ],
    "criterion": {
      "cg:id": "PVS1",
      "cg:type": "Criterion",
      "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
      "shortDescription": "Null Variant in known LOF genes",
      "defaultOutcome": "CG:VS",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll003",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll003",
      "identifier": "http://reg.genome.network/allele/CA238217"
    },
    "condition": [
      {
        "cg:id": "Cond002",
        "cg:type": "Condition",
        "name": "Dilated Cardiomyopathy",
        "code": "217604"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "This variant only occurs as LOF in an uncommon transcript (NM_133437.4).  For most transcripts, the variant is intronic."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess003">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess004</td>
          <td>The assessed variant produces the same amino acid change (p.Phe285Leu) as known pathogenic variant c.853T>C</td>
          <td><div class="example_view" id="CritAssess004"><pre><code>{
  "cg:id": "CritAssess004",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS1",
    "cg:type": "Criterion",
    "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
    "shortDescription": "same as previous pathogenic amino acid change",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll004",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll005",
    "identifier": "http://reg.genome.network/allele/CA261606"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit004",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-02T15:21:59+00:00",
    "endedAtTime": "2016-06-02T15:21:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp008",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "NM_002834.3:c.855T>G Pathogenic for Noonan Syndrome",
        "canonicalAllele": {
          "cg:id": "CanAll005",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll006",
          "identifier": "http://reg.genome.network/allele/CA220155"
        },
        "condition": [
          {
            "cg:id": "Cond003",
            "cg:type": "Condition",
            "name": "Noonan Syndrome",
            "code": "648"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "PS1",
      "cg:type": "Criterion",
      "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
      "shortDescription": "same as previous pathogenic amino acid change",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll004",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll005",
      "identifier": "http://reg.genome.network/allele/CA261606"
    },
    "condition": [
      {
        "cg:id": "Cond003",
        "cg:type": "Condition",
        "name": "Noonan Syndrome",
        "code": "648"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "The assessed variant produces the same amino acid change (p.Phe285Leu) as known pathogenic variant c.853T>C"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess004">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess005</td>
          <td>The assessed variant produces the same amino acid change (p.Met2083Ile) as known pathogenic variant c.6249G>A.  But that change is pathogenic because it has been shown to disrupt splicing and create a premature stop codon. Prediction tools do not suggest c.6249G>C disrupts splicing.</td>
          <td><div class="example_view" id="CritAssess005"><pre><code>{
  "cg:id": "CritAssess005",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS1",
    "cg:type": "Criterion",
    "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
    "shortDescription": "same as previous pathogenic amino acid change",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll007",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll008",
    "identifier": "http://reg.genome.network/allele/CA501220"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit005",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-02T16:21:59+00:00",
    "endedAtTime": "2016-06-02T16:21:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp009",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "NM_001369.2:c.6249G>A Pathogenic for PCD",
        "canonicalAllele": {
          "cg:id": "CanAll008",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll009",
          "identifier": "http://reg.genome.network/allele/CA501221"
        },
        "condition": [
          {
            "cg:id": "Cond004",
            "cg:type": "Condition",
            "name": "Primary Ciliary Dyskenisia",
            "code": "244"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "PS1",
      "cg:type": "Criterion",
      "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
      "shortDescription": "same as previous pathogenic amino acid change",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll007",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll008",
      "identifier": "http://reg.genome.network/allele/CA501220"
    },
    "condition": [
      {
        "cg:id": "Cond004",
        "cg:type": "Condition",
        "name": "Primary Ciliary Dyskenisia",
        "code": "244"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "The assessed variant produces the same amino acid change (p.Met2083Ile) as known pathogenic variant c.6249G>A.  But that change is pathogenic because it has been shown to disrupt splicing and create a premature stop codon. Prediction tools do not suggest c.6249G>C disrupts splicing."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess005">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess006</td>
          <td>While the two amino acid changes are not identical, they are the same amino acid change in analagous residues of HRAS and KRAS.</td>
          <td><div class="example_view" id="CritAssess006"><pre><code>{
  "cg:id": "CritAssess006",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS1",
    "cg:type": "Criterion",
    "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
    "shortDescription": "same as previous pathogenic amino acid change",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll010",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll012",
    "identifier": "http://reg.genome.network/allele/CA341206"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit006",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-02T17:22:00+00:00",
    "endedAtTime": "2016-06-02T17:22:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp010",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Noonan Syndrome 3",
        "canonicalAllele": {
          "cg:id": "CanAll011",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll014",
          "identifier": "http://reg.genome.network/allele/CA256480"
        },
        "condition": [
          {
            "cg:id": "Cond006",
            "cg:type": "Condition",
            "name": "Noonan syndrome 3",
            "code": "609942"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      },
      {
        "cg:id": "MenDisVarInterp011",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Costello syndrome",
        "canonicalAllele": {
          "cg:id": "CanAll011",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll014",
          "identifier": "http://reg.genome.network/allele/CA256480"
        },
        "condition": [
          {
            "cg:id": "Cond045",
            "cg:type": "Condition",
            "name": "Rasopathy",
            "code": "98733"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "PS1",
      "cg:type": "Criterion",
      "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
      "shortDescription": "same as previous pathogenic amino acid change",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll011",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll014",
      "identifier": "http://reg.genome.network/allele/CA256480"
    },
    "condition": [
      {
        "cg:id": "Cond005",
        "cg:type": "Condition",
        "name": "Costello Syndrome",
        "code": "3071"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      },
      {
        "cg:id": "Agent004",
        "cg:type": "Agent",
        "description": "NWG",
        "name": "Noonan/Rasopathy WG (ClinGen)"
      }
    ]
  },
  "explanation": "While the two amino acid changes are not identical, they are the same amino acid change in analagous residues of HRAS and KRAS."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess006">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess007</td>
          <td>IC1: Individual has condition, FH1: No family history, DNA1: variant is denovo (parentage confirmed)</td>
          <td><div class="example_view" id="CritAssess007"><pre><code>{
  "cg:id": "CritAssess007",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS2",
    "cg:type": "Criterion",
    "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
    "shortDescription": "confirmed de novo no family history",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll014",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll017",
    "identifier": "http://reg.genome.network/allele/CA016441"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit007",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-03T08:00:00+00:00",
    "endedAtTime": "2016-06-03T08:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndCond012",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv001",
          "cg:type": "Individual",
          "description": "individual #001"
        },
        "condition": [
          {
            "cg:id": "Cond007",
            "cg:type": "Condition",
            "name": "Cardiomyopathy",
            "code": "167848"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "FamHist013",
        "cg:type": "FamilyHistory",
        "proband": {
          "cg:id": "Indiv001",
          "cg:type": "Individual",
          "description": "individual #001"
        },
        "condition": [
          {
            "cg:id": "Cond007",
            "cg:type": "Condition",
            "name": "Cardiomyopathy",
            "code": "167848"
          }
        ],
        "familyHasCondition": true
      },
      {
        "cg:id": "DeNovo014",
        "cg:type": "DeNovoAllele",
        "explanation": "parentage confirmed via haplotyping.",
        "canonicalAllele": {
          "cg:id": "CanAll014",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll017",
          "identifier": "http://reg.genome.network/allele/CA016441"
        },
        "individual": {
          "cg:id": "Indiv001",
          "cg:type": "Individual",
          "description": "individual #001"
        },
        "maternityConfirmed": true,
        "paternityConfirmed": true
      }
    ],
    "criterion": {
      "cg:id": "PS2",
      "cg:type": "Criterion",
      "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
      "shortDescription": "confirmed de novo no family history",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll014",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll017",
      "identifier": "http://reg.genome.network/allele/CA016441"
    },
    "condition": [
      {
        "cg:id": "Cond007",
        "cg:type": "Condition",
        "name": "Cardiomyopathy",
        "code": "167848"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "IC1: Individual has condition, FH1: No family history, DNA1: variant is denovo (parentage confirmed)"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess007">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess008</td>
          <td>De novo variant occurred on haplotype inherited from the father. Maternity no explicitly confirmed - but if variant occurred on haplotype inherited from the father, then confirmation of maternity not required.</td>
          <td><div class="example_view" id="CritAssess008"><pre><code>{
  "cg:id": "CritAssess008",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS2",
    "cg:type": "Criterion",
    "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
    "shortDescription": "confirmed de novo no family history",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll015",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll018",
    "identifier": "http://reg.genome.network/allele/CA257646"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit008",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-03T09:17:19+00:00",
    "endedAtTime": "2016-06-03T09:17:19+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndCond015",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv002",
          "cg:type": "Individual",
          "description": "individual #002"
        },
        "condition": [
          {
            "cg:id": "Cond008",
            "cg:type": "Condition",
            "name": "desmin related myopathy",
            "code": "98909"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "FamHist016",
        "cg:type": "FamilyHistory",
        "proband": {
          "cg:id": "Indiv002",
          "cg:type": "Individual",
          "description": "individual #002"
        },
        "condition": [
          {
            "cg:id": "Cond008",
            "cg:type": "Condition",
            "name": "desmin related myopathy",
            "code": "98909"
          }
        ],
        "familyHasCondition": true
      },
      {
        "cg:id": "DeNovo017",
        "cg:type": "DeNovoAllele",
        "explanation": "Non-paternity was excluded by assessment with microsatellites",
        "canonicalAllele": {
          "cg:id": "CanAll015",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll018",
          "identifier": "http://reg.genome.network/allele/CA257646"
        },
        "individual": {
          "cg:id": "Indiv002",
          "cg:type": "Individual",
          "description": "individual #002"
        },
        "maternityConfirmed": true,
        "paternityConfirmed": true
      }
    ],
    "criterion": {
      "cg:id": "PS2",
      "cg:type": "Criterion",
      "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
      "shortDescription": "confirmed de novo no family history",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll015",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll018",
      "identifier": "http://reg.genome.network/allele/CA257646"
    },
    "condition": [
      {
        "cg:id": "Cond008",
        "cg:type": "Condition",
        "name": "desmin related myopathy",
        "code": "98909"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "De novo variant occurred on haplotype inherited from the father. Maternity no explicitly confirmed - but if variant occurred on haplotype inherited from the father, then confirmation of maternity not required."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess008">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess009</td>
          <td></td>
          <td><div class="example_view" id="CritAssess009"><pre><code>{
  "cg:id": "CritAssess009",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS2",
    "cg:type": "Criterion",
    "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
    "shortDescription": "confirmed de novo no family history",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll016",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll019"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit009",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-03T10:25:19+00:00",
    "endedAtTime": "2016-06-03T10:25:19+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndCond018",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv003",
          "cg:type": "Individual",
          "description": "individual #003"
        },
        "condition": [
          {
            "cg:id": "Cond009",
            "cg:type": "Condition",
            "name": "Epilepsy with Mental Retardation",
            "code": "???"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "FamHist019",
        "cg:type": "FamilyHistory",
        "proband": {
          "cg:id": "Indiv003",
          "cg:type": "Individual",
          "description": "individual #003"
        },
        "condition": [
          {
            "cg:id": "Cond009",
            "cg:type": "Condition",
            "name": "Epilepsy with Mental Retardation",
            "code": "???"
          }
        ],
        "familyHasCondition": true
      },
      {
        "cg:id": "DeNovo020",
        "cg:type": "DeNovoAllele",
        "explanation": "Variant from trio WES, which confirmed parentage",
        "canonicalAllele": {
          "cg:id": "CanAll016",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll019"
        },
        "individual": {
          "cg:id": "Indiv003",
          "cg:type": "Individual",
          "description": "individual #003"
        },
        "maternityConfirmed": true,
        "paternityConfirmed": true
      }
    ],
    "criterion": {
      "cg:id": "PS2",
      "cg:type": "Criterion",
      "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
      "shortDescription": "confirmed de novo no family history",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll016",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll019"
    },
    "condition": [
      {
        "cg:id": "Cond009",
        "cg:type": "Condition",
        "name": "Epilepsy with Mental Retardation",
        "code": "???"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess009">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess010</td>
          <td></td>
          <td><div class="example_view" id="CritAssess010"><pre><code>{
  "cg:id": "CritAssess010",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS3",
    "cg:type": "Criterion",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "shortDescription": "well-established damaging effect functional studies",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll017",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll020",
    "identifier": "http://reg.genome.network/allele/CA090410"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit010",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-03T14:49:59+00:00",
    "endedAtTime": "2016-06-03T14:49:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Func021",
        "cg:type": "FunctionalData",
        "result": "the mutation lowered the Ca2+  affinity of the reconstituted thin filaments;",
        "dataType": "CG:binding",
        "contextualAllele": {
          "cg:id": "CtxAll021",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll018",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll021",
            "identifier": "http://reg.genome.network/allele/CA501233"
          },
          "alleleName": "NP_001001431.1:p.Arg131Trp"
        },
        "gene": {
          "cg:id": "G011",
          "cg:type": "Gene",
          "symbol": "TNNT2"
        }
      },
      {
        "cg:id": "Func022",
        "cg:type": "FunctionalData",
        "result": "reduced Ca2 sensitivity of activation",
        "dataType": "CG:atpase",
        "contextualAllele": {
          "cg:id": "CtxAll021",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll018",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll021",
            "identifier": "http://reg.genome.network/allele/CA501233"
          },
          "alleleName": "NP_001001431.1:p.Arg131Trp"
        },
        "gene": {
          "cg:id": "G011",
          "cg:type": "Gene",
          "symbol": "TNNT2"
        }
      },
      {
        "cg:id": "Func023",
        "cg:type": "FunctionalData",
        "result": "reduced Ca2 sensitivity of activation",
        "dataType": "CG:motility",
        "contextualAllele": {
          "cg:id": "CtxAll021",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll018",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll021",
            "identifier": "http://reg.genome.network/allele/CA501233"
          },
          "alleleName": "NP_001001431.1:p.Arg131Trp"
        },
        "gene": {
          "cg:id": "G011",
          "cg:type": "Gene",
          "symbol": "TNNT2"
        }
      },
      {
        "cg:id": "Func024",
        "cg:type": "FunctionalData",
        "result": "desensitized Ca2+ binding to the thin filament.",
        "dataType": "CG:binding",
        "contextualAllele": {
          "cg:id": "CtxAll021",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll018",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll021",
            "identifier": "http://reg.genome.network/allele/CA501233"
          },
          "alleleName": "NP_001001431.1:p.Arg131Trp"
        },
        "gene": {
          "cg:id": "G011",
          "cg:type": "Gene",
          "symbol": "TNNT2"
        }
      }
    ],
    "criterion": {
      "cg:id": "PS3",
      "cg:type": "Criterion",
      "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
      "shortDescription": "well-established damaging effect functional studies",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll017",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll020",
      "identifier": "http://reg.genome.network/allele/CA090410"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent003",
        "cg:type": "Agent",
        "description": "TS",
        "name": "Tasha Strande"
      },
      {
        "cg:id": "Agent005",
        "cg:type": "Agent",
        "description": "CSER",
        "name": "CSER Bake Off Project"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess010">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess011</td>
          <td></td>
          <td><div class="example_view" id="CritAssess011"><pre><code>{
  "cg:id": "CritAssess011",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS3",
    "cg:type": "Criterion",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "shortDescription": "well-established damaging effect functional studies",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll019",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll022",
    "identifier": "http://reg.genome.network/allele/CA090884"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit011",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-03T18:08:59+00:00",
    "endedAtTime": "2016-06-03T18:08:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Func025",
        "cg:type": "FunctionalData",
        "explanation": "Expression of human SPG7 (with this mutation) in yeast with damaged m-AAA protease did not restore respiratory growth or MrpL32 processing, if the function of AFG3L2 was impaired.",
        "result": "Functionally Damaging",
        "dataType": "CG:yeastcomp",
        "contextualAllele": {
          "cg:id": "CtxAll022",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll019",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll022",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "NM_003119.2 (SPG7):c.1529C>T"
        },
        "gene": {
          "cg:id": "G012",
          "cg:type": "Gene",
          "symbol": "SPG7"
        }
      }
    ],
    "criterion": {
      "cg:id": "PS3",
      "cg:type": "Criterion",
      "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
      "shortDescription": "well-established damaging effect functional studies",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll019",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll022",
      "identifier": "http://reg.genome.network/allele/CA090884"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent003",
        "cg:type": "Agent",
        "description": "TS",
        "name": "Tasha Strande"
      },
      {
        "cg:id": "Agent005",
        "cg:type": "Agent",
        "description": "CSER",
        "name": "CSER Bake Off Project"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess011">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess012</td>
          <td>The RASopathy group considers MEK and ERK activation assays as well validated.</td>
          <td><div class="example_view" id="CritAssess012"><pre><code>{
  "cg:id": "CritAssess012",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS3",
    "cg:type": "Criterion",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "shortDescription": "well-established damaging effect functional studies",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll020",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll023",
    "identifier": "http://reg.genome.network/allele/CA220161"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit012",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-04T07:12:00+00:00",
    "endedAtTime": "2016-06-04T07:12:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Func026",
        "cg:type": "FunctionalData",
        "result": "abnormal/deceased MEK activtation.",
        "dataType": "CG:activation",
        "contextualAllele": {
          "cg:id": "CtxAll024",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll021",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll024",
            "identifier": "http://reg.genome.network/allele/CA093846"
          },
          "alleleName": "NP_004324.2:p.Gly596Val"
        },
        "gene": {
          "cg:id": "G014",
          "cg:type": "Gene",
          "symbol": "MEK"
        }
      },
      {
        "cg:id": "Func027",
        "cg:type": "FunctionalData",
        "result": "abnormal/deceased ERK activtation.",
        "dataType": "CG:activation",
        "contextualAllele": {
          "cg:id": "CtxAll024",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll021",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll024",
            "identifier": "http://reg.genome.network/allele/CA093846"
          },
          "alleleName": "NP_004324.2:p.Gly596Val"
        },
        "gene": {
          "cg:id": "G015",
          "cg:type": "Gene",
          "symbol": "ERK"
        }
      }
    ],
    "criterion": {
      "cg:id": "PS3",
      "cg:type": "Criterion",
      "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
      "shortDescription": "well-established damaging effect functional studies",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll020",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll023",
      "identifier": "http://reg.genome.network/allele/CA220161"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      },
      {
        "cg:id": "Agent004",
        "cg:type": "Agent",
        "description": "NWG",
        "name": "Noonan/Rasopathy WG (ClinGen)"
      }
    ]
  },
  "explanation": "The RASopathy group considers MEK and ERK activation assays as well validated."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess012">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess013</td>
          <td>statistically significant difference between affects and unaffecteds</td>
          <td><div class="example_view" id="CritAssess013"><pre><code>{
  "cg:id": "CritAssess013",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS4",
    "cg:type": "Criterion",
    "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
    "shortDescription": "Prevelance increase in affected v control",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll022",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll025",
    "identifier": "http://reg.genome.network/allele/CA253316"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit013",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-05T10:24:00+00:00",
    "endedAtTime": "2016-06-05T10:24:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CaseCtrl030",
        "cg:type": "CaseControl",
        "explanation": "Treating ExAC data as controls for",
        "canonicalAllele": {
          "cg:id": "CanAll022",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll025",
          "identifier": "http://reg.genome.network/allele/CA253316"
        },
        "condition": {
          "cg:id": "Cond010",
          "cg:type": "Condition",
          "name": "Hearing Loss",
          "code": "87884"
        },
        "oddsRatio": 9.1361,
        "confidenceLevel": 99.0,
        "confidenceIntervalLower": 6.1425,
        "confidenceIntervalUpper": 13.5889,
        "caseGroupFrequency": {
          "cg:id": "AllFreq029",
          "cg:type": "AlleleFrequency",
          "ascertainment": "Across all published data (and LMM data) with hearing loss.",
          "allele": {
            "cg:id": "CanAll022",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll025",
            "identifier": "http://reg.genome.network/allele/CA253316"
          },
          "alleleCount": 32,
          "alleleNumber": 4076
        },
        "controlGroupFrequency": {
          "cg:id": "AllFreq028",
          "cg:type": "AlleleFrequency",
          "ascertainment": "ExAC",
          "allele": {
            "cg:id": "CanAll022",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll025",
            "identifier": "http://reg.genome.network/allele/CA253316"
          },
          "alleleCount": 105,
          "alleleNumber": 121336
        }
      }
    ],
    "criterion": {
      "cg:id": "PS4",
      "cg:type": "Criterion",
      "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
      "shortDescription": "Prevelance increase in affected v control",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll022",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll025",
      "identifier": "http://reg.genome.network/allele/CA253316"
    },
    "condition": [
      {
        "cg:id": "Cond010",
        "cg:type": "Condition",
        "name": "Hearing Loss",
        "code": "87884"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "statistically significant difference between affects and unaffecteds"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess013">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess014</td>
          <td>Across publications, this variant is thought to account for 2%-10% of patients with Malignant hyperthermia. But variant is also only found in 9/66740 Europeans in ExAC (.01%). Based on these differences, groups applied PS4 to this variant classification (without a formal OR analysis).</td>
          <td><div class="example_view" id="CritAssess014"><pre><code>{
  "cg:id": "CritAssess014",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS4",
    "cg:type": "Criterion",
    "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
    "shortDescription": "Prevelance increase in affected v control",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll023",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll026",
    "identifier": "http://reg.genome.network/allele/CA024311"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit014",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-05T14:39:00+00:00",
    "endedAtTime": "2016-06-05T14:39:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CaseCtrl033",
        "cg:type": "CaseControl",
        "explanation": "Did not calculate formal odds ratio",
        "canonicalAllele": {
          "cg:id": "CanAll023",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll026",
          "identifier": "http://reg.genome.network/allele/CA024311"
        },
        "condition": {
          "cg:id": "Cond011",
          "cg:type": "Condition",
          "name": "Malignant Hyperthermia",
          "code": "423"
        },
        "caseGroupFrequency": {
          "cg:id": "AllFreq032",
          "cg:type": "AlleleFrequency",
          "explanation": "This allele is thought to account for between 2% and 10% of patients with Malignant hyperthermia.",
          "allele": {
            "cg:id": "CanAll023",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll026",
            "identifier": "http://reg.genome.network/allele/CA024311"
          }
        },
        "controlGroupFrequency": {
          "cg:id": "AllFreq031",
          "cg:type": "AlleleFrequency",
          "ascertainment": "ExAC",
          "allele": {
            "cg:id": "CanAll023",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll026",
            "identifier": "http://reg.genome.network/allele/CA024311"
          },
          "alleleCount": 9,
          "alleleNumber": 66740
        }
      }
    ],
    "criterion": {
      "cg:id": "PS4",
      "cg:type": "Criterion",
      "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
      "shortDescription": "Prevelance increase in affected v control",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll023",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll026",
      "identifier": "http://reg.genome.network/allele/CA024311"
    },
    "condition": [
      {
        "cg:id": "Cond011",
        "cg:type": "Condition",
        "name": "Malignant Hyperthermia",
        "code": "423"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "Across publications, this variant is thought to account for 2%-10% of patients with Malignant hyperthermia. But variant is also only found in 9/66740 Europeans in ExAC (.01%). Based on these differences, groups applied PS4 to this variant classification (without a formal OR analysis)."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess014">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess015</td>
          <td></td>
          <td><div class="example_view" id="CritAssess015"><pre><code>{
  "cg:id": "CritAssess015",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PS4",
    "cg:type": "Criterion",
    "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
    "shortDescription": "Prevelance increase in affected v control",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll024",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll027",
    "identifier": "http://reg.genome.network/allele/CA123287"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit015",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-05T16:12:00+00:00",
    "endedAtTime": "2016-06-05T16:12:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CaseCtrl036",
        "cg:type": "CaseControl",
        "wasGeneratedBy": {
          "cg:id": "CapData073",
          "cg:type": "CaptureData",
          "startedAtTime": "2016-09-13T00:00:00+00:00",
          "endedAtTime": "2016-09-13T00:00:00+00:00",
          "dataSource": {
            "cg:id": "DataSrc001",
            "cg:type": "DataSource",
            "externalIdentifier": "PMID:23633196",
            "linkedIRI": "http://www.ncbi.nlm.nih.gov/pubmed/23633196"
          },
          "wasAssociatedWith": [
            {
              "cg:id": "Agent001",
              "cg:type": "Agent",
              "description": "SH",
              "name": "Steven Harrison"
            }
          ]
        },
        "explanation": "Odds ratio is calculated by pooling samples from the six GENIUS T2D cohorts before calculation.",
        "canonicalAllele": {
          "cg:id": "CanAll024",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll027",
          "identifier": "http://reg.genome.network/allele/CA123287"
        },
        "condition": {
          "cg:id": "Cond012",
          "cg:type": "Condition",
          "name": "Abnormal glucose homeostasis",
          "code": "0011014"
        },
        "oddsRatio": 1.06,
        "confidenceLevel": 95.0,
        "caseGroupFrequency": {
          "cg:id": "AllFreq034",
          "cg:type": "AlleleFrequency",
          "ascertainment": "Case group is composed of cases from six cohorts from the GENIUS T2D consortium.  Case status is defined per-cohort.",
          "allele": {
            "cg:id": "CanAll024",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll027",
            "identifier": "http://reg.genome.network/allele/CA123287"
          },
          "alleleCount": 1198,
          "alleleNumber": 7344,
          "individualCount": 3672,
          "homozygousAlleleIndividualCount": 113,
          "heterozygousAlleleIndividualCount": 972,
          "population": "EX:nfe"
        },
        "controlGroupFrequency": {
          "cg:id": "AllFreq035",
          "cg:type": "AlleleFrequency",
          "ascertainment": "Controls group is composed of controls from six cohorts from the GENIUS T2D consortium.  Controls status is defined per-cohort.",
          "allele": {
            "cg:id": "CanAll024",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll027",
            "identifier": "http://reg.genome.network/allele/CA123287"
          },
          "alleleCount": 936,
          "alleleNumber": 5870,
          "individualCount": 2935,
          "homozygousAlleleIndividualCount": 77,
          "heterozygousAlleleIndividualCount": 782,
          "population": "EX:nfe"
        },
        "confidenceIntervalLower": 0.96,
        "confidenceIntervalUpper": 1.18
      }
    ],
    "criterion": {
      "cg:id": "PS4",
      "cg:type": "Criterion",
      "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
      "shortDescription": "Prevelance increase in affected v control",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll024",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll027",
      "identifier": "http://reg.genome.network/allele/CA123287"
    },
    "condition": [
      {
        "cg:id": "Cond012",
        "cg:type": "Condition",
        "name": "Abnormal glucose homeostasis",
        "code": "0011014"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess015">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess016</td>
          <td>Variant is located within the kinase domain (where multiple other pathogenic variants are located)</td>
          <td><div class="example_view" id="CritAssess016"><pre><code>{
  "cg:id": "CritAssess016",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM1",
    "cg:type": "Criterion",
    "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
    "shortDescription": "Located in hot spot w/out benign variation",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll025",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll028",
    "identifier": "http://reg.genome.network/allele/CA090928"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit016",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-06T08:12:00+00:00",
    "endedAtTime": "2016-06-06T08:12:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Reg040",
        "cg:type": "RegionTypesAndAlleles",
        "explanation": "This region is the catalytic (kinase) domain.",
        "referenceSequence": {
          "cg:id": "RefSeq002",
          "cg:type": "ReferenceSequence",
          "identifier": "NP_005219.2"
        },
        "start": 704,
        "stop": 1016,
        "isTypes": [
          "CG:functional"
        ],
        "includedAlleles": [
          {
            "cg:id": "CtxAll126",
            "cg:type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "cg:id": "CanAll120",
              "cg:type": "CanonicalAllele",
              "preferredCtxAllele": "CtxAll126",
              "identifier": "http://reg.genome.network/allele/CA126726"
            },
            "alleleName": "NP_005219.2:p.Thr790Met"
          }
        ]
      }
    ],
    "criterion": {
      "cg:id": "PM1",
      "cg:type": "Criterion",
      "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
      "shortDescription": "Located in hot spot w/out benign variation",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll025",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll028",
      "identifier": "http://reg.genome.network/allele/CA090928"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent003",
        "cg:type": "Agent",
        "description": "TS",
        "name": "Tasha Strande"
      }
    ]
  },
  "explanation": "Variant is located within the kinase domain (where multiple other pathogenic variants are located)"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess016">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess017</td>
          <td>Variant is located in exon 15, which the RASopathy group has decided is an established functional domain for BRAF</td>
          <td><div class="example_view" id="CritAssess017"><pre><code>{
  "cg:id": "CritAssess017",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM1",
    "cg:type": "Criterion",
    "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
    "shortDescription": "Located in hot spot w/out benign variation",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll026",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll029",
    "identifier": "http://reg.genome.network/allele/CA123651"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit017",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-06T10:53:00+00:00",
    "endedAtTime": "2016-06-06T10:53:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Reg043",
        "cg:type": "RegionTypesAndAlleles",
        "explanation": "This region is exon 15 in the transcript, and codes the serine-threonine-protein kinase catalytic domain.",
        "referenceSequence": {
          "cg:id": "RefSeq003",
          "cg:type": "ReferenceSequence",
          "identifier": "NM_004333.4"
        },
        "start": 1803,
        "stop": 1921,
        "isTypes": [
          "CG:functional"
        ],
        "includedAlleles": [
          {
            "cg:id": "CtxAll029",
            "cg:type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "cg:id": "CanAll026",
              "cg:type": "CanonicalAllele",
              "preferredCtxAllele": "CtxAll029",
              "identifier": "http://reg.genome.network/allele/CA123651"
            },
            "alleleName": "NM_004333.4(BRAF):c.1789C>G"
          }
        ]
      }
    ],
    "criterion": {
      "cg:id": "PM1",
      "cg:type": "Criterion",
      "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
      "shortDescription": "Located in hot spot w/out benign variation",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll026",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll029",
      "identifier": "http://reg.genome.network/allele/CA123651"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      },
      {
        "cg:id": "Agent004",
        "cg:type": "Agent",
        "description": "NWG",
        "name": "Noonan/Rasopathy WG (ClinGen)"
      }
    ]
  },
  "explanation": "Variant is located in exon 15, which the RASopathy group has decided is an established functional domain for BRAF"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess017">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess018</td>
          <td>The variant is near 4 other pathogenic variants</td>
          <td><div class="example_view" id="CritAssess018"><pre><code>{
  "cg:id": "CritAssess018",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM1",
    "cg:type": "Criterion",
    "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
    "shortDescription": "Located in hot spot w/out benign variation",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll027",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll030",
    "identifier": "http://reg.genome.network/allele/CA352275"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit018",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-03T07:24:00+00:00",
    "endedAtTime": "2016-06-03T07:24:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Reg046",
        "cg:type": "RegionTypesAndAlleles",
        "explanation": "Region(NM_0001692[996..1012]) is a mutational hotspot and contains the following pathogenic variants: c.886A>G, c.890C>T, c.899T>C, & c.902G>A.",
        "referenceSequence": {
          "cg:id": "RefSeq004",
          "cg:type": "ReferenceSequence",
          "identifier": "NM_000169.2"
        },
        "start": 996,
        "stop": 1012,
        "isTypes": [
          "CG:hotspot"
        ],
        "includedAlleles": [
          {
            "cg:id": "CtxAll031",
            "cg:type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "cg:id": "CanAll028",
              "cg:type": "CanonicalAllele",
              "preferredCtxAllele": "CtxAll031",
              "identifier": "http://reg.genome.network/allele/CA022141"
            },
            "alleleName": "NM_000169.2(GLA): c.886A>G"
          },
          {
            "cg:id": "CtxAll032",
            "cg:type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "cg:id": "CanAll029",
              "cg:type": "CanonicalAllele",
              "preferredCtxAllele": "CtxAll032",
              "identifier": "http://reg.genome.network/allele/CA022153"
            },
            "alleleName": "NM_000169.2(GLA):c.890C>T"
          },
          {
            "cg:id": "CtxAll033",
            "cg:type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "cg:id": "CanAll030",
              "cg:type": "CanonicalAllele",
              "preferredCtxAllele": "CtxAll033",
              "identifier": "http://reg.genome.network/allele/CA022159"
            },
            "alleleName": "NM_000169.2(GLA):c.899T>C"
          },
          {
            "cg:id": "CtxAll034",
            "cg:type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "cg:id": "CanAll031",
              "cg:type": "CanonicalAllele",
              "preferredCtxAllele": "CtxAll034",
              "identifier": "http://reg.genome.network/allele/CA022189"
            },
            "alleleName": "NM_000169.2(GLA):c.902G>A"
          }
        ]
      },
      {
        "cg:id": "BMVR102",
        "cg:type": "BenignMissenseVariationRate",
        "explanation": "ClinVar shows no benign missense variants in (or near) this region.  The closest is NM_000169.2:c.8T>C",
        "referenceSequence": {
          "cg:id": "RefSeq004",
          "cg:type": "ReferenceSequence",
          "identifier": "NM_000169.2"
        },
        "start": 996,
        "stop": 1012,
        "value": "CG:low"
      },
      {
        "cg:id": "MenDisVarInterp047",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll028",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll031",
          "identifier": "http://reg.genome.network/allele/CA022141"
        },
        "condition": [
          {
            "cg:id": "Cond013",
            "cg:type": "Condition",
            "name": "Fabry Disease",
            "code": "324"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      },
      {
        "cg:id": "MenDisVarInterp048",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll029",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll032",
          "identifier": "http://reg.genome.network/allele/CA022153"
        },
        "condition": [
          {
            "cg:id": "Cond013",
            "cg:type": "Condition",
            "name": "Fabry Disease",
            "code": "324"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      },
      {
        "cg:id": "MenDisVarInterp049",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll030",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll033",
          "identifier": "http://reg.genome.network/allele/CA022159"
        },
        "condition": [
          {
            "cg:id": "Cond013",
            "cg:type": "Condition",
            "name": "Fabry Disease",
            "code": "324"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      },
      {
        "cg:id": "MenDisVarInterp050",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll031",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll034",
          "identifier": "http://reg.genome.network/allele/CA022189"
        },
        "condition": [
          {
            "cg:id": "Cond013",
            "cg:type": "Condition",
            "name": "Fabry Disease",
            "code": "324"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "PM1",
      "cg:type": "Criterion",
      "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
      "shortDescription": "Located in hot spot w/out benign variation",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll027",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll030",
      "identifier": "http://reg.genome.network/allele/CA352275"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "The variant is near 4 other pathogenic variants"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess018">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess019</td>
          <td>PM2 is applicable as even though variant is not 100% absent, it is practically absent (a 95% CI for the population goes from 0.00% to 0.01%)</td>
          <td><div class="example_view" id="CritAssess019"><pre><code>{
  "cg:id": "CritAssess019",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM2",
    "cg:type": "Criterion",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "shortDescription": "Absent from controls",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll032",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll035",
    "identifier": "http://reg.genome.network/allele/CA011797"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit019",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-04T09:08:00+00:00",
    "endedAtTime": "2016-06-04T09:08:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AllFreq051",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "EX:nfe",
        "allele": {
          "cg:id": "CanAll032",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll035",
          "identifier": "http://reg.genome.network/allele/CA011797"
        },
        "alleleCount": 66706,
        "alleleNumber": 1,
        "alleleFrequency": 1.499e-05,
        "homozygousAlleleIndividualCount": 0
      },
      {
        "cg:id": "AllFreq052",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "CG:combined",
        "allele": {
          "cg:id": "CanAll032",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll035",
          "identifier": "http://reg.genome.network/allele/CA011797"
        },
        "alleleCount": 121366,
        "alleleNumber": 1,
        "alleleFrequency": 8.24e-06,
        "homozygousAlleleIndividualCount": 0
      }
    ],
    "criterion": {
      "cg:id": "PM2",
      "cg:type": "Criterion",
      "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
      "shortDescription": "Absent from controls",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll032",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll035",
      "identifier": "http://reg.genome.network/allele/CA011797"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "PM2 is applicable as even though variant is not 100% absent, it is practically absent (a 95% CI for the population goes from 0.00% to 0.01%)"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess019">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess020</td>
          <td></td>
          <td><div class="example_view" id="CritAssess020"><pre><code>{
  "cg:id": "CritAssess020",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM2",
    "cg:type": "Criterion",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "shortDescription": "Absent from controls",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll033",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll037",
    "identifier": "http://reg.genome.network/allele/CA011797"
  },
  "outcome": "CG:R",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit020",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-05T15:13:00+00:00",
    "endedAtTime": "2016-06-05T15:13:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AllFreq053",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "EX:fin",
        "allele": {
          "cg:id": "CanAll033",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll037",
          "identifier": "http://reg.genome.network/allele/CA011797"
        },
        "alleleCount": 6614,
        "alleleNumber": 514,
        "alleleFrequency": 0.07771,
        "homozygousAlleleIndividualCount": 21,
        "medianCoverage": 100.0
      }
    ],
    "criterion": {
      "cg:id": "PM2",
      "cg:type": "Criterion",
      "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
      "shortDescription": "Absent from controls",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll033",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll037",
      "identifier": "http://reg.genome.network/allele/CA011797"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess020">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess021</td>
          <td>while the variant is absent from exac, mean coverage (11X) is insufficient to be sure that variant was fully assayed.</td>
          <td><div class="example_view" id="CritAssess021"><pre><code>{
  "cg:id": "CritAssess021",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM2",
    "cg:type": "Criterion",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "shortDescription": "Absent from controls",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll034",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll038",
    "identifier": "http://reg.genome.network/allele/CA273141"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit021",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-06T11:28:00+00:00",
    "endedAtTime": "2016-06-06T11:28:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AllFreq054",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "CG:combined",
        "allele": {
          "cg:id": "CanAll034",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll038",
          "identifier": "http://reg.genome.network/allele/CA273141"
        },
        "alleleCount": 0,
        "alleleNumber": 0,
        "alleleFrequency": 0.0,
        "medianCoverage": 11.0
      }
    ],
    "criterion": {
      "cg:id": "PM2",
      "cg:type": "Criterion",
      "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
      "shortDescription": "Absent from controls",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll034",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll038",
      "identifier": "http://reg.genome.network/allele/CA273141"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "while the variant is absent from exac, mean coverage (11X) is insufficient to be sure that variant was fully assayed."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess021">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess022</td>
          <td></td>
          <td><div class="example_view" id="CritAssess022"><pre><code>{
  "cg:id": "CritAssess022",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM3",
    "cg:type": "Criterion",
    "description": "For recessive disorders, detected in trans with a pathogenic variant",
    "shortDescription": "Recessive in trans w/ Path",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll035",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll039",
    "identifier": "http://reg.genome.network/allele/CA090919"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit022",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-02T08:00:59+00:00",
    "endedAtTime": "2016-06-02T08:00:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondInh055",
        "cg:type": "ConditionInheritance",
        "wasGeneratedBy": {
          "cg:id": "CapData076",
          "cg:type": "CaptureData",
          "startedAtTime": "2016-05-31T08:45:59+00:00",
          "endedAtTime": "2016-05-31T08:45:59+00:00",
          "dataSource": {
            "cg:id": "DataSrc004",
            "cg:type": "DataSource",
            "externalIdentifier": "OMIM:244400",
            "linkedIRI": "http://omim.org/entry/244400",
            "version": "alopez : 09/15/2016"
          },
          "wasAssociatedWith": [
            {
              "cg:id": "Agent008",
              "cg:type": "Agent",
              "description": "MDS",
              "name": "Marina DiStefano"
            }
          ]
        },
        "condition": {
          "cg:id": "Cond004",
          "cg:type": "Condition",
          "name": "Primary Ciliary Dyskenisia",
          "code": "244"
        },
        "modeOfInheritance": "HP:0000007"
      },
      {
        "cg:id": "IndCond056",
        "cg:type": "IndividualCondition",
        "wasGeneratedBy": {
          "cg:id": "CapData077",
          "cg:type": "CaptureData",
          "startedAtTime": "2016-05-31T09:14:59+00:00",
          "endedAtTime": "2016-05-31T09:14:59+00:00",
          "dataSource": {
            "cg:id": "DataSrc005",
            "cg:type": "DataSource",
            "externalIdentifier": "LMM-CASE:CSER-SUBJECT-123"
          },
          "wasAssociatedWith": [
            {
              "cg:id": "Agent001",
              "cg:type": "Agent",
              "description": "SH",
              "name": "Steven Harrison"
            }
          ]
        },
        "individual": {
          "cg:id": "Indiv004",
          "cg:type": "Individual",
          "description": "individual #004"
        },
        "condition": [
          {
            "cg:id": "Cond004",
            "cg:type": "Condition",
            "name": "Primary Ciliary Dyskenisia",
            "code": "244"
          },
          {
            "cg:id": "Cond014",
            "cg:type": "Condition",
            "name": "outer dynein arm (ODA) defects",
            "code": "0200106"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "MenDisVarInterp057",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "wasGeneratedBy": {
          "cg:id": "CapData078",
          "cg:type": "CaptureData",
          "startedAtTime": "2016-05-31T11:00:59+00:00",
          "endedAtTime": "2016-05-31T11:00:59+00:00",
          "dataSource": {
            "cg:id": "DataSrc006",
            "cg:type": "DataSource",
            "externalIdentifier": "CLINVAR:SCV000205842.3",
            "linkedIRI": "https://www.ncbi.nlm.nih.gov/clinvar/SCV000205842.3"
          },
          "wasAssociatedWith": [
            {
              "cg:id": "Agent008",
              "cg:type": "Agent",
              "description": "MDS",
              "name": "Marina DiStefano"
            }
          ]
        },
        "canonicalAllele": {
          "cg:id": "CanAll036",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll040",
          "identifier": "http://reg.genome.network/allele/CA273615"
        },
        "condition": [
          {
            "cg:id": "Cond004",
            "cg:type": "Condition",
            "name": "Primary Ciliary Dyskenisia",
            "code": "244"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      },
      {
        "cg:id": "IndAll058",
        "cg:type": "IndividualAllele",
        "wasGeneratedBy": {
          "cg:id": "CapData077",
          "cg:type": "CaptureData",
          "startedAtTime": "2016-05-31T09:14:59+00:00",
          "endedAtTime": "2016-05-31T09:14:59+00:00",
          "dataSource": {
            "cg:id": "DataSrc005",
            "cg:type": "DataSource",
            "externalIdentifier": "LMM-CASE:CSER-SUBJECT-123"
          },
          "wasAssociatedWith": [
            {
              "cg:id": "Agent001",
              "cg:type": "Agent",
              "description": "SH",
              "name": "Steven Harrison"
            }
          ]
        },
        "explanation": "Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband.",
        "individual": {
          "cg:id": "Indiv004",
          "cg:type": "Individual",
          "description": "individual #004"
        },
        "primaryAllele": {
          "cg:id": "CanAll035",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll039",
          "identifier": "http://reg.genome.network/allele/CA090919"
        },
        "secondaryAllele": {
          "cg:id": "CanAll036",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll040",
          "identifier": "http://reg.genome.network/allele/CA273615"
        },
        "primaryZygosity": "CG:het",
        "secondaryZygosity": "CG:het",
        "phase": "CG:trans"
      }
    ],
    "criterion": {
      "cg:id": "PM3",
      "cg:type": "Criterion",
      "description": "For recessive disorders, detected in trans with a pathogenic variant",
      "shortDescription": "Recessive in trans w/ Path",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll035",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll039",
      "identifier": "http://reg.genome.network/allele/CA090919"
    },
    "condition": [
      {
        "cg:id": "Cond004",
        "cg:type": "Condition",
        "name": "Primary Ciliary Dyskenisia",
        "code": "244"
      },
      {
        "cg:id": "Cond014",
        "cg:type": "Condition",
        "name": "outer dynein arm (ODA) defects",
        "code": "0200106"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess022">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess023</td>
          <td></td>
          <td><div class="example_view" id="CritAssess023"><pre><code>{
  "cg:id": "CritAssess023",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM3",
    "cg:type": "Criterion",
    "description": "For recessive disorders, detected in trans with a pathogenic variant",
    "shortDescription": "Recessive in trans w/ Path",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll037",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll041",
    "identifier": "http://reg.genome.network/allele/CA176145"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit023",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-10T09:01:59+00:00",
    "endedAtTime": "2016-06-10T09:01:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp059",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll038",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll042",
          "identifier": "http://reg.genome.network/allele/CA501223"
        },
        "condition": [
          {
            "cg:id": "Cond010",
            "cg:type": "Condition",
            "name": "Hearing Loss",
            "code": "87884"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      },
      {
        "cg:id": "CondInh060",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond010",
          "cg:type": "Condition",
          "name": "Hearing Loss",
          "code": "87884"
        },
        "modeOfInheritance": "HP:0000007",
        "gene": {
          "cg:id": "G024",
          "cg:type": "Gene",
          "symbol": "GIPC3"
        }
      },
      {
        "cg:id": "IndCond061",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv005",
          "cg:type": "Individual",
          "description": "individual #005"
        },
        "condition": [
          {
            "cg:id": "Cond010",
            "cg:type": "Condition",
            "name": "Hearing Loss",
            "code": "87884"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "IndAll062",
        "cg:type": "IndividualAllele",
        "explanation": "Based on the NGS data, we are able to determine that c.298G>A and c.411+1G>A are in trans as no NGS reads carry both variants.",
        "individual": {
          "cg:id": "Indiv005",
          "cg:type": "Individual",
          "description": "individual #005"
        },
        "primaryAllele": {
          "cg:id": "CanAll037",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll041",
          "identifier": "http://reg.genome.network/allele/CA176145"
        },
        "secondaryAllele": {
          "cg:id": "CanAll038",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll042",
          "identifier": "http://reg.genome.network/allele/CA501223"
        },
        "primaryZygosity": "CG:het",
        "secondaryZygosity": "CG:het",
        "phase": "CG:trans"
      }
    ],
    "criterion": {
      "cg:id": "PM3",
      "cg:type": "Criterion",
      "description": "For recessive disorders, detected in trans with a pathogenic variant",
      "shortDescription": "Recessive in trans w/ Path",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll037",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll041",
      "identifier": "http://reg.genome.network/allele/CA176145"
    },
    "condition": [
      {
        "cg:id": "Cond010",
        "cg:type": "Condition",
        "name": "Hearing Loss",
        "code": "87884"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess023">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess024</td>
          <td>Application of PM3 is incorrect as the variant in trans is know known to be pathogenic (it is the same variant).</td>
          <td><div class="example_view" id="CritAssess024"><pre><code>{
  "cg:id": "CritAssess024",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM3",
    "cg:type": "Criterion",
    "description": "For recessive disorders, detected in trans with a pathogenic variant",
    "shortDescription": "Recessive in trans w/ Path",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll039",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll043",
    "identifier": "http://reg.genome.network/allele/CA090904"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit024",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-10T09:14:59+00:00",
    "endedAtTime": "2016-06-10T09:14:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondInh063",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond015",
          "cg:type": "Condition",
          "name": "neuronal ceroid lipofuscinosis",
          "code": "216"
        },
        "modeOfInheritance": "HP:0000007",
        "gene": {
          "cg:id": "G025",
          "cg:type": "Gene",
          "symbol": "TTP1"
        }
      },
      {
        "cg:id": "IndCond064",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv006",
          "cg:type": "Individual",
          "description": "individual #006"
        },
        "condition": [
          {
            "cg:id": "Cond015",
            "cg:type": "Condition",
            "name": "neuronal ceroid lipofuscinosis",
            "code": "216"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "IndAll065",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv006",
          "cg:type": "Individual",
          "description": "individual #006"
        },
        "primaryAllele": {
          "cg:id": "CanAll039",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll043",
          "identifier": "http://reg.genome.network/allele/CA090904"
        },
        "primaryZygosity": "CG:hom"
      }
    ],
    "criterion": {
      "cg:id": "PM3",
      "cg:type": "Criterion",
      "description": "For recessive disorders, detected in trans with a pathogenic variant",
      "shortDescription": "Recessive in trans w/ Path",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll039",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll043",
      "identifier": "http://reg.genome.network/allele/CA090904"
    },
    "condition": [
      {
        "cg:id": "Cond015",
        "cg:type": "Condition",
        "name": "neuronal ceroid lipofuscinosis",
        "code": "216"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "Application of PM3 is incorrect as the variant in trans is know known to be pathogenic (it is the same variant)."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess024">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess025</td>
          <td>leads to an in-frame deletion of 7 amino acids (not repeat region).</td>
          <td><div class="example_view" id="CritAssess025"><pre><code>{
  "cg:id": "CritAssess025",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM4",
    "cg:type": "Criterion",
    "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
    "shortDescription": "Protein length changes or stop-loss",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll035",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll039",
    "identifier": "http://reg.genome.network/allele/CA090919"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit025",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-12T07:30:00+00:00",
    "endedAtTime": "2016-06-12T07:30:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon066",
        "cg:type": "MolecularConsequence",
        "wasGeneratedBy": {
          "cg:id": "CapData079",
          "cg:type": "CaptureData",
          "startedAtTime": "2016-06-01T14:03:00+00:00",
          "endedAtTime": "2016-06-01T14:03:00+00:00",
          "dataSource": {
            "cg:id": "DataSrc007",
            "cg:type": "DataSource",
            "externalIdentifier": "LMM:GENEINSIGHT-LAB v5.3.2",
            "linkedIRI": "https://geneinsight-lmm-test.partners.org"
          },
          "wasAssociatedWith": [
            {
              "cg:id": "Agent001",
              "cg:type": "Agent",
              "description": "SH",
              "name": "Steven Harrison"
            }
          ]
        },
        "contextualAllele": {
          "cg:id": "CtxAll044",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll040",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll044",
            "identifier": "http://reg.genome.network/allele/CA273136"
          },
          "alleleName": "NP_001360.1:p.Trp2490_Leu2496del"
        },
        "consequence": "SO:0001825"
      },
      {
        "cg:id": "Reg069",
        "cg:type": "RegionTypesAndAlleles",
        "wasGeneratedBy": {
          "cg:id": "CapData080",
          "cg:type": "CaptureData",
          "startedAtTime": "2016-06-01T14:10:00+00:00",
          "endedAtTime": "2016-06-01T14:10:00+00:00",
          "dataSource": {
            "cg:id": "DataSrc008",
            "cg:type": "DataSource",
            "externalIdentifier": "LMM:ALAMUT-PROD v123",
            "linkedIRI": "https://pcpgm.partners.org/lmm"
          },
          "wasAssociatedWith": [
            {
              "cg:id": "Agent008",
              "cg:type": "Agent",
              "description": "MDS",
              "name": "Marina DiStefano"
            }
          ]
        },
        "explanation": "This is not a repreating region.",
        "referenceSequence": {
          "cg:id": "RefSeq005",
          "cg:type": "ReferenceSequence",
          "identifier": "NP_001360.1"
        },
        "start": 2490,
        "stop": 2496,
        "isNotTypes": [
          "CG:repeat"
        ]
      }
    ],
    "criterion": {
      "cg:id": "PM4",
      "cg:type": "Criterion",
      "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
      "shortDescription": "Protein length changes or stop-loss",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll035",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll039",
      "identifier": "http://reg.genome.network/allele/CA090919"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent003",
        "cg:type": "Agent",
        "description": "TS",
        "name": "Tasha Strande"
      }
    ]
  },
  "explanation": "leads to an in-frame deletion of 7 amino acids (not repeat region)."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess025">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess026</td>
          <td>leads to an in-frame deletion of 5 amino acids (not repeat region).</td>
          <td><div class="example_view" id="CritAssess026"><pre><code>{
  "cg:id": "CritAssess026",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM4",
    "cg:type": "Criterion",
    "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
    "shortDescription": "Protein length changes or stop-loss",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll041",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll045",
    "identifier": "http://reg.genome.network/allele/CA090938"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit026",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-12T08:30:59+00:00",
    "endedAtTime": "2016-06-12T08:30:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon070",
        "cg:type": "MolecularConsequence",
        "contextualAllele": {
          "cg:id": "CtxAll046",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll042",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll046",
            "identifier": "http://reg.genome.network/allele/CA322149"
          },
          "alleleName": "NP_056375.2:p.Arg38_Ser43del"
        },
        "consequence": "SO:0001825"
      },
      {
        "cg:id": "Reg073",
        "cg:type": "RegionTypesAndAlleles",
        "explanation": "NP_056375.2[35..43] is not repeating.",
        "referenceSequence": {
          "cg:id": "RefSeq006",
          "cg:type": "ReferenceSequence",
          "identifier": "NP_056375.2"
        },
        "start": 35,
        "stop": 43,
        "isNotTypes": [
          "CG:repeat"
        ]
      }
    ],
    "criterion": {
      "cg:id": "PM4",
      "cg:type": "Criterion",
      "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
      "shortDescription": "Protein length changes or stop-loss",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll041",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll045",
      "identifier": "http://reg.genome.network/allele/CA090938"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent003",
        "cg:type": "Agent",
        "description": "TS",
        "name": "Tasha Strande"
      }
    ]
  },
  "explanation": "leads to an in-frame deletion of 5 amino acids (not repeat region)."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess026">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess027</td>
          <td>This 15bp duplication variant results in an expansion of a 20 residue polyalanine tract to a 25 residue polyalanine tract.</td>
          <td><div class="example_view" id="CritAssess027"><pre><code>{
  "cg:id": "CritAssess027",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM4",
    "cg:type": "Criterion",
    "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
    "shortDescription": "Protein length changes or stop-loss",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll043",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll047",
    "identifier": "http://reg.genome.network/allele/CA501224"
  },
  "outcome": "CG:R",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit027",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-12T09:31:59+00:00",
    "endedAtTime": "2016-06-12T09:31:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon074",
        "cg:type": "MolecularConsequence",
        "contextualAllele": {
          "cg:id": "CtxAll048",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll044",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll048",
            "identifier": "http://reg.genome.network/allele/CA501225"
          },
          "alleleName": "NP_003915.2:p.Ala260_Gly261insAlaAlaAlaAlaAla"
        },
        "consequence": "SO:0001823"
      },
      {
        "cg:id": "Reg077",
        "cg:type": "RegionTypesAndAlleles",
        "explanation": "NP_003915.2[241..260] is repeating.",
        "referenceSequence": {
          "cg:id": "RefSeq007",
          "cg:type": "ReferenceSequence",
          "identifier": "NP_003915.2"
        },
        "start": 241,
        "stop": 260,
        "isTypes": [
          "CG:repeat"
        ]
      }
    ],
    "criterion": {
      "cg:id": "PM4",
      "cg:type": "Criterion",
      "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
      "shortDescription": "Protein length changes or stop-loss",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll043",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll047",
      "identifier": "http://reg.genome.network/allele/CA501224"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "This 15bp duplication variant results in an expansion of a 20 residue polyalanine tract to a 25 residue polyalanine tract."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess027">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess028</td>
          <td>From ClinVar: Doolan et al. (2005) (PMID:15698845) reported mutations at the same residue (R162P) in patients with HCM, supporting the functional importance of this residue in the protein. This rare R162P variant observed multiple times in is Likely Pathogenic in ClinVar with 1 star assertion.  Other variants in the same residue R162W has conflicting evidence of pathogenicity(Path by GeneDx : SCV000209174.2 and VUS by LMM:SCV000203864.2). Considered these two only as they have provided assertion criteria.</td>
          <td><div class="example_view" id="CritAssess028"><pre><code>{
  "cg:id": "CritAssess028",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM5",
    "cg:type": "Criterion",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "shortDescription": "novel missense at same aa residue as known path missense",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll045",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll049",
    "identifier": "http://reg.genome.network/allele/CA021744"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit028",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-12T10:47:59+00:00",
    "endedAtTime": "2016-06-12T10:47:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp078",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "wasGeneratedBy": {
          "cg:id": "CapDataXX"
        },
        "canonicalAllele": {
          "cg:id": "CanAll047",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll051",
          "identifier": "http://reg.genome.network/allele/CA021749"
        },
        "condition": [
          {
            "cg:id": "Cond016",
            "cg:type": "Condition",
            "name": "Primary familial hypertrophic cardiomyopathy",
            "code": "217569"
          }
        ],
        "clinicalSignificance": "LN:LA26332-9"
      },
      {
        "cg:id": "MenDisVarInterp079",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "wasGeneratedBy": {
          "cg:id": "CapDataXX"
        },
        "canonicalAllele": {
          "cg:id": "CanAll049",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll053",
          "identifier": "http://reg.genome.network/allele/CA021738"
        },
        "condition": [
          {
            "cg:id": "Cond016",
            "cg:type": "Condition",
            "name": "Primary familial hypertrophic cardiomyopathy",
            "code": "217569"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      },
      {
        "cg:id": "MenDisVarInterp042",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "wasGeneratedBy": {
          "cg:id": "CapDataXX"
        },
        "explanation": "Uncertain Sig(not specified) NM_000363.4(TNNI3):c.484C>T (p.Arg162Trp)",
        "canonicalAllele": {
          "cg:id": "CanAll049",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll053",
          "identifier": "http://reg.genome.network/allele/CA021738"
        },
        "clinicalSignificance": "LN:LA26333-7"
      }
    ],
    "criterion": {
      "cg:id": "PM5",
      "cg:type": "Criterion",
      "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
      "shortDescription": "novel missense at same aa residue as known path missense",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "condition": [
      {
        "cg:id": "Cond016",
        "cg:type": "Condition",
        "name": "Primary familial hypertrophic cardiomyopathy",
        "code": "217569"
      }
    ],
    "variant": {
      "cg:id": "CanAll045",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll049",
      "identifier": "http://reg.genome.network/allele/CA021744"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      }
    ]
  },
  "explanation": "From ClinVar: Doolan et al. (2005) (PMID:15698845) reported mutations at the same residue (R162P) in patients with HCM, supporting the functional importance of this residue in the protein. This rare R162P variant observed multiple times in is Likely Pathogenic in ClinVar with 1 star assertion.  Other variants in the same residue R162W has conflicting evidence of pathogenicity(Path by GeneDx : SCV000209174.2 and VUS by LMM:SCV000203864.2). Considered these two only as they have provided assertion criteria."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess028">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess029</td>
          <td>Other pathogenic variants at the same residue per ClinVar : NM_000531.5(OTC):c.119G>A (p.Arg40His) However no assertion criteria is provided for these variants. So pathogenicity needs to be determined by literature search. 

So PM5 is applicable when assessing p.Leu1764Il</td>
          <td><div class="example_view" id="CritAssess029"><pre><code>{
  "cg:id": "CritAssess029",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM5",
    "cg:type": "Criterion",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "shortDescription": "novel missense at same aa residue as known path missense",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll051",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll055",
    "identifier": "http://reg.genome.network/allele/CA090910"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit029",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-06-12T11:03:00+00:00",
    "endedAtTime": "2016-06-12T11:03:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp080",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll053",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll057",
          "identifier": "http://reg.genome.network/allele/CA224454"
        },
        "condition": [
          {
            "cg:id": "Cond018",
            "cg:type": "Condition",
            "name": "Birt-Hogg-Dube Syndrome",
            "code": "122"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "PM5",
      "cg:type": "Criterion",
      "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
      "shortDescription": "novel missense at same aa residue as known path missense",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "condition": [
      {
        "cg:id": "Cond017",
        "cg:type": "Condition",
        "name": "Ornithine carbamoyltransferase deficiency",
        "code": "664"
      }
    ],
    "variant": {
      "cg:id": "CanAll051",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll055",
      "identifier": "http://reg.genome.network/allele/CA090910"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      }
    ]
  },
  "explanation": "Other pathogenic variants at the same residue per ClinVar : NM_000531.5(OTC):c.119G>A (p.Arg40His) However no assertion criteria is provided for these variants. So pathogenicity needs to be determined by literature search. \n\nSo PM5 is applicable when assessing p.Leu1764Il"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess029">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess030</td>
          <td>Another variant at this position, but different AA change, has been observed and called pathogenic for breast ovarian cancer by ENIGMA expert panel:</td>
          <td><div class="example_view" id="CritAssess030"><pre><code>{
  "cg:id": "CritAssess030",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM5",
    "cg:type": "Criterion",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "shortDescription": "novel missense at same aa residue as known path missense",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll055",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll059",
    "identifier": "http://reg.genome.network/allele/CA501227"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit030",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-01T09:23:00+00:00",
    "endedAtTime": "2016-07-01T09:23:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp081",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll057",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll061",
          "identifier": "http://reg.genome.network/allele/CA003449"
        },
        "condition": [
          {
            "cg:id": "Cond001",
            "cg:type": "Condition",
            "name": "Breast Ovarian Cancer",
            "code": "D061325"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "PM5",
      "cg:type": "Criterion",
      "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
      "shortDescription": "novel missense at same aa residue as known path missense",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll055",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll059",
      "identifier": "http://reg.genome.network/allele/CA501227"
    },
    "condition": [
      {
        "cg:id": "Cond001",
        "cg:type": "Condition",
        "name": "Breast Ovarian Cancer",
        "code": "D061325"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "Another variant at this position, but different AA change, has been observed and called pathogenic for breast ovarian cancer by ENIGMA expert panel:"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess030">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess031</td>
          <td>while the variant is absent from exac, however
1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”
2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC).</td>
          <td><div class="example_view" id="CritAssess031"><pre><code>{
  "cg:id": "CritAssess031",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM2",
    "cg:type": "Criterion",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "shortDescription": "Absent from controls",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll035",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll039",
    "identifier": "http://reg.genome.network/allele/CA090919"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit031",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-01T10:53:00+00:00",
    "endedAtTime": "2016-07-01T10:53:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AllFreq082",
        "cg:type": "AlleleFrequency",
        "wasGeneratedBy": {
          "cg:id": "CapData081",
          "cg:type": "CaptureData",
          "startedAtTime": "2016-06-01T14:15:00+00:00",
          "endedAtTime": "2016-06-01T14:15:00+00:00",
          "dataSource": {
            "cg:id": "DataSrc009",
            "cg:type": "DataSource",
            "externalIdentifier": "EXAC v0.2",
            "linkedIRI": "ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.2/"
          },
          "wasAssociatedWith": [
            {
              "cg:id": "Agent001",
              "cg:type": "Agent",
              "description": "SH",
              "name": "Steven Harrison"
            }
          ]
        },
        "ascertainment": "ExAC",
        "population": "CG:combined",
        "allele": {
          "cg:id": "CanAll035",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll039",
          "identifier": "http://reg.genome.network/allele/CA090919"
        },
        "alleleCount": 0,
        "alleleFrequency": 0.0,
        "medianCoverage": 11.0
      }
    ],
    "criterion": {
      "cg:id": "PM2",
      "cg:type": "Criterion",
      "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
      "shortDescription": "Absent from controls",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll035",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll039",
      "identifier": "http://reg.genome.network/allele/CA090919"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      },
      {
        "cg:id": "Agent005",
        "cg:type": "Agent",
        "description": "CSER",
        "name": "CSER Bake Off Project"
      }
    ]
  },
  "explanation": "while the variant is absent from exac, however\n1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”\n2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC)."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess031">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess032</td>
          <td>Two sporadic cases of BHD contain this allele</td>
          <td><div class="example_view" id="CritAssess032"><pre><code>{
  "cg:id": "CritAssess032",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM6",
    "cg:type": "Criterion",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "shortDescription": "assumed de novo",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll059",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll063",
    "identifier": "http://reg.genome.network/allele/CA090951"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit034",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-20T07:21:59+00:00",
    "endedAtTime": "2016-07-20T07:21:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndCond083",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Ind007"
        },
        "condition": [
          {
            "cg:id": "Cond018",
            "cg:type": "Condition",
            "name": "Birt-Hogg-Dube Syndrome",
            "code": "122"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "FamHist085",
        "cg:type": "FamilyHistory",
        "proband": {
          "cg:id": "Ind007"
        },
        "condition": [
          {
            "cg:id": "Cond018",
            "cg:type": "Condition",
            "name": "Birt-Hogg-Dube Syndrome",
            "code": "122"
          }
        ],
        "familyHasCondition": true
      },
      {
        "cg:id": "DeNovo087",
        "cg:type": "DeNovoAllele",
        "explanation": "parentage confirmation not mentioned in the source article",
        "canonicalAllele": {
          "cg:id": "CanAll059",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll063",
          "identifier": "http://reg.genome.network/allele/CA090951"
        },
        "individual": {
          "cg:id": "Ind007"
        },
        "maternityConfirmed": true,
        "paternityConfirmed": true
      },
      {
        "cg:id": "IndCond084",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Ind008"
        },
        "condition": [
          {
            "cg:id": "Cond018",
            "cg:type": "Condition",
            "name": "Birt-Hogg-Dube Syndrome",
            "code": "122"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "FamHist086",
        "cg:type": "FamilyHistory",
        "proband": {
          "cg:id": "Ind008"
        },
        "condition": [
          {
            "cg:id": "Cond018",
            "cg:type": "Condition",
            "name": "Birt-Hogg-Dube Syndrome",
            "code": "122"
          }
        ],
        "familyHasCondition": true
      },
      {
        "cg:id": "DeNovo088",
        "cg:type": "DeNovoAllele",
        "explanation": "parentage confirmation not mentioned in the source article",
        "canonicalAllele": {
          "cg:id": "CanAll059",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll063",
          "identifier": "http://reg.genome.network/allele/CA090951"
        },
        "individual": {
          "cg:id": "Ind008"
        },
        "maternityConfirmed": true,
        "paternityConfirmed": true
      }
    ],
    "criterion": {
      "cg:id": "PM6",
      "cg:type": "Criterion",
      "description": "Assumed de novo, but without confirmation of paternity and maternity",
      "shortDescription": "assumed de novo",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll059",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll063",
      "identifier": "http://reg.genome.network/allele/CA090951"
    },
    "condition": [
      {
        "cg:id": "Cond018",
        "cg:type": "Condition",
        "name": "Birt-Hogg-Dube Syndrome",
        "code": "122"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      },
      {
        "cg:id": "Agent005",
        "cg:type": "Agent",
        "description": "CSER",
        "name": "CSER Bake Off Project"
      }
    ]
  },
  "explanation": "Two sporadic cases of BHD contain this allele"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess032">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess033</td>
          <td>affected child with CFC was found to carry c.158T>C. Both parents are unaffected and from Sanger seq neither parent was found to carry c.158T>C variant - so authors conclude variant occurred de novo. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable</td>
          <td><div class="example_view" id="CritAssess033"><pre><code>{
  "cg:id": "CritAssess033",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM6",
    "cg:type": "Criterion",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "shortDescription": "assumed de novo",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll060",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll064",
    "identifier": "http://reg.genome.network/allele/CA279966"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit035",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-20T07:48:59+00:00",
    "endedAtTime": "2016-07-20T07:48:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndCond089",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Ind009"
        },
        "condition": [
          {
            "cg:id": "Cond019",
            "cg:type": "Condition",
            "name": "Cardio-facio-cutaneous (CFC) syndrome",
            "code": "1340"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "FamHist090",
        "cg:type": "FamilyHistory",
        "proband": {
          "cg:id": "Ind009"
        },
        "condition": [
          {
            "cg:id": "Cond019",
            "cg:type": "Condition",
            "name": "Cardio-facio-cutaneous (CFC) syndrome",
            "code": "1340"
          }
        ],
        "familyHasCondition": true
      },
      {
        "cg:id": "DeNovo091",
        "cg:type": "DeNovoAllele",
        "explanation": "parentage confirmation not mentioned in the source article",
        "canonicalAllele": {
          "cg:id": "CanAll060",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll064",
          "identifier": "http://reg.genome.network/allele/CA279966"
        },
        "individual": {
          "cg:id": "Ind009"
        },
        "maternityConfirmed": true,
        "paternityConfirmed": true
      }
    ],
    "criterion": {
      "cg:id": "PM6",
      "cg:type": "Criterion",
      "description": "Assumed de novo, but without confirmation of paternity and maternity",
      "shortDescription": "assumed de novo",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll060",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll064",
      "identifier": "http://reg.genome.network/allele/CA279966"
    },
    "condition": [
      {
        "cg:id": "Cond019",
        "cg:type": "Condition",
        "name": "Cardio-facio-cutaneous (CFC) syndrome",
        "code": "1340"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "affected child with CFC was found to carry c.158T>C. Both parents are unaffected and from Sanger seq neither parent was found to carry c.158T>C variant - so authors conclude variant occurred de novo. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess033">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess034</td>
          <td>3 unrelated probands with CFC were found to carry c.1741A>G variant. There is no family history of CFC in any family and all three sets of parents were confirmed to not have the variant. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable. However, since there are 3 de novo observations, many groups (including Noonan and MYH7) would allow PM6 to be upgraded to Strong (PM6_S)</td>
          <td><div class="example_view" id="CritAssess034"><pre><code>{
  "cg:id": "CritAssess034",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM6",
    "cg:type": "Criterion",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "shortDescription": "assumed de novo",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll061",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll065",
    "identifier": "http://reg.genome.network/allele/CA279976"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit036",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-20T08:01:59+00:00",
    "endedAtTime": "2016-07-20T08:01:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndCond092",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Ind010"
        },
        "condition": [
          {
            "cg:id": "Cond019",
            "cg:type": "Condition",
            "name": "Cardio-facio-cutaneous (CFC) syndrome",
            "code": "1340"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "FamHist093",
        "cg:type": "FamilyHistory",
        "proband": {
          "cg:id": "Ind010"
        },
        "condition": [
          {
            "cg:id": "Cond019",
            "cg:type": "Condition",
            "name": "Cardio-facio-cutaneous (CFC) syndrome",
            "code": "1340"
          }
        ],
        "familyHasCondition": true
      },
      {
        "cg:id": "DeNovo094",
        "cg:type": "DeNovoAllele",
        "canonicalAllele": {
          "cg:id": "CanAll061",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll065",
          "identifier": "http://reg.genome.network/allele/CA279976"
        },
        "individual": {
          "cg:id": "Ind010"
        },
        "maternityConfirmed": true,
        "paternityConfirmed": true
      },
      {
        "cg:id": "IndCond095",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Ind011"
        },
        "condition": [
          {
            "cg:id": "Cond020",
            "cg:type": "Condition",
            "name": "Hypertrophic Cardiomyopathy",
            "code": "217569"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "FamHist096",
        "cg:type": "FamilyHistory",
        "proband": {
          "cg:id": "Ind011"
        },
        "condition": [
          {
            "cg:id": "Cond019",
            "cg:type": "Condition",
            "name": "Cardio-facio-cutaneous (CFC) syndrome",
            "code": "1340"
          }
        ],
        "familyHasCondition": true
      },
      {
        "cg:id": "DeNovo097",
        "cg:type": "DeNovoAllele",
        "canonicalAllele": {
          "cg:id": "CanAll061",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll065",
          "identifier": "http://reg.genome.network/allele/CA279976"
        },
        "individual": {
          "cg:id": "Ind011"
        },
        "maternityConfirmed": true,
        "paternityConfirmed": true
      },
      {
        "cg:id": "IndCond098",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Ind012"
        },
        "condition": [
          {
            "cg:id": "Cond019",
            "cg:type": "Condition",
            "name": "Cardio-facio-cutaneous (CFC) syndrome",
            "code": "1340"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "FamHist099",
        "cg:type": "FamilyHistory",
        "proband": {
          "cg:id": "Ind012"
        },
        "condition": [
          {
            "cg:id": "Cond019",
            "cg:type": "Condition",
            "name": "Cardio-facio-cutaneous (CFC) syndrome",
            "code": "1340"
          }
        ],
        "familyHasCondition": true
      },
      {
        "cg:id": "DeNovo100",
        "cg:type": "DeNovoAllele",
        "canonicalAllele": {
          "cg:id": "CanAll061",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll065",
          "identifier": "http://reg.genome.network/allele/CA279976"
        },
        "individual": {
          "cg:id": "Ind012"
        },
        "maternityConfirmed": true,
        "paternityConfirmed": true
      }
    ],
    "criterion": {
      "cg:id": "PM6",
      "cg:type": "Criterion",
      "description": "Assumed de novo, but without confirmation of paternity and maternity",
      "shortDescription": "assumed de novo",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll061",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll065",
      "identifier": "http://reg.genome.network/allele/CA279976"
    },
    "condition": [
      {
        "cg:id": "Cond019",
        "cg:type": "Condition",
        "name": "Cardio-facio-cutaneous (CFC) syndrome",
        "code": "1340"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "3 unrelated probands with CFC were found to carry c.1741A>G variant. There is no family history of CFC in any family and all three sets of parents were confirmed to not have the variant. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable. However, since there are 3 de novo observations, many groups (including Noonan and MYH7) would allow PM6 to be upgraded to Strong (PM6_S)"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess034">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess035</td>
          <td>In total, 4 segs from 2 families</td>
          <td><div class="example_view" id="CritAssess035"><pre><code>{
  "cg:id": "CritAssess035",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP1",
    "cg:type": "Criterion",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "shortDescription": "segregates with disease in affected family members",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll063",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll067",
    "identifier": "http://reg.genome.network/allele/CA010136"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit037",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-21T06:45:00+00:00",
    "endedAtTime": "2016-07-21T06:45:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondInh106",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond020",
          "cg:type": "Condition",
          "name": "Hypertrophic Cardiomyopathy",
          "code": "217569"
        },
        "modeOfInheritance": "HP:0000006"
      },
      {
        "cg:id": "AggSegData105",
        "cg:type": "AggregateSegregationData",
        "canonicalAllele": {
          "cg:id": "Cond020",
          "cg:type": "Condition",
          "name": "Hypertrophic Cardiomyopathy",
          "code": "217569"
        },
        "condition": {
          "cg:id": "CanAll063",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll067",
          "identifier": "http://reg.genome.network/allele/CA010136"
        },
        "totalNumberOfSegregations": 0,
        "familySegregationData": [
          {
            "cg:id": 4.0
          }
        ]
      }
    ],
    "criterion": {
      "cg:id": "PP1",
      "cg:type": "Criterion",
      "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
      "shortDescription": "segregates with disease in affected family members",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll063",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll067",
      "identifier": "http://reg.genome.network/allele/CA010136"
    },
    "condition": [
      {
        "cg:id": "Cond020",
        "cg:type": "Condition",
        "name": "Hypertrophic Cardiomyopathy",
        "code": "217569"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "In total, 4 segs from 2 families"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess035">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess036</td>
          <td>In total, 8 segs from 2 families (can combine hom segs and compound het segs). Most groups would allow this to be shifted up the Moderate (PP1_M)</td>
          <td><div class="example_view" id="CritAssess036"><pre><code>{
  "cg:id": "CritAssess036",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP1",
    "cg:type": "Criterion",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "shortDescription": "segregates with disease in affected family members",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll062",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll066",
    "identifier": "http://reg.genome.network/allele/CA143213"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit038",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-21T07:02:00+00:00",
    "endedAtTime": "2016-07-21T07:02:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondInh110",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond021",
          "cg:type": "Condition",
          "name": "Usher syndrome",
          "code": "886"
        },
        "modeOfInheritance": "HP:0000007"
      },
      {
        "cg:id": "AggSegData109",
        "cg:type": "AggregateSegregationData",
        "canonicalAllele": {
          "cg:id": "Cond021",
          "cg:type": "Condition",
          "name": "Usher syndrome",
          "code": "886"
        },
        "condition": {
          "cg:id": "CanAll062",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll066",
          "identifier": "http://reg.genome.network/allele/CA143213"
        },
        "totalNumberOfSegregations": 0,
        "familySegregationData": [
          {
            "cg:id": 8.0
          }
        ]
      }
    ],
    "criterion": {
      "cg:id": "PP1",
      "cg:type": "Criterion",
      "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
      "shortDescription": "segregates with disease in affected family members",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll062",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll066",
      "identifier": "http://reg.genome.network/allele/CA143213"
    },
    "condition": [
      {
        "cg:id": "Cond021",
        "cg:type": "Condition",
        "name": "Usher syndrome",
        "code": "886"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "In total, 8 segs from 2 families (can combine hom segs and compound het segs). Most groups would allow this to be shifted up the Moderate (PP1_M)"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess036">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess037</td>
          <td>PP1 is for “Co-segregation with disease in multiple affected family members” PP1 is not applicable for this variant because it’s only a single segregation.</td>
          <td><div class="example_view" id="CritAssess037"><pre><code>{
  "cg:id": "CritAssess037",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP1",
    "cg:type": "Criterion",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "shortDescription": "segregates with disease in affected family members",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll064",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll068",
    "identifier": "http://reg.genome.network/allele/CA501226"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit039",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-21T07:38:59+00:00",
    "endedAtTime": "2016-07-21T07:38:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "FamSegData111",
        "cg:type": "FamilySegregationData",
        "condition": {
          "cg:id": "Cond001",
          "cg:type": "Condition",
          "name": "Breast Ovarian Cancer",
          "code": "D061325"
        },
        "canonicalAllele": {
          "cg:id": "CanAll064",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll068",
          "identifier": "http://reg.genome.network/allele/CA501226"
        },
        "family": {
          "cg:id": "Fam005",
          "cg:type": "Family",
          "description": "Family #005"
        },
        "phenotypePositiveAllelePositive": 2
      },
      {
        "cg:id": "CondInh112",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond001",
          "cg:type": "Condition",
          "name": "Breast Ovarian Cancer",
          "code": "D061325"
        },
        "modeOfInheritance": "HP:0000006"
      }
    ],
    "criterion": {
      "cg:id": "PP1",
      "cg:type": "Criterion",
      "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
      "shortDescription": "segregates with disease in affected family members",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll064",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll068",
      "identifier": "http://reg.genome.network/allele/CA501226"
    },
    "condition": [
      {
        "cg:id": "Cond001",
        "cg:type": "Condition",
        "name": "Breast Ovarian Cancer",
        "code": "D061325"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "PP1 is for “Co-segregation with disease in multiple affected family members” PP1 is not applicable for this variant because it’s only a single segregation."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess037">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess038</td>
          <td></td>
          <td><div class="example_view" id="CritAssess038"><pre><code>{
  "cg:id": "CritAssess038",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP1",
    "cg:type": "Criterion",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "shortDescription": "segregates with disease in affected family members",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll065",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll069",
    "identifier": "http://reg.genome.network/allele/CA006429"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit040",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-21T08:10:59+00:00",
    "endedAtTime": "2016-07-21T08:10:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AggSegData115",
        "cg:type": "AggregateSegregationData",
        "canonicalAllele": {
          "cg:id": "Cond002",
          "cg:type": "Condition",
          "name": "Dilated Cardiomyopathy",
          "code": "217604"
        },
        "condition": {
          "cg:id": "CanAll065",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll069",
          "identifier": "http://reg.genome.network/allele/CA006429"
        },
        "totalNumberOfSegregations": 0,
        "familySegregationData": [
          {
            "cg:id": 9.0
          }
        ]
      },
      {
        "cg:id": "CondInh116",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond002",
          "cg:type": "Condition",
          "name": "Dilated Cardiomyopathy",
          "code": "217604"
        },
        "modeOfInheritance": "HP:0000006"
      }
    ],
    "criterion": {
      "cg:id": "PP1",
      "cg:type": "Criterion",
      "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
      "shortDescription": "segregates with disease in affected family members",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll065",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll069",
      "identifier": "http://reg.genome.network/allele/CA006429"
    },
    "condition": [
      {
        "cg:id": "Cond002",
        "cg:type": "Condition",
        "name": "Dilated Cardiomyopathy",
        "code": "217604"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent002",
        "cg:type": "Agent",
        "description": "CB",
        "name": "Chris Bizon"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess038">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess039</td>
          <td></td>
          <td><div class="example_view" id="CritAssess039"><pre><code>{
  "cg:id": "CritAssess039",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP2",
    "cg:type": "Criterion",
    "description": "Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease",
    "shortDescription": "missense with pathogenic missense common",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll066",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll070",
    "identifier": "http://reg.genome.network/allele/CA016087"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit041",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-21T09:40:59+00:00",
    "endedAtTime": "2016-07-21T09:40:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon117",
        "cg:type": "MolecularConsequence",
        "contextualAllele": {
          "cg:id": "CtxAll070",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll066",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll070",
            "identifier": "http://reg.genome.network/allele/CA016087"
          },
          "alleleName": "NM_000257.3(MYH7):c.5401G>A"
        },
        "consequence": "SO:0001583",
        "lof": true
      },
      {
        "cg:id": "CondMech118",
        "cg:type": "ConditionMechanism",
        "explanation": "Of 178 pathogenic  alleles in ClinVar, 149 are missense.",
        "mechanism": "SO:0001583",
        "strength": "CG:common",
        "gene": {
          "cg:id": "G008",
          "cg:type": "Gene",
          "symbol": "MYH7"
        }
      },
      {
        "cg:id": "BMVR119",
        "cg:type": "BenignMissenseVariationRate",
        "explanation": "Based on the gene size, ExAC expects 740 missense variants to be observed, but actually observes only 420, giving a z score of 6.5, which is in the top 0.02% of genes.",
        "gene": {
          "cg:id": "G008",
          "cg:type": "Gene",
          "symbol": "MYH7"
        },
        "value": "CG:low"
      }
    ],
    "criterion": {
      "cg:id": "PP2",
      "cg:type": "Criterion",
      "description": "Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease",
      "shortDescription": "missense with pathogenic missense common",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll066",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll070",
      "identifier": "http://reg.genome.network/allele/CA016087"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess039">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess040</td>
          <td></td>
          <td><div class="example_view" id="CritAssess040"><pre><code>{
  "cg:id": "CritAssess040",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP3",
    "cg:type": "Criterion",
    "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "shortDescription": "computational evidence supports deleterious\neffect",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll067",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll071",
    "identifier": "http://reg.genome.network/allele/CA090928"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit042",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-21T10:28:59+00:00",
    "endedAtTime": "2016-07-21T10:28:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "InSilicoPred120",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll071",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll067",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll071",
            "identifier": "http://reg.genome.network/allele/CA090928"
          },
          "alleleName": "NM_005228.3(EGFR):c.2369C>T"
        },
        "type": "CG:me",
        "tool": "SIFT",
        "value": "Damaging"
      },
      {
        "cg:id": "InSilicoPred121",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll071",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll067",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll071",
            "identifier": "http://reg.genome.network/allele/CA090928"
          },
          "alleleName": "NM_005228.3(EGFR):c.2369C>T"
        },
        "type": "CG:me",
        "tool": "Polyphen",
        "value": "Probably damaging"
      },
      {
        "cg:id": "InSilicoPred122",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll071",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll067",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll071",
            "identifier": "http://reg.genome.network/allele/CA090928"
          },
          "alleleName": "NM_005228.3(EGFR):c.2369C>T"
        },
        "type": "CG:me",
        "tool": "FATHMM",
        "value": "Tolerated"
      },
      {
        "cg:id": "InSilicoPred123",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll071",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll067",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll071",
            "identifier": "http://reg.genome.network/allele/CA090928"
          },
          "alleleName": "NM_005228.3(EGFR):c.2369C>T"
        },
        "type": "CG:me",
        "tool": "LRT",
        "value": "Damaging"
      },
      {
        "cg:id": "InSilicoPred124",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll071",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll067",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll071",
            "identifier": "http://reg.genome.network/allele/CA090928"
          },
          "alleleName": "NM_005228.3(EGFR):c.2369C>T"
        },
        "type": "CG:me",
        "tool": "MetaSVM",
        "value": "Tolerated"
      },
      {
        "cg:id": "InSilicoPred125",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll071",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll067",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll071",
            "identifier": "http://reg.genome.network/allele/CA090928"
          },
          "alleleName": "NM_005228.3(EGFR):c.2369C>T"
        },
        "type": "CG:me",
        "tool": "Mutation assessor",
        "value": "Tolerated"
      },
      {
        "cg:id": "InSilicoPred126",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll071",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll067",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll071",
            "identifier": "http://reg.genome.network/allele/CA090928"
          },
          "alleleName": "NM_005228.3(EGFR):c.2369C>T"
        },
        "type": "CG:me",
        "tool": "Mutation taster",
        "value": "Disease-causing"
      },
      {
        "cg:id": "InSilicoPred127",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll071",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll067",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll071",
            "identifier": "http://reg.genome.network/allele/CA090928"
          },
          "alleleName": "NM_005228.3(EGFR):c.2369C>T"
        },
        "type": "CG:me",
        "tool": "Provean",
        "value": "Damaging"
      },
      {
        "cg:id": "InSilicoPred128",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll071",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll067",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll071",
            "identifier": "http://reg.genome.network/allele/CA090928"
          },
          "alleleName": "NM_005228.3(EGFR):c.2369C>T"
        },
        "type": "CG:me",
        "tool": "Condel",
        "value": "deleterious"
      }
    ],
    "criterion": {
      "cg:id": "PP3",
      "cg:type": "Criterion",
      "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
      "shortDescription": "computational evidence supports deleterious\neffect",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll067",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll071",
      "identifier": "http://reg.genome.network/allele/CA090928"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      },
      {
        "cg:id": "Agent005",
        "cg:type": "Agent",
        "description": "CSER",
        "name": "CSER Bake Off Project"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess040">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess041</td>
          <td></td>
          <td><div class="example_view" id="CritAssess041"><pre><code>{
  "cg:id": "CritAssess041",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP3",
    "cg:type": "Criterion",
    "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "shortDescription": "computational evidence supports deleterious\neffect",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll068",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll072",
    "identifier": "http://reg.genome.network/allele/CA090884"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit043",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-21T12:14:59+00:00",
    "endedAtTime": "2016-07-21T12:14:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "InSilicoPred129",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll074",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll068",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll072",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "ENST00000566221.5:c.127C>T"
        },
        "type": "CG:me",
        "tool": "SIFT",
        "value": "deleterious"
      },
      {
        "cg:id": "InSilicoPred130",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll074",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll068",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll072",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "ENST00000566221.5:c.127C>T"
        },
        "type": "CG:me",
        "tool": "Polyphen",
        "value": "Probably damaging"
      },
      {
        "cg:id": "InSilicoPred131",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll074",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll068",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll072",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "ENST00000566221.5:c.127C>T"
        },
        "type": "CG:me",
        "tool": "FATHMM",
        "value": "Damaging"
      },
      {
        "cg:id": "InSilicoPred132",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll074",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll068",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll072",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "ENST00000566221.5:c.127C>T"
        },
        "type": "CG:me",
        "tool": "LRT",
        "value": "Damaging"
      },
      {
        "cg:id": "InSilicoPred133",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll074",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll068",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll072",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "ENST00000566221.5:c.127C>T"
        },
        "type": "CG:me",
        "tool": "MetaSVM",
        "value": "Tolerated"
      },
      {
        "cg:id": "InSilicoPred134",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll074",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll068",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll072",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "ENST00000566221.5:c.127C>T"
        },
        "type": "CG:me",
        "tool": "Mutation assessor",
        "value": "Medium functional impact"
      },
      {
        "cg:id": "InSilicoPred135",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll074",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll068",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll072",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "ENST00000566221.5:c.127C>T"
        },
        "type": "CG:me",
        "tool": "Mutation taster",
        "value": "Disease-causing"
      },
      {
        "cg:id": "InSilicoPred136",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll074",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll068",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll072",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "ENST00000566221.5:c.127C>T"
        },
        "type": "CG:me",
        "tool": "Provean",
        "value": "Damaging"
      },
      {
        "cg:id": "InSilicoPred137",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll074",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll068",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll072",
            "identifier": "http://reg.genome.network/allele/CA090884"
          },
          "alleleName": "ENST00000566221.5:c.127C>T"
        },
        "type": "CG:me",
        "tool": "Condel",
        "value": "deleterious"
      }
    ],
    "criterion": {
      "cg:id": "PP3",
      "cg:type": "Criterion",
      "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
      "shortDescription": "computational evidence supports deleterious\neffect",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll068",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll072",
      "identifier": "http://reg.genome.network/allele/CA090884"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess041">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess042</td>
          <td>even though the computational tools for AA changes are NOT predicting an impact, splicing tools are predicting an impact - so PP3 is applicable</td>
          <td><div class="example_view" id="CritAssess042"><pre><code>{
  "cg:id": "CritAssess042",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP3",
    "cg:type": "Criterion",
    "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "shortDescription": "computational evidence supports deleterious\neffect",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll069",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll075",
    "identifier": "http://reg.genome.network/allele/CA022019"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit044",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-07-21T14:45:59+00:00",
    "endedAtTime": "2016-07-21T14:45:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "InSilicoPred138",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:me",
        "tool": "SIFT",
        "value": "Tolerated"
      },
      {
        "cg:id": "InSilicoPred139",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:me",
        "tool": "Polyphen2",
        "value": "benign"
      },
      {
        "cg:id": "InSilicoPred140",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:me",
        "tool": "AlignGVGD",
        "value": "C0 (benign)"
      },
      {
        "cg:id": "InSilicoPred141",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:me",
        "tool": "MAPP",
        "value": "good (benign)"
      },
      {
        "cg:id": "InSilicoPred142",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:me",
        "tool": "MutationTaster",
        "value": "disease-causing"
      },
      {
        "cg:id": "InSilicoPred143",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:sp",
        "tool": "SSF",
        "value": 71.53
      },
      {
        "cg:id": "InSilicoPred144",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:sp",
        "tool": "MaxEnt",
        "value": 7.39
      },
      {
        "cg:id": "InSilicoPred145",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:sp",
        "tool": "NNSPLICE",
        "value": 0.87
      },
      {
        "cg:id": "InSilicoPred146",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:sp",
        "tool": "GeneSplicer",
        "value": 7.75
      },
      {
        "cg:id": "InSilicoPred147",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll075",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll069",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll075",
            "identifier": "http://reg.genome.network/allele/CA022019"
          },
          "alleleName": "NM_000363.4(TNNI3):c.592C>G"
        },
        "type": "CG:sp",
        "tool": "HSF",
        "value": 81.75
      }
    ],
    "criterion": {
      "cg:id": "PP3",
      "cg:type": "Criterion",
      "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
      "shortDescription": "computational evidence supports deleterious\neffect",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll069",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll075",
      "identifier": "http://reg.genome.network/allele/CA022019"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "even though the computational tools for AA changes are NOT predicting an impact, splicing tools are predicting an impact - so PP3 is applicable"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess042">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess044</td>
          <td></td>
          <td><div class="example_view" id="CritAssess044"><pre><code>{
  "cg:id": "CritAssess044",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP4",
    "cg:type": "Criterion",
    "description": "Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology",
    "shortDescription": "disease specific phenotype and family history",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll070",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll076",
    "identifier": "http://reg.genome.network/allele/CA090915"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit046",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-28T14:00:00+00:00",
    "endedAtTime": "2016-08-28T14:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondEtiol148",
        "cg:type": "ConditionEtiology",
        "explanation": "HHT type 1 is only known to be caused by variants in ENG -",
        "condition": {
          "cg:id": "Cond022",
          "cg:type": "Condition",
          "name": "Hereditary Hemorrhagic Telangiectasia Type 1",
          "code": "774"
        },
        "gene": {
          "cg:id": "G035",
          "cg:type": "Gene",
          "symbol": "ENG"
        },
        "specificity": "complete"
      }
    ],
    "criterion": {
      "cg:id": "PP4",
      "cg:type": "Criterion",
      "description": "Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology",
      "shortDescription": "disease specific phenotype and family history",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll070",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll076",
      "identifier": "http://reg.genome.network/allele/CA090915"
    },
    "condition": [
      {
        "cg:id": "Cond022",
        "cg:type": "Condition",
        "name": "Hereditary Hemorrhagic Telangiectasia Type 1",
        "code": "774"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      },
      {
        "cg:id": "Agent005",
        "cg:type": "Agent",
        "description": "CSER",
        "name": "CSER Bake Off Project"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess044">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess045</td>
          <td></td>
          <td><div class="example_view" id="CritAssess045"><pre><code>{
  "cg:id": "CritAssess045",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP4",
    "cg:type": "Criterion",
    "description": "Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology",
    "shortDescription": "disease specific phenotype and family history",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll071",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll077",
    "identifier": "http://reg.genome.network/allele/CA501234"
  },
  "outcome": "CG:R",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit047",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-29T15:00:00+00:00",
    "endedAtTime": "2016-08-29T15:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondEtiol149",
        "cg:type": "ConditionEtiology",
        "explanation": "1.Only ~50% of HCM is genetic (can be caused by environment) \n2. More than 10 genes can cause HCM (specifically 9 sarcomere genes)",
        "condition": {
          "cg:id": "Cond020",
          "cg:type": "Condition",
          "name": "Hypertrophic Cardiomyopathy",
          "code": "217569"
        },
        "gene": {
          "cg:id": "G008",
          "cg:type": "Gene",
          "symbol": "MYH7"
        },
        "specificity": "low"
      }
    ],
    "criterion": {
      "cg:id": "PP4",
      "cg:type": "Criterion",
      "description": "Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology",
      "shortDescription": "disease specific phenotype and family history",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll071",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll077",
      "identifier": "http://reg.genome.network/allele/CA501234"
    },
    "condition": [
      {
        "cg:id": "Cond020",
        "cg:type": "Condition",
        "name": "Hypertrophic Cardiomyopathy",
        "code": "217569"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess045">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess046</td>
          <td></td>
          <td><div class="example_view" id="CritAssess046"><pre><code>{
  "cg:id": "CritAssess046",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP5",
    "cg:type": "Criterion",
    "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
    "shortDescription": "pathogenic from reputable source without shared data",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll072",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll078",
    "identifier": "http://reg.genome.network/allele/CA019369"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit048",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-29T15:59:59+00:00",
    "endedAtTime": "2016-08-29T15:59:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp150",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "Variant has been classified as “Pathogenic” by Sharing Clinical Reports Project (SCV000054057.3)",
        "canonicalAllele": {
          "cg:id": "CanAll072",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll078",
          "identifier": "http://reg.genome.network/allele/CA019369"
        },
        "condition": [
          {
            "cg:id": "Cond023",
            "cg:type": "Condition",
            "name": "Breast-ovarian cancer, familial 2",
            "code": "612555"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "PP5",
      "cg:type": "Criterion",
      "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
      "shortDescription": "pathogenic from reputable source without shared data",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll072",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll078",
      "identifier": "http://reg.genome.network/allele/CA019369"
    },
    "condition": [
      {
        "cg:id": "Cond023",
        "cg:type": "Condition",
        "name": "Breast-ovarian cancer, familial 2",
        "code": "612555"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess046">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess047</td>
          <td>OMIM is not considered an expert or reputable source for variant interpretations.
Further, OMIM’s evidence for this pathogenic call is available (the publication link - PMID:21620354 - and free text summary) so people should use the evidence from OMIM in their assessment NOT OMIM’s interpretation.  This is marked as Insufficient Evidence rather than refuted because the OMIM entry does not rule out a later application of PP5 based on eg. an expert panel.</td>
          <td><div class="example_view" id="CritAssess047"><pre><code>{
  "cg:id": "CritAssess047",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP5",
    "cg:type": "Criterion",
    "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
    "shortDescription": "pathogenic from reputable source without shared data",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll073",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll079",
    "identifier": "http://reg.genome.network/allele/CA128479"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit049",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-29T17:00:00+00:00",
    "endedAtTime": "2016-08-29T17:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp151",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "Variant has been classified as “Pathogenic” by OMIM (SCV000043718.1)",
        "canonicalAllele": {
          "cg:id": "CanAll073",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll079",
          "identifier": "http://reg.genome.network/allele/CA128479"
        },
        "condition": [
          {
            "cg:id": "Cond024",
            "cg:type": "Condition",
            "name": "Myopathy, distal, 4",
            "code": "63273"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "PP5",
      "cg:type": "Criterion",
      "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
      "shortDescription": "pathogenic from reputable source without shared data",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll073",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll079",
      "identifier": "http://reg.genome.network/allele/CA128479"
    },
    "condition": [
      {
        "cg:id": "Cond024",
        "cg:type": "Condition",
        "name": "Myopathy, distal, 4",
        "code": "63273"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "OMIM is not considered an expert or reputable source for variant interpretations.\nFurther, OMIM’s evidence for this pathogenic call is available (the publication link - PMID:21620354 - and free text summary) so people should use the evidence from OMIM in their assessment NOT OMIM’s interpretation.  This is marked as Insufficient Evidence rather than refuted because the OMIM entry does not rule out a later application of PP5 based on eg. an expert panel."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess047">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess048</td>
          <td></td>
          <td><div class="example_view" id="CritAssess048"><pre><code>{
  "cg:id": "CritAssess048",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PP5",
    "cg:type": "Criterion",
    "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
    "shortDescription": "pathogenic from reputable source without shared data",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll074",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll080",
    "identifier": "http://reg.genome.network/allele/CA276116"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit050",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-29T18:00:00+00:00",
    "endedAtTime": "2016-08-29T18:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp152",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "Variant has been classified as “Pathogenic” by Expert panel CFTR (SCV000245981.1)",
        "canonicalAllele": {
          "cg:id": "CanAll074",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll080",
          "identifier": "http://reg.genome.network/allele/CA276116"
        },
        "condition": [
          {
            "cg:id": "Cond026",
            "cg:type": "Condition",
            "name": "Marfan Syndrome",
            "code": "558"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "PP5",
      "cg:type": "Criterion",
      "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
      "shortDescription": "pathogenic from reputable source without shared data",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll074",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll080",
      "identifier": "http://reg.genome.network/allele/CA276116"
    },
    "condition": [
      {
        "cg:id": "Cond025",
        "cg:type": "Condition",
        "name": "Cystic Fibrosis",
        "code": "586"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess048">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess049</td>
          <td></td>
          <td><div class="example_view" id="CritAssess049"><pre><code>{
  "cg:id": "CritAssess049",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BA1",
    "cg:type": "Criterion",
    "description": "Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "shortDescription": "present in population databases at greater than 5%",
    "defaultOutcome": "CG:A",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll075",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll081",
    "identifier": "http://reg.genome.network/allele/CA090891"
  },
  "outcome": "CG:A",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit051",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-29T18:59:59+00:00",
    "endedAtTime": "2016-08-29T18:59:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AllFreq153",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "CG:combined",
        "allele": {
          "cg:id": "CanAll075",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll081",
          "identifier": "http://reg.genome.network/allele/CA090891"
        },
        "alleleCount": 6732,
        "alleleNumber": 120402,
        "alleleFrequency": 0.05591,
        "homozygousAlleleIndividualCount": 223
      }
    ],
    "criterion": {
      "cg:id": "BA1",
      "cg:type": "Criterion",
      "description": "Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
      "shortDescription": "present in population databases at greater than 5%",
      "defaultOutcome": "CG:A",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll075",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll081",
      "identifier": "http://reg.genome.network/allele/CA090891"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess049">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess050</td>
          <td>Globally, this variant has a MAF of 0.0454 (4.5%) which would put it under the 5% rule and thus BA1 would not be applicable. However, variant is enriched in East Asian population - 29% MAF - so way over the 5% cut-off. So now BA1 is applicable</td>
          <td><div class="example_view" id="CritAssess050"><pre><code>{
  "cg:id": "CritAssess050",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BA1",
    "cg:type": "Criterion",
    "description": "Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "shortDescription": "present in population databases at greater than 5%",
    "defaultOutcome": "CG:A",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll076",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll082",
    "identifier": "http://reg.genome.network/allele/CA134997"
  },
  "outcome": "CG:A",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit052",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-29T20:00:00+00:00",
    "endedAtTime": "2016-08-29T20:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AllFreq154",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "CG:combined",
        "allele": {
          "cg:id": "CanAll076",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll082",
          "identifier": "http://reg.genome.network/allele/CA134997"
        },
        "alleleCount": 5507,
        "alleleNumber": 121350,
        "alleleFrequency": 0.04538,
        "homozygousAlleleIndividualCount": 768
      },
      {
        "cg:id": "AllFreq155",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "EX:eas",
        "allele": {
          "cg:id": "CanAll076",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll082",
          "identifier": "http://reg.genome.network/allele/CA134997"
        },
        "alleleCount": 2518,
        "alleleNumber": 8632,
        "alleleFrequency": 0.2917,
        "homozygousAlleleIndividualCount": 405
      }
    ],
    "criterion": {
      "cg:id": "BA1",
      "cg:type": "Criterion",
      "description": "Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
      "shortDescription": "present in population databases at greater than 5%",
      "defaultOutcome": "CG:A",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll076",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll082",
      "identifier": "http://reg.genome.network/allele/CA134997"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "Globally, this variant has a MAF of 0.0454 (4.5%) which would put it under the 5% rule and thus BA1 would not be applicable. However, variant is enriched in East Asian population - 29% MAF - so way over the 5% cut-off. So now BA1 is applicable"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess050">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess051</td>
          <td>Although this variant has been seen in the general population, its frequency is low enough to be consistent with a recessive carrier frequency.</td>
          <td><div class="example_view" id="CritAssess051"><pre><code>{
  "cg:id": "CritAssess051",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PM2",
    "cg:type": "Criterion",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "shortDescription": "Absent from controls",
    "defaultOutcome": "CG:M",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll077",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll083",
    "identifier": "http://reg.genome.network/allele/CA501229"
  },
  "outcome": "CG:M",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit053",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-29T21:00:00+00:00",
    "endedAtTime": "2016-08-29T21:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondInh156",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond004",
          "cg:type": "Condition",
          "name": "Primary Ciliary Dyskenisia",
          "code": "244"
        },
        "modeOfInheritance": "HP:0000007"
      },
      {
        "cg:id": "AllFreq157",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "CG:combined",
        "allele": {
          "cg:id": "CanAll077",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll083",
          "identifier": "http://reg.genome.network/allele/CA501229"
        },
        "alleleCount": 14,
        "alleleNumber": 121100,
        "alleleFrequency": 0.0001158,
        "homozygousAlleleIndividualCount": 0
      }
    ],
    "criterion": {
      "cg:id": "PM2",
      "cg:type": "Criterion",
      "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
      "shortDescription": "Absent from controls",
      "defaultOutcome": "CG:M",
      "targetInterpretation": "LN:LA6668-3"
    },
    "variant": {
      "cg:id": "CanAll077",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll083",
      "identifier": "http://reg.genome.network/allele/CA501229"
    },
    "condition": [
      {
        "cg:id": "Cond004",
        "cg:type": "Condition",
        "name": "Primary Ciliary Dyskenisia",
        "code": "244"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "Although this variant has been seen in the general population, its frequency is low enough to be consistent with a recessive carrier frequency."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess051">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess052</td>
          <td></td>
          <td><div class="example_view" id="CritAssess052"><pre><code>{
  "cg:id": "CritAssess052",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS1",
    "cg:type": "Criterion",
    "description": "Allele frequency is greater than expected for disorder",
    "shortDescription": "MAF too high for disorder",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll078",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll084",
    "identifier": "http://reg.genome.network/allele/CA017597"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit054",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-29T21:59:59+00:00",
    "endedAtTime": "2016-08-29T21:59:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AllFreq158",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "CG:combined",
        "allele": {
          "cg:id": "CanAll078",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll084",
          "identifier": "http://reg.genome.network/allele/CA017597"
        },
        "alleleCount": 89,
        "alleleNumber": 121410,
        "alleleFrequency": 0.0007,
        "homozygousAlleleIndividualCount": 1
      },
      {
        "cg:id": "CondPrev159",
        "cg:type": "ConditionPrevelance",
        "condition": [
          {
            "cg:id": "Cond028",
            "cg:type": "Condition",
            "name": "Perrault Syndrome",
            "code": "2855"
          }
        ],
        "minimum": 0.0017,
        "maximum": 0.0007,
        "population": "CG:overall"
      },
      {
        "cg:id": "CondPenet160",
        "cg:type": "ConditionPenetrance",
        "condition": [
          {
            "cg:id": "Cond028",
            "cg:type": "Condition",
            "name": "Perrault Syndrome",
            "code": "2855"
          }
        ],
        "penetrance": "CG:high"
      },
      {
        "cg:id": "CondInh161",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond028",
          "cg:type": "Condition",
          "name": "Perrault Syndrome",
          "code": "2855"
        },
        "modeOfInheritance": "HP:0000006"
      }
    ],
    "criterion": {
      "cg:id": "BS1",
      "cg:type": "Criterion",
      "description": "Allele frequency is greater than expected for disorder",
      "shortDescription": "MAF too high for disorder",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll078",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll084",
      "identifier": "http://reg.genome.network/allele/CA017597"
    },
    "condition": [
      {
        "cg:id": "Cond026",
        "cg:type": "Condition",
        "name": "Marfan Syndrome",
        "code": "558"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess052">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess053</td>
          <td></td>
          <td><div class="example_view" id="CritAssess053"><pre><code>{
  "cg:id": "CritAssess053",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS1",
    "cg:type": "Criterion",
    "description": "Allele frequency is greater than expected for disorder",
    "shortDescription": "MAF too high for disorder",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll079",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll085",
    "identifier": "http://reg.genome.network/allele/CA090926"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit055",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-29T23:00:00+00:00",
    "endedAtTime": "2016-08-29T23:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AllFreq162",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "CG:combined",
        "allele": {
          "cg:id": "CanAll079",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll085",
          "identifier": "http://reg.genome.network/allele/CA090926"
        },
        "alleleCount": 545,
        "alleleNumber": 87696,
        "alleleFrequency": 0.00625,
        "homozygousAlleleIndividualCount": 1,
        "hemizygousAlleleIndividualCount": 206
      },
      {
        "cg:id": "CondPrev163",
        "cg:type": "ConditionPrevelance",
        "condition": [
          {
            "cg:id": "Cond027",
            "cg:type": "Condition",
            "name": "Mitochondrial complex 1 deficiency",
            "code": "2609"
          }
        ],
        "value": 0.00012,
        "population": "CG:overall"
      },
      {
        "cg:id": "CondPenet164",
        "cg:type": "ConditionPenetrance",
        "condition": [
          {
            "cg:id": "Cond027",
            "cg:type": "Condition",
            "name": "Mitochondrial complex 1 deficiency",
            "code": "2609"
          }
        ],
        "penetrance": "CG:high"
      },
      {
        "cg:id": "CondInh165",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond027",
          "cg:type": "Condition",
          "name": "Mitochondrial complex 1 deficiency",
          "code": "2609"
        },
        "modeOfInheritance": "HP:0001423"
      }
    ],
    "criterion": {
      "cg:id": "BS1",
      "cg:type": "Criterion",
      "description": "Allele frequency is greater than expected for disorder",
      "shortDescription": "MAF too high for disorder",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll079",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll085",
      "identifier": "http://reg.genome.network/allele/CA090926"
    },
    "condition": [
      {
        "cg:id": "Cond027",
        "cg:type": "Condition",
        "name": "Mitochondrial complex 1 deficiency",
        "code": "2609"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      },
      {
        "cg:id": "Agent005",
        "cg:type": "Agent",
        "description": "CSER",
        "name": "CSER Bake Off Project"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess053">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess054</td>
          <td>Perrault syndrome may be underreported due to misdiagnosis.  If the prevalence estimates rise, this assessment may need to be revisited</td>
          <td><div class="example_view" id="CritAssess054"><pre><code>{
  "cg:id": "CritAssess054",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS1",
    "cg:type": "Criterion",
    "description": "Allele frequency is greater than expected for disorder",
    "shortDescription": "MAF too high for disorder",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll080",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll086",
    "identifier": "http://reg.genome.network/allele/CA501230"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit056",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-08-30T06:00:00+00:00",
    "endedAtTime": "2016-08-30T06:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "AllFreq166",
        "cg:type": "AlleleFrequency",
        "ascertainment": "ExAC",
        "population": "CG:combined",
        "allele": {
          "cg:id": "CanAll080",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll086",
          "identifier": "http://reg.genome.network/allele/CA501230"
        },
        "alleleCount": 3,
        "alleleNumber": 120870,
        "alleleFrequency": 2.4e-05
      },
      {
        "cg:id": "CondPrev167",
        "cg:type": "ConditionPrevelance",
        "condition": [
          {
            "cg:id": "Cond028",
            "cg:type": "Condition",
            "name": "Perrault Syndrome",
            "code": "2855"
          }
        ],
        "maximum": 1.0e-06,
        "population": "CG:overall"
      },
      {
        "cg:id": "CondPenet168",
        "cg:type": "ConditionPenetrance",
        "condition": [
          {
            "cg:id": "Cond028",
            "cg:type": "Condition",
            "name": "Perrault Syndrome",
            "code": "2855"
          }
        ],
        "penetrance": "CG:high"
      },
      {
        "cg:id": "CondInh169",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond028",
          "cg:type": "Condition",
          "name": "Perrault Syndrome",
          "code": "2855"
        },
        "modeOfInheritance": "HP:0000007"
      }
    ],
    "criterion": {
      "cg:id": "BS1",
      "cg:type": "Criterion",
      "description": "Allele frequency is greater than expected for disorder",
      "shortDescription": "MAF too high for disorder",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll080",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll086",
      "identifier": "http://reg.genome.network/allele/CA501230"
    },
    "condition": [
      {
        "cg:id": "Cond028",
        "cg:type": "Condition",
        "name": "Perrault Syndrome",
        "code": "2855"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  },
  "explanation": "Perrault syndrome may be underreported due to misdiagnosis.  If the prevalence estimates rise, this assessment may need to be revisited"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess054">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess055</td>
          <td>According to Baylor internal data on this variant, it has been observed multiple times in asymptomatic parents internally and in controls and observed once as a homozygotes internally in unaffected.  Although the evidence for full penetrance is weak, the fact that there are multiple asymptomatic occurrences helps the case.</td>
          <td><div class="example_view" id="CritAssess055"><pre><code>{
  "cg:id": "CritAssess055",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS2",
    "cg:type": "Criterion",
    "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
    "shortDescription": "control frequency higher than penetrance predicts",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll081",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll087",
    "identifier": "http://reg.genome.network/allele/CA090918"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit057",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-05T06:00:00+00:00",
    "endedAtTime": "2016-09-05T06:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondPenet170",
        "cg:type": "ConditionPenetrance",
        "explanation": "The evidence for full penetrance is not great, due to the rarity of the disease, but the fact that it's diagnosed at a young age and majority of cases are de novo is suggestive",
        "condition": [
          {
            "cg:id": "Cond031",
            "cg:type": "Condition",
            "name": "Pendred syndrome",
            "code": "705"
          }
        ],
        "penetrance": "CG:complete"
      },
      {
        "cg:id": "CondInh171",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond031",
          "cg:type": "Condition",
          "name": "Pendred syndrome",
          "code": "705"
        },
        "modeOfInheritance": "HP:0000006"
      },
      {
        "cg:id": "IndAll172",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv013",
          "cg:type": "Individual",
          "description": "individual #013"
        },
        "primaryAllele": {
          "cg:id": "CanAll081",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll087",
          "identifier": "http://reg.genome.network/allele/CA090918"
        },
        "primaryZygosity": "CG:het"
      },
      {
        "cg:id": "IndAll173",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv014",
          "cg:type": "Individual",
          "description": "individual #014"
        },
        "primaryAllele": {
          "cg:id": "CanAll081",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll087",
          "identifier": "http://reg.genome.network/allele/CA090918"
        },
        "primaryZygosity": "CG:hom"
      },
      {
        "cg:id": "IndCond174",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv013",
          "cg:type": "Individual",
          "description": "individual #013"
        },
        "condition": [
          {
            "cg:id": "Cond031",
            "cg:type": "Condition",
            "name": "Pendred syndrome",
            "code": "705"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "IndCond175",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv014",
          "cg:type": "Individual",
          "description": "individual #014"
        },
        "condition": [
          {
            "cg:id": "Cond031",
            "cg:type": "Condition",
            "name": "Pendred syndrome",
            "code": "705"
          }
        ],
        "hasCondition": true
      }
    ],
    "criterion": {
      "cg:id": "BS2",
      "cg:type": "Criterion",
      "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
      "shortDescription": "control frequency higher than penetrance predicts",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll081",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll087",
      "identifier": "http://reg.genome.network/allele/CA090918"
    },
    "condition": [
      {
        "cg:id": "Cond029",
        "cg:type": "Condition",
        "name": "Wiedemann-Steiner syndrome",
        "code": "319182"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "According to Baylor internal data on this variant, it has been observed multiple times in asymptomatic parents internally and in controls and observed once as a homozygotes internally in unaffected.  Although the evidence for full penetrance is weak, the fact that there are multiple asymptomatic occurrences helps the case."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess055">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess056</td>
          <td>This variant was observed in unaffected healthy control chromosomes. PMIDs: 15122587, 18199528, 22703879, 24728327</td>
          <td><div class="example_view" id="CritAssess056"><pre><code>{
  "cg:id": "CritAssess056",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS2",
    "cg:type": "Criterion",
    "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
    "shortDescription": "control frequency higher than penetrance predicts",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll082",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll088",
    "identifier": "http://reg.genome.network/allele/CA008351"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit058",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-05T06:30:00+00:00",
    "endedAtTime": "2016-09-05T06:30:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondPenet176",
        "cg:type": "ConditionPenetrance",
        "explanation": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early.",
        "condition": [
          {
            "cg:id": "Cond030",
            "cg:type": "Condition",
            "name": "Familial Adenomatous Polyposis",
            "code": "733"
          }
        ],
        "penetrance": "CG:complete"
      },
      {
        "cg:id": "CondInh177",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond030",
          "cg:type": "Condition",
          "name": "Familial Adenomatous Polyposis",
          "code": "733"
        },
        "modeOfInheritance": "HP:0000006"
      },
      {
        "cg:id": "IndAll178",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv015",
          "cg:type": "Individual",
          "description": "individual #015"
        },
        "primaryAllele": {
          "cg:id": "CanAll082",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll088",
          "identifier": "http://reg.genome.network/allele/CA008351"
        },
        "primaryZygosity": "CG:het"
      },
      {
        "cg:id": "IndAll179",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv016",
          "cg:type": "Individual",
          "description": "individual #016"
        },
        "primaryAllele": {
          "cg:id": "CanAll082",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll088",
          "identifier": "http://reg.genome.network/allele/CA008351"
        },
        "primaryZygosity": "CG:het"
      },
      {
        "cg:id": "IndAll180",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv017",
          "cg:type": "Individual",
          "description": "individual #017"
        },
        "primaryAllele": {
          "cg:id": "CanAll082",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll088",
          "identifier": "http://reg.genome.network/allele/CA008351"
        },
        "primaryZygosity": "CG:het"
      },
      {
        "cg:id": "IndAll181",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv018",
          "cg:type": "Individual",
          "description": "individual #018"
        },
        "primaryAllele": {
          "cg:id": "CanAll082",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll088",
          "identifier": "http://reg.genome.network/allele/CA008351"
        },
        "primaryZygosity": "CG:het"
      },
      {
        "cg:id": "IndCond182",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv015",
          "cg:type": "Individual",
          "description": "individual #015"
        },
        "condition": [
          {
            "cg:id": "Cond030",
            "cg:type": "Condition",
            "name": "Familial Adenomatous Polyposis",
            "code": "733"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "IndCond183",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv016",
          "cg:type": "Individual",
          "description": "individual #016"
        },
        "condition": [
          {
            "cg:id": "Cond030",
            "cg:type": "Condition",
            "name": "Familial Adenomatous Polyposis",
            "code": "733"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "IndCond184",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv017",
          "cg:type": "Individual",
          "description": "individual #017"
        },
        "condition": [
          {
            "cg:id": "Cond030",
            "cg:type": "Condition",
            "name": "Familial Adenomatous Polyposis",
            "code": "733"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "IndCond185",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv018",
          "cg:type": "Individual",
          "description": "individual #018"
        },
        "condition": [
          {
            "cg:id": "Cond030",
            "cg:type": "Condition",
            "name": "Familial Adenomatous Polyposis",
            "code": "733"
          }
        ],
        "hasCondition": true
      }
    ],
    "criterion": {
      "cg:id": "BS2",
      "cg:type": "Criterion",
      "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
      "shortDescription": "control frequency higher than penetrance predicts",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll082",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll088",
      "identifier": "http://reg.genome.network/allele/CA008351"
    },
    "condition": [
      {
        "cg:id": "Cond030",
        "cg:type": "Condition",
        "name": "Familial Adenomatous Polyposis",
        "code": "733"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      },
      {
        "cg:id": "Agent005",
        "cg:type": "Agent",
        "description": "CSER",
        "name": "CSER Bake Off Project"
      }
    ]
  },
  "explanation": "This variant was observed in unaffected healthy control chromosomes. PMIDs: 15122587, 18199528, 22703879, 24728327"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess056">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess057</td>
          <td>Although this variant was found in 5 individuals with Marfan Syndrome, it was identified in 2 unaffected relatives from two separate families. PMID: 12402346</td>
          <td><div class="example_view" id="CritAssess057"><pre><code>{
  "cg:id": "CritAssess057",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS2",
    "cg:type": "Criterion",
    "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
    "shortDescription": "control frequency higher than penetrance predicts",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll083",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll089",
    "identifier": "http://reg.genome.network/allele/CA013604"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit059",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-05T07:00:59+00:00",
    "endedAtTime": "2016-09-05T07:00:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "CondPenet186",
        "cg:type": "ConditionPenetrance",
        "explanation": "Marfan syndrome involves a range of skeletal features that should be identifiable at birth, if not during adolescence. The ocular and cardiovascular aspects may manifest later. 1/4 of mutations are de novo.",
        "condition": [
          {
            "cg:id": "Cond026",
            "cg:type": "Condition",
            "name": "Marfan Syndrome",
            "code": "558"
          }
        ],
        "penetrance": "CG:complete"
      },
      {
        "cg:id": "CondInh187",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond026",
          "cg:type": "Condition",
          "name": "Marfan Syndrome",
          "code": "558"
        },
        "modeOfInheritance": "HP:0000006"
      },
      {
        "cg:id": "IndAll188",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv019",
          "cg:type": "Individual",
          "description": "individual #019"
        },
        "primaryAllele": {
          "cg:id": "CanAll083",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll089",
          "identifier": "http://reg.genome.network/allele/CA013604"
        },
        "primaryZygosity": "CG:het"
      },
      {
        "cg:id": "IndAll189",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv020",
          "cg:type": "Individual",
          "description": "individual #020"
        },
        "primaryAllele": {
          "cg:id": "CanAll083",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll089",
          "identifier": "http://reg.genome.network/allele/CA013604"
        },
        "primaryZygosity": "CG:het"
      },
      {
        "cg:id": "IndCond190",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv019",
          "cg:type": "Individual",
          "description": "individual #019"
        },
        "condition": [
          {
            "cg:id": "Cond026",
            "cg:type": "Condition",
            "name": "Marfan Syndrome",
            "code": "558"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "IndCond191",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv020",
          "cg:type": "Individual",
          "description": "individual #020"
        },
        "condition": [
          {
            "cg:id": "Cond026",
            "cg:type": "Condition",
            "name": "Marfan Syndrome",
            "code": "558"
          }
        ],
        "hasCondition": true
      }
    ],
    "criterion": {
      "cg:id": "BS2",
      "cg:type": "Criterion",
      "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
      "shortDescription": "control frequency higher than penetrance predicts",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll083",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll089",
      "identifier": "http://reg.genome.network/allele/CA013604"
    },
    "condition": [
      {
        "cg:id": "Cond026",
        "cg:type": "Condition",
        "name": "Marfan Syndrome",
        "code": "558"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      },
      {
        "cg:id": "Agent005",
        "cg:type": "Agent",
        "description": "CSER",
        "name": "CSER Bake Off Project"
      }
    ]
  },
  "explanation": "Although this variant was found in 5 individuals with Marfan Syndrome, it was identified in 2 unaffected relatives from two separate families. PMID: 12402346"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess057">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess059</td>
          <td></td>
          <td><div class="example_view" id="CritAssess059"><pre><code>{
  "cg:id": "CritAssess059",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS3",
    "cg:type": "Criterion",
    "description": "Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing",
    "shortDescription": "well-established no damaging effect functional studies",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll085",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll091",
    "identifier": "http://reg.genome.network/allele/CA132727"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit061",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-05T09:01:00+00:00",
    "endedAtTime": "2016-09-05T09:01:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Func192",
        "cg:type": "FunctionalData",
        "result": "Unchanged from reference",
        "dataType": "CG:iodeff",
        "contextualAllele": {
          "cg:id": "CtxAll092",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll086",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll092",
            "identifier": "http://reg.genome.network/allele/CA109417"
          },
          "alleleName": "NP_000432..1:p.Leu117Phe"
        },
        "gene": {
          "cg:id": "G016",
          "cg:type": "Gene",
          "symbol": "SLC26A4"
        }
      },
      {
        "cg:id": "Func193",
        "cg:type": "FunctionalData",
        "explanation": "Both wild type and NP_000432..1:p.Leu117Phe localize to the same location",
        "result": "localized to apical membrane",
        "dataType": "CG:celloc",
        "contextualAllele": {
          "cg:id": "CtxAll092",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll086",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll092",
            "identifier": "http://reg.genome.network/allele/CA109417"
          },
          "alleleName": "NP_000432..1:p.Leu117Phe"
        },
        "gene": {
          "cg:id": "G016",
          "cg:type": "Gene",
          "symbol": "SLC26A4"
        }
      }
    ],
    "criterion": {
      "cg:id": "BS3",
      "cg:type": "Criterion",
      "description": "Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing",
      "shortDescription": "well-established no damaging effect functional studies",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll085",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll091",
      "identifier": "http://reg.genome.network/allele/CA132727"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess059">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess060</td>
          <td></td>
          <td><div class="example_view" id="CritAssess060"><pre><code>{
  "cg:id": "CritAssess060",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS3",
    "cg:type": "Criterion",
    "description": "Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing",
    "shortDescription": "well-established no damaging effect functional studies",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll087",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll093",
    "identifier": "http://reg.genome.network/allele/CA501231"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit062",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-05T09:30:59+00:00",
    "endedAtTime": "2016-09-05T09:30:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Func194",
        "cg:type": "FunctionalData",
        "result": "Unchanged from reference",
        "dataType": "CG:neuritegrowth",
        "contextualAllele": {
          "cg:id": "CtxAll093",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll087",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll093",
            "identifier": "http://reg.genome.network/allele/CA501231"
          },
          "alleleName": "NM_152594.2(SPRED1): c.587C>T"
        },
        "gene": {
          "cg:id": "G045",
          "cg:type": "Gene",
          "symbol": "SPRED1"
        }
      },
      {
        "cg:id": "Func195",
        "cg:type": "FunctionalData",
        "result": "localized to apical membrane",
        "dataType": "CG:elk1phosvialucif",
        "contextualAllele": {
          "cg:id": "CtxAll093",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll087",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll093",
            "identifier": "http://reg.genome.network/allele/CA501231"
          },
          "alleleName": "NM_152594.2(SPRED1): c.587C>T"
        },
        "gene": {
          "cg:id": "G045",
          "cg:type": "Gene",
          "symbol": "SPRED1"
        }
      }
    ],
    "criterion": {
      "cg:id": "BS3",
      "cg:type": "Criterion",
      "description": "Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing",
      "shortDescription": "well-established no damaging effect functional studies",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll087",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll093",
      "identifier": "http://reg.genome.network/allele/CA501231"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess060">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess061</td>
          <td></td>
          <td><div class="example_view" id="CritAssess061"><pre><code>{
  "cg:id": "CritAssess061",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS4",
    "cg:type": "Criterion",
    "description": "Lack of segregation in affected members of a family",
    "shortDescription": "does not segregate with disease",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll088",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll094",
    "identifier": "http://reg.genome.network/allele/CA023690"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit063",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-05T09:50:59+00:00",
    "endedAtTime": "2016-09-05T09:50:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "FamSegData196",
        "cg:type": "FamilySegregationData",
        "explanation": "observed in one case diagnosed with Familial Hypercholesterolemia, characterized by elevated LDLC levels. Two siblings of the proband also have elevated LDLC levels but neither carry the 313+2T>C variant.",
        "condition": {
          "cg:id": "Cond035",
          "cg:type": "Condition",
          "name": "Familial Hypercholesterolemia",
          "code": "D006938"
        },
        "canonicalAllele": {
          "cg:id": "CanAll088",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll094",
          "identifier": "http://reg.genome.network/allele/CA023690"
        },
        "family": {
          "cg:id": "Fam008",
          "cg:type": "Family",
          "description": "Family #008"
        },
        "phenotypePositiveAllelePositive": 0,
        "phenotypePositiveAlleleNegative": 2,
        "anyInconsistentSegregations": true
      }
    ],
    "criterion": {
      "cg:id": "BS4",
      "cg:type": "Criterion",
      "description": "Lack of segregation in affected members of a family",
      "shortDescription": "does not segregate with disease",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll088",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll094",
      "identifier": "http://reg.genome.network/allele/CA023690"
    },
    "condition": [
      {
        "cg:id": "Cond035",
        "cg:type": "Condition",
        "name": "Familial Hypercholesterolemia",
        "code": "D006938"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess061">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess062</td>
          <td>The inconsistent segregation data is reasonably explainable as a phenocopy, and therefore does not provide evidence for the benignity of this allele.</td>
          <td><div class="example_view" id="CritAssess062"><pre><code>{
  "cg:id": "CritAssess062",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS4",
    "cg:type": "Criterion",
    "description": "Lack of segregation in affected members of a family",
    "shortDescription": "does not segregate with disease",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll089",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll095",
    "identifier": "http://reg.genome.network/allele/CA010777"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit064",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-05T10:50:59+00:00",
    "endedAtTime": "2016-09-05T10:50:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "FamSegData197",
        "cg:type": "FamilySegregationData",
        "explanation": "The individual with HCM but without the allele is likely a phenocopy explained\nby environmental factors (obese, alcohol)",
        "condition": {
          "cg:id": "Cond023",
          "cg:type": "Condition",
          "name": "Breast-ovarian cancer, familial 2",
          "code": "612555"
        },
        "canonicalAllele": {
          "cg:id": "CanAll089",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll095",
          "identifier": "http://reg.genome.network/allele/CA010777"
        },
        "family": {
          "cg:id": "Fam009",
          "cg:type": "Family",
          "description": "Family #009"
        },
        "phenotypePositiveAllelePositive": 3,
        "phenotypePositiveAlleleNegative": 1,
        "anyInconsistentSegregations": true
      }
    ],
    "criterion": {
      "cg:id": "BS4",
      "cg:type": "Criterion",
      "description": "Lack of segregation in affected members of a family",
      "shortDescription": "does not segregate with disease",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll089",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll095",
      "identifier": "http://reg.genome.network/allele/CA010777"
    },
    "condition": [
      {
        "cg:id": "Cond020",
        "cg:type": "Condition",
        "name": "Hypertrophic Cardiomyopathy",
        "code": "217569"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "The inconsistent segregation data is reasonably explainable as a phenocopy, and therefore does not provide evidence for the benignity of this allele."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess062">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess063</td>
          <td>In a family tested by the LMM, the c.1484C>T variant was identified in a proband with clinical diagnosis of HCM. This proband has two relatives that also have clinical diagnosis of HCM however never relative carries the c.1484C>T variant.</td>
          <td><div class="example_view" id="CritAssess063"><pre><code>{
  "cg:id": "CritAssess063",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BS4",
    "cg:type": "Criterion",
    "description": "Lack of segregation in affected members of a family",
    "shortDescription": "does not segregate with disease",
    "defaultOutcome": "CG:S",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll090",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll096",
    "identifier": "http://reg.genome.network/allele/CA133141"
  },
  "outcome": "CG:S",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit065",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-05T11:00:00+00:00",
    "endedAtTime": "2016-09-05T11:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "FamSegData271",
        "cg:type": "FamilySegregationData",
        "condition": {
          "cg:id": "Cond020",
          "cg:type": "Condition",
          "name": "Hypertrophic Cardiomyopathy",
          "code": "217569"
        },
        "canonicalAllele": {
          "cg:id": "CanAll090",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll096",
          "identifier": "http://reg.genome.network/allele/CA133141"
        },
        "family": {
          "cg:id": "Fam010"
        },
        "phenotypePositiveAllelePositive": 0,
        "phenotypePositiveAlleleNegative": 2,
        "anyInconsistentSegregations": true
      }
    ],
    "criterion": {
      "cg:id": "BS4",
      "cg:type": "Criterion",
      "description": "Lack of segregation in affected members of a family",
      "shortDescription": "does not segregate with disease",
      "defaultOutcome": "CG:S",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll090",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll096",
      "identifier": "http://reg.genome.network/allele/CA133141"
    },
    "condition": [
      {
        "cg:id": "Cond020",
        "cg:type": "Condition",
        "name": "Hypertrophic Cardiomyopathy",
        "code": "217569"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent001",
        "cg:type": "Agent",
        "description": "SH",
        "name": "Steven Harrison"
      }
    ]
  },
  "explanation": "In a family tested by the LMM, the c.1484C>T variant was identified in a proband with clinical diagnosis of HCM. This proband has two relatives that also have clinical diagnosis of HCM however never relative carries the c.1484C>T variant."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess063">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess064</td>
          <td></td>
          <td><div class="example_view" id="CritAssess064"><pre><code>{
  "cg:id": "CritAssess064",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP1",
    "cg:type": "Criterion",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "shortDescription": "missense variant when truncating is pathogenic\n",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll091",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll097",
    "identifier": "http://reg.genome.network/allele/CA020822"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit066",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-06T08:00:00+00:00",
    "endedAtTime": "2016-09-06T08:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon198",
        "cg:type": "MolecularConsequence",
        "contextualAllele": {
          "cg:id": "CtxAll097",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll091",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll097",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2) :c.4779A>C"
        },
        "consequence": "SO:0001583",
        "lof": true
      },
      {
        "cg:id": "CondMech199",
        "cg:type": "ConditionMechanism",
        "explanation": "ClinVar shows 421 variants in BRCA2 that are pathogenic for HBOC. 402 of these (~95%) are frameshift, nonsense, or splice-site variants.",
        "mechanism": "SO:0002054",
        "strength": "CG:primary",
        "gene": {
          "cg:id": "G002",
          "cg:type": "Gene",
          "symbol": "BRCA2"
        },
        "condition": {
          "cg:id": "Cond034",
          "cg:type": "Condition",
          "name": "macrocephaly",
          "code": "0000256"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP1",
      "cg:type": "Criterion",
      "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
      "shortDescription": "missense variant when truncating is pathogenic\n",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll091",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll097",
      "identifier": "http://reg.genome.network/allele/CA020822"
    },
    "condition": [
      {
        "cg:id": "Cond036",
        "cg:type": "Condition",
        "name": "Hereditary Breast and Ovarian Cancer",
        "code": "D061325"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess064">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess065</td>
          <td></td>
          <td><div class="example_view" id="CritAssess065"><pre><code>{
  "cg:id": "CritAssess065",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP1",
    "cg:type": "Criterion",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "shortDescription": "missense variant when truncating is pathogenic\n",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll091",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll097",
    "identifier": "http://reg.genome.network/allele/CA020822"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit067",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-06T09:00:00+00:00",
    "endedAtTime": "2016-09-06T09:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon198",
        "cg:type": "MolecularConsequence",
        "contextualAllele": {
          "cg:id": "CtxAll097",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll091",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll097",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2) :c.4779A>C"
        },
        "consequence": "SO:0001583",
        "lof": true
      },
      {
        "cg:id": "CondMech200",
        "cg:type": "ConditionMechanism",
        "explanation": "ClinVar shows 61 variants in BRCA2 that are pathogenic for Fanconi Anemia. 55 of these (~90%) are frameshift, nonsense, or splice-site variants.",
        "mechanism": "SO:0002054",
        "strength": "CG:primary",
        "gene": {
          "cg:id": "G002",
          "cg:type": "Gene",
          "symbol": "BRCA2"
        },
        "condition": {
          "cg:id": "Cond035",
          "cg:type": "Condition",
          "name": "Familial Hypercholesterolemia",
          "code": "D006938"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP1",
      "cg:type": "Criterion",
      "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
      "shortDescription": "missense variant when truncating is pathogenic\n",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll091",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll097",
      "identifier": "http://reg.genome.network/allele/CA020822"
    },
    "condition": [
      {
        "cg:id": "Cond037",
        "cg:type": "Condition",
        "name": "Fanconi Anemia",
        "code": "84"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess065">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess066</td>
          <td></td>
          <td><div class="example_view" id="CritAssess066"><pre><code>{
  "cg:id": "CritAssess066",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP1",
    "cg:type": "Criterion",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "shortDescription": "missense variant when truncating is pathogenic\n",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll092",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll098",
    "identifier": "http://reg.genome.network/allele/CA090950"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit068",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-06T09:59:59+00:00",
    "endedAtTime": "2016-09-06T09:59:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon201",
        "cg:type": "MolecularConsequence",
        "contextualAllele": {
          "cg:id": "CtxAll098",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll092",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll098",
            "identifier": "http://reg.genome.network/allele/CA090950"
          },
          "alleleName": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)"
        },
        "consequence": "SO:0001583",
        "lof": true
      },
      {
        "cg:id": "CondMech202",
        "cg:type": "ConditionMechanism",
        "explanation": "shown in both humans and mouse models",
        "mechanism": "SO:0002054",
        "strength": "CG:primary",
        "gene": {
          "cg:id": "G048",
          "cg:type": "Gene",
          "symbol": "LOXHD1"
        },
        "condition": {
          "cg:id": "Cond010",
          "cg:type": "Condition",
          "name": "Hearing Loss",
          "code": "87884"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP1",
      "cg:type": "Criterion",
      "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
      "shortDescription": "missense variant when truncating is pathogenic\n",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll092",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll098",
      "identifier": "http://reg.genome.network/allele/CA090950"
    },
    "condition": [
      {
        "cg:id": "Cond010",
        "cg:type": "Condition",
        "name": "Hearing Loss",
        "code": "87884"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess066">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess067</td>
          <td>While truncations are usually the cause of TTN related DCM, missense variants in certain regions can also cause disease.</td>
          <td><div class="example_view" id="CritAssess067"><pre><code>{
  "cg:id": "CritAssess067",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP1",
    "cg:type": "Criterion",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "shortDescription": "missense variant when truncating is pathogenic\n",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll093",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll099",
    "identifier": "http://reg.genome.network/allele/CA256496"
  },
  "outcome": "CG:R",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit069",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-06T11:00:00+00:00",
    "endedAtTime": "2016-09-06T11:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MolCon203",
        "cg:type": "MolecularConsequence",
        "contextualAllele": {
          "cg:id": "CtxAll099",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll093",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll099",
            "identifier": "http://reg.genome.network/allele/CA256496"
          },
          "alleleName": "NM_133378.4(TTN): c.2926T>C"
        },
        "consequence": "SO:0001583",
        "lof": true
      },
      {
        "cg:id": "CondMech204",
        "cg:type": "ConditionMechanism",
        "mechanism": "SO:0002054",
        "strength": "CG:primary",
        "gene": {
          "cg:id": "G003",
          "cg:type": "Gene",
          "symbol": "TTN"
        },
        "condition": {
          "cg:id": "Cond002",
          "cg:type": "Condition",
          "name": "Dilated Cardiomyopathy",
          "code": "217604"
        }
      },
      {
        "cg:id": "CondMech205",
        "cg:type": "ConditionMechanism",
        "explanation": "Truncating variants in TTN are associated with DCM. However, if missense variants are located in the A-band, impact the A-band, or are in a highly expressed region, they can also be associated with DCM.",
        "mechanism": "SO:0001583",
        "strength": "CG:established",
        "gene": {
          "cg:id": "G003",
          "cg:type": "Gene",
          "symbol": "TTN"
        },
        "condition": {
          "cg:id": "Cond002",
          "cg:type": "Condition",
          "name": "Dilated Cardiomyopathy",
          "code": "217604"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP1",
      "cg:type": "Criterion",
      "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
      "shortDescription": "missense variant when truncating is pathogenic\n",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll093",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll099",
      "identifier": "http://reg.genome.network/allele/CA256496"
    },
    "condition": [
      {
        "cg:id": "Cond002",
        "cg:type": "Condition",
        "name": "Dilated Cardiomyopathy",
        "code": "217604"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  },
  "explanation": "While truncations are usually the cause of TTN related DCM, missense variants in certain regions can also cause disease."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess067">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess068</td>
          <td></td>
          <td><div class="example_view" id="CritAssess068"><pre><code>{
  "cg:id": "CritAssess068",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP2",
    "cg:type": "Criterion",
    "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
    "shortDescription": "in trans with dominant OR in cis with path variant",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll094",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll100",
    "identifier": "http://reg.genome.network/allele/CA143572"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit070",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-06T12:00:00+00:00",
    "endedAtTime": "2016-09-06T12:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndAll206",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv021",
          "cg:type": "Individual",
          "description": "individual #021"
        },
        "primaryAllele": {
          "cg:id": "CanAll094",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll100",
          "identifier": "http://reg.genome.network/allele/CA143572"
        },
        "secondaryAllele": {
          "cg:id": "CanAll095",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll101",
          "identifier": "http://reg.genome.network/allele/CA252233"
        },
        "phase": "CG:cis"
      },
      {
        "cg:id": "IndAll207",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv022",
          "cg:type": "Individual",
          "description": "individual #022"
        },
        "primaryAllele": {
          "cg:id": "CanAll094",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll100",
          "identifier": "http://reg.genome.network/allele/CA143572"
        },
        "secondaryAllele": {
          "cg:id": "CanAll095",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll101",
          "identifier": "http://reg.genome.network/allele/CA252233"
        },
        "phase": "CG:cis"
      },
      {
        "cg:id": "IndCond208",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv021",
          "cg:type": "Individual",
          "description": "individual #021"
        },
        "condition": [
          {
            "cg:id": "Cond038",
            "cg:type": "Condition",
            "name": "Retinitis Pigmentosa",
            "code": "791"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "IndCond209",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv022",
          "cg:type": "Individual",
          "description": "individual #022"
        },
        "condition": [
          {
            "cg:id": "Cond038",
            "cg:type": "Condition",
            "name": "Retinitis Pigmentosa",
            "code": "791"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "MenDisVarInterp210",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll095",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll101",
          "identifier": "http://reg.genome.network/allele/CA252233"
        },
        "condition": [
          {
            "cg:id": "Cond038",
            "cg:type": "Condition",
            "name": "Retinitis Pigmentosa",
            "code": "791"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "BP2",
      "cg:type": "Criterion",
      "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
      "shortDescription": "in trans with dominant OR in cis with path variant",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll094",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll100",
      "identifier": "http://reg.genome.network/allele/CA143572"
    },
    "condition": [
      {
        "cg:id": "Cond038",
        "cg:type": "Condition",
        "name": "Retinitis Pigmentosa",
        "code": "791"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess068">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess069</td>
          <td></td>
          <td><div class="example_view" id="CritAssess069"><pre><code>{
  "cg:id": "CritAssess069",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP2",
    "cg:type": "Criterion",
    "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
    "shortDescription": "in trans with dominant OR in cis with path variant",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll096",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll102",
    "identifier": "http://reg.genome.network/allele/CA182840"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit071",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-06T12:59:59+00:00",
    "endedAtTime": "2016-09-06T12:59:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndAll211",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv023",
          "cg:type": "Individual",
          "description": "individual #023"
        },
        "primaryAllele": {
          "cg:id": "CanAll096",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll102",
          "identifier": "http://reg.genome.network/allele/CA182840"
        },
        "secondaryAllele": {
          "cg:id": "CanAll097",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll103",
          "identifier": "http://reg.genome.network/allele/CA182839"
        },
        "phase": "CG:cis"
      },
      {
        "cg:id": "IndCond212",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv023",
          "cg:type": "Individual",
          "description": "individual #023"
        },
        "condition": [
          {
            "cg:id": "Cond039",
            "cg:type": "Condition",
            "name": "Congenital Absence of the Vas Deferens",
            "code": "0012873"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "MenDisVarInterp213",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll097",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll103",
          "identifier": "http://reg.genome.network/allele/CA182839"
        },
        "condition": [
          {
            "cg:id": "Cond039",
            "cg:type": "Condition",
            "name": "Congenital Absence of the Vas Deferens",
            "code": "0012873"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "BP2",
      "cg:type": "Criterion",
      "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
      "shortDescription": "in trans with dominant OR in cis with path variant",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll096",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll102",
      "identifier": "http://reg.genome.network/allele/CA182840"
    },
    "condition": [
      {
        "cg:id": "Cond039",
        "cg:type": "Condition",
        "name": "Congenital Absence of the Vas Deferens",
        "code": "0012873"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess069">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess070</td>
          <td>although this variant was found in trans with another variant in the same gene, that variant is VUS and not an established pathogenic variant and cardiomyopathies can be variable in penetrance. </td>
          <td><div class="example_view" id="CritAssess070"><pre><code>{
  "cg:id": "CritAssess070",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP2",
    "cg:type": "Criterion",
    "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
    "shortDescription": "in trans with dominant OR in cis with path variant",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll098",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll104",
    "identifier": "http://reg.genome.network/allele/CA501235"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit072",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-06T14:00:00+00:00",
    "endedAtTime": "2016-09-06T14:00:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndAll214",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv024",
          "cg:type": "Individual",
          "description": "individual #024"
        },
        "primaryAllele": {
          "cg:id": "CanAll098",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll104",
          "identifier": "http://reg.genome.network/allele/CA501235"
        },
        "secondaryAllele": {
          "cg:id": "CanAll099",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll105",
          "identifier": "http://reg.genome.network/allele/CA501236"
        },
        "phase": "CG:trans"
      },
      {
        "cg:id": "IndCond215",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv024",
          "cg:type": "Individual",
          "description": "individual #024"
        },
        "condition": [
          {
            "cg:id": "Cond007",
            "cg:type": "Condition",
            "name": "Cardiomyopathy",
            "code": "167848"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "MenDisVarInterp216",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll099",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll105",
          "identifier": "http://reg.genome.network/allele/CA501236"
        },
        "condition": [
          {
            "cg:id": "Cond007",
            "cg:type": "Condition",
            "name": "Cardiomyopathy",
            "code": "167848"
          }
        ],
        "clinicalSignificance": "LN:LA26333-7"
      },
      {
        "cg:id": "CondInh217",
        "cg:type": "ConditionInheritance",
        "condition": {
          "cg:id": "Cond038",
          "cg:type": "Condition",
          "name": "Retinitis Pigmentosa",
          "code": "791"
        },
        "modeOfInheritance": "HP:0000006"
      },
      {
        "cg:id": "CondPenet218",
        "cg:type": "ConditionPenetrance",
        "condition": [
          {
            "cg:id": "Cond007",
            "cg:type": "Condition",
            "name": "Cardiomyopathy",
            "code": "167848"
          }
        ],
        "penetrance": "CG:incomplete"
      }
    ],
    "criterion": {
      "cg:id": "BP2",
      "cg:type": "Criterion",
      "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
      "shortDescription": "in trans with dominant OR in cis with path variant",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll098",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll104",
      "identifier": "http://reg.genome.network/allele/CA501235"
    },
    "condition": [
      {
        "cg:id": "Cond007",
        "cg:type": "Condition",
        "name": "Cardiomyopathy",
        "code": "167848"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  },
  "explanation": "although this variant was found in trans with another variant in the same gene, that variant is VUS and not an established pathogenic variant and cardiomyopathies can be variable in penetrance. "
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess070">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess071</td>
          <td></td>
          <td><div class="example_view" id="CritAssess071"><pre><code>{
  "cg:id": "CritAssess071",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP3",
    "cg:type": "Criterion",
    "description": "In-frame deletions/insertions in a repetitive region without a known function",
    "shortDescription": "in-frame indel unknown function",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll100",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll106",
    "identifier": "http://reg.genome.network/allele/CA148212"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit082",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-15T08:12:00+00:00",
    "endedAtTime": "2016-09-15T08:12:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Reg220",
        "cg:type": "RegionTypesAndAlleles",
        "explanation": "region is repeating but does not have a known function.",
        "referenceSequence": {
          "cg:id": "RefSeq008",
          "cg:type": "ReferenceSequence",
          "identifier": "NM_001271208.1"
        },
        "start": 154,
        "stop": 183,
        "isNotTypes": [
          "CG:functional"
        ],
        "isTypes": [
          "CG:repeat"
        ],
        "includedAlleles": [
          {
            "cg:id": "CtxAll106",
            "cg:type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "cg:id": "CanAll100",
              "cg:type": "CanonicalAllele",
              "preferredCtxAllele": "CtxAll106",
              "identifier": "http://reg.genome.network/allele/CA148212"
            },
            "alleleName": "NM_001271208.1(NEB):c.169_183delCTGGCACAGCCAGCA"
          }
        ]
      },
      {
        "cg:id": "MolCon223",
        "cg:type": "MolecularConsequence",
        "consequence": "SO:0001825",
        "contextualAllele": {
          "cg:id": "CtxAll106",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll100",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll106",
            "identifier": "http://reg.genome.network/allele/CA148212"
          },
          "alleleName": "NM_001271208.1(NEB):c.169_183delCTGGCACAGCCAGCA"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP3",
      "cg:type": "Criterion",
      "description": "In-frame deletions/insertions in a repetitive region without a known function",
      "shortDescription": "in-frame indel unknown function",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll100",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll106",
      "identifier": "http://reg.genome.network/allele/CA148212"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess071">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess072</td>
          <td>BP3 does not apply for this variant because it is not an in-frame deletion (it's a deletion of 10 bp) and, although the I Band is a repetitive region that contains tandem arrays of immunoglobulin domains, these folds may be an important component for TTN's elasticity.</td>
          <td><div class="example_view" id="CritAssess072"><pre><code>{
  "cg:id": "CritAssess072",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP3",
    "cg:type": "Criterion",
    "description": "In-frame deletions/insertions in a repetitive region without a known function",
    "shortDescription": "in-frame indel unknown function",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll101",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll107",
    "identifier": "http://reg.genome.network/allele/CA273551"
  },
  "outcome": "CG:R",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit083",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-15T09:23:00+00:00",
    "endedAtTime": "2016-09-15T09:23:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Reg225",
        "cg:type": "RegionTypesAndAlleles",
        "explanation": "The I-Band is a large region of repetitive Igg domains. Allele NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC is contained by this region.",
        "referenceSequence": {
          "cg:id": "RegSeq009"
        },
        "start": 6509,
        "stop": 39262,
        "isTypes": [
          "CG:repeat"
        ],
        "includedAlleles": [
          {
            "cg:id": "CtxAll107",
            "cg:type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "cg:id": "CanAll101",
              "cg:type": "CanonicalAllele",
              "preferredCtxAllele": "CtxAll107",
              "identifier": "http://reg.genome.network/allele/CA273551"
            },
            "alleleName": "NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC"
          }
        ]
      },
      {
        "cg:id": "MolCon227",
        "cg:type": "MolecularConsequence",
        "consequence": "SO:0001589",
        "contextualAllele": {
          "cg:id": "CtxAll107",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll101",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll107",
            "identifier": "http://reg.genome.network/allele/CA273551"
          },
          "alleleName": "NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP3",
      "cg:type": "Criterion",
      "description": "In-frame deletions/insertions in a repetitive region without a known function",
      "shortDescription": "in-frame indel unknown function",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll101",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll107",
      "identifier": "http://reg.genome.network/allele/CA273551"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  },
  "explanation": "BP3 does not apply for this variant because it is not an in-frame deletion (it's a deletion of 10 bp) and, although the I Band is a repetitive region that contains tandem arrays of immunoglobulin domains, these folds may be an important component for TTN's elasticity."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess072">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess073</td>
          <td></td>
          <td><div class="example_view" id="CritAssess073"><pre><code>{
  "cg:id": "CritAssess073",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP3",
    "cg:type": "Criterion",
    "description": "In-frame deletions/insertions in a repetitive region without a known function",
    "shortDescription": "in-frame indel unknown function",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll102",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll108",
    "identifier": "http://reg.genome.network/allele/CA184168"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit084",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-09-15T10:40:00+00:00",
    "endedAtTime": "2016-09-15T10:40:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "Reg229",
        "cg:type": "RegionTypesAndAlleles",
        "referenceSequence": {
          "cg:id": "RefSeq010",
          "cg:type": "ReferenceSequence",
          "identifier": "NM_147196.2"
        },
        "start": 522,
        "stop": 548,
        "isTypes": [
          "CG:repeat"
        ],
        "includedAlleles": [
          {
            "cg:id": "CtxAll108",
            "cg:type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "cg:id": "CanAll102",
              "cg:type": "CanonicalAllele",
              "preferredCtxAllele": "CtxAll108",
              "identifier": "http://reg.genome.network/allele/CA184168"
            },
            "alleleName": "NM_147196.2(TMIE):c.391_393dupAAG"
          }
        ]
      },
      {
        "cg:id": "MolCon231",
        "cg:type": "MolecularConsequence",
        "consequence": "SO:0001823",
        "contextualAllele": {
          "cg:id": "CtxAll108",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll102",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll108",
            "identifier": "http://reg.genome.network/allele/CA184168"
          },
          "alleleName": "NM_147196.2(TMIE):c.391_393dupAAG"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP3",
      "cg:type": "Criterion",
      "description": "In-frame deletions/insertions in a repetitive region without a known function",
      "shortDescription": "in-frame indel unknown function",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll102",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll108",
      "identifier": "http://reg.genome.network/allele/CA184168"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess073">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess074</td>
          <td>Seven of the nine predictions found this variant to be benign, which we consider sufficient to support.</td>
          <td><div class="example_view" id="CritAssess074"><pre><code>{
  "cg:id": "CritAssess074",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP4",
    "cg:type": "Criterion",
    "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "shortDescription": "computational evidence for\nno gene impact",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll103",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll109",
    "identifier": "http://reg.genome.network/allele/CA020822"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit085",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-04T08:01:59+00:00",
    "endedAtTime": "2016-10-04T08:01:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "InSilicoPred232",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll109",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll103",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll109",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
        },
        "type": "CG:me",
        "tool": "SIFT",
        "value": "Benign"
      },
      {
        "cg:id": "InSilicoPred233",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll109",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll103",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll109",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
        },
        "type": "CG:me",
        "tool": "Polyphen",
        "value": "Possibly damaging"
      },
      {
        "cg:id": "InSilicoPred234",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll109",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll103",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll109",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
        },
        "type": "CG:me",
        "tool": "FATHMM",
        "value": "Benign"
      },
      {
        "cg:id": "InSilicoPred235",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll109",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll103",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll109",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
        },
        "type": "CG:me",
        "tool": "LRT",
        "value": "Benign"
      },
      {
        "cg:id": "InSilicoPred236",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll109",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll103",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll109",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
        },
        "type": "CG:me",
        "tool": "MetaSVM",
        "value": "Benign"
      },
      {
        "cg:id": "InSilicoPred237",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll109",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll103",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll109",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
        },
        "type": "CG:me",
        "tool": "Mutation assessor",
        "value": "Benign"
      },
      {
        "cg:id": "InSilicoPred238",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll109",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll103",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll109",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
        },
        "type": "CG:me",
        "tool": "Mutation taster",
        "value": "Benign"
      },
      {
        "cg:id": "InSilicoPred239",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll109",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll103",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll109",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
        },
        "type": "CG:me",
        "tool": "Provean",
        "value": "Benign"
      },
      {
        "cg:id": "InSilicoPred240",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll109",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll103",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll109",
            "identifier": "http://reg.genome.network/allele/CA020822"
          },
          "alleleName": "NM_000059.3(BRCA2):c.4779A>C"
        },
        "type": "CG:me",
        "tool": "Condel",
        "value": "deleterious"
      }
    ],
    "criterion": {
      "cg:id": "BP4",
      "cg:type": "Criterion",
      "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
      "shortDescription": "computational evidence for\nno gene impact",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll103",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll109",
      "identifier": "http://reg.genome.network/allele/CA020822"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent006",
        "cg:type": "Agent",
        "description": "RG",
        "name": "Rajarshi Ghosh"
      }
    ]
  },
  "explanation": "Seven of the nine predictions found this variant to be benign, which we consider sufficient to support."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess074">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess076</td>
          <td></td>
          <td><div class="example_view" id="CritAssess076"><pre><code>{
  "cg:id": "CritAssess076",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP4",
    "cg:type": "Criterion",
    "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "shortDescription": "computational evidence for\nno gene impact",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll105",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll111",
    "identifier": "http://reg.genome.network/allele/CA501237"
  },
  "wasGeneratedBy": {
    "cg:id": "AssessCrit087",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-04T10:37:00+00:00",
    "endedAtTime": "2016-10-04T10:37:00+00:00",
    "criterion": {
      "cg:id": "BP4",
      "cg:type": "Criterion",
      "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
      "shortDescription": "computational evidence for\nno gene impact",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll105",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll111",
      "identifier": "http://reg.genome.network/allele/CA501237"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess076">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess077</td>
          <td>The evidence here only explicitly describes a single individual with a likely pathogenic variant in TNNT2, but there are numerous other examples that have been observed in internal LMM data.</td>
          <td><div class="example_view" id="CritAssess077"><pre><code>{
  "cg:id": "CritAssess077",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP5",
    "cg:type": "Criterion",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "shortDescription": "alternate cause of disease",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll106",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll112",
    "identifier": "http://reg.genome.network/allele/CA012381"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit088",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-04T11:42:00+00:00",
    "endedAtTime": "2016-10-04T11:42:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndAll241",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv025",
          "cg:type": "Individual",
          "description": "individual #025"
        },
        "primaryAllele": {
          "cg:id": "CanAll106",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll112",
          "identifier": "http://reg.genome.network/allele/CA012381"
        },
        "secondaryAllele": {
          "cg:id": "CanAll107",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll113",
          "identifier": "http://reg.genome.network/allele/CA005304"
        }
      },
      {
        "cg:id": "IndCond242",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv025",
          "cg:type": "Individual",
          "description": "individual #025"
        },
        "condition": [
          {
            "cg:id": "Cond020",
            "cg:type": "Condition",
            "name": "Hypertrophic Cardiomyopathy",
            "code": "217569"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "MenDisVarInterp243",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll106",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll112",
          "identifier": "http://reg.genome.network/allele/CA012381"
        },
        "condition": [
          {
            "cg:id": "Cond020",
            "cg:type": "Condition",
            "name": "Hypertrophic Cardiomyopathy",
            "code": "217569"
          }
        ],
        "clinicalSignificance": "LN:LA26332-9"
      }
    ],
    "criterion": {
      "cg:id": "BP5",
      "cg:type": "Criterion",
      "description": "Variant found in a case with an alternate molecular basis for disease",
      "shortDescription": "alternate cause of disease",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll106",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll112",
      "identifier": "http://reg.genome.network/allele/CA012381"
    },
    "condition": [
      {
        "cg:id": "Cond020",
        "cg:type": "Condition",
        "name": "Hypertrophic Cardiomyopathy",
        "code": "217569"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  },
  "explanation": "The evidence here only explicitly describes a single individual with a likely pathogenic variant in TNNT2, but there are numerous other examples that have been observed in internal LMM data."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess077">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess078</td>
          <td></td>
          <td><div class="example_view" id="CritAssess078"><pre><code>{
  "cg:id": "CritAssess078",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP5",
    "cg:type": "Criterion",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "shortDescription": "alternate cause of disease",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll108",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll114",
    "identifier": "http://reg.genome.network/allele/CA137502"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit089",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-06T08:29:59+00:00",
    "endedAtTime": "2016-10-06T08:29:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndAll244",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv026",
          "cg:type": "Individual",
          "description": "individual #026"
        },
        "primaryAllele": {
          "cg:id": "CanAll108",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll114",
          "identifier": "http://reg.genome.network/allele/CA137502"
        }
      },
      {
        "cg:id": "IndAll245",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv026",
          "cg:type": "Individual",
          "description": "individual #026"
        },
        "primaryAllele": {
          "cg:id": "CanAll109",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll115",
          "identifier": "http://reg.genome.network/allele/CA172240"
        },
        "secondaryAllele": {
          "cg:id": "CanAll110",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll116",
          "identifier": "http://reg.genome.network/allele/CA274460"
        },
        "primaryZygosity": "CG:het",
        "secondaryZygosity": "CG:het",
        "phase": "CG:trans"
      },
      {
        "cg:id": "IndCond246",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv026",
          "cg:type": "Individual",
          "description": "individual #026"
        },
        "condition": [
          {
            "cg:id": "Cond010",
            "cg:type": "Condition",
            "name": "Hearing Loss",
            "code": "87884"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "MenDisVarInterp247",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll109",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll115",
          "identifier": "http://reg.genome.network/allele/CA172240"
        },
        "condition": [
          {
            "cg:id": "Cond010",
            "cg:type": "Condition",
            "name": "Hearing Loss",
            "code": "87884"
          }
        ],
        "clinicalSignificance": "LN:LA26332-9"
      },
      {
        "cg:id": "MenDisVarInterp248",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll110",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll116",
          "identifier": "http://reg.genome.network/allele/CA274460"
        },
        "condition": [
          {
            "cg:id": "Cond010",
            "cg:type": "Condition",
            "name": "Hearing Loss",
            "code": "87884"
          }
        ],
        "clinicalSignificance": "LN:LA26332-9"
      }
    ],
    "criterion": {
      "cg:id": "BP5",
      "cg:type": "Criterion",
      "description": "Variant found in a case with an alternate molecular basis for disease",
      "shortDescription": "alternate cause of disease",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll108",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll114",
      "identifier": "http://reg.genome.network/allele/CA137502"
    },
    "condition": [
      {
        "cg:id": "Cond010",
        "cg:type": "Condition",
        "name": "Hearing Loss",
        "code": "87884"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess078">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess079</td>
          <td></td>
          <td><div class="example_view" id="CritAssess079"><pre><code>{
  "cg:id": "CritAssess079",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP5",
    "cg:type": "Criterion",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "shortDescription": "alternate cause of disease",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll108",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll114",
    "identifier": "http://reg.genome.network/allele/CA137502"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit090",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-06T08:54:00+00:00",
    "endedAtTime": "2016-10-06T08:54:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndAll249",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv027",
          "cg:type": "Individual",
          "description": "individual #027"
        },
        "primaryAllele": {
          "cg:id": "CanAll108",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll114",
          "identifier": "http://reg.genome.network/allele/CA137502"
        },
        "secondaryAllele": {
          "cg:id": "CanAll111",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll117",
          "identifier": "http://reg.genome.network/allele/CA277980"
        },
        "secondaryZygosity": "CG:hom"
      },
      {
        "cg:id": "IndCond250",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv027",
          "cg:type": "Individual",
          "description": "individual #027"
        },
        "condition": [
          {
            "cg:id": "Cond021",
            "cg:type": "Condition",
            "name": "Usher syndrome",
            "code": "886"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "MenDisVarInterp251",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll108",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll114",
          "identifier": "http://reg.genome.network/allele/CA137502"
        },
        "condition": [
          {
            "cg:id": "Cond021",
            "cg:type": "Condition",
            "name": "Usher syndrome",
            "code": "886"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      }
    ],
    "criterion": {
      "cg:id": "BP5",
      "cg:type": "Criterion",
      "description": "Variant found in a case with an alternate molecular basis for disease",
      "shortDescription": "alternate cause of disease",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll108",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll114",
      "identifier": "http://reg.genome.network/allele/CA137502"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess079">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess080</td>
          <td>Variants in COL2A1 are associated with dominant spondyloepiphyseal dysplasia congenita (a primary bone dysplasia). CA112 was identified in a  proband with CA113, which is pathogenic for another primary bone dysplasia, dominant Pseudoachondroplasia. However, this does not provide supporting evidence for CA112 being benign because the combination of a COL2A1 and COMP variant are known to produce a more severe phenotype, Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome, which is the phenotype of the proband.  Because the combination of alleles may produce a qualitatively different phenotype, the bone dysplasia caused by the COMP variant does not provide evidence that the COL2A1 variant is benign for any bone dysplasias in general.
</td>
          <td><div class="example_view" id="CritAssess080"><pre><code>{
  "cg:id": "CritAssess080",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP5",
    "cg:type": "Criterion",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "shortDescription": "alternate cause of disease",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll112",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll118",
    "identifier": "http://reg.genome.network/allele/CA204551"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit091",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-06T09:08:59+00:00",
    "endedAtTime": "2016-10-06T09:08:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "IndCond252",
        "cg:type": "IndividualCondition",
        "individual": {
          "cg:id": "Indiv028",
          "cg:type": "Individual",
          "description": "individual #028"
        },
        "condition": [
          {
            "cg:id": "Cond041",
            "cg:type": "Condition",
            "name": "Pseudoachondroplasia",
            "code": "750"
          }
        ],
        "hasCondition": true
      },
      {
        "cg:id": "IndAll253",
        "cg:type": "IndividualAllele",
        "individual": {
          "cg:id": "Indiv028",
          "cg:type": "Individual",
          "description": "individual #028"
        },
        "primaryAllele": {
          "cg:id": "CanAll112",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll118",
          "identifier": "http://reg.genome.network/allele/CA204551"
        },
        "secondaryAllele": {
          "cg:id": "CanAll113",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll119",
          "identifier": "http://reg.genome.network/allele/CA254703"
        }
      },
      {
        "cg:id": "MenDisVarInterp254",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "canonicalAllele": {
          "cg:id": "CanAll113",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll119",
          "identifier": "http://reg.genome.network/allele/CA254703"
        },
        "condition": [
          {
            "cg:id": "Cond042",
            "cg:type": "Condition",
            "name": "Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome",
            "code": "750"
          }
        ],
        "clinicalSignificance": "LN:LA6668-3"
      },
      {
        "cg:id": "MenDisVarInterp270",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome is caused by a combination of a pathogenic variant in COL2A1 and another in COMP.   So CA113 may be involved, but is not the sole cause.",
        "canonicalAllele": {
          "cg:id": "CanAll113",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll119",
          "identifier": "http://reg.genome.network/allele/CA254703"
        },
        "condition": [
          {
            "cg:id": "Cond041",
            "cg:type": "Condition",
            "name": "Pseudoachondroplasia",
            "code": "750"
          }
        ],
        "clinicalSignificance": "LN:LA26333-7"
      }
    ],
    "criterion": {
      "cg:id": "BP5",
      "cg:type": "Criterion",
      "description": "Variant found in a case with an alternate molecular basis for disease",
      "shortDescription": "alternate cause of disease",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll112",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll118",
      "identifier": "http://reg.genome.network/allele/CA204551"
    },
    "condition": [
      {
        "cg:id": "Cond040",
        "cg:type": "Condition",
        "name": "Primary bone dysplasia",
        "code": "94068"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  },
  "explanation": "Variants in COL2A1 are associated with dominant spondyloepiphyseal dysplasia congenita (a primary bone dysplasia). CA112 was identified in a  proband with CA113, which is pathogenic for another primary bone dysplasia, dominant Pseudoachondroplasia. However, this does not provide supporting evidence for CA112 being benign because the combination of a COL2A1 and COMP variant are known to produce a more severe phenotype, Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome, which is the phenotype of the proband.  Because the combination of alleles may produce a qualitatively different phenotype, the bone dysplasia caused by the COMP variant does not provide evidence that the COL2A1 variant is benign for any bone dysplasias in general.\n"
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess080">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess081</td>
          <td></td>
          <td><div class="example_view" id="CritAssess081"><pre><code>{
  "cg:id": "CritAssess081",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP6",
    "cg:type": "Criterion",
    "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
    "shortDescription": "benign from reputable source without shared data",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll114",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll120"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit092",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-06T09:45:00+00:00",
    "endedAtTime": "2016-10-06T09:45:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp255",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "classified by ENIGMA.",
        "canonicalAllele": {
          "cg:id": "CanAll114",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll120"
        },
        "condition": [
          {
            "cg:id": "Cond036",
            "cg:type": "Condition",
            "name": "Hereditary Breast and Ovarian Cancer",
            "code": "D061325"
          }
        ],
        "clinicalSignificance": "LN:LA6675-8"
      }
    ],
    "criterion": {
      "cg:id": "BP6",
      "cg:type": "Criterion",
      "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
      "shortDescription": "benign from reputable source without shared data",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll114",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll120"
    },
    "condition": [
      {
        "cg:id": "Cond036",
        "cg:type": "Condition",
        "name": "Hereditary Breast and Ovarian Cancer",
        "code": "D061325"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess081">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess082</td>
          <td></td>
          <td><div class="example_view" id="CritAssess082"><pre><code>{
  "cg:id": "CritAssess082",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP6",
    "cg:type": "Criterion",
    "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
    "shortDescription": "benign from reputable source without shared data",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll115",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll121",
    "identifier": "http://reg.genome.network/allele/CA016113"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit093",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-10T08:01:59+00:00",
    "endedAtTime": "2016-10-10T08:01:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp256",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "classified by Sharing Clinical Reports Project.",
        "canonicalAllele": {
          "cg:id": "CanAll115",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll121",
          "identifier": "http://reg.genome.network/allele/CA016113"
        },
        "condition": [
          {
            "cg:id": "Cond043",
            "cg:type": "Condition",
            "name": "Familial Cancer of Breast",
            "code": "114480"
          }
        ],
        "clinicalSignificance": "LN:LA6675-8"
      }
    ],
    "criterion": {
      "cg:id": "BP6",
      "cg:type": "Criterion",
      "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
      "shortDescription": "benign from reputable source without shared data",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll115",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll121",
      "identifier": "http://reg.genome.network/allele/CA016113"
    },
    "condition": [
      {
        "cg:id": "Cond043",
        "cg:type": "Condition",
        "name": "Familial Cancer of Breast",
        "code": "114480"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess082">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess083</td>
          <td></td>
          <td><div class="example_view" id="CritAssess083"><pre><code>{
  "cg:id": "CritAssess083",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP6",
    "cg:type": "Criterion",
    "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
    "shortDescription": "benign from reputable source without shared data",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll116",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll122",
    "identifier": "http://reg.genome.network/allele/CA113946"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit094",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-10T08:03:00+00:00",
    "endedAtTime": "2016-10-10T08:03:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "MenDisVarInterp257",
        "cg:type": "MendelianDiseaseVariantPathogenicityInterpretation",
        "explanation": "classified by OMIM",
        "canonicalAllele": {
          "cg:id": "CanAll116",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll122",
          "identifier": "http://reg.genome.network/allele/CA113946"
        },
        "condition": [
          {
            "cg:id": "Cond044",
            "cg:type": "Condition",
            "name": "Gyrate atrophy of choroid and retina",
            "code": "414"
          }
        ],
        "clinicalSignificance": "LN:LA6675-8"
      }
    ],
    "criterion": {
      "cg:id": "BP6",
      "cg:type": "Criterion",
      "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
      "shortDescription": "benign from reputable source without shared data",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll116",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll122",
      "identifier": "http://reg.genome.network/allele/CA113946"
    },
    "condition": [
      {
        "cg:id": "Cond044",
        "cg:type": "Condition",
        "name": "Gyrate atrophy of choroid and retina",
        "code": "414"
      }
    ],
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess083">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess084</td>
          <td></td>
          <td><div class="example_view" id="CritAssess084"><pre><code>{
  "cg:id": "CritAssess084",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP7",
    "cg:type": "Criterion",
    "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
    "shortDescription": "silent variant no splicing \nimpact  and low conservation",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll117",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll123",
    "identifier": "http://reg.genome.network/allele/CA501232"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit095",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-10T08:03:59+00:00",
    "endedAtTime": "2016-10-10T08:03:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "InSilicoPred258",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll123",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll117",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll123",
            "identifier": "http://reg.genome.network/allele/CA501232"
          },
          "alleleName": "NM_138691.2 (TMC1): c.684C>T"
        },
        "type": "CG:sp",
        "value": "no impact"
      },
      {
        "cg:id": "Conserv259",
        "cg:type": "Conservation",
        "allele": {
          "cg:id": "CanAll117",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll123",
          "identifier": "http://reg.genome.network/allele/CA501232"
        },
        "conserved": true,
        "method": "phyloP",
        "score": -2.297
      },
      {
        "cg:id": "Conserv260",
        "cg:type": "Conservation",
        "allele": {
          "cg:id": "CanAll117",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll123",
          "identifier": "http://reg.genome.network/allele/CA501232"
        },
        "conserved": true,
        "method": "PhastCons",
        "score": 0.039
      },
      {
        "cg:id": "MolCon261",
        "cg:type": "MolecularConsequence",
        "consequence": "SO:0001819",
        "contextualAllele": {
          "cg:id": "CtxAll123",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll117",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll123",
            "identifier": "http://reg.genome.network/allele/CA501232"
          },
          "alleleName": "NM_138691.2 (TMC1): c.684C>T"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP7",
      "cg:type": "Criterion",
      "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
      "shortDescription": "silent variant no splicing \nimpact  and low conservation",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll117",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll123",
      "identifier": "http://reg.genome.network/allele/CA501232"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess084">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess085</td>
          <td></td>
          <td><div class="example_view" id="CritAssess085"><pre><code>{
  "cg:id": "CritAssess085",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP7",
    "cg:type": "Criterion",
    "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
    "shortDescription": "silent variant no splicing \nimpact  and low conservation",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll118",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll124",
    "identifier": "http://reg.genome.network/allele/CA013536"
  },
  "outcome": "CG:P",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit096",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-10T08:04:59+00:00",
    "endedAtTime": "2016-10-10T08:04:59+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "InSilicoPred262",
        "cg:type": "InSilicoPrediction",
        "explanation": "Provenance would show that this comes from Alamut",
        "contextualAllele": {
          "cg:id": "CtxAll124",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll118",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll124",
            "identifier": "http://reg.genome.network/allele/CA013536"
          },
          "alleleName": "NM_000257.2(MYH7):c.327C>T"
        },
        "type": "CG:sp",
        "value": "no impact"
      },
      {
        "cg:id": "Conserv263",
        "cg:type": "Conservation",
        "allele": {
          "cg:id": "CanAll118",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll124",
          "identifier": "http://reg.genome.network/allele/CA013536"
        },
        "conserved": true,
        "method": "phyloP",
        "score": -0.76
      },
      {
        "cg:id": "Conserv264",
        "cg:type": "Conservation",
        "allele": {
          "cg:id": "CanAll118",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll124",
          "identifier": "http://reg.genome.network/allele/CA013536"
        },
        "conserved": true,
        "method": "PhastCons",
        "score": 0.63
      },
      {
        "cg:id": "MolCon265",
        "cg:type": "MolecularConsequence",
        "consequence": "SO:0001819",
        "contextualAllele": {
          "cg:id": "CtxAll124",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll118",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll124",
            "identifier": "http://reg.genome.network/allele/CA013536"
          },
          "alleleName": "NM_000257.2(MYH7):c.327C>T"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP7",
      "cg:type": "Criterion",
      "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
      "shortDescription": "silent variant no splicing \nimpact  and low conservation",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll118",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll124",
      "identifier": "http://reg.genome.network/allele/CA013536"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  }
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess085">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess086</td>
          <td>This variant is a silent variant that does not result in an amino acid change. It is not very conserved and splicing tools suggest no change. However, BP7 does not apply because the variant is located in the first base of exon 4 of RAF1, which is part of the 3’ splice region of the exon. For that reason, we can’t fully trust computational tools. Functional assays would have to confirm that this variant truly does not alter splicing.</td>
          <td><div class="example_view" id="CritAssess086"><pre><code>{
  "cg:id": "CritAssess086",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "BP7",
    "cg:type": "Criterion",
    "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
    "shortDescription": "silent variant no splicing \nimpact  and low conservation",
    "defaultOutcome": "CG:P",
    "targetInterpretation": "LN:LA6675-8"
  },
  "variant": {
    "cg:id": "CanAll119",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll125",
    "identifier": "http://reg.genome.network/allele/CA177682"
  },
  "outcome": "CG:I",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit097",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2016-10-10T08:06:00+00:00",
    "endedAtTime": "2016-10-10T08:06:00+00:00",
    "hasSupportingEvidence": [
      {
        "cg:id": "InSilicoPred266",
        "cg:type": "InSilicoPrediction",
        "contextualAllele": {
          "cg:id": "CtxAll125",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll119",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll125",
            "identifier": "http://reg.genome.network/allele/CA177682"
          },
          "alleleName": "NM_002880.3(RAF1):c.321T>C"
        },
        "type": "CG:sp",
        "value": "no impact"
      },
      {
        "cg:id": "Conserv267",
        "cg:type": "Conservation",
        "allele": {
          "cg:id": "CanAll119",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll125",
          "identifier": "http://reg.genome.network/allele/CA177682"
        },
        "conserved": true,
        "method": "phyloP",
        "score": 0.45
      },
      {
        "cg:id": "Conserv268",
        "cg:type": "Conservation",
        "allele": {
          "cg:id": "CanAll119",
          "cg:type": "CanonicalAllele",
          "preferredCtxAllele": "CtxAll125",
          "identifier": "http://reg.genome.network/allele/CA177682"
        },
        "conserved": true,
        "method": "PhastCons",
        "score": 1.0
      },
      {
        "cg:id": "MolCon269",
        "cg:type": "MolecularConsequence",
        "consequence": "SO:0001819",
        "contextualAllele": {
          "cg:id": "CtxAll125",
          "cg:type": "ContextualAllele",
          "relatedCanonicalAllele": {
            "cg:id": "CanAll119",
            "cg:type": "CanonicalAllele",
            "preferredCtxAllele": "CtxAll125",
            "identifier": "http://reg.genome.network/allele/CA177682"
          },
          "alleleName": "NM_002880.3(RAF1):c.321T>C"
        }
      }
    ],
    "criterion": {
      "cg:id": "BP7",
      "cg:type": "Criterion",
      "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
      "shortDescription": "silent variant no splicing \nimpact  and low conservation",
      "defaultOutcome": "CG:P",
      "targetInterpretation": "LN:LA6675-8"
    },
    "variant": {
      "cg:id": "CanAll119",
      "cg:type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll125",
      "identifier": "http://reg.genome.network/allele/CA177682"
    },
    "wasAssociatedWith": [
      {
        "cg:id": "Agent008",
        "cg:type": "Agent",
        "description": "MDS",
        "name": "Marina DiStefano"
      }
    ]
  },
  "explanation": "This variant is a silent variant that does not result in an amino acid change. It is not very conserved and splicing tools suggest no change. However, BP7 does not apply because the variant is located in the first base of exon 4 of RAF1, which is part of the 3’ splice region of the exon. For that reason, we can’t fully trust computational tools. Functional assays would have to confirm that this variant truly does not alter splicing."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess086">diagram</a></td>
        </tr>

        <tr>
          <td>CritAssess087</td>
          <td>This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).</td>
          <td><div class="example_view" id="CritAssess087"><pre><code>{
  "cg:id": "CritAssess087",
  "cg:type": "CriterionAssessment",
  "criterion": {
    "cg:id": "PVS1",
    "cg:type": "Criterion",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "shortDescription": "Null Variant in known LOF genes",
    "defaultOutcome": "CG:VS",
    "targetInterpretation": "LN:LA6668-3"
  },
  "variant": {
    "cg:id": "CanAll121",
    "cg:type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll127"
  },
  "outcome": "CG:VS",
  "wasGeneratedBy": {
    "cg:id": "AssessCrit098",
    "cg:type": "AssessCriterion",
    "startedAtTime": "2017-01-17T09:11:59+00:00",
    "endedAtTime": "2017-01-17T09:11:59+00:00"
  },
  "explanation": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID: 17186466). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/)."
}
          </code></pre></div></td>
          <td><a href="../../examples/diagram.html#CritAssess087">diagram</a></td>
        </tr>

      </tbody>
    </table>
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("cg:id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("cg:id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../javascripts/jquery.js"></script><script src="../../../javascripts/bootstrap.js"></script><script src="../../../javascripts/jquery.jsonview.js"></script>

  </body>
</html>
