<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

    <link href="../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/model/entities/variant_interpretation.html">0.1</a></li>
                  <li><a href="/development/tech/model/entities/variant_interpretation.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../../../">Interpretation Model</a></li>
<li><a href="../../">Technical Reference</a></li>
<li><a href="../">Model</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../../../images/clingen-doc-logo.png" />
            </li>

            <li><a href="../../../welcome/">Welcome</a></li>
            <li><a href="../../../user/">User Guide</a></li>
            <li class="active"><a href="../../">Technical Reference</a></li><ul><li><a href="../../api/">API</a></li><li><a href="../../examples/">Examples</a></li><li class="active"><a href="../">Model</a></li><ul><li><a href="../activities/">Activity</a></li><li><a href="../agents/">Agent</a></li><li class="active"><a href="./">Entity</a></li><ul><li><a href="criterion_assessment.html">CriterionAssessment</a></li><li><a href="evidence/">Data</a></li><li><a href="evidence_data-source.html">DataSource</a></li><li><a href="publication_status.html">PublicationStatus</a></li><li class="active"><a href="variant_interpretation.html">VariantInterpretation</a></li></ul><li><a href="../secondary/">Secondary Types</a></li></ul><li><a href="../../value_sets/">Value Sets</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>VariantInterpretation</h1>
            <p>Evidence-based conclusion about the pathogenicity of a sequence variant</p>

                <p>Descended from Entity</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>A VariantInterpretation is a statement about the pathogenicity of a given CanonicalAllele, usually for a given Condition.  A VariantInterpretation does not explicitly contain the reasoning and evidence supporting the conclusion, but that information is referenced in the provenance of the VariantInterpretation entity.  That is, the VariantInterpretation is, for example, the statement that a variant is pathogenic, the reasoning for that conclusion is found by following the wasGenerated attribute of the VariantInterpretation.</p>

<p>VariantInterpretations for pathogenic variants will include a Condition for which the conclusion was reached.  However, if the variant is determined to be benign (such as in the case of a very common polymorphism), then the Condition attribute may be unused indicating that the allele is benign for all conditions.  If a Condition is specified for a benign variant, then the conclusion that the allele is benign is limited to that specific condition.</p>

<p>The definition of many Conditions implies a particular mode of inheritance, but exceptions frequently exist.  Therefore, VariantInterpretations that include Condition should also include a modeOfInheritance attribute to explicitly state the modeOfInheritance about which this VariantInterpretation is made.</p>

<p>The criteriaSet describes the framework under which the VariantInterpretation was created.  If this attribute is not used, it indicates that no formal process was used in the creation of the VariantInterpretation.</p>

<p>While the CriterionAssessments and the criteriaSet should be sufficient to understand the reasoning behind a given VariantInterpretation, a human-readable statement of this reasoning may be included in the explanation attribute (inherited from Entity).</p>

<p>VariantInterpretations are generated by a GenerateVariantInterpretation Activity.</p>

<p>VariantInterpretations are frequently updated when new information becomes available, or during periodic review.  In such a case, a new VariantInterpretation is created.  A tie to the original VariantInterpretation can be maintained using the wasDerivedFrom attribute, which indicates the previous version of the current VariantInterpretation.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>variant</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>The variant whose pathogenicity is being described</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>Condition</td>
                <td>0..*</td>
                <td>The condition for which the variant's pathogenicity is described</td>
            </tr>
            <tr>
                <td>interpretationCode</td>
                <td>CodeableConcept</td>
                <td>1..1</td>
                <td>The coded pathogenicity of the variant for the condition</td>
            </tr>
            <tr>
                <td>method</td>
                <td>Method</td>
                <td>0..1</td>
                <td>The  framework that formally defines the methodology used in creating this interpretation.</td>
            </tr>
            <tr>
                <td>wasDerivedFrom</td>
                <td>VariantInterpretation</td>
                <td>0..1</td>
                <td>A previous interpretation from which the current interpretation was derived</td>
            </tr>
            </tbody>
        </table>

    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>wasGeneratedBy</td>
                    <td>Activity</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>The activity that produced this entity</td>
                </tr>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>A human-readable description of the entity</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <div id="example_view"></div>
    <script>
        run_on_load = function () {return $("#example_view").JSONView(
          {"VarInterp001":{"cg:id":"VarInterp001","cg:type":"VariantInterpretation","variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"interpretationCode":"LN:LA6669-1","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"wasDerivedFrom":{"cg:id":"-"},"wasGeneratedBy":{"cg:id":"GenVarInterp032","cg:type":"GenerateVariantInterpretation","when":"2016-07-12T11:00:59+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent007","cg:type":"Agent","description":"CPC","name":"ClinGen Pathogenicity Calculator v1"},"role":"generator"},{"agent":{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"},"role":"generator"}],"used":{"Method":[null],"CriterionAssessment":[{"cg:id":"CritAssess031","cg:type":"CriterionAssessment","criterion":{"cg:id":"PM2","cg:type":"Criterion","description":"Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium","shortDescription":"Absent from controls","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"outcome":"CG:I","wasGeneratedBy":{"cg:id":"AssessCrit031","cg:type":"AssessCriterion","when":"2016-07-01T10:53:00+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"},"role":"assessor"},{"agent":{"cg:id":"Agent005","cg:type":"Agent","description":"CSER","name":"CSER Bake Off Project"},"role":"assessor"}],"used":{"Data":[{"cg:id":"PAF082","cg:type":"PopulationAlleleFrequency","wasGeneratedBy":{"cg:id":"CapData081","cg:type":"CaptureData","when":"2016-06-01T14:15:00+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"},"role":"capturer"}],"used":{"DataSource":[{"cg:id":"DataSrc009","cg:type":"DataSource","externalIdentifier":"EXAC v0.2","linkedIRI":"ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.2/"}]}},"ascertainment":"ExAC","population":"CG:combined","contextualAllele":{"cg:id":"CtxAll039","cg:type":"ContextualAllele","relatedCanonicalAllele":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"alleleName":"NM_001369.2(DNAH5):c.7468_7488del"},"alleleCount":0,"alleleFrequency":0.0,"medianCoverage":11.0}],"Criterion":[{"cg:id":"PM2","cg:type":"Criterion","description":"Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium","shortDescription":"Absent from controls","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"}],"CanonicalAllele":[{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"}]}},"explanation":"while the variant is absent from exac, however\n1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”\n2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC)."},{"cg:id":"CritAssess022","cg:type":"CriterionAssessment","criterion":{"cg:id":"PM3","cg:type":"Criterion","description":"For recessive disorders, detected in trans with a pathogenic variant","shortDescription":"Recessive in trans w/ Path","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"outcome":"CG:M","wasGeneratedBy":{"cg:id":"AssessCrit022","cg:type":"AssessCriterion","when":"2016-06-02T08:00:59+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"},"role":"assessor"}],"used":{"Data":[{"cg:id":"CondInh055","cg:type":"ConditionInheritance","wasGeneratedBy":{"cg:id":"CapData076","cg:type":"CaptureData","when":"2016-05-31T08:45:59+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent008","cg:type":"Agent","description":"MDS","name":"Marina DiStefano"},"role":"capturer"}],"used":{"DataSource":[{"cg:id":"DataSrc004","cg:type":"DataSource","externalIdentifier":"OMIM:244400","linkedIRI":"http://omim.org/entry/244400","version":"alopez : 09/15/2016"}]}},"condition":{"cg:id":"Cond004","cg:type":"Condition","name":"Primary Ciliary Dyskenisia","code":"244"},"modeOfInheritance":"HP:0000007"},{"cg:id":"IndCond056","cg:type":"IndividualCondition","wasGeneratedBy":{"cg:id":"CapData077","cg:type":"CaptureData","when":"2016-05-31T09:14:59+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"},"role":"capturer"}],"used":{"DataSource":[{"cg:id":"DataSrc005","cg:type":"DataSource","externalIdentifier":"LMM-CASE:CSER-SUBJECT-123"}]}},"individual":{"cg:id":"Indiv004","cg:type":"Individual","description":"individual #004"},"condition":{"cg:id":"Cond014","cg:type":"Condition","name":"outer dynein arm (ODA) defects","code":"0200106"},"hasCondition":true},{"cg:id":"MenDisVarInterp057","cg:type":"MendelianDiseaseVariantPathogenicityInterpretation","wasGeneratedBy":{"cg:id":"CapData078","cg:type":"CaptureData","when":"2016-05-31T11:00:59+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent008","cg:type":"Agent","description":"MDS","name":"Marina DiStefano"},"role":"capturer"}],"used":{"DataSource":[{"cg:id":"DataSrc006","cg:type":"DataSource","externalIdentifier":"CLINVAR:SCV000205842.3","linkedIRI":"https://www.ncbi.nlm.nih.gov/clinvar/SCV000205842.3"}]}},"canonicalAllele":{"cg:id":"CanAll036","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll040","identifier":"http://reg.genome.network/allele/CA273615"},"condition":{"cg:id":"Cond004","cg:type":"Condition","name":"Primary Ciliary Dyskenisia","code":"244"},"clinicalSignificance":"LN:LA6668-3"},{"cg:id":"IndAll058","cg:type":"IndividualAllele","wasGeneratedBy":{"cg:id":"CapData077","cg:type":"CaptureData","when":"2016-05-31T09:14:59+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"},"role":"capturer"}],"used":{"DataSource":[{"cg:id":"DataSrc005","cg:type":"DataSource","externalIdentifier":"LMM-CASE:CSER-SUBJECT-123"}]}},"explanation":"Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband.","individual":{"cg:id":"Indiv004","cg:type":"Individual","description":"individual #004"},"primaryAllele":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"secondaryAllele":{"cg:id":"CanAll036","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll040","identifier":"http://reg.genome.network/allele/CA273615"},"primaryZygosity":"CG:het","secondaryZygosity":"CG:het","phase":"CG:trans"}],"Criterion":[{"cg:id":"PM3","cg:type":"Criterion","description":"For recessive disorders, detected in trans with a pathogenic variant","shortDescription":"Recessive in trans w/ Path","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"}],"CanonicalAllele":[{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"}],"Condition":[{"cg:id":"Cond004","cg:type":"Condition","name":"Primary Ciliary Dyskenisia","code":"244"},{"cg:id":"Cond014","cg:type":"Condition","name":"outer dynein arm (ODA) defects","code":"0200106"}]}}},{"cg:id":"CritAssess025","cg:type":"CriterionAssessment","criterion":{"cg:id":"PM4","cg:type":"Criterion","description":"Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants","shortDescription":"Protein length changes or stop-loss","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"},"variant":{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"},"outcome":"CG:M","wasGeneratedBy":{"cg:id":"AssessCrit025","cg:type":"AssessCriterion","when":"2016-06-12T07:30:00+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent003","cg:type":"Agent","description":"TS","name":"Tasha Strande"},"role":"assessor"}],"used":{"Data":[{"cg:id":"MolCon066","cg:type":"MolecularConsequence","wasGeneratedBy":{"cg:id":"CapData079","cg:type":"CaptureData","when":"2016-06-01T14:03:00+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent001","cg:type":"Agent","description":"SH","name":"Steven Harrison"},"role":"capturer"}],"used":{"DataSource":[{"cg:id":"DataSrc007","cg:type":"DataSource","externalIdentifier":"LMM:GENEINSIGHT-LAB v5.3.2","linkedIRI":"https://geneinsight-lmm-test.partners.org"}]}},"contextualAllele":{"cg:id":"CtxAll044","cg:type":"ContextualAllele","relatedCanonicalAllele":{"cg:id":"CanAll040","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll044","identifier":"http://reg.genome.network/allele/CA273136"},"alleleName":"NP_001360.1:p.Trp2490_Leu2496del"},"consequence":"SO:0001825"},{"cg:id":"RegAnno069","cg:type":"RegionAnnotation","wasGeneratedBy":{"cg:id":"CapData080","cg:type":"CaptureData","when":"2016-06-01T14:10:00+00:00","wasAssociatedWith":[{"agent":{"cg:id":"Agent008","cg:type":"Agent","description":"MDS","name":"Marina DiStefano"},"role":"capturer"}],"used":{"DataSource":[{"cg:id":"DataSrc008","cg:type":"DataSource","externalIdentifier":"LMM:ALAMUT-PROD v123","linkedIRI":"https://pcpgm.partners.org/lmm"}]}},"region":{"cg:id":"CtxReg004","cg:type":"ContextualRegion","referenceSequence":{"cg:id":"RefSeq005","cg:type":"ReferenceSequence","identifier":"NP_001360.1"},"start":2490,"stop":2496},"value":true,"type":"CG:repeat"}],"Criterion":[{"cg:id":"PM4","cg:type":"Criterion","description":"Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants","shortDescription":"Protein length changes or stop-loss","method":{"cg:id":"ACMGv2015","cg:type":"Method","description":"ACMG ISV guidelines 2015","url":"https://www.acmg.net/docs/Standards_Guidelines_for_the_Interpretation_of_Sequence_Variants.pdf"},"defaultOutcome":"CG:M","targetInterpretation":"LN:LA6668-3"}],"CanonicalAllele":[{"cg:id":"CanAll040","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll044","identifier":"http://reg.genome.network/allele/CA273136"},{"cg:id":"CanAll041","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll045","identifier":"http://reg.genome.network/allele/CA090938"}]}},"explanation":"leads to an in-frame deletion of 7 amino acids (not repeat region)."}],"Condition":[{"cg:id":"Cond004","cg:type":"Condition","name":"Primary Ciliary Dyskenisia","code":"244"}],"CanonicalAllele":[{"cg:id":"CanAll035","cg:type":"CanonicalAllele","preferredCtxAllele":"CtxAll039","identifier":"http://reg.genome.network/allele/CA090919"}]}},"explanation":"The Trp2490_Leu2496del variant in DNAH5 leads to an in-frame deletion of 7 amino acids. This variant has been reported together with a second DNAH5 variant (Met2083Ile) in one individual with PCD and situs inversus (Berg 2011). In addition, this variant has been identified in trans configuration with a disease-causing variant in one affected proband (LMM unpublished data). Data from large population studies is insufficient to determine whether this variant is present in the general population. In summary, this variant is likely pathogenic, though additional studies are required to fully establish its clinical significance."}},
          { collapsed: true }
        )}
    </script>


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../javascripts/jquery.js"></script><script src="../../../javascripts/bootstrap.js"></script><script src="../../../javascripts/jquery.jsonview.js"></script><script src="../../../javascripts/run-run_on_load.js"></script>

  </body>
</html>
