<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a class="navbar-brand" href="/">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../../../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a target="_blank" href="http://www.clinicalgenome.org"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>
    


    
        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">AggregateSegregation</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../../../">Interpretation Model</a></li>
<li><a href="../../">Technical Reference</a></li>
<li><a href="../">Entity Definitions</a></li>
<li><a href="./">Evidence Item</a></li>

              </ol>
                <blockquote><p>A statement describing the correlation between the presense of a particular allele and the manifestation of a disease or phenotypes, as determined by examining inheritance patterns across multiple families (i.e. co-segregation of an allele and condition, aggregated across multiple family pedigrees).</p>
                <STRONG>
                  <div>SEPIO:0000248</div>
                      <i>Descended from Statement</i>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>Co-segregation of alleles and conditions is evaluated per-family, but may then be aggregated across families.  This Data type represents such an aggregation.  Instances may refer to FamilySegregationData that represent the per-family segregation results that are aggregated together, but this is not required.</p>

<p>AggregateSegregationData requires a particular CanonicalAllele and Condition, and it is the co-segregation of these entities that is described.   In addition, a modeOfInheritance may be given, indicating the inheritance pattern to be used for interpreting the segregations.</p>

<p>AggregateSegregationData also provides numerous attributes summarizing data across families, counting the numbers of segregations and sets of individuals in the families, based on their allele and condition status.</p>

<p>If LOD scores are used in the family segregations, these LODS may be combined to a total LOD score as well, though this is not required.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>canonicalAllele</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>Allele being tested for cosegregation</td>
                <td>SEPIO:0000275</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>GeneticCondition</td>
                <td>1..1</td>
                <td>Condition being tested for cosegregation</td>
                <td>SEPIO:0000276</td>
            </tr>
            <tr>
                <td>totalNumberOfSegregations</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Total number of segregations across families</td>
                <td>SEPIO:0000288</td>
            </tr>
            <tr>
                <td>familySegregation</td>
                <td>FamilySegregation</td>
                <td>0..*</td>
                <td>List of FamilySegregrationData that have been aggregated</td>
                <td>SEPIO:0000287</td>
            </tr>
            <tr>
                <td>modeOfInheritance</td>
                <td>ModeOfInheritance</td>
                <td>0..1</td>
                <td>Mode Of Inheritance used in evaluation of segregation</td>
                <td></td>
            </tr>
            <tr>
                <td>totalOfAllelePosConditionPosIndividuals</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals across families that have the allele and the condition</td>
                <td>SEPIO:0000292</td>
            </tr>
            <tr>
                <td>totalOfAlleleNegConditionNegIndividuals</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals across families that lack both the allele and the condition</td>
                <td>SEPIO:0000289</td>
            </tr>
            <tr>
                <td>totalOfAllelePosConditionNegIndividuals</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals across families that have the allele but not the condition</td>
                <td>SEPIO:0000291</td>
            </tr>
            <tr>
                <td>totalOfAlleleNegConditionPosIndividuals</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals across families that do not have the allele but do have the condition</td>
                <td>SEPIO:0000290</td>
            </tr>
            <tr>
                <td>totalOfUntestedConditionPosIndividuals</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals across families that have the condition but whose allele is untested</td>
                <td>SEPIO:0000295</td>
            </tr>
            <tr>
                <td>totalOfUntestedConditionNegIndividuals</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals across families that do not have the condition but whose allele is untested</td>
                <td>SEPIO:0000294</td>
            </tr>
            <tr>
                <td>totalOfFamiliesWithInconsistentSegregations</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of families containing segregations that are inconsistent with the allele causing the condition</td>
                <td>SEPIO:0000293</td>
            </tr>
            </tbody>
        </table>

<!--
    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>qualifier</td>
                    <td>string</td>
                    <td>0..1</td>
                    <td>Use “NOT” as the value of this property to assert that the statement is negated.</td>
                    <td>SEPIO:0000346</td>
                </tr>
                <tr>
                    <td>evidenceLine</td>
                    <td>EvidenceLine</td>
                    <td>0..*</td>
                    <td>supporting evidence</td>
                    <td>SEPIO:0000006</td>
                </tr>
                <tr>
                    <td>description</td>
                    <td>string</td>
                    <td>0..1</td>
                    <td>A human-readable description of the entity</td>
                    <td>DC:description</td>
                </tr>
                <tr>
                    <td>contribution</td>
                    <td>Contribution</td>
                    <td>0..*</td>
                    <td>The contributions (agent, time and role) made to this entity</td>
                    <td>SEPIO:0000159</td>
                </tr>
                <tr>
                    <td>source</td>
                    <td>string</td>
                    <td>0..*</td>
                    <td>A string indicating the source of a Statement</td>
                    <td>DC:source</td>
                </tr>
            </tbody>
        </table>
-->

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>qualifier</th>
        <th>evidenceLine</th>
        <th>description</th>
        <th>canonicalAllele</th>
        <th>condition</th>
        <th>totalNumberOfSegregations</th>
        <th>familySegregation</th>
        <th>modeOfInheritance</th>
        <th>totalOfAllelePosConditionPosIndividuals</th>
        <th>totalOfAlleleNegConditionNegIndividuals</th>
        <th>totalOfAllelePosConditionNegIndividuals</th>
        <th>totalOfAlleleNegConditionPosIndividuals</th>
        <th>totalOfUntestedConditionPosIndividuals</th>
        <th>totalOfUntestedConditionNegIndividuals</th>
        <th>totalOfFamiliesWithInconsistentSegregations</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../../details/details.html#CG-EX:AggSeg105">CG-EX:AggSeg105</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA010136">CAR:CA010136</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CG-EX:GenCond020">CG-EX:GenCond020</a></p>
         </td>
          <td>
              <p>4</p>
         </td>
          <td>
              <p><a href="../../details/details.html#CG-EX:FamSeg103">CG-EX:FamSeg103</a></p><p><a href="../../details/details.html#CG-EX:FamSeg104">CG-EX:FamSeg104</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CG-EX:AggSeg105"><pre><code>{
  "id": "CG-EX:AggSeg105",
  "type": "AggregateSegregation",
  "condition": {
    "id": "CG-EX:GenCond020",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "Orphanet:217569",
        "type": "Disease",
        "label": "hypertrophic cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CAR:CA010136",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CG-EX:CtxAll067",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll063",
      "alleleName": "NM_000256.3(MYBPC3)c.1351+1G>A"
    }
  },
  "familySegregation": [
    {
      "id": "CG-EX:FamSeg103",
      "type": "FamilySegregation",
      "condition": {
        "id": "CG-EX:GenCond020",
        "type": "GeneticCondition",
        "disease": [
          {
            "id": "Orphanet:217569",
            "type": "Disease",
            "label": "hypertrophic cardiomyopathy"
          }
        ],
        "modeOfInheritance": {
          "id": "HP:0000006",
          "type": "ModeOfInheritance",
          "label": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "id": "CAR:CA010136",
        "type": "CanonicalAllele",
        "preferredCtxAllele": {
          "id": "CG-EX:CtxAll067",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll063",
          "alleleName": "NM_000256.3(MYBPC3)c.1351+1G>A"
        }
      },
      "family": {
        "id": "_b8500",
        "type": "Family",
        "label": "12A",
        "description": "Family #001"
      },
      "phenotypePositiveAllelePositive": 3,
      "phenotypeNegativeAllelePositive": 2,
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CG-EX:Agent007",
            "type": "Agent",
            "label": "Stefan DiMarino"
          },
          "contributionDate": "9/13/16",
          "contributionRole": {
            "id": "SEPIO:0000156",
            "type": "ContributoryRole",
            "label": "curator role"
          }
        }
      ],
      "source": [
        "http://www.ncbi.nlm.nih.gov/pubmed/17081393"
      ],
      "description": "identified in 1 individual with HCM. Segregation analysis determined 5 other members of the family also carried this variant. However, only 3 of those family members had HCM (aged 48-75). The 2 other family members were young and asymptomatic. So from this paper, only count 3 segregations. The two apparently refuting segregations can be ignored due to the age-dependent penetrance of HCM."
    },
    {
      "id": "CG-EX:FamSeg104",
      "type": "FamilySegregation",
      "condition": {
        "id": "CG-EX:GenCond020",
        "type": "GeneticCondition",
        "disease": [
          {
            "id": "Orphanet:217569",
            "type": "Disease",
            "label": "hypertrophic cardiomyopathy"
          }
        ],
        "modeOfInheritance": {
          "id": "HP:0000006",
          "type": "ModeOfInheritance",
          "label": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "id": "CAR:CA010136",
        "type": "CanonicalAllele",
        "preferredCtxAllele": {
          "id": "CG-EX:CtxAll067",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll063",
          "alleleName": "NM_000256.3(MYBPC3)c.1351+1G>A"
        }
      },
      "family": {
        "id": "_b8501",
        "type": "Family",
        "description": "Family #002"
      },
      "phenotypePositiveAllelePositive": 1,
      "contribution": [
        {
          "type": "Contribution",
          "agent": {
            "id": "CG-EX:Agent001",
            "type": "Agent",
            "label": "Harry Stevenson"
          },
          "contributionDate": "9/13/16",
          "contributionRole": {
            "id": "SEPIO:0000156",
            "type": "ContributoryRole",
            "label": "curator role"
          }
        }
      ],
      "source": [
        "http://partners.org/ppm/lmm/gi/variant/12345"
      ],
      "description": "In LMM internal family, c.1351+1G>A was identified in 1 individual with HCM. Variant also found in individual’s sister - who also had clinical dx of HCM."
    }
  ],
  "totalNumberOfSegregations": 4
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CG-EX:AggSeg109">CG-EX:AggSeg109</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA143213">CAR:CA143213</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CG-EX:GenCond021">CG-EX:GenCond021</a></p>
         </td>
          <td>
              <p>8</p>
         </td>
          <td>
              <p><a href="../../details/details.html#CG-EX:FamSeg107">CG-EX:FamSeg107</a></p><p><a href="../../details/details.html#CG-EX:FamSeg108">CG-EX:FamSeg108</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CG-EX:AggSeg109"><pre><code>{
  "id": "CG-EX:AggSeg109",
  "type": "AggregateSegregation",
  "condition": {
    "id": "CG-EX:GenCond021",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "Orphanet:886",
        "type": "Disease",
        "label": "Usher syndrome"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000007",
      "type": "ModeOfInheritance",
      "label": "Autosomal Recessive"
    }
  },
  "canonicalAllele": {
    "id": "CAR:CA143213",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CG-EX:CtxAll066",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll062",
      "alleleName": "NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)"
    }
  },
  "familySegregation": [
    {
      "id": "CG-EX:FamSeg107",
      "type": "FamilySegregation",
      "condition": {
        "id": "CG-EX:GenCond021",
        "type": "GeneticCondition",
        "disease": [
          {
            "id": "Orphanet:886",
            "type": "Disease",
            "label": "Usher syndrome"
          }
        ],
        "modeOfInheritance": {
          "id": "HP:0000007",
          "type": "ModeOfInheritance",
          "label": "Autosomal Recessive"
        }
      },
      "canonicalAllele": {
        "id": "CAR:CA143213",
        "type": "CanonicalAllele",
        "preferredCtxAllele": {
          "id": "CG-EX:CtxAll066",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll062",
          "alleleName": "NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)"
        }
      },
      "family": {
        "id": "_b8502",
        "type": "Family",
        "description": "Family #003"
      },
      "phenotypePositiveAllelePositive": 5,
      "description": "proband and 5 affected family members all homozygous for NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)."
    },
    {
      "id": "CG-EX:FamSeg108",
      "type": "FamilySegregation",
      "condition": {
        "id": "CG-EX:GenCond021",
        "type": "GeneticCondition",
        "disease": [
          {
            "id": "Orphanet:886",
            "type": "Disease",
            "label": "Usher syndrome"
          }
        ],
        "modeOfInheritance": {
          "id": "HP:0000007",
          "type": "ModeOfInheritance",
          "label": "Autosomal Recessive"
        }
      },
      "canonicalAllele": {
        "id": "CAR:CA143213",
        "type": "CanonicalAllele",
        "preferredCtxAllele": {
          "id": "CG-EX:CtxAll066",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll062",
          "alleleName": "NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)"
        }
      },
      "family": {
        "id": "_b8503",
        "type": "Family",
        "description": "Family #004"
      },
      "description": "c.1000C>T identified in het state in 1 individual with Usher. Individual compound het with Gly1840Val USH2A variant - confirmed in trans. Gly1840Val called Pathogenic. Individual has 3 affected sibs all of whom are compound het with 1000C>T & Gly1840Val. One additional sib only carried 1000C>T and is unaffected."
    }
  ],
  "totalNumberOfSegregations": 8
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CG-EX:AggSeg115">CG-EX:AggSeg115</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CAR:CA006429">CAR:CA006429</a></p>
         </td>
          <td>
              <p><a href="../../details/details.html#CG-EX:GenCond002">CG-EX:GenCond002</a></p>
         </td>
          <td>
              <p>9</p>
         </td>
          <td>
              <p><a href="../../details/details.html#CG-EX:FamSeg113">CG-EX:FamSeg113</a></p><p><a href="../../details/details.html#CG-EX:FamSeg114">CG-EX:FamSeg114</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CG-EX:AggSeg115"><pre><code>{
  "id": "CG-EX:AggSeg115",
  "type": "AggregateSegregation",
  "canonicalAllele": {
    "id": "CAR:CA006429",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CG-EX:CtxAll069",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll065",
      "alleleName": "NM_001001430.2(TNNT2):c.629_631delAGA"
    }
  },
  "condition": {
    "id": "CG-EX:GenCond002",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "Orphanet:217604",
        "type": "Disease",
        "label": "dilated cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "familySegregation": [
    {
      "id": "CG-EX:FamSeg113",
      "type": "FamilySegregation",
      "condition": {
        "id": "CG-EX:GenCond002",
        "type": "GeneticCondition",
        "disease": [
          {
            "id": "Orphanet:217604",
            "type": "Disease",
            "label": "dilated cardiomyopathy"
          }
        ],
        "modeOfInheritance": {
          "id": "HP:0000006",
          "type": "ModeOfInheritance",
          "label": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "id": "CAR:CA006429",
        "type": "CanonicalAllele",
        "preferredCtxAllele": {
          "id": "CG-EX:CtxAll069",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll065",
          "alleleName": "NM_001001430.2(TNNT2):c.629_631delAGA"
        }
      },
      "family": {
        "id": "_b8505",
        "type": "Family",
        "description": "Family #006"
      },
      "phenotypePositiveAllelePositive": 1,
      "phenotypeNegativeAllelePositive": 1,
      "columns": [
        "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”]"
      ],
      "affectedValues": [
        "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
      ],
      "genotypeValues": [
        "{0: “Unknown”,1:”Reference”, 2: CA65}"
      ],
      "pedigree": [
        "[[Fam006,Fam006.1,0,0,1,2,0/0],\n   [Fam006,Fam006.2,0,0,2,1,0/0],\n   [Fam006,Fam006.3,Fam006.1,Fam006.2,2,3,0/0],\n   [Fam006,Fam006.4,Fam006.1,Fam006.2,1,2,1/2],\n   [Fam006,Fam006.5,Fam006.1,Fam006.2,2,1,1/1],\n   [Fam006,Fam006.6,Fam006.1,Fam006.2,1,2,1/2]]"
      ]
    },
    {
      "id": "CG-EX:FamSeg114",
      "type": "FamilySegregation",
      "condition": {
        "id": "CG-EX:GenCond002",
        "type": "GeneticCondition",
        "disease": [
          {
            "id": "Orphanet:217604",
            "type": "Disease",
            "label": "dilated cardiomyopathy"
          }
        ],
        "modeOfInheritance": {
          "id": "HP:0000006",
          "type": "ModeOfInheritance",
          "label": "Autosomal Dominant"
        }
      },
      "canonicalAllele": {
        "id": "CAR:CA006429",
        "type": "CanonicalAllele",
        "preferredCtxAllele": {
          "id": "CG-EX:CtxAll069",
          "type": "ContextualAllele",
          "relatedCanonicalAllele": "CanAll065",
          "alleleName": "NM_001001430.2(TNNT2):c.629_631delAGA"
        }
      },
      "family": {
        "id": "_b8506",
        "type": "Family",
        "description": "Family #007"
      },
      "phenotypePositiveAllelePositive": 2,
      "phenotypeNegativeAllelePositive": 3,
      "columns": [
        "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”,”ObligateGenotype”]"
      ],
      "affectedValues": [
        "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
      ],
      "genotypeValues": [
        "{0: “Unknown”,1:”Reference”, 2: CA65}"
      ],
      "pedigree": [
        "[[Fam007, Fam007.1, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.2, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.3, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.4, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.5, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.6, Fam007.1, Fam007.2, 2, 3, 1/2, 1],\n   [Fam007, Fam007.7,0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.8, Fam007.1, Fam007.2, 2, 1, 1/2, 1],\n   [Fam007, Fam007.9,Fam007.3, Fam007.4, 1, 1, 0/0, 0],\n   [Fam007, Fam007.10, 0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.11, Fam007.5, Fam007.6, 2, 2, 1/2, 0],\n   [Fam007, Fam007.12, Fam007.5, Fam007.6, 1,2, 0/0, 0],\n   [Fam007, Fam007.13, Fam007.5, Fam007.6, 2,1, 1/1,0],\n   [Fam007, Fam007.14, Fam007.5, Fam007.6, 1, 1, 1/1, 0],\n   [Fam007, Fam007.15, Fam007.7, Fam007.8, 1, 2, 1/2, 0],\n   [Fam007, Fam007.16, Fam007.8, Fam007.9, 1, 2, 1/2, 0],\n   [Fam007, Fam007.17, Fam007.8, Fam007.9, 1, 2, 0/0,0],\n   [Fam007, Fam007.18, Fam007.8, Fam007.9, 1, 2, 0/0, 0],\n   [Fam007, Fam007.19, Fam007.10, Fam007.11, 1, 2, 0/0,0],\n   [Fam007.20,Fam007.10, Fam007.11, 1,1,1/1,0]]"
      ]
    }
  ],
  "totalNumberOfSegregations": 9
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../../../">Interpretation Overview</a></h5>
              </li>
              <li class='list-group-item'><h5><a href="../../../user/">User Guide</a></h5></li>
              <li class="list-group-item"><h5><a href="../../">Technical Reference</a></h5><ul class='list-unstyled'><li class="active"><a href="../">Entity Definitions</a></li><ul class='list-unstyled'><li><a href="../core/">Core</a></li><li><a href="../domain_entities/">Domain Entities</a></li><li class="active"><a href="./">Evidence Item</a></li><ul class='list-unstyled'><li class="active"><a href="aggregate_segregation.html">AggregateSegregation</a></li><li><a href="allele_conservation.html">AlleleConservation</a></li><li><a href="allele_conservation_score.html">AlleleConservationScore</a></li><li><a href="allele_frequency.html">AlleleFrequency</a></li><li><a href="allele_functional_impact.html">AlleleFunctionalImpact</a></li><li><a href="allele_molecular_consequence.html">AlleleMolecularConsequence</a></li><li><a href="benign_missense_variation_rate.html">BenignMissenseVariationRate</a></li><li><a href="case_control.html">CaseControl</a></li><li><a href="condition_mechanism.html">ConditionMechanism</a></li><li><a href="condition_penetrance.html">ConditionPenetrance</a></li><li><a href="condition_prevelance.html">ConditionPrevelance</a></li><li><a href="excluded_region_type.html">Haplotype</a></li><li><a href="family_condition.html">FamilyCondition</a></li><li><a href="family_segregation.html">FamilySegregation</a></li><li><a href="in_silico_prediction.html">InSilicoPrediction</a></li><li><a href="in_silico_prediction_score.html">InSilicoPredictionScore</a></li><li><a href="individual_allele_inheritance.html">IndividualAlleleInheritance</a></li><li><a href="individual_condition.html">IndividualCondition</a></li><li><a href="individual_genotype.html">IndividualGenotype</a></li><li><a href="locus_heterogeneity.html">LocusHeterogeneity</a></li><li><a href="null_allele.html">NullAllele</a></li><li><a href="region_alleles.html">RegionAlleles</a></li><li><a href="region_type.html">RegionType</a></li></ul><li><a href="../provenance/">Provenance</a></li><li><a href="../terminology/">Terminology</a></li></ul><li><a href="../../json_schema.html">JSON Schema</a></li></ul></li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2017 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../javascripts/jquery.js"></script><script src="../../../javascripts/bootstrap.js"></script><script src="../../../javascripts/jquery.jsonview.js"></script><script src="../../../javascripts/all.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
