<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a class="navbar-brand" href="/">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../../../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a target="_blank" href="http://www.clinicalgenome.org"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>
    


    
        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">BenignMissenseVariationRateStatement</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../../../">Variant Pathogenicity Interpretation Model</a></li>
<li><a href="../../">Technical Reference</a></li>
<li><a href="../">Entity Definitions</a></li>
<li><a href="./">Evidence Item</a></li>

              </ol>
                <blockquote><p>A statement about the frequency at which missense variants in a particular region are tolerated (i.e. do not cause a pathological condition)</p>
                <STRONG>
                  <div>SEPIO:0000246</div>
                      <i>Descended from Statement</i>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>If a gene, or a region within a gene, has a low rate of benign missense alleles, then missense alleles in that gene or region are less likely to be benign.  This Data represents the rate at which benign missense alleles occur.</p>

<p>The unit over which the benign missense rate is defined may either be an entire Gene, or a portion of a gene as defined by a ContextualRegion.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
                <th>Defined in</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>region</td>
                <td>SequenceLocation</td>
                <td>0..1</td>
                <td>Sequence on which the region is defined</td>
                <td>SEPIO:0000279</td>
                <td>BenignMissenseVariationRateStatement</td>
            </tr>
            <tr>
                <td>gene</td>
                <td>@id</td>
                <td>0..1</td>
                <td>Gene that contains the missense variants</td>
                <td>SEPIO:0000278</td>
                <td>BenignMissenseVariationRateStatement</td>
            </tr>
            <tr>
                <td>statementOutcome</td>
                <td>@id</td>
                <td>1..1</td>
                <td>The rate at which Benign missense variants occur within the region</td>
                <td>SEPIO:0000197</td>
                <td>BenignMissenseVariationRateStatement</td>
            </tr>
            <tr>
                <td>label</td>
                <td>string</td>
                <td>0..1</td>
                <td>A name given to the resource.</td>
                <td>RDFS:label</td>
                <td>Entity</td>
            </tr>
            <tr>
                <td>description</td>
                <td>string</td>
                <td>0..1</td>
                <td>Description may include but is not limited to: an abstract, a table of contents, a graphical representation, or a free-text account of the resource.</td>
                <td>DC:description</td>
                <td>Entity</td>
            </tr>
            <tr>
                <td>userLabelDictionary</td>
                <td>UserLabel</td>
                <td>0..*</td>
                <td>An optional label defined by the user. Used for custom entities or to clarify the preferred user label on existing entities with non-preferred labels.</td>
                <td>SEPIO:0000422</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>outcomeQualifier</td>
                <td>string</td>
                <td>0..1</td>
                <td>Use “NOT” as the value of this property to assert that the statement is negated.</td>
                <td>SEPIO:0000346</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>evidenceLine</td>
                <td>EvidenceLine</td>
                <td>0..*</td>
                <td>supporting evidence</td>
                <td>SEPIO:0000006</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>contribution</td>
                <td>Contribution</td>
                <td>0..*</td>
                <td>The contributions (agent, time and role) made to this entity</td>
                <td>SEPIO:0000159</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>source</td>
                <td>string</td>
                <td>0..*</td>
                <td>A string indicating the source of a Statement</td>
                <td>DC:source</td>
                <td>Statement</td>
            </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>label</th>
        <th>description</th>
        <th>userLabelDictionary</th>
        <th>outcomeQualifier</th>
        <th>evidenceLine</th>
        <th>region</th>
        <th>gene</th>
        <th>statementOutcome</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../../details/details.html#CGEX:BMVR161">CGEX:BMVR161</a></p></td>
          <td>
              
         </td>
          <td>
              <p>In LMM data alone, we have over 150 benign missense variants in TTN.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#HGNC:12403">HGNC:12403</a></p> (TTN)
         </td>
          <td>
              <p><a href="../../details/details.html#SEPIO:0000265">SEPIO:0000265</a></p> (High)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:BMVR161"><pre><code>{
  "id": "CGEX:BMVR161",
  "type": "BenignMissenseVariationRateStatement",
  "description": "In LMM data alone, we have over 150 benign missense variants in TTN.",
  "gene": {
    "id": "HGNC:12403",
    "label": "TTN"
  },
  "statementOutcome": {
    "id": "SEPIO:0000265",
    "label": "High"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:BMVR119">CGEX:BMVR119</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Based on the gene size, ExAC expects 740 missense variants to be observed, but actually observes only 420, giving a z score of 6.5, which is in the top 0.02% of genes.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#HGNC:7577">HGNC:7577</a></p> (MYH7)
         </td>
          <td>
              <p><a href="../../details/details.html#SEPIO:0000266">SEPIO:0000266</a></p> (Low)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:BMVR119"><pre><code>{
  "id": "CGEX:BMVR119",
  "type": "BenignMissenseVariationRateStatement",
  "description": "Based on the gene size, ExAC expects 740 missense variants to be observed, but actually observes only 420, giving a z score of 6.5, which is in the top 0.02% of genes.",
  "gene": {
    "id": "HGNC:7577",
    "label": "MYH7"
  },
  "statementOutcome": {
    "id": "SEPIO:0000266",
    "label": "Low"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../../details/details.html#CGEX:BMVR102">CGEX:BMVR102</a></p></td>
          <td>
              
         </td>
          <td>
              <p>ClinVar shows no benign missense variants in (or near) this region.  The closest is NM_000169.2:c.8T>C</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#CGEX:Loc434">CGEX:Loc434</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../../details/details.html#SEPIO:0000266">SEPIO:0000266</a></p> (Low)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:BMVR102"><pre><code>{
  "id": "CGEX:BMVR102",
  "type": "BenignMissenseVariationRateStatement",
  "description": "ClinVar shows no benign missense variants in (or near) this region.  The closest is NM_000169.2:c.8T>C",
  "region": {
    "id": "CGEX:Loc434",
    "type": "SequenceLocation",
    "referenceSequence": {
      "id": "REFSEQ:NM_000169.2",
      "label": "NM_000169.2"
    },
    "start": 996,
    "stop": 1012
  },
  "statementOutcome": {
    "id": "SEPIO:0000266",
    "label": "Low"
  }
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../../../">Interpretation Overview</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../../../getting_started.html">Getting Started</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../../../deeper_dive.html">A Deeper Dive</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../../../faq.html">FAQ</a></h5>
              </li>
              <li class="list-group-item"><h5><a href="../../">Entities</a></h5><ul class='list-unstyled'><li class="active"><a href="../">Entity Definitions</a></li><ul class='list-unstyled'><li><a href="../core/">Core</a></li><li><a href="../domain_entities/">Domain Entities</a></li><li class="active"><a href="./">Evidence Item</a></li><ul class='list-unstyled'><li><a href="aggregate_segregation.html">AggregateCosegregationStatement</a></li><li><a href="allele_conservation.html">AlleleConservationStatement</a></li><li><a href="allele_conservation_score.html">AlleleConservationScoreStatement</a></li><li><a href="allele_frequency.html">PopulationAlleleFrequencyStatement</a></li><li><a href="allele_functional_impact.html">AlleleFunctionalImpactStatement</a></li><li><a href="allele_molecular_consequence.html">AlleleMolecularConsequenceStatement</a></li><li class="active"><a href="benign_missense_variation_rate.html">BenignMissenseVariationRateStatement</a></li><li><a href="case_control.html">CaseControlAlleleFrequencyStatement</a></li><li><a href="condition_mechanism.html">ConditionMechanismStatement</a></li><li><a href="condition_penetrance.html">ConditionPenetranceStatement</a></li><li><a href="condition_prevelance.html">ConditionPrevelanceStatement</a></li><li><a href="excluded_region_type.html">Haplotype</a></li><li><a href="family_condition.html">FamilyConditionStatement</a></li><li><a href="family_segregation.html">FamilyCosegregationStatement</a></li><li><a href="in_silico_prediction.html">InSilicoPredictionStatement</a></li><li><a href="in_silico_prediction_score.html">InSilicoPredictionScoreStatement</a></li><li><a href="individual_allele_inheritance.html">IndividualAlleleInheritanceStatement</a></li><li><a href="individual_condition.html">IndividualConditionStatement</a></li><li><a href="individual_genotype.html">IndividualGenotypeStatement</a></li><li><a href="locus_heterogeneity.html">LocusHeterogeneityStatement</a></li><li><a href="null_allele.html">NullAlleleStatement</a></li><li><a href="region_alleles.html">RegionAllelesStatement</a></li><li><a href="region_type.html">RegionTypeStatement</a></li></ul><li><a href="../provenance/">Provenance</a></li><li><a href="../terminology/">Terminology</a></li></ul><li><a href="../../json_schema.html">JSON Schema</a></li></ul></li>
              <li class='list-group-item'><h5><a href="../../../terminology/">Terminology</a></h5></li>
              <li class="list-group-item">
                <h5><a href="../../../projects.html">Projects and Tools</a></h5>
              </li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2017 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../javascripts/jquery.js"></script><script src="../../../javascripts/bootstrap.js"></script><script src="../../../javascripts/jquery.jsonview.js"></script><script src="../../../javascripts/all.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
