<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a class="navbar-brand" href="/">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a target="_blank" href="http://www.clinicalgenome.org"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>
    


    
        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">CaseControl</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../../">Interpretation Model</a></li>
<li><a href="../">Technical Reference and Examples</a></li>
<li><a href="./">Domain Evidence</a></li>

              </ol>
                <blockquote><p>A statement describing allele frequencies identified in a particular study group as a function of case-control status (i.e frequency in affected vs unaffected study groups)</p>
                <STRONG>
                  <div>SEPIO:0000242</div>
                      <i>Descended from Statement</i>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>CaseControl represents data relating the frequency of a given CanonicalAllele to the case-control status for a given Condition.   The frequencies of the allele in these groups is represented by GroupAlleleFrequency elements.   Because PopulationAlleleFrequency descends from GroupAlleleFrequency, it may also be used, usually for the control group frequency.</p>

<p>Results of case control statistical analysis are presented as an odds ratio with a confidence level indicating the degree of confidence that the odds ratio is between the given upper and lower bounds.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>canonicalAllele</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>The allele studied in the case-control experiment</td>
                <td>SEPIO:0000275</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>GeneticCondition</td>
                <td>1..1</td>
                <td>The condition studied in the case-control experiment</td>
                <td>SEPIO:0000276</td>
            </tr>
            <tr>
                <td>caseGroupFrequency</td>
                <td>AlleleFrequency</td>
                <td>0..1</td>
                <td>The group of individuals with the condition</td>
                <td>SEPIO:0000304</td>
            </tr>
            <tr>
                <td>controlGroupFrequency</td>
                <td>AlleleFrequency</td>
                <td>0..1</td>
                <td>The group of individuals without the condition</td>
                <td>SEPIO:0000305</td>
            </tr>
            <tr>
                <td>oddsRatio</td>
                <td>number</td>
                <td>0..1</td>
                <td>Ratio of the probability of having the condition given the allele to the probability of the condition without the allele.</td>
                <td>SEPIO:0000309</td>
            </tr>
            <tr>
                <td>confidenceLevel</td>
                <td>number</td>
                <td>0..1</td>
                <td>Confidence level in percent</td>
                <td>SEPIO:0000308</td>
            </tr>
            <tr>
                <td>confidenceIntervalLower</td>
                <td>number</td>
                <td>0..1</td>
                <td>Lower bound of the confidence interval for the odds ratio</td>
                <td>SEPIO:0000306</td>
            </tr>
            <tr>
                <td>confidenceIntervalUpper</td>
                <td>number</td>
                <td>0..1</td>
                <td>Upper bound of the confidence interval for the odds ratio</td>
                <td>SEPIO:0000307</td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>qualifier</td>
                    <td>string</td>
                    <td>0..1</td>
                    <td></td>
                    <td>Use “NOT” as the value of this property to assert that the statement is negated.</td>
                    <td>??</td>
                </tr>
                <tr>
                    <td>evidenceLine</td>
                    <td>EvidenceLine</td>
                    <td>0..*</td>
                    <td></td>
                    <td>supporting evidence</td>
                    <td>SEPIO:0000006</td>
                </tr>
                <tr>
                    <td>description</td>
                    <td>string</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A human-readable description of the entity</td>
                    <td>DC:description</td>
                </tr>
                <tr>
                    <td>contribution</td>
                    <td>Contribution</td>
                    <td>0..*</td>
                    <td></td>
                    <td>The contributions (agent, time and role) made to this entity</td>
                    <td>SEPIO:0000159</td>
                </tr>
                <tr>
                    <td>source</td>
                    <td>string</td>
                    <td>0..*</td>
                    <td></td>
                    <td></td>
                    <td>DC:source</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>qualifier</th>
        <th>evidenceLine</th>
        <th>description</th>
        <th>canonicalAllele</th>
        <th>condition</th>
        <th>caseGroupFrequency</th>
        <th>controlGroupFrequency</th>
        <th>oddsRatio</th>
        <th>confidenceLevel</th>
        <th>confidenceIntervalLower</th>
        <th>confidenceIntervalUpper</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../details/details.html#CaseCtrl030">CaseCtrl030</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>Treating ExAC data as controls for</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll022">CanAll022</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond048">GenCond048</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#AllFreq029">AllFreq029</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#AllFreq028">AllFreq028</a></p>
         </td>
          <td>
              <p>9.1361</p>
         </td>
          <td>
              <p>99.0</p>
         </td>
          <td>
              <p>6.1425</p>
         </td>
          <td>
              <p>13.5889</p>
         </td>
          <!--
          <td><div class="example_view" id="CaseCtrl030"><pre><code>{
  "id": "CaseCtrl030",
  "type": "CaseControl",
  "canonicalAllele": {
    "id": "CanAll022",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll025",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll022",
      "alleleName": "NM_000441.1(SLC26A4):c.1003T>C"
    }
  },
  "condition": {
    "id": "GenCond048",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID183",
        "type": "Disease",
        "label": "nonsyndromic deafness"
      }
    ],
    "name": "Hearing Loss"
  },
  "oddsRatio": 9.1361,
  "confidenceLevel": 99.0,
  "confidenceIntervalLower": 6.1425,
  "confidenceIntervalUpper": 13.5889,
  "caseGroupFrequency": {
    "id": "AllFreq029",
    "type": "AlleleFrequency",
    "ascertainment": {
      "id": "AscrtID206",
      "type": "Ascertainment",
      "userLabel": {
        "id": "UsrLabl002",
        "type": "UserLabel",
        "description": "Case group is composed of cases from six cohorts from the GENIUS T2D consortium.  Case status is defined per-cohort."
      }
    },
    "allele": {
      "id": "CanAll022",
      "type": "CanonicalAllele",
      "preferredCtxAllele": {
        "id": "CtxAll025",
        "type": "ContextualAllele",
        "relatedCanonicalAllele": "CanAll022",
        "alleleName": "NM_000441.1(SLC26A4):c.1003T>C"
      }
    },
    "alleleCount": 32,
    "alleleNumber": 4076
  },
  "controlGroupFrequency": {
    "id": "AllFreq028",
    "type": "AlleleFrequency",
    "ascertainment": {
      "id": "AscrtID203",
      "type": "Ascertainment",
      "label": "ExAc ascertainment method"
    },
    "allele": {
      "id": "CanAll022",
      "type": "CanonicalAllele",
      "preferredCtxAllele": {
        "id": "CtxAll025",
        "type": "ContextualAllele",
        "relatedCanonicalAllele": "CanAll022",
        "alleleName": "NM_000441.1(SLC26A4):c.1003T>C"
      }
    },
    "alleleCount": 105,
    "alleleNumber": 121336
  },
  "description": "Treating ExAC data as controls for"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CaseCtrl033">CaseCtrl033</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>Did not calculate formal odds ratio</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll023">CanAll023</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond011">GenCond011</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#AllFreq032">AllFreq032</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#AllFreq031">AllFreq031</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CaseCtrl033"><pre><code>{
  "id": "CaseCtrl033",
  "type": "CaseControl",
  "canonicalAllele": {
    "id": "CanAll023",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll026",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll023",
      "alleleName": "NM_000540.2 (RYR1):c.1840C>T"
    }
  },
  "condition": {
    "id": "GenCond011",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID172",
        "type": "Disease",
        "label": "Malignant hyperthermia"
      }
    ],
    "modeOfInheritance": {
      "id": "ModeOfInhID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "caseGroupFrequency": {
    "id": "AllFreq032",
    "type": "AlleleFrequency",
    "allele": {
      "id": "CanAll023",
      "type": "CanonicalAllele",
      "preferredCtxAllele": {
        "id": "CtxAll026",
        "type": "ContextualAllele",
        "relatedCanonicalAllele": "CanAll023",
        "alleleName": "NM_000540.2 (RYR1):c.1840C>T"
      }
    },
    "description": "This allele is thought to account for between 2% and 10% of patients with Malignant hyperthermia."
  },
  "controlGroupFrequency": {
    "id": "AllFreq031",
    "type": "AlleleFrequency",
    "ascertainment": {
      "id": "AscrtID203",
      "type": "Ascertainment",
      "label": "ExAc ascertainment method"
    },
    "allele": {
      "id": "CanAll023",
      "type": "CanonicalAllele",
      "preferredCtxAllele": {
        "id": "CtxAll026",
        "type": "ContextualAllele",
        "relatedCanonicalAllele": "CanAll023",
        "alleleName": "NM_000540.2 (RYR1):c.1840C>T"
      }
    },
    "alleleCount": 9,
    "alleleNumber": 66740
  },
  "description": "Did not calculate formal odds ratio"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#CaseCtrl036">CaseCtrl036</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>Odds ratio is calculated by pooling samples from the six GENIUS T2D cohorts before calculation.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll024">CanAll024</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond012">GenCond012</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#AllFreq034">AllFreq034</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#AllFreq035">AllFreq035</a></p>
         </td>
          <td>
              <p>1.06</p>
         </td>
          <td>
              <p>95.0</p>
         </td>
          <td>
              <p>0.96</p>
         </td>
          <td>
              <p>1.18</p>
         </td>
          <!--
          <td><div class="example_view" id="CaseCtrl036"><pre><code>{
  "id": "CaseCtrl036",
  "type": "CaseControl",
  "canonicalAllele": {
    "id": "CanAll024",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll027",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll024",
      "alleleName": "NM_006208.2(ENPP1):c.517A>C"
    }
  },
  "condition": {
    "id": "GenCond012",
    "type": "GeneticCondition",
    "phenotype": [
      {
        "id": "PhenID189",
        "type": "Phenotype",
        "label": "Abnormal Glucose Homeostasis"
      }
    ]
  },
  "oddsRatio": 1.06,
  "confidenceLevel": 95.0,
  "caseGroupFrequency": {
    "id": "AllFreq034",
    "type": "AlleleFrequency",
    "ascertainment": {
      "id": "AscrtID207",
      "type": "Ascertainment",
      "userLabel": {
        "id": "UsrLabl002",
        "type": "UserLabel",
        "description": "Case group is composed of cases from six cohorts from the GENIUS T2D consortium.  Case status is defined per-cohort."
      }
    },
    "allele": {
      "id": "CanAll024",
      "type": "CanonicalAllele",
      "preferredCtxAllele": {
        "id": "CtxAll027",
        "type": "ContextualAllele",
        "relatedCanonicalAllele": "CanAll024",
        "alleleName": "NM_006208.2(ENPP1):c.517A>C"
      }
    },
    "alleleCount": 1198,
    "alleleNumber": 7344,
    "individualCount": 3672,
    "homozygousAlleleIndividualCount": 113,
    "heterozygousAlleleIndividualCount": 972,
    "population": {
      "id": "PopID090",
      "type": "Population",
      "label": "Non-Finnish European"
    }
  },
  "controlGroupFrequency": {
    "id": "AllFreq035",
    "type": "AlleleFrequency",
    "ascertainment": {
      "id": "AscrtID208",
      "type": "Ascertainment",
      "userLabel": {
        "id": "UsrLabl003",
        "type": "UserLabel",
        "description": "Controls group is composed of controls from six cohorts from the GENIUS T2D consortium.  Controls status is defined per-cohort."
      }
    },
    "allele": {
      "id": "CanAll024",
      "type": "CanonicalAllele",
      "preferredCtxAllele": {
        "id": "CtxAll027",
        "type": "ContextualAllele",
        "relatedCanonicalAllele": "CanAll024",
        "alleleName": "NM_006208.2(ENPP1):c.517A>C"
      }
    },
    "alleleCount": 936,
    "alleleNumber": 5870,
    "individualCount": 2935,
    "homozygousAlleleIndividualCount": 77,
    "heterozygousAlleleIndividualCount": 782,
    "population": {
      "id": "PopID090",
      "type": "Population",
      "label": "Non-Finnish European"
    }
  },
  "confidenceIntervalLower": 0.96,
  "confidenceIntervalUpper": 1.18,
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/13/16",
      "contributionRole": {
        "id": "ContribRoleID194",
        "type": "ContributoryRole",
        "label": "curator role"
      }
    }
  ],
  "source": [
    "http://www.ncbi.nlm.nih.gov/pubmed/23633196"
  ],
  "description": "Odds ratio is calculated by pooling samples from the six GENIUS T2D cohorts before calculation."
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../../">Interpretation Overview</a></h5>
              </li>
              <li class='list-group-item'><h5><a href="../../user/">User Guide</a></h5></li>
              <li class="list-group-item"><h5><a href="../">Technical Reference</a></h5><ul class='list-unstyled'><li><a href="../core/">Core</a></li><li><a href="../domain_entities/">Domain Entities</a></li><li class="active"><a href="./">Domain Evidence</a></li><ul class='list-unstyled'><li><a href="aggregate_segregation.html">AggregateSegregation</a></li><li><a href="allele_conservation.html">AlleleConservation</a></li><li><a href="allele_frequency.html">AlleleFrequency</a></li><li><a href="allele_functional_impact.html">AlleleFunctionalImpact</a></li><li><a href="benign_missense_variation_rate.html">BenignMissenseVariationRate</a></li><li class="active"><a href="case_control.html">CaseControl</a></li><li><a href="condition_mechanism.html">ConditionMechanism</a></li><li><a href="condition_penetrance.html">ConditionPenetrance</a></li><li><a href="condition_prevelance.html">ConditionPrevelance</a></li><li><a href="criterion_assessment.html">CriterionAssessment</a></li><li><a href="excluded_region_type.html">Haplotype</a></li><li><a href="family_condition.html">FamilyCondition</a></li><li><a href="family_segregation.html">FamilySegregation</a></li><li><a href="in_silico_prediction.html">InSilicoPrediction</a></li><li><a href="individual_allele.html">IndividualGenotype</a></li><li><a href="individual_allele_inheritance.html">IndividualAlleleInheritance</a></li><li><a href="individual_condition.html">IndividualCondition</a></li><li><a href="locus_heterogeneity.html">LocusHeterogeneity</a></li><li><a href="molecular_consequence.html">AlleleMolecularConsequence</a></li><li><a href="null_allele.html">NullAllele</a></li><li><a href="region_alleles.html">RegionAlleles</a></li><li><a href="region_type.html">RegionType</a></li><li><a href="sequence_location.html">SequenceLocation</a></li></ul><li><a href="../provenance/">Provenance</a></li><li><a href="../terminology/">Terminology</a></li></ul></li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2017 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../javascripts/jquery.js"></script><script src="../../javascripts/bootstrap.js"></script><script src="../../javascripts/jquery.jsonview.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
