<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a class="navbar-brand" href="/">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a target="_blank" href="http://www.clinicalgenome.org"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>
    


    
        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">FamilySegregation</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../../">Interpretation Model</a></li>
<li><a href="../">Technical Reference and Examples</a></li>
<li><a href="/tech/domain evidence/">Domain Evidence</a></li>

              </ol>
                <blockquote><p>A statement describing the correlation between the presence of a particular allele and the manifestation of a mendelian condition, as inherited through a single family pedigree (i.e. co-segregation of an allele and condition in a single family).

</p>
                <STRONG>
                  <div>SEPIO:0000247</div>
                      <i>Descended from Statement</i>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>The FamilySegregationData element represents co-segregation of an allele and a condition within a particular family.   The segregation can be characterized using a series of aggregate counts, such as the number of individuals that have both the allele and the condition, or those with the allele but not the condition.</p>

<p>In addition, or instead, the specific family details can be provided in pedigree file format.  The contents of a pedigree file are stored as a string in the pedigree attribute, and the columns, genotypeValues, and affectedValues provide the metadata required to fully interpret the coding of the pedigree file, for instance, mapping 0/1 genotype codes in the pedigree file to named alleles.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>canonicalAllele</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>Allele being tested for cosegregation</td>
                <td>SEPIO:0000275</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>GeneticCondition</td>
                <td>1..1</td>
                <td>Condition being tested for cosegregation</td>
                <td>SEPIO:0000276</td>
            </tr>
            <tr>
                <td>family</td>
                <td>Family</td>
                <td>1..1</td>
                <td>Family in which segregation is being tested</td>
                <td>SEPIO:0000282</td>
            </tr>
            <tr>
                <td>phenotypePositiveAllelePositive</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of (non-proband) individuals with both the condition and the allele</td>
                <td>SEPIO:0000292</td>
            </tr>
            <tr>
                <td>phenotypePositiveAlleleNegative</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of (non-proband) individuals with the condition and without the allele</td>
                <td>SEPIO:0000290</td>
            </tr>
            <tr>
                <td>phenotypeNegativeAllelePositive</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of (non-proband) individuals without the condition and with the allele</td>
                <td>SEPIO:0000291</td>
            </tr>
            <tr>
                <td>phenotypeNegativeAlleleNegative</td>
                <td>int</td>
                <td>0..1</td>
                <td>The number of non-proband family members that have neither the allele nor the phenotype</td>
                <td>SEPIO:0000289</td>
            </tr>
            <tr>
                <td>pedigree</td>
                <td>String</td>
                <td>0..*</td>
                <td>Detailed information about the family structure, affected status, and genotypes in pedigree/linkage format.</td>
                <td>SEPIO:0000318</td>
            </tr>
            <tr>
                <td>columns</td>
                <td>String</td>
                <td>0..*</td>
                <td>Column names for the pedigree data element</td>
                <td>SEPIO:0000316</td>
            </tr>
            <tr>
                <td>genotypeValues</td>
                <td>String</td>
                <td>0..*</td>
                <td>Map describing how genotypes are represented in the pedigree</td>
                <td>SEPIO:0000317</td>
            </tr>
            <tr>
                <td>affectedValues</td>
                <td>String</td>
                <td>0..*</td>
                <td>Map describing how affected status is represented in the pedigree</td>
                <td>SEPIO:0000313</td>
            </tr>
            <tr>
                <td>inconsistentSegregationsObserved</td>
                <td>InconsistentSegregationObserved</td>
                <td>0..1</td>
                <td>Whether phenotype positive/allele negative or phenotype negative/allele positive individuals are in the family</td>
                <td>SEPIO:0000314</td>
            </tr>
            <tr>
                <td>inconsistentSegregationCount</td>
                <td>int</td>
                <td>0..1</td>
                <td></td>
                <td>SEPIO:0000315</td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
                <th>IRI</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>description</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A human-readable description of the entity</td>
                    <td>DC:description</td>
                </tr>
                <tr>
                    <td>contribution</td>
                    <td>Contribution</td>
                    <td>0..*</td>
                    <td></td>
                    <td>The contributions (agent, time and role) made to this entity</td>
                    <td>SEPIO:0000159</td>
                </tr>
                <tr>
                    <td>source</td>
                    <td>String</td>
                    <td>0..*</td>
                    <td></td>
                    <td></td>
                    <td>DC:source</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>description</th>
        <th>canonicalAllele</th>
        <th>condition</th>
        <th>family</th>
        <th>phenotypePositiveAllelePositive</th>
        <th>phenotypePositiveAlleleNegative</th>
        <th>phenotypeNegativeAllelePositive</th>
        <th>phenotypeNegativeAlleleNegative</th>
        <th>pedigree</th>
        <th>columns</th>
        <th>genotypeValues</th>
        <th>affectedValues</th>
        <th>inconsistentSegregationsObserved</th>
        <th>inconsistentSegregationCount</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../details/details.html#FamSeg076">FamSeg076</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll124">CanAll124</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond056">GenCond056</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam373">Fam373</a></p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#ISegObsID220">ISegObsID220</a></p> (present)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg076"><pre><code>{
  "id": "FamSeg076",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond056",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID175",
        "type": "Disease",
        "label": "noonan syndrome"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll124",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll130",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll124",
      "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
    }
  },
  "family": {
    "id": "Fam373",
    "type": "Family",
    "description": "Family #011"
  },
  "phenotypePositiveAllelePositive": 0,
  "phenotypeNegativeAllelePositive": 2,
  "phenotypePositiveAlleleNegative": 0,
  "inconsistentSegregationsObserved": {
    "id": "ISegObsID220",
    "type": "InconsistentSegregationObserved",
    "label": "present"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg103">FamSeg103</a></p></td>
          <td>
              <p>identified in 1 individual with HCM. Segregation analysis determined 5 other members of the family also carried this variant. However, only 3 of those family members had HCM (aged 48-75). The 2 other family members were young and asymptomatic. So from this paper, only count 3 segregations. The two apparently refuting segregations can be ignored due to the age-dependent penetrance of HCM.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll063">CanAll063</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond020">GenCond020</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam363">Fam363</a></p> (12A)
         </td>
          <td>
              <p>3</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg103"><pre><code>{
  "id": "FamSeg103",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond020",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID160",
        "type": "Disease",
        "label": "hypertrophic cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll063",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll067",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll063",
      "alleleName": "NM_000256.3(MYBPC3)c.1351+1G>A"
    }
  },
  "family": {
    "id": "Fam363",
    "type": "Family",
    "label": "12A",
    "description": "Family #001"
  },
  "phenotypePositiveAllelePositive": 3,
  "phenotypeNegativeAllelePositive": 2,
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/13/16",
      "contributionRole": {
        "id": "CRID194",
        "type": "ContributoryRole",
        "label": "curator role"
      }
    }
  ],
  "source": [
    "http://www.ncbi.nlm.nih.gov/pubmed/17081393"
  ],
  "description": "identified in 1 individual with HCM. Segregation analysis determined 5 other members of the family also carried this variant. However, only 3 of those family members had HCM (aged 48-75). The 2 other family members were young and asymptomatic. So from this paper, only count 3 segregations. The two apparently refuting segregations can be ignored due to the age-dependent penetrance of HCM."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg104">FamSeg104</a></p></td>
          <td>
              <p>In LMM internal family, c.1351+1G>A was identified in 1 individual with HCM. Variant also found in individual’s sister - who also had clinical dx of HCM.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll063">CanAll063</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond020">GenCond020</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam364">Fam364</a></p>
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg104"><pre><code>{
  "id": "FamSeg104",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond020",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID160",
        "type": "Disease",
        "label": "hypertrophic cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll063",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll067",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll063",
      "alleleName": "NM_000256.3(MYBPC3)c.1351+1G>A"
    }
  },
  "family": {
    "id": "Fam364",
    "type": "Family",
    "description": "Family #002"
  },
  "phenotypePositiveAllelePositive": 1,
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/13/16",
      "contributionRole": {
        "id": "CRID194",
        "type": "ContributoryRole",
        "label": "curator role"
      }
    }
  ],
  "source": [
    "http://partners.org/ppm/lmm/gi/variant/12345"
  ],
  "description": "In LMM internal family, c.1351+1G>A was identified in 1 individual with HCM. Variant also found in individual’s sister - who also had clinical dx of HCM."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg107">FamSeg107</a></p></td>
          <td>
              <p>proband and 5 affected family members all homozygous for NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly).</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll062">CanAll062</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond021">GenCond021</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam365">Fam365</a></p>
         </td>
          <td>
              <p>5</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg107"><pre><code>{
  "id": "FamSeg107",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond021",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID184",
        "type": "Disease",
        "label": "Usher syndrome"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID052",
      "type": "ModeOfInheritance",
      "label": "Autosomal Recessive"
    }
  },
  "canonicalAllele": {
    "id": "CanAll062",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll066",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll062",
      "alleleName": "NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)"
    }
  },
  "family": {
    "id": "Fam365",
    "type": "Family",
    "description": "Family #003"
  },
  "phenotypePositiveAllelePositive": 5,
  "description": "proband and 5 affected family members all homozygous for NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg108">FamSeg108</a></p></td>
          <td>
              <p>c.1000C>T identified in het state in 1 individual with Usher. Individual compound het with Gly1840Val USH2A variant - confirmed in trans. Gly1840Val called Pathogenic. Individual has 3 affected sibs all of whom are compound het with 1000C>T & Gly1840Val. One additional sib only carried 1000C>T and is unaffected.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll062">CanAll062</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond021">GenCond021</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam366">Fam366</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg108"><pre><code>{
  "id": "FamSeg108",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond021",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID184",
        "type": "Disease",
        "label": "Usher syndrome"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID052",
      "type": "ModeOfInheritance",
      "label": "Autosomal Recessive"
    }
  },
  "canonicalAllele": {
    "id": "CanAll062",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll066",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll062",
      "alleleName": "NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly)"
    }
  },
  "family": {
    "id": "Fam366",
    "type": "Family",
    "description": "Family #004"
  },
  "description": "c.1000C>T identified in het state in 1 individual with Usher. Individual compound het with Gly1840Val USH2A variant - confirmed in trans. Gly1840Val called Pathogenic. Individual has 3 affected sibs all of whom are compound het with 1000C>T & Gly1840Val. One additional sib only carried 1000C>T and is unaffected."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg111">FamSeg111</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll064">CanAll064</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond051">GenCond051</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam367">Fam367</a></p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg111"><pre><code>{
  "id": "FamSeg111",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond051",
    "type": "GeneticCondition",
    "name": "Breast Ovarian Cancer, Autosomal Dominant",
    "disease": [
      {
        "id": "DisID148",
        "type": "Disease",
        "label": "hereditary breast ovarian cancer"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll064",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll068",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll064",
      "alleleName": "NM_000059.3(BRCA2):c.1832C>G"
    }
  },
  "family": {
    "id": "Fam367",
    "type": "Family",
    "description": "Family #005"
  },
  "phenotypePositiveAllelePositive": 2
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg113">FamSeg113</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll065">CanAll065</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond002">GenCond002</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam368">Fam368</a></p>
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>[[Fam006,Fam006.1,0,0,1,2,0/0],
   [Fam006,Fam006.2,0,0,2,1,0/0],
   [Fam006,Fam006.3,Fam006.1,Fam006.2,2,3,0/0],
   [Fam006,Fam006.4,Fam006.1,Fam006.2,1,2,1/2],
   [Fam006,Fam006.5,Fam006.1,Fam006.2,2,1,1/1],
   [Fam006,Fam006.6,Fam006.1,Fam006.2,1,2,1/2]]</p>
         </td>
          <td>
              <p>[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”]</p>
         </td>
          <td>
              <p>{0: “Unknown”,1:”Reference”, 2: CA65}</p>
         </td>
          <td>
              <p>{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg113"><pre><code>{
  "id": "FamSeg113",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond002",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID162",
        "type": "Disease",
        "label": "dilated cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll065",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll069",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll065",
      "alleleName": "NM_001001430.2(TNNT2):c.629_631delAGA"
    }
  },
  "family": {
    "id": "Fam368",
    "type": "Family",
    "description": "Family #006"
  },
  "phenotypePositiveAllelePositive": 1,
  "phenotypeNegativeAllelePositive": 1,
  "columns": [
    "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”]"
  ],
  "affectedValues": [
    "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
  ],
  "genotypeValues": [
    "{0: “Unknown”,1:”Reference”, 2: CA65}"
  ],
  "pedigree": [
    "[[Fam006,Fam006.1,0,0,1,2,0/0],\n   [Fam006,Fam006.2,0,0,2,1,0/0],\n   [Fam006,Fam006.3,Fam006.1,Fam006.2,2,3,0/0],\n   [Fam006,Fam006.4,Fam006.1,Fam006.2,1,2,1/2],\n   [Fam006,Fam006.5,Fam006.1,Fam006.2,2,1,1/1],\n   [Fam006,Fam006.6,Fam006.1,Fam006.2,1,2,1/2]]"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg114">FamSeg114</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll065">CanAll065</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond002">GenCond002</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam369">Fam369</a></p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>3</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>[[Fam007, Fam007.1, 0, 0, 1, 3, 0/0, 0],
   [Fam007, Fam007.2, 0, 0, 2, 1, 0/0, 0],
   [Fam007, Fam007.3, 0, 0, 1, 3, 0/0, 0],
   [Fam007, Fam007.4, 0, 0, 2, 1, 0/0, 0],
   [Fam007, Fam007.5, 0, 0, 1, 3, 0/0, 0],
   [Fam007, Fam007.6, Fam007.1, Fam007.2, 2, 3, 1/2, 1],
   [Fam007, Fam007.7,0, 0, 1, 1, 0/0, 0],
   [Fam007, Fam007.8, Fam007.1, Fam007.2, 2, 1, 1/2, 1],
   [Fam007, Fam007.9,Fam007.3, Fam007.4, 1, 1, 0/0, 0],
   [Fam007, Fam007.10, 0, 0, 1, 1, 0/0, 0],
   [Fam007, Fam007.11, Fam007.5, Fam007.6, 2, 2, 1/2, 0],
   [Fam007, Fam007.12, Fam007.5, Fam007.6, 1,2, 0/0, 0],
   [Fam007, Fam007.13, Fam007.5, Fam007.6, 2,1, 1/1,0],
   [Fam007, Fam007.14, Fam007.5, Fam007.6, 1, 1, 1/1, 0],
   [Fam007, Fam007.15, Fam007.7, Fam007.8, 1, 2, 1/2, 0],
   [Fam007, Fam007.16, Fam007.8, Fam007.9, 1, 2, 1/2, 0],
   [Fam007, Fam007.17, Fam007.8, Fam007.9, 1, 2, 0/0,0],
   [Fam007, Fam007.18, Fam007.8, Fam007.9, 1, 2, 0/0, 0],
   [Fam007, Fam007.19, Fam007.10, Fam007.11, 1, 2, 0/0,0],
   [Fam007.20,Fam007.10, Fam007.11, 1,1,1/1,0]]</p>
         </td>
          <td>
              <p>[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”,”ObligateGenotype”]</p>
         </td>
          <td>
              <p>{0: “Unknown”,1:”Reference”, 2: CA65}</p>
         </td>
          <td>
              <p>{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg114"><pre><code>{
  "id": "FamSeg114",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond002",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID162",
        "type": "Disease",
        "label": "dilated cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll065",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll069",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll065",
      "alleleName": "NM_001001430.2(TNNT2):c.629_631delAGA"
    }
  },
  "family": {
    "id": "Fam369",
    "type": "Family",
    "description": "Family #007"
  },
  "phenotypePositiveAllelePositive": 2,
  "phenotypeNegativeAllelePositive": 3,
  "columns": [
    "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”,”ObligateGenotype”]"
  ],
  "affectedValues": [
    "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
  ],
  "genotypeValues": [
    "{0: “Unknown”,1:”Reference”, 2: CA65}"
  ],
  "pedigree": [
    "[[Fam007, Fam007.1, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.2, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.3, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.4, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.5, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.6, Fam007.1, Fam007.2, 2, 3, 1/2, 1],\n   [Fam007, Fam007.7,0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.8, Fam007.1, Fam007.2, 2, 1, 1/2, 1],\n   [Fam007, Fam007.9,Fam007.3, Fam007.4, 1, 1, 0/0, 0],\n   [Fam007, Fam007.10, 0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.11, Fam007.5, Fam007.6, 2, 2, 1/2, 0],\n   [Fam007, Fam007.12, Fam007.5, Fam007.6, 1,2, 0/0, 0],\n   [Fam007, Fam007.13, Fam007.5, Fam007.6, 2,1, 1/1,0],\n   [Fam007, Fam007.14, Fam007.5, Fam007.6, 1, 1, 1/1, 0],\n   [Fam007, Fam007.15, Fam007.7, Fam007.8, 1, 2, 1/2, 0],\n   [Fam007, Fam007.16, Fam007.8, Fam007.9, 1, 2, 1/2, 0],\n   [Fam007, Fam007.17, Fam007.8, Fam007.9, 1, 2, 0/0,0],\n   [Fam007, Fam007.18, Fam007.8, Fam007.9, 1, 2, 0/0, 0],\n   [Fam007, Fam007.19, Fam007.10, Fam007.11, 1, 2, 0/0,0],\n   [Fam007.20,Fam007.10, Fam007.11, 1,1,1/1,0]]"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg196">FamSeg196</a></p></td>
          <td>
              <p>observed in one case diagnosed with Familial Hypercholesterolemia, characterized by elevated LDLC levels. Two siblings of the proband also have elevated LDLC levels but neither carry the 313+2T>C variant.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll088">CanAll088</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond035">GenCond035</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam370">Fam370</a></p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#ISegObsID220">ISegObsID220</a></p> (present)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg196"><pre><code>{
  "id": "FamSeg196",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond035",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID150",
        "type": "Disease",
        "label": "Hypercholesterolemia, Familial"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll088",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll094",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll088",
      "alleleName": "NM_000527.4(LDLR):c.313+2T>C"
    }
  },
  "family": {
    "id": "Fam370",
    "type": "Family",
    "description": "Family #008"
  },
  "phenotypePositiveAllelePositive": 0,
  "phenotypePositiveAlleleNegative": 2,
  "inconsistentSegregationsObserved": {
    "id": "ISegObsID220",
    "type": "InconsistentSegregationObserved",
    "label": "present"
  },
  "description": "observed in one case diagnosed with Familial Hypercholesterolemia, characterized by elevated LDLC levels. Two siblings of the proband also have elevated LDLC levels but neither carry the 313+2T>C variant."
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg197">FamSeg197</a></p></td>
          <td>
              <p>The individual with HCM but without the allele is likely a phenocopy explained
by environmental factors (obese, alcohol)</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll089">CanAll089</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond065">GenCond065</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam371">Fam371</a></p>
         </td>
          <td>
              <p>3</p>
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#ISegObsID220">ISegObsID220</a></p> (present)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg197"><pre><code>{
  "id": "FamSeg197",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond065",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID160",
        "type": "Disease",
        "label": "hypertrophic cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll089",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll095",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll089",
      "alleleName": "NM_000256.3(MYBPC3):c.1624+4A>T"
    }
  },
  "family": {
    "id": "Fam371",
    "type": "Family",
    "description": "Family #009"
  },
  "phenotypePositiveAllelePositive": 3,
  "phenotypePositiveAlleleNegative": 1,
  "inconsistentSegregationsObserved": {
    "id": "ISegObsID220",
    "type": "InconsistentSegregationObserved",
    "label": "present"
  },
  "description": "The individual with HCM but without the allele is likely a phenocopy explained\nby environmental factors (obese, alcohol)"
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSeg271">FamSeg271</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll090">CanAll090</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#GenCond066">GenCond066</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam372">Fam372</a></p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#ISegObsID220">ISegObsID220</a></p> (present)
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSeg271"><pre><code>{
  "id": "FamSeg271",
  "type": "FamilySegregation",
  "condition": {
    "id": "GenCond066",
    "type": "GeneticCondition",
    "disease": [
      {
        "id": "DisID160",
        "type": "Disease",
        "label": "hypertrophic cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "MOIID051",
      "type": "ModeOfInheritance",
      "label": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll090",
    "type": "CanonicalAllele",
    "preferredCtxAllele": {
      "id": "CtxAll096",
      "type": "ContextualAllele",
      "relatedCanonicalAllele": "CanAll090",
      "alleleName": "NM_001103.3(ACTN2):c.1484C>T (p.Thr495Met)"
    }
  },
  "family": {
    "id": "Fam372",
    "type": "Family",
    "description": "Family #010"
  },
  "phenotypePositiveAllelePositive": 0,
  "phenotypePositiveAlleleNegative": 2,
  "inconsistentSegregationsObserved": {
    "id": "ISegObsID220",
    "type": "InconsistentSegregationObserved",
    "label": "present"
  }
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../../">Interpretation Overview</a></h5>
              </li>
              <li class='list-group-item'><h5><a href="../../user/">User Guide</a></h5></li>
              <li class="list-group-item"><h5><a href="../">Technical Reference</a></h5><ul class='list-unstyled'><li><a href="../core/">Core</a></li><li class="active"><a href="/tech/domain evidence/">Domain Evidence</a></li><ul class='list-unstyled'><li><a href="/tech/domain evidence/aggregate_segregation.html">AggregateSegregation</a></li><li><a href="/tech/domain evidence/allele_conservation.html">AlleleConservation</a></li><li><a href="/tech/domain evidence/allele_frequency.html">AlleleFrequency</a></li><li><a href="/tech/domain evidence/allele_functional_impact.html">AlleleFunctionalImpact</a></li><li><a href="/tech/domain evidence/benign_missense_variation_rate.html">BenignMissenseVariationRate</a></li><li><a href="/tech/domain evidence/case_control.html">CaseControl</a></li><li><a href="/tech/domain evidence/condition_mechanism.html">ConditionMechanism</a></li><li><a href="/tech/domain evidence/condition_penetrance.html">ConditionPenetrance</a></li><li><a href="/tech/domain evidence/condition_prevelance.html">ConditionPrevelance</a></li><li><a href="/tech/domain evidence/criterion_assessment.html">CriterionAssessment</a></li><li><a href="/tech/domain evidence/excluded_region_type.html">ExcludedRegionType</a></li><li><a href="/tech/domain evidence/family_condition.html">FamilyCondition</a></li><li class="active"><a href="/tech/domain evidence/family_segregation.html">FamilySegregation</a></li><li><a href="/tech/domain evidence/in_silico_prediction.html">InSilicoPrediction</a></li><li><a href="/tech/domain evidence/individual_allele.html">IndividualAllele</a></li><li><a href="/tech/domain evidence/individual_allele_inheritance.html">IndividualAlleleInheritance</a></li><li><a href="/tech/domain evidence/individual_condition.html">IndividualCondition</a></li><li><a href="/tech/domain evidence/locus_heterogeneity.html">LocusHeterogeneity</a></li><li><a href="/tech/domain evidence/molecular_consequence.html">MolecularConsequence</a></li><li><a href="/tech/domain evidence/null_allele.html">NullAllele</a></li><li><a href="/tech/domain evidence/region_alleles.html">RegionAlleles</a></li><li><a href="/tech/domain evidence/region_type.html">RegionType</a></li><li><a href="/tech/domain evidence/sequence_location.html">SequenceLocation</a></li></ul><li><a href="../domain_entities/">Domain Entities</a></li><li><a href="../provenance/">Provenance</a></li><li><a href="../terminology/">Terminology</a></li></ul></li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2017 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../javascripts/jquery.js"></script><script src="../../javascripts/bootstrap.js"></script><script src="../../javascripts/jquery.jsonview.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
