<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen DataExchange Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../stylesheets/brand2.css" rel="stylesheet" /><link href="../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a href="/" class="navbar-brand">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a href="http://www.clinicalgenome.org" target="_blank"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>

        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">CriterionAssessment</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../">Interpretation Model</a></li>
<li><a href="./">Entities</a></li>

              </ol>
                <blockquote><p>An assertion of whether sufficient evidence exists to satisfy some pre-defined 'variant interpretation criterion', which defines requirements for a particular type of argument relevant to the pathogenicity of a genetic variant.</p>
                <STRONG>
                  <div>SEPIO:0000191</div>
                      <i>Descended from Statement</i>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            

    <!-- this is primarily the Scope and Usage section -->
    <h2>Scope and Usage</h2>
    <p>Variant interpretation criteria are provided as part of frameworks such as the ACMG Guidelines for charaterizing the pathogenicty of genetic variants. These criteria define relevant data types and rules for interpreting this data as evidence, to make a specific type of assertion about a variant that is ultimately relevant to its pathogenicity interpretation.  For example, the PM2 ACMG criterion may be met based on a study showing a variant to be absent in a particular population of healthy individuals (e.g. non-Finnish Europeans in the ExAC database). Implicit in an this criterion being 'met' is the broader assertion that the frequency of the variant is sufficiently low in healthy populations to support a meaningful argument for the pathogenicity of the variant.</p>

<p>In the process of generating a variant interpretation through the ACMG workflow, criterion assessments are treate as 'intermediate' assertions that establish whether a particuar criterion is met or not for the variant under review.  Evidence relevant to the merit of this assertion is organized around one or more evidence lines, and the criterion assessment is itself used as evidence for the final interpretation of the variant's pathogenicity.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
                <th>Defined in</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>criterion</td>
                <td>Criterion</td>
                <td>0..1</td>
                <td>The rule describing how the data is being used</td>
                <td>SEPIO:0000041</td>
                <td>CriterionAssessment</td>
            </tr>
            <tr>
                <td>variant</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>Variant about which the assemssment is made</td>
                <td>SEPIO:0000275</td>
                <td>CriterionAssessment</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>GeneticCondition</td>
                <td>0..*</td>
                <td>Condition for which the assessment is made</td>
                <td>SEPIO:0000276</td>
                <td>CriterionAssessment</td>
            </tr>
            <tr>
                <td>statementOutcome</td>
                <td>@id</td>
                <td>1..1</td>
                <td>Result of assessing the data and criterion</td>
                <td>SEPIO:0000197</td>
                <td>CriterionAssessment</td>
            </tr>
            <tr>
                <td>label</td>
                <td>string</td>
                <td>0..1</td>
                <td>A name given to the resource.</td>
                <td>RDFS:label</td>
                <td>Entity</td>
            </tr>
            <tr>
                <td>description</td>
                <td>string</td>
                <td>0..1</td>
                <td>Description may include but is not limited to: an abstract, a table of contents, a graphical representation, or a free-text account of the resource.</td>
                <td>DC:description</td>
                <td>Entity</td>
            </tr>
            <tr>
                <td>userLabelDictionary</td>
                <td>UserLabel</td>
                <td>0..*</td>
                <td>An optional label defined by the user. Used for custom entities or to clarify the preferred user label on existing entities with non-preferred labels.</td>
                <td>SEPIO:0000422</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>outcomeQualifier</td>
                <td>string</td>
                <td>0..1</td>
                <td>Use “NOT” as the value of this property to assert that the statement is negated.</td>
                <td>SEPIO:0000346</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>evidenceLine</td>
                <td>EvidenceLine</td>
                <td>0..*</td>
                <td>supporting evidence</td>
                <td>SEPIO:0000006</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>contribution</td>
                <td>Contribution</td>
                <td>0..*</td>
                <td>The contributions (agent, time and role) made to this entity</td>
                <td>SEPIO:0000159</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>source</td>
                <td>string</td>
                <td>0..*</td>
                <td>A string indicating the source of a Statement</td>
                <td>DC:source</td>
                <td>Statement</td>
            </tr>
            </tbody>
        </table>


    <h2>Instances</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>label</th>
        <th>description</th>
        <th>userLabelDictionary</th>
        <th>outcomeQualifier</th>
        <th>evidenceLine</th>
        <th>criterion</th>
        <th>variant</th>
        <th>condition</th>
        <th>statementOutcome</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess156">CGEX:CritAssess156</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Most pathogenic variants in TTN are truncating. However, a small number of missense variants in TTN are associated with ARVC (Taylor 2011 PMID:21810661 ) Because TTN is such a large gene, the statistical expectation of benign missense variants is very high.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn132">CGEX:EvLn132</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99034">SEPIO-CG:99034</a></p> (PP2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA256496">CAR:CA256496</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess156"><pre><code>{
  "id": "CGEX:CritAssess156",
  "type": "CriterionAssessment",
  "description": "Most pathogenic variants in TTN are truncating. However, a small number of missense variants in TTN are associated with ARVC (Taylor 2011 PMID:21810661 ) Because TTN is such a large gene, the statistical expectation of benign missense variants is very high.",
  "criterion": {
    "id": "SEPIO-CG:99034",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP2",
    "description": "Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease",
    "usageNotes": "Seems no one has established firm rules for when this rule is applicable so not sure how great any examples will be\nAgree with the lack of firm rules. Seems like one criteria may be observed missense variants over 5% allele frequency in ExAC. If this seems reasonable then I could generate a couple of examples.",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA256496",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll099"
  },
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn132",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:BMVR161",
          "type": "BenignMissenseVariationRateStatement",
          "description": "In LMM data alone, we have over 150 benign missense variants in TTN.",
          "gene": {
            "id": "HGNC:12403",
            "label": "TTN"
          },
          "statementOutcome": {
            "id": "SEPIO:0000265",
            "label": "High"
          }
        },
        {
          "id": "CGEX:MolCon169",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll099",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA256496",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll099"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9099",
              "type": "ContextualAlleleName",
              "name": "NM_133378.4(TTN): c.2926T>C",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001583",
            "label": "missense variant"
          }
        },
        {
          "id": "CGEX:Null406",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll099",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA256496",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll099"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9099",
              "type": "ContextualAlleleName",
              "name": "NM_133378.4(TTN): c.2926T>C",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99015",
            "label": "function not lost"
          }
        },
        {
          "id": "CGEX:CondMech165",
          "type": "ConditionMechanismStatement",
          "statementOutcome": {
            "id": "SO:0001583",
            "label": "missense variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000268",
            "label": "common"
          },
          "gene": {
            "id": "HGNC:12403",
            "label": "TTN"
          },
          "condition": {
            "id": "CGEX:GenCond054",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:247",
                "label": "arrhythmogenic right ventricular cardiomyopathy"
              }
            ]
          },
          "source": [
            "PMID:21810661"
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib519",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent010",
        "type": "Agent",
        "label": "ACME Molecular Lab"
      },
      "contributionDate": "7/2/2012 8:45 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ],
  "source": [
    "PMID:21810661"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess275">CGEX:CritAssess275</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Variant is not LOF and LOF is not a known mechanism of disease associated with this gene for any condition.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn001">CGEX:EvLn001</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99022">SEPIO-CG:99022</a></p> (PVS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090943">CAR:CA090943</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess275"><pre><code>{
  "id": "CGEX:CritAssess275",
  "type": "CriterionAssessment",
  "description": "Variant is not LOF and LOF is not a known mechanism of disease associated with this gene for any condition.",
  "criterion": {
    "id": "SEPIO-CG:99022",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PVS1",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "usageNotes": "The two main concerns for applying this rule is:\na. Is LOF an established disease mechanism?\nb. Is LOF actually occurring?",
    "defaultStrength": {
      "id": "SEPIO:0000220",
      "label": "Pathogenic Very Strong"
    }
  },
  "variant": {
    "id": "CAR:CA090943",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll001"
  },
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn001",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon001",
          "type": "AlleleMolecularConsequenceStatement",
          "description": "Protein ends at p.826 and this frameshift would create stop at p.852 (769+83) thus NMD would not occur. Instead an elongated protein",
          "contextualAllele": {
            "id": "CGEX:CtxAll001",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA090943",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll001"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9001",
              "type": "ContextualAlleleName",
              "name": "NM_031844.2(HNRNPU):c.2304_2305del (p.Gly769Glufs*83)",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001589",
            "label": "frameshift variant"
          }
        },
        {
          "id": "CGEX:Null396",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll001",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA090943",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll001"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9001",
              "type": "ContextualAlleleName",
              "name": "NM_031844.2(HNRNPU):c.2304_2305del (p.Gly769Glufs*83)",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99015",
            "label": "function not lost"
          }
        },
        {
          "id": "CGEX:CondMech002",
          "type": "ConditionMechanismStatement",
          "description": "HNRNPU is not associated with any conditions, and therefore LOF is not an established mechanism of any condition for this gene.",
          "statementOutcome": {
            "id": "SO:0002054",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000269",
            "label": "established"
          },
          "gene": {
            "id": "HGNC:5048",
            "label": "HNRNPU"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib396",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/1/16 10:10 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess277">CGEX:CritAssess277</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This variant only occurs as LOF in an uncommon transcript (NM_133437.4).  For most transcripts, the variant is intronic.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn003">CGEX:EvLn003</a></p><p><a href="details/details.html#CGEX:EvLn004">CGEX:EvLn004</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99022">SEPIO-CG:99022</a></p> (PVS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA238217">CAR:CA238217</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond047">CGEX:GenCond047</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess277"><pre><code>{
  "id": "CGEX:CritAssess277",
  "type": "CriterionAssessment",
  "description": "This variant only occurs as LOF in an uncommon transcript (NM_133437.4).  For most transcripts, the variant is intronic.",
  "criterion": {
    "id": "SEPIO-CG:99022",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PVS1",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "usageNotes": "The two main concerns for applying this rule is:\na. Is LOF an established disease mechanism?\nb. Is LOF actually occurring?",
    "defaultStrength": {
      "id": "SEPIO:0000220",
      "label": "Pathogenic Very Strong"
    }
  },
  "variant": {
    "id": "CAR:CA238217",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll003"
  },
  "condition": [
    {
      "id": "CGEX:GenCond047",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217604",
          "label": "dilated cardiomyopathy"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn003",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon005",
          "type": "AlleleMolecularConsequenceStatement",
          "description": "LOF is predicted but this transcript is rare",
          "contextualAllele": {
            "id": "CGEX:CtxAll003",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA238217",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll003"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9003",
              "type": "ContextualAlleleName",
              "name": "NM_133437.4(TTN):c.10670dupG",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001589",
            "label": "frameshift variant"
          }
        },
        {
          "id": "CGEX:Null398",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll003",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA238217",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll003"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9003",
              "type": "ContextualAlleleName",
              "name": "NM_133437.4(TTN):c.10670dupG",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0002055",
            "label": "null mutation"
          }
        },
        {
          "id": "CGEX:CondMech007",
          "type": "ConditionMechanismStatement",
          "statementOutcome": {
            "id": "SO:0002054",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000269",
            "label": "established"
          },
          "gene": {
            "id": "HGNC:12403",
            "label": "TTN"
          },
          "condition": {
            "id": "CGEX:GenCond047",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:217604",
                "label": "dilated cardiomyopathy"
              }
            ]
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn004",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon006",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll004",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA238217",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll003"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9004",
              "type": "ContextualAlleleName",
              "name": "LRG_391t2:c.10303+2691dupG",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0000191",
            "label": "interior intron"
          }
        },
        {
          "id": "CGEX:Null399",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll004",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA238217",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll003"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9004",
              "type": "ContextualAlleleName",
              "name": "LRG_391t2:c.10303+2691dupG",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99015",
            "label": "function not lost"
          }
        },
        {
          "id": "CGEX:CondMech007",
          "type": "ConditionMechanismStatement",
          "statementOutcome": {
            "id": "SO:0002054",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000269",
            "label": "established"
          },
          "gene": {
            "id": "HGNC:12403",
            "label": "TTN"
          },
          "condition": {
            "id": "CGEX:GenCond047",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:217604",
                "label": "dilated cardiomyopathy"
              }
            ]
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib398",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 2:22 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess278">CGEX:CritAssess278</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The assessed variant produces the same amino acid change (p.Phe285Leu) as known pathogenic variant c.853T>C</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn005">CGEX:EvLn005</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99023">SEPIO-CG:99023</a></p> (PS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA261606">CAR:CA261606</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond003">CGEX:GenCond003</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess278"><pre><code>{
  "id": "CGEX:CritAssess278",
  "type": "CriterionAssessment",
  "description": "The assessed variant produces the same amino acid change (p.Phe285Leu) as known pathogenic variant c.853T>C",
  "criterion": {
    "id": "SEPIO-CG:99023",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS1",
    "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
    "usageNotes": "Beware of changes that impact splicing rather than at the amino acid/protein level\nFrom CSER paper: One site erroneously used this to apply prior publication of the same exact variant while this rule, as described in more depth on the ACMG/AMP guideline, only applies when the established pathogenic variant has a different nucleotide change than the variant being interpreted.",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA261606",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll005"
  },
  "condition": [
    {
      "id": "CGEX:GenCond003",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:648",
          "label": "noonan syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn005",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp008",
          "type": "VariantPathogenicityInterpretation",
          "description": "NM_002834.3:c.855T>G Pathogenic for Noonan Syndrome",
          "variant": {
            "id": "CAR:CA220155",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll006"
          },
          "condition": [
            {
              "id": "CGEX:GenCond003",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:648",
                  "label": "noonan syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib399",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 3:22 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess279">CGEX:CritAssess279</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The assessed variant produces the same amino acid change (p.Met2083Ile) as known pathogenic variant c.6249G>A.  But that change is pathogenic because it has been shown to disrupt splicing and create a premature stop codon. Prediction tools do not suggest c.6249G>C disrupts splicing.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn006">CGEX:EvLn006</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99023">SEPIO-CG:99023</a></p> (PS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501220">CAR:CA501220</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond004">CGEX:GenCond004</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess279"><pre><code>{
  "id": "CGEX:CritAssess279",
  "type": "CriterionAssessment",
  "description": "The assessed variant produces the same amino acid change (p.Met2083Ile) as known pathogenic variant c.6249G>A.  But that change is pathogenic because it has been shown to disrupt splicing and create a premature stop codon. Prediction tools do not suggest c.6249G>C disrupts splicing.",
  "criterion": {
    "id": "SEPIO-CG:99023",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS1",
    "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
    "usageNotes": "Beware of changes that impact splicing rather than at the amino acid/protein level\nFrom CSER paper: One site erroneously used this to apply prior publication of the same exact variant while this rule, as described in more depth on the ACMG/AMP guideline, only applies when the established pathogenic variant has a different nucleotide change than the variant being interpreted.",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA501220",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll008"
  },
  "condition": [
    {
      "id": "CGEX:GenCond004",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:244",
          "label": "Primary ciliary dyskenisia"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn006",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp009",
          "type": "VariantPathogenicityInterpretation",
          "description": "NM_001369.2:c.6249G>A Pathogenic for PCD",
          "variant": {
            "id": "CAR:CA501221",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll009"
          },
          "condition": [
            {
              "id": "CGEX:GenCond004",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:244",
                  "label": "Primary ciliary dyskenisia"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib400",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 4:22 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess280">CGEX:CritAssess280</a></p></td>
          <td>
              
         </td>
          <td>
              <p>While the two amino acid changes are not identical, they are the same amino acid change in analagous residues of HRAS and KRAS.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn007">CGEX:EvLn007</a></p><p><a href="details/details.html#CGEX:EvLn008">CGEX:EvLn008</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99023">SEPIO-CG:99023</a></p> (PS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA341206">CAR:CA341206</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond005">CGEX:GenCond005</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess280"><pre><code>{
  "id": "CGEX:CritAssess280",
  "type": "CriterionAssessment",
  "description": "While the two amino acid changes are not identical, they are the same amino acid change in analagous residues of HRAS and KRAS.",
  "criterion": {
    "id": "SEPIO-CG:99023",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS1",
    "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
    "usageNotes": "Beware of changes that impact splicing rather than at the amino acid/protein level\nFrom CSER paper: One site erroneously used this to apply prior publication of the same exact variant while this rule, as described in more depth on the ACMG/AMP guideline, only applies when the established pathogenic variant has a different nucleotide change than the variant being interpreted.",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA341206",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll012"
  },
  "condition": [
    {
      "id": "CGEX:GenCond005",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:3071",
          "label": "costello syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn007",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp010",
          "type": "VariantPathogenicityInterpretation",
          "description": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Noonan Syndrome 3",
          "variant": {
            "id": "CAR:CA256480",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll014"
          },
          "condition": [
            {
              "id": "CGEX:GenCond006",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "OMIM:609942",
                  "label": "noonan syndrome 3"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn008",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp011",
          "type": "VariantPathogenicityInterpretation",
          "description": "NM_004985.4(KRAS):c.173C>T (p.Thr58Ile) Pathogenic for Costello syndrome",
          "variant": {
            "id": "CAR:CA256480",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll014"
          },
          "condition": [
            {
              "id": "CGEX:GenCond055",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:648",
                  "label": "noonan syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib401",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 5:22 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib402",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent002",
        "type": "Agent",
        "label": "Noonan/Rasopathy WG (ClinGen)"
      },
      "contributionDate": "6/2/16 5:22 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess281">CGEX:CritAssess281</a></p></td>
          <td>
              
         </td>
          <td>
              <p>IC1: Individual has condition, FH1: No family history, DNA1: variant is denovo (parentage confirmed)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn009">CGEX:EvLn009</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99024">SEPIO-CG:99024</a></p> (PS2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA016441">CAR:CA016441</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond052">CGEX:GenCond052</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess281"><pre><code>{
  "id": "CGEX:CritAssess281",
  "type": "CriterionAssessment",
  "description": "IC1: Individual has condition, FH1: No family history, DNA1: variant is denovo (parentage confirmed)",
  "criterion": {
    "id": "SEPIO-CG:99024",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS2",
    "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
    "usageNotes": "MYH7 is adding specificity to this rule with\na. No family history = parents have had ECHO and ECG\nb. Only paternity confirmation required",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA016441",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll017"
  },
  "condition": [
    {
      "id": "CGEX:GenCond052",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217604",
          "label": "dilated cardiomyopathy"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn009",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond012",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv333",
            "label": "individual #001"
          },
          "condition": [
            {
              "id": "CGEX:GenCond052",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:217604",
                  "label": "dilated cardiomyopathy"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:FamCond013",
          "type": "FamilyConditionStatement",
          "proband": {
            "id": "CGEX:Indiv333",
            "label": "individual #001"
          },
          "condition": [
            {
              "id": "CGEX:GenCond052",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:217604",
                  "label": "dilated cardiomyopathy"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        },
        {
          "id": "CGEX:IndAllInh014",
          "type": "IndividualAlleleInheritanceStatement",
          "description": "parentage confirmed via haplotyping.",
          "canonicalAllele": {
            "id": "CAR:CA016441",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll017"
          },
          "individual": {
            "id": "CGEX:Indiv333",
            "label": "individual #001"
          },
          "parentalConfirmation": {
            "id": "SEPIO-CG:99007",
            "label": "maternal and paternal confirmation"
          },
          "statementOutcome": [
            {
              "id": "GENO:0000880",
              "label": "de novo"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib403",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/3/16 8:00 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess282">CGEX:CritAssess282</a></p></td>
          <td>
              
         </td>
          <td>
              <p>De novo variant occurred on haplotype inherited from the father. Maternity no explicitly confirmed - but if variant occurred on haplotype inherited from the father, then confirmation of maternity not required.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn010">CGEX:EvLn010</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99024">SEPIO-CG:99024</a></p> (PS2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA257646">CAR:CA257646</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond008">CGEX:GenCond008</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess282"><pre><code>{
  "id": "CGEX:CritAssess282",
  "type": "CriterionAssessment",
  "description": "De novo variant occurred on haplotype inherited from the father. Maternity no explicitly confirmed - but if variant occurred on haplotype inherited from the father, then confirmation of maternity not required.",
  "criterion": {
    "id": "SEPIO-CG:99024",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS2",
    "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
    "usageNotes": "MYH7 is adding specificity to this rule with\na. No family history = parents have had ECHO and ECG\nb. Only paternity confirmation required",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA257646",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll018"
  },
  "condition": [
    {
      "id": "CGEX:GenCond008",
      "type": "GeneticCondition",
      "description": "desmin related myopathy",
      "disease": [
        {
          "id": "OMIM:601419",
          "label": "myofibrillar myopathy 1"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn010",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond015",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv334",
            "label": "individual #002"
          },
          "condition": [
            {
              "id": "CGEX:GenCond008",
              "type": "GeneticCondition",
              "description": "desmin related myopathy",
              "disease": [
                {
                  "id": "OMIM:601419",
                  "label": "myofibrillar myopathy 1"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:FamCond016",
          "type": "FamilyConditionStatement",
          "proband": {
            "id": "CGEX:Indiv334",
            "label": "individual #002"
          },
          "condition": [
            {
              "id": "CGEX:GenCond008",
              "type": "GeneticCondition",
              "description": "desmin related myopathy",
              "disease": [
                {
                  "id": "OMIM:601419",
                  "label": "myofibrillar myopathy 1"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        },
        {
          "id": "CGEX:IndAllInh017",
          "type": "IndividualAlleleInheritanceStatement",
          "description": "Non-paternity was excluded by assessment with microsatellites",
          "canonicalAllele": {
            "id": "CAR:CA257646",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll018"
          },
          "individual": {
            "id": "CGEX:Indiv334",
            "label": "individual #002"
          },
          "parentalConfirmation": {
            "id": "SEPIO-CG:99006",
            "label": "paternal confirmation"
          },
          "statementOutcome": [
            {
              "id": "GENO:0000880",
              "label": "de novo"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib404",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/3/16 9:17 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess286">CGEX:CritAssess286</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The RASopathy group considers MEK and ERK activation assays as well validated.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn017">CGEX:EvLn017</a></p><p><a href="details/details.html#CGEX:EvLn018">CGEX:EvLn018</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99025">SEPIO-CG:99025</a></p> (PS3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA220161">CAR:CA220161</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess286"><pre><code>{
  "id": "CGEX:CritAssess286",
  "type": "CriterionAssessment",
  "description": "The RASopathy group considers MEK and ERK activation assays as well validated.",
  "criterion": {
    "id": "SEPIO-CG:99025",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS3",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "usageNotes": "This particular rule will probably be downgraded often, given that there are not many “Functional studies that have been validated and shown to be reproducible and robust in a clinical diagnostic laboratory setting”",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA220161",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll023"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn017",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func026",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "abnormal/deceased MEK activtation.",
          "assayType": {
            "id": "CGEX:AFAMID076",
            "label": "Activation Assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll024",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA093846",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll024"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9024",
              "type": "ContextualAlleleName",
              "name": "NP_004324.2:p.Gly596Val",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:6847",
            "label": "MEK"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn018",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func027",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "abnormal/deceased ERK activtation.",
          "assayType": {
            "id": "CGEX:AFAMID076",
            "label": "Activation Assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll024",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA093846",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll024"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9024",
              "type": "ContextualAlleleName",
              "name": "NP_004324.2:p.Gly596Val",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:6871",
            "label": "ERK"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib410",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/4/16 7:12 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib411",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent002",
        "type": "Agent",
        "label": "Noonan/Rasopathy WG (ClinGen)"
      },
      "contributionDate": "6/4/16 7:12 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess287">CGEX:CritAssess287</a></p></td>
          <td>
              
         </td>
          <td>
              <p>statistically significant difference between affects and unaffecteds</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn019">CGEX:EvLn019</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99026">SEPIO-CG:99026</a></p> (PS4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA253316">CAR:CA253316</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond048">CGEX:GenCond048</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess287"><pre><code>{
  "id": "CGEX:CritAssess287",
  "type": "CriterionAssessment",
  "description": "statistically significant difference between affects and unaffecteds",
  "criterion": {
    "id": "SEPIO-CG:99026",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS4",
    "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA253316",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll025"
  },
  "condition": [
    {
      "id": "CGEX:GenCond048",
      "type": "GeneticCondition",
      "description": "Hearing Loss",
      "disease": [
        {
          "id": "Orphanet:87884",
          "label": "nonsyndromic deafness"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn019",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CaseCtrl030",
          "type": "CaseControlAlleleFrequencyStatement",
          "description": "Treating ExAC data as controls",
          "canonicalAllele": {
            "id": "CAR:CA253316",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll025"
          },
          "condition": {
            "id": "CGEX:GenCond048",
            "type": "GeneticCondition",
            "description": "Hearing Loss",
            "disease": [
              {
                "id": "Orphanet:87884",
                "label": "nonsyndromic deafness"
              }
            ]
          },
          "oddsRatio": 9.1361,
          "confidenceLevel": 99.0,
          "confidenceIntervalLower": 6.1425,
          "confidenceIntervalUpper": 13.5889,
          "caseGroupFrequency": {
            "id": "CGEX:AllFreq029",
            "type": "PopulationAlleleFrequencyStatement",
            "ascertainment": {
              "id": "CGEX:Ascrt0001",
              "label": "LMM Hearing Loss"
            },
            "allele": {
              "id": "CAR:CA253316",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll025"
            },
            "alleleCount": 32,
            "alleleNumber": 4076,
            "userLabelDictionary": [
              {
                "id": "CGEX:UsrLabl001",
                "type": "UserLabel",
                "description": "Across all published data (and LMM data) with hearing loss.",
                "labelFor": {
                  "id": "CGEX:Ascrt0001",
                  "label": "LMM Hearing Loss"
                }
              }
            ]
          },
          "controlGroupFrequency": {
            "id": "CGEX:AllFreq028",
            "type": "PopulationAlleleFrequencyStatement",
            "ascertainment": {
              "id": "SEPIO:0000332",
              "label": "ExAC ascertainment method"
            },
            "allele": {
              "id": "CAR:CA253316",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll025"
            },
            "alleleCount": 105,
            "alleleNumber": 121336
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib412",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/5/16 10:24 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess288">CGEX:CritAssess288</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Across publications, this variant is thought to account for 2%-10% of patients with Malignant hyperthermia. But variant is also only found in 9/66740 Europeans in ExAC (.01%). Based on these differences, groups applied PS4 to this variant classification (without a formal OR analysis).</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn020">CGEX:EvLn020</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99026">SEPIO-CG:99026</a></p> (PS4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA024311">CAR:CA024311</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond011">CGEX:GenCond011</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess288"><pre><code>{
  "id": "CGEX:CritAssess288",
  "type": "CriterionAssessment",
  "description": "Across publications, this variant is thought to account for 2%-10% of patients with Malignant hyperthermia. But variant is also only found in 9/66740 Europeans in ExAC (.01%). Based on these differences, groups applied PS4 to this variant classification (without a formal OR analysis).",
  "criterion": {
    "id": "SEPIO-CG:99026",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS4",
    "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA024311",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll026"
  },
  "condition": [
    {
      "id": "CGEX:GenCond011",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:423",
          "label": "Malignant hyperthermia"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn020",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CaseCtrl033",
          "type": "CaseControlAlleleFrequencyStatement",
          "description": "Did not calculate formal odds ratio",
          "canonicalAllele": {
            "id": "CAR:CA024311",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll026"
          },
          "condition": {
            "id": "CGEX:GenCond011",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:423",
                "label": "Malignant hyperthermia"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          },
          "caseGroupFrequency": {
            "id": "CGEX:AllFreq032",
            "type": "PopulationAlleleFrequencyStatement",
            "description": "This allele is thought to account for between 2% and 10% of patients with Malignant hyperthermia.",
            "allele": {
              "id": "CAR:CA024311",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll026"
            }
          },
          "controlGroupFrequency": {
            "id": "CGEX:AllFreq031",
            "type": "PopulationAlleleFrequencyStatement",
            "ascertainment": {
              "id": "SEPIO:0000332",
              "label": "ExAC ascertainment method"
            },
            "allele": {
              "id": "CAR:CA024311",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll026"
            },
            "alleleCount": 9,
            "alleleNumber": 66740
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib413",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/5/16 2:39 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess290">CGEX:CritAssess290</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Variant is located within the kinase domain (where multiple other pathogenic variants are located)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn022">CGEX:EvLn022</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99027">SEPIO-CG:99027</a></p> (PM1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090928">CAR:CA090928</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess290"><pre><code>{
  "id": "CGEX:CritAssess290",
  "type": "CriterionAssessment",
  "description": "Variant is located within the kinase domain (where multiple other pathogenic variants are located)",
  "criterion": {
    "id": "SEPIO-CG:99027",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM1",
    "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
    "usageNotes": "Suggested that this rule is NOT appropriate for LOF variants.\nAlso, note that when the region of interest is a “well-established functional domain” then it’s not necessary to show through evidence that it is intolerant of benign missense variants.  However, when a mutational hotspot is being defined, then showing it does not have benign missense variants is important.",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA090928",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll071"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn022",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:RegTyp040",
          "type": "RegionTypeStatement",
          "description": "This region is the catalytic (kinase) domain.",
          "region": {
            "id": "CGEX:Loc432",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NP_005219.2",
              "label": "NP_005219.2"
            },
            "start": 704,
            "stop": 1016
          },
          "statementOutcome": [
            {
              "id": "SO:0002188",
              "label": "functional region"
            }
          ]
        },
        {
          "id": "CGEX:RegAll380",
          "type": "RegionAllelesStatement",
          "description": "This region is the catalytic (kinase) domain.",
          "region": {
            "id": "CGEX:Loc432",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NP_005219.2",
              "label": "NP_005219.2"
            },
            "start": 704,
            "stop": 1016
          },
          "allele": [
            {
              "id": "CGEX:CtxAll126",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": {
                "id": "CAR:CA126726",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll126"
              },
              "alleleName": {
                "id": "CGEX:CtxNm9126",
                "type": "ContextualAlleleName",
                "name": "NP_005219.2:p.Thr790Met",
                "preferred": true
              }
            }
          ],
          "statementOutcome": {
            "id": "RO:0002656",
            "label": "overlaps region"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib415",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/6/16 8:12 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess291">CGEX:CritAssess291</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Variant is located in exon 15, which the RASopathy group has decided is an established functional domain for BRAF</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn023">CGEX:EvLn023</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99027">SEPIO-CG:99027</a></p> (PM1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA123651">CAR:CA123651</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess291"><pre><code>{
  "id": "CGEX:CritAssess291",
  "type": "CriterionAssessment",
  "description": "Variant is located in exon 15, which the RASopathy group has decided is an established functional domain for BRAF",
  "criterion": {
    "id": "SEPIO-CG:99027",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM1",
    "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
    "usageNotes": "Suggested that this rule is NOT appropriate for LOF variants.\nAlso, note that when the region of interest is a “well-established functional domain” then it’s not necessary to show through evidence that it is intolerant of benign missense variants.  However, when a mutational hotspot is being defined, then showing it does not have benign missense variants is important.",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA123651",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll029"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn023",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:RegTyp043",
          "type": "RegionTypeStatement",
          "description": "This region is exon 15 in the transcript, and codes the serine-threonine-protein kinase catalytic domain.",
          "region": {
            "id": "CGEX:Loc433",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_004333.4",
              "label": "NM_004333.4"
            },
            "start": 1803,
            "stop": 1921
          },
          "statementOutcome": [
            {
              "id": "SO:0002188",
              "label": "functional region"
            }
          ]
        },
        {
          "id": "CGEX:RegAll382",
          "type": "RegionAllelesStatement",
          "description": "This region is exon 15 in the transcript, and codes the serine-threonine-protein kinase catalytic domain.",
          "region": {
            "id": "CGEX:Loc433",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_004333.4",
              "label": "NM_004333.4"
            },
            "start": 1803,
            "stop": 1921
          },
          "allele": [
            {
              "id": "CGEX:CtxAll029",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": {
                "id": "CAR:CA123651",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll029"
              },
              "alleleName": {
                "id": "CGEX:CtxNm9029",
                "type": "ContextualAlleleName",
                "name": "NM_004333.4(BRAF):c.1789C>G",
                "preferred": true
              }
            }
          ],
          "statementOutcome": {
            "id": "RO:0002656",
            "label": "overlaps region"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib416",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/6/16 10:53 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib417",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent002",
        "type": "Agent",
        "label": "Noonan/Rasopathy WG (ClinGen)"
      },
      "contributionDate": "6/6/16 10:53 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess292">CGEX:CritAssess292</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The variant is near 4 other pathogenic variants</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn024">CGEX:EvLn024</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99027">SEPIO-CG:99027</a></p> (PM1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA352275">CAR:CA352275</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess292"><pre><code>{
  "id": "CGEX:CritAssess292",
  "type": "CriterionAssessment",
  "description": "The variant is near 4 other pathogenic variants",
  "criterion": {
    "id": "SEPIO-CG:99027",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM1",
    "description": "Located in a mutational hot spot and/or critical and well-established functional domain (e.g., active site of an enzyme) without benign variation",
    "usageNotes": "Suggested that this rule is NOT appropriate for LOF variants.\nAlso, note that when the region of interest is a “well-established functional domain” then it’s not necessary to show through evidence that it is intolerant of benign missense variants.  However, when a mutational hotspot is being defined, then showing it does not have benign missense variants is important.",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA352275",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll030"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn024",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:RegTyp046",
          "type": "RegionTypeStatement",
          "description": "Region(NM_0001692[996..1012]) is a mutational hotspot.",
          "region": {
            "id": "CGEX:Loc434",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_000169.2",
              "label": "NM_000169.2"
            },
            "start": 996,
            "stop": 1012
          },
          "statementOutcome": [
            {
              "id": "SO:0002186",
              "label": "mutational hotspot"
            }
          ]
        },
        {
          "id": "CGEX:RegAll384",
          "type": "RegionAllelesStatement",
          "description": "The region contains the following pathogenic variants: c.886A>G, c.890C>T, c.899T>C, & c.902G>A.",
          "region": {
            "id": "CGEX:Loc434",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_000169.2",
              "label": "NM_000169.2"
            },
            "start": 996,
            "stop": 1012
          },
          "allele": [
            {
              "id": "CGEX:CtxAll031",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": {
                "id": "CAR:CA022141",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll031"
              },
              "alleleName": {
                "id": "CGEX:CtxNm9031",
                "type": "ContextualAlleleName",
                "name": "NM_000169.2(GLA): c.886A>G",
                "preferred": true
              }
            },
            {
              "id": "CGEX:CtxAll032",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": {
                "id": "CAR:CA022153",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll032"
              },
              "alleleName": {
                "id": "CGEX:CtxNm9032",
                "type": "ContextualAlleleName",
                "name": "NM_000169.2(GLA):c.890C>T",
                "preferred": true
              }
            },
            {
              "id": "CGEX:CtxAll033",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": {
                "id": "CAR:CA022159",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll033"
              },
              "alleleName": {
                "id": "CGEX:CtxNm9033",
                "type": "ContextualAlleleName",
                "name": "NM_000169.2(GLA):c.899T>C",
                "preferred": true
              }
            },
            {
              "id": "CGEX:CtxAll034",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": {
                "id": "CAR:CA022189",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll034"
              },
              "alleleName": {
                "id": "CGEX:CtxNm9034",
                "type": "ContextualAlleleName",
                "name": "NM_000169.2(GLA):c.902G>A",
                "preferred": true
              }
            }
          ],
          "statementOutcome": {
            "id": "RO:0002656",
            "label": "overlaps region"
          }
        },
        {
          "id": "CGEX:VarInterp047",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA022141",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll031"
          },
          "condition": [
            {
              "id": "CGEX:GenCond013",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:324",
                  "label": "Fabry disease"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0001419",
                "label": "X linked recessive inheritance"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        },
        {
          "id": "CGEX:VarInterp048",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA022153",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll032"
          },
          "condition": [
            {
              "id": "CGEX:GenCond013",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:324",
                  "label": "Fabry disease"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0001419",
                "label": "X linked recessive inheritance"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        },
        {
          "id": "CGEX:VarInterp049",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA022159",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll033"
          },
          "condition": [
            {
              "id": "CGEX:GenCond013",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:324",
                  "label": "Fabry disease"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0001419",
                "label": "X linked recessive inheritance"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        },
        {
          "id": "CGEX:VarInterp050",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA022189",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll034"
          },
          "condition": [
            {
              "id": "CGEX:GenCond013",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:324",
                  "label": "Fabry disease"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0001419",
                "label": "X linked recessive inheritance"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        },
        {
          "id": "CGEX:BMVR102",
          "type": "BenignMissenseVariationRateStatement",
          "description": "ClinVar shows no benign missense variants in (or near) this region.  The closest is NM_000169.2:c.8T>C",
          "region": {
            "id": "CGEX:Loc434",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_000169.2",
              "label": "NM_000169.2"
            },
            "start": 996,
            "stop": 1012
          },
          "statementOutcome": {
            "id": "SEPIO:0000266",
            "label": "Low"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib418",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/3/16 7:24 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess293">CGEX:CritAssess293</a></p></td>
          <td>
              
         </td>
          <td>
              <p>PM2 is applicable as even though variant is not 100% absent, it is practically absent (a 95% CI for the population goes from 0.00% to 0.01%)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn025">CGEX:EvLn025</a></p><p><a href="details/details.html#CGEX:EvLn026">CGEX:EvLn026</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99028">SEPIO-CG:99028</a></p> (PM2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA011797">CAR:CA011797</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess293"><pre><code>{
  "id": "CGEX:CritAssess293",
  "type": "CriterionAssessment",
  "description": "PM2 is applicable as even though variant is not 100% absent, it is practically absent (a 95% CI for the population goes from 0.00% to 0.01%)",
  "criterion": {
    "id": "SEPIO-CG:99028",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA011797",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll037"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn025",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq051",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "GNOMAD:nfe",
            "label": "Non-Finnish European"
          },
          "allele": {
            "id": "CAR:CA011797",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll037"
          },
          "alleleCount": 66706,
          "alleleNumber": 1,
          "alleleFrequency": 1.499e-05,
          "homozygousAlleleIndividualCount": 0
        }
      ]
    },
    {
      "id": "CGEX:EvLn026",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq052",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA011797",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll037"
          },
          "alleleCount": 121366,
          "alleleNumber": 1,
          "alleleFrequency": 8.24e-06,
          "homozygousAlleleIndividualCount": 0
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib419",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/4/16 9:08 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess295">CGEX:CritAssess295</a></p></td>
          <td>
              
         </td>
          <td>
              <p>while the variant is absent from exac, mean coverage (11X) is insufficient to be sure that variant was fully assayed.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn028">CGEX:EvLn028</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99028">SEPIO-CG:99028</a></p> (PM2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA273141">CAR:CA273141</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess295"><pre><code>{
  "id": "CGEX:CritAssess295",
  "type": "CriterionAssessment",
  "description": "while the variant is absent from exac, mean coverage (11X) is insufficient to be sure that variant was fully assayed.",
  "criterion": {
    "id": "SEPIO-CG:99028",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA273141",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll038"
  },
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn028",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq054",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA273141",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll038"
          },
          "alleleCount": 0,
          "alleleNumber": 0,
          "alleleFrequency": 0.0,
          "medianCoverage": 11.0
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib421",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/6/16 11:28 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess298">CGEX:CritAssess298</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Application of PM3 is incorrect as the variant in trans is know known to be pathogenic (it is the same variant).</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn031">CGEX:EvLn031</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99029">SEPIO-CG:99029</a></p> (PM3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090904">CAR:CA090904</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond015">CGEX:GenCond015</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess298"><pre><code>{
  "id": "CGEX:CritAssess298",
  "type": "CriterionAssessment",
  "description": "Application of PM3 is incorrect as the variant in trans is know known to be pathogenic (it is the same variant).",
  "criterion": {
    "id": "SEPIO-CG:99029",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM3",
    "description": "For recessive disorders, detected in trans with a pathogenic variant",
    "usageNotes": "CSER - Laboratories discussed when to modify the strength of PM3, the variant is seen in trans with a pathogenic variant for recessive disorders. Published literature may not always explicitly state the phase of variants found in affected individuals which raises a challenge for invoking PM3.\nWhen phase has not been established, some felt that PM3 could be invoked as supporting evidence. Also, if the variant is seen in trans with a pathogenic variant in more than one individual it was felt that PM3 can be upgraded to strong. However, sites did not agree on how many additional observations were necessary to call the evidence strong (2 vs. 3) but concluded that such guidance would be useful.\n \nLMM - INCREASES IN STRENGTH IF VARIANT SEEN IN MULTIPLE COMPOUND HETS WITH DIFFERENT LIKELY PATHOGENIC/PATHOGENIC VARIANTS",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA090904",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll043"
  },
  "condition": [
    {
      "id": "CGEX:GenCond015",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:216",
          "label": "neuronal ceroid lipofuscinosis"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "label": "Autosomal Recessive"
      },
      "gene": {
        "id": "HGNC:28848",
        "label": "TTP1"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn031",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond064",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv338",
            "label": "individual #006"
          },
          "condition": [
            {
              "id": "CGEX:GenCond015",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:216",
                  "label": "neuronal ceroid lipofuscinosis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              },
              "gene": {
                "id": "HGNC:28848",
                "label": "TTP1"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:IndGeno065",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv338",
            "label": "individual #006"
          },
          "genotype": [
            {
              "id": "CGEX:Geno383",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo382",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA090904",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll043"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000136",
                "label": "homozygous"
              }
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib424",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/10/16 9:15 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess299">CGEX:CritAssess299</a></p></td>
          <td>
              
         </td>
          <td>
              <p>leads to an in-frame deletion of 7 amino acids (not repeat region).</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn032">CGEX:EvLn032</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99030">SEPIO-CG:99030</a></p> (PM4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090919">CAR:CA090919</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess299"><pre><code>{
  "id": "CGEX:CritAssess299",
  "type": "CriterionAssessment",
  "description": "leads to an in-frame deletion of 7 amino acids (not repeat region).",
  "criterion": {
    "id": "SEPIO-CG:99030",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM4",
    "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
    "usageNotes": "Need to ensure that variant is not located within a repeat region\n\nThis rule is also (likely) applicable for frameshift variants that result in extension of the protein as opposed to truncation (similar to stop loss).",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA090919",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll039"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn032",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon066",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll044",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA273136",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll044"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9044",
              "type": "ContextualAlleleName",
              "name": "NP_001360.1:p.Trp2490_Leu2496del",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001825",
            "label": "conservative Inframe Deletion"
          },
          "contribution": [
            {
              "id": "CGEX:Contrib486",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "6/1/16 2:03 PM",
              "contributionRole": {
                "id": "SEPIO:0000156",
                "label": "curator"
              }
            }
          ],
          "source": [
            "https://geneinsight-lmm-test.partners.org"
          ]
        },
        {
          "id": "CGEX:RegTyp069",
          "type": "RegionTypeStatement",
          "description": "This is not a repreating region.",
          "region": {
            "id": "CGEX:Loc435",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NP_001360.1",
              "label": "NP_001360.1"
            },
            "start": 2490,
            "stop": 2496
          },
          "statementOutcome": [
            {
              "id": "SO:0000657",
              "label": "repeat region "
            }
          ],
          "outcomeQualifier": "NOT",
          "contribution": [
            {
              "id": "CGEX:Contrib487",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "6/1/16 2:10 PM",
              "contributionRole": {
                "id": "SEPIO:0000156",
                "label": "curator"
              }
            }
          ],
          "source": [
            "https://pcpgm.partners.org/lmm/alamut"
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib425",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/12/16 7:30 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess300">CGEX:CritAssess300</a></p></td>
          <td>
              
         </td>
          <td>
              <p>leads to an in-frame deletion of 5 amino acids (not repeat region).</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn033">CGEX:EvLn033</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99030">SEPIO-CG:99030</a></p> (PM4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090938">CAR:CA090938</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess300"><pre><code>{
  "id": "CGEX:CritAssess300",
  "type": "CriterionAssessment",
  "description": "leads to an in-frame deletion of 5 amino acids (not repeat region).",
  "criterion": {
    "id": "SEPIO-CG:99030",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM4",
    "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
    "usageNotes": "Need to ensure that variant is not located within a repeat region\n\nThis rule is also (likely) applicable for frameshift variants that result in extension of the protein as opposed to truncation (similar to stop loss).",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA090938",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll045"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn033",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon070",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll046",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA322149",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll046"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9046",
              "type": "ContextualAlleleName",
              "name": "NP_056375.2:p.Arg38_Ser43del",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001825",
            "label": "conservative Inframe Deletion"
          }
        },
        {
          "id": "CGEX:RegTyp073",
          "type": "RegionTypeStatement",
          "description": "NP_056375.2[35..43] is not repeating.",
          "region": {
            "id": "CGEX:Loc436",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NP_056375.2",
              "label": "NP_056375.2"
            },
            "start": 35,
            "stop": 43
          },
          "statementOutcome": [
            {
              "id": "SO:0000657",
              "label": "repeat region "
            }
          ],
          "outcomeQualifier": "NOT"
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib426",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/12/16 8:31 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess301">CGEX:CritAssess301</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This 15bp duplication variant results in an expansion of a 20 residue polyalanine tract to a 25 residue polyalanine tract.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn034">CGEX:EvLn034</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99030">SEPIO-CG:99030</a></p> (PM4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501224">CAR:CA501224</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess301"><pre><code>{
  "id": "CGEX:CritAssess301",
  "type": "CriterionAssessment",
  "description": "This 15bp duplication variant results in an expansion of a 20 residue polyalanine tract to a 25 residue polyalanine tract.",
  "criterion": {
    "id": "SEPIO-CG:99030",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM4",
    "description": "Protein length changes as a result of in-frame deletions/insertions in a nonrepeat region or stop-loss variants",
    "usageNotes": "Need to ensure that variant is not located within a repeat region\n\nThis rule is also (likely) applicable for frameshift variants that result in extension of the protein as opposed to truncation (similar to stop loss).",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA501224",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll047"
  },
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn034",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon074",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll048",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA501225",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll048"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9048",
              "type": "ContextualAlleleName",
              "name": "NP_003915.2:p.Ala260_Gly261insAlaAlaAlaAlaAla",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001823",
            "label": "conservative Inframe Insertion"
          }
        },
        {
          "id": "CGEX:RegTyp077",
          "type": "RegionTypeStatement",
          "description": "NP_003915.2[241..260] is repeating.",
          "region": {
            "id": "CGEX:Loc437",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NP_003915.2",
              "label": "NP_003915.2"
            },
            "start": 241,
            "stop": 260
          },
          "statementOutcome": [
            {
              "id": "SO:0000657",
              "label": "repeat region "
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib427",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/12/16 9:32 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess302">CGEX:CritAssess302</a></p></td>
          <td>
              
         </td>
          <td>
              <p>From CLINVAR: Doolan et al. (2005) (PMID:15698845) reported mutations at the same residue (R162P) in patients with HCM, supporting the functional importance of this residue in the protein. This rare R162P variant observed multiple times in is Likely Pathogenic in ClinVar with 1 star assertion.  Other variants in the same residue R162W has conflicting evidence of pathogenicity(Path by GeneDx : SCV000209174.2 and VUS by LMM:SCV000203864.2). Considered these two only as they have provided assertion criteria.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn035">CGEX:EvLn035</a></p><p><a href="details/details.html#CGEX:EvLn036">CGEX:EvLn036</a></p><p><a href="details/details.html#CGEX:EvLn037">CGEX:EvLn037</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99031">SEPIO-CG:99031</a></p> (PM5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA021744">CAR:CA021744</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond016">CGEX:GenCond016</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess302"><pre><code>{
  "id": "CGEX:CritAssess302",
  "type": "CriterionAssessment",
  "description": "From CLINVAR: Doolan et al. (2005) (PMID:15698845) reported mutations at the same residue (R162P) in patients with HCM, supporting the functional importance of this residue in the protein. This rare R162P variant observed multiple times in is Likely Pathogenic in ClinVar with 1 star assertion.  Other variants in the same residue R162W has conflicting evidence of pathogenicity(Path by GeneDx : SCV000209174.2 and VUS by LMM:SCV000203864.2). Considered these two only as they have provided assertion criteria.",
  "criterion": {
    "id": "SEPIO-CG:99031",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM5",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "usageNotes": "Need to be sure that pathogenic variant exists at the residue in question. Sometimes discrepancy resolution may be required. Some thing like Variantexplorer.org may be useful.\nSometimes no assertion is provided in ClinVar ...may require curation by lit. search.",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA021744",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll049"
  },
  "condition": [
    {
      "id": "CGEX:GenCond016",
      "type": "GeneticCondition",
      "description": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant",
      "disease": [
        {
          "id": "Orphanet:217569",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn035",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp078",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA021749",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll051"
          },
          "condition": [
            {
              "id": "CGEX:GenCond016",
              "type": "GeneticCondition",
              "description": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant",
              "disease": [
                {
                  "id": "Orphanet:217569",
                  "label": "hypertrophic cardiomyopathy"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA26332-9",
            "label": "Likely Pathogenic"
          },
          "contribution": [
            {
              "id": "CGEX:Contrib520",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent011",
                "type": "Agent",
                "label": "Genetics Diagnostic Laboratory,Children's Hospital of Eastern Ontario"
              },
              "contributionDate": "6/1/2014 9:02 AM",
              "contributionRole": {
                "id": "SEPIO:0000331",
                "label": "interpreter"
              }
            }
          ],
          "source": [
            "https://www.ncbi.nlm.nih.gov/clinvar/SCV000059953.4"
          ]
        }
      ]
    },
    {
      "id": "CGEX:EvLn036",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp079",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA021738",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll053"
          },
          "condition": [
            {
              "id": "CGEX:GenCond016",
              "type": "GeneticCondition",
              "description": "Primary familial hypertrophic cardiomyopathy, Autosomal Dominant",
              "disease": [
                {
                  "id": "Orphanet:217569",
                  "label": "hypertrophic cardiomyopathy"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          },
          "contribution": [
            {
              "id": "CGEX:Contrib521",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent012",
                "type": "Agent",
                "label": "GeneDx"
              },
              "contributionDate": "6/20/2016 11:12 AM",
              "contributionRole": {
                "id": "SEPIO:0000331",
                "label": "interpreter"
              }
            }
          ],
          "source": [
            "https://www.ncbi.nlm.nih.gov/clinvar/SCV000203864.3"
          ]
        }
      ]
    },
    {
      "id": "CGEX:EvLn037",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp042",
          "type": "VariantPathogenicityInterpretation",
          "description": "Uncertain Sig(not specified) NM_000363.4(TNNI3):c.484C>T (p.Arg162Trp)",
          "variant": {
            "id": "CAR:CA021738",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll053"
          },
          "statementOutcome": {
            "id": "LOINC:LA26333-7",
            "label": "Uncertain Significance"
          },
          "contribution": [
            {
              "id": "CGEX:Contrib434",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "7/13/16 8:46 AM",
              "contributionRole": {
                "id": "SEPIO:0000331",
                "label": "interpreter"
              }
            }
          ],
          "source": [
            "https://www.ncbi.nlm.nih.gov/clinvar/SCV000190642.1"
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib428",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "6/12/16 10:48 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ],
  "source": [
    "PMID:15698845",
    "CLINVAR:SCV000209174.2",
    "CLINVAR:SCV000203864.2"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess303">CGEX:CritAssess303</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Other pathogenic variants at the same residue per ClinVar : NM_000531.5(OTC):c.119G>A (p.Arg40His) However no assertion criteria is provided for these variants. So pathogenicity needs to be determined by literature search. 

So PM5 is applicable when assessing p.Leu1764Il</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn038">CGEX:EvLn038</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99031">SEPIO-CG:99031</a></p> (PM5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090910">CAR:CA090910</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond017">CGEX:GenCond017</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess303"><pre><code>{
  "id": "CGEX:CritAssess303",
  "type": "CriterionAssessment",
  "description": "Other pathogenic variants at the same residue per ClinVar : NM_000531.5(OTC):c.119G>A (p.Arg40His) However no assertion criteria is provided for these variants. So pathogenicity needs to be determined by literature search. \n\nSo PM5 is applicable when assessing p.Leu1764Il",
  "criterion": {
    "id": "SEPIO-CG:99031",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM5",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "usageNotes": "Need to be sure that pathogenic variant exists at the residue in question. Sometimes discrepancy resolution may be required. Some thing like Variantexplorer.org may be useful.\nSometimes no assertion is provided in ClinVar ...may require curation by lit. search.",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA090910",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll055"
  },
  "condition": [
    {
      "id": "CGEX:GenCond017",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:664",
          "label": "Ornithine carbamoyltransferase deficiency"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn038",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp080",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA224454",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll057"
          },
          "condition": [
            {
              "id": "CGEX:GenCond017",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:664",
                  "label": "Ornithine carbamoyltransferase deficiency"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib429",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "6/12/16 11:03 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess304">CGEX:CritAssess304</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Another variant at this position, but different AA change, has been observed and called pathogenic for breast ovarian cancer by ENIGMA expert panel:</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn039">CGEX:EvLn039</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99031">SEPIO-CG:99031</a></p> (PM5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501227">CAR:CA501227</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond050">CGEX:GenCond050</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess304"><pre><code>{
  "id": "CGEX:CritAssess304",
  "type": "CriterionAssessment",
  "description": "Another variant at this position, but different AA change, has been observed and called pathogenic for breast ovarian cancer by ENIGMA expert panel:",
  "criterion": {
    "id": "SEPIO-CG:99031",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM5",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "usageNotes": "Need to be sure that pathogenic variant exists at the residue in question. Sometimes discrepancy resolution may be required. Some thing like Variantexplorer.org may be useful.\nSometimes no assertion is provided in ClinVar ...may require curation by lit. search.",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA501227",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll059"
  },
  "condition": [
    {
      "id": "CGEX:GenCond050",
      "type": "GeneticCondition",
      "description": "Breast Ovarian Cancer, Autosomal Dominant",
      "disease": [
        {
          "id": "MESH:D061325",
          "label": "hereditary breast ovarian cancer"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn039",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp081",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA003449",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll061"
          },
          "condition": [
            {
              "id": "CGEX:GenCond050",
              "type": "GeneticCondition",
              "description": "Breast Ovarian Cancer, Autosomal Dominant",
              "disease": [
                {
                  "id": "MESH:D061325",
                  "label": "hereditary breast ovarian cancer"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib430",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/1/16 9:23 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess305">CGEX:CritAssess305</a></p></td>
          <td>
              
         </td>
          <td>
              <p>while the variant is absent from exac, however
1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”
2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC).</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn040">CGEX:EvLn040</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99028">SEPIO-CG:99028</a></p> (PM2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090919">CAR:CA090919</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess305"><pre><code>{
  "id": "CGEX:CritAssess305",
  "type": "CriterionAssessment",
  "description": "while the variant is absent from exac, however\n1) very low coverage across this region (~5X) and usually 20X is used as a cut-off to say “absent”\n2) variant is 21 nt deletion - databases may not be able to call a variant of that size (I rarely see an indel of this size in ExAC).",
  "criterion": {
    "id": "SEPIO-CG:99028",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA090919",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll039"
  },
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn040",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq082",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA090919",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll039"
          },
          "alleleCount": 0,
          "alleleFrequency": 0.0,
          "medianCoverage": 11.0,
          "contribution": [
            {
              "id": "CGEX:Contrib488",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "6/1/16 2:15 PM",
              "contributionRole": {
                "id": "SEPIO:0000156",
                "label": "curator"
              }
            }
          ],
          "source": [
            "ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.2/"
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib431",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/1/16 10:53 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib432",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "7/1/16 10:53 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess306">CGEX:CritAssess306</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Two sporadic cases of BHD contain this allele</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn041">CGEX:EvLn041</a></p><p><a href="details/details.html#CGEX:EvLn042">CGEX:EvLn042</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99032">SEPIO-CG:99032</a></p> (PM6)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090951">CAR:CA090951</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond018">CGEX:GenCond018</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess306"><pre><code>{
  "id": "CGEX:CritAssess306",
  "type": "CriterionAssessment",
  "description": "Two sporadic cases of BHD contain this allele",
  "criterion": {
    "id": "SEPIO-CG:99032",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM6",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA090951",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll063"
  },
  "condition": [
    {
      "id": "CGEX:GenCond018",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:122",
          "label": "Birt-Hogg-Dube Syndrome"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn041",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond083",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv339",
            "label": "individual #007"
          },
          "condition": [
            {
              "id": "CGEX:GenCond018",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:122",
                  "label": "Birt-Hogg-Dube Syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:FamCond085",
          "type": "FamilyConditionStatement",
          "proband": {
            "id": "CGEX:Indiv339",
            "label": "individual #007"
          },
          "condition": [
            {
              "id": "CGEX:GenCond018",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:122",
                  "label": "Birt-Hogg-Dube Syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        },
        {
          "id": "CGEX:IndAllInh087",
          "type": "IndividualAlleleInheritanceStatement",
          "description": "parentage confirmation not mentioned in the source article",
          "canonicalAllele": {
            "id": "CAR:CA090951",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll063"
          },
          "individual": {
            "id": "CGEX:Indiv339",
            "label": "individual #007"
          },
          "parentalConfirmation": {
            "id": "SEPIO-CG:99008",
            "label": "no parental confirmation"
          },
          "statementOutcome": [
            {
              "id": "GENO:0000880",
              "label": "de novo"
            }
          ]
        }
      ]
    },
    {
      "id": "CGEX:EvLn042",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond084",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv340",
            "label": "individual #008"
          },
          "condition": [
            {
              "id": "CGEX:GenCond018",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:122",
                  "label": "Birt-Hogg-Dube Syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:FamCond086",
          "type": "FamilyConditionStatement",
          "proband": {
            "id": "CGEX:Indiv340",
            "label": "individual #008"
          },
          "condition": [
            {
              "id": "CGEX:GenCond018",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:122",
                  "label": "Birt-Hogg-Dube Syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        },
        {
          "id": "CGEX:IndAllInh088",
          "type": "IndividualAlleleInheritanceStatement",
          "description": "parentage confirmation not mentioned in the source article",
          "canonicalAllele": {
            "id": "CAR:CA090951",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll063"
          },
          "individual": {
            "id": "CGEX:Indiv340",
            "label": "individual #008"
          },
          "parentalConfirmation": {
            "id": "SEPIO-CG:99008",
            "label": "no parental confirmation"
          },
          "statementOutcome": [
            {
              "id": "GENO:0000880",
              "label": "de novo"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib435",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "7/20/16 7:22 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib436",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "7/20/16 7:22 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess307">CGEX:CritAssess307</a></p></td>
          <td>
              
         </td>
          <td>
              <p>affected child with CFC was found to carry c.158T>C. Both parents are unaffected and from Sanger seq neither parent was found to carry c.158T>C variant - so authors conclude variant occurred de novo. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn043">CGEX:EvLn043</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99032">SEPIO-CG:99032</a></p> (PM6)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA279966">CAR:CA279966</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond019">CGEX:GenCond019</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess307"><pre><code>{
  "id": "CGEX:CritAssess307",
  "type": "CriterionAssessment",
  "description": "affected child with CFC was found to carry c.158T>C. Both parents are unaffected and from Sanger seq neither parent was found to carry c.158T>C variant - so authors conclude variant occurred de novo. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable",
  "criterion": {
    "id": "SEPIO-CG:99032",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM6",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA279966",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll064"
  },
  "condition": [
    {
      "id": "CGEX:GenCond019",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:1340",
          "label": "cardiofaciocutaneous syndrome"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn043",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond089",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv341",
            "label": "individual #009"
          },
          "condition": [
            {
              "id": "CGEX:GenCond019",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:FamCond090",
          "type": "FamilyConditionStatement",
          "proband": {
            "id": "CGEX:Indiv341",
            "label": "individual #009"
          },
          "condition": [
            {
              "id": "CGEX:GenCond019",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        },
        {
          "id": "CGEX:IndAllInh091",
          "type": "IndividualAlleleInheritanceStatement",
          "description": "parentage confirmation not mentioned in the source article",
          "canonicalAllele": {
            "id": "CAR:CA279966",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll064"
          },
          "individual": {
            "id": "CGEX:Indiv341",
            "label": "individual #009"
          },
          "parentalConfirmation": {
            "id": "SEPIO-CG:99008",
            "label": "no parental confirmation"
          },
          "statementOutcome": [
            {
              "id": "GENO:0000880",
              "label": "de novo"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib437",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/20/16 7:49 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess308">CGEX:CritAssess308</a></p></td>
          <td>
              
         </td>
          <td>
              <p>3 unrelated probands with CFC were found to carry c.1741A>G variant. There is no family history of CFC in any family and all three sets of parents were confirmed to not have the variant. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable. However, since there are 3 de novo observations, many groups (including Noonan and MYH7) would allow PM6 to be upgraded to Strong (PM6_S)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn044">CGEX:EvLn044</a></p><p><a href="details/details.html#CGEX:EvLn045">CGEX:EvLn045</a></p><p><a href="details/details.html#CGEX:EvLn046">CGEX:EvLn046</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99032">SEPIO-CG:99032</a></p> (PM6)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA279976">CAR:CA279976</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond068">CGEX:GenCond068</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess308"><pre><code>{
  "id": "CGEX:CritAssess308",
  "type": "CriterionAssessment",
  "description": "3 unrelated probands with CFC were found to carry c.1741A>G variant. There is no family history of CFC in any family and all three sets of parents were confirmed to not have the variant. Since there was no mention of parental confirmation, PS2 cannot be used but PM6 is applicable. However, since there are 3 de novo observations, many groups (including Noonan and MYH7) would allow PM6 to be upgraded to Strong (PM6_S)",
  "criterion": {
    "id": "SEPIO-CG:99032",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM6",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA279976",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll065"
  },
  "condition": [
    {
      "id": "CGEX:GenCond068",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:1340",
          "label": "cardiofaciocutaneous syndrome"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn044",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond092",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv342",
            "label": "individual #010"
          },
          "condition": [
            {
              "id": "CGEX:GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:FamCond093",
          "type": "FamilyConditionStatement",
          "proband": {
            "id": "CGEX:Indiv342",
            "label": "individual #010"
          },
          "condition": [
            {
              "id": "CGEX:GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        },
        {
          "id": "CGEX:IndAllInh094",
          "type": "IndividualAlleleInheritanceStatement",
          "canonicalAllele": {
            "id": "CAR:CA279976",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll065"
          },
          "individual": {
            "id": "CGEX:Indiv342",
            "label": "individual #010"
          },
          "parentalConfirmation": {
            "id": "SEPIO-CG:99008",
            "label": "no parental confirmation"
          },
          "statementOutcome": [
            {
              "id": "GENO:0000880",
              "label": "de novo"
            }
          ]
        }
      ]
    },
    {
      "id": "CGEX:EvLn045",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond095",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv343",
            "label": "individual #011"
          },
          "condition": [
            {
              "id": "CGEX:GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:FamCond096",
          "type": "FamilyConditionStatement",
          "proband": {
            "id": "CGEX:Indiv343",
            "label": "individual #011"
          },
          "condition": [
            {
              "id": "CGEX:GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        },
        {
          "id": "CGEX:IndAllInh097",
          "type": "IndividualAlleleInheritanceStatement",
          "canonicalAllele": {
            "id": "CAR:CA279976",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll065"
          },
          "individual": {
            "id": "CGEX:Indiv343",
            "label": "individual #011"
          },
          "parentalConfirmation": {
            "id": "SEPIO-CG:99008",
            "label": "no parental confirmation"
          },
          "statementOutcome": [
            {
              "id": "GENO:0000880",
              "label": "de novo"
            }
          ]
        }
      ]
    },
    {
      "id": "CGEX:EvLn046",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond098",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv344",
            "label": "individual #012"
          },
          "condition": [
            {
              "id": "CGEX:GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:FamCond099",
          "type": "FamilyConditionStatement",
          "proband": {
            "id": "CGEX:Indiv344",
            "label": "individual #012"
          },
          "condition": [
            {
              "id": "CGEX:GenCond068",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        },
        {
          "id": "CGEX:IndAllInh100",
          "type": "IndividualAlleleInheritanceStatement",
          "canonicalAllele": {
            "id": "CAR:CA279976",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll065"
          },
          "individual": {
            "id": "CGEX:Indiv344",
            "label": "individual #012"
          },
          "parentalConfirmation": {
            "id": "SEPIO-CG:99008",
            "label": "no parental confirmation"
          },
          "statementOutcome": [
            {
              "id": "GENO:0000880",
              "label": "de novo"
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib438",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/20/16 8:02 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess309">CGEX:CritAssess309</a></p></td>
          <td>
              
         </td>
          <td>
              <p>In total, 4 segs from 2 families</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn047">CGEX:EvLn047</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99033">SEPIO-CG:99033</a></p> (PP1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA010136">CAR:CA010136</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond020">CGEX:GenCond020</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess309"><pre><code>{
  "id": "CGEX:CritAssess309",
  "type": "CriterionAssessment",
  "description": "In total, 4 segs from 2 families",
  "criterion": {
    "id": "SEPIO-CG:99033",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP1",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA010136",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll067"
  },
  "condition": [
    {
      "id": "CGEX:GenCond020",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217569",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn047",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AggSeg105",
          "type": "AggregateCosegregationStatement",
          "condition": {
            "id": "CGEX:GenCond020",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:217569",
                "label": "hypertrophic cardiomyopathy"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CAR:CA010136",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll067"
          },
          "familySegregation": [
            {
              "id": "CGEX:FamSeg103",
              "type": "FamilyCosegregationStatement",
              "description": "identified in 1 individual with HCM. Segregation analysis determined 5 other members of the family also carried this variant. However, only 3 of those family members had HCM (aged 48-75). The 2 other family members were young and asymptomatic. So from this paper, only count 3 segregations. The two apparently refuting segregations can be ignored due to the age-dependent penetrance of HCM.",
              "condition": {
                "id": "CGEX:GenCond020",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "Orphanet:217569",
                    "label": "hypertrophic cardiomyopathy"
                  }
                ],
                "inheritancePattern": {
                  "id": "HP:0000006",
                  "label": "Autosomal Dominant"
                }
              },
              "canonicalAllele": {
                "id": "CAR:CA010136",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll067"
              },
              "family": {
                "id": "CGEX:Fam363",
                "label": "Family #001"
              },
              "phenotypePositiveAllelePositive": 3,
              "phenotypeNegativeAllelePositive": 2,
              "contribution": [
                {
                  "id": "CGEX:Contrib481",
                  "type": "Contribution",
                  "agent": {
                    "id": "CGEX:Agent007",
                    "type": "Agent",
                    "label": "Stefan DiMarino"
                  },
                  "contributionDate": "9/13/16",
                  "contributionRole": {
                    "id": "SEPIO:0000156",
                    "label": "curator"
                  }
                }
              ],
              "source": [
                "http://www.ncbi.nlm.nih.gov/pubmed/17081393"
              ]
            },
            {
              "id": "CGEX:FamSeg104",
              "type": "FamilyCosegregationStatement",
              "description": "In LMM internal family, c.1351+1G>A was identified in 1 individual with HCM. Variant also found in individual’s sister - who also had clinical dx of HCM.",
              "condition": {
                "id": "CGEX:GenCond020",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "Orphanet:217569",
                    "label": "hypertrophic cardiomyopathy"
                  }
                ],
                "inheritancePattern": {
                  "id": "HP:0000006",
                  "label": "Autosomal Dominant"
                }
              },
              "canonicalAllele": {
                "id": "CAR:CA010136",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll067"
              },
              "family": {
                "id": "CGEX:Fam364",
                "label": "Family #002"
              },
              "phenotypePositiveAllelePositive": 1,
              "contribution": [
                {
                  "id": "CGEX:Contrib482",
                  "type": "Contribution",
                  "agent": {
                    "id": "CGEX:Agent001",
                    "type": "Agent",
                    "label": "Harry Stevenson"
                  },
                  "contributionDate": "9/13/16",
                  "contributionRole": {
                    "id": "SEPIO:0000156",
                    "label": "curator"
                  }
                }
              ],
              "source": [
                "http://partners.org/ppm/lmm/gi/variant/12345"
              ]
            }
          ],
          "totalNumberOfSegregations": 4
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib439",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 6:45 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess310">CGEX:CritAssess310</a></p></td>
          <td>
              
         </td>
          <td>
              <p>In total, 8 segs from 2 families (can combine hom segs and compound het segs). Most groups would allow this to be shifted up the Moderate (PP1_M)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn048">CGEX:EvLn048</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99033">SEPIO-CG:99033</a></p> (PP1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA143213">CAR:CA143213</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond021">CGEX:GenCond021</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess310"><pre><code>{
  "id": "CGEX:CritAssess310",
  "type": "CriterionAssessment",
  "description": "In total, 8 segs from 2 families (can combine hom segs and compound het segs). Most groups would allow this to be shifted up the Moderate (PP1_M)",
  "criterion": {
    "id": "SEPIO-CG:99033",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP1",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA143213",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll066"
  },
  "condition": [
    {
      "id": "CGEX:GenCond021",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:886",
          "label": "Usher syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn048",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AggSeg109",
          "type": "AggregateCosegregationStatement",
          "condition": {
            "id": "CGEX:GenCond021",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:886",
                "label": "Usher syndrome"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000007",
              "label": "Autosomal Recessive"
            }
          },
          "canonicalAllele": {
            "id": "CAR:CA143213",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll066"
          },
          "familySegregation": [
            {
              "id": "CGEX:FamSeg107",
              "type": "FamilyCosegregationStatement",
              "description": "proband and 5 affected family members all homozygous for NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly).",
              "condition": {
                "id": "CGEX:GenCond021",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "Orphanet:886",
                    "label": "Usher syndrome"
                  }
                ],
                "inheritancePattern": {
                  "id": "HP:0000007",
                  "label": "Autosomal Recessive"
                }
              },
              "canonicalAllele": {
                "id": "CAR:CA143213",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll066"
              },
              "family": {
                "id": "CGEX:Fam365",
                "label": "Family #003"
              },
              "phenotypePositiveAllelePositive": 5
            },
            {
              "id": "CGEX:FamSeg108",
              "type": "FamilyCosegregationStatement",
              "description": "c.1000C>T identified in het state in 1 individual with Usher. Individual compound het with Gly1840Val USH2A variant - confirmed in trans. Gly1840Val called Pathogenic. Individual has 3 affected sibs all of whom are compound het with 1000C>T & Gly1840Val. One additional sib only carried 1000C>T and is unaffected.",
              "condition": {
                "id": "CGEX:GenCond021",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "Orphanet:886",
                    "label": "Usher syndrome"
                  }
                ],
                "inheritancePattern": {
                  "id": "HP:0000007",
                  "label": "Autosomal Recessive"
                }
              },
              "canonicalAllele": {
                "id": "CAR:CA143213",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll066"
              },
              "family": {
                "id": "CGEX:Fam366",
                "label": "Family #004"
              }
            }
          ],
          "totalNumberOfSegregations": 8
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib440",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 7:02 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess311">CGEX:CritAssess311</a></p></td>
          <td>
              
         </td>
          <td>
              <p>PP1 is for “Co-segregation with disease in multiple affected family members” PP1 is not applicable for this variant because it’s only a single segregation.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn049">CGEX:EvLn049</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99033">SEPIO-CG:99033</a></p> (PP1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501226">CAR:CA501226</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond051">CGEX:GenCond051</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess311"><pre><code>{
  "id": "CGEX:CritAssess311",
  "type": "CriterionAssessment",
  "description": "PP1 is for “Co-segregation with disease in multiple affected family members” PP1 is not applicable for this variant because it’s only a single segregation.",
  "criterion": {
    "id": "SEPIO-CG:99033",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP1",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA501226",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll068"
  },
  "condition": [
    {
      "id": "CGEX:GenCond051",
      "type": "GeneticCondition",
      "description": "Breast Ovarian Cancer, Autosomal Dominant",
      "disease": [
        {
          "id": "MESH:D061325",
          "label": "hereditary breast ovarian cancer"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn049",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:FamSeg111",
          "type": "FamilyCosegregationStatement",
          "condition": {
            "id": "CGEX:GenCond051",
            "type": "GeneticCondition",
            "description": "Breast Ovarian Cancer, Autosomal Dominant",
            "disease": [
              {
                "id": "MESH:D061325",
                "label": "hereditary breast ovarian cancer"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CAR:CA501226",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll068"
          },
          "family": {
            "id": "CGEX:Fam367",
            "label": "Family #005"
          },
          "phenotypePositiveAllelePositive": 2
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib441",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 7:39 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess316">CGEX:CritAssess316</a></p></td>
          <td>
              
         </td>
          <td>
              <p>even though the computational tools for AA changes are NOT predicting an impact, splicing tools are predicting an impact - so PP3 is applicable</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn054">CGEX:EvLn054</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99035">SEPIO-CG:99035</a></p> (PP3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA022019">CAR:CA022019</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess316"><pre><code>{
  "id": "CGEX:CritAssess316",
  "type": "CriterionAssessment",
  "description": "even though the computational tools for AA changes are NOT predicting an impact, splicing tools are predicting an impact - so PP3 is applicable",
  "criterion": {
    "id": "SEPIO-CG:99035",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP3",
    "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "usageNotes": "Don’t know if there is a definition of “multiple” . If >1 then the algorithms chosen are at the discretion of the curator/lab.  There is a difference between lines of evidence and number of implementations.",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA022019",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll075"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn054",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:InSilicoPred138",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "SIFT",
          "statementOutcome": {
            "id": "CGEX:PredOutcome011",
            "label": "Tolerated"
          }
        },
        {
          "id": "CGEX:InSilicoPred139",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Polyphen2",
          "statementOutcome": {
            "id": "CGEX:PredOutcome001",
            "label": "Benign"
          }
        },
        {
          "id": "CGEX:InSilicoPred140",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "AlignGVGD",
          "statementOutcome": {
            "id": "CGEX:PredOutcome002",
            "label": "C0 (benign)"
          }
        },
        {
          "id": "CGEX:InSilicoPred141",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "MAPP",
          "statementOutcome": {
            "id": "CGEX:PredOutcome005",
            "label": "Disease-causing"
          }
        },
        {
          "id": "CGEX:InSilicoPred142",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "MutationTaster",
          "statementOutcome": {
            "id": "CGEX:PredOutcome005",
            "label": "Disease-causing"
          }
        },
        {
          "id": "CGEX:InSilicoPredScore143",
          "type": "InSilicoPredictionScoreStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000323",
            "label": "splicing prediction"
          },
          "algorithm": "SSF",
          "prediction": 71.53
        },
        {
          "id": "CGEX:InSilicoPredScore144",
          "type": "InSilicoPredictionScoreStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000323",
            "label": "splicing prediction"
          },
          "algorithm": "MaxEnt",
          "prediction": 7.39
        },
        {
          "id": "CGEX:InSilicoPredScore145",
          "type": "InSilicoPredictionScoreStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000323",
            "label": "splicing prediction"
          },
          "algorithm": "NNSPLICE",
          "prediction": 0.87
        },
        {
          "id": "CGEX:InSilicoPredScore146",
          "type": "InSilicoPredictionScoreStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000323",
            "label": "splicing prediction"
          },
          "algorithm": "GeneSplicer",
          "prediction": 7.75
        },
        {
          "id": "CGEX:InSilicoPredScore147",
          "type": "InSilicoPredictionScoreStatement",
          "transcript": {
            "id": "REFSEQ:NM_000363.4",
            "label": "NM_000363.4"
          },
          "canonicalAllele": {
            "id": "CAR:CA022019",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll075"
          },
          "predictionType": {
            "id": "SEPIO:0000323",
            "label": "splicing prediction"
          },
          "algorithm": "HSF",
          "prediction": 81.75
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib447",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 2:46 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess321">CGEX:CritAssess321</a></p></td>
          <td>
              
         </td>
          <td>
              <p>OMIM is not considered an expert or reputable source for variant interpretations.
Further, OMIM’s evidence for this pathogenic call is available (the publication link - PMID:21620354 - and free text summary) so people should use the evidence from OMIM in their assessment NOT OMIM’s interpretation.  This is marked as Insufficient Evidence rather than refuted because the OMIM entry does not rule out a later application of PP5 based on eg. an expert panel.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn058">CGEX:EvLn058</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99037">SEPIO-CG:99037</a></p> (PP5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA128479">CAR:CA128479</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond024">CGEX:GenCond024</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess321"><pre><code>{
  "id": "CGEX:CritAssess321",
  "type": "CriterionAssessment",
  "description": "OMIM is not considered an expert or reputable source for variant interpretations.\nFurther, OMIM’s evidence for this pathogenic call is available (the publication link - PMID:21620354 - and free text summary) so people should use the evidence from OMIM in their assessment NOT OMIM’s interpretation.  This is marked as Insufficient Evidence rather than refuted because the OMIM entry does not rule out a later application of PP5 based on eg. an expert panel.",
  "criterion": {
    "id": "SEPIO-CG:99037",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP5",
    "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA128479",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll079"
  },
  "condition": [
    {
      "id": "CGEX:GenCond024",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "OMIM:614065",
          "label": "Myopathy, Distal, type 4"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn058",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp151",
          "type": "VariantPathogenicityInterpretation",
          "description": "Variant has been classified as “Pathogenic” by OMIM (SCV000043718.1)",
          "variant": {
            "id": "CAR:CA128479",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll079"
          },
          "condition": [
            {
              "id": "CGEX:GenCond024",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "OMIM:614065",
                  "label": "Myopathy, Distal, type 4"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib453",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 5:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ],
  "source": [
    "PMID:21620354"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess324">CGEX:CritAssess324</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Globally, this variant has a MAF of 0.0454 (4.5%) which would put it under the 5% rule and thus BA1 would not be applicable. However, variant is enriched in East Asian population - 29% MAF - so way over the 5% cut-off. So now BA1 is applicable</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn061">CGEX:EvLn061</a></p><p><a href="details/details.html#CGEX:EvLn062">CGEX:EvLn062</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99038">SEPIO-CG:99038</a></p> (BA1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA134997">CAR:CA134997</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess324"><pre><code>{
  "id": "CGEX:CritAssess324",
  "type": "CriterionAssessment",
  "description": "Globally, this variant has a MAF of 0.0454 (4.5%) which would put it under the 5% rule and thus BA1 would not be applicable. However, variant is enriched in East Asian population - 29% MAF - so way over the 5% cut-off. So now BA1 is applicable",
  "criterion": {
    "id": "SEPIO-CG:99038",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BA1",
    "description": "Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "usageNotes": "Several groups have lowered the BA1 threshold (e.g. PTEN-1%). Also the sequence variant interpretation group is working on a modified version of this guideline including the minimum # of alleles that need to be examined and the population stratification effects, if any.",
    "defaultStrength": {
      "id": "SEPIO:0000325",
      "label": "Benign Stand Alone"
    }
  },
  "variant": {
    "id": "CAR:CA134997",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll082"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn061",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq154",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA134997",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll082"
          },
          "alleleCount": 5507,
          "alleleNumber": 121350,
          "alleleFrequency": 0.04538,
          "homozygousAlleleIndividualCount": 768
        }
      ]
    },
    {
      "id": "CGEX:EvLn062",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq155",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "GNOMAD:eas",
            "label": "East Asian"
          },
          "allele": {
            "id": "CAR:CA134997",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll082"
          },
          "alleleCount": 2518,
          "alleleNumber": 8632,
          "alleleFrequency": 0.2917,
          "homozygousAlleleIndividualCount": 405
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib456",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 8:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess325">CGEX:CritAssess325</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Although this variant has been seen in the general population, its frequency is low enough to be consistent with a recessive carrier frequency.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn063">CGEX:EvLn063</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99028">SEPIO-CG:99028</a></p> (PM2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501229">CAR:CA501229</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond045">CGEX:GenCond045</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess325"><pre><code>{
  "id": "CGEX:CritAssess325",
  "type": "CriterionAssessment",
  "description": "Although this variant has been seen in the general population, its frequency is low enough to be consistent with a recessive carrier frequency.",
  "criterion": {
    "id": "SEPIO-CG:99028",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA501229",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll083"
  },
  "condition": [
    {
      "id": "CGEX:GenCond045",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:244",
          "label": "Primary ciliary dyskenisia"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn063",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq157",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA501229",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll083"
          },
          "alleleCount": 14,
          "alleleNumber": 121100,
          "alleleFrequency": 0.0001158,
          "homozygousAlleleIndividualCount": 0
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib457",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 9:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess328">CGEX:CritAssess328</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Perrault syndrome may be underreported due to misdiagnosis.  If the prevalence estimates rise, this assessment may need to be revisited</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn066">CGEX:EvLn066</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99039">SEPIO-CG:99039</a></p> (BS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501230">CAR:CA501230</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond028">CGEX:GenCond028</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess328"><pre><code>{
  "id": "CGEX:CritAssess328",
  "type": "CriterionAssessment",
  "description": "Perrault syndrome may be underreported due to misdiagnosis.  If the prevalence estimates rise, this assessment may need to be revisited",
  "criterion": {
    "id": "SEPIO-CG:99039",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA501230",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll086"
  },
  "condition": [
    {
      "id": "CGEX:GenCond028",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:2855",
          "label": "Perrault Syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn066",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq166",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA501230",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll086"
          },
          "alleleCount": 3,
          "alleleNumber": 120870,
          "alleleFrequency": 2.4e-05
        },
        {
          "id": "CGEX:CondPrev167",
          "type": "ConditionPrevelanceStatement",
          "condition": [
            {
              "id": "CGEX:GenCond028",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:2855",
                  "label": "Perrault Syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "maximum": 1.0e-06,
          "population": {
            "id": "SEPIO-CG:98002",
            "label": "Overall"
          }
        },
        {
          "id": "CGEX:CondPenet168",
          "type": "ConditionPenetranceStatement",
          "condition": [
            {
              "id": "CGEX:GenCond028",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:2855",
                  "label": "Perrault Syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX2",
            "label": "High"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib461",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "8/30/16 6:00 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess329">CGEX:CritAssess329</a></p></td>
          <td>
              
         </td>
          <td>
              <p>According to Baylor internal data on this variant, it has been observed multiple times in asymptomatic parents internally and in controls and observed once as a homozygotes internally in unaffected.  Although the evidence for full penetrance is weak, the fact that there are multiple asymptomatic occurrences helps the case.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn067">CGEX:EvLn067</a></p><p><a href="details/details.html#CGEX:EvLn068">CGEX:EvLn068</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99040">SEPIO-CG:99040</a></p> (BS2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090918">CAR:CA090918</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond029">CGEX:GenCond029</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess329"><pre><code>{
  "id": "CGEX:CritAssess329",
  "type": "CriterionAssessment",
  "description": "According to Baylor internal data on this variant, it has been observed multiple times in asymptomatic parents internally and in controls and observed once as a homozygotes internally in unaffected.  Although the evidence for full penetrance is weak, the fact that there are multiple asymptomatic occurrences helps the case.",
  "criterion": {
    "id": "SEPIO-CG:99040",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS2",
    "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
    "usageNotes": "This rule should not be used for diseases with reduced penetrance or late/variable age of  onset (such as certain cardiomyopathies).",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA090918",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll087"
  },
  "condition": [
    {
      "id": "CGEX:GenCond029",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:319182",
          "label": "Wiedemann-Steiner syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn067",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CondPenet170",
          "type": "ConditionPenetranceStatement",
          "description": "The evidence for full penetrance is not great, due to the rarity of the disease, but the fact that it's diagnosed at a young age and majority of cases are de novo is suggestive",
          "condition": [
            {
              "id": "CGEX:GenCond029",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:319182",
                  "label": "Wiedemann-Steiner syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX1",
            "label": "Complete"
          }
        },
        {
          "id": "CGEX:IndGeno172",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv345",
            "label": "individual #013"
          },
          "genotype": [
            {
              "id": "CGEX:Geno385",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo384",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA090918",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll087"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000135",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "CGEX:IndCond174",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv345",
            "label": "individual #013"
          },
          "condition": [
            {
              "id": "CGEX:GenCond029",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:319182",
                  "label": "Wiedemann-Steiner syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn068",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CondPenet170",
          "type": "ConditionPenetranceStatement",
          "description": "The evidence for full penetrance is not great, due to the rarity of the disease, but the fact that it's diagnosed at a young age and majority of cases are de novo is suggestive",
          "condition": [
            {
              "id": "CGEX:GenCond029",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:319182",
                  "label": "Wiedemann-Steiner syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX1",
            "label": "Complete"
          }
        },
        {
          "id": "CGEX:IndGeno173",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv346",
            "label": "individual #014"
          },
          "genotype": [
            {
              "id": "CGEX:Geno387",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo386",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA090918",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll087"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000136",
                "label": "homozygous"
              }
            }
          ]
        },
        {
          "id": "CGEX:IndCond175",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv346",
            "label": "individual #014"
          },
          "condition": [
            {
              "id": "CGEX:GenCond029",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:319182",
                  "label": "Wiedemann-Steiner syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib462",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/5/16 6:00 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess330">CGEX:CritAssess330</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This variant was observed in unaffected healthy control chromosomes. PMIDs: 15122587, 18199528, 22703879, 24728327</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn069">CGEX:EvLn069</a></p><p><a href="details/details.html#CGEX:EvLn070">CGEX:EvLn070</a></p><p><a href="details/details.html#CGEX:EvLn071">CGEX:EvLn071</a></p><p><a href="details/details.html#CGEX:EvLn072">CGEX:EvLn072</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99040">SEPIO-CG:99040</a></p> (BS2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA008351">CAR:CA008351</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond030">CGEX:GenCond030</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess330"><pre><code>{
  "id": "CGEX:CritAssess330",
  "type": "CriterionAssessment",
  "description": "This variant was observed in unaffected healthy control chromosomes. PMIDs: 15122587, 18199528, 22703879, 24728327",
  "criterion": {
    "id": "SEPIO-CG:99040",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS2",
    "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
    "usageNotes": "This rule should not be used for diseases with reduced penetrance or late/variable age of  onset (such as certain cardiomyopathies).",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA008351",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll088"
  },
  "condition": [
    {
      "id": "CGEX:GenCond030",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:733",
          "label": "Familial Adenomatous Polyposis"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn069",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CondPenet176",
          "type": "ConditionPenetranceStatement",
          "description": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early.",
          "condition": [
            {
              "id": "CGEX:GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:733",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX1",
            "label": "Complete"
          }
        },
        {
          "id": "CGEX:IndGeno178",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv347",
            "label": "individual #015"
          },
          "genotype": [
            {
              "id": "CGEX:Geno389",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo388",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA008351",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll088"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000135",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "CGEX:IndCond182",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv347",
            "label": "individual #015"
          },
          "condition": [
            {
              "id": "CGEX:GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:733",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn070",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CondPenet176",
          "type": "ConditionPenetranceStatement",
          "description": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early.",
          "condition": [
            {
              "id": "CGEX:GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:733",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX1",
            "label": "Complete"
          }
        },
        {
          "id": "CGEX:IndGeno179",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv348",
            "label": "individual #016"
          },
          "genotype": [
            {
              "id": "CGEX:Geno391",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo390",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA008351",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll088"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000135",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "CGEX:IndCond183",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv348",
            "label": "individual #016"
          },
          "condition": [
            {
              "id": "CGEX:GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:733",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn071",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CondPenet176",
          "type": "ConditionPenetranceStatement",
          "description": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early.",
          "condition": [
            {
              "id": "CGEX:GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:733",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX1",
            "label": "Complete"
          }
        },
        {
          "id": "CGEX:IndGeno180",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv349",
            "label": "individual #017"
          },
          "genotype": [
            {
              "id": "CGEX:Geno393",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo392",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA008351",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll088"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000135",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "CGEX:IndCond184",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv349",
            "label": "individual #017"
          },
          "condition": [
            {
              "id": "CGEX:GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:733",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn072",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CondPenet176",
          "type": "ConditionPenetranceStatement",
          "description": "the disease manifests as hundreds of colorectal ademonas during adolescence. This can progress to colorectal cancer later in life, but the adenomas manifest early.",
          "condition": [
            {
              "id": "CGEX:GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:733",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX1",
            "label": "Complete"
          }
        },
        {
          "id": "CGEX:IndGeno181",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv350",
            "label": "individual #018"
          },
          "genotype": [
            {
              "id": "CGEX:Geno395",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo394",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA008351",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll088"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000135",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "CGEX:IndCond185",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv350",
            "label": "individual #018"
          },
          "condition": [
            {
              "id": "CGEX:GenCond030",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:733",
                  "label": "Familial Adenomatous Polyposis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib463",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/5/16 6:30 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib464",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "9/5/16 6:30 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess331">CGEX:CritAssess331</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Although this variant was found in 5 individuals with Marfan Syndrome, it was identified in 2 unaffected relatives from two separate families. PMID:12402346</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn073">CGEX:EvLn073</a></p><p><a href="details/details.html#CGEX:EvLn074">CGEX:EvLn074</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99040">SEPIO-CG:99040</a></p> (BS2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA013604">CAR:CA013604</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond072">CGEX:GenCond072</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess331"><pre><code>{
  "id": "CGEX:CritAssess331",
  "type": "CriterionAssessment",
  "description": "Although this variant was found in 5 individuals with Marfan Syndrome, it was identified in 2 unaffected relatives from two separate families. PMID:12402346",
  "criterion": {
    "id": "SEPIO-CG:99040",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS2",
    "description": "Observed in a healthy adult individual for a recessive (homozygous), dominant (heterozygous), or X-linked (hemizygous) disorder, with full penetrance expected at an early age",
    "usageNotes": "This rule should not be used for diseases with reduced penetrance or late/variable age of  onset (such as certain cardiomyopathies).",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA013604",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll089"
  },
  "condition": [
    {
      "id": "CGEX:GenCond072",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:558",
          "label": "Marfan Syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn073",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CondPenet186",
          "type": "ConditionPenetranceStatement",
          "description": "Marfan syndrome involves a range of skeletal features that should be identifiable at birth, if not during adolescence. The ocular and cardiovascular aspects may manifest later. 1/4 of mutations are de novo.",
          "condition": [
            {
              "id": "CGEX:GenCond072",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:558",
                  "label": "Marfan Syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX1",
            "label": "Complete"
          }
        },
        {
          "id": "CGEX:IndGeno188",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv351",
            "label": "individual #019"
          },
          "genotype": [
            {
              "id": "CGEX:Geno397",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo396",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA013604",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll089"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000135",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "CGEX:IndCond190",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv351",
            "label": "individual #019"
          },
          "condition": [
            {
              "id": "CGEX:GenCond072",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:558",
                  "label": "Marfan Syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn074",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CondPenet186",
          "type": "ConditionPenetranceStatement",
          "description": "Marfan syndrome involves a range of skeletal features that should be identifiable at birth, if not during adolescence. The ocular and cardiovascular aspects may manifest later. 1/4 of mutations are de novo.",
          "condition": [
            {
              "id": "CGEX:GenCond072",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:558",
                  "label": "Marfan Syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX1",
            "label": "Complete"
          }
        },
        {
          "id": "CGEX:IndGeno189",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv352",
            "label": "individual #020"
          },
          "genotype": [
            {
              "id": "CGEX:Geno399",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo398",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA013604",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll089"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000135",
                "label": "heterozygous"
              }
            }
          ]
        },
        {
          "id": "CGEX:IndCond191",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv352",
            "label": "individual #020"
          },
          "condition": [
            {
              "id": "CGEX:GenCond072",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:558",
                  "label": "Marfan Syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib465",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/5/16 7:01 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib466",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "9/5/16 7:01 AM",
      "contributionRole": {
        "id": "SEPIO:0000331",
        "label": "interpreter"
      }
    }
  ],
  "source": [
    "PMID:12402346"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess336">CGEX:CritAssess336</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The inconsistent segregation data is reasonably explainable as a phenocopy, and therefore does not provide evidence for the benignity of this allele.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn080">CGEX:EvLn080</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99042">SEPIO-CG:99042</a></p> (BS4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA010777">CAR:CA010777</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond065">CGEX:GenCond065</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess336"><pre><code>{
  "id": "CGEX:CritAssess336",
  "type": "CriterionAssessment",
  "description": "The inconsistent segregation data is reasonably explainable as a phenocopy, and therefore does not provide evidence for the benignity of this allele.",
  "criterion": {
    "id": "SEPIO-CG:99042",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS4",
    "description": "Lack of segregation in affected members of a family",
    "usageNotes": "This rule is for “lack of segregation” meaning phenotype positive / genotype negative (and not pheno neg / geno pos). Also need to be careful of phenocopies. Also need rules for how many cases are needed in use this rule",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA010777",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll095"
  },
  "condition": [
    {
      "id": "CGEX:GenCond065",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217569",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn080",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:FamSeg197",
          "type": "FamilyCosegregationStatement",
          "description": "The individual with HCM but without the allele is likely a phenocopy explained\nby environmental factors (obese, alcohol)",
          "condition": {
            "id": "CGEX:GenCond065",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:217569",
                "label": "hypertrophic cardiomyopathy"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CAR:CA010777",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll095"
          },
          "family": {
            "id": "CGEX:Fam371",
            "label": "Family #009"
          },
          "phenotypePositiveAllelePositive": 3,
          "phenotypePositiveAlleleNegative": 1,
          "inconsistentSegregationsObserved": {
            "id": "SEPIO-CG:99011",
            "label": "present"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib471",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/5/16 10:51 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess337">CGEX:CritAssess337</a></p></td>
          <td>
              
         </td>
          <td>
              <p>In a family tested by the LMM, the c.1484C>T variant was identified in a proband with clinical diagnosis of HCM. This proband has two relatives that also have clinical diagnosis of HCM however never relative carries the c.1484C>T variant.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn104">CGEX:EvLn104</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99042">SEPIO-CG:99042</a></p> (BS4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA133141">CAR:CA133141</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond066">CGEX:GenCond066</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess337"><pre><code>{
  "id": "CGEX:CritAssess337",
  "type": "CriterionAssessment",
  "description": "In a family tested by the LMM, the c.1484C>T variant was identified in a proband with clinical diagnosis of HCM. This proband has two relatives that also have clinical diagnosis of HCM however never relative carries the c.1484C>T variant.",
  "criterion": {
    "id": "SEPIO-CG:99042",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS4",
    "description": "Lack of segregation in affected members of a family",
    "usageNotes": "This rule is for “lack of segregation” meaning phenotype positive / genotype negative (and not pheno neg / geno pos). Also need to be careful of phenocopies. Also need rules for how many cases are needed in use this rule",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA133141",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll096"
  },
  "condition": [
    {
      "id": "CGEX:GenCond066",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217569",
          "label": "hypertrophic cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn104",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:FamSeg271",
          "type": "FamilyCosegregationStatement",
          "condition": {
            "id": "CGEX:GenCond066",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:217569",
                "label": "hypertrophic cardiomyopathy"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CAR:CA133141",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll096"
          },
          "family": {
            "id": "CGEX:Fam372",
            "label": "Family #010"
          },
          "phenotypePositiveAllelePositive": 0,
          "phenotypePositiveAlleleNegative": 2,
          "inconsistentSegregationsObserved": {
            "id": "SEPIO-CG:99011",
            "label": "present"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib472",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/5/16 11:00 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess340">CGEX:CritAssess340</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Homozygous truncating variants in LOXHD1 are associated with autosomal recessive hearing loss. This was demonstrated by both identification in human patients and a mouse model. (I used the OMIM entry as a source, but you could also directly use the papers.)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn083">CGEX:EvLn083</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99043">SEPIO-CG:99043</a></p> (BP1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090950">CAR:CA090950</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond059">CGEX:GenCond059</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess340"><pre><code>{
  "id": "CGEX:CritAssess340",
  "type": "CriterionAssessment",
  "description": "Homozygous truncating variants in LOXHD1 are associated with autosomal recessive hearing loss. This was demonstrated by both identification in human patients and a mouse model. (I used the OMIM entry as a source, but you could also directly use the papers.)",
  "criterion": {
    "id": "SEPIO-CG:99043",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP1",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "usageNotes": "Is there any guidance on the proportion of pathogenic variants that are LOF to apply this rule?  99%? 90%? 75%?",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA090950",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll098"
  },
  "condition": [
    {
      "id": "CGEX:GenCond059",
      "type": "GeneticCondition",
      "description": "Hearing Loss, Autosomal Recessive",
      "disease": [
        {
          "id": "Orphanet:87884",
          "label": "nonsyndromic deafness"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn083",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon201",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll098",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA090950",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll098"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9098",
              "type": "ContextualAlleleName",
              "name": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001583",
            "label": "missense variant"
          }
        },
        {
          "id": "CGEX:Null415",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll098",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA090950",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll098"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9098",
              "type": "ContextualAlleleName",
              "name": "NM_144612.6(LOXHD1 ): c.1028G>A (p.Arg343His)",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99015",
            "label": "function not lost"
          }
        },
        {
          "id": "CGEX:CondMech202",
          "type": "ConditionMechanismStatement",
          "description": "shown in both humans and mouse models",
          "statementOutcome": {
            "id": "SO:0002054",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000270",
            "label": "primary"
          },
          "gene": {
            "id": "HGNC:26521",
            "label": "LOXHD1"
          },
          "condition": {
            "id": "CGEX:GenCond059",
            "type": "GeneticCondition",
            "description": "Hearing Loss, Autosomal Recessive",
            "disease": [
              {
                "id": "Orphanet:87884",
                "label": "nonsyndromic deafness"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000007",
              "label": "Autosomal Recessive"
            }
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib475",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 10:00 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess341">CGEX:CritAssess341</a></p></td>
          <td>
              
         </td>
          <td>
              <p>While truncations are usually the cause of TTN related DCM, missense variants in certain regions can also cause disease.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn084">CGEX:EvLn084</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99043">SEPIO-CG:99043</a></p> (BP1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA256496">CAR:CA256496</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond053">CGEX:GenCond053</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess341"><pre><code>{
  "id": "CGEX:CritAssess341",
  "type": "CriterionAssessment",
  "description": "While truncations are usually the cause of TTN related DCM, missense variants in certain regions can also cause disease.",
  "criterion": {
    "id": "SEPIO-CG:99043",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP1",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "usageNotes": "Is there any guidance on the proportion of pathogenic variants that are LOF to apply this rule?  99%? 90%? 75%?",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA256496",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll099"
  },
  "condition": [
    {
      "id": "CGEX:GenCond053",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217604",
          "label": "dilated cardiomyopathy"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn084",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon203",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll099",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA256496",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll099"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9099",
              "type": "ContextualAlleleName",
              "name": "NM_133378.4(TTN): c.2926T>C",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001583",
            "label": "missense variant"
          }
        },
        {
          "id": "CGEX:Null416",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll099",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA256496",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll099"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9099",
              "type": "ContextualAlleleName",
              "name": "NM_133378.4(TTN): c.2926T>C",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0002055",
            "label": "null mutation"
          }
        },
        {
          "id": "CGEX:CondMech204",
          "type": "ConditionMechanismStatement",
          "statementOutcome": {
            "id": "SO:0002054",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000270",
            "label": "primary"
          },
          "gene": {
            "id": "HGNC:12403",
            "label": "TTN"
          },
          "condition": {
            "id": "CGEX:GenCond053",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:217604",
                "label": "dilated cardiomyopathy"
              }
            ]
          }
        },
        {
          "id": "CGEX:CondMech205",
          "type": "ConditionMechanismStatement",
          "description": "Truncating variants in TTN are associated with DCM. However, if missense variants are located in the A-band, impact the A-band, or are in a highly expressed region, they can also be associated with DCM.",
          "statementOutcome": {
            "id": "SO:0001583",
            "label": "missense variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000269",
            "label": "established"
          },
          "gene": {
            "id": "HGNC:12403",
            "label": "TTN"
          },
          "condition": {
            "id": "CGEX:GenCond053",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:217604",
                "label": "dilated cardiomyopathy"
              }
            ]
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib476",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 11:00 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess344">CGEX:CritAssess344</a></p></td>
          <td>
              
         </td>
          <td>
              <p>although this variant was found in trans with another variant in the same gene, that variant is VUS and not an established pathogenic variant and cardiomyopathies can be variable in penetrance. </p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn088">CGEX:EvLn088</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99044">SEPIO-CG:99044</a></p> (BP2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501235">CAR:CA501235</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond007">CGEX:GenCond007</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess344"><pre><code>{
  "id": "CGEX:CritAssess344",
  "type": "CriterionAssessment",
  "description": "although this variant was found in trans with another variant in the same gene, that variant is VUS and not an established pathogenic variant and cardiomyopathies can be variable in penetrance. ",
  "criterion": {
    "id": "SEPIO-CG:99044",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP2",
    "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA501235",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll104"
  },
  "condition": [
    {
      "id": "CGEX:GenCond007",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:167848",
          "label": "cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn088",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndGeno214",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv356",
            "label": "individual #024"
          },
          "genotype": [
            {
              "id": "CGEX:Geno408",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo406",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA501235",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll104"
                    }
                  ]
                },
                {
                  "id": "CGEX:Haplo407",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA501236",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll105"
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:IndCond215",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv356",
            "label": "individual #024"
          },
          "condition": [
            {
              "id": "CGEX:GenCond007",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:167848",
                  "label": "cardiomyopathy"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:VarInterp216",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA501236",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll105"
          },
          "condition": [
            {
              "id": "CGEX:GenCond007",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:167848",
                  "label": "cardiomyopathy"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA26333-7",
            "label": "Uncertain Significance"
          }
        },
        {
          "id": "CGEX:CondPenet218",
          "type": "ConditionPenetranceStatement",
          "condition": [
            {
              "id": "CGEX:GenCond007",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:167848",
                  "label": "cardiomyopathy"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:0003829",
            "label": "Incomplete"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib479",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 2:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess346">CGEX:CritAssess346</a></p></td>
          <td>
              
         </td>
          <td>
              <p>BP3 does not apply for this variant because it is not an in-frame deletion (it's a deletion of 10 bp) and, although the I Band is a repetitive region that contains tandem arrays of immunoglobulin domains, these folds may be an important component for TTN's elasticity.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn090">CGEX:EvLn090</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99045">SEPIO-CG:99045</a></p> (BP3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA273551">CAR:CA273551</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess346"><pre><code>{
  "id": "CGEX:CritAssess346",
  "type": "CriterionAssessment",
  "description": "BP3 does not apply for this variant because it is not an in-frame deletion (it's a deletion of 10 bp) and, although the I Band is a repetitive region that contains tandem arrays of immunoglobulin domains, these folds may be an important component for TTN's elasticity.",
  "criterion": {
    "id": "SEPIO-CG:99045",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP3",
    "description": "In-frame deletions/insertions in a repetitive region without a known function",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA273551",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll107"
  },
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn090",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:RegTyp225",
          "type": "RegionTypeStatement",
          "description": "The I-Band is a large region of repetitive Igg domains. ",
          "region": {
            "id": "CGEX:Loc439",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_133378.4",
              "label": "NM_133378.4"
            },
            "start": 6509,
            "stop": 39262
          },
          "statementOutcome": [
            {
              "id": "SO:0000657",
              "label": "repeat region "
            }
          ]
        },
        {
          "id": "CGEX:RegAll393",
          "type": "RegionAllelesStatement",
          "description": "Allele NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC is contained by this region.",
          "region": {
            "id": "CGEX:Loc439",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_133378.4",
              "label": "NM_133378.4"
            },
            "start": 6509,
            "stop": 39262
          },
          "allele": [
            {
              "id": "CGEX:CtxAll107",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": {
                "id": "CAR:CA273551",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll107"
              },
              "alleleName": {
                "id": "CGEX:CtxNm9107",
                "type": "ContextualAlleleName",
                "name": "NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC",
                "preferred": true
              }
            }
          ],
          "statementOutcome": {
            "id": "RO:0002656",
            "label": "overlaps region"
          }
        },
        {
          "id": "CGEX:MolCon227",
          "type": "AlleleMolecularConsequenceStatement",
          "statementOutcome": {
            "id": "SO:0001589",
            "label": "frameshift variant"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll107",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA273551",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll107"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9107",
              "type": "ContextualAlleleName",
              "name": "NM_133378.4 (TTN): c.26752_26761delACGGCAGAGC",
              "preferred": true
            }
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib490",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/15/16 9:23 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess350">CGEX:CritAssess350</a></p></td>
          <td>
              
         </td>
          <td>
              <p>While AI111 is considered probably damaging by at least one in silico predictor, the transcript for AI111 ( NM_133379.3 ) is not clinically relevant.  In clinically relevant transcripts, thisthis canonical allele is intronic.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn105">CGEX:EvLn105</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99046">SEPIO-CG:99046</a></p> (BP4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501237">CAR:CA501237</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess350"><pre><code>{
  "id": "CGEX:CritAssess350",
  "type": "CriterionAssessment",
  "description": "While AI111 is considered probably damaging by at least one in silico predictor, the transcript for AI111 ( NM_133379.3 ) is not clinically relevant.  In clinically relevant transcripts, thisthis canonical allele is intronic.",
  "criterion": {
    "id": "SEPIO-CG:99046",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP4",
    "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "usageNotes": "same issues as PP3.",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA501237",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll111"
  },
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn105",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:InSilicoPred362",
          "type": "InSilicoPredictionStatement",
          "canonicalAllele": {
            "id": "CGEX:CtxAll111",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA501237",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll111"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9111",
              "type": "ContextualAlleleName",
              "name": "NM_133379.3(TTN) : c.15283T>C",
              "preferred": true
            }
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "statementOutcome": {
            "id": "CGEX:PredOutcome010",
            "label": "Probably damaging"
          },
          "algorithm": "Polyphen-2 HumDiv"
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib494",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/4/16 10:37 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess351">CGEX:CritAssess351</a></p></td>
          <td>
              
         </td>
          <td>
              <p>The evidence here only explicitly describes a single individual with a likely pathogenic variant in TNNT2, but there are numerous other examples that have been observed in internal LMM data.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn093">CGEX:EvLn093</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99047">SEPIO-CG:99047</a></p> (BP5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA012381">CAR:CA012381</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond067">CGEX:GenCond067</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess351"><pre><code>{
  "id": "CGEX:CritAssess351",
  "type": "CriterionAssessment",
  "description": "The evidence here only explicitly describes a single individual with a likely pathogenic variant in TNNT2, but there are numerous other examples that have been observed in internal LMM data.",
  "criterion": {
    "id": "SEPIO-CG:99047",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA012381",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll112"
  },
  "condition": [
    {
      "id": "CGEX:GenCond067",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217569",
          "label": "hypertrophic cardiomyopathy"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn093",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndGeno241",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv357",
            "label": "individual #025"
          },
          "genotype": [
            {
              "id": "CGEX:Geno410",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo409",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA012381",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll112"
                    }
                  ]
                }
              ]
            },
            {
              "id": "CGEX:Geno412",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo411",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA005304",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll113"
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:IndCond242",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv357",
            "label": "individual #025"
          },
          "condition": [
            {
              "id": "CGEX:GenCond067",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:217569",
                  "label": "hypertrophic cardiomyopathy"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:VarInterp243",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA012381",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll112"
          },
          "condition": [
            {
              "id": "CGEX:GenCond067",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:217569",
                  "label": "hypertrophic cardiomyopathy"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA26332-9",
            "label": "Likely Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib495",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/4/16 11:42 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess354">CGEX:CritAssess354</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Variants in COL2A1 are associated with dominant spondyloepiphyseal dysplasia congenita (a primary bone dysplasia). CA112 was identified in a  proband with CA113, which is pathogenic for another primary bone dysplasia, dominant Pseudoachondroplasia. However, this does not provide supporting evidence for CA112 being benign because the combination of a COL2A1 and COMP variant are known to produce a more severe phenotype, Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome, which is the phenotype of the proband.  Because the combination of alleles may produce a qualitatively different phenotype, the bone dysplasia caused by the COMP variant does not provide evidence that the COL2A1 variant is benign for any bone dysplasias in general.
</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn096">CGEX:EvLn096</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99047">SEPIO-CG:99047</a></p> (BP5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA204551">CAR:CA204551</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond040">CGEX:GenCond040</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess354"><pre><code>{
  "id": "CGEX:CritAssess354",
  "type": "CriterionAssessment",
  "description": "Variants in COL2A1 are associated with dominant spondyloepiphyseal dysplasia congenita (a primary bone dysplasia). CA112 was identified in a  proband with CA113, which is pathogenic for another primary bone dysplasia, dominant Pseudoachondroplasia. However, this does not provide supporting evidence for CA112 being benign because the combination of a COL2A1 and COMP variant are known to produce a more severe phenotype, Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome, which is the phenotype of the proband.  Because the combination of alleles may produce a qualitatively different phenotype, the bone dysplasia caused by the COMP variant does not provide evidence that the COL2A1 variant is benign for any bone dysplasias in general.\n",
  "criterion": {
    "id": "SEPIO-CG:99047",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA204551",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll118"
  },
  "condition": [
    {
      "id": "CGEX:GenCond040",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:364526",
          "label": "Primary bone dysplasia"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn096",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond252",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv360",
            "label": "individual #028"
          },
          "condition": [
            {
              "id": "CGEX:GenCond041",
              "type": "GeneticCondition",
              "description": "Pseudoachondroplastic spondyloepiphyseal dysplasia",
              "disease": [
                {
                  "id": "Orphanet:750",
                  "label": "pseudoachondroplasia"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:IndGeno253",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv360",
            "label": "individual #028"
          },
          "genotype": [
            {
              "id": "CGEX:Geno423",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo422",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA204551",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll118"
                    }
                  ]
                }
              ]
            },
            {
              "id": "CGEX:Geno425",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo424",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA254703",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll119"
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:VarInterp254",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA254703",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll119"
          },
          "condition": [
            {
              "id": "CGEX:GenCond042",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:750",
                  "label": "pseudoachondroplasia"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        },
        {
          "id": "CGEX:VarInterp270",
          "type": "VariantPathogenicityInterpretation",
          "description": "Pseudoachondroplastic spondyloepiphyseal dysplasia syndrome is caused by a combination of a pathogenic variant in COL2A1 and another in COMP.   So CA113 may be involved, but is not the sole cause.",
          "variant": {
            "id": "CAR:CA254703",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll119"
          },
          "condition": [
            {
              "id": "CGEX:GenCond041",
              "type": "GeneticCondition",
              "description": "Pseudoachondroplastic spondyloepiphyseal dysplasia",
              "disease": [
                {
                  "id": "Orphanet:750",
                  "label": "pseudoachondroplasia"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA26333-7",
            "label": "Uncertain Significance"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib498",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/6/16 9:09 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess348">CGEX:CritAssess348</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Seven of the nine predictions found this variant to be benign, which we consider sufficient to support.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn092">CGEX:EvLn092</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99046">SEPIO-CG:99046</a></p> (BP4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA020822">CAR:CA020822</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess348"><pre><code>{
  "id": "CGEX:CritAssess348",
  "type": "CriterionAssessment",
  "description": "Seven of the nine predictions found this variant to be benign, which we consider sufficient to support.",
  "criterion": {
    "id": "SEPIO-CG:99046",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP4",
    "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "usageNotes": "same issues as PP3.",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA020822",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll109"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn092",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:InSilicoPred232",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000059.3",
            "label": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA020822",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll109"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "SIFT",
          "statementOutcome": {
            "id": "CGEX:PredOutcome001",
            "label": "Benign"
          }
        },
        {
          "id": "CGEX:InSilicoPred233",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000059.3",
            "label": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA020822",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll109"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Polyphen",
          "statementOutcome": {
            "id": "CGEX:PredOutcome009",
            "label": "Possibly damaging"
          }
        },
        {
          "id": "CGEX:InSilicoPred234",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000059.3",
            "label": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA020822",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll109"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "FATHMM",
          "statementOutcome": {
            "id": "CGEX:PredOutcome001",
            "label": "Benign"
          }
        },
        {
          "id": "CGEX:InSilicoPred235",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000059.3",
            "label": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA020822",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll109"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "LRT",
          "statementOutcome": {
            "id": "CGEX:PredOutcome001",
            "label": "Benign"
          }
        },
        {
          "id": "CGEX:InSilicoPred236",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000059.3",
            "label": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA020822",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll109"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "MetaSVM",
          "statementOutcome": {
            "id": "CGEX:PredOutcome001",
            "label": "Benign"
          }
        },
        {
          "id": "CGEX:InSilicoPred237",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000059.3",
            "label": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA020822",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll109"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Mutation assessor",
          "statementOutcome": {
            "id": "CGEX:PredOutcome001",
            "label": "Benign"
          }
        },
        {
          "id": "CGEX:InSilicoPred238",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000059.3",
            "label": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA020822",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll109"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Mutation taster",
          "statementOutcome": {
            "id": "CGEX:PredOutcome001",
            "label": "Benign"
          }
        },
        {
          "id": "CGEX:InSilicoPred239",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000059.3",
            "label": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA020822",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll109"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Provean",
          "statementOutcome": {
            "id": "CGEX:PredOutcome001",
            "label": "Benign"
          }
        },
        {
          "id": "CGEX:InSilicoPred240",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_000059.3",
            "label": "NM_000059.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA020822",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll109"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Condel",
          "statementOutcome": {
            "id": "CGEX:PredOutcome004",
            "label": "Deleterious"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib492",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "10/4/16 8:02 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess360">CGEX:CritAssess360</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This variant is a silent variant that does not result in an amino acid change. It is not very conserved and splicing tools suggest no change. However, BP7 does not apply because the variant is located in the first base of exon 4 of RAF1, which is part of the 3’ splice region of the exon. For that reason, we can’t fully trust computational tools. Functional assays would have to confirm that this variant truly does not alter splicing.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn102">CGEX:EvLn102</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99049">SEPIO-CG:99049</a></p> (BP7)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA177682">CAR:CA177682</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess360"><pre><code>{
  "id": "CGEX:CritAssess360",
  "type": "CriterionAssessment",
  "description": "This variant is a silent variant that does not result in an amino acid change. It is not very conserved and splicing tools suggest no change. However, BP7 does not apply because the variant is located in the first base of exon 4 of RAF1, which is part of the 3’ splice region of the exon. For that reason, we can’t fully trust computational tools. Functional assays would have to confirm that this variant truly does not alter splicing.",
  "criterion": {
    "id": "SEPIO-CG:99049",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP7",
    "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA177682",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll125"
  },
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn102",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:InSilicoPred266",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_002880.3",
            "label": "NM_002880.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA177682",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll125"
          },
          "predictionType": {
            "id": "SEPIO:0000323",
            "label": "splicing prediction"
          },
          "statementOutcome": {
            "id": "CGEX:PredOutcome008",
            "label": "No impact"
          }
        },
        {
          "id": "CGEX:AllConserv267",
          "type": "AlleleConservationStatement",
          "allele": {
            "id": "CAR:CA177682",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll125"
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99017",
            "label": "not conserved region"
          },
          "algorithm": "phyloP",
          "evidenceLine": [
            {
              "id": "CGEX:EvLn139",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllConservScore222",
                  "type": "AlleleConservationScoreStatement",
                  "allele": {
                    "id": "CAR:CA177682",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll125"
                  },
                  "algorithm": "phyloP",
                  "score": 0.45
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:AllConserv268",
          "type": "AlleleConservationStatement",
          "allele": {
            "id": "CAR:CA177682",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll125"
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99017",
            "label": "not conserved region"
          },
          "algorithm": "PhastCons",
          "evidenceLine": [
            {
              "id": "CGEX:EvLn140",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllConservScore224",
                  "type": "AlleleConservationScoreStatement",
                  "allele": {
                    "id": "CAR:CA177682",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll125"
                  },
                  "algorithm": "PhastCons",
                  "score": 1.0
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:MolCon269",
          "type": "AlleleMolecularConsequenceStatement",
          "statementOutcome": {
            "id": "SO:0001819",
            "label": "synonymous variant"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll125",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA177682",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll125"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9125",
              "type": "ContextualAlleleName",
              "name": "NM_002880.3(RAF1):c.321T>C",
              "preferred": true
            }
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib504",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:06 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess361">CGEX:CritAssess361</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID:17186466). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn103">CGEX:EvLn103</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99022">SEPIO-CG:99022</a></p> (PVS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA022880">CAR:CA022880</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond046">CGEX:GenCond046</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess361"><pre><code>{
  "id": "CGEX:CritAssess361",
  "type": "CriterionAssessment",
  "description": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID:17186466). Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).",
  "criterion": {
    "id": "SEPIO-CG:99022",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PVS1",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "usageNotes": "The two main concerns for applying this rule is:\na. Is LOF an established disease mechanism?\nb. Is LOF actually occurring?",
    "defaultStrength": {
      "id": "SEPIO:0000220",
      "label": "Pathogenic Very Strong"
    }
  },
  "variant": {
    "id": "CAR:CA022880",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll127"
  },
  "condition": [
    {
      "id": "CGEX:GenCond046",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:247",
          "label": "arrhythmogenic right ventricular cardiomyopathy"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn103",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon272",
          "type": "AlleleMolecularConsequenceStatement",
          "description": "This variant occurs in the invariant region (+/- 1,2) of the splice consensus sequence and in vitro studies confirmed that it leads to aberrant splicing and reduced DSC2 protein levels (Heuser 2006 PMID:17186466).",
          "contextualAllele": {
            "id": "CGEX:CtxAll127",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA022880",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll127"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9127",
              "type": "ContextualAlleleName",
              "name": "NM_024422.3(DSC2): c.631-2A>G (p.?)",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001629",
            "label": "splice-site variant"
          },
          "source": [
            "PMID:17186466"
          ]
        },
        {
          "id": "CGEX:CondMech273",
          "type": "ConditionMechanismStatement",
          "description": "Splice and other loss-of-function variants in DSC2 have been reported in individuals with ARVC (http://arvcdatabase.info/).",
          "statementOutcome": {
            "id": "SO:0002054",
            "label": "loss of function variant"
          },
          "condition": {
            "id": "CGEX:GenCond046",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:247",
                "label": "arrhythmogenic right ventricular cardiomyopathy"
              }
            ]
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000269",
            "label": "established"
          },
          "gene": {
            "id": "HGNC:3036",
            "label": "DSC2"
          }
        },
        {
          "id": "CGEX:Null404",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll127",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA022880",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll127"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9127",
              "type": "ContextualAlleleName",
              "name": "NM_024422.3(DSC2): c.631-2A>G (p.?)",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0002055",
            "label": "null mutation"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib505",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "01/17/17 9:12 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ],
  "source": [
    "PMID:17186466"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess363">CGEX:CritAssess363</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This variant was identified in one individual with pathogenic variants in another gene. Internal LMM data: The proband had one pathogenic and one likely pathogenic variant in (in trans) in SLC26A4. This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome. SLC26A4 is the only gene associated with Pendred Syndrome</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn106">CGEX:EvLn106</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99047">SEPIO-CG:99047</a></p> (BP5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090950">CAR:CA090950</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond058">CGEX:GenCond058</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess363"><pre><code>{
  "id": "CGEX:CritAssess363",
  "type": "CriterionAssessment",
  "description": "This variant was identified in one individual with pathogenic variants in another gene. Internal LMM data: The proband had one pathogenic and one likely pathogenic variant in (in trans) in SLC26A4. This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome. SLC26A4 is the only gene associated with Pendred Syndrome",
  "criterion": {
    "id": "SEPIO-CG:99047",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA090950",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll098"
  },
  "condition": [
    {
      "id": "CGEX:GenCond058",
      "type": "GeneticCondition",
      "description": "Hearing Loss",
      "disease": [
        {
          "id": "Orphanet:87884",
          "label": "nonsyndromic deafness"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn106",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp364",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA253304",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll128"
          },
          "condition": [
            {
              "id": "CGEX:GenCond058",
              "type": "GeneticCondition",
              "description": "Hearing Loss",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA26332-9",
            "label": "Likely Pathogenic"
          }
        },
        {
          "id": "CGEX:VarInterp365",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA273665",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll129"
          },
          "condition": [
            {
              "id": "CGEX:GenCond058",
              "type": "GeneticCondition",
              "description": "Hearing Loss",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        },
        {
          "id": "CGEX:IndGeno366",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv361",
            "label": "individual #029"
          },
          "genotype": [
            {
              "id": "CGEX:Geno428",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo426",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA090950",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll098"
                    }
                  ]
                },
                {
                  "id": "CGEX:Haplo427",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA253304",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll128"
                    }
                  ]
                }
              ]
            },
            {
              "id": "CGEX:Geno431",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo429",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA090950",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll098"
                    }
                  ]
                },
                {
                  "id": "CGEX:Haplo430",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA273665",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll129"
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:IndCond368",
          "type": "IndividualConditionStatement",
          "description": "This proband had mild to profound hearing loss with EVA, which is characteristic of Pendred Syndrome.",
          "individual": {
            "id": "CGEX:Indiv361",
            "label": "individual #029"
          },
          "condition": [
            {
              "id": "CGEX:GenCond058",
              "type": "GeneticCondition",
              "description": "Hearing Loss",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib506",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "03/07/2017 4:30 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess369">CGEX:CritAssess369</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042).</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn107">CGEX:EvLn107</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99028">SEPIO-CG:99028</a></p> (PM2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA022880">CAR:CA022880</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess369"><pre><code>{
  "id": "CGEX:CritAssess369",
  "type": "CriterionAssessment",
  "description": "This variant has been reported in 1/66564 European chromosomes by the Exome Aggregation Consortium (ExAC, http://exac.broadinstitute.org; dbSNP rs397514042).",
  "criterion": {
    "id": "SEPIO-CG:99028",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA022880",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll127"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn107",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq370",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "GNOMAD:nfe",
            "label": "Non-Finnish European"
          },
          "allele": {
            "id": "CAR:CA022880",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll127"
          },
          "alleleCount": 1,
          "alleleNumber": 66564,
          "alleleFrequency": 1.5e-05,
          "homozygousAlleleIndividualCount": 0
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib507",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "03/07/2017 5:22 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess371">CGEX:CritAssess371</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This variant is present in 0.2% (31/16512) of South Asian Chromosomes in ExAC.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn108">CGEX:EvLn108</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99039">SEPIO-CG:99039</a></p> (BS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA013536">CAR:CA013536</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess371"><pre><code>{
  "id": "CGEX:CritAssess371",
  "type": "CriterionAssessment",
  "description": "This variant is present in 0.2% (31/16512) of South Asian Chromosomes in ExAC.",
  "criterion": {
    "id": "SEPIO-CG:99039",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA013536",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll124"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn108",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq037",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "allele": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "alleleCount": 31,
          "alleleNumber": 16512,
          "alleleFrequency": 0.02,
          "population": {
            "id": "GNOMAD:sas",
            "label": "South Asian"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib508",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "03/08/2017 7:43 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess372">CGEX:CritAssess372</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This variant is present in 2% (27/1384) of African American Chromosomes in ESP.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn109">CGEX:EvLn109</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99039">SEPIO-CG:99039</a></p> (BS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090950">CAR:CA090950</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond057">CGEX:GenCond057</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess372"><pre><code>{
  "id": "CGEX:CritAssess372",
  "type": "CriterionAssessment",
  "description": "This variant is present in 2% (27/1384) of African American Chromosomes in ESP.",
  "criterion": {
    "id": "SEPIO-CG:99039",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA090950",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll098"
  },
  "condition": [
    {
      "id": "CGEX:GenCond057",
      "type": "GeneticCondition",
      "description": "Hearing Loss",
      "disease": [
        {
          "id": "Orphanet:87884",
          "label": "nonsyndromic deafness"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn109",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq038",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "allele": {
            "id": "CAR:CA090950",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll098"
          },
          "alleleCount": 27,
          "alleleNumber": 1384,
          "alleleFrequency": 0.02,
          "population": {
            "id": "GNOMAD:afr",
            "label": "African/African American"
          }
        },
        {
          "id": "CGEX:CondPrev039",
          "type": "ConditionPrevelanceStatement",
          "condition": [
            {
              "id": "CGEX:GenCond057",
              "type": "GeneticCondition",
              "description": "Hearing Loss",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "minimum": 1.0,
          "maximum": 1.0,
          "population": {
            "id": "SEPIO-CG:98002",
            "label": "Overall"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib509",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "03/08/2017 9:46 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess373">CGEX:CritAssess373</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This is a synonymous variant with little conservation and no predicted impact on splicing. AllConservation and splicing algorithm values are listed below.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn117">CGEX:EvLn117</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99046">SEPIO-CG:99046</a></p> (BP4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA013536">CAR:CA013536</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess373"><pre><code>{
  "id": "CGEX:CritAssess373",
  "type": "CriterionAssessment",
  "description": "This is a synonymous variant with little conservation and no predicted impact on splicing. AllConservation and splicing algorithm values are listed below.",
  "criterion": {
    "id": "SEPIO-CG:99046",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP4",
    "description": "Multiple lines of computational evidence suggest no impact on gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "usageNotes": "same issues as PP3.",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA013536",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll124"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn117",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:InSilicoPred068",
          "type": "InSilicoPredictionStatement",
          "description": "Provenance would show that this comes from Alamut",
          "transcript": {
            "id": "REFSEQ:NM_000257.2",
            "label": "NM_000257.2"
          },
          "canonicalAllele": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "predictionType": {
            "id": "SEPIO:0000323",
            "label": "splicing prediction"
          },
          "statementOutcome": {
            "id": "CGEX:PredOutcome008",
            "label": "No impact"
          }
        },
        {
          "id": "CGEX:AllConserv071",
          "type": "AlleleConservationStatement",
          "allele": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99017",
            "label": "not conserved region"
          },
          "algorithm": "phyloP",
          "evidenceLine": [
            {
              "id": "CGEX:EvLn133",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllConservScore116",
                  "type": "AlleleConservationScoreStatement",
                  "allele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "algorithm": "phyloP",
                  "score": -0.76
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:AllConserv072",
          "type": "AlleleConservationStatement",
          "allele": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99017",
            "label": "not conserved region"
          },
          "algorithm": "PhastCons",
          "evidenceLine": [
            {
              "id": "CGEX:EvLn134",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllConservScore171",
                  "type": "AlleleConservationScoreStatement",
                  "allele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "algorithm": "PhastCons",
                  "score": 0.63
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:MolCon075",
          "type": "AlleleMolecularConsequenceStatement",
          "statementOutcome": {
            "id": "SO:0001819",
            "label": "synonymous variant"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll124",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA013536",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll124"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9124",
              "type": "ContextualAlleleName",
              "name": "NM_000257.2(MYH7):c.327C>T",
              "preferred": true
            }
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib510",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "03/08/2017 10:02 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess374">CGEX:CritAssess374</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Flag-tagged construct with the variant was transfected into HEK cells. As per the RASopathy working group’s approved assays (see example 3 above), Smith et al. showed that the Y337C mutation increased levels of pERK in starved HEK cells via western blotting.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn116">CGEX:EvLn116</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99025">SEPIO-CG:99025</a></p> (PS3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090930">CAR:CA090930</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess374"><pre><code>{
  "id": "CGEX:CritAssess374",
  "type": "CriterionAssessment",
  "description": "Flag-tagged construct with the variant was transfected into HEK cells. As per the RASopathy working group’s approved assays (see example 3 above), Smith et al. showed that the Y337C mutation increased levels of pERK in starved HEK cells via western blotting.",
  "criterion": {
    "id": "SEPIO-CG:99025",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS3",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "usageNotes": "This particular rule will probably be downgraded often, given that there are not many “Functional studies that have been validated and shown to be reproducible and robust in a clinical diagnostic laboratory setting”",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA090930",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll130"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn116",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func067",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "increased ERK activtation",
          "assayType": {
            "id": "CGEX:AFAMID076",
            "label": "Activation Assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll130",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA090930",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll130"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9130",
              "type": "ContextualAlleleName",
              "name": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:11187",
            "label": "SOS1"
          },
          "source": [
            "PMID:23487764"
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib513",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "3/14/2017 10:29 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess375">CGEX:CritAssess375</a></p></td>
          <td>
              
         </td>
          <td>
              <p>A personal communication to our lab informed us of an individual and his son who both harbored the variant but with clinical workup did not appear to have Noonan Syndrome or symptoms indicative of Noonan Syndrome.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn123">CGEX:EvLn123</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99042">SEPIO-CG:99042</a></p> (BS4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090930">CAR:CA090930</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond056">CGEX:GenCond056</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess375"><pre><code>{
  "id": "CGEX:CritAssess375",
  "type": "CriterionAssessment",
  "description": "A personal communication to our lab informed us of an individual and his son who both harbored the variant but with clinical workup did not appear to have Noonan Syndrome or symptoms indicative of Noonan Syndrome.",
  "criterion": {
    "id": "SEPIO-CG:99042",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS4",
    "description": "Lack of segregation in affected members of a family",
    "usageNotes": "This rule is for “lack of segregation” meaning phenotype positive / genotype negative (and not pheno neg / geno pos). Also need to be careful of phenocopies. Also need rules for how many cases are needed in use this rule",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA090930",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll130"
  },
  "condition": [
    {
      "id": "CGEX:GenCond056",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:648",
          "label": "noonan syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn123",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:FamSeg076",
          "type": "FamilyCosegregationStatement",
          "condition": {
            "id": "CGEX:GenCond056",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:648",
                "label": "noonan syndrome"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CAR:CA090930",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll130"
          },
          "family": {
            "id": "CGEX:Fam373",
            "label": "Family #011"
          },
          "phenotypePositiveAllelePositive": 0,
          "phenotypeNegativeAllelePositive": 2,
          "phenotypePositiveAlleleNegative": 0,
          "inconsistentSegregationsObserved": {
            "id": "SEPIO-CG:99011",
            "label": "present"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib514",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "3/14/2017 10:45 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess377">CGEX:CritAssess377</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Parental testing in our lab confirmed this variant was present de novo in 1 proband with clinical features of CFC. (LMM internal Data)</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn128">CGEX:EvLn128</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99032">SEPIO-CG:99032</a></p> (PM6)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA280055">CAR:CA280055</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond069">CGEX:GenCond069</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess377"><pre><code>{
  "id": "CGEX:CritAssess377",
  "type": "CriterionAssessment",
  "description": "Parental testing in our lab confirmed this variant was present de novo in 1 proband with clinical features of CFC. (LMM internal Data)",
  "criterion": {
    "id": "SEPIO-CG:99032",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM6",
    "description": "Assumed de novo, but without confirmation of paternity and maternity",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA280055",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll131"
  },
  "condition": [
    {
      "id": "CGEX:GenCond069",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:1340",
          "label": "cardiofaciocutaneous syndrome"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn128",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndAllInh106",
          "type": "IndividualAlleleInheritanceStatement",
          "canonicalAllele": {
            "id": "CAR:CA280055",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll131"
          },
          "individual": {
            "id": "CGEX:Indiv362",
            "label": "individual #030"
          },
          "parentalConfirmation": {
            "id": "SEPIO-CG:99008",
            "label": "no parental confirmation"
          },
          "statementOutcome": [
            {
              "id": "GENO:0000880",
              "label": "de novo"
            }
          ]
        },
        {
          "id": "CGEX:IndCond110",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv362",
            "label": "individual #030"
          },
          "condition": [
            {
              "id": "CGEX:GenCond069",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:FamCond112",
          "type": "FamilyConditionStatement",
          "proband": {
            "id": "CGEX:Indiv362",
            "label": "individual #030"
          },
          "condition": [
            {
              "id": "CGEX:GenCond069",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:1340",
                  "label": "cardiofaciocutaneous syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99010",
            "label": "unaffected"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib516",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent010",
        "type": "Agent",
        "label": "ACME Molecular Lab"
      },
      "contributionDate": "6/11/2011 12:45:00",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess378">CGEX:CritAssess378</a></p></td>
          <td>
              
         </td>
          <td>
              <p>Another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). The other variant: NM_004333.4(BRAF):c.1449A>C (p.Lys483Asn) is classified as likely pathogenic by LMM for Rasopathy.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn130">CGEX:EvLn130</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99031">SEPIO-CG:99031</a></p> (PM5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA280055">CAR:CA280055</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond049">CGEX:GenCond049</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess378"><pre><code>{
  "id": "CGEX:CritAssess378",
  "type": "CriterionAssessment",
  "description": "Another variant at the same residue (Lys483Asn) was identified as a de novo variant in an individual with features of a Noonan spectrum disorder (LMM unpublished data). The other variant: NM_004333.4(BRAF):c.1449A>C (p.Lys483Asn) is classified as likely pathogenic by LMM for Rasopathy.",
  "criterion": {
    "id": "SEPIO-CG:99031",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM5",
    "description": "Novel missense change at an amino acid residue where a different missense change determined to be pathogenic has been seen before",
    "usageNotes": "Need to be sure that pathogenic variant exists at the residue in question. Sometimes discrepancy resolution may be required. Some thing like Variantexplorer.org may be useful.\nSometimes no assertion is provided in ClinVar ...may require curation by lit. search.",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA280055",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll131"
  },
  "condition": [
    {
      "id": "CGEX:GenCond049",
      "type": "GeneticCondition",
      "description": "Rasopathy",
      "disease": [
        {
          "id": "Orphanet:98733",
          "label": "Noonan syndrome and noonan-related syndrom"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn130",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp060",
          "type": "VariantPathogenicityInterpretation",
          "description": "The Lys483Asn variant in BRAF has not been previously reported in the literature, but has been identified as a de novo variant in one proband in our laboratory. In addition, this residue is highly conserved across evolutionarily distinct species and computational analyses (PolyPhen2, SIFT, AlignGVGD) predict that this variant will impact the normal function of BRAF. It should be noted that the sensitivity and specificity of these computational programs has not been determined by our laboratory. Therefore, this variant is likely to be pathogenic.",
          "variant": {
            "id": "CAR:CA273507",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll133"
          },
          "condition": [
            {
              "id": "CGEX:GenCond049",
              "type": "GeneticCondition",
              "description": "Rasopathy",
              "disease": [
                {
                  "id": "Orphanet:98733",
                  "label": "Noonan syndrome and noonan-related syndrom"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA26332-9",
            "label": "Likely Pathogenic"
          },
          "contribution": [
            {
              "id": "CGEX:Contrib518",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent010",
                "type": "Agent",
                "label": "ACME Molecular Lab"
              },
              "contributionDate": "6/12/2011 9:00 AM",
              "contributionRole": {
                "id": "SEPIO:0000331",
                "label": "interpreter"
              }
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib517",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent010",
        "type": "Agent",
        "label": "ACME Molecular Lab"
      },
      "contributionDate": "6/10/2011 1:45 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess276">CGEX:CritAssess276</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn002">CGEX:EvLn002</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99022">SEPIO-CG:99022</a></p> (PVS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090898">CAR:CA090898</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond001">CGEX:GenCond001</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess276"><pre><code>{
  "id": "CGEX:CritAssess276",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99022",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PVS1",
    "description": "null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single or multiexon deletion) in a gene where LOF is a known mechanism of disease",
    "usageNotes": "The two main concerns for applying this rule is:\na. Is LOF an established disease mechanism?\nb. Is LOF actually occurring?",
    "defaultStrength": {
      "id": "SEPIO:0000220",
      "label": "Pathogenic Very Strong"
    }
  },
  "variant": {
    "id": "CAR:CA090898",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll002"
  },
  "condition": [
    {
      "id": "CGEX:GenCond001",
      "type": "GeneticCondition",
      "description": "Breast Ovarian Cancer, Autosomal Dominant",
      "disease": [
        {
          "id": "MESH:D061325",
          "label": "hereditary breast ovarian cancer"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn002",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon003",
          "type": "AlleleMolecularConsequenceStatement",
          "description": "LOF is predicted to occur as variant is >55bp from penultimate exon",
          "contextualAllele": {
            "id": "CGEX:CtxAll002",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA090898",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll002"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9002",
              "type": "ContextualAlleleName",
              "name": "NM_000059.3(BRCA2):c.7762_7764delinsTT",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001589",
            "label": "frameshift variant"
          }
        },
        {
          "id": "CGEX:Null397",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll002",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA090898",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll002"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9002",
              "type": "ContextualAlleleName",
              "name": "NM_000059.3(BRCA2):c.7762_7764delinsTT",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0002055",
            "label": "null mutation"
          }
        },
        {
          "id": "CGEX:CondMech004",
          "type": "ConditionMechanismStatement",
          "description": "This is so well-known that it does not require a citation.",
          "statementOutcome": {
            "id": "SO:0002054",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000269",
            "label": "established"
          },
          "gene": {
            "id": "HGNC:1101",
            "label": "BRCA2"
          },
          "condition": {
            "id": "CGEX:GenCond001",
            "type": "GeneticCondition",
            "description": "Breast Ovarian Cancer, Autosomal Dominant",
            "disease": [
              {
                "id": "MESH:D061325",
                "label": "hereditary breast ovarian cancer"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib397",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 12:15 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess283">CGEX:CritAssess283</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99024">SEPIO-CG:99024</a></p> (PS2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA354186">CAR:CA354186</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond009">CGEX:GenCond009</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess283"><pre><code>{
  "id": "CGEX:CritAssess283",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99024",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS2",
    "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
    "usageNotes": "MYH7 is adding specificity to this rule with\na. No family history = parents have had ECHO and ECG\nb. Only paternity confirmation required",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA354186",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll019"
  },
  "condition": [
    {
      "id": "CGEX:GenCond009",
      "type": "GeneticCondition",
      "description": "Epilepsy with Mental Retardation"
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "phenotypePositiveAlleleNegative": 0,
  "contribution": [
    {
      "id": "CGEX:Contrib405",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/3/16 10:25 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess284">CGEX:CritAssess284</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn012">CGEX:EvLn012</a></p><p><a href="details/details.html#CGEX:EvLn013">CGEX:EvLn013</a></p><p><a href="details/details.html#CGEX:EvLn014">CGEX:EvLn014</a></p><p><a href="details/details.html#CGEX:EvLn015">CGEX:EvLn015</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99025">SEPIO-CG:99025</a></p> (PS3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090410">CAR:CA090410</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess284"><pre><code>{
  "id": "CGEX:CritAssess284",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99025",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS3",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "usageNotes": "This particular rule will probably be downgraded often, given that there are not many “Functional studies that have been validated and shown to be reproducible and robust in a clinical diagnostic laboratory setting”",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA090410",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll020"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn012",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func021",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "the mutation lowered the Ca2+  affinity of the reconstituted thin filaments;",
          "assayType": {
            "id": "CGEX:AFAMID078",
            "label": "Binding assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll021",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA501233",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll021"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9021",
              "type": "ContextualAlleleName",
              "name": "NP_001001431.1:p.Arg131Trp",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:11949",
            "label": "TNNT2"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn013",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func022",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "reduced Ca2 sensitivity of activation",
          "assayType": {
            "id": "CGEX:AFAMID077",
            "label": "ATPase assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll021",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA501233",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll021"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9021",
              "type": "ContextualAlleleName",
              "name": "NP_001001431.1:p.Arg131Trp",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:11949",
            "label": "TNNT2"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn014",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func023",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "reduced Ca2 sensitivity of activation",
          "assayType": {
            "id": "CGEX:AFAMID082",
            "label": "motility assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll021",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA501233",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll021"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9021",
              "type": "ContextualAlleleName",
              "name": "NP_001001431.1:p.Arg131Trp",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:11949",
            "label": "TNNT2"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn015",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func024",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "desensitized Ca2+ binding to the thin filament.",
          "assayType": {
            "id": "CGEX:AFAMID078",
            "label": "Binding assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll021",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA501233",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll021"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9021",
              "type": "ContextualAlleleName",
              "name": "NP_001001431.1:p.Arg131Trp",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:11949",
            "label": "TNNT2"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib406",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/3/16 2:50 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib407",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "6/3/16 2:50 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess285">CGEX:CritAssess285</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn016">CGEX:EvLn016</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99025">SEPIO-CG:99025</a></p> (PS3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090884">CAR:CA090884</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess285"><pre><code>{
  "id": "CGEX:CritAssess285",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99025",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS3",
    "description": "Well-established in vitro or in vivo functional studies supportive of a damaging effect on the gene or gene product",
    "usageNotes": "This particular rule will probably be downgraded often, given that there are not many “Functional studies that have been validated and shown to be reproducible and robust in a clinical diagnostic laboratory setting”",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA090884",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll072"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn016",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func025",
          "type": "AlleleFunctionalImpactStatement",
          "description": "Expression of human SPG7 (with this mutation) in yeast with damaged m-AAA protease did not restore respiratory growth or MrpL32 processing, if the function of AFG3L2 was impaired.",
          "resultDescription": "Functionally Damaging",
          "assayType": {
            "id": "CGEX:AFAMID084",
            "label": "Yeast complementation assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll022",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA090884",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll072"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9022",
              "type": "ContextualAlleleName",
              "name": "NM_003119.2 (SPG7):c.1529C>T",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:11237",
            "label": "SPG7"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib408",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent003",
        "type": "Agent",
        "label": "Sasha Rande"
      },
      "contributionDate": "6/3/16 6:09 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib409",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "6/3/16 6:09 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess289">CGEX:CritAssess289</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn021">CGEX:EvLn021</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99026">SEPIO-CG:99026</a></p> (PS4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA123287">CAR:CA123287</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond012">CGEX:GenCond012</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess289"><pre><code>{
  "id": "CGEX:CritAssess289",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99026",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PS4",
    "description": "The prevalence of the variant in affected individuals is significantly increased compared with the prevalence in controls",
    "defaultStrength": {
      "id": "SEPIO:0000330",
      "label": "Pathogenic Strong"
    }
  },
  "variant": {
    "id": "CAR:CA123287",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll027"
  },
  "condition": [
    {
      "id": "CGEX:GenCond012",
      "type": "GeneticCondition",
      "phenotype": [
        {
          "id": "HP:0011014",
          "label": "Abnormal Glucose Homeostasis"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn021",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:CaseCtrl036",
          "type": "CaseControlAlleleFrequencyStatement",
          "description": "Odds ratio is calculated by pooling samples from the six GENIUS T2D cohorts before calculation.",
          "canonicalAllele": {
            "id": "CAR:CA123287",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll027"
          },
          "condition": {
            "id": "CGEX:GenCond012",
            "type": "GeneticCondition",
            "phenotype": [
              {
                "id": "HP:0011014",
                "label": "Abnormal Glucose Homeostasis"
              }
            ]
          },
          "oddsRatio": 1.06,
          "confidenceLevel": 95.0,
          "caseGroupFrequency": {
            "id": "CGEX:AllFreq034",
            "type": "PopulationAlleleFrequencyStatement",
            "ascertainment": {
              "id": "CGEX:Ascrt0002",
              "label": "GENIUS T2D Cases"
            },
            "allele": {
              "id": "CAR:CA123287",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll027"
            },
            "alleleCount": 1198,
            "alleleNumber": 7344,
            "individualCount": 3672,
            "homozygousAlleleIndividualCount": 113,
            "heterozygousAlleleIndividualCount": 972,
            "population": {
              "id": "GNOMAD:nfe",
              "label": "Non-Finnish European"
            },
            "userLabelDictionary": [
              {
                "id": "CGEX:UsrLabl002",
                "type": "UserLabel",
                "description": "Case group is composed of cases from six cohorts from the GENIUS T2D consortium.  Case status is defined per-cohort.",
                "labelFor": {
                  "id": "CGEX:Ascrt0002",
                  "label": "GENIUS T2D Cases"
                }
              }
            ]
          },
          "controlGroupFrequency": {
            "id": "CGEX:AllFreq035",
            "type": "PopulationAlleleFrequencyStatement",
            "ascertainment": {
              "id": "CGEX:Ascrt0003",
              "label": "GENIUS T2D Controls"
            },
            "allele": {
              "id": "CAR:CA123287",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll027"
            },
            "alleleCount": 936,
            "alleleNumber": 5870,
            "individualCount": 2935,
            "homozygousAlleleIndividualCount": 77,
            "heterozygousAlleleIndividualCount": 782,
            "population": {
              "id": "GNOMAD:nfe",
              "label": "Non-Finnish European"
            },
            "userLabelDictionary": [
              {
                "id": "CGEX:UsrLabl003",
                "type": "UserLabel",
                "description": "Controls group is composed of controls from six cohorts from the GENIUS T2D consortium.  Controls status is defined per-cohort.",
                "labelFor": {
                  "id": "CGEX:Ascrt0003",
                  "label": "GENIUS T2D Controls"
                }
              }
            ]
          },
          "confidenceIntervalLower": 0.96,
          "confidenceIntervalUpper": 1.18,
          "contribution": [
            {
              "id": "CGEX:Contrib480",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "9/13/16",
              "contributionRole": {
                "id": "SEPIO:0000156",
                "label": "curator"
              }
            }
          ],
          "source": [
            "http://www.ncbi.nlm.nih.gov/pubmed/23633196"
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib414",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/5/16 4:12 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess294">CGEX:CritAssess294</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn027">CGEX:EvLn027</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99028">SEPIO-CG:99028</a></p> (PM2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA011797">CAR:CA011797</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess294"><pre><code>{
  "id": "CGEX:CritAssess294",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99028",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA011797",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll037"
  },
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn027",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq053",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "GNOMAD:fin",
            "label": "Finnish"
          },
          "allele": {
            "id": "CAR:CA011797",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll037"
          },
          "alleleCount": 6614,
          "alleleNumber": 514,
          "alleleFrequency": 0.07771,
          "homozygousAlleleIndividualCount": 21,
          "medianCoverage": 100.0
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib420",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/5/16 3:13 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess296">CGEX:CritAssess296</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn029">CGEX:EvLn029</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99029">SEPIO-CG:99029</a></p> (PM3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090919">CAR:CA090919</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond062">CGEX:GenCond062</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess296"><pre><code>{
  "id": "CGEX:CritAssess296",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99029",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM3",
    "description": "For recessive disorders, detected in trans with a pathogenic variant",
    "usageNotes": "CSER - Laboratories discussed when to modify the strength of PM3, the variant is seen in trans with a pathogenic variant for recessive disorders. Published literature may not always explicitly state the phase of variants found in affected individuals which raises a challenge for invoking PM3.\nWhen phase has not been established, some felt that PM3 could be invoked as supporting evidence. Also, if the variant is seen in trans with a pathogenic variant in more than one individual it was felt that PM3 can be upgraded to strong. However, sites did not agree on how many additional observations were necessary to call the evidence strong (2 vs. 3) but concluded that such guidance would be useful.\n \nLMM - INCREASES IN STRENGTH IF VARIANT SEEN IN MULTIPLE COMPOUND HETS WITH DIFFERENT LIKELY PATHOGENIC/PATHOGENIC VARIANTS",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA090919",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll039"
  },
  "condition": [
    {
      "id": "CGEX:GenCond062",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:244",
          "label": "Primary ciliary dyskenisia"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn029",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndCond056",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv336",
            "label": "individual #004"
          },
          "condition": [
            {
              "id": "CGEX:GenCond014",
              "type": "GeneticCondition",
              "description": "PCD autosomal recessive w/ outer dynein arm (ODA) defects",
              "disease": [
                {
                  "id": "Orphanet:244",
                  "label": "Primary ciliary dyskenisia"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              },
              "phenotype": [
                {
                  "id": "HP:0200106",
                  "label": "Absent/shortened dynein arms"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          },
          "contribution": [
            {
              "id": "CGEX:Contrib483",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "5/31/16 8:46 AM",
              "contributionRole": {
                "id": "SEPIO:0000156",
                "label": "curator"
              }
            }
          ],
          "source": [
            "http://partners.org/ppm/lmm/gi/case/123"
          ]
        },
        {
          "id": "CGEX:VarInterp057",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA273615",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll040"
          },
          "condition": [
            {
              "id": "CGEX:GenCond062",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:244",
                  "label": "Primary ciliary dyskenisia"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          },
          "contribution": [
            {
              "id": "CGEX:Contrib485",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent007",
                "type": "Agent",
                "label": "Stefan DiMarino"
              },
              "contributionDate": "5/31/16 11:01 AM",
              "contributionRole": {
                "id": "SEPIO:0000331",
                "label": "interpreter"
              }
            }
          ],
          "source": [
            "https://www.ncbi.nlm.nih.gov/clinvar/SCV000205842.3"
          ]
        },
        {
          "id": "CGEX:IndGeno058",
          "type": "IndividualGenotypeStatement",
          "description": "Father was het for c.9449delG variant and negative for c.7468_7488del variant. Mother was het for c.7468_7488del variant and negative for c.9449delG variant. Thus, variants are in trans in this proband.",
          "individual": {
            "id": "CGEX:Indiv336",
            "label": "individual #004"
          },
          "genotype": [
            {
              "id": "CGEX:Geno378",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo376",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA090919",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll039"
                    }
                  ]
                },
                {
                  "id": "CGEX:Haplo377",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA273615",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll040"
                    }
                  ]
                }
              ]
            }
          ],
          "contribution": [
            {
              "id": "CGEX:Contrib484",
              "type": "Contribution",
              "agent": {
                "id": "CGEX:Agent001",
                "type": "Agent",
                "label": "Harry Stevenson"
              },
              "contributionDate": "5/31/16 9:15 AM",
              "contributionRole": {
                "id": "SEPIO:0000156",
                "label": "curator"
              }
            }
          ],
          "source": [
            "http://partners.org/ppm/lmm/gi/case/123"
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib422",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/2/16 8:01 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess297">CGEX:CritAssess297</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn030">CGEX:EvLn030</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99029">SEPIO-CG:99029</a></p> (PM3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA176145">CAR:CA176145</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond010">CGEX:GenCond010</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess297"><pre><code>{
  "id": "CGEX:CritAssess297",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99029",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM3",
    "description": "For recessive disorders, detected in trans with a pathogenic variant",
    "usageNotes": "CSER - Laboratories discussed when to modify the strength of PM3, the variant is seen in trans with a pathogenic variant for recessive disorders. Published literature may not always explicitly state the phase of variants found in affected individuals which raises a challenge for invoking PM3.\nWhen phase has not been established, some felt that PM3 could be invoked as supporting evidence. Also, if the variant is seen in trans with a pathogenic variant in more than one individual it was felt that PM3 can be upgraded to strong. However, sites did not agree on how many additional observations were necessary to call the evidence strong (2 vs. 3) but concluded that such guidance would be useful.\n \nLMM - INCREASES IN STRENGTH IF VARIANT SEEN IN MULTIPLE COMPOUND HETS WITH DIFFERENT LIKELY PATHOGENIC/PATHOGENIC VARIANTS",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA176145",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll041"
  },
  "condition": [
    {
      "id": "CGEX:GenCond010",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:87884",
          "label": "nonsyndromic deafness"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "label": "Autosomal Recessive"
      },
      "gene": {
        "id": "HGNC:18183",
        "label": "GIPC3"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn030",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp059",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA501223",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll042"
          },
          "condition": [
            {
              "id": "CGEX:GenCond010",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "label": "nonsyndromic deafness"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              },
              "gene": {
                "id": "HGNC:18183",
                "label": "GIPC3"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        },
        {
          "id": "CGEX:IndCond061",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv337",
            "label": "individual #005"
          },
          "condition": [
            {
              "id": "CGEX:GenCond010",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "label": "nonsyndromic deafness"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              },
              "gene": {
                "id": "HGNC:18183",
                "label": "GIPC3"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:IndGeno062",
          "type": "IndividualGenotypeStatement",
          "description": "Based on the NGS data, we are able to determine that c.298G>A and c.411+1G>A are in trans as no NGS reads carry both variants.",
          "individual": {
            "id": "CGEX:Indiv337",
            "label": "individual #005"
          },
          "genotype": [
            {
              "id": "CGEX:Geno381",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo379",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA176145",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll041"
                    }
                  ]
                },
                {
                  "id": "CGEX:Haplo380",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA501223",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll042"
                    }
                  ]
                }
              ]
            }
          ]
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib423",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/10/16 9:02 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess312">CGEX:CritAssess312</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn050">CGEX:EvLn050</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99033">SEPIO-CG:99033</a></p> (PP1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA006429">CAR:CA006429</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond002">CGEX:GenCond002</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess312"><pre><code>{
  "id": "CGEX:CritAssess312",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99033",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP1",
    "description": "Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA006429",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll069"
  },
  "condition": [
    {
      "id": "CGEX:GenCond002",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217604",
          "label": "dilated cardiomyopathy"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn050",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AggSeg115",
          "type": "AggregateCosegregationStatement",
          "canonicalAllele": {
            "id": "CAR:CA006429",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll069"
          },
          "condition": {
            "id": "CGEX:GenCond002",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:217604",
                "label": "dilated cardiomyopathy"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          },
          "familySegregation": [
            {
              "id": "CGEX:FamSeg113",
              "type": "FamilyCosegregationStatement",
              "condition": {
                "id": "CGEX:GenCond002",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "Orphanet:217604",
                    "label": "dilated cardiomyopathy"
                  }
                ],
                "inheritancePattern": {
                  "id": "HP:0000006",
                  "label": "Autosomal Dominant"
                }
              },
              "canonicalAllele": {
                "id": "CAR:CA006429",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll069"
              },
              "family": {
                "id": "CGEX:Fam368",
                "label": "Family #006"
              },
              "phenotypePositiveAllelePositive": 1,
              "phenotypeNegativeAllelePositive": 1,
              "columns": [
                "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”]"
              ],
              "affectedValues": [
                "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
              ],
              "genotypeValues": [
                "{0: “Unknown”,1:”Reference”, 2: CA65}"
              ],
              "pedigree": [
                "[[Fam006,Fam006.1,0,0,1,2,0/0],\n   [Fam006,Fam006.2,0,0,2,1,0/0],\n   [Fam006,Fam006.3,Fam006.1,Fam006.2,2,3,0/0],\n   [Fam006,Fam006.4,Fam006.1,Fam006.2,1,2,1/2],\n   [Fam006,Fam006.5,Fam006.1,Fam006.2,2,1,1/1],\n   [Fam006,Fam006.6,Fam006.1,Fam006.2,1,2,1/2]]"
              ]
            },
            {
              "id": "CGEX:FamSeg114",
              "type": "FamilyCosegregationStatement",
              "condition": {
                "id": "CGEX:GenCond002",
                "type": "GeneticCondition",
                "disease": [
                  {
                    "id": "Orphanet:217604",
                    "label": "dilated cardiomyopathy"
                  }
                ],
                "inheritancePattern": {
                  "id": "HP:0000006",
                  "label": "Autosomal Dominant"
                }
              },
              "canonicalAllele": {
                "id": "CAR:CA006429",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll069"
              },
              "family": {
                "id": "CGEX:Fam369",
                "label": "Family #007"
              },
              "phenotypePositiveAllelePositive": 2,
              "phenotypeNegativeAllelePositive": 3,
              "columns": [
                "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”,”ObligateGenotype”]"
              ],
              "affectedValues": [
                "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
              ],
              "genotypeValues": [
                "{0: “Unknown”,1:”Reference”, 2: CA65}"
              ],
              "pedigree": [
                "[[Fam007, Fam007.1, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.2, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.3, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.4, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.5, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.6, Fam007.1, Fam007.2, 2, 3, 1/2, 1],\n   [Fam007, Fam007.7,0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.8, Fam007.1, Fam007.2, 2, 1, 1/2, 1],\n   [Fam007, Fam007.9,Fam007.3, Fam007.4, 1, 1, 0/0, 0],\n   [Fam007, Fam007.10, 0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.11, Fam007.5, Fam007.6, 2, 2, 1/2, 0],\n   [Fam007, Fam007.12, Fam007.5, Fam007.6, 1,2, 0/0, 0],\n   [Fam007, Fam007.13, Fam007.5, Fam007.6, 2,1, 1/1,0],\n   [Fam007, Fam007.14, Fam007.5, Fam007.6, 1, 1, 1/1, 0],\n   [Fam007, Fam007.15, Fam007.7, Fam007.8, 1, 2, 1/2, 0],\n   [Fam007, Fam007.16, Fam007.8, Fam007.9, 1, 2, 1/2, 0],\n   [Fam007, Fam007.17, Fam007.8, Fam007.9, 1, 2, 0/0,0],\n   [Fam007, Fam007.18, Fam007.8, Fam007.9, 1, 2, 0/0, 0],\n   [Fam007, Fam007.19, Fam007.10, Fam007.11, 1, 2, 0/0,0],\n   [Fam007.20,Fam007.10, Fam007.11, 1,1,1/1,0]]"
              ]
            }
          ],
          "totalNumberOfSegregations": 9
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib442",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent002",
        "type": "Agent",
        "label": "Noonan/Rasopathy WG (ClinGen)"
      },
      "contributionDate": "7/21/16 8:11 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess313">CGEX:CritAssess313</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn051">CGEX:EvLn051</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99034">SEPIO-CG:99034</a></p> (PP2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA016087">CAR:CA016087</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess313"><pre><code>{
  "id": "CGEX:CritAssess313",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99034",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP2",
    "description": "Missense variant in a gene that has a low rate of benign missense variation and in which missense variants are a common mechanism of disease",
    "usageNotes": "Seems no one has established firm rules for when this rule is applicable so not sure how great any examples will be\nAgree with the lack of firm rules. Seems like one criteria may be observed missense variants over 5% allele frequency in ExAC. If this seems reasonable then I could generate a couple of examples.",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA016087",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll070"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn051",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon117",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll070",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA016087",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll070"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9070",
              "type": "ContextualAlleleName",
              "name": "NM_000257.3(MYH7):c.5401G>A",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001583",
            "label": "missense variant"
          }
        },
        {
          "id": "CGEX:Null405",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll070",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA016087",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll070"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9070",
              "type": "ContextualAlleleName",
              "name": "NM_000257.3(MYH7):c.5401G>A",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99015",
            "label": "function not lost"
          }
        },
        {
          "id": "CGEX:CondMech118",
          "type": "ConditionMechanismStatement",
          "description": "Of 178 pathogenic  alleles in ClinVar, 149 are missense.",
          "statementOutcome": {
            "id": "SO:0001583",
            "label": "missense variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000268",
            "label": "common"
          },
          "gene": {
            "id": "HGNC:7577",
            "label": "MYH7"
          }
        },
        {
          "id": "CGEX:BMVR119",
          "type": "BenignMissenseVariationRateStatement",
          "description": "Based on the gene size, ExAC expects 740 missense variants to be observed, but actually observes only 420, giving a z score of 6.5, which is in the top 0.02% of genes.",
          "gene": {
            "id": "HGNC:7577",
            "label": "MYH7"
          },
          "statementOutcome": {
            "id": "SEPIO:0000266",
            "label": "Low"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib443",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "7/21/16 9:41 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess314">CGEX:CritAssess314</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn052">CGEX:EvLn052</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99035">SEPIO-CG:99035</a></p> (PP3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090928">CAR:CA090928</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess314"><pre><code>{
  "id": "CGEX:CritAssess314",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99035",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP3",
    "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "usageNotes": "Don’t know if there is a definition of “multiple” . If >1 then the algorithms chosen are at the discretion of the curator/lab.  There is a difference between lines of evidence and number of implementations.",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA090928",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll071"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn052",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:InSilicoPred120",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_005228.3",
            "label": "NM_005228.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA090928",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll071"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "SIFT",
          "statementOutcome": {
            "id": "CGEX:PredOutcome003",
            "label": "Damaging"
          }
        },
        {
          "id": "CGEX:InSilicoPred121",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_005228.3",
            "label": "NM_005228.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA090928",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll071"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Polyphen",
          "statementOutcome": {
            "id": "CGEX:PredOutcome010",
            "label": "Probably damaging"
          }
        },
        {
          "id": "CGEX:InSilicoPred122",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_005228.3",
            "label": "NM_005228.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA090928",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll071"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "FATHMM",
          "statementOutcome": {
            "id": "CGEX:PredOutcome011",
            "label": "Tolerated"
          }
        },
        {
          "id": "CGEX:InSilicoPred123",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_005228.3",
            "label": "NM_005228.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA090928",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll071"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "LRT",
          "statementOutcome": {
            "id": "CGEX:PredOutcome003",
            "label": "Damaging"
          }
        },
        {
          "id": "CGEX:InSilicoPred124",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_005228.3",
            "label": "NM_005228.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA090928",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll071"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "MetaSVM",
          "statementOutcome": {
            "id": "CGEX:PredOutcome011",
            "label": "Tolerated"
          }
        },
        {
          "id": "CGEX:InSilicoPred125",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_005228.3",
            "label": "NM_005228.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA090928",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll071"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Mutation assessor",
          "statementOutcome": {
            "id": "CGEX:PredOutcome011",
            "label": "Tolerated"
          }
        },
        {
          "id": "CGEX:InSilicoPred126",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_005228.3",
            "label": "NM_005228.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA090928",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll071"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Mutation taster",
          "statementOutcome": {
            "id": "CGEX:PredOutcome005",
            "label": "Disease-causing"
          }
        },
        {
          "id": "CGEX:InSilicoPred127",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_005228.3",
            "label": "NM_005228.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA090928",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll071"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Provean",
          "statementOutcome": {
            "id": "CGEX:PredOutcome003",
            "label": "Damaging"
          }
        },
        {
          "id": "CGEX:InSilicoPred128",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_005228.3",
            "label": "NM_005228.3"
          },
          "canonicalAllele": {
            "id": "CAR:CA090928",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll071"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Condel",
          "statementOutcome": {
            "id": "CGEX:PredOutcome004",
            "label": "Deleterious"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib444",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "7/21/16 10:29 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib445",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "7/21/16 10:29 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess315">CGEX:CritAssess315</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn053">CGEX:EvLn053</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99035">SEPIO-CG:99035</a></p> (PP3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090884">CAR:CA090884</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess315"><pre><code>{
  "id": "CGEX:CritAssess315",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99035",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP3",
    "description": "Multiple lines of computational evidence support a deleterious effect on the gene or gene product (conservation, evolutionary, splicing impact, etc.)",
    "usageNotes": "Don’t know if there is a definition of “multiple” . If >1 then the algorithms chosen are at the discretion of the curator/lab.  There is a difference between lines of evidence and number of implementations.",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA090884",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll072"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn053",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:InSilicoPred129",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "ENSEMBL:ENST00000566221.5",
            "label": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CAR:CA090884",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll072"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "SIFT",
          "statementOutcome": {
            "id": "CGEX:PredOutcome004",
            "label": "Deleterious"
          }
        },
        {
          "id": "CGEX:InSilicoPred130",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "ENSEMBL:ENST00000566221.5",
            "label": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CAR:CA090884",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll072"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Polyphen",
          "statementOutcome": {
            "id": "CGEX:PredOutcome010",
            "label": "Probably damaging"
          }
        },
        {
          "id": "CGEX:InSilicoPred131",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "ENSEMBL:ENST00000566221.5",
            "label": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CAR:CA090884",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll072"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "FATHMM",
          "statementOutcome": {
            "id": "CGEX:PredOutcome003",
            "label": "Damaging"
          }
        },
        {
          "id": "CGEX:InSilicoPred132",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "ENSEMBL:ENST00000566221.5",
            "label": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CAR:CA090884",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll072"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "LRT",
          "statementOutcome": {
            "id": "CGEX:PredOutcome003",
            "label": "Damaging"
          }
        },
        {
          "id": "CGEX:InSilicoPred133",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "ENSEMBL:ENST00000566221.5",
            "label": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CAR:CA090884",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll072"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "MetaSVM",
          "statementOutcome": {
            "id": "CGEX:PredOutcome011",
            "label": "Tolerated"
          }
        },
        {
          "id": "CGEX:InSilicoPred134",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "ENSEMBL:ENST00000566221.5",
            "label": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CAR:CA090884",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll072"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Mutation assessor",
          "statementOutcome": {
            "id": "CGEX:PredOutcome007",
            "label": "Medium functional impact"
          }
        },
        {
          "id": "CGEX:InSilicoPred135",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "ENSEMBL:ENST00000566221.5",
            "label": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CAR:CA090884",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll072"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Mutation taster",
          "statementOutcome": {
            "id": "CGEX:PredOutcome005",
            "label": "Disease-causing"
          }
        },
        {
          "id": "CGEX:InSilicoPred136",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "ENSEMBL:ENST00000566221.5",
            "label": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CAR:CA090884",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll072"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Provean",
          "statementOutcome": {
            "id": "CGEX:PredOutcome003",
            "label": "Damaging"
          }
        },
        {
          "id": "CGEX:InSilicoPred137",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "ENSEMBL:ENST00000566221.5",
            "label": "ENST00000566221.5"
          },
          "canonicalAllele": {
            "id": "CAR:CA090884",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll072"
          },
          "predictionType": {
            "id": "SEPIO:0000322",
            "label": "missense effect"
          },
          "algorithm": "Condel",
          "statementOutcome": {
            "id": "CGEX:PredOutcome004",
            "label": "Deleterious"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib446",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "7/21/16 12:15 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess318">CGEX:CritAssess318</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn055">CGEX:EvLn055</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99036">SEPIO-CG:99036</a></p> (PP4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090915">CAR:CA090915</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond022">CGEX:GenCond022</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess318"><pre><code>{
  "id": "CGEX:CritAssess318",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99036",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP4",
    "description": "Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA090915",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll076"
  },
  "condition": [
    {
      "id": "CGEX:GenCond022",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:774",
          "label": "hereditary hemorrhagic telangiectasia"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn055",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:LocHet148",
          "type": "LocusHeterogeneityStatement",
          "description": "HHT type 1 is only known to be caused by variants in ENG -",
          "condition": {
            "id": "CGEX:GenCond022",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:774",
                "label": "hereditary hemorrhagic telangiectasia"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          },
          "gene": {
            "id": "HGNC:3349",
            "label": "ENG"
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99002",
            "label": "complete"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib449",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/28/16 2:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib450",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "8/28/16 2:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess319">CGEX:CritAssess319</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn056">CGEX:EvLn056</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99036">SEPIO-CG:99036</a></p> (PP4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501234">CAR:CA501234</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond064">CGEX:GenCond064</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess319"><pre><code>{
  "id": "CGEX:CritAssess319",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99036",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP4",
    "description": "Patient’s phenotype or family history is highly specific for a disease with a single genetic etiology",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA501234",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll077"
  },
  "condition": [
    {
      "id": "CGEX:GenCond064",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:217569",
          "label": "hypertrophic cardiomyopathy"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn056",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:LocHet149",
          "type": "LocusHeterogeneityStatement",
          "description": "1.Only ~50% of HCM is genetic (can be caused by environment) \n2. More than 10 genes can cause HCM (specifically 9 sarcomere genes)",
          "condition": {
            "id": "CGEX:GenCond064",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:217569",
                "label": "hypertrophic cardiomyopathy"
              }
            ]
          },
          "gene": {
            "id": "HGNC:7577",
            "label": "MYH7"
          },
          "statementOutcome": {
            "id": "SEPIO:0000273",
            "label": "low"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib451",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 3:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess320">CGEX:CritAssess320</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn057">CGEX:EvLn057</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99037">SEPIO-CG:99037</a></p> (PP5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA019369">CAR:CA019369</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond023">CGEX:GenCond023</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess320"><pre><code>{
  "id": "CGEX:CritAssess320",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99037",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP5",
    "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA019369",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll078"
  },
  "condition": [
    {
      "id": "CGEX:GenCond023",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "OMIM:612555",
          "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn057",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp150",
          "type": "VariantPathogenicityInterpretation",
          "description": "Variant has been classified as “Pathogenic” by Sharing Clinical Reports Project (SCV000054057.3)",
          "variant": {
            "id": "CAR:CA019369",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll078"
          },
          "condition": [
            {
              "id": "CGEX:GenCond023",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "OMIM:612555",
                  "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib452",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 4:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess322">CGEX:CritAssess322</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn059">CGEX:EvLn059</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99037">SEPIO-CG:99037</a></p> (PP5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA276116">CAR:CA276116</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond025">CGEX:GenCond025</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess322"><pre><code>{
  "id": "CGEX:CritAssess322",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99037",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PP5",
    "description": "Reputable source recently reports variant as pathogenic, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "SEPIO:0000329",
      "label": "Pathogenic Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA276116",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll080"
  },
  "condition": [
    {
      "id": "CGEX:GenCond025",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:586",
          "label": "Cystic Fibrosis"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn059",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp152",
          "type": "VariantPathogenicityInterpretation",
          "description": "Variant has been classified as “Pathogenic” by Expert panel CFTR (SCV000245981.1)",
          "variant": {
            "id": "CAR:CA276116",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll080"
          },
          "condition": [
            {
              "id": "CGEX:GenCond025",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:586",
                  "label": "Cystic Fibrosis"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib454",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 6:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess323">CGEX:CritAssess323</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn060">CGEX:EvLn060</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99038">SEPIO-CG:99038</a></p> (BA1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090891">CAR:CA090891</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess323"><pre><code>{
  "id": "CGEX:CritAssess323",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99038",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BA1",
    "description": "Allele frequency is >5% in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "usageNotes": "Several groups have lowered the BA1 threshold (e.g. PTEN-1%). Also the sequence variant interpretation group is working on a modified version of this guideline including the minimum # of alleles that need to be examined and the population stratification effects, if any.",
    "defaultStrength": {
      "id": "SEPIO:0000325",
      "label": "Benign Stand Alone"
    }
  },
  "variant": {
    "id": "CAR:CA090891",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll081"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn060",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq153",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA090891",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll081"
          },
          "alleleCount": 6732,
          "alleleNumber": 120402,
          "alleleFrequency": 0.05591,
          "homozygousAlleleIndividualCount": 223
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib455",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "8/29/16 7:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess326">CGEX:CritAssess326</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn064">CGEX:EvLn064</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99039">SEPIO-CG:99039</a></p> (BS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA017597">CAR:CA017597</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond026">CGEX:GenCond026</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess326"><pre><code>{
  "id": "CGEX:CritAssess326",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99039",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA017597",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll084"
  },
  "condition": [
    {
      "id": "CGEX:GenCond026",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:558",
          "label": "Marfan Syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn064",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq158",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA017597",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll084"
          },
          "alleleCount": 89,
          "alleleNumber": 121410,
          "alleleFrequency": 0.0007,
          "homozygousAlleleIndividualCount": 1
        },
        {
          "id": "CGEX:CondPrev159",
          "type": "ConditionPrevelanceStatement",
          "condition": [
            {
              "id": "CGEX:GenCond026",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:558",
                  "label": "Marfan Syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "minimum": 0.0017,
          "maximum": 0.0007,
          "population": {
            "id": "SEPIO-CG:98002",
            "label": "Overall"
          }
        },
        {
          "id": "CGEX:CondPenet160",
          "type": "ConditionPenetranceStatement",
          "condition": [
            {
              "id": "CGEX:GenCond026",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:558",
                  "label": "Marfan Syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000006",
                "label": "Autosomal Dominant"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX2",
            "label": "High"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib458",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "8/29/16 10:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess327">CGEX:CritAssess327</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn065">CGEX:EvLn065</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99039">SEPIO-CG:99039</a></p> (BS1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090926">CAR:CA090926</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond027">CGEX:GenCond027</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess327"><pre><code>{
  "id": "CGEX:CritAssess327",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99039",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS1",
    "description": "Allele frequency is greater than expected for disorder",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA090926",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll085"
  },
  "condition": [
    {
      "id": "CGEX:GenCond027",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:2609",
          "label": "Mitochondrial complex 1 deficiency"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0001423",
        "label": "X-linked dominant inheritance"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn065",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq162",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA090926",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll085"
          },
          "alleleCount": 545,
          "alleleNumber": 87696,
          "alleleFrequency": 0.00625,
          "homozygousAlleleIndividualCount": 1,
          "hemizygousAlleleIndividualCount": 206
        },
        {
          "id": "CGEX:CondPrev163",
          "type": "ConditionPrevelanceStatement",
          "condition": [
            {
              "id": "CGEX:GenCond027",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:2609",
                  "label": "Mitochondrial complex 1 deficiency"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0001423",
                "label": "X-linked dominant inheritance"
              }
            }
          ],
          "prevelance": 0.00012,
          "population": {
            "id": "SEPIO-CG:98002",
            "label": "Overall"
          }
        },
        {
          "id": "CGEX:CondPenet164",
          "type": "ConditionPenetranceStatement",
          "condition": [
            {
              "id": "CGEX:GenCond027",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:2609",
                  "label": "Mitochondrial complex 1 deficiency"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0001423",
                "label": "X-linked dominant inheritance"
              }
            }
          ],
          "statementOutcome": {
            "id": "HP:000XXX2",
            "label": "High"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib459",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "8/29/16 11:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    },
    {
      "id": "CGEX:Contrib460",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent004",
        "type": "Agent",
        "label": "CSER Bakeoff Project"
      },
      "contributionDate": "8/29/16 11:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess333">CGEX:CritAssess333</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn075">CGEX:EvLn075</a></p><p><a href="details/details.html#CGEX:EvLn076">CGEX:EvLn076</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99041">SEPIO-CG:99041</a></p> (BS3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA132727">CAR:CA132727</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess333"><pre><code>{
  "id": "CGEX:CritAssess333",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99041",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS3",
    "description": "Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing",
    "usageNotes": "This will all depend on what is “established”, which could end up being defined by the different clinical domain working groups.",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA132727",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll091"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn075",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func192",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "Unchanged from reference",
          "assayType": {
            "id": "CGEX:AFAMID081",
            "label": "iodide efflux"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll092",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA109417",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll092"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9092",
              "type": "ContextualAlleleName",
              "name": "NP_000432..1:p.Leu117Phe",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:8818",
            "label": "SLC26A4"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn076",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func193",
          "type": "AlleleFunctionalImpactStatement",
          "description": "Both wild type and NP_000432..1:p.Leu117Phe localize to the same location",
          "resultDescription": "localized to apical membrane",
          "assayType": {
            "id": "CGEX:AFAMID079",
            "label": "cellular localization"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll092",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA109417",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll092"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9092",
              "type": "ContextualAlleleName",
              "name": "NP_000432..1:p.Leu117Phe",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:8818",
            "label": "SLC26A4"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib468",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/5/16 9:01 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess334">CGEX:CritAssess334</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn077">CGEX:EvLn077</a></p><p><a href="details/details.html#CGEX:EvLn078">CGEX:EvLn078</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99041">SEPIO-CG:99041</a></p> (BS3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501231">CAR:CA501231</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess334"><pre><code>{
  "id": "CGEX:CritAssess334",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99041",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS3",
    "description": "Well-established in vitro or in vivo functional studies show no damaging effect on protein function or splicing",
    "usageNotes": "This will all depend on what is “established”, which could end up being defined by the different clinical domain working groups.",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA501231",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll093"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn077",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func194",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "Unchanged from reference",
          "assayType": {
            "id": "CGEX:AFAMID080",
            "label": "Elk1 pohsphorylation via luciferase assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll093",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA501231",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll093"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9093",
              "type": "ContextualAlleleName",
              "name": "NM_152594.2(SPRED1): c.587C>T",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:20249",
            "label": "SPRED1"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn078",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:Func195",
          "type": "AlleleFunctionalImpactStatement",
          "resultDescription": "localized to apical membrane",
          "assayType": {
            "id": "CGEX:AFAMID080",
            "label": "Elk1 pohsphorylation via luciferase assay"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll093",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA501231",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll093"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9093",
              "type": "ContextualAlleleName",
              "name": "NM_152594.2(SPRED1): c.587C>T",
              "preferred": true
            }
          },
          "gene": {
            "id": "HGNC:20249",
            "label": "SPRED1"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib469",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/5/16 9:31 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess335">CGEX:CritAssess335</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn079">CGEX:EvLn079</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99042">SEPIO-CG:99042</a></p> (BS4)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA023690">CAR:CA023690</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond035">CGEX:GenCond035</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess335"><pre><code>{
  "id": "CGEX:CritAssess335",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99042",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BS4",
    "description": "Lack of segregation in affected members of a family",
    "usageNotes": "This rule is for “lack of segregation” meaning phenotype positive / genotype negative (and not pheno neg / geno pos). Also need to be careful of phenocopies. Also need rules for how many cases are needed in use this rule",
    "defaultStrength": {
      "id": "SEPIO:0000328",
      "label": "Benign Strong"
    }
  },
  "variant": {
    "id": "CAR:CA023690",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll094"
  },
  "condition": [
    {
      "id": "CGEX:GenCond035",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "OMIM:143890",
          "label": "Hypercholesterolemia, Familial"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000006",
        "label": "Autosomal Dominant"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn079",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:FamSeg196",
          "type": "FamilyCosegregationStatement",
          "description": "observed in one case diagnosed with Familial Hypercholesterolemia, characterized by elevated LDLC levels. Two siblings of the proband also have elevated LDLC levels but neither carry the 313+2T>C variant.",
          "condition": {
            "id": "CGEX:GenCond035",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "OMIM:143890",
                "label": "Hypercholesterolemia, Familial"
              }
            ],
            "inheritancePattern": {
              "id": "HP:0000006",
              "label": "Autosomal Dominant"
            }
          },
          "canonicalAllele": {
            "id": "CAR:CA023690",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll094"
          },
          "family": {
            "id": "CGEX:Fam370",
            "label": "Family #008"
          },
          "phenotypePositiveAllelePositive": 0,
          "phenotypePositiveAlleleNegative": 2,
          "inconsistentSegregationsObserved": {
            "id": "SEPIO-CG:99011",
            "label": "present"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib470",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "9/5/16 9:51 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess338">CGEX:CritAssess338</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn081">CGEX:EvLn081</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99043">SEPIO-CG:99043</a></p> (BP1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA020822">CAR:CA020822</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond036">CGEX:GenCond036</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess338"><pre><code>{
  "id": "CGEX:CritAssess338",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99043",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP1",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "usageNotes": "Is there any guidance on the proportion of pathogenic variants that are LOF to apply this rule?  99%? 90%? 75%?",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA020822",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll109"
  },
  "condition": [
    {
      "id": "CGEX:GenCond036",
      "type": "GeneticCondition",
      "description": "hereditary breast and ovarian cancer",
      "disease": [
        {
          "id": "Orphanet:145",
          "label": "hereditary breast and ovarian cancer syndrome"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn081",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon198",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll097",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA020822",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll109"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9097",
              "type": "ContextualAlleleName",
              "name": "NM_000059.3(BRCA2) :c.4779A>C",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001583",
            "label": "missense variant"
          }
        },
        {
          "id": "CGEX:Null407",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll097",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA020822",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll109"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9097",
              "type": "ContextualAlleleName",
              "name": "NM_000059.3(BRCA2) :c.4779A>C",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99015",
            "label": "function not lost"
          }
        },
        {
          "id": "CGEX:CondMech199",
          "type": "ConditionMechanismStatement",
          "description": "ClinVar shows 421 variants in BRCA2 that are pathogenic for HBOC. 402 of these (~95%) are frameshift, nonsense, or splice-site variants.",
          "statementOutcome": {
            "id": "SO:0002054",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000270",
            "label": "primary"
          },
          "gene": {
            "id": "HGNC:1101",
            "label": "BRCA2"
          },
          "condition": {
            "id": "CGEX:GenCond036",
            "type": "GeneticCondition",
            "description": "hereditary breast and ovarian cancer",
            "disease": [
              {
                "id": "Orphanet:145",
                "label": "hereditary breast and ovarian cancer syndrome"
              }
            ]
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib473",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "9/6/16 8:00 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess339">CGEX:CritAssess339</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn082">CGEX:EvLn082</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99043">SEPIO-CG:99043</a></p> (BP1)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA020822">CAR:CA020822</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond037">CGEX:GenCond037</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess339"><pre><code>{
  "id": "CGEX:CritAssess339",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99043",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP1",
    "description": "Missense variant in a gene for which primarily truncating variants are known to cause disease",
    "usageNotes": "Is there any guidance on the proportion of pathogenic variants that are LOF to apply this rule?  99%? 90%? 75%?",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA020822",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll109"
  },
  "condition": [
    {
      "id": "CGEX:GenCond037",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:84",
          "label": "Fanconi Anemia"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn082",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:MolCon198",
          "type": "AlleleMolecularConsequenceStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll097",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA020822",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll109"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9097",
              "type": "ContextualAlleleName",
              "name": "NM_000059.3(BRCA2) :c.4779A>C",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SO:0001583",
            "label": "missense variant"
          }
        },
        {
          "id": "CGEX:Null408",
          "type": "NullAlleleStatement",
          "contextualAllele": {
            "id": "CGEX:CtxAll097",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA020822",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll109"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9097",
              "type": "ContextualAlleleName",
              "name": "NM_000059.3(BRCA2) :c.4779A>C",
              "preferred": true
            }
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99015",
            "label": "function not lost"
          }
        },
        {
          "id": "CGEX:CondMech200",
          "type": "ConditionMechanismStatement",
          "description": "ClinVar shows 61 variants in BRCA2 that are pathogenic for Fanconi Anemia. 55 of these (~90%) are frameshift, nonsense, or splice-site variants.",
          "statementOutcome": {
            "id": "SO:0002054",
            "label": "loss of function variant"
          },
          "mechanismConfidence": {
            "id": "SEPIO:0000270",
            "label": "primary"
          },
          "gene": {
            "id": "HGNC:1101",
            "label": "BRCA2"
          },
          "condition": {
            "id": "CGEX:GenCond037",
            "type": "GeneticCondition",
            "disease": [
              {
                "id": "Orphanet:84",
                "label": "Fanconi Anemia"
              }
            ]
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib474",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent005",
        "type": "Agent",
        "label": "G. Rajar"
      },
      "contributionDate": "9/6/16 9:00 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess342">CGEX:CritAssess342</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn085">CGEX:EvLn085</a></p><p><a href="details/details.html#CGEX:EvLn086">CGEX:EvLn086</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99044">SEPIO-CG:99044</a></p> (BP2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA143572">CAR:CA143572</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond038">CGEX:GenCond038</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess342"><pre><code>{
  "id": "CGEX:CritAssess342",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99044",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP2",
    "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA143572",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll100"
  },
  "condition": [
    {
      "id": "CGEX:GenCond038",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:791",
          "label": "Retinitis Pigmentosa"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn085",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndGeno206",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv353",
            "label": "individual #021"
          },
          "genotype": [
            {
              "id": "CGEX:Geno401",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo400",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA143572",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll100"
                    },
                    {
                      "id": "CAR:CA252233",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll101"
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:IndCond208",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv353",
            "label": "individual #021"
          },
          "condition": [
            {
              "id": "CGEX:GenCond038",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:791",
                  "label": "Retinitis Pigmentosa"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:VarInterp210",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA252233",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll101"
          },
          "condition": [
            {
              "id": "CGEX:GenCond038",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:791",
                  "label": "Retinitis Pigmentosa"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    },
    {
      "id": "CGEX:EvLn086",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndGeno207",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv354",
            "label": "individual #022"
          },
          "genotype": [
            {
              "id": "CGEX:Geno403",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo402",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA143572",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll100"
                    },
                    {
                      "id": "CAR:CA252233",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll101"
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:IndCond209",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv354",
            "label": "individual #022"
          },
          "condition": [
            {
              "id": "CGEX:GenCond038",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:791",
                  "label": "Retinitis Pigmentosa"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:VarInterp210",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA252233",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll101"
          },
          "condition": [
            {
              "id": "CGEX:GenCond038",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:791",
                  "label": "Retinitis Pigmentosa"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib477",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 12:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess343">CGEX:CritAssess343</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn087">CGEX:EvLn087</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99044">SEPIO-CG:99044</a></p> (BP2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA182840">CAR:CA182840</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond039">CGEX:GenCond039</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess343"><pre><code>{
  "id": "CGEX:CritAssess343",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99044",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP2",
    "description": "Observed in trans with a pathogenic variant for a fully penetrant dominant gene/disorder or observed in cis with a pathogenic variant in any inheritance pattern",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA182840",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll102"
  },
  "condition": [
    {
      "id": "CGEX:GenCond039",
      "type": "GeneticCondition",
      "phenotype": [
        {
          "id": "HP:0012873",
          "label": "Absent vas deferens"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn087",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndGeno211",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv355",
            "label": "individual #023"
          },
          "genotype": [
            {
              "id": "CGEX:Geno405",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo404",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA182840",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll102"
                    },
                    {
                      "id": "CAR:CA182839",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll103"
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:IndCond212",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv355",
            "label": "individual #023"
          },
          "condition": [
            {
              "id": "CGEX:GenCond039",
              "type": "GeneticCondition",
              "phenotype": [
                {
                  "id": "HP:0012873",
                  "label": "Absent vas deferens"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:VarInterp213",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA182839",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll103"
          },
          "condition": [
            {
              "id": "CGEX:GenCond039",
              "type": "GeneticCondition",
              "phenotype": [
                {
                  "id": "HP:0012873",
                  "label": "Absent vas deferens"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib478",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/6/16 1:00 PM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess345">CGEX:CritAssess345</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn089">CGEX:EvLn089</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99045">SEPIO-CG:99045</a></p> (BP3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA148212">CAR:CA148212</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess345"><pre><code>{
  "id": "CGEX:CritAssess345",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99045",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP3",
    "description": "In-frame deletions/insertions in a repetitive region without a known function",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA148212",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll106"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn089",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:RegTyp220",
          "type": "RegionTypeStatement",
          "description": "region does not have a known function.",
          "region": {
            "id": "CGEX:Loc438",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_001271208.1",
              "label": "NM_001271208.1"
            },
            "start": 154,
            "stop": 183
          },
          "statementOutcome": [
            {
              "id": "SO:0002188",
              "label": "functional region"
            }
          ],
          "outcomeQualifier": "NOT"
        },
        {
          "id": "CGEX:RegTyp221",
          "type": "RegionTypeStatement",
          "description": "region is repeating.",
          "region": {
            "id": "CGEX:Loc438",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_001271208.1",
              "label": "NM_001271208.1"
            },
            "start": 154,
            "stop": 183
          },
          "statementOutcome": [
            {
              "id": "SO:0000657",
              "label": "repeat region "
            }
          ]
        },
        {
          "id": "CGEX:RegAll391",
          "type": "RegionAllelesStatement",
          "description": "region is repeating but does not have a known function.",
          "region": {
            "id": "CGEX:Loc438",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_001271208.1",
              "label": "NM_001271208.1"
            },
            "start": 154,
            "stop": 183
          },
          "allele": [
            {
              "id": "CGEX:CtxAll106",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": {
                "id": "CAR:CA148212",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll106"
              },
              "alleleName": {
                "id": "CGEX:CtxNm9106",
                "type": "ContextualAlleleName",
                "name": "NM_001271208.1(NEB):c.169_183delCTGGCACAGCCAGCA",
                "preferred": true
              }
            }
          ],
          "statementOutcome": {
            "id": "RO:0002656",
            "label": "overlaps region"
          }
        },
        {
          "id": "CGEX:MolCon223",
          "type": "AlleleMolecularConsequenceStatement",
          "statementOutcome": {
            "id": "SO:0001825",
            "label": "conservative Inframe Deletion"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll106",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA148212",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll106"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9106",
              "type": "ContextualAlleleName",
              "name": "NM_001271208.1(NEB):c.169_183delCTGGCACAGCCAGCA",
              "preferred": true
            }
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib489",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/15/16 8:12 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess347">CGEX:CritAssess347</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn091">CGEX:EvLn091</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99045">SEPIO-CG:99045</a></p> (BP3)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA184168">CAR:CA184168</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess347"><pre><code>{
  "id": "CGEX:CritAssess347",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99045",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP3",
    "description": "In-frame deletions/insertions in a repetitive region without a known function",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA184168",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll108"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn091",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:RegTyp229",
          "type": "RegionTypeStatement",
          "region": {
            "id": "CGEX:Loc440",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_147196.2",
              "label": "NM_147196.2"
            },
            "start": 522,
            "stop": 548
          },
          "statementOutcome": [
            {
              "id": "SO:0000657",
              "label": "repeat region "
            }
          ]
        },
        {
          "id": "CGEX:RegAll395",
          "type": "RegionAllelesStatement",
          "region": {
            "id": "CGEX:Loc440",
            "type": "SequenceLocation",
            "referenceSequence": {
              "id": "REFSEQ:NM_147196.2",
              "label": "NM_147196.2"
            },
            "start": 522,
            "stop": 548
          },
          "allele": [
            {
              "id": "CGEX:CtxAll108",
              "type": "ContextualAllele",
              "relatedCanonicalAllele": {
                "id": "CAR:CA184168",
                "type": "CanonicalAllele",
                "relatedContextualAllele": "CGEX:CtxAll108"
              },
              "alleleName": {
                "id": "CGEX:CtxNm9108",
                "type": "ContextualAlleleName",
                "name": "NM_147196.2(TMIE):c.391_393dupAAG",
                "preferred": true
              }
            }
          ],
          "statementOutcome": {
            "id": "RO:0002656",
            "label": "overlaps region"
          }
        },
        {
          "id": "CGEX:MolCon231",
          "type": "AlleleMolecularConsequenceStatement",
          "statementOutcome": {
            "id": "SO:0001823",
            "label": "conservative Inframe Insertion"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll108",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA184168",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll108"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9108",
              "type": "ContextualAlleleName",
              "name": "NM_147196.2(TMIE):c.391_393dupAAG",
              "preferred": true
            }
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib491",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "9/15/16 10:40 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess352">CGEX:CritAssess352</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn094">CGEX:EvLn094</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99047">SEPIO-CG:99047</a></p> (BP5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA137502">CAR:CA137502</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond060">CGEX:GenCond060</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess352"><pre><code>{
  "id": "CGEX:CritAssess352",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99047",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA137502",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll114"
  },
  "condition": [
    {
      "id": "CGEX:GenCond060",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:87884",
          "label": "nonsyndromic deafness"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn094",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndGeno244",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv358",
            "label": "individual #026"
          },
          "genotype": [
            {
              "id": "CGEX:Geno414",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo413",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA137502",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll114"
                    }
                  ]
                }
              ]
            },
            {
              "id": "CGEX:Geno417",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo415",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA172240",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll115"
                    }
                  ]
                },
                {
                  "id": "CGEX:Haplo416",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA274460",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll116"
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:IndCond246",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv358",
            "label": "individual #026"
          },
          "condition": [
            {
              "id": "CGEX:GenCond060",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:VarInterp247",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA172240",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll115"
          },
          "condition": [
            {
              "id": "CGEX:GenCond060",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA26332-9",
            "label": "Likely Pathogenic"
          }
        },
        {
          "id": "CGEX:VarInterp248",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA274460",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll116"
          },
          "condition": [
            {
              "id": "CGEX:GenCond060",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:87884",
                  "label": "nonsyndromic deafness"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA26332-9",
            "label": "Likely Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib496",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/6/16 8:30 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess353">CGEX:CritAssess353</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn095">CGEX:EvLn095</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99047">SEPIO-CG:99047</a></p> (BP5)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA137502">CAR:CA137502</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond071">CGEX:GenCond071</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess353"><pre><code>{
  "id": "CGEX:CritAssess353",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99047",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP5",
    "description": "Variant found in a case with an alternate molecular basis for disease",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA137502",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll114"
  },
  "condition": [
    {
      "id": "CGEX:GenCond071",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:886",
          "label": "Usher syndrome"
        }
      ],
      "inheritancePattern": {
        "id": "HP:0000007",
        "label": "Autosomal Recessive"
      }
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn095",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:IndGeno249",
          "type": "IndividualGenotypeStatement",
          "individual": {
            "id": "CGEX:Indiv359",
            "label": "individual #027"
          },
          "genotype": [
            {
              "id": "CGEX:Geno419",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo418",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA137502",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll114"
                    }
                  ]
                }
              ],
              "zygosity": {
                "id": "GENO:0000136",
                "label": "homozygous"
              }
            },
            {
              "id": "CGEX:Geno421",
              "type": "Genotype",
              "hasPart": [
                {
                  "id": "CGEX:Haplo420",
                  "type": "Haplotype",
                  "hasPart": [
                    {
                      "id": "CAR:CA277980",
                      "type": "CanonicalAllele",
                      "relatedContextualAllele": "CGEX:CtxAll117"
                    }
                  ]
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:IndCond250",
          "type": "IndividualConditionStatement",
          "individual": {
            "id": "CGEX:Indiv359",
            "label": "individual #027"
          },
          "condition": [
            {
              "id": "CGEX:GenCond071",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:886",
                  "label": "Usher syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "statementOutcome": {
            "id": "SEPIO-CG:99009",
            "label": "affected"
          }
        },
        {
          "id": "CGEX:VarInterp251",
          "type": "VariantPathogenicityInterpretation",
          "variant": {
            "id": "CAR:CA137502",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll114"
          },
          "condition": [
            {
              "id": "CGEX:GenCond071",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:886",
                  "label": "Usher syndrome"
                }
              ],
              "inheritancePattern": {
                "id": "HP:0000007",
                "label": "Autosomal Recessive"
              }
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6668-3",
            "label": "Pathogenic"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib497",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/6/16 8:54 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess355">CGEX:CritAssess355</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn097">CGEX:EvLn097</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99048">SEPIO-CG:99048</a></p> (BP6)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA276195">CAR:CA276195</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond073">CGEX:GenCond073</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess355"><pre><code>{
  "id": "CGEX:CritAssess355",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99048",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP6",
    "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA276195",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll120"
  },
  "condition": [
    {
      "id": "CGEX:GenCond073",
      "type": "GeneticCondition",
      "description": "hereditary breast and ovarian cancer",
      "disease": [
        {
          "id": "Orphanet:145",
          "label": "hereditary breast and ovarian cancer syndrome"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn097",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp255",
          "type": "VariantPathogenicityInterpretation",
          "description": "classified by ENIGMA.",
          "variant": {
            "id": "CAR:CA276195",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll120"
          },
          "condition": [
            {
              "id": "CGEX:GenCond073",
              "type": "GeneticCondition",
              "description": "hereditary breast and ovarian cancer",
              "disease": [
                {
                  "id": "Orphanet:145",
                  "label": "hereditary breast and ovarian cancer syndrome"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6675-8",
            "label": "Benign"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib499",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/6/16 9:45 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess356">CGEX:CritAssess356</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn098">CGEX:EvLn098</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99048">SEPIO-CG:99048</a></p> (BP6)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA016113">CAR:CA016113</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond043">CGEX:GenCond043</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess356"><pre><code>{
  "id": "CGEX:CritAssess356",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99048",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP6",
    "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA016113",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll121"
  },
  "condition": [
    {
      "id": "CGEX:GenCond043",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "OMIM:612555",
          "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn098",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp256",
          "type": "VariantPathogenicityInterpretation",
          "description": "classified by Sharing Clinical Reports Project.",
          "variant": {
            "id": "CAR:CA016113",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll121"
          },
          "condition": [
            {
              "id": "CGEX:GenCond043",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "OMIM:612555",
                  "label": "Breast-Ovarian Cancer, Familial, Susceptibility To, 2"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6675-8",
            "label": "Benign"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib500",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:02 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess357">CGEX:CritAssess357</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn099">CGEX:EvLn099</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99048">SEPIO-CG:99048</a></p> (BP6)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA113946">CAR:CA113946</a></p>
         </td>
          <td>
              <p><a href="details/details.html#CGEX:GenCond044">CGEX:GenCond044</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000224">SEPIO:0000224</a></p> (Not Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess357"><pre><code>{
  "id": "CGEX:CritAssess357",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99048",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP6",
    "description": "Reputable source recently reports variant as benign, but the evidence is not available to the laboratory to perform an independent evaluation",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA113946",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll122"
  },
  "condition": [
    {
      "id": "CGEX:GenCond044",
      "type": "GeneticCondition",
      "disease": [
        {
          "id": "Orphanet:414",
          "label": "Gyrate atrophy of choroid and retina"
        }
      ]
    }
  ],
  "statementOutcome": {
    "id": "SEPIO:0000224",
    "label": "Not Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn099",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:VarInterp257",
          "type": "VariantPathogenicityInterpretation",
          "description": "classified by OMIM",
          "variant": {
            "id": "CAR:CA113946",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll122"
          },
          "condition": [
            {
              "id": "CGEX:GenCond044",
              "type": "GeneticCondition",
              "disease": [
                {
                  "id": "Orphanet:414",
                  "label": "Gyrate atrophy of choroid and retina"
                }
              ]
            }
          ],
          "statementOutcome": {
            "id": "LOINC:LA6675-8",
            "label": "Benign"
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib501",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:03 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess358">CGEX:CritAssess358</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn100">CGEX:EvLn100</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99049">SEPIO-CG:99049</a></p> (BP7)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501232">CAR:CA501232</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess358"><pre><code>{
  "id": "CGEX:CritAssess358",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99049",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP7",
    "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA501232",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll123"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn100",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:InSilicoPred258",
          "type": "InSilicoPredictionStatement",
          "transcript": {
            "id": "REFSEQ:NM_138691.2",
            "label": "NM_138691.2"
          },
          "canonicalAllele": {
            "id": "CAR:CA501232",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll123"
          },
          "predictionType": {
            "id": "SEPIO:0000323",
            "label": "splicing prediction"
          },
          "statementOutcome": {
            "id": "CGEX:PredOutcome008",
            "label": "No impact"
          }
        },
        {
          "id": "CGEX:AllConserv259",
          "type": "AlleleConservationStatement",
          "allele": {
            "id": "CAR:CA501232",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll123"
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99017",
            "label": "not conserved region"
          },
          "algorithm": "phyloP",
          "evidenceLine": [
            {
              "id": "CGEX:EvLn135",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllConservScore245",
                  "type": "AlleleConservationScoreStatement",
                  "allele": {
                    "id": "CAR:CA501232",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll123"
                  },
                  "algorithm": "phyloP",
                  "score": -2.297
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:AllConserv260",
          "type": "AlleleConservationStatement",
          "allele": {
            "id": "CAR:CA501232",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll123"
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99017",
            "label": "not conserved region"
          },
          "algorithm": "PhastCons",
          "evidenceLine": [
            {
              "id": "CGEX:EvLn136",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllConservScore228",
                  "type": "AlleleConservationScoreStatement",
                  "allele": {
                    "id": "CAR:CA501232",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll123"
                  },
                  "algorithm": "PhastCons",
                  "score": 0.039
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:MolCon261",
          "type": "AlleleMolecularConsequenceStatement",
          "statementOutcome": {
            "id": "SO:0001819",
            "label": "synonymous variant"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll123",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA501232",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll123"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9123",
              "type": "ContextualAlleleName",
              "name": "NM_138691.2 (TMC1): c.684C>T",
              "preferred": true
            }
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib502",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:04 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess359">CGEX:CritAssess359</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn101">CGEX:EvLn101</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99049">SEPIO-CG:99049</a></p> (BP7)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA013536">CAR:CA013536</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess359"><pre><code>{
  "id": "CGEX:CritAssess359",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99049",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "BP7",
    "description": "A synonymous (silent) variant for which splicing prediction algorithms predict no impact to the splice consensus sequence nor the creation of a new splice site AND the nucleotide is not highly conserved",
    "defaultStrength": {
      "id": "SEPIO:0000327",
      "label": "Benign Supporting"
    }
  },
  "variant": {
    "id": "CAR:CA013536",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll124"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn101",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:InSilicoPred262",
          "type": "InSilicoPredictionStatement",
          "description": "Provenance would show that this comes from Alamut",
          "transcript": {
            "id": "REFSEQ:NM_000257.2",
            "label": "NM_000257.2"
          },
          "canonicalAllele": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "predictionType": {
            "id": "SEPIO:0000323",
            "label": "splicing prediction"
          },
          "statementOutcome": {
            "id": "CGEX:PredOutcome008",
            "label": "No impact"
          }
        },
        {
          "id": "CGEX:AllConserv263",
          "type": "AlleleConservationStatement",
          "allele": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99017",
            "label": "not conserved region"
          },
          "algorithm": "phyloP",
          "evidenceLine": [
            {
              "id": "CGEX:EvLn137",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllConservScore230",
                  "type": "AlleleConservationScoreStatement",
                  "allele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "algorithm": "phyloP",
                  "score": -0.76
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:AllConserv264",
          "type": "AlleleConservationStatement",
          "allele": {
            "id": "CAR:CA013536",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll124"
          },
          "statementOutcome": {
            "id": "SEPIO-CG:99017",
            "label": "not conserved region"
          },
          "algorithm": "PhastCons",
          "evidenceLine": [
            {
              "id": "CGEX:EvLn138",
              "type": "EvidenceLine",
              "evidenceItem": [
                {
                  "id": "CGEX:AllConservScore226",
                  "type": "AlleleConservationScoreStatement",
                  "allele": {
                    "id": "CAR:CA013536",
                    "type": "CanonicalAllele",
                    "relatedContextualAllele": "CGEX:CtxAll124"
                  },
                  "algorithm": "PhastCons",
                  "score": 0.63
                }
              ]
            }
          ]
        },
        {
          "id": "CGEX:MolCon265",
          "type": "AlleleMolecularConsequenceStatement",
          "statementOutcome": {
            "id": "SO:0001819",
            "label": "synonymous variant"
          },
          "contextualAllele": {
            "id": "CGEX:CtxAll124",
            "type": "ContextualAllele",
            "relatedCanonicalAllele": {
              "id": "CAR:CA013536",
              "type": "CanonicalAllele",
              "relatedContextualAllele": "CGEX:CtxAll124"
            },
            "alleleName": {
              "id": "CGEX:CtxNm9124",
              "type": "ContextualAlleleName",
              "name": "NM_000257.2(MYH7):c.327C>T",
              "preferred": true
            }
          }
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib503",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent007",
        "type": "Agent",
        "label": "Stefan DiMarino"
      },
      "contributionDate": "10/10/16 8:05 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:CritAssess376">CGEX:CritAssess376</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:EvLn126">CGEX:EvLn126</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:99028">SEPIO-CG:99028</a></p> (PM2)
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA280055">CAR:CA280055</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000223">SEPIO:0000223</a></p> (Met)
         </td>
          <!--
          <td><div class="example_view" id="CGEX:CritAssess376"><pre><code>{
  "id": "CGEX:CritAssess376",
  "type": "CriterionAssessment",
  "criterion": {
    "id": "SEPIO-CG:99028",
    "type": "VariantPathogenicityInterpretationCriterion",
    "label": "PM2",
    "description": "Absent from controls (or at extremely low frequency if recessive) (Table 6) in Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium",
    "usageNotes": "Some groups are allowing to rule to be used if even not 100% absent\nFor instance, Cardio has allowed PM2 to be used if MAF is <0.05% with 95% CI if\nCI lower is ~0.00% (at 2 decimal places) AND CI upper is below 0.05%\n Caveat: Population data for insertions/deletions may be poorly called by next-generation sequencing",
    "defaultStrength": {
      "id": "SEPIO:0000216",
      "label": "Pathogenic Moderate"
    }
  },
  "variant": {
    "id": "CAR:CA280055",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll131"
  },
  "statementOutcome": {
    "id": "SEPIO:0000223",
    "label": "Met"
  },
  "evidenceLine": [
    {
      "id": "CGEX:EvLn126",
      "type": "EvidenceLine",
      "evidenceItem": [
        {
          "id": "CGEX:AllFreq101",
          "type": "PopulationAlleleFrequencyStatement",
          "ascertainment": {
            "id": "SEPIO:0000332",
            "label": "ExAC ascertainment method"
          },
          "population": {
            "id": "SEPIO-CG:98001",
            "label": "Combined"
          },
          "allele": {
            "id": "CAR:CA280055",
            "type": "CanonicalAllele",
            "relatedContextualAllele": "CGEX:CtxAll131"
          },
          "alleleCount": 0,
          "alleleFrequency": 0.0,
          "medianCoverage": 57.0
        }
      ]
    }
  ],
  "contribution": [
    {
      "id": "CGEX:Contrib515",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent010",
        "type": "Agent",
        "label": "ACME Molecular Lab"
      },
      "contributionDate": "6/11/2011 11:45 AM",
      "contributionRole": {
        "id": "SEPIO:0000154",
        "label": "assessor"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../">Model Overview</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../getting_started.html">Getting Started</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../deeper_dive.html">A Deeper Dive</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../faq.html">FAQ</a></h5>
              </li>
              <li class="list-group-item"><h5><a href="./">Entities</a></h5><ul class='list-unstyled'><li><a href="Agent.html"></a></li><li><a href="AggregateCosegregationStatement.html"></a></li><li><a href="AlleleConservationScoreStatement.html"></a></li><li><a href="AlleleConservationStatement.html"></a></li><li><a href="AlleleFunctionalImpactStatement.html"></a></li><li><a href="AlleleMolecularConsequenceStatement.html"></a></li><li><a href="BenignMissenseVariationRateStatement.html"></a></li><li><a href="CanonicalAllele.html"></a></li><li><a href="CaseControlAlleleFrequencyStatement.html"></a></li><li><a href="ConditionMechanismStatement.html"></a></li><li><a href="ConditionPenetranceStatement.html"></a></li><li><a href="ConditionPrevelanceStatement.html"></a></li><li><a href="ContextualAllele.html"></a></li><li><a href="ContextualAlleleName.html"></a></li><li><a href="Contribution.html"></a></li><li class="active"><a href="CriterionAssessment.html"></a></li><li><a href="DomainEntity.html"></a></li><li><a href="Entity.html"></a></li><li><a href="EvidenceLine.html"></a></li><li><a href="FamilyConditionStatement.html"></a></li><li><a href="FamilyCosegregationStatement.html"></a></li><li><a href="GeneticCondition.html"></a></li><li><a href="Genotype.html"></a></li><li><a href="Haplotype.html"></a></li><li><a href="IdentifierSystem.html"></a></li><li><a href="InSilicoPredictionScoreStatement.html"></a></li><li><a href="InSilicoPredictionStatement.html"></a></li><li><a href="IndividualAlleleInheritanceStatement.html"></a></li><li><a href="IndividualConditionStatement.html"></a></li><li><a href="IndividualGenotypeStatement.html"></a></li><li><a href="LocusHeterogeneityStatement.html"></a></li><li><a href="NullAlleleStatement.html"></a></li><li><a href="PopulationAlleleFrequencyStatement.html"></a></li><li><a href="Position.html"></a></li><li><a href="ReferenceCoordinate.html"></a></li><li><a href="RegionAllelesStatement.html"></a></li><li><a href="RegionTypeStatement.html"></a></li><li><a href="SequenceLocation.html"></a></li><li><a href="Statement.html"></a></li><li><a href="UserLabel.html"></a></li><li><a href="ValueSet.html"></a></li><li><a href="VariantPathogenicityInterpretation.html"></a></li><li><a href="VariantPathogenicityInterpretationCriterion.html"></a></li><li><a href="VariantPathogenicityInterpretationGuideline.html"></a></li></ul></li>
              <li class='list-group-item'><h5><a href="../terminologies/">Terminologies</a></h5></li>
              <li class="list-group-item">
                <h5><a href="../tools.html">Tools</a></h5>
              </li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2018 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../javascripts/jquery.js"></script><script src="../javascripts/bootstrap.js"></script><script src="../javascripts/jquery.jsonview.js"></script><script src="../javascripts/all.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
