<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen DataExchange Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../stylesheets/brand2.css" rel="stylesheet" /><link href="../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a href="/" class="navbar-brand">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="../images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a href="http://www.clinicalgenome.org" target="_blank"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>

        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">PopulationAlleleFrequencyStatement</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="../">Interpretation Model</a></li>
<li><a href="./">Entities</a></li>

              </ol>
                <blockquote><p>A statement that describes the observed frequency of an allele as determined in a defined group of individuals (typically a representative sampling that serves as a proxy for some larger population of individuals).
</p>
                <STRONG>
                  <div>SEPIO:0000227</div>
                      <i>Descended from Statement</i>
                </STRONG>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            

    <!-- this is primarily the Scope and Usage section -->
    <h2>Scope and Usage</h2>
    <p>The population in which allele frequency is determined is defined based on features such as  ethnicity, nationality, geography, socio-economic status, etc.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
                <th>IRI</th>
                <th>Defined in</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>allele</td>
                <td>CanonicalAllele</td>
                <td>0..1</td>
                <td>The variant for which the frequency is given</td>
                <td>SEPIO:0000275</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>population</td>
                <td>@id</td>
                <td>0..1</td>
                <td>Ancestry of population or group over which the frequency was measured</td>
                <td>SEPIO:0000277</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>ascertainment</td>
                <td>@id</td>
                <td>0..1</td>
                <td>The criteria applied to create the dataset in which the frequency was measured</td>
                <td>SEPIO:0000255</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>alleleFrequency</td>
                <td>number</td>
                <td>0..1</td>
                <td>Frequency of the named allele in the population</td>
                <td>SEPIO:0000297</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>alleleCount</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Frequency numerator: number of times the allele was observed</td>
                <td>SEPIO:0000296</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>alleleNumber</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Frequency denominator: number of observations</td>
                <td>SEPIO:0000298</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>individualCount</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals sampled</td>
                <td>SEPIO:0000302</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>homozygousAlleleIndividualCount</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals in the group that are homozygous for the allele</td>
                <td>SEPIO:0000301</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>heterozygousAlleleIndividualCount</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals in the group who are heterozygous for the allele.</td>
                <td>SEPIO:0000300</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>hemizygousAlleleIndividualCount</td>
                <td>integer</td>
                <td>0..1</td>
                <td>Number of individuals in the group that are hemizygous for the allele</td>
                <td>SEPIO:0000299</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>medianCoverage</td>
                <td>number</td>
                <td>0..1</td>
                <td>Median coverage at the allele site in the data set</td>
                <td>SEPIO:0000303</td>
                <td>PopulationAlleleFrequencyStatement</td>
            </tr>
            <tr>
                <td>label</td>
                <td>string</td>
                <td>0..1</td>
                <td>A name given to the resource.</td>
                <td>RDFS:label</td>
                <td>Entity</td>
            </tr>
            <tr>
                <td>description</td>
                <td>string</td>
                <td>0..1</td>
                <td>Description may include but is not limited to: an abstract, a table of contents, a graphical representation, or a free-text account of the resource.</td>
                <td>DC:description</td>
                <td>Entity</td>
            </tr>
            <tr>
                <td>userLabelDictionary</td>
                <td>UserLabel</td>
                <td>0..*</td>
                <td>An optional label defined by the user. Used for custom entities or to clarify the preferred user label on existing entities with non-preferred labels.</td>
                <td>SEPIO:0000422</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>outcomeQualifier</td>
                <td>string</td>
                <td>0..1</td>
                <td>Use “NOT” as the value of this property to assert that the statement is negated.</td>
                <td>SEPIO:0000346</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>evidenceLine</td>
                <td>EvidenceLine</td>
                <td>0..*</td>
                <td>supporting evidence</td>
                <td>SEPIO:0000006</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>contribution</td>
                <td>Contribution</td>
                <td>0..*</td>
                <td>The contributions (agent, time and role) made to this entity</td>
                <td>SEPIO:0000159</td>
                <td>Statement</td>
            </tr>
            <tr>
                <td>source</td>
                <td>string</td>
                <td>0..*</td>
                <td>A string indicating the source of a Statement</td>
                <td>DC:source</td>
                <td>Statement</td>
            </tr>
            </tbody>
        </table>


    <h2>Instances</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>label</th>
        <th>description</th>
        <th>userLabelDictionary</th>
        <th>outcomeQualifier</th>
        <th>evidenceLine</th>
        <th>allele</th>
        <th>population</th>
        <th>ascertainment</th>
        <th>alleleFrequency</th>
        <th>alleleCount</th>
        <th>alleleNumber</th>
        <th>individualCount</th>
        <th>homozygousAlleleIndividualCount</th>
        <th>heterozygousAlleleIndividualCount</th>
        <th>hemizygousAlleleIndividualCount</th>
        <th>medianCoverage</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq032">CGEX:AllFreq032</a></p></td>
          <td>
              
         </td>
          <td>
              <p>This allele is thought to account for between 2% and 10% of patients with Malignant hyperthermia.</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA024311">CAR:CA024311</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq032"><pre><code>{
  "id": "CGEX:AllFreq032",
  "type": "PopulationAlleleFrequencyStatement",
  "description": "This allele is thought to account for between 2% and 10% of patients with Malignant hyperthermia.",
  "allele": {
    "id": "CAR:CA024311",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll026"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq028">CGEX:AllFreq028</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA253316">CAR:CA253316</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              
         </td>
          <td>
              <p>105</p>
         </td>
          <td>
              <p>121336</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq028"><pre><code>{
  "id": "CGEX:AllFreq028",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "allele": {
    "id": "CAR:CA253316",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll025"
  },
  "alleleCount": 105,
  "alleleNumber": 121336
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq029">CGEX:AllFreq029</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:UsrLabl001">CGEX:UsrLabl001</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA253316">CAR:CA253316</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:Ascrt0001">CGEX:Ascrt0001</a></p> (LMM Hearing Loss)
         </td>
          <td>
              
         </td>
          <td>
              <p>32</p>
         </td>
          <td>
              <p>4076</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq029"><pre><code>{
  "id": "CGEX:AllFreq029",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "CGEX:Ascrt0001",
    "label": "LMM Hearing Loss"
  },
  "allele": {
    "id": "CAR:CA253316",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll025"
  },
  "alleleCount": 32,
  "alleleNumber": 4076,
  "userLabelDictionary": [
    {
      "id": "CGEX:UsrLabl001",
      "type": "UserLabel",
      "description": "Across all published data (and LMM data) with hearing loss.",
      "labelFor": {
        "id": "CGEX:Ascrt0001",
        "label": "LMM Hearing Loss"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq031">CGEX:AllFreq031</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA024311">CAR:CA024311</a></p>
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              
         </td>
          <td>
              <p>9</p>
         </td>
          <td>
              <p>66740</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq031"><pre><code>{
  "id": "CGEX:AllFreq031",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "allele": {
    "id": "CAR:CA024311",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll026"
  },
  "alleleCount": 9,
  "alleleNumber": 66740
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq034">CGEX:AllFreq034</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:UsrLabl002">CGEX:UsrLabl002</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA123287">CAR:CA123287</a></p>
         </td>
          <td>
              <p><a href="details/details.html#GNOMAD:nfe">GNOMAD:nfe</a></p> (Non-Finnish European)
         </td>
          <td>
              <p><a href="details/details.html#CGEX:Ascrt0002">CGEX:Ascrt0002</a></p> (GENIUS T2D Cases)
         </td>
          <td>
              
         </td>
          <td>
              <p>1198</p>
         </td>
          <td>
              <p>7344</p>
         </td>
          <td>
              <p>3672</p>
         </td>
          <td>
              <p>113</p>
         </td>
          <td>
              <p>972</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq034"><pre><code>{
  "id": "CGEX:AllFreq034",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "CGEX:Ascrt0002",
    "label": "GENIUS T2D Cases"
  },
  "allele": {
    "id": "CAR:CA123287",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll027"
  },
  "alleleCount": 1198,
  "alleleNumber": 7344,
  "individualCount": 3672,
  "homozygousAlleleIndividualCount": 113,
  "heterozygousAlleleIndividualCount": 972,
  "population": {
    "id": "GNOMAD:nfe",
    "label": "Non-Finnish European"
  },
  "userLabelDictionary": [
    {
      "id": "CGEX:UsrLabl002",
      "type": "UserLabel",
      "description": "Case group is composed of cases from six cohorts from the GENIUS T2D consortium.  Case status is defined per-cohort.",
      "labelFor": {
        "id": "CGEX:Ascrt0002",
        "label": "GENIUS T2D Cases"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq035">CGEX:AllFreq035</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CGEX:UsrLabl003">CGEX:UsrLabl003</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA123287">CAR:CA123287</a></p>
         </td>
          <td>
              <p><a href="details/details.html#GNOMAD:nfe">GNOMAD:nfe</a></p> (Non-Finnish European)
         </td>
          <td>
              <p><a href="details/details.html#CGEX:Ascrt0003">CGEX:Ascrt0003</a></p> (GENIUS T2D Controls)
         </td>
          <td>
              
         </td>
          <td>
              <p>936</p>
         </td>
          <td>
              <p>5870</p>
         </td>
          <td>
              <p>2935</p>
         </td>
          <td>
              <p>77</p>
         </td>
          <td>
              <p>782</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq035"><pre><code>{
  "id": "CGEX:AllFreq035",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "CGEX:Ascrt0003",
    "label": "GENIUS T2D Controls"
  },
  "allele": {
    "id": "CAR:CA123287",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll027"
  },
  "alleleCount": 936,
  "alleleNumber": 5870,
  "individualCount": 2935,
  "homozygousAlleleIndividualCount": 77,
  "heterozygousAlleleIndividualCount": 782,
  "population": {
    "id": "GNOMAD:nfe",
    "label": "Non-Finnish European"
  },
  "userLabelDictionary": [
    {
      "id": "CGEX:UsrLabl003",
      "type": "UserLabel",
      "description": "Controls group is composed of controls from six cohorts from the GENIUS T2D consortium.  Controls status is defined per-cohort.",
      "labelFor": {
        "id": "CGEX:Ascrt0003",
        "label": "GENIUS T2D Controls"
      }
    }
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq037">CGEX:AllFreq037</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA013536">CAR:CA013536</a></p>
         </td>
          <td>
              <p><a href="details/details.html#GNOMAD:sas">GNOMAD:sas</a></p> (South Asian)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.02</p>
         </td>
          <td>
              <p>31</p>
         </td>
          <td>
              <p>16512</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq037"><pre><code>{
  "id": "CGEX:AllFreq037",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "allele": {
    "id": "CAR:CA013536",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll124"
  },
  "alleleCount": 31,
  "alleleNumber": 16512,
  "alleleFrequency": 0.02,
  "population": {
    "id": "GNOMAD:sas",
    "label": "South Asian"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq038">CGEX:AllFreq038</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090950">CAR:CA090950</a></p>
         </td>
          <td>
              <p><a href="details/details.html#GNOMAD:afr">GNOMAD:afr</a></p> (African/African American)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.02</p>
         </td>
          <td>
              <p>27</p>
         </td>
          <td>
              <p>1384</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq038"><pre><code>{
  "id": "CGEX:AllFreq038",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "allele": {
    "id": "CAR:CA090950",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll098"
  },
  "alleleCount": 27,
  "alleleNumber": 1384,
  "alleleFrequency": 0.02,
  "population": {
    "id": "GNOMAD:afr",
    "label": "African/African American"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq051">CGEX:AllFreq051</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA011797">CAR:CA011797</a></p>
         </td>
          <td>
              <p><a href="details/details.html#GNOMAD:nfe">GNOMAD:nfe</a></p> (Non-Finnish European)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>1.499e-05</p>
         </td>
          <td>
              <p>66706</p>
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq051"><pre><code>{
  "id": "CGEX:AllFreq051",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "GNOMAD:nfe",
    "label": "Non-Finnish European"
  },
  "allele": {
    "id": "CAR:CA011797",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll037"
  },
  "alleleCount": 66706,
  "alleleNumber": 1,
  "alleleFrequency": 1.499e-05,
  "homozygousAlleleIndividualCount": 0
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq052">CGEX:AllFreq052</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA011797">CAR:CA011797</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>8.24e-06</p>
         </td>
          <td>
              <p>121366</p>
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq052"><pre><code>{
  "id": "CGEX:AllFreq052",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA011797",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll037"
  },
  "alleleCount": 121366,
  "alleleNumber": 1,
  "alleleFrequency": 8.24e-06,
  "homozygousAlleleIndividualCount": 0
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq053">CGEX:AllFreq053</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA011797">CAR:CA011797</a></p>
         </td>
          <td>
              <p><a href="details/details.html#GNOMAD:fin">GNOMAD:fin</a></p> (Finnish)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.07771</p>
         </td>
          <td>
              <p>6614</p>
         </td>
          <td>
              <p>514</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>21</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>100.0</p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq053"><pre><code>{
  "id": "CGEX:AllFreq053",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "GNOMAD:fin",
    "label": "Finnish"
  },
  "allele": {
    "id": "CAR:CA011797",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll037"
  },
  "alleleCount": 6614,
  "alleleNumber": 514,
  "alleleFrequency": 0.07771,
  "homozygousAlleleIndividualCount": 21,
  "medianCoverage": 100.0
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq054">CGEX:AllFreq054</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA273141">CAR:CA273141</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.0</p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>11.0</p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq054"><pre><code>{
  "id": "CGEX:AllFreq054",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA273141",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll038"
  },
  "alleleCount": 0,
  "alleleNumber": 0,
  "alleleFrequency": 0.0,
  "medianCoverage": 11.0
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq082">CGEX:AllFreq082</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090919">CAR:CA090919</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.0</p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>11.0</p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq082"><pre><code>{
  "id": "CGEX:AllFreq082",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA090919",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll039"
  },
  "alleleCount": 0,
  "alleleFrequency": 0.0,
  "medianCoverage": 11.0,
  "contribution": [
    {
      "id": "CGEX:Contrib488",
      "type": "Contribution",
      "agent": {
        "id": "CGEX:Agent001",
        "type": "Agent",
        "label": "Harry Stevenson"
      },
      "contributionDate": "6/1/16 2:15 PM",
      "contributionRole": {
        "id": "SEPIO:0000156",
        "label": "curator"
      }
    }
  ],
  "source": [
    "ftp://ftp.broadinstitute.org/pub/ExAC_release/release0.2/"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq101">CGEX:AllFreq101</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA280055">CAR:CA280055</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.0</p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>57.0</p>
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq101"><pre><code>{
  "id": "CGEX:AllFreq101",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA280055",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll131"
  },
  "alleleCount": 0,
  "alleleFrequency": 0.0,
  "medianCoverage": 57.0
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq153">CGEX:AllFreq153</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090891">CAR:CA090891</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.05591</p>
         </td>
          <td>
              <p>6732</p>
         </td>
          <td>
              <p>120402</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>223</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq153"><pre><code>{
  "id": "CGEX:AllFreq153",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA090891",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll081"
  },
  "alleleCount": 6732,
  "alleleNumber": 120402,
  "alleleFrequency": 0.05591,
  "homozygousAlleleIndividualCount": 223
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq154">CGEX:AllFreq154</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA134997">CAR:CA134997</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.04538</p>
         </td>
          <td>
              <p>5507</p>
         </td>
          <td>
              <p>121350</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>768</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq154"><pre><code>{
  "id": "CGEX:AllFreq154",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA134997",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll082"
  },
  "alleleCount": 5507,
  "alleleNumber": 121350,
  "alleleFrequency": 0.04538,
  "homozygousAlleleIndividualCount": 768
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq155">CGEX:AllFreq155</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA134997">CAR:CA134997</a></p>
         </td>
          <td>
              <p><a href="details/details.html#GNOMAD:eas">GNOMAD:eas</a></p> (East Asian)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.2917</p>
         </td>
          <td>
              <p>2518</p>
         </td>
          <td>
              <p>8632</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>405</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq155"><pre><code>{
  "id": "CGEX:AllFreq155",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "GNOMAD:eas",
    "label": "East Asian"
  },
  "allele": {
    "id": "CAR:CA134997",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll082"
  },
  "alleleCount": 2518,
  "alleleNumber": 8632,
  "alleleFrequency": 0.2917,
  "homozygousAlleleIndividualCount": 405
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq157">CGEX:AllFreq157</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501229">CAR:CA501229</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.0001158</p>
         </td>
          <td>
              <p>14</p>
         </td>
          <td>
              <p>121100</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq157"><pre><code>{
  "id": "CGEX:AllFreq157",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA501229",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll083"
  },
  "alleleCount": 14,
  "alleleNumber": 121100,
  "alleleFrequency": 0.0001158,
  "homozygousAlleleIndividualCount": 0
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq158">CGEX:AllFreq158</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA017597">CAR:CA017597</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.0007</p>
         </td>
          <td>
              <p>89</p>
         </td>
          <td>
              <p>121410</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq158"><pre><code>{
  "id": "CGEX:AllFreq158",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA017597",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll084"
  },
  "alleleCount": 89,
  "alleleNumber": 121410,
  "alleleFrequency": 0.0007,
  "homozygousAlleleIndividualCount": 1
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq162">CGEX:AllFreq162</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA090926">CAR:CA090926</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>0.00625</p>
         </td>
          <td>
              <p>545</p>
         </td>
          <td>
              <p>87696</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>206</p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq162"><pre><code>{
  "id": "CGEX:AllFreq162",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA090926",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll085"
  },
  "alleleCount": 545,
  "alleleNumber": 87696,
  "alleleFrequency": 0.00625,
  "homozygousAlleleIndividualCount": 1,
  "hemizygousAlleleIndividualCount": 206
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq166">CGEX:AllFreq166</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA501230">CAR:CA501230</a></p>
         </td>
          <td>
              <p><a href="details/details.html#SEPIO-CG:98001">SEPIO-CG:98001</a></p> (Combined)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>2.4e-05</p>
         </td>
          <td>
              <p>3</p>
         </td>
          <td>
              <p>120870</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq166"><pre><code>{
  "id": "CGEX:AllFreq166",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "SEPIO-CG:98001",
    "label": "Combined"
  },
  "allele": {
    "id": "CAR:CA501230",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll086"
  },
  "alleleCount": 3,
  "alleleNumber": 120870,
  "alleleFrequency": 2.4e-05
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="details/details.html#CGEX:AllFreq370">CGEX:AllFreq370</a></p></td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p><a href="details/details.html#CAR:CA022880">CAR:CA022880</a></p>
         </td>
          <td>
              <p><a href="details/details.html#GNOMAD:nfe">GNOMAD:nfe</a></p> (Non-Finnish European)
         </td>
          <td>
              <p><a href="details/details.html#SEPIO:0000332">SEPIO:0000332</a></p> (ExAC ascertainment method)
         </td>
          <td>
              <p>1.5e-05</p>
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              <p>66564</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="CGEX:AllFreq370"><pre><code>{
  "id": "CGEX:AllFreq370",
  "type": "PopulationAlleleFrequencyStatement",
  "ascertainment": {
    "id": "SEPIO:0000332",
    "label": "ExAC ascertainment method"
  },
  "population": {
    "id": "GNOMAD:nfe",
    "label": "Non-Finnish European"
  },
  "allele": {
    "id": "CAR:CA022880",
    "type": "CanonicalAllele",
    "relatedContextualAllele": "CGEX:CtxAll127"
  },
  "alleleCount": 1,
  "alleleNumber": 66564,
  "alleleFrequency": 1.5e-05,
  "homozygousAlleleIndividualCount": 0
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="../">Model Overview</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../getting_started.html">Getting Started</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../deeper_dive.html">A Deeper Dive</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="../faq.html">FAQ</a></h5>
              </li>
              <li class="list-group-item"><h5><a href="./">Entities</a></h5><ul class='list-unstyled'><li><a href="Agent.html"></a></li><li><a href="AggregateCosegregationStatement.html"></a></li><li><a href="AlleleConservationScoreStatement.html"></a></li><li><a href="AlleleConservationStatement.html"></a></li><li><a href="AlleleFunctionalImpactStatement.html"></a></li><li><a href="AlleleMolecularConsequenceStatement.html"></a></li><li><a href="BenignMissenseVariationRateStatement.html"></a></li><li><a href="CanonicalAllele.html"></a></li><li><a href="CaseControlAlleleFrequencyStatement.html"></a></li><li><a href="ConditionMechanismStatement.html"></a></li><li><a href="ConditionPenetranceStatement.html"></a></li><li><a href="ConditionPrevelanceStatement.html"></a></li><li><a href="ContextualAllele.html"></a></li><li><a href="ContextualAlleleName.html"></a></li><li><a href="Contribution.html"></a></li><li><a href="CriterionAssessment.html"></a></li><li><a href="DomainEntity.html"></a></li><li><a href="Entity.html"></a></li><li><a href="EvidenceLine.html"></a></li><li><a href="FamilyConditionStatement.html"></a></li><li><a href="FamilyCosegregationStatement.html"></a></li><li><a href="GeneticCondition.html"></a></li><li><a href="Genotype.html"></a></li><li><a href="Haplotype.html"></a></li><li><a href="IdentifierSystem.html"></a></li><li><a href="InSilicoPredictionScoreStatement.html"></a></li><li><a href="InSilicoPredictionStatement.html"></a></li><li><a href="IndividualAlleleInheritanceStatement.html"></a></li><li><a href="IndividualConditionStatement.html"></a></li><li><a href="IndividualGenotypeStatement.html"></a></li><li><a href="LocusHeterogeneityStatement.html"></a></li><li><a href="NullAlleleStatement.html"></a></li><li class="active"><a href="PopulationAlleleFrequencyStatement.html"></a></li><li><a href="Position.html"></a></li><li><a href="ReferenceCoordinate.html"></a></li><li><a href="RegionAllelesStatement.html"></a></li><li><a href="RegionTypeStatement.html"></a></li><li><a href="SequenceLocation.html"></a></li><li><a href="Statement.html"></a></li><li><a href="UserLabel.html"></a></li><li><a href="ValueSet.html"></a></li><li><a href="VariantPathogenicityInterpretation.html"></a></li><li><a href="VariantPathogenicityInterpretationCriterion.html"></a></li><li><a href="VariantPathogenicityInterpretationGuideline.html"></a></li></ul></li>
              <li class='list-group-item'><h5><a href="../terminologies/">Terminologies</a></h5></li>
              <li class="list-group-item">
                <h5><a href="../tools.html">Tools</a></h5>
              </li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          © 2018 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../javascripts/jquery.js"></script><script src="../javascripts/bootstrap.js"></script><script src="../javascripts/jquery.jsonview.js"></script><script src="../javascripts/all.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
