<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

    <link href="../../../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/model/entities/evidence/condition_mechanism.html">0.1</a></li>
                  <li><a href="/development/tech/model/entities/evidence/condition_mechanism.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../../../../">Interpretation Model</a></li>
<li><a href="../../../">Technical Reference</a></li>
<li><a href="../../">Model</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
          <ul class="nav nav-sidebar">
            <li class='nav-sidebar-title'>
              <img width="438" height="108" alt="ClinGen Documentation" class="img-responsive" src="../../../../images/clingen-doc-logo.png" />
            </li>

            <li><a href="../../../../welcome/">Welcome</a></li>
            <li><a href="../../../../user/">User Guide</a></li>
            <li class="active"><a href="../../../">Technical Reference</a></li><ul><li><a href="../../../api/">API</a></li><li><a href="../../../examples/">Examples</a></li><li class="active"><a href="../../">Model</a></li><ul><li><a href="../../activities/">Activity</a></li><li><a href="../../agents/">Agent</a></li><li class="active"><a href="../">Entity</a></li><ul><li><a href="../criterion_assessment.html">CriterionAssessment</a></li><li class="active"><a href="./">Data</a></li><ul><li><a href="aggregation_segregation_data.html">AggregateSegregationData</a></li><li><a href="benign_missense_variation_rate.html">BenignMissenseVariationRate</a></li><li><a href="case_control.html">CaseControl</a></li><li><a href="condition_etiology.html">ConditionEtiology</a></li><li><a href="condition_inheritance.html">ConditionInheritance</a></li><li class="active"><a href="condition_mechanism.html">ConditionMechanism</a></li><li><a href="condition_penetrance.html">ConditionPenetrance</a></li><li><a href="condition_prevelance.html">ConditionPrevelance</a></li><li><a href="conservation.html">Conservation</a></li><li><a href="de_novo_allele.html">DeNovoAllele</a></li><li><a href="family_history.html">FamilyHistory</a></li><li><a href="family_segregation_data.html">FamilySegregationData</a></li><li><a href="functional_data.html">FunctionalData</a></li><li><a href="group_allele_frequency.html">GroupAlleleFrequency</a></li><li><a href="in_silico_prediction.html">InSilicoPrediction</a></li><li><a href="individual_allele.html">IndividualAllele</a></li><li><a href="individual_condition.html">IndividualCondition</a></li><li><a href="mendelian_disease_variant_interpretation.html">MendelianDiseaseVariantPathogenicityInterpretation</a></li><li><a href="molecular_consequence.html">MolecularConsequence</a></li><li><a href="population_allele_frequency.html">PopulationAlleleFrequency</a></li><li><a href="region_annotation.html">RegionAnnotation</a></li><li><a href="region_contains_allele.html">RegionContainsAllele</a></li></ul><li><a href="../evidence_data-source.html">DataSource</a></li><li><a href="../publication_status.html">PublicationStatus</a></li><li><a href="../variant_interpretation.html">VariantInterpretation</a></li></ul><li><a href="../../secondary/">Secondary Types</a></li></ul><li><a href="../../../value_sets/">Value Sets</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>ConditionMechanism</h1>
            <p>Qualitative information about the types of alleles that cause a condition</p>

                <p>Descended from Data</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>Some conditions are differentially driven by particular class of alleles, such as nonsense variants within a given gene.  ConditionMechanism represents such statements.  The particular class of alleles that cause disease is listed in the mechanism attribute, and the strength of the association may be one of four values:</p>

<p>1) primary: alleles of the given class account for most cases
2) common: alleles of the given class account for many, but not all, cases
3) established: some alleles of the class cause the condition
4) suggested: it is unknown whether any alleles of the class cause the condition, though there is some evidence to suggest that they do.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>condition</td>
                <td>Condition</td>
                <td>1..1</td>
                <td>The condition whose mechanism is being described</td>
            </tr>
            <tr>
                <td>gene</td>
                <td>Gene</td>
                <td>1..1</td>
                <td>Gene in which the given variants occur</td>
            </tr>
            <tr>
                <td>mechanism</td>
                <td>CodeableConcept</td>
                <td>1..1</td>
                <td>SO term for a class of variants</td>
            </tr>
            <tr>
                <td>established</td>
                <td>boolean</td>
                <td>0..1</td>
                <td>True if the class of variants are known to cause the condition</td>
            </tr>
            <tr>
                <td>primary</td>
                <td>boolean</td>
                <td>0..1</td>
                <td>True if the class of variants are the main causes of the condition</td>
            </tr>
            <tr>
                <td>suggested</td>
                <td>boolean</td>
                <td>0..1</td>
                <td>True if the class of variants has been suggested to cause the condition</td>
            </tr>
            <tr>
                <td>common</td>
                <td>boolean</td>
                <td>0..1</td>
                <td>Whether the mechanism is common cause of the condition</td>
            </tr>
            </tbody>
        </table>

    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>wasGeneratedBy</td>
                    <td>Activity</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>The activity that produced this entity</td>
                </tr>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td>Entity</td>
                    <td>A human-readable description of the entity</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <div id="example_view"></div>
    <script>
window.addEventListener('load', function () {
    var view = $("#example_view").JSONView(
          {"CondMech002":{"cg:id":"CondMech002","cg:type":"ConditionMechanism","explanation":"HNRNPU is not associated with any conditions, and therefore LOF is not an established mechanism of any condition for this gene.","mechanism":"SO:0002054","established":true,"gene":{"cg:id":"G001","cg:type":"Gene","symbol":"HNRNPU"}},"CondMech004":{"cg:id":"CondMech004","cg:type":"ConditionMechanism","explanation":"This is so well-known that it does not require a citation.","mechanism":"SO:0002054","established":true,"gene":{"cg:id":"G002","cg:type":"Gene","symbol":"BRCA2"},"condition":{"cg:id":"Cond001","cg:type":"Condition","name":"Breast Ovarian Cancer","code":"D061325"}},"CondMech007":{"cg:id":"CondMech007","cg:type":"ConditionMechanism","mechanism":"SO:0002054","established":true,"gene":{"cg:id":"G003","cg:type":"Gene","symbol":"TTN"},"condition":{"cg:id":"Cond002","cg:type":"Condition","name":"Dilated Cardiomyopathy","code":"217604"}},"CondMech118":{"cg:id":"CondMech118","cg:type":"ConditionMechanism","explanation":"Of 178 pathogenic  alleles in ClinVar, 149 are missense.","mechanism":"SO:0001583","common":true,"gene":{"cg:id":"G008","cg:type":"Gene","symbol":"MYH7"}},"CondMech199":{"cg:id":"CondMech199","cg:type":"ConditionMechanism","explanation":"ClinVar shows 421 variants in BRCA2 that are pathogenic for HBOC. 402 of these (~95%) are frameshift, nonsense, or splice-site variants.","mechanism":"SO:0002054","primary":true,"gene":{"cg:id":"G002","cg:type":"Gene","symbol":"BRCA2"},"condition":{"cg:id":"Cond034","cg:type":"Condition","name":"macrocephaly","code":"0000256"}},"CondMech200":{"cg:id":"CondMech200","cg:type":"ConditionMechanism","explanation":"ClinVar shows 61 variants in BRCA2 that are pathogenic for Fanconi Anemia. 55 of these (~90%) are frameshift, nonsense, or splice-site variants.","mechanism":"SO:0002054","primary":true,"gene":{"cg:id":"G002","cg:type":"Gene","symbol":"BRCA2"},"condition":{"cg:id":"Cond035","cg:type":"Condition","name":"Familial Hypercholesterolemia","code":"D006938"}},"CondMech202":{"cg:id":"CondMech202","cg:type":"ConditionMechanism","explanation":"shown in both humans and mouse models","mechanism":"SO:0002054","primary":true,"gene":{"cg:id":"G048","cg:type":"Gene","symbol":"LOXHD1"},"condition":{"cg:id":"Cond010","cg:type":"Condition","name":"Hearing Loss","code":"87884"}},"CondMech204":{"cg:id":"CondMech204","cg:type":"ConditionMechanism","mechanism":"SO:0002054","primary":true,"gene":{"cg:id":"G003","cg:type":"Gene","symbol":"TTN"},"condition":{"cg:id":"Cond002","cg:type":"Condition","name":"Dilated Cardiomyopathy","code":"217604"}},"CondMech205":{"cg:id":"CondMech205","cg:type":"ConditionMechanism","explanation":"Truncating variants in TTN are associated with DCM. However, if missense variants are located in the A-band, impact the A-band, or are in a highly expressed region, they can also be associated with DCM.","mechanism":"SO:0001583","established":true,"primary":true,"gene":{"cg:id":"G003","cg:type":"Gene","symbol":"TTN"},"condition":{"cg:id":"Cond002","cg:type":"Condition","name":"Dilated Cardiomyopathy","code":"217604"}}},
          { collapsed: true }
    )

    var fix_ellipses = function () {
        $('#example_view ul.obj li a:contains("cg:id")').parent().parent().parent().children('.ellipsis').each(function () {
            $(this).html('<em>' + $(this).next('ul').find('a:contains("cg:id")').next('.string')[0].innerText + ', …</em>')
        })

        $('#example_view ul.array').each(function () {
            $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
        })
    }
    fix_ellipses()
    $('#example_view .collapser').click(fix_ellipses)
    $('#example_view .collapser').next('a.prop').click(fix_ellipses)
})
    </script>


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../../../javascripts/jquery.js"></script><script src="../../../../javascripts/bootstrap.js"></script><script src="../../../../javascripts/jquery.jsonview.js"></script>

  </body>
</html>
