<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      Deeper Dive into the Model
    </title>

  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="stylesheets/bootstrap.css" rel="stylesheet" /><link href="stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="stylesheets/brand2.css" rel="stylesheet" /><link href="stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">
    
      <div class="container-fluid hero-background">
      <div class="container">
        <div class="row">
          <nav class="navbar navbar-default">
            <div class="navbar-header">
              <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
              </button>
<a class="navbar-brand" href="/">                <img width="240px" alt="ClinGen Data Model WG Documentation" class="img-responsive" src="images/clingen-doc-logo.png" />
</a>            </div>
            <div id="navbar" class="navbar-collapse collapse">
              <ul class="nav navbar-nav navbar-right">
                <li>
                  <a href="/"><i class="glyphicon glyphicon-home"></i> Home</a>
                </li>
                    <li class="allele" >
                      <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span> Allele Model</a></li>
                    </li>
                    <li class="interpretation" >
                      <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span> Interpretation Model</a></li>
                    </li>
                <li>
                    <a target="_blank" href="http://www.clinicalgenome.org"><i class="glyphicon glyphicon-new-window"></i> About ClinGen</a>
                </li>
              </ul>
            </div>
        </nav>
          </div>

        <div class="row">
          <div class="col-sm-12">
              <h1 class="header">Deeper Dive into the Model</h1>
              <ol class="breadcrumb bg-none margin-none">
                <li><a href="./">Interpretation Model</a></li>
<li><a href="deeper_dive.html">Deeper Dive into the Model</a></li>

              </ol>
                <blockquote><p>A more detailed look at how the Variant Pathogenicity Interpretation Model is designed on top of the SEPIO model.</p>
                </blockquote>
          </div>
        </div>
      </div>
    </div>
    <div class="container">
      <div class="row">
        <div class="col-sm-8 col-xs-12 col-md-9 col-lg-10">
          <article>
            <h2 id="introduction">Introduction</h2>

<p>An interpretation is the outcome of structured reasoning applied to evidence.   Interpretations may be made about many kinds of entities using many kinds of evidence, and different formalized reasoning strategies.  We've seen in <a href="getting_started.html">Getting Started</a> how a variant pathogenicity interpretation can be structured as a chain of conclusions based on evidence, including possibly other supported conclusions.  We believe that this is a general structure for the interpretation of evidence, and we want our model to reflect this generality, so that we will be able to apply the same basic model to a wide variety of situations such as gene-disease curation, gene actionability, interpretation of somatic variants, and so on.</p>

<h2 id="sepio">SEPIO</h2>

<p>The interpretation model requires a structure describing the relationship between a conclusion and the evidence that led to this conclusion;  the Scientific Evidence and Provenance Information Ontology (<a href="./sepio.html">SEPIO</a>) provides such a structure.  In SEPIO, an <a target="sepioassertion" href="https://github.com/monarch-initiative/SEPIO-ontology/wiki/Assertion">assertion</a> is the conclusion drawn from reasoning about <a target="sepioevidenceitem" href="https://github.com/monarch-initiative/SEPIO-ontology/wiki/Evidence-Item">Evidence Items</a>.   This information can be either data, like an allele frequency, or a prior assertion.</p>

<p><img alt="Interpretation Figure 1" src="images/SEPIO_UML.jpg" /></p>

<p>This ability for one SEPIO assertion to support another is integral to our use of SEPIO.   A VariantInterpretation is a SEPIO assertion, and it is supported by a CriteriaAssessment, which is also a ClinVar assertion. This CriteriaAssessment is, in turn, supported by measured data.  In other words, the data is used to create an intermediate assertion: a CriteriaAssessment saying that, for example, BA1 is satisfied for this variant.   This CriteriaAssessment (and potentially other such CriteriaAssessments) are then used to create the final VariantInterpretation in a second cycle of the basic SEPIO model.</p>

<h2 id="evidence-lines">Evidence Lines</h2>

<p>In the ClinGen interpretation model, as well as in SEPIO, assertions are not directly linked to their underlying evidence.  Rather, this link is mediated by an EvidenceLine node.  These nodes serve two purposes: First, they are structural nodes used to combine multiple individually-coded evidence statements that should be considered together.  Second, properties of EvidenceLines can be used for values that are not specific to the assertion or the supporting data, but to the relationship between them.</p>

<p>In the ClinGen interpretation model,  one such property is the strength with which a CriteriaAssessment supports an Interpretation.  When an expert applies a criteria, they may find that a particular criteria is satisfied by a given set of data.   The particular criteria (such as PS1 or BP2) has a default strength describing how much weight should be given to this finding in creating the overall interpretation.  However, the expert analyst may override the default and find that in the case of this interpretation, this assessment may be judged to have increased or decreased strength.  This strength, then, is a property of the EvidenceLine, because it is a property of how much the assessment is used in the context of this specific interpretation.</p>

<h2 id="contributions">Contributions</h2>

<p>In addition to capturing the structure of interpretation arguments, the model captures granular statements about the provenance of particular entities.   Interpretations, Assessments, EvidenceLines, and Statements can each be tagged with a Contribution, defining <strong>when</strong> the entity was created it and <strong>who</strong> created it, along with the <strong>role</strong> in which they were acting.</p>

<p>With this approach, multiple agents, both human and computational, can be noted as contributing to a specific element in a document, and different agents can be assigned as contributing to different precisely defined aspects of creating an interpretation.</p>

<h2 id="model-structure">Model Structure</h2>

<p>The following diagram displays the structure of a variant interpretation message.  An interpretation is essentially two cycles of the base SEPIO model.  First, an interpretation is supported by Criteria Assessments. Second, each Criteria Assessment is supported by Statements (usually Evidence Items).</p>

<p><img alt="Interpretation Figure 2" src="images/CG_UML.jpg" /></p>

<p>At each stage, the conclusions and evidence lines can be tagged with Contributions tracking provenance.  Furthermore, each assertion (the interpretation and the assessment) can both indicate the pre-specified rule by which they were created.  Information about the strength of support that evidence lends to a conclusion is attached to the evidence line between them.</p>

<h2 id="json-ld-and-sepio-mapping">JSON-LD and SEPIO Mapping</h2>

<p>The native encoding of ClinGen interpretations is JSON-LD.  In particular, instances of classes in the interpretation model are expressed as JSON objects with types matching the name of the class.  Property names in the model appear in the JSON-LD serialization as keys.   In JSON-LD, a context file specifies IRIs for types and properties so that they can be unambiguously interpreted.</p>

<p>Many terms needed for variant pathogenicity interpretations were previously defined SEPIO. Variant pathogenicity interpretations also require specialized data elements related to specific types of entities (Alleles, Genes) or evidence (Segregation, Allele Frequencies).  In addition, ClinGen interpretations define new subclasses on SEPIO classes, such as distinguishing VariantInterpretations from CriteriaAssessments.  Each is a SEPIO interpretation, but with different properties. These terms do not make sense to add to the core SEPIO product, but have been added to ClinGen-specific extensions within the SEPIO project which are jointly maintained by the Monarch and ClinGen projects.</p>

<p>For instance, the lines in the <a href="http://dataexchange.clinicalgenome.org/interpretation/json/context">ClinGen interpretation context</a></p>

<p><code>"VariantInterpretation": {
   "@id": "SEPIO:0000190"
},
</code>
indicate that a VariantInterpretation in a ClinGen JSON-LD document is the same entity as the type defined by the id SEPIO:0000190. That SEPIO id is defined in the <a target="sepio" href="https://github.com/monarch-initiative/SEPIO-ontology/blob/master/src/ontology/extensions/sepio-clingen.owl">ClinGen-specific SEPIO extension here</a> as a variant pathogenicity interpretation, which is a subclass of SEPIO:0000001.  SEPIO:0000001 can be found in the <a target="sepio" href="https://raw.githubusercontent.com/monarch-initiative/SEPIO-ontology/master/src/ontology/sepio.owl">main SEPIO OWL file</a>, where it is defined as an <a target="sepio" href="http://github.com/monarch-initiative/SEPIO-ontology/wiki/Assertion">Assertion</a> in SEPIO.</p>

<h2 id="further-information">Further Information</h2>

<p>This document has described the connection to the SEPIO model and the detailed structure of a message.   For details of the individual classes see the <a href="/entities">Entities</a>.  For implementation topics, see the <a href="faq.html">FAQ</a>.</p>

          </article>
        </div>
        <div class="col-sm-4 col-md-3 col-lg-2">
          <div class="list-group sidenav">
            <ul class="list-unstyled">
              <li class="list-group-item">
                <h5><a href="./">Model Overview</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="getting_started.html">Getting Started</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="deeper_dive.html">A Deeper Dive</a></h5>
              </li>
              <li class="list-group-item">
                <h5><a href="faq.html">FAQ</a></h5>
              </li>
              <li class='list-group-item'><h5><a href="entities/">Entities</a></h5></li>
              <li class='list-group-item'><h5><a href="terminologies/">Terminologies</a></h5></li>
              <li class="list-group-item">
                <h5><a href="projects.html">Projects & Tools</a></h5>
              </li>
              <!-- Old style.  We might still need the model here? -->
              <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div id="footer" class="container  background-trans padding-top-xl">
      <div class="row">
        <hr />
        <div class="col-md-col-sm-12 text-center"> 
          Â© 2018 <a href="/" title="ClinGen" class="">ClinGen</a> - All rights reserved
        </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="javascripts/jquery.js"></script><script src="javascripts/bootstrap.js"></script><script src="javascripts/jquery.jsonview.js"></script><script src="javascripts/all.js"></script>
  <script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
