<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/information/conservation.html">0.1</a></li>
                  <li><a href="/development/tech/information/conservation.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../../">Interpretation Model</a></li>
<li><a href="../">Technical Reference and Examples</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
         <div class='nav-sidebar nav-sidebar-title'>
<a href="https://clinicalgenome.org">            	<img width="80%" height="80%" alt="ClinGen Documentation" class="img-responsive" src="../../images/clingen-doc-logo.png" />
</a>        	</div>
          <ul class="nav nav-sidebar">
            
            <li class="active">
              <a href="../../">Welcome</a>
            </li>
            <li><a href="../../user/">User Guide</a></li>
            <li class="active"><a href="../">Technical Reference</a></li><ul><li><a href="../criterion_assessment.html">CriterionAssessment</a></li><li><a href="../evidence_line.html">EvidenceLine</a></li><li class="active"><a href="./">Information</a></li><ul><li><a href="agent.html">Agent</a></li><li><a href="aggregation_segregation_data.html">AggregateSegregationData</a></li><li><a href="allele_frequency.html">AlleleFrequency</a></li><li><a href="benign_missense_variation_rate.html">BenignMissenseVariationRate</a></li><li><a href="case_control.html">CaseControl</a></li><li><a href="condition_etiology.html">ConditionEtiology</a></li><li><a href="condition_mechanism.html">ConditionMechanism</a></li><li><a href="condition_penetrance.html">ConditionPenetrance</a></li><li><a href="condition_prevelance.html">ConditionPrevelance</a></li><li class="active"><a href="conservation.html">Conservation</a></li><li><a href="criterion.html">Criterion</a></li><li><a href="de_novo_allele.html">DeNovoAllele</a></li><li><a href="family_history.html">FamilyHistory</a></li><li><a href="family_segregation_data.html">FamilySegregationData</a></li><li><a href="functional_data.html">FunctionalData</a></li><li><a href="guideline.html">AssertionMethod</a></li><li><a href="guideline_algorithm.html">ScoringAlgorithm</a></li><li><a href="in_silico_prediction.html">InSilicoPrediction</a></li><li><a href="individual_allele.html">IndividualAllele</a></li><li><a href="individual_condition.html">IndividualCondition</a></li><li><a href="mendelian_condition.html">MendelianCondition</a></li><li><a href="molecular_consequence.html">MolecularConsequence</a></li><li><a href="region_annotation.html">RegionAnnotation</a></li></ul><li><a href="../value_set.html">ValueSet</a></li><li><a href="../variant_interpretation.html">VariantInterpretation</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>Conservation</h1>
            <p>Characterizes whether the allele is fixed across species</p>

                <p>Descended from Information</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>Conservation represents the degree to which an allele is the same (conserved) across different species.   If an allele is the same in many species, it often represents an important functional portion of a gene; variations in the allele may therefore tend to be more pathogenic.</p>

<p>Most conservation calculations are given on a computational score based on a set of multiple sequence alignments across species. To properly describe a conservation data element, it is important to include both the method used and the score that the method provided.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>allele</td>
                <td>CanonicalAllele</td>
                <td>0..1</td>
                <td>The allele for which conservation values are given</td>
            </tr>
            <tr>
                <td>algorithm</td>
                <td>String</td>
                <td>0..1</td>
                <td>Method or program used to calculate conservation</td>
            </tr>
            <tr>
                <td>score</td>
                <td>float</td>
                <td>0..1</td>
                <td>Numerical score for the conservation of the allele</td>
            </tr>
            <tr>
                <td>conserved</td>
                <td>boolean</td>
                <td>0..1</td>
                <td>Whether the allele is conserved</td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>source</td>
                    <td>String</td>
                    <td>0..*</td>
                    <td></td>
                    <td></td>
                </tr>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A human-readable description of the entity</td>
                </tr>
                <tr>
                    <td>contribution</td>
                    <td>Contribution</td>
                    <td>0..*</td>
                    <td></td>
                    <td>The contributions (agent, time and role) made to this entity</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>explanation</th>
        <th>allele</th>
        <th>algorithm</th>
        <th>score</th>
        <th>conserved</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../details/details.html#Conserv071">Conserv071</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll118">CanAll118</a></p>
         </td>
          <td>
              <p>phyloP</p>
         </td>
          <td>
              <p>-0.76</p>
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="Conserv071"><pre><code>{
  "id": "Conserv071",
  "type": "Conservation",
  "allele": {
    "id": "CanAll118",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll124",
    "identifier": "http://reg.genome.network/allele/CA013536"
  },
  "conserved": true,
  "algorithm": "phyloP",
  "score": -0.76
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Conserv072">Conserv072</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll118">CanAll118</a></p>
         </td>
          <td>
              <p>PhastCons</p>
         </td>
          <td>
              <p>0.63</p>
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="Conserv072"><pre><code>{
  "id": "Conserv072",
  "type": "Conservation",
  "allele": {
    "id": "CanAll118",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll124",
    "identifier": "http://reg.genome.network/allele/CA013536"
  },
  "conserved": true,
  "algorithm": "PhastCons",
  "score": 0.63
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Conserv259">Conserv259</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll117">CanAll117</a></p>
         </td>
          <td>
              <p>phyloP</p>
         </td>
          <td>
              <p>-2.297</p>
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="Conserv259"><pre><code>{
  "id": "Conserv259",
  "type": "Conservation",
  "allele": {
    "id": "CanAll117",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll123",
    "identifier": "http://reg.genome.network/allele/CA501232"
  },
  "conserved": true,
  "algorithm": "phyloP",
  "score": -2.297
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Conserv260">Conserv260</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll117">CanAll117</a></p>
         </td>
          <td>
              <p>PhastCons</p>
         </td>
          <td>
              <p>0.039</p>
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="Conserv260"><pre><code>{
  "id": "Conserv260",
  "type": "Conservation",
  "allele": {
    "id": "CanAll117",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll123",
    "identifier": "http://reg.genome.network/allele/CA501232"
  },
  "conserved": true,
  "algorithm": "PhastCons",
  "score": 0.039
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Conserv263">Conserv263</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll118">CanAll118</a></p>
         </td>
          <td>
              <p>phyloP</p>
         </td>
          <td>
              <p>-0.76</p>
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="Conserv263"><pre><code>{
  "id": "Conserv263",
  "type": "Conservation",
  "allele": {
    "id": "CanAll118",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll124",
    "identifier": "http://reg.genome.network/allele/CA013536"
  },
  "conserved": true,
  "algorithm": "phyloP",
  "score": -0.76
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Conserv264">Conserv264</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll118">CanAll118</a></p>
         </td>
          <td>
              <p>PhastCons</p>
         </td>
          <td>
              <p>0.63</p>
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="Conserv264"><pre><code>{
  "id": "Conserv264",
  "type": "Conservation",
  "allele": {
    "id": "CanAll118",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll124",
    "identifier": "http://reg.genome.network/allele/CA013536"
  },
  "conserved": true,
  "algorithm": "PhastCons",
  "score": 0.63
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Conserv267">Conserv267</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll119">CanAll119</a></p>
         </td>
          <td>
              <p>phyloP</p>
         </td>
          <td>
              <p>0.45</p>
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="Conserv267"><pre><code>{
  "id": "Conserv267",
  "type": "Conservation",
  "allele": {
    "id": "CanAll119",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll125",
    "identifier": "http://reg.genome.network/allele/CA177682"
  },
  "conserved": true,
  "algorithm": "phyloP",
  "score": 0.45
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Conserv268">Conserv268</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll119">CanAll119</a></p>
         </td>
          <td>
              <p>PhastCons</p>
         </td>
          <td>
              <p>1.0</p>
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="Conserv268"><pre><code>{
  "id": "Conserv268",
  "type": "Conservation",
  "allele": {
    "id": "CanAll119",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll125",
    "identifier": "http://reg.genome.network/allele/CA177682"
  },
  "conserved": true,
  "algorithm": "PhastCons",
  "score": 1.0
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../javascripts/jquery.js"></script><script src="../../javascripts/bootstrap.js"></script><script src="../../javascripts/jquery.jsonview.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
