<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/information/family_segregation_data.html">0.1</a></li>
                  <li><a href="/development/tech/information/family_segregation_data.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../../">Interpretation Model</a></li>
<li><a href="../">Technical Reference and Examples</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
         <div class='nav-sidebar nav-sidebar-title'>
<a href="https://clinicalgenome.org">            	<img width="80%" height="80%" alt="ClinGen Documentation" class="img-responsive" src="../../images/clingen-doc-logo.png" />
</a>        	</div>
          <ul class="nav nav-sidebar">
            
            <li class="active">
              <a href="../../">Welcome</a>
            </li>
            <li><a href="../../user/">User Guide</a></li>
            <li class="active"><a href="../">Technical Reference</a></li><ul><li><a href="../criterion_assessment.html">CriterionAssessment</a></li><li><a href="../evidence_line.html">EvidenceLine</a></li><li class="active"><a href="./">Information</a></li><ul><li><a href="agent.html">Agent</a></li><li><a href="aggregation_segregation_data.html">AggregateSegregationData</a></li><li><a href="allele_frequency.html">AlleleFrequency</a></li><li><a href="benign_missense_variation_rate.html">BenignMissenseVariationRate</a></li><li><a href="case_control.html">CaseControl</a></li><li><a href="condition_etiology.html">ConditionEtiology</a></li><li><a href="condition_mechanism.html">ConditionMechanism</a></li><li><a href="condition_penetrance.html">ConditionPenetrance</a></li><li><a href="condition_prevelance.html">ConditionPrevelance</a></li><li><a href="conservation.html">Conservation</a></li><li><a href="criterion.html">Criterion</a></li><li><a href="de_novo_allele.html">DeNovoAllele</a></li><li><a href="family_history.html">FamilyHistory</a></li><li class="active"><a href="family_segregation_data.html">FamilySegregationData</a></li><li><a href="functional_data.html">FunctionalData</a></li><li><a href="guideline.html">AssertionMethod</a></li><li><a href="guideline_algorithm.html">ScoringAlgorithm</a></li><li><a href="in_silico_prediction.html">InSilicoPrediction</a></li><li><a href="individual_allele.html">IndividualAllele</a></li><li><a href="individual_condition.html">IndividualCondition</a></li><li><a href="mendelian_condition.html">MendelianCondition</a></li><li><a href="molecular_consequence.html">MolecularConsequence</a></li><li><a href="region_annotation.html">RegionAnnotation</a></li></ul><li><a href="../value_set.html">ValueSet</a></li><li><a href="../variant_interpretation.html">VariantInterpretation</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>FamilySegregationData</h1>
            <p>Cosegregation of an allele and a condition in a single family</p>

                <p>Descended from Information</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>The FamilySegregationData element represents co-segregation of an allele and a condition within a particular family.   The segregation can be characterized using a series of aggregate counts, such as the number of individuals that have both the allele and the condition, or those with the allele but not the condition.</p>

<p>In addition, or instead, the specific family details can be provided in pedigree file format.  The contents of a pedigree file are stored as a string in the pedigree attribute, and the columns, genotypeValues, and affectedValues provide the metadata required to fully interpret the coding of the pedigree file, for instance, mapping 0/1 genotype codes in the pedigree file to named alleles.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>canonicalAllele</td>
                <td>CanonicalAllele</td>
                <td>1..1</td>
                <td>Allele being tested for cosegregation</td>
            </tr>
            <tr>
                <td>condition</td>
                <td>MendelianCondition</td>
                <td>1..1</td>
                <td>Condition being tested for cosegregation</td>
            </tr>
            <tr>
                <td>family</td>
                <td>Family</td>
                <td>1..1</td>
                <td>Family in which segregation is being tested</td>
            </tr>
            <tr>
                <td>phenotypePositiveAllelePositive</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of (non-proband) individuals with both the condition and the allele</td>
            </tr>
            <tr>
                <td>phenotypePositiveAlleleNegative</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of (non-proband) individuals with the condition and without the allele</td>
            </tr>
            <tr>
                <td>phenotypeNegativeAllelePositive</td>
                <td>int</td>
                <td>0..1</td>
                <td>Number of (non-proband) individuals without the condition and with the allele</td>
            </tr>
            <tr>
                <td>phenotypeNegativeAlleleNegative</td>
                <td>int</td>
                <td>0..1</td>
                <td>The number of non-proband family members that have neither the allele nor the phenotype</td>
            </tr>
            <tr>
                <td>pedigree</td>
                <td>String</td>
                <td>0..*</td>
                <td>Detailed information about the family structure, affected status, and genotypes in pedigree/linkage format.</td>
            </tr>
            <tr>
                <td>columns</td>
                <td>String</td>
                <td>0..*</td>
                <td>Column names for the pedigree data element</td>
            </tr>
            <tr>
                <td>genotypeValues</td>
                <td>String</td>
                <td>0..*</td>
                <td>Map describing how genotypes are represented in the pedigree</td>
            </tr>
            <tr>
                <td>affectedValues</td>
                <td>String</td>
                <td>0..*</td>
                <td>Map describing how affected status is represented in the pedigree</td>
            </tr>
            <tr>
                <td>anyInconsistentSegregations</td>
                <td>boolean</td>
                <td>0..1</td>
                <td>Whether phenotype positive/allele negative or phenotype negative/allele positive individuals are in the family</td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>source</td>
                    <td>String</td>
                    <td>0..*</td>
                    <td></td>
                    <td></td>
                </tr>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A human-readable description of the entity</td>
                </tr>
                <tr>
                    <td>contribution</td>
                    <td>Contribution</td>
                    <td>0..*</td>
                    <td></td>
                    <td>The contributions (agent, time and role) made to this entity</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>explanation</th>
        <th>canonicalAllele</th>
        <th>condition</th>
        <th>family</th>
        <th>phenotypePositiveAllelePositive</th>
        <th>phenotypePositiveAlleleNegative</th>
        <th>phenotypeNegativeAllelePositive</th>
        <th>phenotypeNegativeAlleleNegative</th>
        <th>pedigree</th>
        <th>columns</th>
        <th>genotypeValues</th>
        <th>affectedValues</th>
        <th>anyInconsistentSegregations</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../details/details.html#FamSegData076">FamSegData076</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll124">CanAll124</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond056">MendCond056</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam011">Fam011</a></p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="FamSegData076"><pre><code>{
  "id": "FamSegData076",
  "type": "FamilySegregationData",
  "condition": {
    "id": "MendCond056",
    "type": "MendelianCondition",
    "explanation": "noonan syndrome, Autosomal Dominant",
    "disease": [
      {
        "id": "Orphanet:648",
        "type": "Coding",
        "code": "648",
        "system": "http://www.orpha.net/",
        "display": "noonan syndrome"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "Coding",
      "code": "0000006",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll124",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll130",
    "identifier": "http://reg.genome.network/allele/CA090930"
  },
  "family": {
    "id": "Fam011",
    "type": "Family",
    "description": "Family #011"
  },
  "phenotypePositiveAllelePositive": 0,
  "phenotypeNegativeAllelePositive": 2,
  "phenotypePositiveAlleleNegative": 0,
  "anyInconsistentSegregations": true
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData103">FamSegData103</a></p></td>
          <td>
              <p>identified in 1 individual with HCM. Segregation analysis determined 5 other members of the family also carried this variant. However, only 3 of those family members had HCM (aged 48-75). The 2 other family members were young and asymptomatic. So from this paper, only count 3 segregations. The two apparently refuting segregations can be ignored due to the age-dependent penetrance of HCM.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll063">CanAll063</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond020">MendCond020</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam001">Fam001</a></p>
         </td>
          <td>
              <p>3</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSegData103"><pre><code>{
  "id": "FamSegData103",
  "type": "FamilySegregationData",
  "explanation": "identified in 1 individual with HCM. Segregation analysis determined 5 other members of the family also carried this variant. However, only 3 of those family members had HCM (aged 48-75). The 2 other family members were young and asymptomatic. So from this paper, only count 3 segregations. The two apparently refuting segregations can be ignored due to the age-dependent penetrance of HCM.",
  "condition": {
    "id": "MendCond020",
    "type": "MendelianCondition",
    "explanation": "hypertrophic cardiomyopathy, Autosomal Dominant",
    "disease": [
      {
        "id": "Orphanet:217569",
        "type": "Coding",
        "code": "217569",
        "system": "http://www.orpha.net/",
        "display": "hypertrophic cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "Coding",
      "code": "0000006",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll063",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll067",
    "identifier": "http://reg.genome.network/allele/CA010136"
  },
  "family": {
    "id": "Fam001",
    "type": "Family",
    "label": "12A",
    "description": "Family #001"
  },
  "phenotypePositiveAllelePositive": 3,
  "phenotypeNegativeAllelePositive": 2,
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent007",
        "type": "Agent",
        "name": "Stefan DiMarino"
      },
      "onDate": "9/13/16",
      "role": {
        "id": "CC383",
        "type": "CodeableConcept",
        "coding": [
          {
            "id": "CG-contributory-role:curator",
            "type": "Coding",
            "code": "curator",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "curator"
          }
        ]
      }
    }
  ],
  "source": [
    "http://www.ncbi.nlm.nih.gov/pubmed/17081393"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData104">FamSegData104</a></p></td>
          <td>
              <p>In LMM internal family, c.1351+1G>A was identified in 1 individual with HCM. Variant also found in individual’s sister - who also had clinical dx of HCM.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll063">CanAll063</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond020">MendCond020</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam002">Fam002</a></p>
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSegData104"><pre><code>{
  "id": "FamSegData104",
  "type": "FamilySegregationData",
  "explanation": "In LMM internal family, c.1351+1G>A was identified in 1 individual with HCM. Variant also found in individual’s sister - who also had clinical dx of HCM.",
  "condition": {
    "id": "MendCond020",
    "type": "MendelianCondition",
    "explanation": "hypertrophic cardiomyopathy, Autosomal Dominant",
    "disease": [
      {
        "id": "Orphanet:217569",
        "type": "Coding",
        "code": "217569",
        "system": "http://www.orpha.net/",
        "display": "hypertrophic cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "Coding",
      "code": "0000006",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll063",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll067",
    "identifier": "http://reg.genome.network/allele/CA010136"
  },
  "family": {
    "id": "Fam002",
    "type": "Family",
    "description": "Family #002"
  },
  "phenotypePositiveAllelePositive": 1,
  "contribution": [
    {
      "type": "Contribution",
      "agent": {
        "id": "Agent001",
        "type": "Agent",
        "name": "Harry Stevenson"
      },
      "onDate": "9/13/16",
      "role": {
        "id": "CC384",
        "type": "CodeableConcept",
        "coding": [
          {
            "id": "CG-contributory-role:curator",
            "type": "Coding",
            "code": "curator",
            "system": "http://clinicalgenome.org/datamodel/contributory-role/",
            "display": "curator"
          }
        ]
      }
    }
  ],
  "source": [
    "http://partners.org/ppm/lmm/gi/variant/12345"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData107">FamSegData107</a></p></td>
          <td>
              <p>proband and 5 affected family members all homozygous for NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly).</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll062">CanAll062</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond021">MendCond021</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam003">Fam003</a></p>
         </td>
          <td>
              <p>5</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSegData107"><pre><code>{
  "id": "FamSegData107",
  "type": "FamilySegregationData",
  "explanation": "proband and 5 affected family members all homozygous for NM_206933.2(USH2A):c.1000C>G (p.Arg334Gly).",
  "condition": {
    "id": "MendCond021",
    "type": "MendelianCondition",
    "explanation": "Usher syndrome, Autosomal Recessive",
    "disease": [
      {
        "id": "Orphanet:886",
        "type": "Coding",
        "code": "886",
        "system": "http://www.orpha.net/",
        "display": "Usher syndrome"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000007",
      "type": "Coding",
      "code": "0000007",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Recessive"
    }
  },
  "canonicalAllele": {
    "id": "CanAll062",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll066",
    "identifier": "http://reg.genome.network/allele/CA143213"
  },
  "family": {
    "id": "Fam003",
    "type": "Family",
    "description": "Family #003"
  },
  "phenotypePositiveAllelePositive": 5
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData108">FamSegData108</a></p></td>
          <td>
              <p>c.1000C>T identified in het state in 1 individual with Usher. Individual compound het with Gly1840Val USH2A variant - confirmed in trans. Gly1840Val called Pathogenic. Individual has 3 affected sibs all of whom are compound het with 1000C>T & Gly1840Val. One additional sib only carried 1000C>T and is unaffected.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll062">CanAll062</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond021">MendCond021</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam004">Fam004</a></p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSegData108"><pre><code>{
  "id": "FamSegData108",
  "type": "FamilySegregationData",
  "explanation": "c.1000C>T identified in het state in 1 individual with Usher. Individual compound het with Gly1840Val USH2A variant - confirmed in trans. Gly1840Val called Pathogenic. Individual has 3 affected sibs all of whom are compound het with 1000C>T & Gly1840Val. One additional sib only carried 1000C>T and is unaffected.",
  "condition": {
    "id": "MendCond021",
    "type": "MendelianCondition",
    "explanation": "Usher syndrome, Autosomal Recessive",
    "disease": [
      {
        "id": "Orphanet:886",
        "type": "Coding",
        "code": "886",
        "system": "http://www.orpha.net/",
        "display": "Usher syndrome"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000007",
      "type": "Coding",
      "code": "0000007",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Recessive"
    }
  },
  "canonicalAllele": {
    "id": "CanAll062",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll066",
    "identifier": "http://reg.genome.network/allele/CA143213"
  },
  "family": {
    "id": "Fam004",
    "type": "Family",
    "description": "Family #004"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData111">FamSegData111</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll064">CanAll064</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond051">MendCond051</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam005">Fam005</a></p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSegData111"><pre><code>{
  "id": "FamSegData111",
  "type": "FamilySegregationData",
  "condition": {
    "id": "MendCond051",
    "type": "MendelianCondition",
    "explanation": "Breast Ovarian Cancer, Autosomal Dominant",
    "name": "Breast Ovarian Cancer, Autosomal Dominant",
    "disease": [
      {
        "id": "MESH:D061325",
        "type": "Coding",
        "code": "D061325",
        "system": "https://meshb.nlm.nih.gov/",
        "display": "hereditary breast ovarian cancer"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "Coding",
      "code": "0000006",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll064",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll068",
    "identifier": "http://reg.genome.network/allele/CA501226"
  },
  "family": {
    "id": "Fam005",
    "type": "Family",
    "description": "Family #005"
  },
  "phenotypePositiveAllelePositive": 2
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData113">FamSegData113</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll065">CanAll065</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond002">MendCond002</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam006">Fam006</a></p>
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>[[Fam006,Fam006.1,0,0,1,2,0/0],
   [Fam006,Fam006.2,0,0,2,1,0/0],
   [Fam006,Fam006.3,Fam006.1,Fam006.2,2,3,0/0],
   [Fam006,Fam006.4,Fam006.1,Fam006.2,1,2,1/2],
   [Fam006,Fam006.5,Fam006.1,Fam006.2,2,1,1/1],
   [Fam006,Fam006.6,Fam006.1,Fam006.2,1,2,1/2]]</p>
         </td>
          <td>
              <p>[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”]</p>
         </td>
          <td>
              <p>{0: “Unknown”,1:”Reference”, 2: CA65}</p>
         </td>
          <td>
              <p>{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}</p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSegData113"><pre><code>{
  "id": "FamSegData113",
  "type": "FamilySegregationData",
  "condition": {
    "id": "MendCond002",
    "type": "MendelianCondition",
    "explanation": "dilated cardiomyopathy, Autosomal Dominant",
    "disease": [
      {
        "id": "Orphanet:217604",
        "type": "Coding",
        "code": "217604",
        "system": "http://www.orpha.net/",
        "display": "dilated cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "Coding",
      "code": "0000006",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll065",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll069",
    "identifier": "http://reg.genome.network/allele/CA006429"
  },
  "family": {
    "id": "Fam006",
    "type": "Family",
    "description": "Family #006"
  },
  "phenotypePositiveAllelePositive": 1,
  "phenotypeNegativeAllelePositive": 1,
  "columns": [
    "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”]"
  ],
  "affectedValues": [
    "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
  ],
  "genotypeValues": [
    "{0: “Unknown”,1:”Reference”, 2: CA65}"
  ],
  "pedigree": [
    "[[Fam006,Fam006.1,0,0,1,2,0/0],\n   [Fam006,Fam006.2,0,0,2,1,0/0],\n   [Fam006,Fam006.3,Fam006.1,Fam006.2,2,3,0/0],\n   [Fam006,Fam006.4,Fam006.1,Fam006.2,1,2,1/2],\n   [Fam006,Fam006.5,Fam006.1,Fam006.2,2,1,1/1],\n   [Fam006,Fam006.6,Fam006.1,Fam006.2,1,2,1/2]]"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData114">FamSegData114</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll065">CanAll065</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond002">MendCond002</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam007">Fam007</a></p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>3</p>
         </td>
          <td>
              
         </td>
          <td>
              <p>[[Fam007, Fam007.1, 0, 0, 1, 3, 0/0, 0],
   [Fam007, Fam007.2, 0, 0, 2, 1, 0/0, 0],
   [Fam007, Fam007.3, 0, 0, 1, 3, 0/0, 0],
   [Fam007, Fam007.4, 0, 0, 2, 1, 0/0, 0],
   [Fam007, Fam007.5, 0, 0, 1, 3, 0/0, 0],
   [Fam007, Fam007.6, Fam007.1, Fam007.2, 2, 3, 1/2, 1],
   [Fam007, Fam007.7,0, 0, 1, 1, 0/0, 0],
   [Fam007, Fam007.8, Fam007.1, Fam007.2, 2, 1, 1/2, 1],
   [Fam007, Fam007.9,Fam007.3, Fam007.4, 1, 1, 0/0, 0],
   [Fam007, Fam007.10, 0, 0, 1, 1, 0/0, 0],
   [Fam007, Fam007.11, Fam007.5, Fam007.6, 2, 2, 1/2, 0],
   [Fam007, Fam007.12, Fam007.5, Fam007.6, 1,2, 0/0, 0],
   [Fam007, Fam007.13, Fam007.5, Fam007.6, 2,1, 1/1,0],
   [Fam007, Fam007.14, Fam007.5, Fam007.6, 1, 1, 1/1, 0],
   [Fam007, Fam007.15, Fam007.7, Fam007.8, 1, 2, 1/2, 0],
   [Fam007, Fam007.16, Fam007.8, Fam007.9, 1, 2, 1/2, 0],
   [Fam007, Fam007.17, Fam007.8, Fam007.9, 1, 2, 0/0,0],
   [Fam007, Fam007.18, Fam007.8, Fam007.9, 1, 2, 0/0, 0],
   [Fam007, Fam007.19, Fam007.10, Fam007.11, 1, 2, 0/0,0],
   [Fam007.20,Fam007.10, Fam007.11, 1,1,1/1,0]]</p>
         </td>
          <td>
              <p>[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”,”ObligateGenotype”]</p>
         </td>
          <td>
              <p>{0: “Unknown”,1:”Reference”, 2: CA65}</p>
         </td>
          <td>
              <p>{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}</p>
         </td>
          <td>
              
         </td>
          <!--
          <td><div class="example_view" id="FamSegData114"><pre><code>{
  "id": "FamSegData114",
  "type": "FamilySegregationData",
  "condition": {
    "id": "MendCond002",
    "type": "MendelianCondition",
    "explanation": "dilated cardiomyopathy, Autosomal Dominant",
    "disease": [
      {
        "id": "Orphanet:217604",
        "type": "Coding",
        "code": "217604",
        "system": "http://www.orpha.net/",
        "display": "dilated cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "Coding",
      "code": "0000006",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll065",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll069",
    "identifier": "http://reg.genome.network/allele/CA006429"
  },
  "family": {
    "id": "Fam007",
    "type": "Family",
    "description": "Family #007"
  },
  "phenotypePositiveAllelePositive": 2,
  "phenotypeNegativeAllelePositive": 3,
  "columns": [
    "[“Family”,”Individual”,”Father”,”Mother”,”Sex”,”Affected”,”Genotype”,”ObligateGenotype”]"
  ],
  "affectedValues": [
    "{1: “Unaffected”, 2: “Affected”, 3: “Any Cardiovascular Abnormality”}"
  ],
  "genotypeValues": [
    "{0: “Unknown”,1:”Reference”, 2: CA65}"
  ],
  "pedigree": [
    "[[Fam007, Fam007.1, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.2, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.3, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.4, 0, 0, 2, 1, 0/0, 0],\n   [Fam007, Fam007.5, 0, 0, 1, 3, 0/0, 0],\n   [Fam007, Fam007.6, Fam007.1, Fam007.2, 2, 3, 1/2, 1],\n   [Fam007, Fam007.7,0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.8, Fam007.1, Fam007.2, 2, 1, 1/2, 1],\n   [Fam007, Fam007.9,Fam007.3, Fam007.4, 1, 1, 0/0, 0],\n   [Fam007, Fam007.10, 0, 0, 1, 1, 0/0, 0],\n   [Fam007, Fam007.11, Fam007.5, Fam007.6, 2, 2, 1/2, 0],\n   [Fam007, Fam007.12, Fam007.5, Fam007.6, 1,2, 0/0, 0],\n   [Fam007, Fam007.13, Fam007.5, Fam007.6, 2,1, 1/1,0],\n   [Fam007, Fam007.14, Fam007.5, Fam007.6, 1, 1, 1/1, 0],\n   [Fam007, Fam007.15, Fam007.7, Fam007.8, 1, 2, 1/2, 0],\n   [Fam007, Fam007.16, Fam007.8, Fam007.9, 1, 2, 1/2, 0],\n   [Fam007, Fam007.17, Fam007.8, Fam007.9, 1, 2, 0/0,0],\n   [Fam007, Fam007.18, Fam007.8, Fam007.9, 1, 2, 0/0, 0],\n   [Fam007, Fam007.19, Fam007.10, Fam007.11, 1, 2, 0/0,0],\n   [Fam007.20,Fam007.10, Fam007.11, 1,1,1/1,0]]"
  ]
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData196">FamSegData196</a></p></td>
          <td>
              <p>observed in one case diagnosed with Familial Hypercholesterolemia, characterized by elevated LDLC levels. Two siblings of the proband also have elevated LDLC levels but neither carry the 313+2T>C variant.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll088">CanAll088</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond035">MendCond035</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam008">Fam008</a></p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="FamSegData196"><pre><code>{
  "id": "FamSegData196",
  "type": "FamilySegregationData",
  "explanation": "observed in one case diagnosed with Familial Hypercholesterolemia, characterized by elevated LDLC levels. Two siblings of the proband also have elevated LDLC levels but neither carry the 313+2T>C variant.",
  "condition": {
    "id": "MendCond035",
    "type": "MendelianCondition",
    "explanation": "Hypercholesterolemia, Familial, Autosomal Dominant",
    "disease": [
      {
        "id": "OMIM:143890",
        "type": "Coding",
        "code": "143890",
        "system": "http://www.omim.org/",
        "display": "Hypercholesterolemia, Familial"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "Coding",
      "code": "0000006",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll088",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll094",
    "identifier": "http://reg.genome.network/allele/CA023690"
  },
  "family": {
    "id": "Fam008",
    "type": "Family",
    "description": "Family #008"
  },
  "phenotypePositiveAllelePositive": 0,
  "phenotypePositiveAlleleNegative": 2,
  "anyInconsistentSegregations": true
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData197">FamSegData197</a></p></td>
          <td>
              <p>The individual with HCM but without the allele is likely a phenocopy explained
by environmental factors (obese, alcohol)</p>
         </td>
          <td>
              <p><a href="../details/details.html#CanAll089">CanAll089</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond065">MendCond065</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam009">Fam009</a></p>
         </td>
          <td>
              <p>3</p>
         </td>
          <td>
              <p>1</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="FamSegData197"><pre><code>{
  "id": "FamSegData197",
  "type": "FamilySegregationData",
  "explanation": "The individual with HCM but without the allele is likely a phenocopy explained\nby environmental factors (obese, alcohol)",
  "condition": {
    "id": "MendCond065",
    "type": "MendelianCondition",
    "explanation": "hypertrophic cardiomyopathy, Autosomal Dominant",
    "disease": [
      {
        "id": "Orphanet:217569",
        "type": "Coding",
        "code": "217569",
        "system": "http://www.orpha.net/",
        "display": "hypertrophic cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "Coding",
      "code": "0000006",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll089",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll095",
    "identifier": "http://reg.genome.network/allele/CA010777"
  },
  "family": {
    "id": "Fam009",
    "type": "Family",
    "description": "Family #009"
  },
  "phenotypePositiveAllelePositive": 3,
  "phenotypePositiveAlleleNegative": 1,
  "anyInconsistentSegregations": true
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#FamSegData271">FamSegData271</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CanAll090">CanAll090</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#MendCond066">MendCond066</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#Fam010">Fam010</a></p>
         </td>
          <td>
              <p>0</p>
         </td>
          <td>
              <p>2</p>
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              
         </td>
          <td>
              <p>true</p>
         </td>
          <!--
          <td><div class="example_view" id="FamSegData271"><pre><code>{
  "id": "FamSegData271",
  "type": "FamilySegregationData",
  "condition": {
    "id": "MendCond066",
    "type": "MendelianCondition",
    "explanation": "hypertrophic cardiomyopathy, Autosomal Dominant",
    "disease": [
      {
        "id": "Orphanet:217569",
        "type": "Coding",
        "code": "217569",
        "system": "http://www.orpha.net/",
        "display": "hypertrophic cardiomyopathy"
      }
    ],
    "modeOfInheritance": {
      "id": "HP:0000006",
      "type": "Coding",
      "code": "0000006",
      "system": "http://human-phenotype-ontology.github.io/",
      "display": "Autosomal Dominant"
    }
  },
  "canonicalAllele": {
    "id": "CanAll090",
    "type": "CanonicalAllele",
    "preferredCtxAllele": "CtxAll096",
    "identifier": "http://reg.genome.network/allele/CA133141"
  },
  "family": {
    "id": "Fam010",
    "type": "Family",
    "description": "Family #010"
  },
  "phenotypePositiveAllelePositive": 0,
  "phenotypePositiveAlleleNegative": 2,
  "anyInconsistentSegregations": true
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../javascripts/jquery.js"></script><script src="../../javascripts/bootstrap.js"></script><script src="../../javascripts/jquery.jsonview.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
