<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <link rel="icon" href="#######">

    <title>
      ClinGen Data Model
    </title>

	<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.css"/>

    <link href="../../stylesheets/bootstrap.css" rel="stylesheet" /><link href="../../stylesheets/bootstrap-theme.css" rel="stylesheet" /><link href="../../stylesheets/brand2.css" rel="stylesheet" /><link href="../../stylesheets/jquery.jsonview.css" rel="stylesheet" />

      <script>
       (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
       })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

       ga('create', 'UA-49947422-3', 'auto');
       ga('send', 'pageview');
      </script>

  </head>

  <body class="documentation">

    <nav class="navbar navbar-inverse">
      <div class="container-fluid">
        <div class="navbar-header col-sm-4 col-md-3">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand " href="#">ClinGen Documentation</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-left">
            <li>
              <a href="/">Home</a>
            </li>
            <li>
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >
                Models
                  ::
                  <span class="glyphicon leaf"></span>Interpretation
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu" role="menu">
                  <li class="allele" >
                    <li class="allele"><a href="/allele"><span class="glyphicon glyphicon-leaf"></span>Allele</a></li>
                  </li>
                  <li class="interpretation" >
                    <li class="interpretation"><a href="/interpretation"><span class="glyphicon glyphicon-star"></span>Interpretation</a></li>
                  </li>
              </ul>
            </li>
          </ul>
          <ul class="nav navbar-nav navbar-right">
            <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" >development release<span class="caret"></span></a>
              <ul class="dropdown-menu" role="menu">
                  <li><a href="/0.1/tech/information/functional_data.html">0.1</a></li>
                  <li><a href="/development/tech/information/functional_data.html">development</a></li>
              </ul>
            </li>
          </ul>
        </div>
      </div>
    </nav>

    <div class='container-fluid bg-lblue padding-left-none padding-right-none'>
      <div class='col-sm-offset-4 col-md-offset-3'>
        <ol class="breadcrumb bg-none margin-none">
          <li><a href="../../">Interpretation Model</a></li>
<li><a href="../">Technical Reference and Examples</a></li>

        </ol>
      </div>
    </div>
    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-4 col-md-3 sidebar">
         <div class='nav-sidebar nav-sidebar-title'>
<a href="https://clinicalgenome.org">            	<img width="80%" height="80%" alt="ClinGen Documentation" class="img-responsive" src="../../images/clingen-doc-logo.png" />
</a>        	</div>
          <ul class="nav nav-sidebar">
            
            <li class="active">
              <a href="../../">Welcome</a>
            </li>
            <li><a href="../../user/">User Guide</a></li>
            <li class="active"><a href="../">Technical Reference</a></li><ul><li><a href="../criterion_assessment.html">CriterionAssessment</a></li><li><a href="../evidence_line.html">EvidenceLine</a></li><li class="active"><a href="./">Information</a></li><ul><li><a href="agent.html">Agent</a></li><li><a href="aggregation_segregation_data.html">AggregateSegregationData</a></li><li><a href="allele_frequency.html">AlleleFrequency</a></li><li><a href="benign_missense_variation_rate.html">BenignMissenseVariationRate</a></li><li><a href="case_control.html">CaseControl</a></li><li><a href="condition_etiology.html">ConditionEtiology</a></li><li><a href="condition_mechanism.html">ConditionMechanism</a></li><li><a href="condition_penetrance.html">ConditionPenetrance</a></li><li><a href="condition_prevelance.html">ConditionPrevelance</a></li><li><a href="conservation.html">Conservation</a></li><li><a href="criterion.html">Criterion</a></li><li><a href="de_novo_allele.html">DeNovoAllele</a></li><li><a href="family_history.html">FamilyHistory</a></li><li><a href="family_segregation_data.html">FamilySegregationData</a></li><li class="active"><a href="functional_data.html">FunctionalData</a></li><li><a href="guideline.html">AssertionMethod</a></li><li><a href="guideline_algorithm.html">ScoringAlgorithm</a></li><li><a href="in_silico_prediction.html">InSilicoPrediction</a></li><li><a href="individual_allele.html">IndividualAllele</a></li><li><a href="individual_condition.html">IndividualCondition</a></li><li><a href="mendelian_condition.html">MendelianCondition</a></li><li><a href="molecular_consequence.html">MolecularConsequence</a></li><li><a href="region_annotation.html">RegionAnnotation</a></li></ul><li><a href="../value_set.html">ValueSet</a></li><li><a href="../variant_interpretation.html">VariantInterpretation</a></li></ul>
            <!-- Old style.  We might still need the model here? -->
            <!--%= link_with_local_index "Entities", "/#{current_page.data.model}/vampi/entities/index.html" %-->
          </ul>
        </div>
        <div class="col-sm-8 col-sm-offset-4 col-md-9 col-md-offset-3 main">
          <article>


            <h1>FunctionalData</h1>
            <p>Data about the function of a gene product</p>

                <p>Descended from Information</p>

            <hr class='col-sm-12' />


            

    <!-- this is primarily the Scope and Usage section -->
    <h2 id="scope-and-usage">Scope and Usage</h2>

<p>FunctionalData comprises any experimental measurement of the function of a gene, usually (but not always) in relation to the presence of an allele.   Many classes of in vivo and in vitro experimental data may be included.  Rather than attempt a full classification of the types of functional data and models for each, this model provides a single type of functional data with several optional elements to tag relevant entities such as genes or alleles.</p>


    <!-- image here if available -->

    <h2>Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
            <tr>
                <td>contextualAllele</td>
                <td>ContextualAllele</td>
                <td>0..1</td>
                <td>Allele defined with respect to the measured transcript</td>
            </tr>
            <tr>
                <td>gene</td>
                <td>Gene</td>
                <td>0..1</td>
                <td>Gene to which the functional data applies</td>
            </tr>
            <tr>
                <td>result</td>
                <td>String</td>
                <td>0..1</td>
                <td>Result of the functional experiment</td>
            </tr>
            <tr>
                <td>dataType</td>
                <td>CodeableConcept</td>
                <td>0..1</td>
                <td>Specific type of functional data collected</td>
            </tr>
            </tbody>
        </table>


    <h2>Inherited Attributes</h2>

        <table class="table table-striped table-bordered" cellspacing="0" width="100%">
            <thead>
            <tr>
                <th>Name</th>
                <th>Type</th>
                <th>Cardinality</th>
                <th>Inherited From</th>
                <th>Description</th>
            </tr>
            </thead>
            <tbody>
                <tr>
                    <td>source</td>
                    <td>String</td>
                    <td>0..*</td>
                    <td></td>
                    <td></td>
                </tr>
                <tr>
                    <td>explanation</td>
                    <td>String</td>
                    <td>0..1</td>
                    <td></td>
                    <td>A human-readable description of the entity</td>
                </tr>
                <tr>
                    <td>contribution</td>
                    <td>Contribution</td>
                    <td>0..*</td>
                    <td></td>
                    <td>The contributions (agent, time and role) made to this entity</td>
                </tr>
            </tbody>
        </table>

    <h2>Examples</h2>
    <table class="datatables table table-striped table-bordered" cellspacing=0 width="100%">
      <thead><tr>
        <th>ID</th>
        <th>explanation</th>
        <th>contextualAllele</th>
        <th>gene</th>
        <th>result</th>
        <th>dataType</th>
      </tr></thead>
      <tbody>
        <tr>
          <td><p><a href="../details/details.html#Func021">Func021</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll021">CtxAll021</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:TNNT2">hgnc:TNNT2</a></p>
         </td>
          <td>
              <p>the mutation lowered the Ca2+  affinity of the reconstituted thin filaments;</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC015">CC015</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func021"><pre><code>{
  "id": "Func021",
  "type": "FunctionalData",
  "result": "the mutation lowered the Ca2+  affinity of the reconstituted thin filaments;",
  "dataType": {
    "id": "CC015",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:binding",
        "type": "Coding",
        "code": "binding",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "Binding assay"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll021",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll018",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll021",
      "identifier": "http://reg.genome.network/allele/CA501233"
    },
    "alleleName": "NP_001001431.1:p.Arg131Trp"
  },
  "gene": {
    "id": "hgnc:TNNT2",
    "type": "Gene",
    "symbol": "TNNT2"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func022">Func022</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll021">CtxAll021</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:TNNT2">hgnc:TNNT2</a></p>
         </td>
          <td>
              <p>reduced Ca2 sensitivity of activation</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC016">CC016</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func022"><pre><code>{
  "id": "Func022",
  "type": "FunctionalData",
  "result": "reduced Ca2 sensitivity of activation",
  "dataType": {
    "id": "CC016",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:atpase",
        "type": "Coding",
        "code": "atpase",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "ATPase assay"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll021",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll018",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll021",
      "identifier": "http://reg.genome.network/allele/CA501233"
    },
    "alleleName": "NP_001001431.1:p.Arg131Trp"
  },
  "gene": {
    "id": "hgnc:TNNT2",
    "type": "Gene",
    "symbol": "TNNT2"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func023">Func023</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll021">CtxAll021</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:TNNT2">hgnc:TNNT2</a></p>
         </td>
          <td>
              <p>reduced Ca2 sensitivity of activation</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC017">CC017</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func023"><pre><code>{
  "id": "Func023",
  "type": "FunctionalData",
  "result": "reduced Ca2 sensitivity of activation",
  "dataType": {
    "id": "CC017",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:motility",
        "type": "Coding",
        "code": "motility",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "motility assay"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll021",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll018",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll021",
      "identifier": "http://reg.genome.network/allele/CA501233"
    },
    "alleleName": "NP_001001431.1:p.Arg131Trp"
  },
  "gene": {
    "id": "hgnc:TNNT2",
    "type": "Gene",
    "symbol": "TNNT2"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func024">Func024</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll021">CtxAll021</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:TNNT2">hgnc:TNNT2</a></p>
         </td>
          <td>
              <p>desensitized Ca2+ binding to the thin filament.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC018">CC018</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func024"><pre><code>{
  "id": "Func024",
  "type": "FunctionalData",
  "result": "desensitized Ca2+ binding to the thin filament.",
  "dataType": {
    "id": "CC018",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:binding",
        "type": "Coding",
        "code": "binding",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "Binding assay"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll021",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll018",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll021",
      "identifier": "http://reg.genome.network/allele/CA501233"
    },
    "alleleName": "NP_001001431.1:p.Arg131Trp"
  },
  "gene": {
    "id": "hgnc:TNNT2",
    "type": "Gene",
    "symbol": "TNNT2"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func025">Func025</a></p></td>
          <td>
              <p>Expression of human SPG7 (with this mutation) in yeast with damaged m-AAA protease did not restore respiratory growth or MrpL32 processing, if the function of AFG3L2 was impaired.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll022">CtxAll022</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:SPG7">hgnc:SPG7</a></p>
         </td>
          <td>
              <p>Functionally Damaging</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC019">CC019</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func025"><pre><code>{
  "id": "Func025",
  "type": "FunctionalData",
  "explanation": "Expression of human SPG7 (with this mutation) in yeast with damaged m-AAA protease did not restore respiratory growth or MrpL32 processing, if the function of AFG3L2 was impaired.",
  "result": "Functionally Damaging",
  "dataType": {
    "id": "CC019",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:yeastcomp",
        "type": "Coding",
        "code": "yeastcomp",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "Yeast complementation assay"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll022",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll019",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll022",
      "identifier": "http://reg.genome.network/allele/CA090884"
    },
    "alleleName": "NM_003119.2 (SPG7):c.1529C>T"
  },
  "gene": {
    "id": "hgnc:SPG7",
    "type": "Gene",
    "symbol": "SPG7"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func026">Func026</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll024">CtxAll024</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:MEK">hgnc:MEK</a></p>
         </td>
          <td>
              <p>abnormal/deceased MEK activtation.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC020">CC020</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func026"><pre><code>{
  "id": "Func026",
  "type": "FunctionalData",
  "result": "abnormal/deceased MEK activtation.",
  "dataType": {
    "id": "CC020",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:activation",
        "type": "Coding",
        "code": "activation",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "Activation Assay"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll024",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll021",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll024",
      "identifier": "http://reg.genome.network/allele/CA093846"
    },
    "alleleName": "NP_004324.2:p.Gly596Val"
  },
  "gene": {
    "id": "hgnc:MEK",
    "type": "Gene",
    "symbol": "MEK"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func027">Func027</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll024">CtxAll024</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:ERK">hgnc:ERK</a></p>
         </td>
          <td>
              <p>abnormal/deceased ERK activtation.</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC021">CC021</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func027"><pre><code>{
  "id": "Func027",
  "type": "FunctionalData",
  "result": "abnormal/deceased ERK activtation.",
  "dataType": {
    "id": "CC021",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:activation",
        "type": "Coding",
        "code": "activation",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "Activation Assay"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll024",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll021",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll024",
      "identifier": "http://reg.genome.network/allele/CA093846"
    },
    "alleleName": "NP_004324.2:p.Gly596Val"
  },
  "gene": {
    "id": "hgnc:ERK",
    "type": "Gene",
    "symbol": "ERK"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func067">Func067</a></p></td>
          <td>
              <p>PMID: 23487764</p>
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll130">CtxAll130</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:SOS1">hgnc:SOS1</a></p>
         </td>
          <td>
              <p>increased ERK activtation</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC055">CC055</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func067"><pre><code>{
  "id": "Func067",
  "type": "FunctionalData",
  "explanation": "PMID: 23487764",
  "result": "increased ERK activtation",
  "dataType": {
    "id": "CC055",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:activation",
        "type": "Coding",
        "code": "activation",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "Activation Assay"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll130",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll124",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll130",
      "identifier": "http://reg.genome.network/allele/CA090930"
    },
    "alleleName": "NM_005633.3(SOS1):c.1010A>G (p.Tyr337Cys)"
  },
  "gene": {
    "id": "hgnc:SOS1",
    "type": "Gene",
    "symbol": "SOS1"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func192">Func192</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll092">CtxAll092</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:SLC26A4">hgnc:SLC26A4</a></p>
         </td>
          <td>
              <p>Unchanged from reference</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC134">CC134</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func192"><pre><code>{
  "id": "Func192",
  "type": "FunctionalData",
  "result": "Unchanged from reference",
  "dataType": {
    "id": "CC134",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:iodeff",
        "type": "Coding",
        "code": "iodeff",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "iodide efflux"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll092",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll086",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll092",
      "identifier": "http://reg.genome.network/allele/CA109417"
    },
    "alleleName": "NP_000432..1:p.Leu117Phe"
  },
  "gene": {
    "id": "hgnc:SLC26A4",
    "type": "Gene",
    "symbol": "SLC26A4"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func193">Func193</a></p></td>
          <td>
              <p>Both wild type and NP_000432..1:p.Leu117Phe localize to the same location</p>
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll092">CtxAll092</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:SLC26A4">hgnc:SLC26A4</a></p>
         </td>
          <td>
              <p>localized to apical membrane</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC135">CC135</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func193"><pre><code>{
  "id": "Func193",
  "type": "FunctionalData",
  "explanation": "Both wild type and NP_000432..1:p.Leu117Phe localize to the same location",
  "result": "localized to apical membrane",
  "dataType": {
    "id": "CC135",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:celloc",
        "type": "Coding",
        "code": "celloc",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "cellular localization"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll092",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll086",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll092",
      "identifier": "http://reg.genome.network/allele/CA109417"
    },
    "alleleName": "NP_000432..1:p.Leu117Phe"
  },
  "gene": {
    "id": "hgnc:SLC26A4",
    "type": "Gene",
    "symbol": "SLC26A4"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func194">Func194</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll093">CtxAll093</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:SPRED1">hgnc:SPRED1</a></p>
         </td>
          <td>
              <p>Unchanged from reference</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC136">CC136</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func194"><pre><code>{
  "id": "Func194",
  "type": "FunctionalData",
  "result": "Unchanged from reference",
  "dataType": {
    "id": "CC136",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:neuritegrowth",
        "type": "Coding",
        "code": "neuritegrowth",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "neurite outgrowth"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll093",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll087",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll093",
      "identifier": "http://reg.genome.network/allele/CA501231"
    },
    "alleleName": "NM_152594.2(SPRED1): c.587C>T"
  },
  "gene": {
    "id": "hgnc:SPRED1",
    "type": "Gene",
    "symbol": "SPRED1"
  }
}
          </code></pre></div></td>
          -->
        </tr>

        <tr>
          <td><p><a href="../details/details.html#Func195">Func195</a></p></td>
          <td>
              
         </td>
          <td>
              <p><a href="../details/details.html#CtxAll093">CtxAll093</a></p>
         </td>
          <td>
              <p><a href="../details/details.html#hgnc:SPRED1">hgnc:SPRED1</a></p>
         </td>
          <td>
              <p>localized to apical membrane</p>
         </td>
          <td>
              <p><a href="../details/details.html#CC137">CC137</a></p>
         </td>
          <!--
          <td><div class="example_view" id="Func195"><pre><code>{
  "id": "Func195",
  "type": "FunctionalData",
  "result": "localized to apical membrane",
  "dataType": {
    "id": "CC137",
    "type": "CodeableConcept",
    "coding": [
      {
        "id": "CG-func-data-type:elk1phosvialucif",
        "type": "Coding",
        "code": "elk1phosvialucif",
        "system": "http://clinicalgenome.org/datamodel/functional-data-type/",
        "display": "Elk1 pohsphorylation via luciferase assay"
      }
    ]
  },
  "contextualAllele": {
    "id": "CtxAll093",
    "type": "ContextualAllele",
    "relatedCanonicalAllele": {
      "id": "CanAll087",
      "type": "CanonicalAllele",
      "preferredCtxAllele": "CtxAll093",
      "identifier": "http://reg.genome.network/allele/CA501231"
    },
    "alleleName": "NM_152594.2(SPRED1): c.587C>T"
  },
  "gene": {
    "id": "hgnc:SPRED1",
    "type": "Gene",
    "symbol": "SPRED1"
  }
}
          </code></pre></div></td>
          -->
        </tr>

      </tbody>
    </table>
    <script>
window.addEventListener('load', function() {
    $('.datatables').DataTable();
});
    </script>
    <!--
    <script>
var fix_ellipses = function ($view) {
    $view.find('ul.obj li a:contains("id")').parent().parent().parent().children('.ellipsis').each(function () {
        $(this).html('<em>' + $(this).next('ul').find('a:contains("id")').next('.string')[0].innerText + ', …</em>')
    })
    $view.find('ul.array').each(function () {
        $(this).parent().children('.ellipsis').html('<em>… [' + $(this).children('li').size() + ']</em>')
    })
}

window.addEventListener('load', function () {
    $('.example_view').each(function() {
        var $view = $(this);
        $view.JSONView($(this).text(), {collapsed: true})
        fix_ellipses($view)
        $view.find('.collapser').click(function (e) { fix_ellipses($view); })
        $view.next('a.prop').click(function (e) { fix_ellipses($view); })
    })

})
    </script>
    -->


          </article>
        </div>
      </div>
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../../javascripts/jquery.js"></script><script src="../../javascripts/bootstrap.js"></script><script src="../../javascripts/jquery.jsonview.js"></script>
	<script type="text/javascript" src="https://cdn.datatables.net/v/bs/dt-1.10.13/fh-3.1.2/datatables.min.js"></script>

  </body>
</html>
